---
author-meta:
- Joshua A. Shapiro
- Candace L. Savonen
- Chante J. Bethell
- Krutika S. Gaonkar
- Run Jin
- Yuankun Zhu
- Miguel A. Brown
- Nhat Duong
- Komal S. Rathi
- Nighat Noureen
- Bo Zhang
- Brian M. Ennis
- Stephanie J. Spielman
- Laura E. Egolf
- Bailey Farrow
- Nicolas Van Kuren
- Tejaswi Koganti
- Shrivats Kannan
- Pichai Raman
- Jennifer Mason
- Daniel P. Miller
- Anna R. Poetsch
- Payal Jain
- Adam A. Kraya
- Allison P. Heath
- Mateusz P. Koptyra
- Shannon Robbins
- Yiran Guo
- Xiaoyan Huang
- Jessica Wong
- Mariarita Santi
- Angela Viaene
- Laura Scolaro
- Angela Waanders
- Derek Hanson
- Hongbo M. Xie
- Siyuan Zheng
- Cassie N. Kline
- Jena V. Lilly
- Philip B. Storm
- Adam C. Resnick
- Casey S. Greene
- Jo Lynne Rokita*
- Jaclyn N. Taroni*
- Children's Brain Tumor Network
- Pacific Pediatric Neurooncology Consortium
bibliography:
- content/manual-references.json
date-meta: '2022-03-31'
header-includes: "<!--\nManubot generated metadata rendered from header-includes-template.html.\nSuggest improvements at https://github.com/manubot/manubot/blob/master/manubot/process/header-includes-template.html\n-->\n<meta name=\"dc.format\" content=\"text/html\" />\n<meta name=\"dc.title\" content=\"An Open Pediatric Brain Tumor Atlas\" />\n<meta name=\"citation_title\" content=\"An Open Pediatric Brain Tumor Atlas\" />\n<meta property=\"og:title\" content=\"An Open Pediatric Brain Tumor Atlas\" />\n<meta property=\"twitter:title\" content=\"An Open Pediatric Brain Tumor Atlas\" />\n<meta name=\"dc.date\" content=\"2022-03-31\" />\n<meta name=\"citation_publication_date\" content=\"2022-03-31\" />\n<meta name=\"dc.language\" content=\"en-US\" />\n<meta name=\"citation_language\" content=\"en-US\" />\n<meta name=\"dc.relation.ispartof\" content=\"Manubot\" />\n<meta name=\"dc.publisher\" content=\"Manubot\" />\n<meta name=\"citation_journal_title\" content=\"Manubot\" />\n<meta name=\"citation_technical_report_institution\" content=\"Manubot\" />\n<meta name=\"citation_author\" content=\"Joshua A. Shapiro\" />\n<meta name=\"citation_author_institution\" content=\"Childhood Cancer Data Lab, Alex&#39;s Lemonade Stand Foundation, Philadelphia, PA, USA\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0002-6224-0347\" />\n<meta name=\"twitter:creator\" content=\"@jashapiro\" />\n<meta name=\"citation_author\" content=\"Candace L. Savonen\" />\n<meta name=\"citation_author_institution\" content=\"Childhood Cancer Data Lab, Alex&#39;s Lemonade Stand Foundation, Philadelphia, PA, USA\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0001-6331-7070\" />\n<meta name=\"twitter:creator\" content=\"@cansavvy\" />\n<meta name=\"citation_author\" content=\"Chante J. Bethell\" />\n<meta name=\"citation_author_institution\" content=\"Childhood Cancer Data Lab, Alex&#39;s Lemonade Stand Foundation, Philadelphia, PA, USA\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0001-9653-8128\" />\n<meta name=\"twitter:creator\" content=\"@cjbethell\" />\n<meta name=\"citation_author\" content=\"Krutika S. Gaonkar\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Division of Neurosurgery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Department of Bioinformatics and Health Informatics, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0003-0838-2405\" />\n<meta name=\"twitter:creator\" content=\"@aggokittu\" />\n<meta name=\"citation_author\" content=\"Run Jin\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Division of Neurosurgery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0002-8958-9266\" />\n<meta name=\"twitter:creator\" content=\"@runjin\" />\n<meta name=\"citation_author\" content=\"Yuankun Zhu\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Division of Neurosurgery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0002-2455-9525\" />\n<meta name=\"twitter:creator\" content=\"@zhuyuankun\" />\n<meta name=\"citation_author\" content=\"Miguel A. Brown\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Division of Neurosurgery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0001-6782-1442\" />\n<meta name=\"twitter:creator\" content=\"@migbro\" />\n<meta name=\"citation_author\" content=\"Nhat Duong\" />\n<meta name=\"citation_author_institution\" content=\"Department of Bioinformatics and Health Informatics, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0003-2852-4263\" />\n<meta name=\"twitter:creator\" content=\"@asiannhat\" />\n<meta name=\"citation_author\" content=\"Komal S. Rathi\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Department of Bioinformatics and Health Informatics, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0001-5534-6904\" />\n<meta name=\"twitter:creator\" content=\"@komalsrathi\" />\n<meta name=\"citation_author\" content=\"Nighat Noureen\" />\n<meta name=\"citation_author_institution\" content=\"Greehey Children&#39;s Cancer Research Institute, UT Health San Antonio\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0001-7495-8201\" />\n<meta name=\"citation_author\" content=\"Bo Zhang\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Division of Neurosurgery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0002-0743-5379\" />\n<meta name=\"citation_author\" content=\"Brian M. Ennis\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Division of Neurosurgery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0002-2653-5009\" />\n<meta name=\"citation_author\" content=\"Stephanie J. Spielman\" />\n<meta name=\"citation_author_institution\" content=\"Childhood Cancer Data Lab, Alex&#39;s Lemonade Stand Foundation, Philadelphia, PA, USA&lt;sup&gt;\u262F&lt;/sup&gt;\" />\n<meta name=\"citation_author_institution\" content=\"Rowan University, Glassboro, NJ, USA\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0002-9090-4788\" />\n<meta name=\"twitter:creator\" content=\"@stephspiel\" />\n<meta name=\"citation_author\" content=\"Laura E. Egolf\" />\n<meta name=\"citation_author_institution\" content=\"Cell and Molecular Biology Graduate Group, Perelman School of Medicine at the University of Pennsylvania\" />\n<meta name=\"citation_author_institution\" content=\"Division of Oncology, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0002-7103-4801\" />\n<meta name=\"twitter:creator\" content=\"@LauraEgolf\" />\n<meta name=\"citation_author\" content=\"Bailey Farrow\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Division of Neurosurgery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0001-6727-6333\" />\n<meta name=\"citation_author\" content=\"Nicolas Van Kuren\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Division of Neurosurgery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0002-7414-9516\" />\n<meta name=\"citation_author\" content=\"Tejaswi Koganti\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Division of Neurosurgery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0002-7733-6480\" />\n<meta name=\"citation_author\" content=\"Shrivats Kannan\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Division of Neurosurgery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0002-1460-920X\" />\n<meta name=\"twitter:creator\" content=\"@kshrivats\" />\n<meta name=\"citation_author\" content=\"Pichai Raman\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Department of Bioinformatics and Health Informatics, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0001-6948-2157\" />\n<meta name=\"twitter:creator\" content=\"@PichaiRaman\" />\n<meta name=\"citation_author\" content=\"Jennifer Mason\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Division of Neurosurgery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"twitter:creator\" content=\"@jenn0307\" />\n<meta name=\"citation_author\" content=\"Daniel P. Miller\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Division of Neurosurgery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0002-2032-4358\" />\n<meta name=\"citation_author\" content=\"Anna R. Poetsch\" />\n<meta name=\"citation_author_institution\" content=\"Biotechnology Center, Technical University Dresden, Germany\" />\n<meta name=\"citation_author_institution\" content=\"National Center for Tumor Diseases, Dresden, Germany\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0003-3056-4360\" />\n<meta name=\"twitter:creator\" content=\"@APoetsch\" />\n<meta name=\"citation_author\" content=\"Payal Jain\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Division of Neurosurgery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0002-5914-9083\" />\n<meta name=\"twitter:creator\" content=\"@jainpayal022\" />\n<meta name=\"citation_author\" content=\"Adam A. Kraya\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Division of Neurosurgery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0002-8526-5694\" />\n<meta name=\"citation_author\" content=\"Allison P. Heath\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Division of Neurosurgery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0002-2583-9668\" />\n<meta name=\"twitter:creator\" content=\"@allig8r\" />\n<meta name=\"citation_author\" content=\"Mateusz P. Koptyra\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Division of Neurosurgery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0002-3857-6633\" />\n<meta name=\"twitter:creator\" content=\"@koptyram\" />\n<meta name=\"citation_author\" content=\"Shannon Robbins\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Division of Neurosurgery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author\" content=\"Yiran Guo\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Division of Neurosurgery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0002-6549-8589\" />\n<meta name=\"twitter:creator\" content=\"@YiranGuo3\" />\n<meta name=\"citation_author\" content=\"Xiaoyan Huang\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Division of Neurosurgery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0001-7267-4512\" />\n<meta name=\"citation_author\" content=\"Jessica Wong\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Division of Neurosurgery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0003-1508-7631\" />\n<meta name=\"twitter:creator\" content=\"@jessicawongbfx\" />\n<meta name=\"citation_author\" content=\"Mariarita Santi\" />\n<meta name=\"citation_author_institution\" content=\"Department of Pathology and Laboratory Medicine, Children\u2019s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Department of Pathology and Laboratory Medicine, University of Pennsylvania Perelman School of Medicine\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0002-6728-3450\" />\n<meta name=\"citation_author\" content=\"Angela Viaene\" />\n<meta name=\"citation_author_institution\" content=\"Department of Pathology and Laboratory Medicine, Children\u2019s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Department of Pathology and Laboratory Medicine, University of Pennsylvania Perelman School of Medicine\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0001-6430-8360\" />\n<meta name=\"citation_author\" content=\"Laura Scolaro\" />\n<meta name=\"citation_author_institution\" content=\"Division of Oncology, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author\" content=\"Angela Waanders\" />\n<meta name=\"citation_author_institution\" content=\"Department of Oncology, Ann &amp; Robert H. Lurie Children\u2019s Hospital of Chicago\" />\n<meta name=\"citation_author_institution\" content=\"Department of Pediatrics, Northwestern University Feinberg School of Medicine\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0002-0571-2889\" />\n<meta name=\"citation_author\" content=\"Derek Hanson\" />\n<meta name=\"citation_author_institution\" content=\"Hackensack Meridian School of Medicine\" />\n<meta name=\"citation_author_institution\" content=\"Hackensack University Medical Center\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0002-0024-5142\" />\n<meta name=\"citation_author\" content=\"Hongbo M. Xie\" />\n<meta name=\"citation_author_institution\" content=\"Department of Bioinformatics and Health Informatics, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0003-2223-0029\" />\n<meta name=\"twitter:creator\" content=\"@xiehb\" />\n<meta name=\"citation_author\" content=\"Siyuan Zheng\" />\n<meta name=\"citation_author_institution\" content=\"Greehey Children&#39;s Cancer Research Institute, UT Health San Antonio\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0002-1031-9424\" />\n<meta name=\"twitter:creator\" content=\"@zhengsiyuan\" />\n<meta name=\"citation_author\" content=\"Cassie N. Kline\" />\n<meta name=\"citation_author_institution\" content=\"Division of Oncology, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0001-7765-7690\" />\n<meta name=\"twitter:creator\" content=\"@cnkline13\" />\n<meta name=\"citation_author\" content=\"Jena V. Lilly\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Division of Neurosurgery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0003-1439-6045\" />\n<meta name=\"twitter:creator\" content=\"@jvlilly\" />\n<meta name=\"citation_author\" content=\"Philip B. Storm\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Division of Neurosurgery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0002-7964-2449\" />\n<meta name=\"citation_author\" content=\"Adam C. Resnick\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Division of Neurosurgery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0003-0436-4189\" />\n<meta name=\"twitter:creator\" content=\"@adamcresnick\" />\n<meta name=\"citation_author\" content=\"Casey S. Greene\" />\n<meta name=\"citation_author_institution\" content=\"Department of Systems Pharmacology and Translational Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA, USA\" />\n<meta name=\"citation_author_institution\" content=\"Childhood Cancer Data Lab, Alex&#39;s Lemonade Stand Foundation, Philadelphia, PA, USA\" />\n<meta name=\"citation_author_institution\" content=\"Center for Health AI, University of Colorado School of Medicine, Aurora, CO, USA\" />\n<meta name=\"citation_author_institution\" content=\"Department of Biochemistry and Molecular Genetics, University of Colorado School of Medicine, Aurora, CO, USA\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0001-8713-9213\" />\n<meta name=\"twitter:creator\" content=\"@greenescientist\" />\n<meta name=\"citation_author\" content=\"Jo Lynne Rokita*\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Division of Neurosurgery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Department of Bioinformatics and Health Informatics, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0003-2171-3627\" />\n<meta name=\"twitter:creator\" content=\"@jolynnerokita\" />\n<meta name=\"citation_author\" content=\"Jaclyn N. Taroni*\" />\n<meta name=\"citation_author_institution\" content=\"Childhood Cancer Data Lab, Alex&#39;s Lemonade Stand Foundation, Philadelphia, PA, USA\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0003-4734-4508\" />\n<meta name=\"twitter:creator\" content=\"@jaclyn_taroni\" />\n<meta name=\"citation_author\" content=\"Children&#39;s Brain Tumor Network\" />\n<meta name=\"citation_author\" content=\"Pacific Pediatric Neurooncology Consortium\" />\n<link rel=\"canonical\" href=\"https://AlexsLemonade.github.io/OpenPBTA-manuscript/\" />\n<meta property=\"og:url\" content=\"https://AlexsLemonade.github.io/OpenPBTA-manuscript/\" />\n<meta property=\"twitter:url\" content=\"https://AlexsLemonade.github.io/OpenPBTA-manuscript/\" />\n<meta name=\"citation_fulltext_html_url\" content=\"https://AlexsLemonade.github.io/OpenPBTA-manuscript/\" />\n<meta name=\"citation_pdf_url\" content=\"https://AlexsLemonade.github.io/OpenPBTA-manuscript/manuscript.pdf\" />\n<link rel=\"alternate\" type=\"application/pdf\" href=\"https://AlexsLemonade.github.io/OpenPBTA-manuscript/manuscript.pdf\" />\n<link rel=\"alternate\" type=\"text/html\" href=\"https://AlexsLemonade.github.io/OpenPBTA-manuscript/v/0d77be1a643b2800042657630fa2f994c481404f/\" />\n<meta name=\"manubot_html_url_versioned\" content=\"https://AlexsLemonade.github.io/OpenPBTA-manuscript/v/0d77be1a643b2800042657630fa2f994c481404f/\" />\n<meta name=\"manubot_pdf_url_versioned\" content=\"https://AlexsLemonade.github.io/OpenPBTA-manuscript/v/0d77be1a643b2800042657630fa2f994c481404f/manuscript.pdf\" />\n<meta property=\"og:type\" content=\"article\" />\n<meta property=\"twitter:card\" content=\"summary_large_image\" />\n<link rel=\"icon\" type=\"image/png\" sizes=\"192x192\" href=\"https://manubot.org/favicon-192x192.png\" />\n<link rel=\"mask-icon\" href=\"https://manubot.org/safari-pinned-tab.svg\" color=\"#ad1457\" />\n<meta name=\"theme-color\" content=\"#ad1457\" />\n<!-- end Manubot generated metadata -->"
keywords:
- pediatric cancer
- brain tumor
- tumor atlas
lang: en-US
manubot-clear-requests-cache: false
manubot-output-bibliography: output/references.json
manubot-output-citekeys: output/citations.tsv
manubot-requests-cache-path: ci/cache/requests-cache
title: An Open Pediatric Brain Tumor Atlas
...






<small><em>
This manuscript
([permalink](https://AlexsLemonade.github.io/OpenPBTA-manuscript/v/0d77be1a643b2800042657630fa2f994c481404f/))
was automatically generated
from [AlexsLemonade/OpenPBTA-manuscript@0d77be1](https://github.com/AlexsLemonade/OpenPBTA-manuscript/tree/0d77be1a643b2800042657630fa2f994c481404f)
on March 31, 2022.
</em></small>

## Authors



+ **Joshua A. Shapiro**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-6224-0347](https://orcid.org/0000-0002-6224-0347)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [jashapiro](https://github.com/jashapiro)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [jashapiro](https://twitter.com/jashapiro)<br>
  <small>   
     Childhood Cancer Data Lab, Alex's Lemonade Stand Foundation, Philadelphia, PA, USA
     · Funded by Alex's Lemonade Stand Foundation Childhood Cancer Data Lab (CCDL)
  </small>

+ **Candace L. Savonen**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0001-6331-7070](https://orcid.org/0000-0001-6331-7070)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [cansavvy](https://github.com/cansavvy)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [cansavvy](https://twitter.com/cansavvy)<br>
  <small>   
     Childhood Cancer Data Lab, Alex's Lemonade Stand Foundation, Philadelphia, PA, USA
     · Funded by Alex's Lemonade Stand Foundation Childhood Cancer Data Lab (CCDL)
  </small>

+ **Chante J. Bethell**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0001-9653-8128](https://orcid.org/0000-0001-9653-8128)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [cbethell](https://github.com/cbethell)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [cjbethell](https://twitter.com/cjbethell)<br>
  <small>   
     Childhood Cancer Data Lab, Alex's Lemonade Stand Foundation, Philadelphia, PA, USA
     · Funded by Alex's Lemonade Stand Foundation Childhood Cancer Data Lab (CCDL)
  </small>

+ **Krutika S. Gaonkar**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0003-0838-2405](https://orcid.org/0000-0003-0838-2405)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [kgaonkar6](https://github.com/kgaonkar6)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [aggokittu](https://twitter.com/aggokittu)<br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Division of Neurosurgery, Children's Hospital of Philadelphia; Department of Bioinformatics and Health Informatics, Children's Hospital of Philadelphia
  </small>

+ **Run Jin**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-8958-9266](https://orcid.org/0000-0002-8958-9266)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [runjin326](https://github.com/runjin326)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [runjin](https://twitter.com/runjin)<br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Division of Neurosurgery, Children's Hospital of Philadelphia
  </small>

+ **Yuankun Zhu**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-2455-9525](https://orcid.org/0000-0002-2455-9525)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [yuankunzhu](https://github.com/yuankunzhu)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [zhuyuankun](https://twitter.com/zhuyuankun)<br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Division of Neurosurgery, Children's Hospital of Philadelphia
  </small>

+ **Miguel A. Brown**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0001-6782-1442](https://orcid.org/0000-0001-6782-1442)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [migbro](https://github.com/migbro)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [migbro](https://twitter.com/migbro)<br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Division of Neurosurgery, Children's Hospital of Philadelphia
  </small>

+ **Nhat Duong**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0003-2852-4263](https://orcid.org/0000-0003-2852-4263)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [fingerfen](https://github.com/fingerfen)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [asiannhat](https://twitter.com/asiannhat)<br>
  <small>   
     Department of Bioinformatics and Health Informatics, Children's Hospital of Philadelphia
  </small>

+ **Komal S. Rathi**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0001-5534-6904](https://orcid.org/0000-0001-5534-6904)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [komalsrathi](https://github.com/komalsrathi)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [komalsrathi](https://twitter.com/komalsrathi)<br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Department of Bioinformatics and Health Informatics, Children's Hospital of Philadelphia
  </small>

+ **Nighat Noureen**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0001-7495-8201](https://orcid.org/0000-0001-7495-8201)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [NNoureen](https://github.com/NNoureen)<br>
  <small>   
     Greehey Children's Cancer Research Institute, UT Health San Antonio
  </small>

+ **Bo Zhang**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-0743-5379](https://orcid.org/0000-0002-0743-5379)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [zhangb1](https://github.com/zhangb1)<br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Division of Neurosurgery, Children's Hospital of Philadelphia
  </small>

+ **Brian M. Ennis**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-2653-5009](https://orcid.org/0000-0002-2653-5009)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [bmennis](https://github.com/bmennis)<br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Division of Neurosurgery, Children's Hospital of Philadelphia
  </small>

+ **Stephanie J. Spielman**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-9090-4788](https://orcid.org/0000-0002-9090-4788)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [sjspielman](https://github.com/sjspielman)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [stephspiel](https://twitter.com/stephspiel)<br>
  <small>   
     Childhood Cancer Data Lab, Alex's Lemonade Stand Foundation, Philadelphia, PA, USA<sup>☯</sup>; Rowan University, Glassboro, NJ, USA
     · Funded by Alex's Lemonade Stand Foundation Childhood Cancer Data Lab (CCDL)
      <br>
      <sup>☯</sup>Current affiliation
  </small>

+ **Laura E. Egolf**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-7103-4801](https://orcid.org/0000-0002-7103-4801)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [LauraEgolf](https://github.com/LauraEgolf)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [LauraEgolf](https://twitter.com/LauraEgolf)<br>
  <small>   
     Cell and Molecular Biology Graduate Group, Perelman School of Medicine at the University of Pennsylvania; Division of Oncology, Children's Hospital of Philadelphia
  </small>

+ **Bailey Farrow**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0001-6727-6333](https://orcid.org/0000-0001-6727-6333)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [baileyckelly](https://github.com/baileyckelly)<br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Division of Neurosurgery, Children's Hospital of Philadelphia
  </small>

+ **Nicolas Van Kuren**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-7414-9516](https://orcid.org/0000-0002-7414-9516)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [nicholasvk](https://github.com/nicholasvk)<br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Division of Neurosurgery, Children's Hospital of Philadelphia
  </small>

+ **Tejaswi Koganti**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-7733-6480](https://orcid.org/0000-0002-7733-6480)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [tkoganti](https://github.com/tkoganti)<br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Division of Neurosurgery, Children's Hospital of Philadelphia
  </small>

+ **Shrivats Kannan**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-1460-920X](https://orcid.org/0000-0002-1460-920X)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [shrivatsk](https://github.com/shrivatsk)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [kshrivats](https://twitter.com/kshrivats)<br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Division of Neurosurgery, Children's Hospital of Philadelphia
  </small>

+ **Pichai Raman**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0001-6948-2157](https://orcid.org/0000-0001-6948-2157)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [pichairaman](https://github.com/pichairaman)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [PichaiRaman](https://twitter.com/PichaiRaman)<br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Department of Bioinformatics and Health Informatics, Children's Hospital of Philadelphia
  </small>

+ **Jennifer Mason**<br>
    · ![GitHub icon](images/github.svg){.inline_icon}
    [jenn0307](https://github.com/jenn0307)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [jenn0307](https://twitter.com/jenn0307)<br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Division of Neurosurgery, Children's Hospital of Philadelphia
  </small>

+ **Daniel P. Miller**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-2032-4358](https://orcid.org/0000-0002-2032-4358)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [dmiller15](https://github.com/dmiller15)<br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Division of Neurosurgery, Children's Hospital of Philadelphia
  </small>

+ **Anna R. Poetsch**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0003-3056-4360](https://orcid.org/0000-0003-3056-4360)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [arpoe](https://github.com/arpoe)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [APoetsch](https://twitter.com/APoetsch)<br>
  <small>   
     Biotechnology Center, Technical University Dresden, Germany; National Center for Tumor Diseases, Dresden, Germany
  </small>

+ **Payal Jain**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-5914-9083](https://orcid.org/0000-0002-5914-9083)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [jainpayal022](https://github.com/jainpayal022)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [jainpayal022](https://twitter.com/jainpayal022)<br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Division of Neurosurgery, Children's Hospital of Philadelphia
  </small>

+ **Adam A. Kraya**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-8526-5694](https://orcid.org/0000-0002-8526-5694)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [aadamk](https://github.com/aadamk)<br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Division of Neurosurgery, Children's Hospital of Philadelphia
  </small>

+ **Allison P. Heath**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-2583-9668](https://orcid.org/0000-0002-2583-9668)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [allisonheath](https://github.com/allisonheath)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [allig8r](https://twitter.com/allig8r)<br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Division of Neurosurgery, Children's Hospital of Philadelphia
     · Funded by NIH U2C HL138346-03; NCI/NIH Contract No. 75N91019D00024, Task Order No. 75N91020F00003; Australian Government, Department of Education
  </small>

+ **Mateusz P. Koptyra**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-3857-6633](https://orcid.org/0000-0002-3857-6633)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [mkoptyra](https://github.com/mkoptyra)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [koptyram](https://twitter.com/koptyram)<br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Division of Neurosurgery, Children's Hospital of Philadelphia
  </small>

+ **Shannon Robbins**<br><br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Division of Neurosurgery, Children's Hospital of Philadelphia
  </small>

+ **Yiran Guo**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-6549-8589](https://orcid.org/0000-0002-6549-8589)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [Yiran-Guo](https://github.com/Yiran-Guo)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [YiranGuo3](https://twitter.com/YiranGuo3)<br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Division of Neurosurgery, Children's Hospital of Philadelphia
  </small>

+ **Xiaoyan Huang**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0001-7267-4512](https://orcid.org/0000-0001-7267-4512)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [HuangXiaoyan0106](https://github.com/HuangXiaoyan0106)<br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Division of Neurosurgery, Children's Hospital of Philadelphia
  </small>

+ **Jessica Wong**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0003-1508-7631](https://orcid.org/0000-0003-1508-7631)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [wongjessica93](https://github.com/wongjessica93)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [jessicawongbfx](https://twitter.com/jessicawongbfx)<br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Division of Neurosurgery, Children's Hospital of Philadelphia
  </small>

+ **Mariarita Santi**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-6728-3450](https://orcid.org/0000-0002-6728-3450)<br>
  <small>   
     Department of Pathology and Laboratory Medicine, Children’s Hospital of Philadelphia; Department of Pathology and Laboratory Medicine, University of Pennsylvania Perelman School of Medicine
  </small>

+ **Angela Viaene**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0001-6430-8360](https://orcid.org/0000-0001-6430-8360)<br>
  <small>   
     Department of Pathology and Laboratory Medicine, Children’s Hospital of Philadelphia; Department of Pathology and Laboratory Medicine, University of Pennsylvania Perelman School of Medicine
  </small>

+ **Laura Scolaro**<br><br>
  <small>   
     Division of Oncology, Children's Hospital of Philadelphia
  </small>

+ **Angela Waanders**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-0571-2889](https://orcid.org/0000-0002-0571-2889)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [awaanders](https://github.com/awaanders)<br>
  <small>   
     Department of Oncology, Ann & Robert H. Lurie Children’s Hospital of Chicago; Department of Pediatrics, Northwestern University Feinberg School of Medicine
  </small>

+ **Derek Hanson**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-0024-5142](https://orcid.org/0000-0002-0024-5142)<br>
  <small>   
     Hackensack Meridian School of Medicine; Hackensack University Medical Center
  </small>

+ **Hongbo M. Xie**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0003-2223-0029](https://orcid.org/0000-0003-2223-0029)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [xiehongbo](https://github.com/xiehongbo)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [xiehb](https://twitter.com/xiehb)<br>
  <small>   
     Department of Bioinformatics and Health Informatics, Children's Hospital of Philadelphia
  </small>

+ **Siyuan Zheng**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-1031-9424](https://orcid.org/0000-0002-1031-9424)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [syzheng](https://github.com/syzheng)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [zhengsiyuan](https://twitter.com/zhengsiyuan)<br>
  <small>   
     Greehey Children's Cancer Research Institute, UT Health San Antonio
  </small>

+ **Cassie N. Kline**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0001-7765-7690](https://orcid.org/0000-0001-7765-7690)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [cnkline13](https://twitter.com/cnkline13)<br>
  <small>   
     Division of Oncology, Children's Hospital of Philadelphia
  </small>

+ **Jena V. Lilly**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0003-1439-6045](https://orcid.org/0000-0003-1439-6045)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [jvlilly](https://github.com/jvlilly)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [jvlilly](https://twitter.com/jvlilly)<br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Division of Neurosurgery, Children's Hospital of Philadelphia
  </small>

+ **Philip B. Storm**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-7964-2449](https://orcid.org/0000-0002-7964-2449)<br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Division of Neurosurgery, Children's Hospital of Philadelphia
     · Funded by Alex's Lemonade Stand Foundation (Catalyst); Children’s Hospital of Philadelphia Division of Neurosurgery
  </small>

+ **Adam C. Resnick**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0003-0436-4189](https://orcid.org/0000-0003-0436-4189)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [adamcresnick](https://github.com/adamcresnick)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [adamcresnick](https://twitter.com/adamcresnick)<br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Division of Neurosurgery, Children's Hospital of Philadelphia
     · Funded by Alex's Lemonade Stand Foundation (Catalyst); Children's Brain Tumor Network; NIH 3P30 CA016520-44S5, U2C HL138346-03, U24 CA220457-03; NCI/NIH Contract No. 75N91019D00024, Task Order No. 75N91020F00003; Children’s Hospital of Philadelphia Division of Neurosurgery
  </small>

+ **Casey S. Greene**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0001-8713-9213](https://orcid.org/0000-0001-8713-9213)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [cgreene](https://github.com/cgreene)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [greenescientist](https://twitter.com/greenescientist)<br>
  <small>   
     Department of Systems Pharmacology and Translational Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA, USA; Childhood Cancer Data Lab, Alex's Lemonade Stand Foundation, Philadelphia, PA, USA; Center for Health AI, University of Colorado School of Medicine, Aurora, CO, USA; Department of Biochemistry and Molecular Genetics, University of Colorado School of Medicine, Aurora, CO, USA
     · Funded by Alex's Lemonade Stand Foundation Childhood Cancer Data Lab (CCDL)
  </small>

+ **Jo Lynne Rokita***<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0003-2171-3627](https://orcid.org/0000-0003-2171-3627)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [jharenza](https://github.com/jharenza)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [jolynnerokita](https://twitter.com/jolynnerokita)<br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Division of Neurosurgery, Children's Hospital of Philadelphia; Department of Bioinformatics and Health Informatics, Children's Hospital of Philadelphia
     · Funded by Alex's Lemonade Stand Foundation (Young Investigator, Catalyst); NCI/NIH Contract No. 75N91019D00024, Task Order No. 75N91020F00003
  </small>

+ **Jaclyn N. Taroni***<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0003-4734-4508](https://orcid.org/0000-0003-4734-4508)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [jaclyn-taroni](https://github.com/jaclyn-taroni)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [jaclyn_taroni](https://twitter.com/jaclyn_taroni)<br>
  <small>   
     Childhood Cancer Data Lab, Alex's Lemonade Stand Foundation, Philadelphia, PA, USA
     · Funded by Alex's Lemonade Stand Foundation Childhood Cancer Data Lab (CCDL)
  </small>

+ **Children's Brain Tumor Network**<br><br>
  <small>
  </small>

+ **Pacific Pediatric Neurooncology Consortium**<br><br>
  <small>
  </small>


## Contact information
\*Correspondence: jaclyn.taroni@ccdatalab.org, rokita@chop.edu

## In Brief
<!-- to add -->

## Highlights
<!-- Highlights are bullet points that convey the core findings of your paper. You may include up to four highlights. The length of each highlight cannot exceed 85 characters (including spaces).-->



## Summary
<!-- The summary is a single paragraph no longer than 150 words. An effective summary includes the following elements: (1) a brief background of the question that avoids statements about how a process is not well understood; (2) a description of the results and approaches/model systems framed in the context of their conceptual interest; and (3) an indication of the broader significance of the work. We discourage novelty claims (e.g., use of the word “novel”) because they are overused, tend not to add meaning, and are difficult to verify. Please do not include references in the summary. -->

## Keywords
<!-- We encourage you to include up to ten keywords with your paper. These keywords will be associated with your paper on Cell Press platforms and on PubMed. These keywords should be listed in the manuscript after the summary, separated by commas. -->
pediatric brain tumors, somatic variation, open science, classification  




## Introduction

Pediatric brain and spinal cord tumors are the second most common tumors in children after leukemia, yet they represent the leading disease-related cause of death in children [@doi:10.1093/neuonc/noz150]. 
Five-year survival rates vary widely across different histologic and molecular classifications of brain tumors.
For example, most high-grade and embryonal tumors carry a universally fatal prognosis while children with pilocytic astrocytoma have an estimated 10-year survival rate of 92% [@doi:10.1093/neuonc/now207]. 
Despite their relative rarity, the years of potential life lost due to brain tumors in 2009 was estimated at 47,631 years for children and adolescents aged 0-19 in the United States [@doi:10.1159/00009389610.1159/000093896]. 
The low survival rates for some tumors are clearly multifactorial but can be explained in part by our lack of understanding of the ever-evolving array of brain tumor molecular subtypes, difficulty drugging these entities, and the shortage of drugs specifically labeled for pediatric malignancies. 
Historically, some of the most fatal, inoperable brain tumors, such as diffuse midline gliomas, were not routinely biopsied due to perceived risks of biopsy and the paucity of therapeutic options that would require tissue. 
Limited access to tissue to develop patient-derived cell line and mouse models has been a barrier to research. 
Furthermore, the incidence of any single molecular tumor entity is relatively low due to the rarity of pediatric tumors in general. 
Together, these factors have hindered research progress and have led to multiple national and international center and consortia efforts to collaboratively share specimens and data to accelerate breakthroughs and clinical translation. 

There has been significant progress in recent years to elucidate the landscape of somatic variation responsible for pediatric brain tumor formation and progression, however, translation of therapeutic agents to phase II or III clinical trials and subsequent FDA approval has been slow. 
Within the last 20 years, the FDA has approved only five drugs for the treatment of pediatric brain tumors: mTOR inhibitor, everolimus, for subependymal giant cell astrocytoma; anti-PD-1 immunotherapy, pembrolizumab, for microsatellite instability–high or mismatch repair–deficient tumors; NTRK inhibitors larotrectinib and entrectinib for tumors with an NTRK 1/2/3 gene fusions; MEK1/2 inhibitor, selumetinib, for neurofibromatosis type 1 (NF1) and symptomatic, inoperable plexiform neurofibromas. 
This is, in part, due to pharmaceutical company priorities and/or concerns regarding toxicity that have resulted in an inability to obtain drugs for pediatric clinical trials, ultimately delaying access to new agents.
An amendment to the Pediatric Research Equity Act called the Research to Accelerate Cures and Equity (RACE) for Children Act mandates that as of August 18, 2020 all new adult oncology drugs also be tested in children when the molecular targets are relevant to a particular childhood cancer. 
Here, we present a comprehensive, collaborative, open genomic analysis of 943 patient tumors from 59 distinct brain tumor histologies which can be used to support the RACE Act and accelerate rational clinical trial design.


## Results

### Crowd-sourced Somatic Analysis to create an Open Pediatric Brain Tumor Atlas

We previously performed whole genome sequencing (WGS), whole exome sequencing (WXS), and RNA sequencing (RNA-Seq) on matched tumor and normal tissues as well as selected cell lines from 943 patient tumors from the Pediatric Brain Tumor Atlas (PBTA) [@doi:10.1093/neuonc/noz192], consisting of samples from the Children’s Brain Tumor Network (CBTN) [@url:https://CBTN.org] and the PNOC003 DMG clinical trial [@doi:10.1002/ijc.32258] of the Pacific Pediatric Neuro-oncology Symposium (PNOC) (**Figure {@fig:Fig1}A**). 
We then harnessed the benchmarking efforts of the Kids First Data Resource Center to develop a robust and reproducible data analysis workflow within the CAVATICA platform to perform the following primary somatic analyses: variant calling of single nucleotide variants (SNVs), copy number variants (CNVs), structural variants (SVs), and fusions (**Figure S1}** - red boxes and **STAR Methods**). 
Next, we created a public Github analysis repository (https://github.com/AlexsLemonade/OpenPBTA-analysis) with continuous integration to ensure analysis reproducibility.
We additionally created a GitHub manuscript repository (https://github.com/AlexsLemonade/OpenPBTA-manuscript) with ManuBot [@url:https://manubot.org/ ; @doi:10.1371/journal.pcbi.1007128] integration to enable manuscript creation using Markdown within GitHub.
We maintained a data release folder on Amazon S3 containing merged files for each analysis, downloadable from the GitHub repository or open access CAVATICA project.  

<!-- TODO: Stub for the process for contributing analytical code (**Figure {@fig:FigS1}** and **STAR Methods**) and to the manuscript. -->

![**Figure 1. Overview of the OpenPBTA Project.** A, The Children's Brain Tumor Network and the Pacific Pediatric Neuro-Oncology Consortium collected tumor samples from 943 patients. To date, 38 cell lines were created from tumor tissue and over 2000 specimens were sequenced (N = 1035 RNA-Seq, N = 940 WGS, and N = 32 WXS or Targeted panel). Data was harmonized by the Kids First Data Resource Center using an Amazon S3 framework within CAVATICA. B, Processed data was released through the OpenPBTA GitHub project, through which contributors filed issues and developed code for analyses within docker. Additional analysis details can be found in **Figure S1**. Simultaneously, this manuscript was written in GitHub using Manubot. All analysis and manuscript code was reviewed and continuously integrated using CircleCI. C, Stacked bar plot summary of the number of biospecimens per phase of therapy per broad histology.](images/main/Figure1.png){#fig:Fig1}

### Molecular Subtyping of OpenPBTA CNS Tumors

Over the past two decades, together with the World Health Organization (WHO), experts in neurooncology have iteratively redefined the classifications of central nervous system (CNS) tumors [@doi: 10.1093/jnen/61.3.215; @doi:10.1007/s00401-007-0243-4]. 
More recently, in 2016 and 2021 [@doi:10.1007/s00401-016-1545-1; @doi:10.1093/neuonc/noab106], molecular subtypes have been integrated into these entities. 
The Children’s Brain Tumor Tissue Consortium (CBTTC), currently the Children’s Brain Tumor Network (CBTN), opened its protocol for brain tumor and matched normal sample collection in 2011 and as such, the majority of the samples within the OpenPBTA dataset lack molecular subtype annotation. 
In the absence of methylation arrays for tumor classification, we utilized key genomic features of entities described by the WHO in 2016, as well as Ryall and colleagues [@doi:10.1016/j.ccell.2020.03.011], coupled with clinician and pathologist review to subtype 64% (1,281/2,007) of tumor biospecimens with high confidence (**Table S1**).
Importantly, this collaborative molecular subtyping process allowed us to identify data entry errors (e.g., an ETMR entered as a medulloblastoma) and mis-identified specimens (e.g., Ewing sarcoma sample labeled as a craniopharyngioma), update diagnoses using modern terms (e.g., primitive neuro-ectodermal tumor [PNET] diagnoses), and discover rarer tumor entities within the OpenPBTA (e.g., H3-mutant ependymoma, meningioma with _YAP1::FAM118B_ fusion).
**Table 1** {#tbl:subtypes} lists the subtypes we defined within the OpenPBTA, comprising of LGATs (N = 569), high-grade gliomas (N = 314), embryonal tumors (N = 229), ependymomas (N = 65), tumors of sellar region (N = 51), mesenchymal non-meningothelial tumors (N = 21), glialneuronal tumors (N = 20), and chordomas (N = 12). For methods, see **STAR Methods** and **Figure S1**. 

| Broad histology                               | molecular subtype                        | n    |
|-----------------------------------------------|------------------------------------------|------|
| Chordoma                                      | CHDM, conventional                       | 4    |
| Chordoma                                      | CHDM, poorly differentiated              | 8    |
| Diffuse astrocytic and oligodendroglial tumor | DMG, H3 K28                              | 54   |
| Diffuse astrocytic and oligodendroglial tumor | DMG, H3 K28, TP53 activated              | 26   |
| Diffuse astrocytic and oligodendroglial tumor | DMG, H3 K28, TP53 loss                   | 91   |
| Diffuse astrocytic and oligodendroglial tumor | HGG, H3 G35                              | 7    |
| Diffuse astrocytic and oligodendroglial tumor | HGG, H3 G35, TP53 loss                   | 2    |
| Diffuse astrocytic and oligodendroglial tumor | HGG, H3 wildtype                         | 74   |
| Diffuse astrocytic and oligodendroglial tumor | HGG, H3 wildtype, TP53 activated         | 10   |
| Diffuse astrocytic and oligodendroglial tumor | HGG, H3 wildtype, TP53 loss              | 45   |
| Diffuse astrocytic and oligodendroglial tumor | HGG, IDH, TP53 activated                 | 3    |
| Diffuse astrocytic and oligodendroglial tumor | HGG, IDH, TP53 loss                      | 2    |
| Embryonal tumor                               | CNS Embryonal, NOS                       | 24   |
| Embryonal tumor                               | CNS HGNET-MN1                            | 1    |
| Embryonal tumor                               | CNS NB-FOXR2                             | 5    |
| Embryonal tumor                               | ETMR, C19MC-altered                      | 8    |
| Embryonal tumor                               | ETMR, NOS                                | 1    |
| Embryonal tumor                               | MB, Group3                               | 24   |
| Embryonal tumor                               | MB, Group4                               | 91   |
| Embryonal tumor                               | MB, SHH                                  | 55   |
| Embryonal tumor                               | MB, WNT                                  | 20   |
| Ependymal tumor                               | EPN, H3 K28                              | 2    |
| Ependymal tumor                               | EPN, PF A                                | 6    |
| Ependymal tumor                               | EPN, ST RELA                             | 51   |
| Ependymal tumor                               | EPN, ST YAP1                             | 6    |
| Low-grade astrocytic tumor                    | GNG, BRAF V600E                          | 25   |
| Low-grade astrocytic tumor                    | GNG, BRAF V600E, CDKN2A/B                | 2    |
| Low-grade astrocytic tumor                    | GNG, FGFR                                | 2    |
| Low-grade astrocytic tumor                    | GNG, H3                                  | 2    |
| Low-grade astrocytic tumor                    | GNG, IDH                                 | 4    |
| Low-grade astrocytic tumor                    | GNG, KIAA1549-BRAF                       | 10   |
| Low-grade astrocytic tumor                    | GNG, MYB/MYBL1                           | 2    |
| Low-grade astrocytic tumor                    | GNG, NF1-germline                        | 2    |
| Low-grade astrocytic tumor                    | GNG, NF1-somatic, BRAF V600E             | 1    |
| Low-grade astrocytic tumor                    | GNG, other MAPK                          | 7    |
| Low-grade astrocytic tumor                    | GNG, other MAPK, IDH                     | 2    |
| Low-grade astrocytic tumor                    | GNG, RTK                                 | 6    |
| Low-grade astrocytic tumor                    | GNG, wildtype                            | 28   |
| Low-grade astrocytic tumor                    | LGG, BRAF V600E                          | 53   |
| Low-grade astrocytic tumor                    | LGG, BRAF V600E, CDKN2A/B                | 10   |
| Low-grade astrocytic tumor                    | LGG, FGFR                                | 16   |
| Low-grade astrocytic tumor                    | LGG, IDH                                 | 6    |
| Low-grade astrocytic tumor                    | LGG, KIAA1549-BRAF                       | 222  |
| Low-grade astrocytic tumor                    | LGG, KIAA1549-BRAF, other MAPK           | 2    |
| Low-grade astrocytic tumor                    | LGG, MYB/MYBL1                           | 4    |
| Low-grade astrocytic tumor                    | LGG, NF1-germline                        | 12   |
| Low-grade astrocytic tumor                    | LGG, NF1-germline, CDKN2A/B              | 2    |
| Low-grade astrocytic tumor                    | LGG, NF1-germline, FGFR                  | 4    |
| Low-grade astrocytic tumor                    | LGG, NF1-somatic                         | 4    |
| Low-grade astrocytic tumor                    | LGG, NF1-somatic, FGFR                   | 2    |
| Low-grade astrocytic tumor                    | LGG, NF1-somatic, NF1-germline, CDKN2A/B | 2    |
| Low-grade astrocytic tumor                    | LGG, other MAPK                          | 23   |
| Low-grade astrocytic tumor                    | LGG, RTK                                 | 22   |
| Low-grade astrocytic tumor                    | LGG, RTK, CDKN2A/B                       | 2    |
| Low-grade astrocytic tumor                    | LGG, wildtype                            | 84   |
| Low-grade astrocytic tumor                    | SEGA, wildtype                           | 6    |
| Mesenchymal non-meningothelial tumor          | EWS                                      | 21   |
| Neuronal and mixed neuronal-glial tumor       | CNC                                      | 4    |
| Neuronal and mixed neuronal-glial tumor       | EVN                                      | 2    |
| Neuronal and mixed neuronal-glial tumor       | GNT, BRAF V600E                          | 2    |
| Neuronal and mixed neuronal-glial tumor       | GNT, KIAA1549-BRAF                       | 4    |
| Neuronal and mixed neuronal-glial tumor       | GNT, other MAPK                          | 2    |
| Neuronal and mixed neuronal-glial tumor       | GNT, other MAPK, FGFR                    | 2    |
| Neuronal and mixed neuronal-glial tumor       | GNT, RTK                                 | 4    |
| Tumors of sellar region                       | CRANIO, ADAM                             | 51   |
|                                               | Total                                    | 1281 |


Table 1: Molecular subtypes determined across OpenPBTA samples. {#tbl:subtypes}


### Somatic Mutational Landscape of Pediatric Brain Tumors

We  performed a comprehensive genomic analysis of somatic SNVs, CNVs, SVs, and fusions across 1,969 tumors (N = 1,019 RNA-Seq, N = 1,719 WGS, N = 64 WXS/Panel) and 38 cell lines (N = 16 RNA-Seq, N = 22 WGS) from 943 patients.
Following SNV consensus calling (**Figure S1** and **Figure S2, panels A-F**), we observed lower expected tumor mutation burden (TMB) **Figure S2, panel G** in pediatric tumors compared to adult brain tumors from The Cancer Genome Atlas (TCGA), with hypermutant (> 10 Mut/Mb) and ultra-hypermutant (> 100 Mut/Mb) tumors [@doi:10.1016/j.cell.2017.09.048] only found within HGATs.  

#### Low-grade astrocytic tumors
**Figure {@fig:Fig2}A** depicts an oncoprint of driver genes for 227 primary low-grade astrocytic tumors. As expected, the majority (62%, 140/227) of these tumors harbor a somatic alteration in _BRAF_, with canonical _BRAF::KIAA1549_ fusions as the major oncogenic driver. 
We observed additional mutations in _FGFR1_ (2%), _PIK3CA_ (2%), _KRAS_ (2%), _TP53_ (1%), and _ATRX_ (1%) and fusions in _NTRK2_ (2%), _RAF1_ (2%), _MYB_ (1%), _QKI_ (1%), _ROS1_ (1%), and _FGFR2_ (1%), concordant with previous studies reporting the near universal upregulation of the RAS/MAPK pathway in these tumors resulting from activating mutations and/or oncogenic fusions [@doi:10.1186/s40478-020-00902-z; @doi:10.1016/j.ccell.2020.03.011]. 
Indeed, we observed significant upregulation (ANOVA p < 0.01) of the KRAS signaling pathway in LGATs (**Figure {@fig:Fig4}B**).

#### Embryonal tumors
**Figure {@fig:Fig2}B** shows the mutational landscape for 128 primary embryonal tumors. 
The majority (N = 95) are medulloblastomas and span the spectrum of molecular subtypes: WNT, SHH, Group3, and Group 4 (see **Molecular Subtyping of CNS Tumors**), with their canonical mutations. 
We detected canonical _SMARCB1/SMARCA4_ deletions or inactivating mutations in atypical teratoid rhabdoid tumors (ATRTs) and C19MC amplification in the embryonal tumors with multilayer rosettes (ETMRs) [@doi:10.1007/s00401-020-02182-2; @doi:10.1093/neuonc/noab178; @doi:10.1186/s40478-020-00984-9; @doi:10.1038/nature22973].

#### Diffuse astrocytic and oligodendroglial tumors
In **Figure {@fig:Fig2}C**, we show genomic alterations in diffuse midline gliomas (DMGs, N = 34) and non-midline high-grade gliomas (N = 26) biopsied at diagnosis. 
The single oligodendroglioma sample in the OpenPBTA does not contain mutations in the genes shown and is therefore not present in this oncoprint. 
Across HGATs, we found _TP53_ (57%, 35/61) and _H3F3A_ (52%, 32/61) to be the most mutated and co-occurring genes (**Figure {@fig:Fig2}A**), followed by frequent mutations in _ATRX_ (30% 18/61). 
We found recurrent amplifications and fusions in _EGFR_, _MET_, _PDGFRA_, and _KIT_, highlighting that these tumors utilize multiple oncogenic mechanisms to activate tyrosine kinases, as has been previously reported [@doi:10.1007/s12254-021-00680-x; @doi:10.1002/ijc.32258; @doi:10.1016/j.ccell.2017.08.017; @doi:10.1186/s40478-020-00905-w]. 
Gene set enrichment analysis showed upregulation (ANOVA p < 0.01) of DNA repair, G2M checkpoint, and MYC pathways as well as downregulation of the TP53 pathway (**Figure {@fig:Fig4}B**).
The two tumors with ultra-high tumor mutation burden (TMB) (> 100 Mutations/Mb) were from patients with known mismatch repair deficiency syndrome [@doi:10.1093/neuonc/noz192].

#### Other CNS tumors
**Figure {@fig:Fig2}D** depicts an oncoprint for the remaining primary CNS tumors (N = 195). 
We observed 25% (15/60) of ependymoma tumors to be _C11orf95::RELA_ (now, _ZFTA::RELA_) fusion positive ependymomas and 70% (21/30) of craniopharyngiomas to be driven by mutations in _CTNNB1_. 
Multiple histologies contained somatic mutations or fusions in _NF2_: 41% (7/17) of meningiomas, 5% (3/60) of ependymomas,  and 27% (3/11) schwannomas. 
Rare fusions in _ERBB4_, _YAP1_, _KRAS_, and _MAML2_ were observed in 10% (6/60) of ependymoma tumors. 
DNETs harbored alterations in MAPK/PI3K pathway genes as previously reported [@doi:10.1093/jnen/nlz101], including _FGFR1_ (21%, 4/19), _PDGFRA_ (10%, 2/19), and _BRAF_ (5%, 1/19).
Frequent mutations in rarer brain tumor histologies (N < 5) are depicted in **Figure {@fig:FigS3}A**.

![**Figure 2. Mutational landscape of PBTA tumors.** Shown are frequencies of canonical somatic gene mutations, CNVs, fusions, and TMB (top bar plot) for the top 20 genes mutated across primary tumors within the OpenPBTA dataset. A, Low-grade astrocytic tumors (N = 227): low-grade glioma astrocytoma (N = 187), ganglioglioma (N = 35), subependymal giant cell astrocytoma (N = 2), diffuse fibrillary astrocytoma (N = 1), pilocytic astrocytoma (N = 1), and pleomorphic xanthoastrocytoma (N = 1); B, Embryonal tumors (N = 128): medulloblastomas (N = 95), atypical teratoid rhabdoid tumors (N = 24), embryonal tumors with multilayer rosettes (N = 2), other CNS embryonal tumors (N = 5), ganglioneuroblastoma (N = 1), and CNS neuroblastoma (N = 1); C, Diffuse astrocytic and oligodendroglial tumors (N = 61): diffuse midline gliomas (N = 34) and non-midline high-grade gliomas (N = 26), oligodendroglioma (N = 1); D, Other CNS tumors (N = 195): ependymomas (N = 60), dysembryoplastic neuroepithelial tumors (N = 19), meningiomas (N = 17), schwannoma (N = 11), neurofibroma plexiform (N = 7). Other CNS tumors with N < 5 are displayed in **Figure {@fig:FigS3}A**. Patient sex (`germline_sex_estimate`) and tumor histology (`cancer_group`) are displayed as annotations at the bottom of each plot. Only samples with mutations in the listed genes are shown. Multiple CNVs are denoted as a complex event.](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/figures/pngs/primary_only_oncoprint_landscape.png){#fig:Fig2}


### Mutational co-occurrence and signatures highlight key oncogenic drivers

The top 50 mutated genes in primary tumors are shown **Figure {@fig:Fig3}** by tumor type (**A**, bar plots), with co-occurrence scores illustrated in the heatmap (**B**). 
We observed _TP53_ to be the most frequently mutated gene across OpenPBTA tumors (8.4%, 56/666), significantly co-occurring with _H3F3A_ (OR = 32, 95% CI: 15.3 - 66.7, q = 8.46e-17), _ATRX_ (OR = 20, 95% CI: 8.4 - 47.7, q = 4.43e-8), _NF1_ (OR = 8.62, 95% CI: 3.7 - 20.2, q = 5.45e-5), and _EGFR_ (OR = 18.2, 95% CI: 5 - 66.5, q = 1.6e-4). 
Other canonical cancer driver genes were frequently mutated: _BRAF_, _H3F3A_, _CTNNB1_, _NF1_, _ATRX_, _FGFR1_, and _PIK3CA_. 
Although LGG and embryonal tumors make up the majority of tumor types within the OpenPBTA, most of the significant gene interactions stem from HGATs (N = 847/872). 
At the broad histology level, _CTNNB1_ significantly co-occurs with _TP53_ (OR = 42.9, 95% CI: 7 - 261.4, q = 1.63e-3) and _DDX3X_ (OR = 21.1, 95% CI: 4.6 - 96.3, q = 4.46e-3) in embryonal tumors, _FGFR1_ and _PIK3CA_ significantly co-occur in LGGs (OR = 76.1, 95% CI: 9.85 - 588.1, q = 3.26e-3), consistent with previous findings [@doi:10.1200/JCO.2010.31.1670; 10.1186/s40478-020-01027-z].
_TP53_ and _PPM1D_ mutations have been shown to be mutually exclusive in HGATs, and our data recapitulates that trend (52/54 or 96.3% of tumors have a mutation in either gene, OR = 0.188, 95% CI: 0.04 - 0.94, p = 4.13e-2, q = 5.87e-2) [@doi:10.1038/ng.2938].
We next assessed the contributions of eight previously identified adult CNS-specific mutational signatures [@doi:10.1038/s43018-020-0027-5] (RefSig) across cancer groups **Figure {@fig:Fig3}C** and samples **Figure {@fig:FigS4}A**.
Stage 0 and/or 1 tumors characterized by low TMBs **Figure {@fig:FigS2}G** such as LGGs, gangliogliomas, craniopharyngiomas, DNETs, and schwannomas are expectedly dominated by Signature 1, which results from the normal process of spontaneous deamination of 5-methylcytosine.
Signature N6 is CNS-specific signature which we observe nearly universally across samples.
Drivers of Signature 18, _TP53_, _APC_, _NOTCH1_ (https://signal.mutationalsignatures.com/explore/referenceCancerSignature/31/drivers), are also canonical drivers of medulloblastoma tumors, and indeed, we observe Signature 18 as the most common signature in medulloblastoma tumors.
Signatures 3, 8, 18, and MMR2 are prevalent in HGGs, including DMGs.
Finally, we observe that the weight of Signature 1 is higher at diagnosis (pre-treatment) and is almost always lower in tumors at later phases of therapy (progression, recurrence, post-mortem, secondary malignancy) **Figure {@fig:FigS4}B**.
This trend may be the result of therapy-induced mutations which produce additional signatures (e.g., temozolomide treatment drives Signature 11), subclonal expansion, and/or acquisition of additional driver mutations during tumor progression, leading to higher overall TMBs and additional signatures.

<!-- TODO: possibly explore gene mutations driving signature 8, specifically for meningiomas, and describe -->

### Genomic instability of pediatric brain tumors 

We developed a method for generating consensus copy number altered regions from MantaSV, CNVkit, and ControlFREEC (**STAR Methods**). 
This resulted in high-confidence gains, losses, amplifications, and deep deletion calls used as input for breakpoint density (**Table S2**) and chromothripsis analysis.
We observed that HGATs, followed by medulloblastomas, have the most unstable genomes (**Figure S3A**).
By contrast, craniopharyngiomas generally lack somatic copy number variation.
These patterns of copy number variation largely align with estimates of tumor mutational burden (**Figure S2G**). 
The number of SV and CNV breakpoints were significantly correlated across tumors (p = 1.08e-37) (**Figure {@fig:Fig3}D**) and as expected, the number of chromothripsis regions called increases as breakpoint density increases (**Figure S3B-C**).
Chromothripsis events were observed in 41% (N = 19/46) of non-midline high-grade gliomas and 28.2% (N = 11/39) of DMGs (**Figure {@fig:Fig3}E**). 
We also found evidence of chromothripsis in over 15% of embryonal tumors, ependymomas, meningiomas, germinomas, glial-neuronal tumors, chordomas, metastatic secondary tumors, and sarcomas, highlighting the genomic instability and complexity of pediatric brain tumors.

![**Figure 3. Mutational co-occurrence and signatures highlight key oncogenic drivers.** A, Bar plot of occurrence and co-occurrence of nonsynonymous mutations for the 50 most commonly mutated genes across all tumor types (annotated from `cancer_group` if N >= 10 or `Other` if N < 10); B, Co-occurrence and mutual exclusivity of nonsynonymous mutations between genes; The co-occurrence score is defined as $I(-\log_{10}(P))$ where $P$ is defined by Fisher's exact test and $I$ is 1 when mutations co-occur more often than expected and -1 when exclusivity is more common; C, Sina plots of RefSig signature weights for signatures 1, 3, 8, 11, 18, 19, N6, MMR2, and Other across cancer groups. Box plot lines represent the first quartile, median, and third quartile. D, The number of SV breaks significantly correlate with CNV breaks (Adjusted R = 0.436, p = 1.08e-37). E, Chromothripsis frequency across pediatric brain tumors shown by `cancer_group` with N >= 3.](images/main/Figure3.png){#fig:Fig3}

### Transcriptomic Landscape of Pediatric Brain Tumors

#### Histologic and oncogenic pathway clustering

UMAP visualization of gene expression variation across brain tumors (**Figure {@fig:Fig4}A**) shows the expected clustering of brain tumors by histology.
We observed medulloblastomas cluster by molecular subtype with WNT and SHH in distinct clusters and Groups 3 and 4 showing some overlap (**Figure {@fig:FigS5}A**), as expected.
Of note, two samples annotated as the SHH subtype do not cluster with the MB samples and one clusters with Group 3 and 4 samples, suggesting potential subtype misclassification or different underlying biology of these tumors.
Additionally, except for three outliers, _C11orf95::RELA_ (_ZFTA::RELA_) fusion positive ependymomas fall within distinct clusters (**Figure {@fig:FigS5}B**).
_BRAF_-driven low-grade gliomas (**Figure {@fig:FigS5}C**) were present in three separate clusters, suggesting that there might be distinct underlying biology within these tumors.
Histone H3 G35-mutant HGATs generally clustered together, away from K28-mutant tumors (**Figure {@fig:FigS5}D**).
Interestingly, although H3 K28-mutant tumors have different biological drivers than H3 wildtype tumors, they did not form distinct clusters, suggesting they may either be driven by common transcriptional programs or our sample size is too small to detect transcriptional differences.

We performed gene set variant analysis (GSVA) for Hallmark cancer gene sets, demonstrating activation of underlying oncogenic pathways (**Figure {@fig:Fig4}B**).	

<!-- TODO: Add immune deconvolution figure 4C information. -->

![**Figure 4. Transcriptomic overview** A, First two dimensions from UMAP of sample transcriptome data.
Points are colored by `cancer_group` of the samples they represent. B, Heatmap of GSVA scores for Hallmark gene sets with significant differences, with samples ordered by `cancer_group`. C, <!-- TODO: Add description about immune deconvolution image we use as main. -->](images/main/Figure4.png){#fig:Fig4}


#### Prediction of _TP53_ oncogenicity and telomerase activity
To understand the _TP53_ phenotype in each tumor, we ran a classifier previously trained on TCGA [@doi:10.1016/j.celrep.2018.03.076] to infer _TP53_ inactivation status.
Using high-confidence SNVs, CNVs, SVs, and fusions in _TP53_ as true positive alterations, we achieved a high accuracy (AUROC = 0.85) for rRNA-depleted, stranded samples compared to randomly shuffled _TP53_ scores (**Figure {@fig:Fig5}A**). 
The classifier did not perform well on the poly-A samples (**Figure {@fig:FigS5}E-F**), potentially due to the low number of _TP53_ altered (N = 29) and/or total poly-A samples in our dataset (N = 58) rather than library type, as a previous study demonstrated high accuracy of this classifier on another poly-A dataset [@doi:10.1016/j.celrep.2019.09.071].
We annotated _TP53_ alterations as "activated" if samples harbored one of p.R273C or p.R248W mutations [@doi:10.1038/ng0593-42], "loss" if the patient had a Li Fraumeni Syndrome (LFS) predisposition diagnosis or if any other SNV, CNV, SV, or fusion fell within a functional domain.
If the _TP53_ mutation did not reside within the DNA-binding domain or if an alteration was not detected in _TP53_, we annotated the tumor as "other".
Interestingly, we observed that samples annotated as either "activated" or "loss" had significantly higher _TP53_ scores than those annotated as "other" (**Figure {@fig:Fig5}B**, p<sub>adj</sub> loss vs. other < 2e-16, p<sub>adj</sub> activated vs. other = 4.0e-5), suggesting that the classifier detects an oncogenic, or altered, _TP53_ phenotype (scores > 0.5) rather than solely _TP53_ inactivation, as interpreted previously [@doi:10.1016/j.celrep.2018.03.076].
Moreover, tumors with "activating" _TP53_ mutations had evidence of higher _TP53_ expression than those with _TP53_ "loss" mutations (Wilcoxon p = 3.5e-3, **Figure {@fig:Fig5}C**.
To further validate the classifier's accuracy, we assessed _TP53_ scores for patients with LFS, hypothesizing all of these tumors would have high scores.
Indeed, we observed higher scores in LFS tumors (N = 8) for which we detected high-confidence _TP53_ somatic alterations. 
Although we were unable to detect canonical somatic _TP53_ mutations in two LFS patient tumors with low _TP53_ scores, we confirmed the LFS diagnosis from pathology reports and found each to have a germline variant in _TP53_.
The tumor purity of these samples was low (16% and 37%), suggesting the classifier requires a certain level of tumor purity to achieve good performance, as we expect _TP53_ to be intact in normal cells.
Tumors with the highest median _TP53_ scores were those known to harbor somatic _TP53_ alterations: choroid plexus tumors, embryonal tumors, HGATs, and pineal tumors (**Figure {@fig:Fig5}E**), while melanocytic tumors, meningiomas, and tumors of cranial and paraspinal nerves had the lowest scores.
	
We next used gene expression data to predict telomerase activity using EXpression-based Telomerase ENzymatic activity Detection (EXTEND) [@doi:10.1038/s41467-020-20474-9] as a surrogate measure of malignant potential [@doi:10.1038/s41467-020-20474-9; 10.1093/carcin/bgp268].
EXTEND scores significantly correlated with _TERT_ (R = 0.55) and _TERC_ (R = 0.58) expression (**Figure {@fig:FigS5}G-H**).  
We found aggressive tumors such as CNS lymphoma, ETMR, ATRT, DMG, and HGG had high EXTEND scores (**Figure {@fig:Fig5}G**), while benign lesions such as teratomas, dysplasias, and hemangioblastomas had the lowest scores (**Table S3**).

<!-- TODO: Fig 5 will likely be changing and need an update. -->

![**Figure 5. _TP53_ and telomerase activity** A, Receiver Operating Characteristic for _TP53_ classifier run on FPKM of stranded RNA-Seq samples. B, Violin and box plots of _TP53_ scores plotted by _TP53_ alteration type. C, Violin and box plots of _TP53_ RNA expression plotted by _TP53_ activation status. D, Box plots of _TP53_ scores grouped by `broad_histology`. E, Box plots of EXTEND scores grouped by `broad_histology`.](images/main/Figure5.png){#fig:Fig5}


## Discussion

Stub in discussion section   


## Acknowledgments

We graciously thank the patients and families who have donated their tumors to the Children's Brain Tumor Network and/or the Pacific Pediatric Neuro-oncology Consortium, without which, this research would not be possible. 
This work was funded through the Alex’s Lemonade Stand Foundation (ALSF) Childhood Cancer Data Lab (JNT, CSG, JAS, CLS, CJB), ALSF Young Investigator Award (JLR), ALSF Catalyst Award (JLR, ACR, PBS), Children’s Hospital of Philadelphia Division of Neurosurgery (PBS and ACR), the Australian Government, Department of Education (APH), and NIH Grants 3P30 CA016520-44S5 (ACR), U2C HL138346-03 (ACR, APH), and U24 CA220457-03 (ACR).
This project has been funded in part with Federal funds from the National Cancer Institute, National Institutes of Health, under Contract No. 75N91019D00024, Task Order No. 75N91020F00003 (JLR, ACR, APH). 
The content of this publication does not necessarily reflect the views or policies of the Department of Health and Human Services, nor does mention of trade names, commercial products or organizations imply endorsement by the U.S. Government.


## Author Contributions

|Author|Contributions|
|---|---|
|Joshua A. Shapiro|Methodology, Software, Validation, Formal analysis, Investigation, Writing - Original draft, Writing - Review and editing, Visualization, Supervision|
|Candace L. Savonen|Methodology, Software, Validation, Formal analysis, Investigation, Writing - Original draft, Visualization|
|Chante J. Bethell|Methodology, Validation, Formal analysis, Investigation, Writing - Original draft, Visualization|
|Krutika S. Gaonkar|Data curation, Formal Analysis, Investigation, Methodology, Software, Writing – original draft|
|Run Jin|Data curation, Formal Analysis, Visualization, Writing – original draft|
|Yuankun Zhu|Data curation, Formal Analysis, Investigation, Methodology, Supervision|
|Miguel A. Brown|Data curation, Methodology|
|Nhat Duong|Formal Analysis, Investigation, Methodology|
|Komal S. Rathi|Formal Analysis, Investigation, Methodology, Writing – original draft|
|Nighat Noureen|Formal analysis, Visualization, Writing - Original draft|
|Bo Zhang|Data curation, Formal Analysis|
|Brian M. Ennis|Data curation, Formal Analysis|
|Stephanie J. Spielman|Validation, Formal analysis, Writing - Review and editing, Visualization, Supervision|
|Laura E. Egolf|Formal analysis, Writing - Original draft|
|Bailey Farrow|Data curation, Software|
|Nicolas Van Kuren|Data curation, Software|
|Tejaswi Koganti|Formal Analysis, Investigation|
|Shrivats Kannan|Formal Analysis, Methodology, Writing – original draft|
|Pichai Raman|Conceptualization, Formal Analysis, Methodology|
|Jennifer Mason|Supervision|
|Daniel P. Miller|Formal Analysis|
|Anna R. Poetsch|Formal Analysis|
|Payal Jain|Data curation, Investigation, Validation|
|Adam A. Kraya|Methodology|
|Allison P. Heath|Project administration|
|Mateusz P. Koptyra|Formal Analysis, Writing – original draft|
|Shannon Robbins|Data curation|
|Yiran Guo|Formal Analysis|
|Xiaoyan Huang|Formal Analysis|
|Jessica Wong|Writing – original draft|
|Mariarita Santi|Investigation, Validation|
|Angela Viaene|Investigation, Validation|
|Laura Scolaro|Data Curation|
|Angela Waanders|Supervison|
|Derek Hanson|Validation|
|Hongbo M. Xie|Methodology, Supervision|
|Siyuan Zheng|Formal analysis, Visualization, Writing - Original draft, Supervision|
|Cassie N. Kline|Supervision|
|Jena V. Lilly|Conceptualization, Funding acquisition, Project administration|
|Philip B. Storm|Conceptualization, Funding acquisition, Resources|
|Adam C. Resnick|Conceptualization, Funding acquisition, Resources, Supervision|
|Casey S. Greene|Conceptualization, Funding acquisition, Methodology, Project administration, Software, Supervision, Writing – review & editing|
|Jo Lynne Rokita*|Conceptualization, Data curation, Formal Analysis, Funding acquisition, Investigation, Methodology, Software, Supervision, Writing – original draft|
|Jaclyn N. Taroni*|Methodology, Software, Validation, Formal analysis, Investigation, Data curation, Writing - Review and editing, Visualization, Supervision, Project administration|
|Children's Brain Tumor Network|Conceptualization|
|Pacific Pediatric Neurooncology Consortium|Conceptualization|


## Declarations of Interest

CSG's spouse was an employee of Alex's Lemonade Stand Foundation, which was a sponsor of this research.
JAS, CLS, CJB, SJS, and JNT are or were employees of Alex's Lemonade Stand Foundation, a sponsor of this research.

## Figure Titles and Legends
**Figure 1. Overview of the OpenPBTA Project.**

## STAR METHODS

### RESOURCE AVAILABILITY

#### Lead contact

Requests for access to OpenPBTA raw data and/or specimens may be directed to, and will be fulfilled by Jo Lynne Rokita (rokita@chop.edu).

#### Materials availability

This study did not create new, unique reagents.

#### Data and code availability

Raw and harmonized WGS, WXS, and RNA-Seq data derived from human samples are available within the KidsFirst Portal [@doi:10.24370/OPENPBTA] upon access request to the CBTN (https://cbtn.org/) as of the date of the publication.
In addition, merged summary files are openly accessible at https://cavatica.sbgenomics.com/u/cavatica/openpbta or via download script from https://github.com/AlexsLemonade/OpenPBTA-analysis/.
Summary data are visible within Pedcbioportal at https://pedcbioportal.kidsfirstdrc.org/study/summary?id=openpbta.
Links or DOIs are listed in the *Key Resources Table*.
<!-- TODO: create data dois -->

All original code has been deposited in the following repositories and is publicly available as of the date of the publication:
- Primary data analyses: https://github.com/d3b-center/OpenPBTA-workflows/
- Downstream data analyses: https://github.com/AlexsLemonade/OpenPBTA-analysis/
- Manuscript code: https://github.com/AlexsLemonade/OpenPBTA-manuscript
Links or DOIs are listed in the *Key Resources Table*.
<!-- TODO: create code dois -->

Any additional information required to reanalyze the data reported in this paper is available
from the lead contact upon request.


### METHOD DETAILS

#### Biospecimen Collection

The Pediatric Brain Tumor Atlas specimens are comprised of samples from Children's Brain Tumor Network (CBTN) and the Pediatric Pacific Neuro-oncology Consortium (PNOC).
The CBTN [@url:https://CBTN.org] is a collaborative, multi-institutional (18 institutions worldwide) research program dedicated to the study of childhood brain tumors.
The Pacific Pediatric Neuro-Oncology Consortium (PNOC) [@url:https://www.pnoc.us/] is an international consortium dedicated to bringing new therapies to children and young adults with brain tumors.
PNOC collected blood and tumor biospecimens from newly-diagnosed diffuse intrinsic pontine glioma (DIPG) patients as part of the clinical trial [PNOC003/NCT02274987](https://clinicaltrials.gov/ct2/show/NCT02274987) [@doi:10.1002/ijc.32258].

All CBTN data can be download from the Gabriella Miller Kids First Data Resource Center, [@url:https://kidsfirstdrc.org].
The de-identified patient's blood and tumor tissue were prospectively collected by the consortium from patients enrolled within the CBTN.

The cell lines were generated by the CBTN from either fresh tumor tissue obtained directly from surgery performed at Children’s Hospital of Philadelphia (CHOP) or from prospectively collected tumor specimens stored in Recover Cell Culture Freezing media (cat# 12648010, Gibco).
The tissue was dissociated using enzymatic method with papain as described [@doi:10.1101/656587].
Briefly, tissue was washed with HBSS (cat# 14175095, Gibco), minced and incubated with activated papain solution (cat# LS003124, SciQuest) for up to 45 minutes.
The papain was inactivated using ovomucoid solution (cat# 542000, SciQuest), tissue was briefly treated with DNase (cat#  10104159001, Sigma) and passed through the 100μm cell strainer (cat# 542000, Greiner Bio-One).
Two cell culture conditions were initiated based on the number of cells available.
For cultures utilizing the fetal bovine serum (FBS), a minimum density of 3×10^5 cells/ml were plated in DMEM/F-12 medium (cat# D8062, Sigma) supplemented with 20% FBS (cat# SH30910.03, Hyclone), 1% GlutaMAX (cat# 35050061, Gibco), Penicillin/Streptomycin-Amphotericin B Mixture (cat# 17-745E, Lonza) and 0.2% Normocin (cat# ant-nr-2, Invivogen).
For the serum-free media conditions cells were plated at minimum density of 1×10^6 cells/ml in DMEM/F12 media supplemented with 1% GlutaMAX, 1x B-27 supplement minus vitamin A (cat# 12587-010, Gibco), 1x N-2 supplement (cat# 17502001, Gibco), 20 ng/ml epidermal growth factor (cat# PHG0311L, Gibco), 20 ng/ml basic fibroblast growth factor (cat# 100-18B, PeproTech), 2.5μg/ml heparin (cat# H3149, Sigma), Penicillin/Streptomycin-Amphotericin B Mixture and 0.2% Normocin.

#### Nucleic acids extraction and library preparation

##### PNOC samples

The Translational Genomic Research Institute (TGEN; Phoenix, AZ) performed DNA and RNA extractions on tumor biopsies using a DNA/RNA AllPrep Kit (Qiagen, #80204).
All RNA used for library prep had a minimum RIN of 7 but no QC thresholds were implemented for the DNA.
For library preparation, 500ng of nucleic acids were used as input for RNA-Seq, WXS, and targeted DNA panel (panel).
The RNA prep was performed using the TruSeq RNA Sample Prep Kit (Illumina, #FC-122-1001) and the exome prep was performed using KAPA Library Preparation Kit (Kapa Biosystems, #KK8201) using Agilent's SureSelect Human All Exon V5 backbone with custom probes.
The targeted DNA panel developed by Ashion (formerly known as the GEM Cancer panel) consisted of exonic probes against 541 cancer genes.
Both panel and WXS assays contained 44,000 probes across evenly spaced genomic loci used for genome-wide copy number analysis.
For the panel, additional probes tiled across intronic regions of 22 known tumor suppressor genes and 22 genes involved in common cancer translocations for structural analysis.
All extractions and library preparations were performed according to manufacturer's instructions.

##### CBTN samples

Blood, tissue, and cell line DNA/RNA extractions were performed at the Biorepository Core at CHOP.
Briefly, 10-20 mg frozen tissue, 0.4-1ml of blood or 2×10^6 cells pellet was used for extractions.
Tissues were lysed using a Qiagen TissueLyser II (Qiagen) with 2×30 sec at 18Hz settings using 5 mm steel beads (cat# 69989, Qiagen).
Both tissue and cell pellets processes included a CHCl3 extraction and were run on the QIACube automated platform (Qiagen) using the AllPrep DNA/RNA/miRNA Universal kit (cat# 80224, Qiagen).
Blood was thawed and treated with RNase A (cat#, 19101, Qiagen); 0.4-1ml was processed using the Qiagen QIAsymphony automated platform (Qiagen) using the QIAsymphony DSP DNA Midi Kit (cat# 937255, Qiagen).
DNA and RNA quantity and quality was assessed by PerkinElmer DropletQuant UV-VIS spectrophotometer (PerkinElmer) and an Agilent 4200 TapeStation (Agilent, USA) for RIN and DIN (RNA Integrity Number and DNA Integrity Number, respectively).
Library preparation and sequencing was performed by the NantHealth sequencing center.
Briefly, DNA sequencing libraries were prepared for tumor and matched-normal DNA using the KAPA HyperPrep kit (cat# KK8541, Roche); tumor RNA-Seq libraries were prepared using KAPA Stranded RNA-Seq with RiboErase kit (cat# KK8484, Roche).
Whole genome sequencing (WGS) was performed at an average depth of coverage of 60X for tumor samples and 30X for germline.
The panel tumor sample was sequenced to 470X and the normal panel sample was sequenced to 308X.
RNA samples were sequenced to an average of 200M reads.
All samples were sequenced on the Illumina HiSeq platform (X/400) (Illumina) with 2 × 150bp read length.

#### Data generation

NantHealth Sequencing Center (Culver City, CA) performed whole genome sequencing (WGS) on all paired tumor (~60X) and constitutive (~30X) DNA samples.
WGS libraries were 2x150 bp and sequenced on an Illumina X/400.
NantHealth Sequencing Center performed ribosomal-depleted whole transcriptome stranded RNA-Seq to an average depth of 100M reads for CBTN tumor samples.
The Translational Genomic Research Institute (TGEN; Phoenix, AZ) performed paired tumor (~200X) and constitutive whole exome sequencing (WXS) or targeted DNA panel (panel) and poly-A selected RNA-Seq (~200M reads) for  PNOC tumor samples.
PNOC WXS and RNA-Seq libraries 2x100 bp and sequenced on an Illumina HiSeq 2500.

#### DNA WGS Alignment

We used BWA-MEM [@arxiv:1303.3997] v0.7.17 for alignment of paired-end DNA-seq reads.
We used version 38, patch release 12 of the _Homo sapiens_ genome as our alignment reference, which we obtained as a FASTA file from UCSC [@url:http://hgdownload.soe.ucsc.edu/goldenPath/hg38/bigZips/].
Alignments were further processed using following the Broad Institute's Best Practices [@url:https://software.broadinstitute.org/gatk/best-practices/workflow?id=11165] for processing Binary Alignment/Map files (BAMs) in preparation for variant discovery.
Duplicates were marked using SAMBLASTER [@doi:10/f6kft3] v0.1.24, BAMs merged and sorted using Sambamba [@doi:10/gfzsfw] v0.6.3.
Resultant BAMs were processing using Broad's Genome Analysis Tool Kit [GATK] (https://software.broadinstitute.org/gatk/) v4.0.3.0, BaseRecalibrator submodule.
Lastly, for normal/germline input, we run the GATK HaplotypeCaller [@doi:10.1101/201178] submodule on the recalibrated BAM, generating a genomic variant call format (GVCF) file.
This file is used as the basis for germline calling, described in the "SNV calling for B-allele Frequency (BAF) generation" section.
References can be obtained from the [Broad Genome References on AWS](https://s3.console.aws.amazon.com/s3/buckets/broad-references/hg38/v0/) bucket, with a general description of references here [@url:https://s3.amazonaws.com/broad-references/broad-references-readme.html].

#### Quality Control of Sequencing Data

NGSCheckmate [@doi:10.1093/nar/gkx193] was performed on matched tumor/normal CRAM files to confirm sample matches and remove mismatched samples from the dataset.
CRAM inputs were preprocessed using BCFtools to filter and call 20k common single nucleotide polymorphisms (SNPs) using default parameters[@url:https://github.com/parklab/NGSCheckMate] and the resulting VCFs were used to run NGSCheckmate using [this workflow](https://github.com/d3b-center/ngs_checkmate_wf) in the D3b GitHub repository.
Per author guidelines, <= 0.61 was used as a correlation coefficient cutoff at sequencing depths >10 to predict mismatched samples.
For RNA-Seq, read strandedness was determined by running the [`infer_experiment.py` script](http://rseqc.sourceforge.net/#infer-experiment-py) on the first 200k mapped reads.
If calculated strandedness did not match strandedness information received from the sequencing center, samples were removed from analysis.
We required at least 60% of RNA-Seq reads mapped to the human reference or samples were removed from analysis.
MEND QC [@doi:10.1101/716829] was performed on aligned RNA-Seq reads using [this workflow](https://github.com/d3b-center/publication_workflows/blob/master/openPBTA/kfdrc-mendqc-wf.cwl) to identify mapped exonic non-duplicate reads.

#### Germline Variant Calling

##### SNP calling for B-allele Frequency (BAF) generation

Germline haplotype calls were performed following the [GATK Joint Genotyping Workflow](https://software.broadinstitute.org/gatk/best-practices/workflow?id=11145), except the workflow was run on an individual sample basis.
This workflow was applied to the GVCF output from the alignment workflow on normal/germline samples.
Using only SNPs, we applied the [GATK generic hard filter suggestions](https://gatkforums.broadinstitute.org/gatk/discussion/2806/howto-apply-hard-filters-to-a-call-set) to the VCF, with an additional requirement of 10 reads minimum depth per SNP.
This filtered VCF was used as input to Control-FREEC and CNVkit (below) for generation of BAF files.
GATK v4.0.12.0 was used for all steps except `VariantFiltration`, which used 3.8.0 because as of GATK 4.0.12.0, this tool was beta and known to be unreliable for this purpose.
This single-sample workflow can be found in the [Kids First GitHub repository](https://github.com/kids-first/kf-jointgenotyping-workflow).
References can be obtained from the [Broad Genome References on AWS](https://s3.console.aws.amazon.com/s3/buckets/broad-references/hg38/v0/) bucket, with a general description of references here [@url:https://s3.amazonaws.com/broad-references/broad-references-readme.html].

#### Somatic Mutation Calling

##### SNV and indel calling

For PBTA samples, we used four variant callers to call SNVs and indels from targeted DNA panel, WXS, and WGS data: Strelka2 [@doi:10.1038/s41592-018-0051-x], Mutect2 [@doi:10.1101/861054], Lancet  [@doi:10.1038/s42003-018-0023-9], and VarDict [@doi:10.1093/nar/gkw227].
WXS samples from TCGA were run using Strelka2, Mutect2 and Lancet.
TCGA samples were captured using different WXS target capture kits and all the BED files were downloaded from [`GDC portal`](https://api.gdc.cancer.gov/files).
The input interval BED files for both panel and WXS data for PBTA  samples were provided by the manufacturers.
For both PBTA and TCGA, all panel and WXS  BED files were padded by 100 bp on each side during Strelka2, Mutect2, and VarDict runs and 400 bp for the Lancet run.  
For WGS calling, we utilized the non-padded BROAD Institute interval calling list [`wgs_calling_regions.hg38.interval_list`](https://console.cloud.google.com/storage/browser/_details/genomics-public-data/resources/broad/hg38/v0/wgs_calling_regions.hg38.interval_list), comprised of the full genome minus N bases, unless otherwise noted below.
Strelka2 [@doi:10/gdwrp4] v2.9.3 was run using default parameters for canonical chromosomes (chr1-22, X,Y,M), as recommended by the authors.
The final Strelka2 VCF was filtered for PASS variants.
Mutect2 from GATK v4.1.1.0 was run following Broad best practices outlined from their Workflow Description Language (WDL) [@url:https://github.com/broadinstitute/gatk/blob/4.1.1.0/scripts/mutect2_wdl/mutect2.wdl].
The final Mutect2 VCF was filtered for PASS variants.
To manage memory issues, VarDictJava [@doi:10.1093/nar/gkw227] v1.58 [@url:https://github.com/AstraZeneca-NGS/VarDictJava] was run using 20Kb interval chunks of the input BED, padded by 100 bp on each side, such that if an indel occurred in between intervals, it would be captured.
Parameters and filtering followed [BCBIO standards](https://github.com/bcbio/bcbio-nextgen) except that variants with a variant allele frequency (VAF) >= 0.05 (instead of >= 0.10) were retained.
The 0.05 VAF increased the true positive rate for indels and decreased the false positive rate for SNVs when using VarDict in consensus calling.
The final VCF was filtered for PASS variants with TYPE=StronglySomatic.
Lancet v1.0.7 was run using default parameters, except for those noted below.
For input intervals to Lancet WGS, a reference BED was created by using only the UTR, exome, and start/stop codon features of the GENCODE 31 reference, augmented as recommended with PASS variant calls from Strelka2 and Mutect2 [@doi:10.1101/623702].
These intervals were then padded by 300 bp on each side during Lancet variant calling.
Per recommendations by the New York Genome Center [@doi:10.1101/623702], for WGS samples, the Lancet input intervals described above were augmented with PASS variant calls from Strelka2 and Mutect2 as validation.

##### VCF annotation and MAF creation

Normalization of INDELs using `bcftools norm` [@url:http://samtools.github.io/bcftools/bcftools.html#norm] was performed on all PASS VCFs using the following subworkflow [@url:https://github.com/d3b-center/OpenPBTA-workflows/blob/master/cwl/kfdrc_annot_vcf_sub_wf.cwl], release v3.
The ENSEMBL Variant Effect Predictor [@doi:10/gdz75c], reference release 93, was used to annotate variants and bcftools was used to add population allele frequency (AF) from gnomAD.
SNV and INDEL hotspots from v2 of MSKCC's database [@url:https://www.cancerhotspots.org/#/download] plus the C228T and C250T TERT promoter mutations [@doi:10.3390/ijms21176034] were annotated.
SNVs were annotated by matching amino acid position (`Protein_position` column in MAF file) with SNVs in the MSKCC database, splice sites were matched to `HGVSp_Short`values in the MSKCC database, and INDELs were matched based on amino acid present within the range of INDEL hotspots values in the MSKCC database.
Non-hotspot annotated variants with a normal depth of <= 7 and/or gnomAD AF > 0.001 were removed as potential germline variants.
TERT promoter mutations were matched using hg38 coordinates from [@doi:10.3390/ijms21176034]: C228T occurs at 5:1295113, is annotated as existing variant `s1242535815`, `COSM1716563`, `COSM1716558`, and is 66bp away from TSS and C250T occurs at Chr5:1295135, is annotated as existing variant `COSM1716559`, and is 88 bp away from TSS.

The final set of variants were retained if annotated as PASS or HotSpotAllele=1. 
MAFs were created using MSKCC's vcf2maf [@url:https://github.com/mskcc/vcf2maf] v1.6.17.

##### Gather SNV and INDEL Hotspots
All variant calls from Strelka2, Mutect2, or Lancet that overlap with an SNV or INDEL hotspot from v2 of MSKCC's database [@url:https://www.cancerhotspots.org/#/download] or the C228T and C250T TERT promoter mutations [@doi:10.3390/ijms21176034] were retained in a hotspot-specific MAF file, which was used for select analyses as described in the methods below.
VarDict-only calls were not retained since ~ 39M calls with low VAF were uniquely called and may be potential false positives.

##### Consensus SNV Calling

Our SNV calling process led to separate sets of predicted mutations for each caller.
We considered mutations to describe the same change if they were identical for the following MAF fields: `Chromosome`, `Start_Position`, `Reference_Allele`,  `Allele`, and `Tumor_Sample_Barcode`.
Strelka2 does not call multinucleotide variants (MNV), but instead calls each component SNV as a separate mutation, so we separated MNV calls from Mutect2 and Lancet into consecutive SNVs before comparing them with Strelka2.
We examined the variant allele frequencies produced by each caller and compared their overlap with each other [@url:https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/snv-callers/compare_snv_callers_plots.Rmd].
VarDict calls included many variants that were not identified by other callers [@url:https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/snv-callers/plots/comparison/upset_plot.png], while the other callers produced results that were relatively consistent with one another.
Many of these VarDict-specific calls were variants with low allele frequency [@url:https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/snv-callers/plots/comparison/vaf_violin_plot.png].
We termed mutations shared among the other three callers (Strelka2, Mutect2, and Lancet) to be consensus mutation calls and dropped VarDict due to concerns about it calling a large number of false positives.
In practice, because our filtered set was based on the intersection of these three sets and because VarDict called nearly every mutation from the other three callers plus many that were unique to it, the decision to not consider VarDict calls has little impact on the results.

For some downstream analyses, only coding sequence SNVs (based on GENCODE v27 [@url:https://www.gencodegenes.org/human/release_27.html]) are used, to enhance comparability to other studies.
We considered base pairs to be *effectively surveyed* if they were in the intersection of the genomic ranges considered by the callers used to generate the consensus and where appropriate, regions of interest, such as coding sequences.
This definition of *effectively surveyed* base pairs is what is used to calculate effective genome size for calculations for tumor mutation burden.

#### Somatic Copy Number Variant Calling (WGS samples only)

We used Control-FREEC [@doi:10/ckt4vz; @doi:10/c6bcps] v11.6 and CNVkit [@doi:10.1371/journal.pcbi.1004873] v0.9.3 for copy number variant calls.
For both algorithms, the `germline_sex_estimate` (described below) was used as input for sample sex and germline variant calls (above) were used as input for BAF estimation.
Control-FREEC was run on human genome reference hg38 using the optional parameters of a 0.05 coefficient of variation, ploidy choice of 2-4, and BAF adjustment for tumor-normal pairs.
Theta2 [@doi:10.1093/bioinformatics/btu651] used VarDict germline and somatic calls, filtered on PASS and strongly somatic, to infer tumor purity.
Theta2 purity was added as an optional parameter to CNVkit to adjust copy number calls.
CNVkit was run on human genome reference hg38 using the optional parameters of Theta2 purity and BAF adjustment for tumor-normal pairs.
We used GISTIC [@doi:10.1186/gb-2011-12-4-r41] v.2.0.23 on the CNVkit and the consensus CNV segmentation files to generate gene-level copy number abundance (Log R Ratio) as well as chromosomal arm copy number alterations using the parameters specified in the [OpenPBTA Analysis repository](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/run-gistic/scripts/run-gistic-openpbta.sh).

##### Consensus CNV Calling

For each caller and sample, CNVs were called based on consensus among Control-FREEC [@doi:10/ckt4vz; @doi:10/c6bcps], CNVkit [@doi:10.1371/journal.pcbi.1004873], and Manta [@doi:10/gf3ggb].
CNVs called significant by Control-FREEC (p-value < 0.01) and Manta calls that passed all filters [@url:https://github.com/Illumina/manta/blob/75b5c38d4fcd2f6961197b28a41eb61856f2d976/docs/userGuide/README.md#vcf-format-fields] were included in consensus calling.
Sample and caller combination files with more than 2500 CNVs called were removed from the set; we expect these to be noisy and poor quality samples based on cutoffs used in GISTIC [@doi:10.1186/gb-2011-12-4-r41]. 
For each sample, the following regions are included in the final consensus set: 1) regions with reciprocal overlap of at least 50% between two of the three callers; 2) smaller CNV regions that are at least 90% covered by another caller. 
Any copy number alteration that was not called by two or more callers was not included in the consensus file. 
For the samples that are included in the consensus file, if a certain region has a neutral call, copy number of `NA` is defined for that region.
CNV regions within 10,000 bp of each other with the same direction of gain or loss were merged into single region.
We filtered out any CNVs that overlapped 50% or more with immunoglobulin, telomeric, centromeric, segment duplicated regions or were shorter than 3000 bp. 

#### Somatic Structural Variant Calling (WGS samples only)

We used Manta SV [@doi:10/gf3ggb] v1.4.0 for structural variant (SV) calls.
Manta SV calling was also limited to regions used in Strelka2.
The hg38 reference for SV calling used was limited to canonical chromosome regions.
The somatic DNA workflow for SNV, indel, copy number, and SV calling can be found in the [KidsFirst Github repository](https://github.com/kids-first/kf-somatic-workflow).
Manta SV output was annotated using [AnnotSV v2.1](https://lbgi.fr/AnnotSV/) [@doi:10.1093/bioinformatics/bty304] and the workflow can be found in the [D3b GitHub repository](https://github.com/d3b-center/annotSV/tree/master).

#### Gene Expression

##### Abundance Estimation

We used STAR [@doi:10/f4h523] v2.6.1d to align paired-end RNA-seq reads.
This output was used for all subsequent RNA analysis.
We used Ensembl GENCODE 27 [@url:https://www.gencodegenes.org/human/release_27.html], "Comprehensive gene annotation" as a reference.
We used RSEM [@doi:10/cwg8n5] v1.3.1 for both FPKM and TPM transcript- and gene-level quantification.
We also added a second method of quantification using kallisto [@doi:10.1038/nbt.3519] v0.43.1.
This method differs in that it uses pseudoaligments using FASTQ reads directly to the aforementioned GENCODE 27 reference.

##### Gene Expression Matrices with Unique HUGO Symbols

Algorithms that perform gene set enrichment, molecular subtyping, or immune-profiling, for example, require an RNA-seq gene expression matrix as input, with HUGO gene symbols as row names and sample names as column names.
There is a small proportion of gene symbols that map to multiple Ensembl gene identifiers (in GENCODE v27, 212 gene symbols map to 1866 Ensembl gene identifiers), termed multi-mapped gene symbols.

We first removed genes with no expression from the RSEM abundance data using a cut-off of FPKM > 0 in at least 1 sample across the PBTA cohort.
We computed the mean FPKM across all samples per gene and for each multi-mapped gene symbol, we chose the Ensembl identifier corresponding to the maximum mean FPKM with the goal of choosing the identifier that best represented the expression of the gene.
After collapsing gene identifiers, there were a total of 46,400 unique expressed genes in the poly-A dataset and a total of 53,011 unique expressed genes remaining in the stranded dataset.
More detail can be found in the [`collapse-rnaseq` analysis module](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/collapse-rnaseq).

##### Gene fusion detection

We set up [Arriba v1.1.0](https://github.com/suhrig/arriba/) and STAR-Fusion 1.5.0 [@doi:10.1101/120295] fusion detection tools using CWL on CAVATICA.
For both of these tools, we used aligned BAM and chimeric SAM files from STAR as inputs and GRCh38_gencode_v27 GTF for gene annotation.
We ran STAR-Fusion with default parameters and annotated all fusion calls with GRCh38_v27_CTAT_lib_Feb092018.plug-n-play.tar.gz provided in the STAR-fusion release.
For Arriba, we used a blacklist file (blacklist_hg38_GRCh38_2018-11-04.tsv.gz) from the Arriba release to remove recurrent fusion artifacts and transcripts present in healthy tissue.
We also provided Arriba with strandedness information for stranded samples, or we set it to auto-detection for poly-A samples.
We used [FusionAnnotator](https://github.com/FusionAnnotator/FusionAnnotator) on Arriba fusion calls in order to harmonize annotations with those of STAR-Fusion.
The RNA expression and fusion workflows can be found in the [KidsFirst GitHub repository](https://github.com/kids-first/kf-rnaseq-workflow) and the FusionAnnotator workflow in the [D3b GitHub repository](https://github.com/d3b-center/FusionAnnotator).

### QUANTIFICATION AND STATISTICAL ANALYSIS

##### Recurrently mutated genes and co-occurrence of gene mutations

Using the consensus SNV calls, we identified genes that were recurrently mutated in the cohort, including nonsynonymous mutations with a variant allele frequency greater than 5% among the set of independent samples.
The set of nonsynonymous mutations was determined using ENSEMBL Variant Effect Predictor [@doi:10/gdz75c] annotations, including High and Moderate consequence types as defined in `maftools v. 2.2.10` [@doi:10.1101/gr.239244.118].
For each gene, we then tallied the number of samples that had at least one nonsynonymous mutation.

For genes that contained nonsynonymous mutations in multiple samples, we calculated pairwise mutation co-occurrence scores.
This score was defined as the $I\times -\log_{10}(P)$ where $I$ is 1 when the odds ratio is > 1 (indicating co-occurrence), and -1 when the odds ratio is < 1 (indicating mutual exclusivity), with $P$ defined by  Fisher's Exact Test.

##### Focal Copy Number Calling

We added the ploidy inferred via Control-FREEC to the consensus CNV segmentation file and used the ploidy and copy number values to define gain and loss values broadly at the chromosome level.
We used bedtools coverage [@url:https://bedtools.readthedocs.io/en/latest/content/tools/coverage.html; @doi:10.1093/bioinformatics/btq033] to add cytoband status using the UCSC cytoband file [@url:http://hgdownload.cse.ucsc.edu/goldenpath/hg38/database/cytoBand.txt.gz; @doi:10.1093/nar/gks1048].
The output status call fractions, which are values of the loss, gain and callable fractions of each cytoband region, were used to define dominant status at the cytoband-level.
The weighted means of each status call fraction were calculated using band length.
We used the weighted means to define the dominant status at the chromosome arm-level.

A status was considered dominant if more than half of the region was callable and the status call fraction was greater than 0.9 for that region.
The 0.9 threshold was chosen to ensure that the dominant status fraction call is greater than the remaining status fraction calls in a region.

We also wanted to define focal copy number units to avoid calling adjacent genes in the same cytoband or arm as copy number losses or gains where it would be more appropriate to call the broader region a loss or gain.
For the determination of the most focal units, we first considered the dominant status calls at the chromosome arm-level.
If the chromosome arm dominant status was callable but not clearly defined as a gain or loss, we instead included the cytoband-level status call.
Similarly, if a cytoband dominant status call was callable but not clearly defined as a gain or loss, we instead included gene-level status call.
To obtain the gene-level data, we used the mergeByOverlaps function [@url:https://www.rdocumentation.org/packages/IRanges/versions/2.6.1/topics/findOverlaps-methods] from the IRanges package in R [@doi:10.1371/journal.pcbi.1003118] to find overlaps between the segments in the consensus CNV file and the exons in the GENCODE v27 annotation file [@url:https://www.gencodegenes.org/human/release_27.html].
If the copy number value was 0, we set the status to "deep deletion". 
For autosomes only, we set the status to "amplification" when the copy number value was greater than two times the ploidy value.

#### Chromothripsis Analysis (WGS samples only)

Candidate chromothripsis regions were identified in the set of independent tumor WGS samples with ShatterSeek [@doi:10.1038/s41588-019-0576-7], using Manta SV calls that passed all filters and consensus CNV calls. 
Only chromosomes 1-22 and X were considered. 
The consensus CNV data were modified to fit ShatterSeek input requirements: CNV-neutral or excluded regions (both annotated as `NA` in the consensus data) were filled in with the respective sample’s ploidy value from Control-FREEC, and consecutive segments with the same copy number value were merged. 
Candidate chromothripsis regions were classified as high- or low-confidence by applying the statistical criteria described by the ShatterSeek authors.

##### Immune Profiling and Deconvolution

We used the R package `immunedeconv` [@pmid:31510660; @url:https://github.com/icbi-lab/immunedeconv] with the method `quanTIseq` [@10.1186/s13073-019-0638-6] to deconvolute various immune cell types across tumors from the PBTA cohort in the stranded and poly-A collapsed FPKM RNA-seq datasets ([`immune-deconv` analysis module](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/immune-deconv)). 
The `quanTIseq` deconvolution method directly estimates absolute fractions of 10 immune cell types that represent inferred proportions of the cell types in the mixture. 
Therefore, we utilized `quanTIseq` for inter-sample, intra-sample, and inter-histology score comparisons.

##### Gene Set Variation Analysis

We performed Gene Set Variation Analysis (GSVA) [@doi:10.1186/1471-2105-14-7] on collapsed, log2-transformed RSEM FPKM data using the GSVA Bioconductor package [@doi:10.18129/B9.bioc.GSVA] with setting `mx.diff=TRUE` to obtain Gaussian-distributed scores ([`gene-set-enrichment-analysis` analysis module](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/gene-set-enrichment-analysis)) for each of the MSigDB hallmark gene sets [@doi:10.1016/j.cels.2015.12.004].
We compared GSVA scores among histology groups (`short_histology`) using ANOVA and subsequent Tukey tests; p-values were Bonferroni-corrected for multiple hypothesis testing.

##### Transcriptomic Dimension reduction

We applied Uniform Manifold Approximation and Projection (UMAP) [@arxiv:1802.03426v2] to log2-transformed FPKM data using the umap R package [@url:https://cran.r-project.org/package=umap].
We set the number of neighbors to 15 ([`transcriptomic-dimension-reduction` analysis module](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/transcriptomic-dimension-reduction)).

##### Fusion prioritization

We performed artifact filtering and additional annotation on fusion calls to prioritize putative oncogenic fusions.
Briefly, we considered all in frame and frameshift fusion calls with a minimum of 1 junction reads and at least one gene partner expressed (TPM > 1) to be true calls.
If a fusion call had large number of spanning fragment reads compared to junction reads (spanning fragment minus junction read greater than ten), we removed these calls as potential false positives.
We prioritized a union of fusion calls as true calls if the fused genes were detected by both callers, the same fusion was recurrent within a `broad_histology` (>2 samples) or the fusion was specific to the `broad_histology`.
If either 5' or 3' genes fused to more than five different genes within a sample, we removed these calls as potential false positives.
We annotated putative driver fusions and prioritized fusions based on partners containing known [kinases](http://kinase.com/human/kinome/tables/Kincat_Hsap.08.02.xls), [oncogenes](http://www.bushmanlab.org/assets/doc/allOnco_Feb2017.tsv), [tumor suppressors](https://bioinfo.uth.edu/TSGene/Human_TSGs.txt?csrt=5027697123997809089), curated transcription factors [@doi:10.1016/j.cell.2018.01.029], [COSMIC genes](https://cancer.sanger.ac.uk/census), and/or known [TCGA fusions](https://tumorfusions.org/PanCanFusV2/downloads/pancanfus.txt.gz) from curated [references](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/fusion_filtering/references).
_MYBL1_ [@doi:10.1073/pnas.1300252110], _SNCAIP_ [@doi:10.1038/nature11327], _FOXR2_ [@doi:10.1016/j.cell.2016.01.015], _TTYH1_ [@doi:10.1038/ng.2849], and _TERT_ [@doi:10.1038/ng.3438; @doi:10.1002/gcc.22110; @doi:10.1016/j.canlet.2014.11.057; @doi:10.1007/s11910-017-0722-5] were added to the oncogene list and _BCOR_ [@doi:10.1016/j.cell.2016.01.015] and _QKI_ [@doi:10.1038/ng.3500] were added to the tumor suppressor gene list based on pediatric cancer literature review.
The fusion filtering workflow can be found in the [OpenPBTA Analysis repository](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/fusion_filtering).

#### Oncoprint figure generation
Maftools v. 2.2.10 [@doi:10.1101/gr.239244.118] was used to generate oncoprints depicting the frequencies of canonical somatic gene mutations, CNVs, and fusions for the top 20 genes mutated across primary tumors within broad histologies of the OpenPBTA dataset.
Canonical genes were collated from review of the literature for low-grade astrocytic tumors [@doi:10.1186/s40478-020-00902-z], embryonal tumors [@doi:10.1038/nature22973; @doi:10.1007/s00401-020-02182-2; @doi:10.1186/s40478-020-00984-9; @doi:10.1016/j.ccell.2016.02.001; @doi:10.1038/s41598-020-59812-8], diffuse astrocytic and oligodendroglial tumors [@doi:10.1016/j.ccell.2017.08.017; @doi:10.1002/ijc.32258; @doi:10.1093/neuonc/noab106; @doi:10.1186/s40478-020-00905-w],  and other tumors (ependymal tumors, craniopharyngiomas, neuronal-glial mixed tumors, histiocytic tumors, chordoma, meningioma, and choroid plexus tumors) 
[@doi:10.1186/s40478-020-01056-8; @doi:10.1007/s00381-017-3481-3; @doi:10.1016/j.ccell.2015.04.002; 
@doi:10.1038/nature13109; @doi:10.1038/s41525-017-0014-7; @doi:10.3171/2019.8.JNS191266; @doi:10.1007/s00401-016-1539-z; @doi:10.1093/neuonc/noaa267; @doi:10.1016/s0002-9440(10)64477-x; @doi:10.1016/j.jaad.2017.05.059].

#### Mutational Signatures

We obtained weights (i.e., exposures) for signature sets using the `deconstructSigs` R package function `whichSignatures()` [@doi:10.1186/s13059-016-0893-4 ; @url:https://github.com/raerose01/deconstructSigs] from consensus SNVs with the BSgenome.Hsapiens.UCSC.hg38 annotations [@url:https://bioconductor.org/packages/release/data/annotation/html/BSgenome.Hsapiens.UCSC.hg38.html].
Specifically, we estimated signature weights across samples for eight signatures previously identified in the Signal reference set of signatures ("RefSig") as associated with adult central nervous system (CNS) tumors [@doi:10.1038/s43018-020-0027-5].
These eight RefSig signatures are 1, 3, 8, 11, 18, 19, N6, and MMR2.
Weights for signatures fall in the range zero to one inclusive.
`deconstructSigs` estimates the weights for each signature across samples and allows for a proportion of unassigned weights referred to as "Other" in the text.
These results do not include signatures with small contributions; `deconstructSigs` drops signature weights that are less than 6% [@doi:10.1186/s13059-016-0893-4].

### Tumor Mutation Burden

We consider tumor mutation burden (TMB) to be the number of consensus SNVs per *effectively surveyed* base of the genome.

$$ \textrm{TMB} = \frac{\textrm{\# of coding sequence SNVs}}{\textrm{Size in Mb of {\em effectively surveyed} genome}}
$$

We used the total number coding sequence consensus SNVs for the numerator and the size of the intersection of the regions considered by Strelka2 and Mutect2 with coding regions (CDS from GENCODE v27 annotation [@url:https://www.gencodegenes.org/human/release_27.html]) as the denominator.

Lancet showed a bias for calling mutations with a much lower VAF for WXS than WGS data (**Figure S2, panel G**).
Since TCGA data was all WXS, we dropped Lancet from TMB calculations.  [@url: https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/snv-callers/lancet-wxs-tests].

#### Clinical Data Harmonization
<!-- TODO: pull these "S1 and S2 tables out and append to actual Table S1" -->

##### WHO Classification of Disease Types

The `pathology_diagnosis` field in the `pbta-histologies.tsv` file contains one or more diagnoses from on the patient's pathology report.
The `pathology_free_text_diagnosis` field in the `pbta-histologies.tsv` file contains additional free text diagnosis information gathered from the patient's pathology report.
The `broad_histology` denotes the broad 2016 WHO classification for each tumor.
The `short_histology` is an abbreviated version of either the `broad_histology` or `integrated_diagnosis` for plotting purposes.
Except for LGAT samples, the `integrated_diagnosis` field in the `pbta-histologies.tsv` file was derived to match a standardized 2016 WHO diagnosis [@doi:10.1007/s00401-016-1545-1] based on `pathology_diagnosis`, molecular subtyping, and in some cases, additional pathology review.
The `harmonized_diagnosis` is the final `integrated_diagnosis`, if one exists, or a diagnosis derived from the `pathology_diagnosis` and `pathology_free_text_diagnosis` in the absence of molecular data.
The `cancer_group` is a grouping narrower than `broad_histology` derived within the [molecular subtyping integrate module](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/molecular-subtyping-integrate).
With clinician assistance, the `CNS_region` was categorized as hemispheric, midline, mixed, optic pathway, posterior fossa, spine, suprasellar, ventricles or other based on specimen location (see table below).


| Clinical and Histology Metadata | Definition                                                                                                                                  | Possible values                                                                                                                               |
|---------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------|
| age_at_diagnosis_days           | Patient age at diagnosis in days                                                                                                            | numeric                                                                                                                                       |
| age_last_update_days            | Patient age at the last clinical event/update in days                                                                                       | numeric                                                                                                                                       |
| aliquot_id                      | External aliquot identifier                                                                                                                 | variable                                                                                                                                      |
| broad_histology                 | Broad WHO 2016 classification of cancer type                                                                                                | text                                                                                                                                          |
| cancer_group                    | Harmonized cancer groupings for plots                                                                                                       | text                                                                                                                                          |
| cancer_predispositions          | Reported cancer predisposition syndromes                                                                                                    | text                                                                                                                                          |
| CNS_region                      | Harmonized brain region based on `primary_site`                                                                                             | Hemispheric;Midline;Mixed;Optic pathway;Other;Posterior fossa;Spine;Suprasellar;Ventricles                                                    |
| cohort                          | Scientific cohort                                                                                                                           | CBTN;PNOC                                                                                                                                     |
| cohort_participant_id           | Scientific cohort participant ID                                                                                                            | C#####-C######                                                                                                                                |
| composition                     | Sample composition                                                                                                                          | Derived Cell Line;Not Reported;Peripheral Whole Blood;Saliva;Solid Tissue                                                                     |
| ethnicity                       | Patient reported ethnicity                                                                                                                  | text                                                                                                                                          |
| experimental_strategy           | Sequencing strategy                                                                                                                         | WGS;WXS;RNA-Seq;Panel                                                                                                                         |
| extent_of_tumor_resection       | Amount of tumor resected at time of surgical event                                                                                          | Biopsy only;Partial resection;Gross/Near total resection;Not Reported;Unavailable                                                             |
| germline_sex_estimate           | Predicted sex of patient based on germline X and Y ratio calculation (described in methods)                                                 | Female;Male;Unknown                                                                                                                           |
| harmonized_diagnosis            | `integrated_diagnosis`, if exists, or updated and harmonized diagnosis using pathology_free_text_diagnosis information                      | text                                                                                                                                          |
| integrated_diagnosis            | 2016 WHO diagnosis integrated from pathology diagnosis and molecular subtyping                                                              | text                                                                                                                                          |
| Kids_First_Biospecimen_ID       | KidsFirst biospecimen identifier                                                                                                            | BS_########                                                                                                                                   |
| Kids_First_Participant_ID       | KidsFirst patient identifier                                                                                                                | PT_########                                                                                                                                   |
| molecular_subtype               | Molecular subtype defined by WHO 2016 guidelines                                                                                            | text                                                                                                                                          |
| normal_fraction                 | Theta2 normal DNA fraction estimate                                                                                                         | numeric                                                                                                                                       |
| Notes                           | Free text field describing changes from `pathology_diagnosis` to `integrated_diagnosis` or manner in which molecular_subtype was determined | text                                                                                                                                          |
| OS_days                         | Overall survival in days                                                                                                                    | numeric                                                                                                                                       |
| OS_status                       | Overall survival status                                                                                                                     | DECEASED;LIVING                                                                                                                               |
| parent_aliquot_id               | External identifier combining sample_id, sample_type, aliquot_id, and sequencing_strategy for some samples                                  | text                                                                                                                                          |
| pathology_diagnosis             | Reported and/or harmonized patient diagnosis from pathology reports                                                                         | text                                                                                                                                          |
| pathology_free_text_diagnosis   | Free text patient diagnosis from pathology reports                                                                                          | text                                                                                                                                          |
| PFS_days                        | Progression-free survival in days                                                                                                           | numeric                                                                                                                                       |
| primary_site                    | Bodily site(s) from which specimen was derived                                                                                              | text                                                                                                                                          |
| race                            | Patient reported race                                                                                                                       | text                                                                                                                                          |
| reported_gender                 | Patient reported gender                                                                                                                     | text                                                                                                                                          |
| RNA_library                     | Type of RNA-Sequencing library preparation                                                                                                  | stranded;poly-A                                                                                                                               |
| sample_id                       | External biospecimen identifier                                                                                                             | variable                                                                                                                                      |
| sample_type                     | Broad sample type                                                                                                                           | Normal;Tumor                                                                                                                                  |
| seq_center                      | Sequencing center                                                                                                                           | BGI;BGI@CHOP Genome Center;Genomic Clinical Core at Sidra Medical and Research Center;NantOmics;The Translational Genomics Research Institute |
| short_histology                 | Abbreviated `integrated_diagnosis` or `broad_histology` for plotting purposes                                                               | text                                                                                                                                          |
| tumor_descriptor                | Phase of therapy from which tumor was derived                                                                                               | Initial CNS Tumor;Progressive Progressive Disease Post-Mortem;Recurrence;Second Malignancy;Unavailable                                        |
| tumor_fraction                  | Theta2 tumor DNA fraction estimate                                                                                                          | numeric                                                                                                                                       |
| tumor_ploidy                    | Control-FREEC ploidy                                                                                                                        | numeric                                                                                                                                       |


Table S1. Clinical metadata collected for OpenPBTA. {#tbl:S1}


| CNS_region      | primary_site                                                                                                                   |
|-----------------|--------------------------------------------------------------------------------------------------------------------------------|
| Hemispheric     | Frontal Lobe,Temporal Lobe,Parietal Lobe,Occipital Lobe                                                                        |
| Midline         | Pons/Brainstem,Brain Stem- Midbrain/Tectum,Brain Stem- Pons,Brain Stem-Medulla,Thalamus,Basal Ganglia,Hippocampus,Pineal Gland |
| Spine           | Spinal Cord- Cervical,Spinal Cord- Thoracic,Spinal Cord- Lumbar/Thecal Sac,Spine NOS                                           |
| Ventricles      | Ventricles                                                                                                                     |
| Posterior fossa | Cerebellum/Posterior Fossa                                                                                                     |
| Optic pathway   | Optic Pathway                                                                                                                  |
| Suprasellar     | Suprasellar/Hypothalamic/Pituitary                                                                                             |
| Other           | Meninges/Dura,Other locations NOS,Skull,Cranial Nerves NOS,Brain                                                               |


Table S2. Harmonized CNS brain regions derived from primary site values. {#tbl:S2}


##### Molecular Subtyping

The `molecular_subtype` column in the `pbta-histologies.tsv` file contains molecular subtypes for tumor types selected from  `pathology_diagnosis` and `pathology_free_text_diagnosis` fields as described below, following World Health Organization 2016 classification criteria [@doi:10.1007/s00401-016-1545-1].

Medulloblastoma (MB) subtypes SHH, MYC, Group 3, and Group 4 were predicted using the consensus of two RNA expression classifiers: [Medulloblastoma Classifier]](https://github.com/d3b-center/medullo-classifier-package) and MM2S Classifier [@doi:10.1186/s13029-016-0053-y] on the RSEM FPKM data.

High-grade glioma (HGG) subtypes were derived using the criteria below (additional details in the [analysis README](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/molecular-subtyping-HGG)):

1. If any sample contained an _H3F3A_ p.K28M, _HIST1H3B_ p.K28M, _HIST1H3C_ p.K28M, or _HIST2H3C_ p.K28M mutation and no _BRAF_ p.V600E mutation, it was subtyped as `DMG, H3K28`.
2. If any sample contained an _HIST1H3B_ p.K28M, _HIST1H3C_ p.K28M, or _HIST2H3C_ p.K28M mutation and a _BRAF_ p.V600E mutation, it was subtyped as `DMG, H3 K28, BRAF V600E`.
3. If any sample contained an _H3F3A_ p.G35V or p.G35R mutation, it was subtyped as`HGG, H3 G35`.
4. If any high-grade glioma sample contained an _IDH1_ p.R132 mutation, it was subtyped as `HGG, IDH`.
5. If a sample was initially classified as HGAT, had no defining histone mutations, and a BRAF p.V600E mutation, it was subtyped as `BRAF V600E`.
6. All other high-grade glioma samples that did not meet any of these criteria were subtyped as `HGG, H3 wildtype`.

Embryonal tumors were included in non-MB and non-ATRT embryonal tumor subtyping if they met any of the following criteria:
1. A _TTYH1_ (5' partner) fusion was detected.
2. A _MN1_ (5' partner) fusion was detected, with the exception of `MN1--PATZ1` since it is an entity separate of CNS HGNET-MN1 tumors [@doi:10.1111/nan.12626].
3. Pathology diagnoses included "Supratentorial or Spinal Cord PNET" or "Embryonal Tumor with Multilayered Rosettes".
4. A pathology diagnosis of "Neuroblastoma", where the tumor was not indicated to be peripheral or metastatic and was located in the CNS.
5. Any sample with "embryonal tumor with multilayer rosettes, ros (who grade iv)", "embryonal tumor, nos, congenital type", "ependymoblastoma" or "medulloepithelioma" in pathology free text.

Non-MB and non-ATRT embryonal tumors identified with the above criteria were further subtyped using the criteria below [@pmid:30249036; @doi:10.1007/s00381-017-3551-6; @url:https://www.cancer.gov/types/brain/hp/child-cns-embryonal-treatment-pdq; @doi:10.3390/ijms21051818].
Additional details can be found in the analysis [notebook](https://alexslemonade.github.io/OpenPBTA-analysis/analyses/molecular-subtyping-embryonal/04-table-prep.nb.html).

1. Any RNA-seq biospecimen with _LIN28A_ overexpression, plus a _TYH1_ fusion (5' partner) with a gene adjacent or within the C19MC miRNA cluster and/or copy number amplification of the C19MC region was subtyped as `ETMR, C19MC-altered` (Embryonal tumor with multilayer rosettes, chromosome 19 miRNA cluster altered) [@doi:10.1007/s00401-012-1068-3; 10.1038/ng.2849].
2. Any RNA-seq biospecimen with _LIN28A_ overexpression, a _TTYH1_ fusion (5' partner) with a gene adjacent or within the C19MC miRNA cluster but no evidence of copy number amplification of the C19MC region was subtyped as `ETMR, NOS` (Embryonal tumor with multilayer rosettes, not otherwise specified) [@doi:10.1007/s00401-012-1068-3; @doi:10.1038/ng.2849].
3. Any RNA-seq biospecimen with a fusion having a 5' _MN1_ and 3' _BEND2_ or _CXXC5_ partner were subtyped as `CNS HGNET-MN1` (Central nervous system (CNS) high-grade neuroepithelial tumor with _MN1_ alteration).
4. Non-MB and non-ATRT embryonal tumors with internal tandem duplication (as defined in [@doi:10.1186/s12859-016-1031-8]) of _BCOR_ were subtyped as `CNS HGNET-BCOR` (CNS high-grade neuroepithelial tumor with _BCOR_ alteration).
5. Non-MB and non-ATRT embryonal tumors with over-expression and/or gene fusions in _FOXR2_ were subtyped as `CNS NB-FOXR2` (CNS neuroblastoma with _FOXR2_ activation).
6. Non-MB and non-ATRT embryonal tumors with _CIC-NUTM1_ or other _CIC_ fusions, were subtyped as `CNS EFT-CIC` (CNS Ewing sarcoma family tumor with _CIC_ alteration) [@doi:/j.cell.2016.01.015]
7. Non-MB and non-ATRT embryonal tumors that did not fit any of the above categories were subtyped as `CNS Embryonal, NOS` (CNS Embryonal tumor, not otherwise specified).

Neurocytoma subtypes central neurocytoma (CNC) and extraventricular neurocytoma (EVN) were assigned based on the primary site of the tumor [@doi:10.1007/978-3-319-33432-5_20].
If `primary_site` of the tumor was `Ventricles`, it was subtyped as `CNC`; otherwise, it was subtyped as `EVN`.

Craniopharyngiomas (CRANIO) were subtyped into adamantinomatous (`CRANIO, ADAM`), papillary (`CRANIO, PAP`) or undetermined (`CRANIO, To be classified`) based on the following criteria [@doi:10.3171/jns.1995.83.2.0206; @doi:10.3171/jns.1998.89.4.0547]:
1. Craniopharyngiomas from patients over 40 years old with a _BRAF_ p.V600E mutation were subtyped as `CRANIO, PAP`.
2. Craniopharyngiomas from patients younger than 40 years old with mutations in exon 3 of _CTNNB1_ were subtyped as `CRANIO, ADAM`.
3. Craniopharyngiomas that do not fall into the above two categories were subtyped as `CRANIO, To be classified`. 

A molecular subtype of `EWS` was assigned to any tumor with a _EWSR1_ fusion or with a `pathology_diagnosis` of `Ewings Sarcoma`. 

Low-grade astrocytic tumors (LGAT) or glialneuronal tumors (GNT) were subtyped based on SNV, fusion and CNV status based on @doi:10.1016/j.ccell.2020.03.011, and as described below.
1. If a sample contained a _NF1_ somatic mutation, either nonsense or missense, it was subtyped as `LGG, NF1-somatic`.
2. If a sample contained _NF1_ germline mutation, as indicated by a patient having the neurofibromatosis  cancer predisposition, it was subtyped as `LGG, NF1-germline`.
3. If a sample contained the _IDH_ p.R132 mutation, it was subtyped as `LGG, IDH`.
4. If a sample contained a histone p.K28M mutation in either _H3F3A_, _H3F3B_, _HIST1H3B_, _HIST1H3C_, or _HIST2H3C_, or if it contained a p.G35R or p.G35V mutation in _H3F3A_, it was subtyped as `LGG, H3`.
5. If a sample contained _BRAF_ p.V600E or any other non-canonical _BRAF_ mutations in the kinase (PK_Tyr_Ser-Thr) domain [@url:https://pfam.xfam.org/family/PF07714], it was subtyped as `LGG, BRAF V600E`.
6. If a sample contained `KIAA1549--BRAF` fusion, it was subtyped as `LGG, KIAA1549-BRAF`.
7. If a sample contained SNV or indel in either _KRAS_, _NRAS_, _HRAS_, _MAP2K1_, _MAP2K2_, _MAP2K1_, _ARAF_, _RAF1_, or non-kinase domain of _BRAF_, or if it contained _RAF1_ fusion, or _BRAF_ fusion that was not `KIAA1549--BRAF`, it was subtyped as `LGG, other MAPK`.
8. If a sample contained SNV in either _MET_, _KIT_ or _PDGFRA_, or if it contained fusion in _ALK_, _ROS1_, _NTRK1_, _NTRK2_, _NTRK3_ or _PDGFRA_, it was subtyped as `LGG, RTK`.
9. If a sample contained _FGFR1_ p.N546K, p.K656E, p.N577, or p. K687 hotspot mutations, or tyrosine kinase domain tandem duplication [@url:http://hgdownload.soe.ucsc.edu/goldenPath/hg38/database/pfamDesc.txt.gz], or _FGFR1_ or _FGFR2_ fusions, it was subtyped as `LGG, FGFR`.
10. If a sample contained _MYB_ or _MYBL1_ fusion, it was subtyped as `LGG, MYB/MYBL1`.
11. If a sample contained focal CDKN2A and/or CDKN2B  deletion, it was subtyped as `LGG, CDKN2A/B`.

For LGAT tumors that did not have any of the above molecular alterations, if both RNA and DNA samples were available, it was subtyped as `LGG, wildtype`. 
Otherwise, if either RNA or DNA sample was unavailable, it was subtyped as `LGG, To be classified`.

If pathology diagnosis was `Subependymal Giant Cell Astrocytoma (SEGA)`, the `LGG` portion of molecular subtype was recoded to `SEGA`.

Lastly, for all subtyped samples, if the tumors were glialneuronal in origin, based on `pathology_free_text_diagnosis` entries of `desmoplastic infantile`,`desmoplastic infantile ganglioglioma`, `desmoplastic infantile astrocytoma` or `glioneuronal`, each was recoded as follows:
If pathology diagnosis is `Low-grade glioma/astrocytoma (WHO grade I/II)` or `Ganglioglioma`, the `LGG` portion of the molecular subtype was recoded to `GNT`.

Ependymoma (EPN) were subtyped into `EPN, ST RELA`, `EPN, ST YAP1`, `EPN, PF A` and `EPN, PF B` based on evidence for these molecular subgroups as described in Pajtler et al. [@doi:10.1016/j.ccell.2015.04.002]. 
Briefly, fusion, CNV and gene expression data were used to subtype EPN as followed:
1. Any tumor with fusions containing `RELA` as fusion partner, e.g., `C11orf95--RELA`, `LTBP3--RELA`, was subtyped as `EPN, ST RELA`.
2. Any tumor with fusions containing `YAP1` as fusion partner, such as `C11orf95--YAP1`, `YAP1--MAMLD1` and `YAP1--FAM118B`, was subtyped as `EPN, ST YAP1`.
3. Any tumor with the following molecular characterization would be subtyped as `EPN, PF A`: 
- _CXorf67_ expression z-score of over 3
- _TKTL1_ expression z-score of over 3 and 1q gain
4. Any tumor with the following molecular characterization would be subtyped as `EPN, PF B`:
- _GPBP17_ expression z-score of over 3 and loss of 6q or 6p
- _IFT46_ expression z-score of over 3 and loss of 6q or 6p

Any tumor with the above molecular characteristics would be exclusively subtyped to the designated group. 

For all other remaining EPN tumors without above molecular characteristics, they would be subtyped to `EPN, ST RELA` and `EPN, ST YAP1` in a non-exclusive way (e.g., a tumor could have both `EPN, ST RELA` and `EPN, ST YAP1` subtypes) if any of the following alterations were present.
1. Any tumor with the following alterations was assigned `EPN, ST RELA`:
- `PTEN--TAS2R1` fusion 
- chromosome 9 arm (9p or 9q) loss 
- _RELA_ expression z-score of over 3 
- _L1CAM_ expression z-score of over 3 
2. Any tumor with the following alterations was assigned `EPN, ST YAP1`:
- `C11orf95--MAML2` fusion 
- chromosome 11 short arm (11p) loss  
- chromosome 11 long arm (11q) gain
- _ARL4D_ expression z-score of over 3 
- _CLDN1_ expression z-score of over 3

After all relevant tumor samples were subtyped by the above molecular subtyping modules, the results from these modules, along with other clinical information (such as pathology diagnosis free text), were compiled through `molecular-subtyping-pathology` module. 
The compilation was executed by the following steps:

Firstly, `broad_histology`, `short_histology`, and `integrated_diagnosis` columns in the result files from the above subtyping modules (i.e., `CRANIO_molecular_subtype.tsv`, `EWS_results.tsv`, `EPN_all_data_withsubgroup.tsv`, `HGG_molecular_subtype.tsv` ,`lgat_subtyping.tsv`, `MB_molecular_subtype.tsv`, `embryonal_tumor_molecular_subtypes.tsv`, and `neurocytoma_subtyping.tsv`) were updated based on the molecular subtype of the tumor. 
Detailed information about the updating procedure were included in the analysis [notebook](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/molecular-subtyping-pathology/01-compile-subtyping-results.Rmd).
Notes were also added to indicated that the changes in `broad_histology`, `short_histology` and `integrated_diagnosis` were from OpenPBTA subtyping modules.

Subsequently, `broad_histology`, `short_histology` and `harmonized_diagnosis` columns of tumors with particular pathology diagnosis free text were updated as specified in the following table:

pathology_diagnosis | subtyping module | pathology_free_text_diagnosis | broad_histology | short_histology | harmonized_diagnosis
-- | -- | -- | -- | -- | --
Primary CNS lymphoma | NA | contains `burkitt's lymphoma` | Lymphoma | CNS lymphoma | CNS Burkitt's lymphoma
Other | NA | contains `xanthogranuloma` or `jxg` | Histiocytic tumor | JXG | Juvenile xanthogranuloma
Meningioma | NA | contains `atypical` | Meningioma | Meningioma | Atypical meningioma
Meningioma | NA | contains `anaplastic` | Meningioma | Meningioma | Anaplastic (malignant) meningioma
Meningioma | NA | contains `clear cell meningioma` | Meningioma | Meningioma | Clear cell meningioma
Meningioma | NA | contains `meningothelial` | Meningioma | Meningioma | Meningothelial meningioma
Meningioma | NA | does not contain `atypical`, `anaplastic`, `clear cell`, or `meningothelial` | Meningioma | Meningioma | Meningioma
Choroid plexus papilloma | NA | contains `atypical` | Choroid plexus tumor | Choroid plexus tumor | Atypical choroid plexus papilloma
Craniopharyngioma | CRANIO | contains `adamantinomatous` | Tumors of sellar region | Craniopharyngioma | Adamantinomatous craniopharyngioma 

Similarly, `broad_histology`, `short_histology`, `integrated_diagnosis` and `harmonized_diagnosis` columns of tumors with following pathology diagnosis free text were updated as specified in the table below:

pathology_diagnosis | subtyping module | pathology_free_text_diagnosis | broad_histology | short_histology | integrated_diagnosis
-- | -- | -- | -- | -- | --
Low-grade glioma/astrocytoma (WHO grade I/II) | LGAT | contains `sega or subependymal giant cell astrocytoma` | Low grade astrocytic tumor | LGAT | Subependymal Giant Cell Astrocytoma, <subtype>
Low-grade glioma/astrocytoma (WHO grade I/II) | LGAT | contains `fibrillary` | Low grade astrocytic tumor | LGAT | Diffuse fibrillary astrocytoma, <subtype>
Low-grade glioma/astrocytoma (WHO grade I/II) | LGAT | contains `gliomatosis cerebri, type 1, ia` | Low grade astrocytic tumor | LGAT | Gliomatosis cerebri, <subtype>
Low-grade glioma/astrocytoma (WHO grade I/II) | LGAT | contains `jpa` or `juvenile astrocytoma` or `pilocytic` or `pilocystic` (typo) or `pilomyxoid` but does not contain `fibrillary` | Low grade astrocytic tumor | LGAT | Pilocytic astrocytoma, <subtype>
Low-grade glioma/astrocytoma (WHO grade I/II) | LGAT | contains `oligodendroglioma who ii` | Diffuse astrocytic and oligodendroglial tumor | Oligodendroglioma | Oligodendroglioma, <subtype>
Low-grade glioma/astrocytoma (WHO grade I/II) | LGAT | contains `pxa` or `pleomorphic xanthoastrocytoma` | Low grade astrocytic tumor | LGAT | Pleomorphic xanthoastrocytoma, <subtype>

Additionally, `broad_histology`, `short_histology`, `integrated_diagnosis` and `harmonized_diagnosis` columns of tumors with following pathology diagnosis free text were updated as specified in the table below:

pathology_diagnosis | subtyping module | pathology_free_text_diagnosis | broad_histology | short_histology | integrated_diagnosis | harmonized_diagnosis 
-- | -- | -- | -- | -- | -- | -- 
Low-grade glioma/astrocytoma (WHO grade I/II) | NA, remove from LGAT module | contains `desmoplastic infantile astrocytoma` | Neuronal and mixed neuronal-glial tumor | GNT | Desmoplastic infantile astrocytoma and ganglioglioma, <subtype> | Desmoplastic infantile astrocytoma and ganglioglioma 
Low-grade glioma/astrocytoma (WHO grade I/II) | NA, remove from LGAT module | contains `diffuse leptomeningeal glioneuronal tumor` | Neuronal and mixed neuronal-glial tumor | GNT | Diffuse leptomeningeal glioneuronal tumor, <subtype> | Diffuse leptomeningeal glioneuronal tumor
Low-grade glioma/astrocytoma (WHO grade I/II) | NA, remove from LGAT module | contains `glioneuronal` | Neuronal and mixed neuronal-glial tumor | GNT | Glial-neuronal tumor NOS, <subtype> | Glial-neuronal tumor NOS
Low-grade glioma/astrocytoma (WHO grade I/II) | NA, remove from LGAT module | contains `rosette forming glioneuronal tumor` | Neuronal and mixed neuronal-glial tumor | GNT | Rosette-forming glioneuronal tumor, <subtype> | Rosette-forming glioneuronal tumor

Notes were also added to indicated that the changes in `broad_histology`, `short_histology`, `integrated_diagnosis` and `harmonized_diagnosis` were from pathology diagnosis free text.

For samples with subtype discrepancies, `molecular_subtype` and `integrated_diagnosis` were updated following pathology or clinical review.
Detailed information can be found in the analysis notebooks for [clinical](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/molecular-subtyping-pathology/03-incorporate-clinical-feedback.Rmd) and [pathology](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/molecular-subtyping-pathology/04-incorporate-pathology-feedback.Rmd) feedback.
Finally, the newly compiled subtypes were integrated into the `pbta-histologies.tsv` file in the `molecular-subtyping-integrate` module.


#### TP53 Alteration Annotation

In addition to tumor types mentioned above, TP53 altered status is also annotated for all samples and if a sample is determined to be either `TP53 loss` or `TP53 activated`, this annotation will be included in the `molecular_subtype` column. 
We applied a _TP53_ inactivation classifier originally trained on TCGA PanCan data [@doi:10.1016/j.celrep.2018.03.076] to the matched RNA expression data for each sample. 
Along with the _TP53_ classifier scores, consensus SNV and CNV, SV, and references databases that list TP53 hotspot mutations [@url:https://www.cancerhotspots.org/#/home; @doi:10.1158/2159-8290.CD-17-0321; @doi:10.1038/nbt.3391] and functional domains [@doi:10.1038/sj.cdd.4401904] were used collectively to determine TP53 alteration status for each sample.
The rules for calling either `TP53 loss` or `TP53 activated` are as follows:
If a sample has any of the two well-characterized _TP53_ gain-of-function mutations, p.R273C or p.R248W [@doi:10.1038/ng0593-42], `TP53 activated` status will be assigned.
A sample will be annotated as `TP53 loss` if any of the following conditions is met:
1) It contains a _TP53_ hotspot mutation as defined by IARC TP53 database [@url:https://www.cancerhotspots.org/#/home; @doi:10.1158/2159-8290.CD-17-0321; @doi:10.1038/nbt.3391] 
2) It contains two _TP53_ alterations, including SNV, CNV or SV, which is indicative of probable bi-allelic alterations
3) It contains one _TP53_ somatic alteration, including SNV, CNV, or SV and a germline _TP53_ mutation indicated by the diagnosis of Li-Fraumeni syndrome [@pmid:28270529]
4) It contains one germline _TP53_ mutation indicated by Li-Fraumeni syndrome and the _TP53_ classifier score for matched RNA-Seq is over 0.5. 

#### Survival analysis

Overall survival, denoted `OS_days`, was calculated as days since initial diagnosis.

#### Prediction of participants' genetic sex

The clinical metadata provided included a reported gender.
We used DNA data, in concert with the reported gender, to predict participant genetic sex so that we could identify sexually dimorphic outcomes.
This analysis could also reveal samples that may have been contaminated in certain circumstances.
We used the idxstats utility from SAMtools [@pmid:19505943] to calculate read lengths, the number of mapped reads, and the corresponding chromosomal location for reads to the X and Y chromosomes.
We used the fraction of total normalized X and Y chromosome reads that were attributed to the Y chromosome as a summary statistic.
We reviewed this statistic in the context of reported gender and determined that a threshold of less than 0.2 clearly delineated female samples.
Fractions greater than 0.4 were predicted to be males.
Samples with values in the range [0.2, 0.4] were marked as unknown.
We ran this analysis through [CWL](https://github.com/d3b-center/sex-determination-tool) on CAVATICA.
Resulting calls were added to the clinical metadata as `germline_sex_estimate`.

#### Selection of independent samples

Certain analyses required that we select only a single representative specimen for each individual.
In these cases, we prioritized primary tumors and those with whole-genome sequencing available.
If this filtering still resulted in multiple specimens, we selected from the remaining set randomly.

#### Quantification of Telomerase Activity using Gene Expression Data

We predicted telomerase activity of pediatric brain tumor samples using our recently developed method EXTEND. 
In brief, EXTEND estimates telomerase activity based on the expression of a 13-gene signature. 
This signature was derived by comparing telomerase positive tumors and tumors with activated alternative lengthening of telomeres pathway, a group presumably negative of telomerase activity. 
More details about the algorithm can be found in reference [@doi:https://doi.org/10.1101/2020.05.21.109249]. 
We calculated telomerase activity score for each sample of the PBTA cohort and examined the score distribution across both broad and specific disease histological subtypes. 
We also compared EXTEND scores across the four molecular subgroups of medulloblastoma (Group3, Group4, SHH and WNT) using a series of pairwise two-sample t-tests with a Bonferroni correction for multiple testing. 
EXTEND scores have been further compared, using Spearman rank correlation, between counts and FPKM gene expression values from poly-A and stranded protocols.

### ADDITIONAL RESOURCES
<!-- TODO: Add section -->

### KEY RESOURCES TABLE
| REAGENT or RESOURCE     | SOURCE                                | IDENTIFIER                                                                                   |                               
|-----------------|-------------------------------------------------|------------------------------------------------------------------------|
|Biological samples|||
|                    |                        |                    |
|                    |                        |                    |
|                    |                        |                    |
|                    |                        |                    |
|Critical commercial assays|||
|                    |                        |                    |
|                    |                        |                    |
|                    |                        |                    |
|                    |                        |                    |
|Deposited data|||
|Raw and harmonized WGS, WXS, Panel, RNA-Seq                    |KidsFirst Data Resource Center, this project                        |[@doi:10.24370/OPENPBTA]                    |
|Merged summary files                    |this project                        |https://cavatica.sbgenomics.com/u/cavatica/openpbta                    |
|Merged summary files and downstream analyses                   |this project                        |https://github.com/AlexsLemonade/OpenPBTA-analysis/                    |
|Processed data                    |this project                        |https://pedcbioportal.kidsfirstdrc.org/study/summary?id=openpbta                   |
|Experimental models: Cell lines|||
|                    |                        |                    |
|                    |                        |                    |
|                    |                        |                    |
|                    |                        |                    |
|                    |                        |                    |
|Software and algorithms|||
|OpenPBTA workflows repository                   |this project                        |[@doi:10.5281/zenodo.6363520]                    |
|OpenPBTA analysis repository                    |this project                        |                    |
|OpenPBTA manuscript repository                    |this project                        |                    |
|                    |                        |                    |
|Other|||
|                    |                        |                    |
|                    |                        |                    |
|                    |                        |                    |
|                    |                        |                    |
|                    |                        |                    |

## Supplemental Information Titles and Legends
<!-- TODO: Add complete legends once figures are finalized. -->

**Figure S1. OpenPBTA Project Workflow, Related to Figure 1.** 
Biospecimens and data were collected by CBTN and PNOC. 
Genomic sequencing and harmonization (orange boxes) were performed by the Kids First Data Resource Center (KFDRC). 
Analyses in the green boxes were performed by contributors of the OpenPBTA project. 
Output files are denoted in blue. 
Figure created with biorender.com.

**Figure S2. Validation of Consensus SNV calls and Tumor Mutation Burden, Related to Figures 2 and 3.**  
Correlation (A) and violin (B) plots of mutation variant allele frequencies (VAFs) comparing the variant callers (Lancet, Strelka2, Mutect2, and VarDict) used for PBTA samples. 
Upset plot (C) showing overlap of variant calls. Correlation (D) and violin (E) plots of mutation variant allele frequencies (VAFs) comparing the variant callers (Lancet, Strelka2, and Mutect2) used for TCGA samples. 
Upset plot (F) showing overlap of variant calls. 
Cumulative distribution TMB plots for PBTA (G) and TCGA (H)tumors using consensus SNV calls.

**Figure S3. Genomic instability of pediatric brain tumors, Related to Figures 2 and 3.**
(A) Oncoprint of canonical somatic gene mutations, CNVs, fusions, and TMB (top bar plot) for the top 20 genes mutated across rare CNS tumors (N < 5 each): desmoplastic infantile astrocytoma and ganglioglioma (N = 1), germinoma (N = 4), glial-neuronal NOS (N = 4), metastatic secondary tumors (N = 3), neurocytoma (N = 2), and pineoblastoma (N = 3). 
Patient sex (`germline_sex_estimate`) and tumor histology (`cancer_group`) are displayed as annotations at the bottom of each plot. 
Only samples with mutations in the listed genes are shown. 
Multiple CNVs are denoted as a complex event. 
(B) Genome-wide plot of CNV alterations by broad histology. 
Each row represents one sample. 
Box and whisker plots of number of CNV breaks (C) or SV breaks (D) by number of chromothripsis regions. 

**Figure S4. Related to Figure 3.** 
(A) Sample-specific RefSig signature weights across cancer groups ordered by decreasing Signature 1 exposure. 
(B) Proportion of Signature 1 plotted by phase of therapy for each cancer group.

**Figure S5. Related to Figure 4**.
First two dimensions from UMAP of sample transcriptome data with points colored by `molecular_subtype` for medulloblastoma (A), ependymoma (B), low-grade glioma (C), and high-grade diffuse astrocytic tumors (D). 
(E) Receiver Operating Characteristic for _TP53_ classifier run on FPKM of poly-A RNA-Seq samples. 
(F) Violin and box plots of _TP53_ scores plotted by _TP53_ alteration type in poly-A RNA-Seq samples.
Correlation plots for telomerase scores (EXTEND) with RNA expression of _TERT_ (G) and _TERC_ (H).


**Table S1. Related to Figure 1.** 
Table of specimens and associated metadata, clinical data, and histological data utilized in the OpenPBTA project.

**Table S2. Related to Figures 2 and 3.** 
Table of TMB, eight CNS mutational signatures, and chromothripsis events per sample.

**Table S3. Related to Figures 4 and 5.** Table of _TP53_ scores, telomerase EXTEND scores, and quanTIseq immune scores.




## References {.page_break_before}

<!-- Explicitly insert bibliography here -->
<div id="refs"></div>
