---
author-meta:
- Joshua A. Shapiro
- Candace L. Savonen
- Chante J. Bethell
- Krutika S. Gaonkar
- Run Jin
- Yuankun Zhu
- Miguel A. Brown
- Nhat Duong
- Komal S. Rathi
- Nighat Noureen
- Bo Zhang
- Brian M. Ennis
- Stephanie J. Spielman
- Laura E. Egolf
- Yang Yang
- Bailey Farrow
- Nicolas Van Kuren
- Meen Chul Kim
- Tejaswi Koganti
- Shrivats Kannan
- Pichai Raman
- Jennifer Mason
- Daniel P. Miller
- Anna R. Poetsch
- Payal Jain
- Adam A. Kraya
- Allison P. Heath
- Mateusz P. Koptyra
- Shannon Robbins
- Yiran Guo
- Xiaoyan Huang
- Jessica Wong
- Mariarita Santi
- Angela Viaene
- Laura Scolaro
- Angela Waanders
- Derek Hanson
- Steven M. Foltz
- Hongbo M. Xie
- Siyuan Zheng
- Cassie N. Kline
- Peter J. Madsen
- Jena V. Lilly
- Philip B. Storm
- Adam C. Resnick
- Casey S. Greene
- Jo Lynne Rokita*
- Jaclyn N. Taroni*
- Children's Brain Tumor Network
- Pacific Pediatric Neuro-Oncology Consortium
bibliography:
- content/manual-references.json
date-meta: '2022-07-18'
header-includes: "<!--\nManubot generated metadata rendered from header-includes-template.html.\nSuggest improvements at https://github.com/manubot/manubot/blob/master/manubot/process/header-includes-template.html\n-->\n<meta name=\"dc.format\" content=\"text/html\" />\n<meta name=\"dc.title\" content=\"An Open Pediatric Brain Tumor Atlas\" />\n<meta name=\"citation_title\" content=\"An Open Pediatric Brain Tumor Atlas\" />\n<meta property=\"og:title\" content=\"An Open Pediatric Brain Tumor Atlas\" />\n<meta property=\"twitter:title\" content=\"An Open Pediatric Brain Tumor Atlas\" />\n<meta name=\"dc.date\" content=\"2022-07-18\" />\n<meta name=\"citation_publication_date\" content=\"2022-07-18\" />\n<meta name=\"dc.language\" content=\"en-US\" />\n<meta name=\"citation_language\" content=\"en-US\" />\n<meta name=\"dc.relation.ispartof\" content=\"Manubot\" />\n<meta name=\"dc.publisher\" content=\"Manubot\" />\n<meta name=\"citation_journal_title\" content=\"Manubot\" />\n<meta name=\"citation_technical_report_institution\" content=\"Manubot\" />\n<meta name=\"citation_author\" content=\"Joshua A. Shapiro\" />\n<meta name=\"citation_author_institution\" content=\"Childhood Cancer Data Lab, Alex&#39;s Lemonade Stand Foundation, Bala Cynwyd, PA, USA\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0002-6224-0347\" />\n<meta name=\"twitter:creator\" content=\"@jashapiro\" />\n<meta name=\"citation_author\" content=\"Candace L. Savonen\" />\n<meta name=\"citation_author_institution\" content=\"Childhood Cancer Data Lab, Alex&#39;s Lemonade Stand Foundation, Bala Cynwyd, PA, USA\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0001-6331-7070\" />\n<meta name=\"twitter:creator\" content=\"@cansavvy\" />\n<meta name=\"citation_author\" content=\"Chante J. Bethell\" />\n<meta name=\"citation_author_institution\" content=\"Childhood Cancer Data Lab, Alex&#39;s Lemonade Stand Foundation, Bala Cynwyd, PA, USA\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0001-9653-8128\" />\n<meta name=\"twitter:creator\" content=\"@cjbethell\" />\n<meta name=\"citation_author\" content=\"Krutika S. Gaonkar\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Division of Neurosurgery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Department of Bioinformatics and Health Informatics, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0003-0838-2405\" />\n<meta name=\"twitter:creator\" content=\"@aggokittu\" />\n<meta name=\"citation_author\" content=\"Run Jin\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Division of Neurosurgery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0002-8958-9266\" />\n<meta name=\"twitter:creator\" content=\"@runjin\" />\n<meta name=\"citation_author\" content=\"Yuankun Zhu\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Division of Neurosurgery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0002-2455-9525\" />\n<meta name=\"twitter:creator\" content=\"@zhuyuankun\" />\n<meta name=\"citation_author\" content=\"Miguel A. Brown\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Division of Neurosurgery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0001-6782-1442\" />\n<meta name=\"twitter:creator\" content=\"@migbro\" />\n<meta name=\"citation_author\" content=\"Nhat Duong\" />\n<meta name=\"citation_author_institution\" content=\"Department of Bioinformatics and Health Informatics, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0003-2852-4263\" />\n<meta name=\"twitter:creator\" content=\"@asiannhat\" />\n<meta name=\"citation_author\" content=\"Komal S. Rathi\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Department of Bioinformatics and Health Informatics, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0001-5534-6904\" />\n<meta name=\"twitter:creator\" content=\"@komalsrathi\" />\n<meta name=\"citation_author\" content=\"Nighat Noureen\" />\n<meta name=\"citation_author_institution\" content=\"Greehey Children&#39;s Cancer Research Institute, UT Health San Antonio\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0001-7495-8201\" />\n<meta name=\"citation_author\" content=\"Bo Zhang\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Division of Neurosurgery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0002-0743-5379\" />\n<meta name=\"citation_author\" content=\"Brian M. Ennis\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Division of Neurosurgery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0002-2653-5009\" />\n<meta name=\"citation_author\" content=\"Stephanie J. Spielman\" />\n<meta name=\"citation_author_institution\" content=\"Childhood Cancer Data Lab, Alex&#39;s Lemonade Stand Foundation, Bala Cynwyd, PA, USA&lt;sup&gt;\u262F&lt;/sup&gt;\" />\n<meta name=\"citation_author_institution\" content=\"Rowan University, Glassboro, NJ, USA\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0002-9090-4788\" />\n<meta name=\"twitter:creator\" content=\"@stephspiel\" />\n<meta name=\"citation_author\" content=\"Laura E. Egolf\" />\n<meta name=\"citation_author_institution\" content=\"Cell and Molecular Biology Graduate Group, Perelman School of Medicine at the University of Pennsylvania\" />\n<meta name=\"citation_author_institution\" content=\"Division of Oncology, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0002-7103-4801\" />\n<meta name=\"twitter:creator\" content=\"@LauraEgolf\" />\n<meta name=\"citation_author\" content=\"Yang Yang\" />\n<meta name=\"citation_author_institution\" content=\"Ben May Department for Cancer Research, University of Chicago, Chicago IL, USA\" />\n<meta name=\"citation_author\" content=\"Bailey Farrow\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Division of Neurosurgery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0001-6727-6333\" />\n<meta name=\"citation_author\" content=\"Nicolas Van Kuren\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Division of Neurosurgery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0002-7414-9516\" />\n<meta name=\"citation_author\" content=\"Meen Chul Kim\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Division of Neurosurgery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0002-0308-783X\" />\n<meta name=\"citation_author\" content=\"Tejaswi Koganti\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Division of Neurosurgery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0002-7733-6480\" />\n<meta name=\"citation_author\" content=\"Shrivats Kannan\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Division of Neurosurgery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0002-1460-920X\" />\n<meta name=\"twitter:creator\" content=\"@kshrivats\" />\n<meta name=\"citation_author\" content=\"Pichai Raman\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Department of Bioinformatics and Health Informatics, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0001-6948-2157\" />\n<meta name=\"twitter:creator\" content=\"@PichaiRaman\" />\n<meta name=\"citation_author\" content=\"Jennifer Mason\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Division of Neurosurgery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"twitter:creator\" content=\"@jenn0307\" />\n<meta name=\"citation_author\" content=\"Daniel P. Miller\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Division of Neurosurgery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0002-2032-4358\" />\n<meta name=\"citation_author\" content=\"Anna R. Poetsch\" />\n<meta name=\"citation_author_institution\" content=\"Biotechnology Center, Technical University Dresden, Germany\" />\n<meta name=\"citation_author_institution\" content=\"National Center for Tumor Diseases, Dresden, Germany\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0003-3056-4360\" />\n<meta name=\"twitter:creator\" content=\"@APoetsch\" />\n<meta name=\"citation_author\" content=\"Payal Jain\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Division of Neurosurgery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0002-5914-9083\" />\n<meta name=\"twitter:creator\" content=\"@jainpayal022\" />\n<meta name=\"citation_author\" content=\"Adam A. Kraya\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Division of Neurosurgery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0002-8526-5694\" />\n<meta name=\"citation_author\" content=\"Allison P. Heath\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Division of Neurosurgery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0002-2583-9668\" />\n<meta name=\"twitter:creator\" content=\"@allig8r\" />\n<meta name=\"citation_author\" content=\"Mateusz P. Koptyra\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Division of Neurosurgery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0002-3857-6633\" />\n<meta name=\"twitter:creator\" content=\"@koptyram\" />\n<meta name=\"citation_author\" content=\"Shannon Robbins\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Division of Neurosurgery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author\" content=\"Yiran Guo\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Division of Neurosurgery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0002-6549-8589\" />\n<meta name=\"twitter:creator\" content=\"@YiranGuo3\" />\n<meta name=\"citation_author\" content=\"Xiaoyan Huang\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Division of Neurosurgery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0001-7267-4512\" />\n<meta name=\"citation_author\" content=\"Jessica Wong\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Division of Neurosurgery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0003-1508-7631\" />\n<meta name=\"twitter:creator\" content=\"@jessicawongbfx\" />\n<meta name=\"citation_author\" content=\"Mariarita Santi\" />\n<meta name=\"citation_author_institution\" content=\"Department of Pathology and Laboratory Medicine, Children\u2019s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Department of Pathology and Laboratory Medicine, University of Pennsylvania Perelman School of Medicine\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0002-6728-3450\" />\n<meta name=\"citation_author\" content=\"Angela Viaene\" />\n<meta name=\"citation_author_institution\" content=\"Department of Pathology and Laboratory Medicine, Children\u2019s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Department of Pathology and Laboratory Medicine, University of Pennsylvania Perelman School of Medicine\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0001-6430-8360\" />\n<meta name=\"citation_author\" content=\"Laura Scolaro\" />\n<meta name=\"citation_author_institution\" content=\"Division of Oncology, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author\" content=\"Angela Waanders\" />\n<meta name=\"citation_author_institution\" content=\"Department of Oncology, Ann &amp; Robert H. Lurie Children\u2019s Hospital of Chicago\" />\n<meta name=\"citation_author_institution\" content=\"Department of Pediatrics, Northwestern University Feinberg School of Medicine\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0002-0571-2889\" />\n<meta name=\"citation_author\" content=\"Derek Hanson\" />\n<meta name=\"citation_author_institution\" content=\"Hackensack Meridian School of Medicine\" />\n<meta name=\"citation_author_institution\" content=\"Hackensack University Medical Center\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0002-0024-5142\" />\n<meta name=\"citation_author\" content=\"Steven M. Foltz\" />\n<meta name=\"citation_author_institution\" content=\"Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania\" />\n<meta name=\"citation_author_institution\" content=\"Childhood Cancer Data Lab, Alex&#39;s Lemonade Stand Foundation, Bala Cynwyd, PA, USA\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0002-9526-8194\" />\n<meta name=\"citation_author\" content=\"Hongbo M. Xie\" />\n<meta name=\"citation_author_institution\" content=\"Department of Bioinformatics and Health Informatics, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0003-2223-0029\" />\n<meta name=\"twitter:creator\" content=\"@xiehb\" />\n<meta name=\"citation_author\" content=\"Siyuan Zheng\" />\n<meta name=\"citation_author_institution\" content=\"Greehey Children&#39;s Cancer Research Institute, UT Health San Antonio\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0002-1031-9424\" />\n<meta name=\"twitter:creator\" content=\"@zhengsiyuan\" />\n<meta name=\"citation_author\" content=\"Cassie N. Kline\" />\n<meta name=\"citation_author_institution\" content=\"Division of Oncology, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0001-7765-7690\" />\n<meta name=\"twitter:creator\" content=\"@cnkline13\" />\n<meta name=\"citation_author\" content=\"Peter J. Madsen\" />\n<meta name=\"citation_author_institution\" content=\"Division of Neurosurgery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0001-9266-3685\" />\n<meta name=\"twitter:creator\" content=\"@petermadsenmd\" />\n<meta name=\"citation_author\" content=\"Jena V. Lilly\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Division of Neurosurgery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0003-1439-6045\" />\n<meta name=\"twitter:creator\" content=\"@jvlilly\" />\n<meta name=\"citation_author\" content=\"Philip B. Storm\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Division of Neurosurgery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0002-7964-2449\" />\n<meta name=\"citation_author\" content=\"Adam C. Resnick\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Division of Neurosurgery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0003-0436-4189\" />\n<meta name=\"twitter:creator\" content=\"@adamcresnick\" />\n<meta name=\"citation_author\" content=\"Casey S. Greene\" />\n<meta name=\"citation_author_institution\" content=\"Department of Systems Pharmacology and Translational Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA, USA\" />\n<meta name=\"citation_author_institution\" content=\"Childhood Cancer Data Lab, Alex&#39;s Lemonade Stand Foundation, Bala Cynwyd, PA, USA\" />\n<meta name=\"citation_author_institution\" content=\"Center for Health AI, University of Colorado School of Medicine, Aurora, CO, USA\" />\n<meta name=\"citation_author_institution\" content=\"Department of Biochemistry and Molecular Genetics, University of Colorado School of Medicine, Aurora, CO, USA\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0001-8713-9213\" />\n<meta name=\"twitter:creator\" content=\"@greenescientist\" />\n<meta name=\"citation_author\" content=\"Jo Lynne Rokita*\" />\n<meta name=\"citation_author_institution\" content=\"Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Division of Neurosurgery, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_institution\" content=\"Department of Bioinformatics and Health Informatics, Children&#39;s Hospital of Philadelphia\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0003-2171-3627\" />\n<meta name=\"twitter:creator\" content=\"@jolynnerokita\" />\n<meta name=\"citation_author\" content=\"Jaclyn N. Taroni*\" />\n<meta name=\"citation_author_institution\" content=\"Childhood Cancer Data Lab, Alex&#39;s Lemonade Stand Foundation, Bala Cynwyd, PA, USA\" />\n<meta name=\"citation_author_orcid\" content=\"0000-0003-4734-4508\" />\n<meta name=\"twitter:creator\" content=\"@jaclyn_taroni\" />\n<meta name=\"citation_author\" content=\"Children&#39;s Brain Tumor Network\" />\n<meta name=\"citation_author\" content=\"Pacific Pediatric Neuro-Oncology Consortium\" />\n<link rel=\"canonical\" href=\"https://AlexsLemonade.github.io/OpenPBTA-manuscript/\" />\n<meta property=\"og:url\" content=\"https://AlexsLemonade.github.io/OpenPBTA-manuscript/\" />\n<meta property=\"twitter:url\" content=\"https://AlexsLemonade.github.io/OpenPBTA-manuscript/\" />\n<meta name=\"citation_fulltext_html_url\" content=\"https://AlexsLemonade.github.io/OpenPBTA-manuscript/\" />\n<meta name=\"citation_pdf_url\" content=\"https://AlexsLemonade.github.io/OpenPBTA-manuscript/manuscript.pdf\" />\n<link rel=\"alternate\" type=\"application/pdf\" href=\"https://AlexsLemonade.github.io/OpenPBTA-manuscript/manuscript.pdf\" />\n<link rel=\"alternate\" type=\"text/html\" href=\"https://AlexsLemonade.github.io/OpenPBTA-manuscript/v/022d54780d9ddbfa8b79429fc87124a5eb135905/\" />\n<meta name=\"manubot_html_url_versioned\" content=\"https://AlexsLemonade.github.io/OpenPBTA-manuscript/v/022d54780d9ddbfa8b79429fc87124a5eb135905/\" />\n<meta name=\"manubot_pdf_url_versioned\" content=\"https://AlexsLemonade.github.io/OpenPBTA-manuscript/v/022d54780d9ddbfa8b79429fc87124a5eb135905/manuscript.pdf\" />\n<meta property=\"og:type\" content=\"article\" />\n<meta property=\"twitter:card\" content=\"summary_large_image\" />\n<link rel=\"icon\" type=\"image/png\" sizes=\"192x192\" href=\"https://manubot.org/favicon-192x192.png\" />\n<link rel=\"mask-icon\" href=\"https://manubot.org/safari-pinned-tab.svg\" color=\"#ad1457\" />\n<meta name=\"theme-color\" content=\"#ad1457\" />\n<!-- end Manubot generated metadata -->"
keywords:
- pediatric cancer
- brain tumor
- tumor atlas
lang: en-US
manubot-clear-requests-cache: false
manubot-output-bibliography: output/references.json
manubot-output-citekeys: output/citations.tsv
manubot-requests-cache-path: ci/cache/requests-cache
title: An Open Pediatric Brain Tumor Atlas
...






<small><em>
This manuscript
([permalink](https://AlexsLemonade.github.io/OpenPBTA-manuscript/v/022d54780d9ddbfa8b79429fc87124a5eb135905/))
was automatically generated
from [AlexsLemonade/OpenPBTA-manuscript@022d547](https://github.com/AlexsLemonade/OpenPBTA-manuscript/tree/022d54780d9ddbfa8b79429fc87124a5eb135905)
on July 18, 2022.
</em></small>

## Authors



+ **Joshua A. Shapiro**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-6224-0347](https://orcid.org/0000-0002-6224-0347)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [jashapiro](https://github.com/jashapiro)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [jashapiro](https://twitter.com/jashapiro)<br>
  <small>   
     Childhood Cancer Data Lab, Alex's Lemonade Stand Foundation, Bala Cynwyd, PA, USA
     · Funded by Alex's Lemonade Stand Foundation Childhood Cancer Data Lab (CCDL)
  </small>

+ **Candace L. Savonen**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0001-6331-7070](https://orcid.org/0000-0001-6331-7070)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [cansavvy](https://github.com/cansavvy)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [cansavvy](https://twitter.com/cansavvy)<br>
  <small>   
     Childhood Cancer Data Lab, Alex's Lemonade Stand Foundation, Bala Cynwyd, PA, USA
     · Funded by Alex's Lemonade Stand Foundation Childhood Cancer Data Lab (CCDL)
  </small>

+ **Chante J. Bethell**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0001-9653-8128](https://orcid.org/0000-0001-9653-8128)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [cbethell](https://github.com/cbethell)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [cjbethell](https://twitter.com/cjbethell)<br>
  <small>   
     Childhood Cancer Data Lab, Alex's Lemonade Stand Foundation, Bala Cynwyd, PA, USA
     · Funded by Alex's Lemonade Stand Foundation Childhood Cancer Data Lab (CCDL)
  </small>

+ **Krutika S. Gaonkar**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0003-0838-2405](https://orcid.org/0000-0003-0838-2405)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [kgaonkar6](https://github.com/kgaonkar6)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [aggokittu](https://twitter.com/aggokittu)<br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Division of Neurosurgery, Children's Hospital of Philadelphia; Department of Bioinformatics and Health Informatics, Children's Hospital of Philadelphia
  </small>

+ **Run Jin**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-8958-9266](https://orcid.org/0000-0002-8958-9266)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [runjin326](https://github.com/runjin326)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [runjin](https://twitter.com/runjin)<br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Division of Neurosurgery, Children's Hospital of Philadelphia
  </small>

+ **Yuankun Zhu**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-2455-9525](https://orcid.org/0000-0002-2455-9525)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [yuankunzhu](https://github.com/yuankunzhu)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [zhuyuankun](https://twitter.com/zhuyuankun)<br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Division of Neurosurgery, Children's Hospital of Philadelphia
  </small>

+ **Miguel A. Brown**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0001-6782-1442](https://orcid.org/0000-0001-6782-1442)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [migbro](https://github.com/migbro)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [migbro](https://twitter.com/migbro)<br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Division of Neurosurgery, Children's Hospital of Philadelphia
  </small>

+ **Nhat Duong**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0003-2852-4263](https://orcid.org/0000-0003-2852-4263)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [fingerfen](https://github.com/fingerfen)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [asiannhat](https://twitter.com/asiannhat)<br>
  <small>   
     Department of Bioinformatics and Health Informatics, Children's Hospital of Philadelphia
  </small>

+ **Komal S. Rathi**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0001-5534-6904](https://orcid.org/0000-0001-5534-6904)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [komalsrathi](https://github.com/komalsrathi)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [komalsrathi](https://twitter.com/komalsrathi)<br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Department of Bioinformatics and Health Informatics, Children's Hospital of Philadelphia
  </small>

+ **Nighat Noureen**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0001-7495-8201](https://orcid.org/0000-0001-7495-8201)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [NNoureen](https://github.com/NNoureen)<br>
  <small>   
     Greehey Children's Cancer Research Institute, UT Health San Antonio
  </small>

+ **Bo Zhang**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-0743-5379](https://orcid.org/0000-0002-0743-5379)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [zhangb1](https://github.com/zhangb1)<br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Division of Neurosurgery, Children's Hospital of Philadelphia
  </small>

+ **Brian M. Ennis**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-2653-5009](https://orcid.org/0000-0002-2653-5009)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [bmennis](https://github.com/bmennis)<br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Division of Neurosurgery, Children's Hospital of Philadelphia
  </small>

+ **Stephanie J. Spielman**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-9090-4788](https://orcid.org/0000-0002-9090-4788)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [sjspielman](https://github.com/sjspielman)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [stephspiel](https://twitter.com/stephspiel)<br>
  <small>   
     Childhood Cancer Data Lab, Alex's Lemonade Stand Foundation, Bala Cynwyd, PA, USA<sup>☯</sup>; Rowan University, Glassboro, NJ, USA
     · Funded by Alex's Lemonade Stand Foundation Childhood Cancer Data Lab (CCDL)
      <br>
      <sup>☯</sup>Current affiliation
  </small>

+ **Laura E. Egolf**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-7103-4801](https://orcid.org/0000-0002-7103-4801)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [LauraEgolf](https://github.com/LauraEgolf)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [LauraEgolf](https://twitter.com/LauraEgolf)<br>
  <small>   
     Cell and Molecular Biology Graduate Group, Perelman School of Medicine at the University of Pennsylvania; Division of Oncology, Children's Hospital of Philadelphia
  </small>

+ **Yang Yang**<br>
    · ![GitHub icon](images/github.svg){.inline_icon}
    [yangyangclover](https://github.com/yangyangclover)<br>
  <small>   
     Ben May Department for Cancer Research, University of Chicago, Chicago IL, USA
  </small>

+ **Bailey Farrow**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0001-6727-6333](https://orcid.org/0000-0001-6727-6333)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [baileyckelly](https://github.com/baileyckelly)<br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Division of Neurosurgery, Children's Hospital of Philadelphia
  </small>

+ **Nicolas Van Kuren**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-7414-9516](https://orcid.org/0000-0002-7414-9516)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [nicholasvk](https://github.com/nicholasvk)<br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Division of Neurosurgery, Children's Hospital of Philadelphia
  </small>

+ **Meen Chul Kim**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-0308-783X](https://orcid.org/0000-0002-0308-783X)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [liberaliscomputing](https://github.com/liberaliscomputing)<br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Division of Neurosurgery, Children's Hospital of Philadelphia
  </small>

+ **Tejaswi Koganti**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-7733-6480](https://orcid.org/0000-0002-7733-6480)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [tkoganti](https://github.com/tkoganti)<br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Division of Neurosurgery, Children's Hospital of Philadelphia
  </small>

+ **Shrivats Kannan**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-1460-920X](https://orcid.org/0000-0002-1460-920X)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [shrivatsk](https://github.com/shrivatsk)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [kshrivats](https://twitter.com/kshrivats)<br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Division of Neurosurgery, Children's Hospital of Philadelphia
  </small>

+ **Pichai Raman**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0001-6948-2157](https://orcid.org/0000-0001-6948-2157)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [pichairaman](https://github.com/pichairaman)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [PichaiRaman](https://twitter.com/PichaiRaman)<br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Department of Bioinformatics and Health Informatics, Children's Hospital of Philadelphia
  </small>

+ **Jennifer Mason**<br>
    · ![GitHub icon](images/github.svg){.inline_icon}
    [jenn0307](https://github.com/jenn0307)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [jenn0307](https://twitter.com/jenn0307)<br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Division of Neurosurgery, Children's Hospital of Philadelphia
  </small>

+ **Daniel P. Miller**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-2032-4358](https://orcid.org/0000-0002-2032-4358)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [dmiller15](https://github.com/dmiller15)<br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Division of Neurosurgery, Children's Hospital of Philadelphia
  </small>

+ **Anna R. Poetsch**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0003-3056-4360](https://orcid.org/0000-0003-3056-4360)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [arpoe](https://github.com/arpoe)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [APoetsch](https://twitter.com/APoetsch)<br>
  <small>   
     Biotechnology Center, Technical University Dresden, Germany; National Center for Tumor Diseases, Dresden, Germany
  </small>

+ **Payal Jain**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-5914-9083](https://orcid.org/0000-0002-5914-9083)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [jainpayal022](https://github.com/jainpayal022)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [jainpayal022](https://twitter.com/jainpayal022)<br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Division of Neurosurgery, Children's Hospital of Philadelphia
  </small>

+ **Adam A. Kraya**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-8526-5694](https://orcid.org/0000-0002-8526-5694)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [aadamk](https://github.com/aadamk)<br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Division of Neurosurgery, Children's Hospital of Philadelphia
  </small>

+ **Allison P. Heath**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-2583-9668](https://orcid.org/0000-0002-2583-9668)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [allisonheath](https://github.com/allisonheath)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [allig8r](https://twitter.com/allig8r)<br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Division of Neurosurgery, Children's Hospital of Philadelphia
     · Funded by NIH U2C HL138346-03; NCI/NIH Contract No. 75N91019D00024, Task Order No. 75N91020F00003; Australian Government, Department of Education
  </small>

+ **Mateusz P. Koptyra**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-3857-6633](https://orcid.org/0000-0002-3857-6633)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [mkoptyra](https://github.com/mkoptyra)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [koptyram](https://twitter.com/koptyram)<br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Division of Neurosurgery, Children's Hospital of Philadelphia
  </small>

+ **Shannon Robbins**<br><br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Division of Neurosurgery, Children's Hospital of Philadelphia
  </small>

+ **Yiran Guo**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-6549-8589](https://orcid.org/0000-0002-6549-8589)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [Yiran-Guo](https://github.com/Yiran-Guo)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [YiranGuo3](https://twitter.com/YiranGuo3)<br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Division of Neurosurgery, Children's Hospital of Philadelphia
  </small>

+ **Xiaoyan Huang**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0001-7267-4512](https://orcid.org/0000-0001-7267-4512)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [HuangXiaoyan0106](https://github.com/HuangXiaoyan0106)<br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Division of Neurosurgery, Children's Hospital of Philadelphia
  </small>

+ **Jessica Wong**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0003-1508-7631](https://orcid.org/0000-0003-1508-7631)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [wongjessica93](https://github.com/wongjessica93)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [jessicawongbfx](https://twitter.com/jessicawongbfx)<br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Division of Neurosurgery, Children's Hospital of Philadelphia
  </small>

+ **Mariarita Santi**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-6728-3450](https://orcid.org/0000-0002-6728-3450)<br>
  <small>   
     Department of Pathology and Laboratory Medicine, Children’s Hospital of Philadelphia; Department of Pathology and Laboratory Medicine, University of Pennsylvania Perelman School of Medicine
  </small>

+ **Angela Viaene**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0001-6430-8360](https://orcid.org/0000-0001-6430-8360)<br>
  <small>   
     Department of Pathology and Laboratory Medicine, Children’s Hospital of Philadelphia; Department of Pathology and Laboratory Medicine, University of Pennsylvania Perelman School of Medicine
  </small>

+ **Laura Scolaro**<br><br>
  <small>   
     Division of Oncology, Children's Hospital of Philadelphia
  </small>

+ **Angela Waanders**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-0571-2889](https://orcid.org/0000-0002-0571-2889)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [awaanders](https://github.com/awaanders)<br>
  <small>   
     Department of Oncology, Ann & Robert H. Lurie Children’s Hospital of Chicago; Department of Pediatrics, Northwestern University Feinberg School of Medicine
  </small>

+ **Derek Hanson**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-0024-5142](https://orcid.org/0000-0002-0024-5142)<br>
  <small>   
     Hackensack Meridian School of Medicine; Hackensack University Medical Center
  </small>

+ **Steven M. Foltz**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-9526-8194](https://orcid.org/0000-0002-9526-8194)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [envest](https://github.com/envest)<br>
  <small>   
     Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania; Childhood Cancer Data Lab, Alex's Lemonade Stand Foundation, Bala Cynwyd, PA, USA
     · Funded by Alex's Lemonade Stand Foundation GR-000002471; National Institutes of Health K12GM081259
  </small>

+ **Hongbo M. Xie**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0003-2223-0029](https://orcid.org/0000-0003-2223-0029)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [xiehongbo](https://github.com/xiehongbo)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [xiehb](https://twitter.com/xiehb)<br>
  <small>   
     Department of Bioinformatics and Health Informatics, Children's Hospital of Philadelphia
  </small>

+ **Siyuan Zheng**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-1031-9424](https://orcid.org/0000-0002-1031-9424)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [syzheng](https://github.com/syzheng)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [zhengsiyuan](https://twitter.com/zhengsiyuan)<br>
  <small>   
     Greehey Children's Cancer Research Institute, UT Health San Antonio
  </small>

+ **Cassie N. Kline**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0001-7765-7690](https://orcid.org/0000-0001-7765-7690)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [cnkline13](https://twitter.com/cnkline13)<br>
  <small>   
     Division of Oncology, Children's Hospital of Philadelphia
  </small>

+ **Peter J. Madsen**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0001-9266-3685](https://orcid.org/0000-0001-9266-3685)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [petermadsenmd](https://twitter.com/petermadsenmd)<br>
  <small>   
     Division of Neurosurgery, Children's Hospital of Philadelphia; Center for Data-Driven Discovery, Children's Hospital of Philadelphia
  </small>

+ **Jena V. Lilly**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0003-1439-6045](https://orcid.org/0000-0003-1439-6045)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [jvlilly](https://github.com/jvlilly)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [jvlilly](https://twitter.com/jvlilly)<br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Division of Neurosurgery, Children's Hospital of Philadelphia
  </small>

+ **Philip B. Storm**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0002-7964-2449](https://orcid.org/0000-0002-7964-2449)<br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Division of Neurosurgery, Children's Hospital of Philadelphia
     · Funded by Alex's Lemonade Stand Foundation (Catalyst); Children’s Hospital of Philadelphia Division of Neurosurgery
  </small>

+ **Adam C. Resnick**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0003-0436-4189](https://orcid.org/0000-0003-0436-4189)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [adamcresnick](https://github.com/adamcresnick)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [adamcresnick](https://twitter.com/adamcresnick)<br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Division of Neurosurgery, Children's Hospital of Philadelphia
     · Funded by Alex's Lemonade Stand Foundation (Catalyst); Children's Brain Tumor Network; NIH 3P30 CA016520-44S5, U2C HL138346-03, U24 CA220457-03; NCI/NIH Contract No. 75N91019D00024, Task Order No. 75N91020F00003; Children’s Hospital of Philadelphia Division of Neurosurgery
  </small>

+ **Casey S. Greene**<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0001-8713-9213](https://orcid.org/0000-0001-8713-9213)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [cgreene](https://github.com/cgreene)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [greenescientist](https://twitter.com/greenescientist)<br>
  <small>   
     Department of Systems Pharmacology and Translational Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA, USA; Childhood Cancer Data Lab, Alex's Lemonade Stand Foundation, Bala Cynwyd, PA, USA; Center for Health AI, University of Colorado School of Medicine, Aurora, CO, USA; Department of Biochemistry and Molecular Genetics, University of Colorado School of Medicine, Aurora, CO, USA
     · Funded by Alex's Lemonade Stand Foundation Childhood Cancer Data Lab (CCDL)
  </small>

+ **Jo Lynne Rokita***<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0003-2171-3627](https://orcid.org/0000-0003-2171-3627)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [jharenza](https://github.com/jharenza)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [jolynnerokita](https://twitter.com/jolynnerokita)<br>
  <small>   
     Center for Data-Driven Discovery, Children's Hospital of Philadelphia; Division of Neurosurgery, Children's Hospital of Philadelphia; Department of Bioinformatics and Health Informatics, Children's Hospital of Philadelphia
     · Funded by Alex's Lemonade Stand Foundation (Young Investigator, Catalyst); NCI/NIH Contract No. 75N91019D00024, Task Order No. 75N91020F00003
  </small>

+ **Jaclyn N. Taroni***<br>
    ![ORCID icon](images/orcid.svg){.inline_icon}
    [0000-0003-4734-4508](https://orcid.org/0000-0003-4734-4508)
    · ![GitHub icon](images/github.svg){.inline_icon}
    [jaclyn-taroni](https://github.com/jaclyn-taroni)
    · ![Twitter icon](images/twitter.svg){.inline_icon}
    [jaclyn_taroni](https://twitter.com/jaclyn_taroni)<br>
  <small>   
     Childhood Cancer Data Lab, Alex's Lemonade Stand Foundation, Bala Cynwyd, PA, USA
     · Funded by Alex's Lemonade Stand Foundation Childhood Cancer Data Lab (CCDL)
  </small>

+ **Children's Brain Tumor Network**<br><br>
  <small>
  </small>

+ **Pacific Pediatric Neuro-Oncology Consortium**<br><br>
  <small>
  </small>


## Contact information
\*Correspondence: jaclyn.taroni@ccdatalab.org, rokita@chop.edu

<!-- Cell Genomics author instructions: https://marlin-prod.literatumonline.com/pb-assets/journals/research/cell-genomics/cell-genomics_IfA.pdf -->

## In Brief
<!-- aka the eTOC blurb. A short summary that describes the context and significance of the findings for the broader readership in 80 words or less. -->
The OpenPBTA is a global, collaborative open-science initiative which brought together researchers and clinicians to genomically characterize over 1,000 pediatric brain tumors. 
Shapiro, et. al create over 40 open-source, scalable modules to perform cancer genomics analyses and provide a richly-annotated somatic dataset across 59 brain tumor histologies. 
The OpenPBTA framework can be used as a model for large-scale data integration to inform basic research, therapeutic target identification, and clinical translation. 

## Highlights
<!-- Highlights are bullet points that convey the core findings of your paper. You may include up to four highlights. The length of each highlight cannot exceed 85 characters (including spaces).-->
OpenPBTA collaborative analyses establish resource for 1,253 pediatric brain tumors

NGS-based WHO-aligned integrated diagnoses generated for 1,260 of 2,840 biospecimens

RNA-Seq analysis infers medulloblastoma subtypes, TP53 status, and telomerase activity

OpenPBTA will accelerate therapeutic translation of genomic insights




## Summary
<!-- The summary is a single paragraph no longer than 150 words. An effective summary includes the following elements: (1) a brief background of the question that avoids statements about how a process is not well understood; (2) a description of the results and approaches/model systems framed in the context of their conceptual interest; and (3) an indication of the broader significance of the work. We discourage novelty claims (e.g., use of the word “novel”) because they are overused, tend not to add meaning, and are difficult to verify. Please do not include references in the summary. CURRENT WORD COUNT 146 -->
As pediatric brain and spinal cord tumors are the leading disease-related cause of death in children, we urgently need novel, curative therapeutic strategies for these tumors.
To accelerate such discoveries, the Children's Brain Tumor Network and Pacific Pediatric Neuro-Oncology Consortium created a systematic process for tumor biobanking, model generation, and sequencing while providing immediate access to harmonized data.
Here, we leverage these data to create OpenPBTA, an open and collaborative project to genomically characterize and molecularly subtype pediatric brain tumor specimens in what is, to our knowledge, the largest cohort of such samples to date.
OpenPBTA establishes over 40 scalable analysis modules to analyze these data and recapitulate key oncogenic drivers across histologies.
OpenPBTA has already emerged as a foundational analysis platform actively being applied to study other pediatric cancers and inform molecular tumor board decision-making, making it an invaluable resource to the pediatric oncology community.


## Keywords
<!-- We encourage you to include up to ten keywords with your paper. These keywords will be associated with your paper on Cell Press platforms and on PubMed. These keywords should be listed in the manuscript after the summary, separated by commas. -->
pediatric brain tumors, somatic variation, open science, reproducibility, classification  




## Introduction

Pediatric brain and spinal cord tumors are the second most common tumors in children after leukemia, and they represent the leading disease-related cause of death in children [@doi:10.1093/neuonc/noz150]. 
Five-year survival rates vary widely across different histologic and molecular classifications of brain tumors.
For example, most high-grade and embryonal tumors carry a universally fatal prognosis, while children with pilocytic astrocytoma have an estimated 10-year survival rate of 92% [@doi:10.1093/neuonc/now207]. 
Moreover, estimates from 2009 suggest that children and adolescents aged 0-19 with brain tumors in the United States have lost an average of 47,631 years of potential life [@doi:10.1002/cam4.410].

The low survival rates for some pediatric tumors are clearly multifactorial, explained partly by our lack of comprehensive understanding of the ever-evolving array of brain tumor molecular subtypes, difficulty drugging these tumors, and the shortage of drugs specifically labeled for pediatric malignancies. 
Historically, some of the most fatal, inoperable brain tumors, such as diffuse midline gliomas, were not routinely biopsied due to perceived risks of biopsy and the paucity of therapeutic options that would require tissue. 
Limited access to tissue to develop patient-derived cell lines and mouse models has been a barrier to research. 
Furthermore, the incidence of any single brain tumor molecular subtype is relatively low due to the rarity of pediatric tumors in general. 

To address these long-standing barriers, multiple national and international consortia have come together to collaboratively share specimens and data to accelerate breakthroughs and clinical translation.
The creation of the Pediatric Brain Tumor Atlas (PBTA) in 2018, led by the Children's Brain Tumor Network (CBTN) and the Pacific Pediatric Neuro-Oncology Consortium (PNOC), was one such effort.
Just as cooperation is required to share specimens and data, rigorous cancer genomic analysis requires collaboration among researchers with distinct expertise, such as computational scientists, bench scientists, clinicians, and pathologists.

Although there has been significant progress in recent years to elucidate the landscape of somatic variation responsible for pediatric brain tumor formation and progression, translation of therapeutic agents to phase II or III clinical trials and subsequent FDA approvals have not kept pace.
Within the last 20 years, the FDA has approved only five drugs for the treatment of pediatric brain tumors: mTOR inhibitor, everolimus, for subependymal giant cell astrocytoma [@doi:10.1097/01.COT.0000421359.88140.a8; @url:https://www.fda.gov/drugs/resources-information-approved-drugs/fda-approves-everolimus-tuberous-sclerosis-complex-associated-partial-onset-seizures]; anti-PD-1 immunotherapy, pembrolizumab, for microsatellite instability–high or mismatch repair–deficient tumors [@url:https://www.fda.gov/drugs/drug-approvals-and-databases/fda-approves-pembrolizumab-adults-and-children-tmb-h-solid-tumors]; NTRK inhibitors larotrectinib [@url:https://www.fda.gov/drugs/fda-approves-larotrectinib-solid-tumors-ntrk-gene-fusions] and entrectinib [@url:https://www.fda.gov/drugs/resources-information-approved-drugs/fda-approves-entrectinib-ntrk-solid-tumors-and-ros-1-nsclc] for tumors with an NTRK 1/2/3 gene fusions; MEK1/2 inhibitor, selumetinib, for neurofibromatosis type 1 (NF1) and symptomatic, inoperable plexiform neurofibromas [@url:https://www.fda.gov/drugs/resources-information-approved-drugs/fda-approves-selumetinib-neurofibromatosis-type-1-symptomatic-inoperable-plexiform-neurofibromas].
<!--TODO: Now that this introduction is longer, would we consider using the same citations to refer to the five FDA-approved drugs without listing them all out?-->
This is, in part, due to pharmaceutical company priorities and concerns regarding toxicity, making it challenging for researchers to obtain to new therapeutic agents for pediatric clinical trials.
Critically, as of August 18, 2020, an amendment to the Pediatric Research Equity Act called the "Research to Accelerate Cures and Equity (RACE) for Children Act" mandates that all new adult oncology drugs also be tested in children when the molecular targets are relevant to a particular childhood cancer.
The regulatory change introduced by the RACE Act, coupled with the identification of putative molecular targets in pediatric cancers through genomic characterization, is poised to accelerate identification of novel and effective therapeutic for pediatric diseases that have otherwise been overlooked.


To leverage diverse scientific and analytical expertise to analyze the PBTA data, we created an open science model and incorporated features, such as analytical code review [@doi:10.1093/aje/kwab092; @doi:10.7287/peerj.preprints.3210v1] and continuous integration to test data and code [@doi:10.1038/nbt.3780; @doi:10.7287/peerj.preprints.3210v1], to improve reproducibility throughout the life cycle of our project, termed OpenPBTA.
We anticipated that a model of open collaboration would enhance the value of our effort to the pediatric brain tumor research community. 
Openly sharing data and code in real time allows others to build upon our work more rapidly, and publications that include data and code sharing may have a greater impact [@doi:10.1371/journal.pone.0000308; @doi:10.1371/journal.pcbi.1008867].
Here, we present a comprehensive, collaborative, open genomic analysis of nearly 2,000 tumor specimens and 38 cell lines, comprised of 59 distinct brain tumor histologies from 943 patients. 
These analyses can be used to support the RACE Act in rational clinical trial design to include children with brain tumors.


## Results

### Crowd-sourced Somatic Analysis to create an Open Pediatric Brain Tumor Atlas

We previously performed whole genome sequencing (WGS), whole exome sequencing (WXS), and RNA sequencing (RNA-Seq) on matched tumor and normal tissues as well as selected cell lines from 943 patient tumors from the Pediatric Brain Tumor Atlas (PBTA) [@doi:10.1093/neuonc/noz192], consisting of samples from the [Children’s Brain Tumor Network (CBTN)](https://CBTN.org) and the PNOC003 DMG clinical trial [@doi:10.1002/ijc.32258] of the Pacific Pediatric Neuro-Oncology Consortium (PNOC) (**Figure {@fig:Fig1}A**).
We then harnessed the benchmarking efforts of the Kids First Data Resource Center to develop robust and reproducible data analysis workflows within the CAVATICA platform to perform primary somatic analyses including calling of single nucleotide variants (SNVs), copy number variants (CNVs), structural variants (SVs), and gene fusions, often implementing multiple complementary methods  (**Figure {@fig:S1}**) and **STAR Methods**).

To facilitate analysis and visualization of this large, diverse cohort, we further categorized tumor broad histologies (i.e., broad 2016 WHO classifications) into smaller groupings we denote "cancer groups."
A summarized view of the number of biospecimens per phase of therapy across different broad histologies and cancer groups is shown in (**Figure {@fig:Fig1}B**). 
We maintained a data release folder on Amazon S3, downloadable directly from S3 or through the open-access [CAVATICA project](https://www.cavatica.org/), with merged files for each analysis (See **Data and code availability** section).
As new analytical results (e.g., tumor mutation burden calculations) that we expected to be used across multiple analyses were produced, or issues with the data were identified, new data releases were made available in a versioned manner.

A key innovative feature of this project has been its open contribution model used for both analyses (i.e., analytical code) and scientific manuscript writing. 
We created a public Github analysis repository (https://github.com/AlexsLemonade/OpenPBTA-analysis) to hold all code associated with analyses downstream of the Kids First Data Resource Center workflows and a GitHub manuscript repository (https://github.com/AlexsLemonade/OpenPBTA-manuscript) with ManuBot [@doi:10.1371/journal.pcbi.1007128] integration to enable real-time manuscript creation using Markdown within GitHub.
Importantly, all analyses and manuscript writing were conducted openly throughout the research project, allowing any researcher in the world the opportunity to contribute.

The process for analysis and manuscript contributions is outlined in **Figure {@fig:Fig1}C**.
First, a potential contributor would propose an analysis by filing an issue in the GitHub analysis repository. 
Next, organizers for the project, or other contributors with expertise, had the opportunity to provide feedback about the proposed analysis (**Figure {@fig:Fig1}C**).
The contributor then made a copy (fork) of the analysis repository and added their proposed analysis code and results to their fork.
The contributor would formally request to include their analytical code and results to the main OpenPBTA analysis repository by filing a pull request on GitHub.
All pull requests to the analysis repository underwent peer review by organizers and/or other contributors to ensure scientific accuracy, maintainability, and readability of code and  documentation (**Figure {@fig:Fig1}C-D**).

The collaborative nature of the project required additional steps beyond peer review of analytical code to ensure consistent results for all collaborators and over time (**Figure {@fig:Fig1}D**).
We leveraged Docker® and the Rocker project [@arxiv:1710.03675] to maintain a consistent software development environment, creating a monolithic image that contained all dependencies necessary for analyses.
To ensure that new code would execute in the development environment, we used the continuous integration (CI) service CircleCI® to run analytical code on a small subset of data for testing before formal code review, allowing us to detect code bugs or sensitivity to changes in the underlying data.

We followed a similar process in our Manubot-powered [@doi:10.1371/journal.pcbi.1007128] manuscript repository for additions to the manuscript (**Figure {@fig:Fig1}C**). 
Contributors forked the manuscript repository, added proposed content to their branch, and filed pull requests to the main manuscript repository with their changes.
Similarly, pull requests underwent a peer review process for clarity and correctness, agreement with interpretation, and spell checking via Manubot.

![**Overview of the OpenPBTA Project.** A, The Children's Brain Tumor Network and the Pacific Pediatric Neuro-Oncology Consortium collected tumor samples from 943 patients. To date, 38 cell lines were created from tumor tissue, and over 2000 specimens were sequenced (N = 1035 RNA-Seq, N = 940 WGS, and N = 32 WXS or Targeted panel). Data was harmonized by the Kids First Data Resource Center using an Amazon S3 framework within CAVATICA. B, Stacked bar plot summary of the number of biospecimens per phase of therapy. Each panel denotes a broad histology and each bar denotes a cancer group. (Abbreviations: GNG = ganglioglioma, LGG = low-grade glioma, DIPG = diffuse intrinsic pontine glioma, DMG = diffuse midline glioma, EPN = ependymoma, HGG = high-grade glioma, ATRT = atypical teratoid rhabdoid tumor, CNS-ET = CNS embryonal tumor, MB = medulloblastoma, PNF = plexiform neurofibroma, DNET = dysembryoplastic neuroepithelial tumor, CRANIO = craniopharyngioma, EWS = Ewing sarcoma, CPP = choroid plexus papilloma). C, Overview of the open analysis and manuscript contribution model. In the analysis GitHub repository, a contributor would propose an analysis that other participants can comment on. Contributors would then implement the analysis and file a request to add their changes to the analysis repository ("pull request"). Pull requests underwent review for scientific rigor and correctness of implementation. Pull requests were additionally checked to ensure that all software dependencies were included and the code was not sensitive to underlying data changes using container and continuous integration technologies. Finally, a contributor would file a pull request documenting their methods and results to the Manubot-powered manuscript repository. Pull requests in the manuscript repository were also subject to review. D, A potential path for an analytical pull request. Arrows indicate revisions to a pull request. Prior to review, a pull request was tested for dependency installation and whether or not the code would execute. Pull requests also required approval by organizers and/or other contributors, who checked for scientific correctness. Panel A created with BioRender.com.](https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/0107be711b79f228f858f539fad1a4e8b2ced7c3/figures/pngs/figure1.png?sanitize=true){#fig:Fig1 width="7in"}


### Molecular Subtyping of OpenPBTA CNS Tumors

Over the past two decades, experts in neuro-oncology have worked with the World Health Organization (WHO) to iteratively redefine the classifications of central nervous system (CNS) tumors [@doi:10.1093/jnen/61.3.215; @doi:10.1007/s00401-007-0243-4].
More recently, in 2016 and 2021 [@doi:10.1007/s00401-016-1545-1; @doi:10.1093/neuonc/noab106], molecular subtypes have been integrated into these classifications.
In 2011, the Children’s Brain Tumor Tissue Consortium, now known as the Children’s Brain Tumor Network (CBTN), opened its protocol for brain tumor and matched normal sample collection.
Since the CBTN opened its protocol before molecular data were integrated into classifications, the majority of the samples within the OpenPBTA lacked molecular subtype annotations.
Moreover, the OpenPBTA data does not feature methylation arrays which are now commonly used to perform molecular subtyping.
Therefore, we jointly considered key genomic features of tumor entities described by the WHO in 2016, low-grade glioma (LGG) subtypes described by Ryall and colleagues [@doi:10.1016/j.ccell.2020.03.011], as well as clinician and pathologist review to subtype 64% (1,281/2,007) of tumor biospecimens with high confidence (**Table S1**).
Importantly, this collaborative molecular subtyping process allowed us to identify data entry errors (e.g., an ETMR incorrectly entered as a medulloblastoma) and mis-identified specimens (e.g., Ewing sarcoma sample labeled as a craniopharyngioma), update diagnoses using current WHO terms (e.g., tumors formerly ascribed primitive neuro-ectodermal tumor [PNET] diagnoses), and discover rarer tumor entities within the OpenPBTA (e.g., H3-mutant ependymoma, meningioma with _YAP1::FAM118B_ fusion).
**Table {@tbl:Table1}** lists the subtypes we defined within the OpenPBTA, comprising of LGATs (low grade astrocytic tumors; N = 569), high-grade gliomas (N = 314), embryonal tumors (N = 229), ependymomas (N = 65), tumors of sellar region (N = 51), mesenchymal non-meningothelial tumors (N = 21), glialneuronal tumors (N = 20), and chordomas (N = 12).
For detailed methods, see **STAR Methods** and **Figure {@fig:S1}**. 

| Broad histology                               | OpenPBTA molecular subtype                        | n    |
|-----------------------------------------------|------------------------------------------|------|
| Chordoma                                      | CHDM, conventional                       | 4    |
| Chordoma                                      | CHDM, poorly differentiated              | 8    |
| Diffuse astrocytic and oligodendroglial tumor | DMG, H3 K28                              | 54   |
| Diffuse astrocytic and oligodendroglial tumor | DMG, H3 K28, TP53 activated              | 26   |
| Diffuse astrocytic and oligodendroglial tumor | DMG, H3 K28, TP53 loss                   | 91   |
| Diffuse astrocytic and oligodendroglial tumor | HGG, H3 G35                              | 7    |
| Diffuse astrocytic and oligodendroglial tumor | HGG, H3 G35, TP53 loss                   | 2    |
| Diffuse astrocytic and oligodendroglial tumor | HGG, H3 wildtype                         | 74   |
| Diffuse astrocytic and oligodendroglial tumor | HGG, H3 wildtype, TP53 activated         | 10   |
| Diffuse astrocytic and oligodendroglial tumor | HGG, H3 wildtype, TP53 loss              | 45   |
| Diffuse astrocytic and oligodendroglial tumor | HGG, IDH, TP53 activated                 | 3    |
| Diffuse astrocytic and oligodendroglial tumor | HGG, IDH, TP53 loss                      | 2    |
| Embryonal tumor                               | CNS Embryonal, NOS                       | 24   |
| Embryonal tumor                               | CNS HGNET-MN1                            | 1    |
| Embryonal tumor                               | CNS NB-FOXR2                             | 5    |
| Embryonal tumor                               | ETMR, C19MC-altered                      | 8    |
| Embryonal tumor                               | ETMR, NOS                                | 1    |
| Embryonal tumor                               | MB, Group3                               | 24   |
| Embryonal tumor                               | MB, Group4                               | 91   |
| Embryonal tumor                               | MB, SHH                                  | 55   |
| Embryonal tumor                               | MB, WNT                                  | 20   |
| Ependymal tumor                               | EPN, H3 K28                              | 2    |
| Ependymal tumor                               | EPN, PF A                                | 6    |
| Ependymal tumor                               | EPN, ST RELA                             | 51   |
| Ependymal tumor                               | EPN, ST YAP1                             | 6    |
| Low-grade astrocytic tumor                    | GNG, BRAF V600E                          | 25   |
| Low-grade astrocytic tumor                    | GNG, BRAF V600E, CDKN2A/B                | 2    |
| Low-grade astrocytic tumor                    | GNG, FGFR                                | 2    |
| Low-grade astrocytic tumor                    | GNG, H3                                  | 2    |
| Low-grade astrocytic tumor                    | GNG, IDH                                 | 4    |
| Low-grade astrocytic tumor                    | GNG, KIAA1549-BRAF                       | 10   |
| Low-grade astrocytic tumor                    | GNG, MYB/MYBL1                           | 2    |
| Low-grade astrocytic tumor                    | GNG, NF1-germline                        | 2    |
| Low-grade astrocytic tumor                    | GNG, NF1-somatic, BRAF V600E             | 1    |
| Low-grade astrocytic tumor                    | GNG, other MAPK                          | 7    |
| Low-grade astrocytic tumor                    | GNG, other MAPK, IDH                     | 2    |
| Low-grade astrocytic tumor                    | GNG, RTK                                 | 6    |
| Low-grade astrocytic tumor                    | GNG, wildtype                            | 28   |
| Low-grade astrocytic tumor                    | LGG, BRAF V600E                          | 53   |
| Low-grade astrocytic tumor                    | LGG, BRAF V600E, CDKN2A/B                | 10   |
| Low-grade astrocytic tumor                    | LGG, FGFR                                | 16   |
| Low-grade astrocytic tumor                    | LGG, IDH                                 | 6    |
| Low-grade astrocytic tumor                    | LGG, KIAA1549-BRAF                       | 222  |
| Low-grade astrocytic tumor                    | LGG, KIAA1549-BRAF, other MAPK           | 2    |
| Low-grade astrocytic tumor                    | LGG, MYB/MYBL1                           | 4    |
| Low-grade astrocytic tumor                    | LGG, NF1-germline                        | 12   |
| Low-grade astrocytic tumor                    | LGG, NF1-germline, CDKN2A/B              | 2    |
| Low-grade astrocytic tumor                    | LGG, NF1-germline, FGFR                  | 4    |
| Low-grade astrocytic tumor                    | LGG, NF1-somatic                         | 4    |
| Low-grade astrocytic tumor                    | LGG, NF1-somatic, FGFR                   | 2    |
| Low-grade astrocytic tumor                    | LGG, NF1-somatic, NF1-germline, CDKN2A/B | 2    |
| Low-grade astrocytic tumor                    | LGG, other MAPK                          | 23   |
| Low-grade astrocytic tumor                    | LGG, RTK                                 | 22   |
| Low-grade astrocytic tumor                    | LGG, RTK, CDKN2A/B                       | 2    |
| Low-grade astrocytic tumor                    | LGG, wildtype                            | 84   |
| Low-grade astrocytic tumor                    | SEGA, wildtype                           | 6    |
| Mesenchymal non-meningothelial tumor          | EWS                                      | 21   |
| Neuronal and mixed neuronal-glial tumor       | CNC                                      | 4    |
| Neuronal and mixed neuronal-glial tumor       | EVN                                      | 2    |
| Neuronal and mixed neuronal-glial tumor       | GNT, BRAF V600E                          | 2    |
| Neuronal and mixed neuronal-glial tumor       | GNT, KIAA1549-BRAF                       | 4    |
| Neuronal and mixed neuronal-glial tumor       | GNT, other MAPK                          | 2    |
| Neuronal and mixed neuronal-glial tumor       | GNT, other MAPK, FGFR                    | 2    |
| Neuronal and mixed neuronal-glial tumor       | GNT, RTK                                 | 4    |
| Tumors of sellar region                       | CRANIO, ADAM                             | 51   |
|                                               | Total                                    | 1281 |

Table: **Molecular subtypes generated through the OpenPBTA project.** Listed are broad tumor histologies, molecular subtypes generated, and number of specimens subtyped within the OpenPBTA project. {#tbl:Table1}

### Somatic Mutational Landscape of Pediatric Brain Tumors

We performed a comprehensive genomic analysis of somatic SNVs, CNVs, SVs, and fusions across 1,969 tumors (N = 1,019 RNA-Seq, N = 1,719 WGS, N = 64 WXS/Panel) and 38 cell lines (N = 16 RNA-Seq, N = 22 WGS) from 943 patients.
Following SNV consensus calling (**Figure {@fig:S1}** and **Figure {@fig:S2}A-G**), we observed lower expected tumor mutation burden (TMB) **Figure {@fig:S2}H** in pediatric tumors compared to adult brain tumors from The Cancer Genome Atlas (TCGA), **Figure {@fig:S2}I**, with hypermutant (> 10 Mut/Mb) and ultra-hypermutant (> 100 Mut/Mb) tumors [@doi:10.1016/j.cell.2017.09.048] only found within HGGs.  

#### Low-grade astrocytic tumors
**Figure {@fig:Fig2}A** depicts an oncoprint of driver genes for 227 primary low-grade astrocytic tumors. As expected, the majority (62%, 140/227) of these tumors harbored a somatic alteration in _BRAF_, with canonical _BRAF::KIAA1549_ fusions as the major oncogenic driver [@doi:10.1186/s40478-020-00902-z].
We observed additional mutations in _FGFR1_ (2%), _PIK3CA_ (2%), _KRAS_ (2%), _TP53_ (1%), and _ATRX_ (1%) and fusions in _NTRK2_ (2%), _RAF1_ (2%), _MYB_ (1%), _QKI_ (1%), _ROS1_ (1%), and _FGFR2_ (1%), concordant with previous studies reporting the near universal upregulation of the RAS/MAPK pathway in these tumors resulting from activating mutations and/or oncogenic fusions [@doi:10.1186/s40478-020-00902-z; @doi:10.1016/j.ccell.2020.03.011].
Indeed, we observed significant upregulation (ANOVA p < 0.01) of the KRAS signaling pathway in LGATs (**Figure {@fig:Fig5}B**).

#### Embryonal tumors
**Figure {@fig:Fig2}B** shows the mutational landscape for 128 primary embryonal tumors. 
The majority (N = 95) of samples were medulloblastomas that spanned the spectrum of molecular subtypes (WNT, SHH, Group3, and Group 4; see **Molecular Subtyping of CNS Tumors**), as identified by subtype-specific canonical mutations.
We detected canonical _SMARCB1/SMARCA4_ deletions or inactivating mutations in atypical teratoid rhabdoid tumors (ATRTs) and C19MC amplification in the embryonal tumors with multilayer rosettes (ETMRs) [@doi:10.1007/s00401-020-02182-2; @doi:10.1093/neuonc/noab178; @doi:10.1186/s40478-020-00984-9; @doi:10.1038/nature22973].

#### Diffuse astrocytic and oligodendroglial tumors
In **Figure {@fig:Fig2}C**, we show genomic alterations in 61 HGGs biopsied at diagnosis: diffuse midline gliomas (DMGs, N = 34) and non-midline high-grade gliomas (HGGs, N = 26). 
The single oligodendroglioma sample (N = 1) in the OpenPBTA does not contain mutations in the listed genes and is therefore not shown in this oncoprint.
Across diffuse astrocytic and oligodendroglial tumors, we found that _TP53_ (57%, 35/61) and _H3F3A_ (52%, 32/61) were both most mutated and co-occurring genes (**Figure {@fig:Fig2}A**), followed by frequent mutations in _ATRX_ (30%, 18/61).
We found recurrent amplifications and fusions in _EGFR_, _MET_, _PDGFRA_, and _KIT_, highlighting that these tumors utilize multiple oncogenic mechanisms to activate tyrosine kinases, as has been previously reported [@doi:10.1002/ijc.32258; @doi:10.1016/j.ccell.2017.08.017; @doi:10.1186/s40478-020-00905-w].
Gene set enrichment analysis showed upregulation (ANOVA Bonferroni-corrected p < 0.01) of DNA repair, G2M checkpoint, and MYC pathways as well as downregulation of the TP53 pathway (**Figure {@fig:Fig5}B**). 
The two tumors with ultra-high TMB (> 100 Mutations/Mb) were from patients with known mismatch repair deficiency syndrome [@doi:10.1093/neuonc/noz192].

#### Other CNS tumors
**Figure {@fig:Fig2}D** depicts an oncoprint for the remaining primary CNS tumors in the OpenPBTA (N = 195). 
We observed that 25% (15/60) of ependymoma tumors were _C11orf95::RELA_ (now, _ZFTA::RELA_) fusion-positive ependymomas and that 70% (21/30) of craniopharyngiomas were driven by mutations in _CTNNB1_. 
Multiple histologies contained somatic mutations or fusions in _NF2_: 41% (7/17) of meningiomas, 5% (3/60) of ependymomas,  and 27% (3/11) schwannomas. 
Rare fusions in _ERBB4_, _YAP1_, _KRAS_, and _MAML2_ were observed in 10% (6/60) of ependymoma tumors. 
DNETs harbored alterations in MAPK/PI3K pathway genes as previously reported [@doi:10.1093/jnen/nlz101], including _FGFR1_ (21%, 4/19), _PDGFRA_ (10%, 2/19), and _BRAF_ (5%, 1/19).
Frequent mutations in rarer brain tumor histologies (N < 5 samples in the OpenPBTA) are depicted in **Figure {@fig:S3}A**.

![**Mutational landscape of PBTA tumors.** Shown are frequencies of canonical somatic gene mutations, CNVs, fusions, and TMB (top bar plot) for the top 20 genes mutated across primary tumors within the OpenPBTA dataset. A, Low-grade astrocytic tumors (N = 227): low-grade glioma astrocytoma (N = 187), ganglioglioma (N = 35), subependymal giant cell astrocytoma (N = 2), diffuse fibrillary astrocytoma (N = 1), pilocytic astrocytoma (N = 1), and pleomorphic xanthoastrocytoma (N = 1); B, Embryonal tumors (N = 128): medulloblastomas (N = 95), atypical teratoid rhabdoid tumors (N = 24), embryonal tumors with multilayer rosettes (N = 2), other CNS embryonal tumors (N = 5), ganglioneuroblastoma (N = 1), and CNS neuroblastoma (N = 1); C, Diffuse astrocytic and oligodendroglial tumors (N = 61): diffuse midline gliomas (N = 34) and non-midline high-grade gliomas (N = 26), oligodendroglioma (N = 1); D, Other CNS tumors (N = 195): ependymomas (N = 60), dysembryoplastic neuroepithelial tumors (N = 19), meningiomas (N = 17), schwannoma (N = 11), neurofibroma plexiform (N = 7). Other CNS tumors with N < 5 are displayed in **Figure {@fig:S3}A**. Tumor histology and patient sex are displayed as annotations at the bottom of each plot. Only samples with mutations in the listed genes are shown. Multiple CNVs are denoted as a complex event.](https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/0107be711b79f228f858f539fad1a4e8b2ced7c3/figures/pngs/figure2.png?sanitize=true){#fig:Fig2 width="7in"}


### Mutational co-occurrence, CNV and signatures highlight key oncogenic drivers


We analyzed mutational co-occurrence among OpenPBTA tumors, using a single sequencing sample from each individual with available WGS (N = 666).
The top 50 mutated genes (see **STAR Methods** for details) in primary tumors are shown in **Figure {@fig:Fig3}** by tumor type (**A**, bar plots), with co-occurrence scores illustrated in the heatmap (**B**).
_TP53_ was the most frequently mutated gene across OpenPBTA tumors (8.4%, 56/666), significantly co-occurring with _H3F3A_ (OR = 32, 95% CI: 15.3 - 66.7, q = 8.46e-17), _ATRX_ (OR = 20, 95% CI: 8.4 - 47.7, q = 4.43e-8), _NF1_ (OR = 8.62, 95% CI: 3.7 - 20.2, q = 5.45e-5), and _EGFR_ (OR = 18.2, 95% CI: 5 - 66.5, q = 1.6e-4). 
Other canonical cancer driver genes that were frequently mutated included _BRAF_, _H3F3A_, _CTNNB1_, _NF1_, _ATRX_, _FGFR1_, and _PIK3CA_. 
Although LGG and embryonal tumors made up the majority of tumor types within the OpenPBTA, most of the significant gene interactions stemmed from HGGs (N = 847/872).

At the broad histology level, _CTNNB1_ significantly co-occurred with _TP53_ (OR = 42.9, 95% CI: 7 - 261.4, q = 1.63e-3) and _DDX3X_ (OR = 21.1, 95% CI: 4.6 - 96.3, q = 4.46e-3) in embryonal tumors. _FGFR1_ and _PIK3CA_ significantly co-occur in LGGs (OR = 76.1, 95% CI: 9.85 - 588.1, q = 3.26e-3), consistent with previous findings [@doi:10.1200/JCO.2010.31.1670; @doi:10.1186/s40478-020-01027-z].
Of HGG tumors with mutations in _TP53_ or _PPM1D_, 52/54 (96.3%) had mutations in only one of these genes (OR = 0.188, 95% CI: 0.04 - 0.94, p = 0.0413, q = 0.0587).
This trend recapitulates previous observations that _TP53_ and _PPM1D_ mutations tend to be mutually exclusive in HGGs [@doi:10.1038/ng.2938].

We summarized broad CNV and SV and observed that HGGs and DMGs, followed by medulloblastomas, had the most unstable genomes (**Figure {@fig:S3}A**).
By contrast, craniopharyngiomas and schwannomas generally lacked somatic CNV.
Together, these CNV patterns largely aligned with our estimates of tumor mutational burden (**Figure {@fig:S2}H**). 
The number of SV and CNV breakpoints were significantly correlated across tumors (p = 1.08e-37) (**Figure {@fig:Fig3}C**) and as expected, the number of chromothripsis regions called increased as breakpoint density increased (**Figure {@fig:S3}B-C**).
We identified chromothripsis events in 41% (N = 19/46) of non-midline high-grade gliomas (high-grade glioma astrocytomas) and in 28.2% (N = 11/39) of diffuse midline gliomas (**Figure {@fig:Fig3}D**). 
We also found evidence of chromothripsis in over 15% of sarcomas, metastatic secondary tumors, embryonal tumors, chordomas, glial-neuronal tumors, germinomas, meningiomas, and ependymomas, highlighting the genomic instability and complexity of these pediatric brain tumors. 

We next assessed the contributions of eight previously identified adult CNS-specific mutational signatures from the RefSig database [@doi:10.1038/s43018-020-0027-5] across samples (**Figure {@fig:Fig3}E** and **Figure {@fig:S4}A**).
Stage 0 and/or 1 tumors characterized by low TMBs (**Figure {@fig:S2}G**) such as LGGs, gangliogliomas, craniopharyngiomas, DNETs, and schwannomas were dominated by Signature 1, which results from the normal process of spontaneous deamination of 5-methylcytosine.
Signature N6 is a CNS-specific signature which we observed nearly universally across samples.
Drivers of Signature 18, _TP53_, _APC_, _NOTCH1_ (found at https://signal.mutationalsignatures.com/explore/referenceCancerSignature/31/drivers), are also canonical drivers of medulloblastoma, and indeed, we observed Signature 18 as the most common signature in medulloblastoma tumors.
Signatures 3, 8, 18, and MMR2 were prevalent in HGGs, including DMGs.
Finally, we found that the Signature 1 weight was higher at diagnosis (pre-treatment) and was almost always lower in tumors at later phases of therapy (progression, recurrence, post-mortem, secondary malignancy; **Figure {@fig:S4}B**).
This trend may have resulted from therapy-induced mutations that produced additional signatures (e.g., temozolomide treatment has been suggested to drive Signature 11 [@doi:1053/j.gastro.2014.07.052]), subclonal expansion, and/or acquisition of additional driver mutations during tumor progression, leading to higher overall TMBs and additional signatures.

<!-- To do: possibly explore gene mutations driving signature 8, specifically for meningiomas, and describe -->

![**Mutational co-occurrence and signatures highlight key oncogenic drivers.** A, Bar plot of occurrence and co-occurrence of nonsynonymous mutations for the 50 most commonly mutated genes across all tumor types, which are denoted as "Other" when there are fewer than 10 samples per grouping; B, Co-occurrence and mutual exclusivity of nonsynonymous mutations between genes; The co-occurrence score is defined as $I(-\log_{10}(P))$ where $P$ is defined by Fisher's exact test and $I$ is 1 when mutations co-occur more often than expected and -1 when exclusivity is more common; C, The number of SV breaks significantly correlate with CNV breaks (Adjusted R = 0.436, p = 1.08e-37). D, Chromothripsis frequency across pediatric brain tumors for all cancer groups with N >= 3 samples. E, Sina plots of RefSig signature weights for signatures 1, 3, 8, 11, 18, 19, N6, MMR2, and Other across cancer groups. Box plot lines represent the first quartile, median, and third quartile.](https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/0107be711b79f228f858f539fad1a4e8b2ced7c3/figures/pngs/figure3.png?sanitize=true){#fig:Fig3 width="7in"}

### Transcriptomic Landscape of Pediatric Brain Tumors

#### Prediction of _TP53_ oncogenicity and telomerase activity

To understand the _TP53_ phenotype in each tumor, we ran a classifier previously trained on TCGA [@doi:10.1016/j.celrep.2018.03.076] to calculate a _TP53_ score and infer _TP53_ inactivation status.
We compared results of this classifier to "true positive" alterations derived using high-confidence SNVs, CNVs, SVs, and fusions in _TP53_.
Specifically, we annotated _TP53_ alterations as "activated" if samples harbored one of p.R273C or p.R248W mutations [@doi:10.1038/ng0593-42], or "lost" if the given patient either had a Li Fraumeni Syndrome (LFS) predisposition diagnosis, the tumor harbored a known hotspot mutation, or the tumor contained two hits (e.g. both SNV and CNV), which would suggest both alleles had been affected.
If the _TP53_ mutation did not reside within the DNA-binding domain or we did not detect any alteration in _TP53_, we annotate the tumor as "other," reflecting its unknown _TP53_ alteration status.
The classifier achieved a high accuracy (AUROC = 0.85) for rRNA-depleted, stranded samples compared to randomly shuffled _TP53_ scores (**Figure {@fig:Fig4}A**).
By contrast, while this classifier has previously shown strong performance on poly-A data from both adult [@doi:10.1016/j.celrep.2018.03.076] tumors and pediatric patient-derived xenografts [@doi:10.1016/j.celrep.2019.09.071], it did not perform as well on the poly-A samples in this cohort (AUROC = 0.62; **Figure {@fig:S5}A**).

While we expected that samples annotated as "lost" would have higher _TP53_ scores than would samples annotated as "other," we observed that samples annotated as "activated" had similar _TP53_ scores to those annotated as "lost" (**Figure {@fig:Fig4}B**, Wilcoxon p = 0.23).
This result suggests that the classifier actually detects an oncogenic, or altered, _TP53_ phenotype (scores > 0.5) rather than solely _TP53_ inactivation, as interpreted previously [@doi:10.1016/j.celrep.2018.03.076].
Moreover, tumors with "activating" _TP53_ mutations had evidence of higher _TP53_ expression compared to those with _TP53_ "loss" mutations (Wilcoxon p = 3.5e-3, **Figure {@fig:Fig4}C**).
Tumor types with the highest median _TP53_ scores were those known to harbor somatic _TP53_ alterations and included DMGs, medulloblastomas, HGGs, DNETs, ependymomas, and craniopharyngiomas (**Figure {@fig:Fig4}D**), while gangliogliomas, LGGs, meningiomas, and schwannomas had the lowest median scores.

To further validate the classifier's accuracy, we assessed _TP53_ scores for patients with LFS, hypothesizing that all of these tumors would have high scores.
Indeed, we observed higher scores in LFS tumors (N = 8) for which we detected high-confidence _TP53_ somatic alterations (**Tables S1 and S3**).
Although we did not detect canonical somatic _TP53_ mutations in two LFS patient tumors with low _TP53_ scores (BS_DEHJF4C7 with a score of 0.09 and BS_ZD5HN296 with a score of 0.28), we confirmed the LFS diagnoses from pathology reports and noted that patient had a pathogenic germline variant in _TP53_.
In addition, the tumor purity of these two LFS samples was low (16% and 37%, respectively), suggesting the classifier may require a certain level of tumor purity to achieve good performance, as we expect _TP53_ to be intact in normal cells.


We next used gene expression data to predict telomerase activity using EXpression-based Telomerase ENzymatic activity Detection (EXTEND) [@doi:10.1038/s41467-020-20474-9] as a surrogate measure of malignant potential [@doi:10.1038/s41467-020-20474-9; @doi:10.1093/carcin/bgp268], such that higher EXTEND scores suggest increase malignant potential.
As expected, EXTEND scores significantly correlated with _TERC_ (R = 0.619, p < 0.01) and _TERT_ (R = 0.491, p < 0.01) expression (**Figure {@fig:S5}B-C**).  
We found aggressive tumors such as HGGs (DMGs and other high-grade gliomas) and MB had high EXTEND scores (**Figure {@fig:Fig4}D**), while benign lesions such as schwannomas, GNGs, DNETs, and other low-grade gliomas had among the lowest scores (**Table S3**).
These findings support previous reports of a more aggressive phenotype in tumors with higher telomerase activity	[@doi:10.1007/s13277-016-5045-7; @doi:10.1038/labinvest.3700710; @doi:10.1007/s12032-016-0736-x; @doi:10.1111/j.1750-3639.2010.00372.x].


#### Hypermutant tumors share mutational signatures and have dysregulated **_TP53_**


We further investigated the mutational signature profiles of the hypermutant (TMB > 10 Mut/Mb; N = 2) and ultra-hypermutant (TMB > 100 Mut/Mb; N = 4) HGG tumors and/or derived cell lines from five patients in the OpenPBTA cohort (**Figure {@fig:Fig4}E**).
Signature 11, which has been suggested to result from exposure to temozolomide, was indeed present in tumors with previous exposure to the drug (**Table {@tbl:Table2}**).
We detected the MMR2 signature in tumors of four patients (PT_0SPKM4S8, PT_3CHB9PK5, PT_JNEV57VK, and PT_VTM2STE3) diagnosed with Lynch syndrome (**Table {@tbl:Table2}**), a genetic predisposition syndrome caused by a variant in a mismatch repair gene such as _PMS2_, _MLH1_, _MSH2_, _MSH6_, or others [@doi:10.3390/genes5030497].
Interestingly, while the cell line derived from patient PT_VTM2STE3's tumor at progression was not hypermutated (TMB = 5.74 Mut/Mb), it solely showed the MMR2 signature of the eight CNS signatures examined, suggesting selective pressure to maintain a mismatch repair (MMR) phenotype _in vitro_.
From patient PT_JNEV57VK, only one of the two cell lines derived from the progressive tumor was hypermutated (TMB = 35.9 Mut/Mb).
This hypermutated cell line was strongly weighted towards signature 11, while this patient's non-hypermutated cell line showed a number of lesser signature weights (1, 11, 18, 19, MMR2), highlighting the plasticity of mutational processes and the need to carefully genomically characterize and select models for preclinical studies based on research objectives.

We observed that signature 18, which has been associated with high genomic instability and can lead to a hypermutator phenotype [@doi:10.1038/s43018-020-0027-5], was uniformly represented among hypermutant solid tumors.
Additionally, we found that all of these tumor or cell lines had dysfunctional _TP53_ (**Table {@tbl:Table2}**), consistent with a previous report showing _TP53_ dysregulation is a dependency in tumors with high genomic instability [@doi:10.1038/s43018-020-0027-5].
With one exception, hypermutant and ultra-hypermutant tumors had high  _TP53_ scores (> 0.5) and telomerase activity.
Interestingly, none of the hypermutant samples showed evidence of signature 3 (present in homologous recombination deficient tumors), signature 8 (arises from double nucleotide substitutions/unknown etiology), or signature N6 (a universal CNS tumor signature).
The mutual exclusivity of signatures 3 and MMR2 corroborates a previous report suggesting tumors do not tend to feature both deficient homologous repair and mismatch repair [@doi:10.1016/j.celrep.2018.03.076].


| Kids First Participant ID | Kids First Biospecimen ID | CBTN ID   | Phase of therapy  | Composition       | Therapy post-biopsy                              | Cancer predisposition(s)              | TMB                | OpenPBTA molecular subtype       |
|---------------------------|---------------------------|-----------|-------------------|-------------------|--------------------------------------------------|---------------------------------------|--------------------|----------------------------------|
| PT_0SPKM4S8               | BS_VW4XN9Y7               | 7316-2640 | Initial CNS Tumor | Solid Tissue      | Temozolomide, CCNU, Radiation                    | None documented                       | 187.4 | HGG, H3 wildtype, TP53 activated |
| PT_3CHB9PK5               | BS_20TBZG09               | 7316-515  | Initial CNS Tumor | Solid Tissue      | Temozolomide, Irinotecan, Bevacizumab, Radiation | NF-1,Other inherited conditions NOS   | 307.0  | HGG, H3 wildtype, TP53 loss      |
| PT_3CHB9PK5               | BS_8AY2GM4G               | 7316-2085 | Progressive       | Solid Tissue      | Unknown                                          | NF-1, Lynch syndrome, pathogenic MSH6 | 321.6  | HGG, H3 wildtype, TP53 loss      |
| PT_JNEV57VK               | BS_85Q5P8GF               | 7316-2594 | Initial CNS Tumor | Solid Tissue      | Temozolomide, Radiation                          | Lynch syndrome                        | 4.7  | DMG, H3 K28, TP53 loss           |
| PT_JNEV57VK               | BS_HM5GFJN8               | 7316-3058 | Progressive       | Derived Cell Line | Nivolumab                                        | Lynch Syndrome, pathogenic MSH2       | 35.9  | DMG, H3 K28, TP53 loss           |
| PT_JNEV57VK               | BS_QWM9BPDY               | 7316-3058 | Progressive       | Derived Cell Line | Nivolumab                                        | Lynch Syndrome, pathogenic MSH2       | 7.4  | DMG, H3 K28, TP53 loss           |
| PT_JNEV57VK               | BS_P0QJ1QAH               | 7316-3058 | Progressive       | Solid Tissue      | Nivolumab                                        | Lynch Syndrome, pathogenic MSH2       | 6.3  | DMG, H3 K28, TP53 activated      |
| PT_S0Q27J13               | BS_P3PF53V8               | 7316-2307 | Initial CNS Tumor | Solid Tissue      | Temozolomide, Irinotecan, Radiation              | None documented                       | 15.5 | HGG, H3 wildtype, TP53 activated |
| PT_VTM2STE3               | BS_ERFMPQN3               | 7316-2189 | Progressive       | Derived Cell Line | Unknown                                          | Lynch Syndrome                        | 5.7  | HGG, H3 wildtype, TP53 loss      |
| PT_VTM2STE3               | BS_02YBZSBY               | 7316-2189 | Progressive       | Solid Tissue      | Unknown                                          | Lynch Syndrome                        | 274.5 | HGG, H3 wildtype, TP53 activated |

Table: **Patients with hypermutant tumors.** Listed are patients with at least one hypermutant or ultra-hypermutant tumor or cell line. Coding region TMB, phase of therapy, therapeutic interventions, cancer predispositions, and molecular subtypes are included. {#tbl:Table2}

Next, we asked whether transcriptomic classification of _TP53_ dysregulation and/or telomerase activity recapitulate the known prognostic influence of these oncogenic biomarkers.
To this end, we conducted a multivariate cox regression on overall survival (**Figure {@fig:Fig4}F**; **STAR Methods**), controlling for extent of tumor resection and whether a tumor was low-grade (LGG group) or high-grade (HGG group).
We identified several expected trends, including a significant overall survival benefit if the tumor had been fully resected (HR = 0.36, 95% CI = 0.2 - 0.67, p = 0.001) or if the tumor belonged to the LGG group (HR = 0.068, 95% CI = 0.0091 - 0.5, p = 0.008) as well as a significant risk if the tumor belonged to the HGG group (HR = 5.7, 95% CI = 3.5 - 9.0, p < 0.001).
High telomerase scores were a poor prognostic indicator across brain tumor histologies (HR = 24, 95% CI = 7.1 - 79, p < 0.001), demonstrating that EXTEND scores calculated from RNA-Seq are an effective rapid surrogate measure for telomerase activity.
Although higher _TP53_ scores, which predict _TP53_ gene or pathway dysregulation, were not a significant predictor of risk across the entire OpenPBTA cohort (**Table S4**), we did find a significant survival risk for both DMGs (HR = 1.77e6, 95% CI = 1.98-1.57e12, p = 0.04) and ependymomas (HR = 1612, 95 % CI = 9.1-2.9e5, p = 0.005).
Since we observed the negative prognostic effect of _TP53_ scores for HGGs, we assessed the effect of molecular subtypes within HGG samples on survival risk.
We found that DMG H3 K28 tumors with _TP53_ loss had significantly worse prognosis (HR = 3.2, CI = 1.5-6.6, p = 0.002) than did DMG H3 K28 tumors with wildtype _TP53_ (**Figure {@fig:Fig4}G** and **Figure {@fig:Fig4}H**).

![**_TP53_ and telomerase activity** A, Receiver Operating Characteristic for _TP53_ classifier run on FPKM of stranded RNA-Seq samples. B, Violin and strip plots of _TP53_ scores from stranded RNA-Seq samples plotted by _TP53_ alteration type (N<sub>activated</sub> = 11, N<sub>lost</sub> = 96, N<sub>other</sub> = 862). C, Violin and strip plots of _TP53_ RNA expression from stranded RNA-Seq samples plotted by _TP53_ activation status (N<sub>activated</sub> = 11, N<sub>lost</sub> = 96, N<sub>other</sub> = 862). D, Box plots of _TP53_ and telomerase (EXTEND) scores across cancer groups. Mutation status is highlighted in orange (hypermutant) or red (ultra-hypermutant). E, Heatmap of RefSig mutational signatures for patients who have least one tumor or cell line with a hypermutant phenotype. F, Forest plot depicting the prognostic effects of _TP53_ and telomerase scores on overall survival, controlling for extent of tumor resection, LGG group, and HGG group. G, Forest plot depicting the effect of molecular subtype on overall survival of HGGs. For F and G, hazard ratios (HR) with 95% confidence intervals and p-values are listed. Significant p-values are denoted with black diamonds. Reference groups are denoted by grey diamonds. H, Kaplan-Meier curve of HGG tumors by molecular subtype.](https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/0107be711b79f228f858f539fad1a4e8b2ced7c3/figures/pngs/figure4.png?sanitize=true){#fig:Fig4 width="7in"}

#### Histologic and oncogenic pathway clustering

UMAP visualization of gene expression variation across brain tumors (**Figure {@fig:Fig5}A**) showed the expected clustering of brain tumors by histology.
We additionally explored UMAP projections of gene expression within molecular subtypes for certain cancer groups.
We observed that, except for three outliers, _C11orf95::RELA_ (_ZFTA::RELA_) fusion-positive ependymomas fell within distinct clusters (**Figure {@fig:S6}A**).
Medulloblastoma (MB) samples cluster by molecular subtype, with WNT and SHH in distinct clusters and Groups 3 and 4 showing some overlap (**Figure {@fig:S6}B**), as expected.
Of note, two MB samples annotated as the SHH subtype did not cluster with the other MB samples, and one clustered with Group 3 and 4 samples, suggesting potential subtype misclassification or different underlying biology of these two tumors.
_BRAF_-driven low-grade gliomas (**Figure {@fig:S6}C**) were present in three separate clusters, suggesting that there might be additional shared biology within each cluster.
Histone H3 G35-mutant high-grade gliomas generally clustered together and away from K28-mutant tumors (**Figure {@fig:S6}D**).
Interestingly, although H3 K28-mutant tumors have different biological drivers than do H3 wildtype tumors [@doi:10.1126/science.1232245], they did not form distinct clusters.
This pattern suggests these subtypes may be driven by common transcriptional programs, have other much stronger biological drivers than their known distinct epigenetic drivers, or our sample size is too small to detect transcriptional differences.

We next performed gene set variant analysis (GSVA) for Hallmark cancer gene sets to demonstrate activation of underlying oncogenic pathways (**Figure {@fig:Fig5}B** and quantified immune cell fractions across OpenPBTA tumors using quanTIseq (**Figure {@fig:Fig5}C** and **Figure {@fig:S6}E**).
Through these analyses, we were able to recapitulate previously-described tumor biology.
For example, HGG, DMG, MB, and ATRT tumors are known to upregulate _MYC_ [@doi:10.3390/genes8040107] which in turn activates _E2F_ and S phase [@pmid:11511364].
Indeed, we detected significant (Bonferroni-corrected p < 0.05) upregulation of _MYC_ and _E2F_ targets, as well as G2M (cell cycle phase following S phase) in both embryonal and diffuse astrocytic and oligodendroglial tumors compared to several other cancer groups.
In contrast, low-grade astrocytic tumors showed significant downregulation (Bonferroni-corrected p < 0.05) of these pathways.
Schwannomas and neurofibromas, which have a documented inflammatory immune microenvironment of T and B lymphocytes as well as tumor-associated macrophages (TAMs), are driven by upregulation of cytokines such as IFN$\gamma$, IL-1, and IL-6, and TNF$\alpha$ [@doi:10.1093/noajnl/vdaa023].
Indeed, we observed significant upregulation of these cytokines in GSVA hallmark pathways (Bonferroni-corrected p < 0.05) (**Figure {@fig:Fig5}B**) and found immune cell types dominated by monocytes in these tumors (**Figure {@fig:Fig5}C**).
We also observed significant upregulation of pro-inflammatory cytokines IFN$\alpha$ and IFN$\gamma$ in LGG and craniopharyngiomas compared to medulloblastoma and ependymoma tumors (Bonferroni-corrected p < 0.05), both of which showed significant down-regulation of these cytokines (**Figure {@fig:Fig5}B**).
Together, these results supported previous proteogenomic findings of lower immune infiltration in aggressive medulloblastomas and ependymomas versus higher immune infiltration in _BRAF_-driven LGG and craniopharyngiomas [@doi:10.1016/j.cell.2020.10.044]. 


Although CD8+ T-cell infiltration across all cancer groups was quite low (**Figure {@fig:Fig5}C**), we observed some signal in specific cancer molecular subtypes (medulloblastoma Groups 3 and 4) as well as outlier tumors (BRAF-driven LGG, BRAF-driven and wildtype ganglioglioma, H3 wildtype HGG; **Figure {@fig:S6}E**)
Surprisingly, the classically immunologically-cold HGG and DMG tumors [@doi:10.1186/s40478-018-0553-x; @doi:10.1093/brain/awab155] contained higher overall fractions of immune cells, where monocytes, dendritic cells, and NK cells were the most prevalent (**Figure {@fig:Fig5}C**).
Thus, we suspect that quanTIseq might actually have captured microglia within these immune cell fractions.

While we did not detect notable prognostic effects of immune cell infiltration in HGG or DMG tumors, we did find that high levels of macrophage M1 and monocytes were associated with poorer overall survival (monocyte HR = 5.2e17, 95% CI = 1.2e5 - 2.3e30, p = 0.006) in medulloblastoma tumors (**Figure {@fig:Fig5}D**).
We further reproduced previous findings (**Figure {@fig:Fig5}E**) that medulloblastomas typically have low expression of _CD274_ (PD-L1) [@doi:10.18632/oncotarget.24951].
However, we also found that higher expression of _CD274_ was significantly associated with improved overall prognosis for medulloblastoma samples, although with a marginal effect size (HR = 0.00074, 95% CI = 3.0e-6 - 0.18, p = 0.01) (**Figure {@fig:Fig5}D**).
This result may be explained by the higher expression of _CD274_ found in WNT subtype tumors by us and others [@doi:10.1080/2162402X.2018.1462430], as this diagnosis carries the best prognosis of all medulloblastoma subgroups (**Figure {@fig:Fig5}E**).

Finally, we asked whether any molecular subtypes might show an immunologically-hot phenotype, as roughly defined by a greater proportion of CD8+ to CD4+ T cells [@doi:10.1136/jitc-2021-004012; @doi:10.4236/jct.2013.48164].
While Group 3 and Group 4 medulloblastomas had the highest ratios (**Figure {@fig:S6}F**), very few tumors had ratios greater than 1, highlighting an urgent need to identify novel therapeutics for these immunologically-cold pediatric brain tumors with poor prognosis.

![**Transcriptomic and immune landscape of pediatric brain tumors** A, First two dimensions from UMAP of sample transcriptome data. Points are colored by the broad histology of the samples they represent. B, Heatmap of GSVA scores for Hallmark gene sets with significant differences, with samples ordered by cancer group. C, Box plots of quanTIseq estimates of immune cell proportions in cancer groups with N > 15 samples. D, Forest plot depicting the additive effects of _CD274_ expression, immune cell proportion, and extent of tumor resection on overall survival of medulloblastoma patients. Hazard ratios (HR) with 95% confidence intervals and p-values are listed. Significant p-values are denoted with black diamonds. Reference groups are denoted by grey diamonds. Of note, the Macrophage M1 HR was 0 (coefficient = -8.95e04) with infinite upper and lower CIs, and thus it was not included in the figure. E, Box plot of _CD274_ expression (log<sub>2</sub> FPKM) for medulloblastoma samples grouped by molecular subtype. Bonferroni-corrected p-values from Wilcoxon tests are shown.](https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/0107be711b79f228f858f539fad1a4e8b2ced7c3/figures/pngs/figure5.png?sanitize=true){#fig:Fig5 width="7in"}


## Discussion

We created OpenPBTA to provide an open, reproducible analysis framework to genomically characterize pediatric brain tumors and bring together researchers and clinicians from across the globe.
We provide robust code and data resources to the pediatric oncology community, encouraging interdisciplinary scientists to collaborate on new analyses in order to accelerate therapeutic translation for children with cancer, goals we are seeing play out in real-time.
To our knowledge, this initiative represents the first large-scale, collaborative, open analysis of genomic data coupled with open manuscript writing, in which we comprehensively analyzed the largest cohort of pediatric brain tumors to date, comprising over 1000 tumors across 59 distinct histologies.
We used available WGS, WXS, and RNA-Seq data to generate high-confidence consensus SNV and CNV calls, prioritize putative oncogenic fusions, and establish over 40 scalable modules to perform common downstream cancer genomics analyses, all of which have undergone rigorous scientific and analytical code review.
We detected and showed expected patterns of genomic lesions, mutational signatures, and aberrantly regulated signaling pathways across multiple pediatric brain tumor histologies.

Molecular subtyping information from pathology reports was largely not collected nor available for this cohort, and if available (e.g.: sparse medulloblastoma subtypes), it required manual curation from pathology reports and/or free text clinical data fields.
Furthermore, in the absence of DNA methylation data, rapid classification to derive molecular subtypes could not be performed.
Thus, to enable biological interrogation of specific tumor subtypes, we created RNA- and DNA-based subtyping modules aligned with WHO molecularly-defined diagnoses.
We worked closely with pathologists and clinicians to build modules from which we determined a research-grade integrated diagnosis for 44% (1260/2840) of biospecimens as well as discovered incorrectly diagnosed or mis-identified samples in the OpenPBTA cohort.

We harnessed RNA expression data for a number of analyses, yielding important biological insights across multiple brain tumor histologies.
For example, we performed subtyping of medulloblastoma tumors, for which only 35% (43/122) had subtype information from pathology reports.
Among the subtyped tumors, we accurately recapitulated subtypes using MM2S (91%; 39/43) or medulloPackage (95%; 41/43) [@doi:10.1186/s13029-016-0053-y; @doi:10.1371/journal.pcbi.1008263].
We then applied the consensus of these methods to subtype all medulloblastoma tumors lacking pathology-based subtypes.

We used an expression classifier to determine whether tumors have dysfunctional _TP53_ [@doi:10.1016/j.celrep.2018.03.076] and the EXTEND algorithm to determine their degree of telomerase activity using a 13-gene signature [@doi:10.1038/s41467-020-20474-9].
We found that hypermutant tumors universally displayed dysregulation of _TP53_ and that EXTEND scores are a robust surrogate measure for telomerase activity, as tumors with high scores had a significant adverse risk for overall survival.
Additionally, high _TP53_ scores were a significant negative prognostic indicator for overall survival for patients with certain tumor types, such as H3 K28-altered DMGs and ependymomas. 
By assessing _TP53_ and telomerase activity prospectively from expression data, information usually only attainable with DNA sequencing and/or qPCR, we can quickly incorporate oncogenic biomarker and prognostic knowledge and expand our biological understanding of these tumors.

We identified enrichment of hallmark cancer pathways and characterized the immune cell landscape across pediatric brain tumors, demonstrating tumors in some histologies, such as schwannomas, craniopharyngiomas, and low-grade gliomas, may have a inflammatory tumor microenvironment.
Of note, we observed upregulation of IFN$\gamma$, IL-1, and IL-6, and TNF$\alpha$ in craniopharyngiomas, tumors difficult to resect due to their anatomical location and critical surrounding structures.
Neurotoxic side effects have been reported when interferon alpha immunotherapy is administered to reduce cystic craniopharyngioma tumor size and/or delay progression [@doi:10.3171/2015.2.PEDS14656; @url:http://www.ijcasereportsandimages.com/archive/2013/012-2013-ijcri/013-12-2013-mohammed/ijcri-013122013113-mohammed-full-text.php].
Thus, additional immune vulnerabilities, such as IL-6 inhibition and immune checkpoint blockade, have recently been proposed as therapies for cystic adamantinomatous craniopharyngiomas [@doi:10.1093/neuonc/noy035; @pmid:34966342; @pmid:32075140; @doi:10.1007/s00401-018-1830-2; @doi:10.3389/fonc.2019.00791] and our results noted above support this approach.
Finally, our study reproduced the overall known poor infiltration of CD8+ T cells and general low expression of _CD274_ (PD-L1) in pediatric brain tumors, further highlighting the urgent need to identify novel therapeutic strategies for these immunologically cold tumors.

OpenPBTA has rapidly become a foundational layer for a number of discovery research and translational projects which will continue to add other genomic modalities and analyses, such as germline, methylation, single cell, epigenomic, mRNA splicing, imaging, and model drug response data.
For example, the RNA fusion filtering module created within OpenPBTA set the stage for development of the R package _annoFuse_ [@doi:10.1186/s12859-020-03922-7] and an R Shiny application [_shinyFuse_](http://shiny.imbei.uni-mainz.de:3838/shinyFuse/).
Using medulloblastoma subtyping and immune deconvolution analyses performed herein, Dang and colleagues showed enrichment of monocyte and microglia-derived macrophages within the SHH subgroup which they suggest may accumulate following radiation therapy [@doi:10.1016/j.celrep.2021.108917].
Expression and copy number analyses were used to demonstrate that _GPC2_ is a highly expressed and copy number gained immunotherapeutic target in ETMRs, medulloblastomas, choroid plexus carcinomas, H3 wildtype high-grade gliomas, as well as DMGs [@doi:10.1101/2021.07.06.451385].
This led Foster and colleagues to subsequently develop a chimeric antigen receptor (CAR) directed against _GPC2_, for which they show preclinical efficacy in mouse models [@doi:10.1101/2021.07.06.451385]. 
Moreover, OpenPBTA has enabled a framework to support real-time integration of subjects as each was enrolled on the PNOC008 high-grade glioma clinical trial [@url:https://pnoc.us/clinical-trial/pnoc008/], allowing researchers and clinicians to link tumor biology to translational impact through clinical decision support during tumor board discussions.
Finally, the OpenPBTA project was recently expanded into a pan-pediatric cancer effort to support the RACE Act (https://github.com/PediatricOpenTargets/OpenPedCan-analysis) as part of the the NCI's Childhood Cancer Data Initiative.
The OpenPBTA project has paved the way for collaborative, open, reproducible, and scalable analyses and we anticipate this foundational work will have limitless impact within the pediatric oncology community, ultimately leading to improved outcomes for children with cancer.

All code and processed data are openly available through GitHub, CAVATICA, and PedcBioPortal (see **STAR METHODS**).


## Acknowledgments

We graciously thank the patients and families who have donated their tumors to the Children's Brain Tumor Network and/or the Pacific Pediatric Neuro-oncology Consortium, without which, this research would not be possible. 
This work was funded through the Alex’s Lemonade Stand Foundation (ALSF) Childhood Cancer Data Lab (JNT, CSG, JAS, CLS, CJB, SJS), ALSF Young Investigator Award (JLR), ALSF Catalyst Award (JLR, ACR, PBS), ALSF Catalyst Award (SJS), ALSF CCDL Postdoctoral Training Grant (SMF), Children’s Hospital of Philadelphia Division of Neurosurgery (PBS and ACR), the Australian Government, Department of Education (APH), and NIH Grants 3P30 CA016520-44S5 (ACR), U2C HL138346-03 (ACR, APH), U24 CA220457-03 (ACR), and K12GM081259 (SMF).
This project has been funded in part with Federal funds from the National Cancer Institute, National Institutes of Health, under Contract No. 75N91019D00024, Task Order No. 75N91020F00003 (JLR, ACR, APH). 
The content of this publication does not necessarily reflect the views or policies of the Department of Health and Human Services, nor does mention of trade names, commercial products or organizations imply endorsement by the U.S. Government.

The authors would like to thank the following collaborators who contributed or supervised analyses present in the analysis repository that were not included in the manuscript: William Amadio, Holly Beale, Ellen Kephart, Alfred Lyle, and Olena Vaske. 
Finally, we would like to thank Yuanchao Zhang for adding to the project codebase and Jessica B. Foster for helpful discussions while drafting the manuscript.

## Author Contributions

|Author|Contributions|
|---|---|
|Joshua A. Shapiro|Methodology, Software, Validation, Formal analysis, Investigation, Writing - Original draft, Writing - Review and editing, Visualization, Supervision|
|Candace L. Savonen|Methodology, Software, Validation, Formal analysis, Investigation, Writing - Original draft, Visualization|
|Chante J. Bethell|Methodology, Validation, Formal analysis, Investigation, Writing - Original draft, Visualization|
|Krutika S. Gaonkar|Data curation, Formal Analysis, Investigation, Methodology, Software, Writing – original draft|
|Run Jin|Data curation, Formal Analysis, Visualization, Writing – original draft|
|Yuankun Zhu|Data curation, Formal Analysis, Investigation, Methodology, Supervision|
|Miguel A. Brown|Data curation, Methodology|
|Nhat Duong|Formal Analysis, Investigation, Methodology|
|Komal S. Rathi|Formal Analysis, Investigation, Methodology, Writing – original draft|
|Nighat Noureen|Formal analysis, Visualization, Writing - Original draft|
|Bo Zhang|Data curation, Formal Analysis|
|Brian M. Ennis|Data curation, Formal Analysis|
|Stephanie J. Spielman|Validation, Formal analysis, Writing - Review and editing, Visualization, Supervision|
|Laura E. Egolf|Formal analysis, Writing - Original draft|
|Yang Yang|Formal analysis, Software|
|Bailey Farrow|Data curation, Software|
|Nicolas Van Kuren|Data curation, Software|
|Meen Chul Kim|Data curation|
|Tejaswi Koganti|Formal Analysis, Investigation|
|Shrivats Kannan|Formal Analysis, Methodology, Writing – original draft|
|Pichai Raman|Conceptualization, Formal Analysis, Methodology|
|Jennifer Mason|Supervision|
|Daniel P. Miller|Formal Analysis|
|Anna R. Poetsch|Formal Analysis|
|Payal Jain|Data curation, Investigation, Validation|
|Adam A. Kraya|Methodology|
|Allison P. Heath|Project administration|
|Mateusz P. Koptyra|Formal Analysis, Writing – original draft|
|Shannon Robbins|Data curation|
|Yiran Guo|Formal Analysis|
|Xiaoyan Huang|Formal Analysis|
|Jessica Wong|Writing – original draft|
|Mariarita Santi|Investigation, Validation|
|Angela Viaene|Investigation, Validation|
|Laura Scolaro|Data Curation|
|Angela Waanders|Supervison|
|Derek Hanson|Validation|
|Steven M. Foltz|Validation|
|Hongbo M. Xie|Methodology, Supervision|
|Siyuan Zheng|Formal analysis, Visualization, Writing - Original draft, Supervision|
|Cassie N. Kline|Supervision|
|Peter J. Madsen|Writing – review & editing|
|Jena V. Lilly|Conceptualization, Funding acquisition, Project administration|
|Philip B. Storm|Conceptualization, Funding acquisition, Resources|
|Adam C. Resnick|Conceptualization, Funding acquisition, Resources, Supervision|
|Casey S. Greene|Conceptualization, Funding acquisition, Methodology, Project administration, Software, Supervision, Writing – review & editing|
|Jo Lynne Rokita*|Conceptualization, Data curation, Formal Analysis, Funding acquisition, Investigation, Methodology, Software, Supervision, Writing – original draft|
|Jaclyn N. Taroni*|Methodology, Software, Validation, Formal analysis, Investigation, Data curation, Writing - Review and editing, Visualization, Supervision, Project administration|
|Children's Brain Tumor Network|Conceptualization|
|Pacific Pediatric Neuro-Oncology Consortium|Conceptualization|


## Declarations of Interest

CSG's spouse was an employee of Alex's Lemonade Stand Foundation, which was a sponsor of this research.
JAS, CLS, CJB, SJS, and JNT are or were employees of Alex's Lemonade Stand Foundation, a sponsor of this research.

## Figure Titles and Legends

**Figure 1. Overview of the OpenPBTA Project.** A, The Children's Brain Tumor Network and the Pacific Pediatric Neuro-Oncology Consortium collected tumor samples from 943 patients. To date, 38 cell lines were created from tumor tissue, and over 2000 specimens were sequenced (N = 1035 RNA-Seq, N = 940 WGS, and N = 32 WXS or Targeted panel). Data was harmonized by the Kids First Data Resource Center using an Amazon S3 framework within CAVATICA. B, Stacked bar plot summary of the number of biospecimens per phase of therapy per broad histology (GNG = ganglioglioma, LGG = low-grade glioma, DIPG = diffuse intrinsic pontine glioma, DMG = diffuse midline glioma, EPN = ependymoma, HGG = high-grade glioma, ATRT = atypical teratoid rhabdoid tumor, CNS-ET = CNS embryonal tumor, MB = medulloblastoma, PNF = plexiform neurofibroma, DNET = dysembryoplastic neuroepithelial tumor, CRANIO = craniopharyngioma, EWS = Ewing sarcoma, CPP = choroid plexus papilloma). C, Overview of the open analysis and manuscript contribution model. In the analysis GitHub repository, a contributor would propose an analysis that other participants can comment on. Contributors would then implement the analysis and file a request to add their changes to the analysis repository ("pull request"). Pull requests underwent review for scientific rigor and correctness of implementation. Pull requests were additionally checked to ensure that all software dependencies were included and the code was not sensitive to underlying data changes using container and continuous integration technologies. Finally, a contributor would file a pull request documenting their methods and results to the Manubot-powered manuscript repository. Pull requests in the manuscript repository were also subject to review. D, A potential path for an analytical pull request. Arrows indicate revisions to a pull request. Prior to review, a pull request was tested for dependency installation and whether or not the code would execute. Pull requests also required approval by organizers and/or other contributors, who checked for scientific correctness. Panel A created with [BioRender.com](biorender.com).

**Figure 2. Mutational landscape of PBTA tumors.** 
Shown are frequencies of canonical somatic gene mutations, CNVs, fusions, and TMB (top bar plot) for the top 20 genes mutated across primary tumors within the OpenPBTA dataset. A, Low-grade astrocytic tumors (N = 227): low-grade glioma astrocytoma (N = 187), ganglioglioma (N = 35), subependymal giant cell astrocytoma (N = 2), diffuse fibrillary astrocytoma (N = 1), pilocytic astrocytoma (N = 1), and pleomorphic xanthoastrocytoma (N = 1); B, Embryonal tumors (N = 128): medulloblastomas (N = 95), atypical teratoid rhabdoid tumors (N = 24), embryonal tumors with multilayer rosettes (N = 2), other CNS embryonal tumors (N = 5), ganglioneuroblastoma (N = 1), and CNS neuroblastoma (N = 1); C, Diffuse astrocytic and oligodendroglial tumors (N = 61): diffuse midline gliomas (N = 34) and non-midline high-grade gliomas (N = 26), oligodendroglioma (N = 1); D, Other CNS tumors (N = 195): ependymomas (N = 60), dysembryoplastic neuroepithelial tumors (N = 19), meningiomas (N = 17), schwannoma (N = 11), neurofibroma plexiform (N = 7). Other CNS tumors with N < 5 are displayed in **Figure {@fig:S3}A**. Patient sex (`germline_sex_estimate`) and tumor histology (`cancer_group`) are displayed as annotations at the bottom of each plot. Only samples with mutations in the listed genes are shown. Multiple CNVs are denoted as a complex event.

**Figure 3. Mutational co-occurrence and signatures highlight key oncogenic drivers.** 
A, Bar plot of occurrence and co-occurrence of nonsynonymous mutations for the 50 most commonly mutated genes across all tumor types (annotated from `cancer_group` if N >= 10 or `Other` if N < 10); B, Co-occurrence and mutual exclusivity of nonsynonymous mutations between genes; The co-occurrence score is defined as $I(-\log_{10}(P))$ where $P$ is defined by Fisher's exact test and $I$ is 1 when mutations co-occur more often than expected and -1 when exclusivity is more common; C, The number of SV breaks significantly correlates with the number of CNV breaks (Adjusted R = 0.436, p = 1.08e-37). D, Chromothripsis frequency across pediatric brain tumors shown by `cancer_group` with N >= 3. E, Sina plots of RefSig signature weights for signatures 1, 3, 8, 11, 18, 19, N6, MMR2, and Other across cancer groups. Box plot lines represent the first quartile, median, and third quartile.

**Figure 4. _TP53_ and telomerase activity** 
A, Receiver Operating Characteristic for _TP53_ classifier run on FPKM of stranded RNA-Seq samples. B, Violin and box plots of _TP53_ scores plotted by _TP53_ alteration type (N<sub>activated</sub> = 27, N<sub>lost</sub> = 140, N<sub>other</sub> = 999). C, Violin and box plots of _TP53_ RNA expression plotted by _TP53_ activation status (N<sub>activated</sub> = 11, N<sub>lost</sub> = 96, N<sub>other</sub> = 862). D, Box plots of _TP53_ and telomerase (EXTEND) scores grouped by `cancer_group`. Mutation status is highlighted in orange (hypermutant) or red (ultra-hypermutant). E, Heatmap of RefSig mutational signatures for patients who have least one tumor or cell line with a TMB >= 10 Mut/Mb. F, Forest plot depicting the prognostic effects of _TP53_ and telomerase scores on overall survival, controlling for extent of tumor resection, LGG group, and HGG group. G, Forest plot depicting the effect of molecular subtype on overall survival of HGGs. For F and G, hazard ratios (HR) with 95% confidence intervals and p-values are listed. Significant p-values are denoted with black diamonds. Reference groups are denoted by grey diamonds. H, Kaplan-Meier curve of HGG tumors by molecular subtype.

**Figure 5. Transcriptomic and immune landscape of pediatric brain tumors** 
A, First two dimensions from UMAP of sample transcriptome data. Points are colored by `broad_histology` of the samples they represent. B, Heatmap of GSVA scores for Hallmark gene sets with significant differences, with samples ordered by `cancer_group`. C, Box plots of quanTIseq estimates of immune cell fractions in histologies with N >= 15. D, Forest plot depicting the effect of immune cell fraction or _CD274_ expression on overall survival of medulloblastomas. Hazard ratio (HR) with 95% confidence intervals and p-values are listed and significant p-values are denoted with colored diamonds. Of note, the upper CI for macrophage M1 HR was too large to plot. E, Box plot of _CD274_ expression (log<sub>2</sub> FPKM) for medulloblastoma samples grouped by molecular subtype. Adjusted p-values are shown.

## Table Titles and Legends

**Table 1. Molecular subtypes generated through the OpenPBTA project.**
Listed are broad tumor histologies, molecular subtypes generated, and number of specimens subtyped within the OpenPBTA project.

**Table 2. Patients with hypermutant tumors.**
Listed are patients with at least one hypermutant or ultra-hypermutant tumor or cell line.
Coding region TMB, phase of therapy, therapeutic interventions, cancer predispositions, and molecular subtypes are included.



## STAR METHODS

### RESOURCE AVAILABILITY

#### Lead contact

Requests for access to OpenPBTA raw data and/or specimens may be directed to, and will be fulfilled by Jo Lynne Rokita (rokita@chop.edu).

#### Materials availability

This study did not create new, unique reagents.

#### Data and code availability

Raw and harmonized WGS, WXS, and RNA-Seq data derived from human samples are available within the KidsFirst Portal [@doi:10.24370/OPENPBTA] upon access request to the CBTN (https://cbtn.org/) as of the date of the publication.
In addition, merged summary files are openly accessible at https://cavatica.sbgenomics.com/u/cavatica/openpbta or via download script in the  https://github.com/AlexsLemonade/OpenPBTA-analysis repository.
Summary data are visible within PedcBioPortal at https://pedcbioportal.kidsfirstdrc.org/study/summary?id=openpbta.
Associated DOIs are listed in the **Key Resources Table**.
<!-- TODO: create data dois -->

All original code was developed within the following repositories and is publicly available as follows. 
Primary data analyses can be found at https://github.com/d3b-center/OpenPBTA-workflows. 
Downstream data analyses can be found at https://github.com/AlexsLemonade/OpenPBTA-analysis.
Manuscript code can be found at https://github.com/AlexsLemonade/OpenPBTA-manuscript.
Associated DOIs are listed in the **Key Resources Table**.
Software versions are documented in **Table S5** as an appendix to the **Key Resources Table**.
<!-- TODO: create code dois -->

Any additional information required to reanalyze the data reported in this paper is available
from the lead contact upon request.

### METHOD DETAILS

#### Biospecimen Collection

The Pediatric Brain Tumor Atlas specimens are comprised of samples from Children's Brain Tumor Network (CBTN) and the Pediatric Pacific Neuro-Oncology Consortium (PNOC).
The [CBTN](https://CBTN.org) is a collaborative, multi-institutional (26 institutions worldwide) research program dedicated to the study of childhood brain tumors.
[PNOC](https://www.pnoc.us/) is an international consortium dedicated to bringing new therapies to children and young adults with brain tumors.
We also include blood and tumor biospecimens from newly-diagnosed diffuse intrinsic pontine glioma (DIPG) patients as part of the PNOC003 clinical trial [PNOC003/NCT02274987](https://clinicaltrials.gov/ct2/show/NCT02274987) [@doi:10.1002/ijc.32258].


The CBTN-generated cell lines were derived from either fresh tumor tissue directly obtained from surgery performed at Children’s Hospital of Philadelphia (CHOP) or from prospectively collected tumor specimens stored in Recover Cell Culture Freezing medium (cat# 12648010, Gibco).
We dissociated tumor tissue using enzymatic method with papain as described [@doi:10.1101/656587].
Briefly, we washed tissue with HBSS (cat# 14175095, Gibco), and we minced and incubated the tissue with activated papain solution (cat# LS003124, SciQuest) for up to 45 minutes.
We used ovomucoid solution (cat# 542000, SciQuest) to inactivate the papain, briefly treated tissue with DNase (cat#  10104159001, Roche), passed it through the 100μm cell strainer (cat# 542000, Greiner Bio-One).
We initiated two cell culture conditions based on the number of cells available.
For cultures utilizing the fetal bovine serum (FBS), we plated a minimum density of 3×10<sup>5</sup> cells/mL in DMEM/F-12 medium (cat# D8062, Sigma) supplemented with 20% FBS (cat# SH30910.03, Hyclone), 1% GlutaMAX (cat# 35050061, Gibco), Penicillin/Streptomycin-Amphotericin B Mixture (cat# 17-745E, Lonza), and 0.2% Normocin (cat# ant-nr-2, Invivogen).
For serum-free media conditions, we plated cells at minimum density of 1×10<sup>6</sup> cells/mL in DMEM/F12 medium supplemented with 1% GlutaMAX, 1X B-27 supplement minus vitamin A (cat# 12587-010, Gibco), 1x N-2 supplement (cat# 17502001, Gibco), 20 ng/ml epidermal growth factor (cat# PHG0311L, Gibco), 20 ng/mL basic fibroblast growth factor (cat# 100-18B, PeproTech), 2.5μg/mL heparin (cat# H3149, Sigma), Penicillin/Streptomycin-Amphotericin B Mixture, and 0.2% Normocin.

#### Nucleic acids extraction and library preparation

##### PNOC samples

The Translational Genomic Research Institute (TGEN; Phoenix, AZ) performed DNA and RNA extractions on tumor biopsies using a DNA/RNA AllPrep Kit (Qiagen, #80204).
All RNA used for library prep had a minimum RIN of seven, but no QC thresholds were implemented for the DNA.
For library preparation, 500 ng of nucleic acids were used as input for RNA-Seq, WXS, and targeted DNA panel (panel) sequencing.
RNA library preparation was performed using the TruSeq RNA Sample Prep Kit (Illumina, #FC-122-1001) and the exome prep was performed using KAPA Library Preparation Kit (Roche, #KK8201) using Agilent's SureSelect Human All Exon V5 backbone with custom probes.
The targeted DNA panel developed by Ashion Analytics (formerly known as the GEM Cancer panel) consisted of exonic probes against 541 cancer genes.
Both panel and WXS assays contained 44,000 probes across evenly spaced genomic loci used for genome-wide copy number analysis.
For the panel, additional probes tiled across intronic regions of 22 known tumor suppressor genes and 22 genes involved in common cancer translocations for structural analysis.
All extractions and library preparations were performed according to manufacturer's instructions.

##### CBTN samples

Blood, tissue, and cell line DNA/RNA extractions were performed at the Biorepository Core at CHOP.
Briefly, 10-20 mg frozen tissue, 0.4-1ml of blood, or 2e6 cells pellet was used for extractions.
Tissues were lysed using a Qiagen TissueLyser II (Qiagen) with 2×30 sec at 18Hz settings using 5 mm steel beads (cat# 69989, Qiagen).
Both tissue and cell pellets processes included a CHCl3 extraction and were run on the QIACube automated platform (Qiagen) using the AllPrep DNA/RNA/miRNA Universal kit (cat# 80224, Qiagen).
Blood was thawed and treated with RNase A (cat#, 19101, Qiagen); 0.4-1ml was processed using the Qiagen QIAsymphony automated platform (Qiagen) using the QIAsymphony DSP DNA Midi Kit (cat# 937255, Qiagen).
DNA and RNA quantity and quality was assessed by PerkinElmer DropletQuant UV-VIS spectrophotometer (PerkinElmer) and an Agilent 4200 TapeStation (Agilent, USA) for RIN and DIN (RNA Integrity Number and DNA Integrity Number, respectively).
The NantHealth Sequencing Center, BGI at CHOP, or the Genomic Clinical Core at Sidra Medical and Research Center performed library preparation and sequencing.
Briefly, DNA sequencing libraries were prepared for tumor and matched-normal DNA using the KAPA HyperPrep kit (cat# 08098107702, Roche), and tumor RNA-Seq libraries were prepared using KAPA Stranded RNA-Seq with RiboErase kit (cat# 07962304001, Roche).

#### Data generation

NantHealth and Sidra performed 2x150 bp WGS on paired tumor (~60X) and constitutive DNA (~30X) samples on an Illumina X/400.
BGI at CHOP performed 2x100 bp WGS sequenced at 60X depth for both tumor and normal samples.
NantHealth performed ribosomal-depleted whole transcriptome stranded RNA-Seq to an average depth of 200M.
BGI at CHOP performed poly-A or ribosomal-depleted whole transcriptome stranded RNA-Seq to an average depth of 100M.
The Translational Genomic Research Institute (TGEN; Phoenix, AZ) performed paired tumor (~200X) and constitutive whole exome sequencing (WXS) or targeted DNA panel (panel) and poly-A selected RNA-Seq (~200M reads) for PNOC tumor samples.
The panel tumor sample was sequenced to 470X, and the normal panel sample was sequenced to 308X.
PNOC 2x100 bp WXS and RNA-Seq libraries were sequenced on an Illumina HiSeq 2500.

#### DNA WGS Alignment

We used BWA-MEM [@arxiv:1303.3997] to align paired-end DNA-seq reads to the version 38 patch release 12 of the _Homo sapiens_ genome reference, obtained as a FASTA file from UCSC (see **Key Resources Table**).
Next, we used the [Broad Institute's Best Practices](https://software.broadinstitute.org/gatk/best-practices/workflow?id=11165) to process Binary Alignment/Map files (BAMs) in preparation for variant discovery.
We marked duplicates using SAMBLASTER [@doi:10/f6kft3], and we merged and sorted BAMs using Sambamba [@doi:10/gfzsfw]
We used the Broad's Genome Analysis Tool Kit [GATK](https://software.broadinstitute.org/gatk/) v4.0.3.0, specifically the BaseRecalibrator submodule, to process BAMs. 
Lastly, for normal/germline input, we ran the GATK HaplotypeCaller [@doi:10.1101/201178] submodule on the recalibrated BAM to generate a genomic variant call format (GVCF) file.
This file is used as the basis for germline calling, described in the **SNV calling for B-allele Frequency (BAF) generation** section.

We obtained references from the [Broad Genome References on AWS](https://s3.console.aws.amazon.com/s3/buckets/broad-references/hg38/v0/) bucket with a general description of references at <https://s3.amazonaws.com/broad-references/broad-references-readme.html>.

#### Quality Control of Sequencing Data

To confirm sample matches and remove mis-matched samples from the dataset, we performed NGSCheckMate [@doi:10.1093/nar/gkx193] on matched tumor/normal CRAM files. 
Briefly, we processed CRAMs using BCFtools to filter and call 20k common single nucleotide polymorphisms (SNPs) using default parameters. 
We used the resulting VCFs to run NGSCheckMate following the `bcf_call.cwl` workflow found in the [D3b GitHub repository](https://github.com/d3b-center/OpenPBTA-workflows).
Per NGSCheckMate author recommendations, we used <= 0.61 as a correlation coefficient cutoff at sequencing depths > 10 to predict mis-matched samples.
For RNA-Seq, we determined read strandedness by running the [`infer_experiment.py` script](http://rseqc.sourceforge.net/#infer-experiment-py) on the first 200k mapped reads.
We removed any samples whose calculated strandedness did not match strandedness information provided by the sequencing center.
We required that at least 60% of RNA-Seq reads mapped to the human reference for samples to be included in analysis. 

#### Germline Variant Calling

##### SNP calling for B-allele Frequency (BAF) generation

We performed germline haplotype calls using the [GATK Joint Genotyping Workflow](https://software.broadinstitute.org/gatk/best-practices/workflow?id=11145) on individual GVCFs from the normal sample alignment workflow.
Using only SNPs, we applied the [GATK generic hard filter suggestions](https://gatkforums.broadinstitute.org/gatk/discussion/2806/howto-apply-hard-filters-to-a-call-set) to the VCF, with an additional requirement of 10 reads minimum depth per SNP.
We used the filtered VCF as input to Control-FREEC and CNVkit (below) to generate B-allele frequency (BAF) files. 
Note that GATK v4.0.12.0 was used for all of these steps except VariantFiltration, which used v3.8.0 because this tool was beta in GATK v4.0.12.0 and known to be unreliable for this analysis.
This single-sample workflow is available in the [D3b GitHub repository](https://github.com/d3b-center/OpenPBTA-workflows).
References can be obtained from the [Broad Genome References on AWS](https://s3.console.aws.amazon.com/s3/buckets/broad-references/hg38/v0/) bucket, and a general description of references can be found at https://s3.amazonaws.com/broad-references/broad-references-readme.html.

#### Somatic Mutation Calling

##### SNV and indel calling

For PBTA samples, we used four variant callers to call SNVs and indels from panel, WXS, and WGS data: Strelka2 [@doi:10.1038/s41592-018-0051-x], Mutect2 [@doi:10.1101/861054], Lancet [@doi:10.1038/s42003-018-0023-9], and VarDict [@doi:10.1093/nar/gkw227].
VarDict-only calls were not retained since ~ 39M calls with low VAF were uniquely called and may be potential false positives.
We used only Strelka2, Mutect2 and Lancet to analyze WXS samples from TCGA.
TCGA samples were captured using various WXS target capture kits and we downloaded the BED files from the [`GDC portal`](https://api.gdc.cancer.gov/files).
The manufacturers provided the input interval BED files for both panel and WXS data for PBTA samples.
We padded all panel and WXS BED files were by 100 bp on each side for Strelka2, Mutect2, and VarDict runs and by 400 bp for the Lancet run.  
For WGS calling, we utilized the non-padded BROAD Institute interval calling list [`wgs_calling_regions.hg38.interval_list`](https://console.cloud.google.com/storage/browser/_details/genomics-public-data/resources/broad/hg38/v0/wgs_calling_regions.hg38.interval_list), comprised of the full genome minus N bases, unless otherwise noted below.
We ran Strelka2 [@doi:10/gdwrp4] using default parameters for canonical chromosomes (chr1-22, X,Y,M), as recommended by the authors, and we filtered the final Strelka2 VCF for PASS variants.
We ran Mutect2 from GATK v4.1.1.0 according to Broad best practices outlined from their [Workflow Description Language (WDL)](https://github.com/broadinstitute/gatk/blob/4.1.1.0/scripts/mutect2_wdl/mutect2.wdl), and we filtered the final Mutect2 VCF for PASS variants.
To manage memory issues, we ran VarDictJava [@doi:10.1093/nar/gkw227] using 20 Kb interval chunks of the input BED, padded by 100 bp on each side, such that if an indel occurred in between intervals, it would be captured.
Parameters and filtering followed [BCBIO standards](https://github.com/bcbio/bcbio-nextgen) except that variants with a variant allele frequency (VAF) >= 0.05 (instead of >= 0.10) were retained.
The 0.05 VAF increased the true positive rate for indels and decreased the false positive rate for SNVs when using VarDict in consensus calling.
We filtered the final VarDict VCF for PASS variants with `TYPE=StronglySomatic`.
We ran Lancet using default parameters, except for those noted below.
For input intervals to Lancet WGS, we created a reference BED from only the UTR, exome, and start/stop codon features of the GENCODE 31 reference, augmented as recommended with PASS variant calls from Strelka2 and Mutect2 [@doi:10.1101/623702].
We then padded these intervals by 300 bp on each side during Lancet variant calling.
Per recommendations by the New York Genome Center [@doi:10.1101/623702] for WGS samples, we augmented the Lancet input intervals described above with PASS variant calls from Strelka2 and Mutect2 as validation.

##### VCF annotation and MAF creation

We normalized INDELs with `bcftools norm` on all PASS VCFs using the `kfdrc_annot_vcf_sub_wf.cwl` subworkflow, release v3 (See **Table S5**).
The ENSEMBL Variant Effect Predictor [@doi:10/gdz75c], reference release 93, was used to annotate variants and bcftools was used to add population allele frequency (AF) from gnomAD [@doi:10.1038/s41586-020-2308-7].
We annotated SNV and INDEL hotspots from v2 of Memorial Sloan Kettering Cancer Center's (MSKCC) database (See **Key Resources Table**) as well as the TERT promoter mutations C228T and C250T [@doi:10.3390/ijms21176034].
We annotated SNVs by matching amino acid position (`Protein_position` column in MAF file) with SNVs in the MSKCC database, we matched splice sites to `HGVSp_Short` values in the MSKCC database, and we matched INDELs based on amino acid present within the range of INDEL hotspots values in the MSKCC database.
We removed non-hotspot annotated variants with a normal depth less than or equal to 7 and/or gnomAD allele frequency (AF) greater than 0.001 as potential germline variants.
We matched TERT promoter mutations using hg38 coordinates as indicated in ref. [@doi:10.3390/ijms21176034]: C228T occurs at 5:1295113 is annotated as existing variant `s1242535815`, `COSM1716563`, or `COSM1716558`, and is 66 bp away from the TSS; C250T occurs at Chr5:1295135, is annotated as existing variant `COSM1716559`, and is 88 bp away from the TSS.
We retained variants annotated as PASS or HotSpotAllele=1 in the final set, and we created MAFs using MSKCC's [`vcf2maf`](https://github.com/mskcc/vcf2maf) tool.


##### Gather SNV and INDEL Hotspots
We retained all variant calls from Strelka2, Mutect2, or Lancet that overlapped with an SNV or INDEL hotspot in a hotspot-specific MAF file, which we then used for select analyses as described below.

##### Consensus SNV Calling

Our SNV calling process led to separate sets of predicted mutations for each caller.
We considered mutations to describe the same change if they were identical for the following MAF fields: `Chromosome`, `Start_Position`, `Reference_Allele`,  `Allele`, and `Tumor_Sample_Barcode`.
Strelka2 does not call multinucleotide variants (MNV), but instead calls each component SNV as a separate mutation, so we separated MNV calls from Mutect2 and Lancet into consecutive SNVs before comparing them to Strelka2 calls.
We examined VAFs produced by each caller and compared their overlap with each other (**Figure S2**).
VarDict calls included many variants that were not identified by other callers (**Figure S2C**), while the other callers produced results that were relatively consistent with one another.
Many of these VarDict-specific calls were variants with low allele frequency (**Figure S2B**).
We therefore derived consensus mutation calls as those shared among the other three callers (Strelka2, Mutect2, and Lancet), and we did not further consider VarDict calls due to concerns it called a large number of false positives.
This decision had minimal impact on results because VarDict also identified nearly every mutation that the other three callers identified, in addition to many unique mutations.

#### Somatic Copy Number Variant Calling (WGS samples only)

We used Control-FREEC [@doi:10/ckt4vz; @doi:10/c6bcps] and CNVkit [@doi:10.1371/journal.pcbi.1004873] for copy number variant calls.
For both algorithms, the `germline_sex_estimate` (described below) was used as input for sample sex and germline variant calls (above) were used as input for BAF estimation.
Control-FREEC was run on human genome reference hg38 using the optional parameters of a 0.05 coefficient of variation, ploidy choice of 2-4, and BAF adjustment for tumor-normal pairs.
Theta2 [@doi:10.1093/bioinformatics/btu651] used VarDict germline and somatic calls, filtered on PASS and strongly somatic, to infer tumor purity.
Theta2 purity was added as an optional parameter to CNVkit to adjust copy number calls.
CNVkit was run on human genome reference hg38 using the optional parameters of Theta2 purity and BAF adjustment for tumor-normal pairs.
We used GISTIC [@doi:10.1186/gb-2011-12-4-r41] on the CNVkit and the consensus CNV segmentation files to generate gene-level copy number abundance (Log R Ratio) as well as chromosomal arm copy number alterations using the parameters specified in the (`run-gistic` analysis module in the OpenPBTA Analysis repository).

##### Consensus CNV Calling

For each caller and sample, we called CNVs based on consensus among Control-FREEC [@doi:10/ckt4vz; @doi:10/c6bcps], CNVkit [@doi:10.1371/journal.pcbi.1004873], and Manta [@doi:10/gf3ggb].
We specifically included CNVs called significant by Control-FREEC (p-value < 0.01) and Manta calls that passed all filters in consensus calling.
We removed sample and consensus caller files with more than 2,500 CNVs because we expected these to be noisy and derive poor quality samples based on cutoffs used in GISTIC [@doi:10.1186/gb-2011-12-4-r41]. 
For each sample, we included the regions in the final consensus set: 1) regions with reciprocal overlap of 50% or more between at least two of the callers; 2) smaller CNV regions in which more than 90% of regions are covered by another caller. 
We did not include any copy number alteration called by a single algorithm in the consensus file. 
We defined copy number as `NA` for any regions that had a neutral call for the samples included in the consensus file.
We merged CNV regions within 10,000 bp of each other with the same direction of gain or loss into single region.
We filtered out any CNVs that overlapped 50% or more with immunoglobulin, telomeric, centromeric, segment duplicated regions, or that were shorter than 3000 bp. 

#### Somatic Structural Variant Calling (WGS samples only)

We used Manta [@doi:10/gf3ggb] for structural variant (SV) calls, and we limited to regions used in Strelka2.
The hg38 reference for SV calling used was limited to canonical chromosome regions.
The somatic DNA workflows for SNV, indel, copy number, and SV calling can be found in the [OpenPBTA workflows Github repository](https://github.com/d3b-center/OpenPBTA-workflows/).
We used [AnnotSV v2.1](https://lbgi.fr/AnnotSV/) [@doi:10.1093/bioinformatics/bty304] to annotate Manta output.
This associated workflow is available in the [workflows GitHub repository](https://github.com/d3b-center/OpenPBTA-workflows/).

#### Gene Expression

##### Abundance Estimation


We used STAR [@doi:10/f4h523] to align paired-end RNA-seq reads, and we used the associated alignment for all subsequent RNA analysis.
We used Ensembl GENCODE 27 "Comprehensive gene annotation" (see **Key Resources Table**) as a reference. 
We used RSEM [@doi:10/cwg8n5] for both FPKM and TPM transcript- and gene-level quantification. 

##### Gene Expression Matrices with Unique HUGO Symbols

To enable downstream analyses, we next identified gene symbols that map to multiple Ensembl gene identifiers (in GENCODE v27, 212 gene symbols map to 1866 Ensembl gene identifiers), known as multi-mapped gene symbols, and ensured unique mappings (`collapse-rnaseq` analysis module in the OpenPBTA Analysis repository).
To this end, we first removed genes with no expression from the RSEM abundance data by requiring an FPKM > 0 in at least 1 sample across the PBTA cohort.
We computed the mean FPKM across all samples per gene. 
For each multi-mapped gene symbol, we chose the Ensembl identifier corresponding to the maximum mean FPKM, using the assumption that the gene identifier with the highest expression best represented the expression of the gene.
After collapsing gene identifiers, 46,400 uniquely-expressed genes remained in the poly-A dataset, and 53,011 uniquely-expressed genes remained in the stranded dataset. 

##### Gene fusion detection

We set up Arriba [@doi:10.1101/gr.257246.119] and STAR-Fusion [@doi:10.1101/120295] fusion detection tools using CWL on CAVATICA.
For both of these tools, we used aligned BAM and chimeric SAM files from STAR as inputs and `GRCh38_gencode_v27` GTF for gene annotation.
We ran STAR-Fusion with default parameters and annotated all fusion calls with the `GRCh38_v27_CTAT_lib_Feb092018.plug-n-play.tar.gz` file from the STAR-Fusion release.
For Arriba, we used a blacklist file `blacklist_hg38_GRCh38_2018-11-04.tsv.gz` from the Arriba release to remove recurrent fusion artifacts and transcripts present in healthy tissue.
We provided Arriba with strandedness information for stranded samples, or we set it to auto-detection for poly-A samples.
We used [FusionAnnotator](https://github.com/FusionAnnotator/FusionAnnotator) on Arriba fusion calls to harmonize annotations with those of STAR-Fusion.
The RNA expression and fusion workflows can be found in the [D3b GitHub repository](https://github.com/d3b-center/OpenPBTA-workflows/blob/master/cwl/kfdrc_RNAseq_workflow.cwl).
The FusionAnnotator workflow can also be found in the [D3b GitHub repository](https://github.com/d3b-center/FusionAnnotator).

### QUANTIFICATION AND STATISTICAL ANALYSIS

##### Recurrently mutated genes and co-occurrence of gene mutations (`interation-plots` analysis module)

Using the consensus SNV calls, we identified genes that were recurrently mutated in the OpenPBTA cohort, including nonsynonymous mutations with a VAF > 5% among the set of independent samples.
We used the ENSEMBL Variant Effect Predictor [@doi:10/gdz75c] annotations, including "High" and "Moderate" consequence types as defined in the R package Maftools [@doi:10.1101/gr.239244.118], to determine the set of nonsynonymous mutations.
For each gene, we then tallied the number of samples that had at least one nonsynonymous mutation.

For genes that contained nonsynonymous mutations in multiple samples, we calculated pairwise mutation co-occurrence scores.
This score was defined as $I(-\log_{10}(P))$ where $I$ is 1 when the odds ratio is > 1 (indicating co-occurrence), and -1 when the odds ratio is < 1 (indicating mutual exclusivity), with $P$ defined by Fisher's Exact Test.

##### Focal Copy Number Calling (`focal-cn-file-preparation` analysis module)

We added the ploidy inferred via Control-FREEC to the consensus CNV segmentation file and used the ploidy and copy number values to define gain and loss values broadly at the chromosome level.
We used `bedtools coverage` [@doi:10.1093/bioinformatics/btq033] to add cytoband status using the UCSC cytoband file [@doi:10.1093/nar/gks1048] (See **Key Resources Table**).
The output status call fractions, which are values of the loss, gain, and callable fractions of each cytoband region, were used to define dominant status at the cytoband-level.
We calculated the weighted means of each status call fraction using band length.
We used the weighted means to define the dominant status at the chromosome arm-level.

A status was considered dominant if more than half of the region was callable and the status call fraction was greater than 0.9 for that region.
We adopted this 0.9 threshold to ensure that the dominant status fraction call was greater than the remaining status fraction calls in a region.

We aimed to define focal copy number units to avoid calling adjacent genes in the same cytoband or arm as copy number losses or gains where it would be more appropriate to call the broader region a loss or gain.
To determine the most focal units, we first considered the dominant status calls at the chromosome arm-level.
If the chromosome arm dominant status was callable but not clearly defined as a gain or loss, we instead included the cytoband-level status call.
Similarly, if a cytoband dominant status call was callable but not clearly defined as a gain or loss, we instead included gene-level status call.
To obtain the gene-level data, we used the `mergeByOverlaps` [function](https://www.rdocumentation.org/packages/IRanges/versions/2.6.1/topics/findOverlaps-methods) from the IRanges package in R [@doi:10.1371/journal.pcbi.1003118] to find overlaps between the segments in the consensus CNV file and the exons in the GENCODE v27 annotation file (See **Key Resources Table**)	.
If the copy number value was 0, we set the status to "deep deletion". 
For autosomes only, we set the status to "amplification" when the copy number value was greater than two times the ploidy value.

#### Chromothripsis Analysis (WGS samples only; `chromothripsis` analysis module)

Considering only chromosomes 1-22 and X, we identified candidate chromothripsis regions in the set of independent tumor WGS samples with ShatterSeek [@doi:10.1038/s41588-019-0576-7], using Manta SV calls that passed all filters and consensus CNV calls. 
We modified the consensus CNV data to fit ShatterSeek input requirements as follows: CNV-neutral or excluded regions (both annotated as `NA` in the consensus data) were filled in with the respective sample’s ploidy value from Control-FREEC, and consecutive segments with the same copy number value were merged. 
We classified candidate chromothripsis regions as high- or low-confidence using the statistical criteria described by the ShatterSeek authors.

##### Immune Profiling and Deconvolution (`immune-deconv` analysis module)

We used the R package `immunedeconv` [@pmid:31510660] with the method `quanTIseq` [@doi:10.1186/s13073-019-0638-6] to deconvolute various immune cell types across tumors from the PBTA cohort in the stranded and poly-A collapsed FPKM RNA-seq datasets ([`immune-deconv` analysis module](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/immune-deconv)). 
The `quanTIseq` deconvolution method directly estimates absolute fractions of 10 immune cell types that represent inferred proportions of the cell types in the mixture. 
Therefore, we utilized `quanTIseq` for inter-sample, intra-sample, and inter-histology score comparisons.

##### Gene Set Variation Analysis (`gene-set-enrichment-analysis` analysis module)

We performed Gene Set Variation Analysis (GSVA) [@doi:10.1186/1471-2105-14-7] on collapsed, log2-transformed RSEM FPKM data using the GSVA Bioconductor package [@doi:10.18129/B9.bioc.GSVA].
We specified the parameter `mx.diff=TRUE` to obtain Gaussian-distributed scores for each of the MSigDB hallmark gene sets [@doi:10.1016/j.cels.2015.12.004].
We compared GSVA scores among histology groups using ANOVA and subsequent Tukey tests; p-values were Bonferroni-corrected for multiple hypothesis testing.	

##### Transcriptomic Dimension Reduction (`transcriptomic-dimension-reduction` analysis module)

We applied Uniform Manifold Approximation and Projection (UMAP) [@arxiv:1802.03426v2] to log2-transformed FPKM data using the umap R package (See **Key Resources Table**).
We set the number of neighbors to 15.

##### Fusion prioritization (`fusion_filtering` analysis module)

We performed artifact filtering and additional annotation on fusion calls to prioritize putative oncogenic fusions.
Briefly, we considered all in-frame and frameshift fusion calls with at least one junction read and at least one gene partner expressed (TPM > 1) to be true calls.
If a fusion call had a large number of spanning fragment reads compared to junction reads (spanning fragment minus junction read greater than ten), we removed these calls as potential false positives.
We prioritized a union of fusion calls as true calls if the fused genes were detected by both callers, the same fusion was recurrent within a broad histology grouping (> 2 samples), or the fusion was specific to the given broad histology.
If either 5' or 3' genes fused to more than five different genes within a sample, we removed these calls as potential false positives.
We annotated putative driver fusions and prioritized fusions based on partners containing known [kinases](http://kinase.com/human/kinome/tables/Kincat_Hsap.08.02.xls), [oncogenes](http://www.bushmanlab.org/assets/doc/allOnco_Feb2017.tsv), [tumor suppressors](https://bioinfo.uth.edu/TSGene/Human_TSGs.txt?csrt=5027697123997809089), curated transcription factors [@doi:10.1016/j.cell.2018.01.029], [COSMIC genes](https://cancer.sanger.ac.uk/census), and/or known [TCGA fusions](https://tumorfusions.org/PanCanFusV2/downloads/pancanfus.txt.gz) from curated references.
Based on pediatric cancer literature review, we added _MYBL1_ [@doi:10.1073/pnas.1300252110], _SNCAIP_ [@doi:10.1038/nature11327], _FOXR2_ [@doi:10.1016/j.cell.2016.01.015], _TTYH1_ [@doi:10.1038/ng.2849], and _TERT_ [@doi:10.1038/ng.3438; @doi:10.1002/gcc.22110; @doi:10.1016/j.canlet.2014.11.057; @doi:10.1007/s11910-017-0722-5] to the oncogene list, and we added _BCOR_ [@doi:10.1016/j.cell.2016.01.015] and _QKI_ [@doi:10.1038/ng.3500] to the tumor suppressor gene list.


#### Oncoprint figure generation (`oncoprint-landscape` analysis module)

We used Maftools [@doi:10.1101/gr.239244.118] to generate oncoprints depicting the frequencies of canonical somatic gene mutations, CNVs, and fusions for the top 20 genes mutated across primary tumors within broad histologies of the OpenPBTA dataset.
We collated canonical genes from the literature for low-grade astrocytic tumors [@doi:10.1186/s40478-020-00902-z], embryonal tumors [@doi:10.1038/nature22973; @doi:10.1007/s00401-020-02182-2; @doi:10.1186/s40478-020-00984-9; @doi:10.1016/j.ccell.2016.02.001; @doi:10.1038/s41598-020-59812-8], diffuse astrocytic and oligodendroglial tumors [@doi:10.1016/j.ccell.2017.08.017; @doi:10.1002/ijc.32258; @doi:10.1093/neuonc/noab106; @doi:10.1186/s40478-020-00905-w],  and other tumors: ependymal tumors, craniopharyngiomas, neuronal-glial mixed tumors, histiocytic tumors, chordoma, meningioma, and choroid plexus tumors [@doi:10.1007/s00381-017-3481-3; @doi:10.1016/j.ccell.2015.04.002; @doi:10.1038/nature13109; @doi:10.1038/s41525-017-0014-7; @doi:10.3171/2019.8.JNS191266; @doi:10.1007/s00401-016-1539-z; @doi:10.1093/neuonc/noaa267; @doi:10.1016/s0002-9440(10)64477-x; @doi:10.1016/j.jaad.2017.05.059; @doi:10.1186/s40478-020-01056-8].

#### Mutational Signatures (`mutational-signatures` analysis module)

We obtained weights (i.e., exposures) for signature sets using the `deconstructSigs` R package function `whichSignatures()` [@doi:10.1186/s13059-016-0893-4] from consensus SNVs with the BSgenome.Hsapiens.UCSC.hg38 annotations (see **Key Resources Table**).
Specifically, we estimated signature weights across samples for eight signatures previously identified in the Signal reference set of signatures ("RefSig") as associated with adult central nervous system (CNS) tumors [@doi:10.1038/s43018-020-0027-5].
These eight RefSig signatures are 1, 3, 8, 11, 18, 19, N6, and MMR2.
Weights for signatures fall in the range zero to one inclusive.
`deconstructSigs` estimates the weights for each signature across samples and allows for a proportion of unassigned weights referred to as "Other" in the text.
These results do not include signatures with small contributions; `deconstructSigs` drops signature weights that are less than 6% [@doi:10.1186/s13059-016-0893-4].

### Tumor Mutation Burden (`snv-callers` analysis module)

We consider tumor mutation burden (TMB) to be the number of consensus SNVs per effectively surveyed base of the genome.
We considered base pairs to be effectively surveyed if they were in the intersection of the genomic ranges considered by the callers used to generate the consensus and where appropriate, regions of interest, such as coding sequences.
We calculated TMB as:
$$\textrm{TMB} = \frac{\textrm{# of coding sequence SNVs}}{\textrm{Size in Mb of }\textit{effectively surveyed} \textrm{ genome} }
$$

We used the total number coding sequence consensus SNVs for the numerator and the size of the intersection of the regions considered by Strelka2 and Mutect2 with coding regions (CDS from GENCODE v27 annotation, see **Key Resources Table**) as the denominator.

#### Clinical Data Harmonization

##### WHO Classification of Disease Types

**Table S1** contains a README, along with sample technical, clinical, and additional metadata used for this study.

##### Molecular Subtyping

We performed molecular subtyping on tumors in the OpenPBTA to the extent possible.
The `molecular_subtype` field in `pbta-histologies.tsv` contains molecular subtypes for tumor types selected from `pathology_diagnosis` and `pathology_free_text_diagnosis` fields as described below, following World Health Organization 2016 classification criteria [@doi:10.1007/s00401-016-1545-1].

Medulloblastoma (MB) subtypes SHH, WNT, Group 3, and Group 4 were predicted using the consensus of two RNA expression classifiers: Medulloblastoma Classifier [@doi:10.1371/journal.pcbi.1008263] and MM2S Classifier [@doi:10.1186/s13029-016-0053-y] on the RSEM FPKM data (`molecular-subtyping-MB` analysis module).

High-grade glioma (HGG) subtypes were derived (`molecular-subtyping-HGG` analysis module) using the following criteria:

1. If any sample contained an _H3F3A_ p.K28M, _HIST1H3B_ p.K28M, _HIST1H3C_ p.K28M, or _HIST2H3C_ p.K28M mutation and no _BRAF_ p.V600E mutation, it was subtyped as `DMG, H3K28`.
2. If any sample contained an _HIST1H3B_ p.K28M, _HIST1H3C_ p.K28M, or _HIST2H3C_ p.K28M mutation and a _BRAF_ p.V600E mutation, it was subtyped as `DMG, H3 K28, BRAF V600E`.
3. If any sample contained an _H3F3A_ p.G35V or p.G35R mutation, it was subtyped as`HGG, H3 G35`.
4. If any high-grade glioma sample contained an _IDH1_ p.R132 mutation, it was subtyped as `HGG, IDH`.
5. If a sample was initially classified as HGG, had no defining histone mutations, and a BRAF p.V600E mutation, it was subtyped as `BRAF V600E`.
6. All other high-grade glioma samples that did not meet any of these criteria were subtyped as `HGG, H3 wildtype`.

Embryonal tumors were included in non-MB and non-ATRT embryonal tumor subtyping (`molecular-subtyping-embryonal` analysis module) if they met any of the following criteria:
1. A _TTYH1_ (5' partner) fusion was detected.
2. A _MN1_ (5' partner) fusion was detected, with the exception of `MN1--PATZ1` since it is an entity separate of CNS HGNET-MN1 tumors [@doi:10.1111/nan.12626].
3. Pathology diagnoses included "Supratentorial or Spinal Cord PNET" or "Embryonal Tumor with Multilayered Rosettes".
4. A pathology diagnosis of "Neuroblastoma", where the tumor was not indicated to be peripheral or metastatic and was located in the CNS.
5. Any sample with "embryonal tumor with multilayer rosettes, ros (who grade iv)", "embryonal tumor, nos, congenital type", "ependymoblastoma" or "medulloepithelioma" in pathology free text.

Non-MB and non-ATRT embryonal tumors identified with the above criteria were further subtyped (`molecular-subtyping-embryonal` analysis module) using the criteria below [@pmid:30249036; @doi:10.1007/s00381-017-3551-6; @url:https://www.cancer.gov/types/brain/hp/child-cns-embryonal-treatment-pdq; @doi:10.3390/ijms21051818].

1. Any RNA-seq biospecimen with _LIN28A_ overexpression, plus a _TYH1_ fusion (5' partner) with a gene adjacent or within the C19MC miRNA cluster and/or copy number amplification of the C19MC region was subtyped as `ETMR, C19MC-altered` (Embryonal tumor with multilayer rosettes, chromosome 19 miRNA cluster altered) [@doi:10.1007/s00401-012-1068-3; 10.1038/ng.2849].
2. Any RNA-seq biospecimen with _LIN28A_ overexpression, a _TTYH1_ fusion (5' partner) with a gene adjacent or within the C19MC miRNA cluster but no evidence of copy number amplification of the C19MC region was subtyped as `ETMR, NOS` (Embryonal tumor with multilayer rosettes, not otherwise specified) [@doi:10.1007/s00401-012-1068-3; @doi:10.1038/ng.2849].
3. Any RNA-seq biospecimen with a fusion having a 5' _MN1_ and 3' _BEND2_ or _CXXC5_ partner were subtyped as `CNS HGNET-MN1` [Central nervous system (CNS) high-grade neuroepithelial tumor with _MN1_ alteration].
4. Non-MB and non-ATRT embryonal tumors with internal tandem duplication (as defined in [@doi:10.1186/s12859-016-1031-8]) of _BCOR_ were subtyped as `CNS HGNET-BCOR` (CNS high-grade neuroepithelial tumor with _BCOR_ alteration).
5. Non-MB and non-ATRT embryonal tumors with over-expression and/or gene fusions in _FOXR2_ were subtyped as `CNS NB-FOXR2` (CNS neuroblastoma with _FOXR2_ activation).
6. Non-MB and non-ATRT embryonal tumors with _CIC-NUTM1_ or other _CIC_ fusions, were subtyped as `CNS EFT-CIC` (CNS Ewing sarcoma family tumor with _CIC_ alteration) [@doi:10.1016/j.cell.2016.01.015]
7. Non-MB and non-ATRT embryonal tumors that did not fit any of the above categories were subtyped as `CNS Embryonal, NOS` (CNS Embryonal tumor, not otherwise specified).

Neurocytoma subtypes central neurocytoma (CNC) and extraventricular neurocytoma (EVN) were assigned (`molecular-subtyping-neurocytoma` analysis module) based on the primary site of the tumor [@doi:10.1007/978-3-319-33432-5_20].
If the tumor's primary site was "ventricles," we assigned the subtype as CNC; otherwise, we assigned the subtype as EVN.

Craniopharyngiomas (CRANIO) were subtyped (`molecular-subtyping-CRANIO` analysis module) into adamantinomatous (`CRANIO, ADAM`), papillary (`CRANIO, PAP`) or undetermined (`CRANIO, To be classified`) based on the following criteria [@doi:10.3171/jns.1995.83.2.0206; @doi:10.3171/jns.1998.89.4.0547]:
1. Craniopharyngiomas from patients over 40 years old with a _BRAF_ p.V600E mutation were subtyped as `CRANIO, PAP`.
2. Craniopharyngiomas from patients younger than 40 years old with mutations in exon 3 of _CTNNB1_ were subtyped as `CRANIO, ADAM`.
3. Craniopharyngiomas that did not fall into the above two categories were subtyped as `CRANIO, To be classified`. 

A molecular subtype of `EWS` was assigned to any tumor with a _EWSR1_ fusion or with a `pathology_diagnosis` of `Ewings Sarcoma` (`molecular-subtyping-EWS` analysis module).

Low-grade gliomas (LGG) or glialneuronal tumors (GNT) were subtyped (`molecular-subtyping-LGAT` analysis module). based on SNV, fusion and CNV status based on @doi:10.1016/j.ccell.2020.03.011, and as described below.
1. If a sample contained a _NF1_ somatic mutation, either nonsense or missense, it was subtyped as `LGG, NF1-somatic`.
2. If a sample contained _NF1_ germline mutation, as indicated by a patient having the neurofibromatosis  cancer predisposition, it was subtyped as `LGG, NF1-germline`.
3. If a sample contained the _IDH_ p.R132 mutation, it was subtyped as `LGG, IDH`.
4. If a sample contained a histone p.K28M mutation in either _H3F3A_, _H3F3B_, _HIST1H3B_, _HIST1H3C_, or _HIST2H3C_, or if it contained a p.G35R or p.G35V mutation in _H3F3A_, it was subtyped as `LGG, H3`.
5. If a sample contained _BRAF_ p.V600E or any other non-canonical _BRAF_ mutations in the kinase (PK_Tyr_Ser-Thr) domain PF07714 (see **Key Resources Table**), it was subtyped as `LGG, BRAF V600E`.
6. If a sample contained `KIAA1549--BRAF` fusion, it was subtyped as `LGG, KIAA1549-BRAF`.
7. If a sample contained SNV or indel in either _KRAS_, _NRAS_, _HRAS_, _MAP2K1_, _MAP2K2_, _MAP2K1_, _ARAF_, _RAF1_, or non-kinase domain of _BRAF_, or if it contained _RAF1_ fusion, or _BRAF_ fusion that was not `KIAA1549--BRAF`, it was subtyped as `LGG, other MAPK`.
8. If a sample contained SNV in either _MET_, _KIT_ or _PDGFRA_, or if it contained fusion in _ALK_, _ROS1_, _NTRK1_, _NTRK2_, _NTRK3_ or _PDGFRA_, it was subtyped as `LGG, RTK`.
9. If a sample contained _FGFR1_ p.N546K, p.K656E, p.N577, or p. K687 hotspot mutations, or tyrosine kinase domain tandem duplication (See **Key Resources Table**), or _FGFR1_ or _FGFR2_ fusions, it was subtyped as `LGG, FGFR`.
10. If a sample contained _MYB_ or _MYBL1_ fusion, it was subtyped as `LGG, MYB/MYBL1`.
11. If a sample contained focal CDKN2A and/or CDKN2B  deletion, it was subtyped as `LGG, CDKN2A/B`.

For LGG tumors that did not have any of the above molecular alterations, if both RNA and DNA samples were available, it was subtyped as `LGG, wildtype`. 
Otherwise, if either RNA or DNA sample was unavailable, it was subtyped as `LGG, To be classified`.

If pathology diagnosis was `Subependymal Giant Cell Astrocytoma (SEGA)`, the `LGG` portion of molecular subtype was recoded to `SEGA`.

Lastly, for all LGG- and GNT- subtyped samples, if the tumors were glialneuronal in origin, based on `pathology_free_text_diagnosis` entries of `desmoplastic infantile`,`desmoplastic infantile ganglioglioma`, `desmoplastic infantile astrocytoma` or `glioneuronal`, each was recoded as follows:
If pathology diagnosis is `Low-grade glioma/astrocytoma (WHO grade I/II)` or `Ganglioglioma`, the `LGG` portion of the molecular subtype was recoded to `GNT`.

Ependymomas (EPN) were subtyped (`molecular-subtyping-EPN` analysis module) into `EPN, ST RELA`, `EPN, ST YAP1`, `EPN, PF A` and `EPN, PF B` based on evidence for these molecular subgroups as described in Pajtler et al. [@doi:10.1016/j.ccell.2015.04.002]. 
Briefly, fusion, CNV and gene expression data were used to subtype EPN as follows:

1. Any tumor with fusions containing `RELA` as fusion partner, e.g., `C11orf95--RELA`, `LTBP3--RELA`, was subtyped as `EPN, ST RELA`.
2. Any tumor with fusions containing `YAP1` as fusion partner, such as `C11orf95--YAP1`, `YAP1--MAMLD1` and `YAP1--FAM118B`, was subtyped as `EPN, ST YAP1`.
3. Any tumor with the following molecular characterization would be subtyped as `EPN, PF A`: 
- _CXorf67_ expression z-score of over 3
- _TKTL1_ expression z-score of over 3 and 1q gain
4. Any tumor with the following molecular characterization would be subtyped as `EPN, PF B`:
- _GPBP17_ expression z-score of over 3 and loss of 6q or 6p
- _IFT46_ expression z-score of over 3 and loss of 6q or 6p

Any tumor with the above molecular characteristics would be exclusively subtyped to the designated group. 

For all other remaining EPN tumors without above molecular characteristics, they would be subtyped to `EPN, ST RELA` and `EPN, ST YAP1` in a non-exclusive way (e.g., a tumor could have both `EPN, ST RELA` and `EPN, ST YAP1` subtypes) if any of the following alterations were present.

1. Any tumor with the following alterations was assigned `EPN, ST RELA`:
- `PTEN--TAS2R1` fusion 
- chromosome 9 arm (9p or 9q) loss 
- _RELA_ expression z-score of over 3 
- _L1CAM_ expression z-score of over 3 
2. Any tumor with the following alterations was assigned `EPN, ST YAP1`:
- `C11orf95--MAML2` fusion 
- chromosome 11 short arm (11p) loss  
- chromosome 11 long arm (11q) gain
- _ARL4D_ expression z-score of over 3 
- _CLDN1_ expression z-score of over 3

After all relevant tumor samples were subtyped by the above molecular subtyping modules, the results from these modules, along with other clinical information (such as pathology diagnosis free text), were compiled in the `molecular-subtyping-pathology` module and integrated into the OpenPBTA data in the `molecular-subtyping-integrate` module.


#### TP53 Alteration Annotation (`tp53_nf1_score` analysis module)

We annotated _TP53_ altered HGG samples as either `TP53 lost` or `TP53 activated` and integrated this within the molecular subtype.
To this end, we applied a _TP53_ inactivation classifier originally trained on TCGA pan-cancer data [@doi:10.1016/j.celrep.2018.03.076] to the matched RNA expression data for each sample. 
Along with the _TP53_ classifier scores, we collectively used consensus SNV and CNV, SV, and reference databases that list _TP53_ hotspot mutations [@doi:10.1158/2159-8290.CD-17-0321; @doi:10.1038/nbt.3391] and functional domains [@doi:10.1038/sj.cdd.4401904] to determine _TP53_ alteration status for each sample.
We adopted the following rules for calling either `TP53 lost` or `TP53 activated`:

1. If a sample had either of the two well-characterized _TP53_ gain-of-function mutations, p.R273C or p.R248W [@doi:10.1038/ng0593-42], we assigned `TP53 activated` status.

2. Samples were annotated as `TP53 lost` if they contained i) a _TP53_ hotspot mutation as defined by IARC _TP53_ database or the MSKCC cancer hotspots database [@doi:10.1158/2159-8290.CD-17-0321; @doi:10.1038/nbt.3391] (see also, **Key Resources Table**), ii) two _TP53_ alterations, including SNV, CNV or SV, indicative of probable bi-allelic alterations; iii) one _TP53_ somatic alteration, including SNV, CNV, or SV or a germline _TP53_ mutation indicated by the diagnosis of Li-Fraumeni syndrome [@doi:10.1101/cshperspect.a026187], or iv) one germline _TP53_ mutation indicated by Li-Fraumeni syndrome and the _TP53_ classifier score for matched RNA-Seq was greater than 0.5.

#### Prediction of participants' genetic sex

Participant metadata included a reported gender.
We used WGS germline data, in concert with the reported gender, to predict participant genetic sex so that we could identify sexually dimorphic outcomes.
This analysis may also indicate samples that may have been contaminated.
We used the `idxstats` utility from SAMtools [@pmid:19505943] to calculate read lengths, the number of mapped reads, and the corresponding chromosomal location for reads to the X and Y chromosomes.
We used the fraction of total normalized X and Y chromosome reads that were attributed to the Y chromosome as a summary statistic.
We manually reviewed this statistic in the context of reported gender and determined that a threshold of less than 0.2 clearly delineated female samples. 
We marked fractions greater than 0.4 as predicted males, and we marked samples with values in the inclusive range 0.2-0.4 as unknown.
We performed this analysis through [CWL](https://github.com/d3b-center/sex-determination-tool) on CAVATICA.
We added resulting calls to the histologies file under the column header `germline_sex_estimate`.


#### Selection of independent samples (`independent-samples` analysis module)

Certain analyses required that we select only a single representative specimen for each individual.
In these cases, we identified a single specimen by prioritizing primary tumors and those with whole-genome sequencing available.
If this filtering still resulted in multiple specimens, we randomly selected a single specimen from the remaining set.

#### Quantification of Telomerase Activity using Gene Expression Data (`telomerase-activity-prediction` analysis module)

We predicted telomerase activity of tumor samples using the recently developed EXTEND method [@doi:https://doi.org/10.1101/2020.05.21.109249].
Briefly, EXTEND estimates telomerase activity based on the expression of a 13-gene signature. 
We derived this signature by comparing telomerase-positive tumors and tumors with activated alternative lengthening of telomeres pathway, a group presumably negative of telomerase activity. 

### Survival models (`survival-analysis` analysis module)

We calculated overall survival (OS) as days since initial diagnosis and performed several survival analyses on the OpenPBTA cohort using the [`survival` R package](https://cran.r-project.org/package=survival).
We performed survival analysis for patients by HGG subtype using the Kaplan-Meier estimator [@doi:10.2307/2281868] and a log-rank test (Mantel-Cox test) [@pmid:5910392] on the different HGG subtypes.
Next, we used multivariate cox (proportional hazards) regression analysis [@url:http://www.jstor.org/stable/2985181] to model the following: a) `tp53 scores + telomerase scores + extent of tumor resection + LGG group + HGG group`, in which `tp53 scores` and `telomerase scores` are numeric, `extent of tumor resection` is categorical, and `LGG group` and `HGG group` are binary variables indicating whether the sample is in either broad histology grouping, b) `tp53 scores + telomerase scores + extent of tumor resection` for each `cancer_group` with an N>=3 deceased patients (DIPG, DMG, HGG, MB, and EPN), and c) `quantiseq cell type fractions + CD274 expression + extent of tumor resection` for each `cancer_group` with an N>=3 deceased patients (DIPG, DMG, HGG, MB, and EPN), in which `quantiseq cell type fractions` and `CD274 expression` are numeric.

### KEY RESOURCES TABLE

| REAGENT or RESOURCE     | SOURCE                                | IDENTIFIER                                                                                   |                               
|-----------------|-------------------------------------------------|------------------------------------------------------------------------|
|Critical commercial assays|||
| Recover Cell Culture Freezing media    | Gibco           | 12648010    |
| Hank's Balanced Salt Solution (HBSS)   | Gibco           | 14175095    |
| Papain                                 | SciQuest        | LS003124    |
| Ovomucoid                              | SciQuest        | 542000      |
| DNase                                  | Roche           | 10104159001 |
| 100μm cell strainer                    | Greiner Bio-One | 542000      |
| DMEM/F-12 medium                       | Sigma           | D8062       |
| Fetal Bovine Serum (FBS)               | Hyclone         | SH30910.03  |
| GlutaMAX                               | Gibco           | 35050061    |
| Penicillin/Streptomycin-Amphotericin B | Lonza           | 17-745E     |
| Normocin                               | Invivogen       | ant-nr-2    |
| B-27 supplement minus vitamin A        | Gibco           | 12587-010   |
| N-2 supplement                         | Gibco           | 17502001    |
| Epidermal growth factor                | Gibco           | PHG0311L    |
| Basic fibroblast growth factor         | PeproTech       | 100-18B     |
| Heparin                                | Sigma           | H3149       |
| DNA/RNA AllPrep Kit                    | Qiagen          | 80204       |
| TruSeq RNA Sample Prep Kit             | Illumina        | FC-122-1001 |
| KAPA Library Preparation Kit           | Roche           | KK8201      |
| AllPrep DNA/RNA/miRNA Universal kit    | Qiagen          | 80224       |
| RNase A                                | Qiagen          | 19101       |
| QIAsymphony DSP DNA Midi Kit           | Qiagen          | 937255      |
| KAPA HyperPrep kit                     | Roche           | 08098107702      |
| RiboErase kit                          | Roche           | 07962304001      |
|Deposited data|||
|Raw and harmonized WGS, WXS, Panel, RNA-Seq                    |KidsFirst Data Resource Center, this project                        |[@doi:10.24370/OPENPBTA]                    |
|Merged summary files                    |this project                        |https://cavatica.sbgenomics.com/u/cavatica/openpbta                    |
|Merged summary files and downstream analyses                   |this project                        |https://github.com/AlexsLemonade/OpenPBTA-analysis/                    |
|Processed data                    |this project                        |https://pedcbioportal.kidsfirstdrc.org/study/summary?id=openpbta                   |
|Experimental models: Cell lines|||
|CBTN pediatric brain tumor-derived cell lines                    |[@doi:10.1093/neuonc/noz192]                        |See **Table S1** for identifiers                   |
|Software and algorithms|||
|Data processing and analysis software                    |Multiple                        |See **Table S5** for identifiers                    |
|OpenPBTA workflows repository                   |this project                        |[@doi:10.5281/zenodo.6474116]                    |
|OpenPBTA analysis repository                    |this project                        |                    |
|OpenPBTA manuscript repository                    |this project                        |                    |
|                    |                        |                    |
|Other|||
|TCGA WXS dataset                    |National Institutes of Health The Cancer Genome Atlas (TCGA)
                        |dbGAP phs000178.v11.p8                    |
|Cancer hotspots                    |MSKCC                        |https://www.cancerhotspots.org/#/download (v2)                    |
|Reference genomes                    |Broad                        |https://s3.console.aws.amazon.com/s3/buckets/broad-references/hg38/v0/                    |
|Reference genome hg38, patch release 12                  |UCSC                        |http://hgdownload.soe.ucsc.edu/goldenPath/hg38/bigZips/                    |
|Human Cytoband file                    |UCSC                        |http://hgdownload.cse.ucsc.edu/goldenpath/hg38/database/cytoBand.txt.gz                     |
|CDS from GENCODE v27 annotation                |GENCODE                        |https://www.gencodegenes.org/human/release_27.html                     |
|PFAM domains and locations                    |UCSC                        |http://hgdownload.soe.ucsc.edu/goldenPath/hg38/database/pfamDesc.txt.gz; https://pfam.xfam.org/family/PF07714                      |
|BSgenome.Hsapiens.UCSC.hg38 annotations                   |Bioconductor                        |https://bioconductor.org/packages/release/data/annotation/html/BSgenome.Hsapiens.UCSC.hg38.html                     |
|                    |                        |                     |


## Supplemental Information Titles and Legends

![**OpenPBTA Project Workflow, Related to Figure 1.** Biospecimens and data were collected by CBTN and PNOC. Genomic sequencing and harmonization (orange boxes) were performed by the Kids First Data Resource Center (KFDRC). Analyses in the green boxes were performed by contributors of the OpenPBTA project. Output files are denoted in blue. Figure created with [BioRender.com](biorender.com).](https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/0107be711b79f228f858f539fad1a4e8b2ced7c3/figures/pngs/figureS1.png?sanitize=true){#fig:S1 tag="S1" width="7in"}

![**Validation of Consensus SNV calls and Tumor Mutation Burden, Related to Figures 2 and 3.** Correlation (A) and violin (B) plots of mutation variant allele frequencies (VAFs) comparing the variant callers (Lancet, Strelka2, Mutect2, and VarDict) used for PBTA samples. Upset plot (C) showing overlap of variant calls. Correlation (D) and violin (E) plots of mutation variant allele frequencies (VAFs) comparing the variant callers (Lancet, Strelka2, and Mutect2) used for TCGA samples. Upset plot (F) showing overlap of variant calls. Violin plots (G) showing VAFs for Lancet calls performed on WGS and WXS from the same tumor (N = 52 samples from 13 patients). Cumulative distribution TMB plots for PBTA (H) and TCGA (I) tumors using consensus SNV calls.](https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/0107be711b79f228f858f539fad1a4e8b2ced7c3/figures/pngs/figureS2.png?sanitize=true){#fig:S2 tag="S2" width="7in"}

![**Genomic instability of pediatric brain tumors, Related to Figures 2 and 3.** (A) Oncoprint of canonical somatic gene mutations, CNVs, fusions, and TMB (top bar plot) for the top 20 genes mutated across rare CNS tumors (N < 5 each): desmoplastic infantile astrocytoma and ganglioglioma (N = 1), germinoma (N = 4), glial-neuronal NOS (N = 4), metastatic secondary tumors (N = 3), neurocytoma (N = 2), and pineoblastoma (N = 3). Patient sex (`germline_sex_estimate`) and tumor histology (`cancer_group`) are displayed as annotations at the bottom of each plot. Only primary tumors with mutations in the listed genes are shown. Multiple CNVs are denoted as a complex event. (B) Genome-wide plot of CNV alterations by broad histology. Each row represents one sample. Box and whisker plots of number of CNV breaks (C) or SV breaks (D) by number of chromothripsis regions.](https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/0107be711b79f228f858f539fad1a4e8b2ced7c3/figures/pngs/figureS3.png?sanitize=true){#fig:S3 tag="S3" width="7in"}

![**Mutational signatures in pediatric brain tumors, Related to Figure 3.** (A) Sample-specific RefSig signature weights across cancer groups ordered by decreasing Signature 1 exposure. (B) Proportion of Signature 1 plotted by phase of therapy for each cancer group.](https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/0107be711b79f228f858f539fad1a4e8b2ced7c3/figures/pngs/figureS4.png?sanitize=true){#fig:S4 tag="S4" width="7in"}

![**Quality control metrics for _TP53_ and EXTEND scores, Related to Figure 4**. (A) Receiver Operating Characteristic for _TP53_ classifier run on FPKM of poly-A RNA-Seq samples. Correlation plots for telomerase scores (EXTEND) with RNA expression of _TERT_ (B) and _TERC_ (C).](https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/0107be711b79f228f858f539fad1a4e8b2ced7c3/figures/pngs/figureS5.png?sanitize=true){#fig:S5 tag="S5" width="7in"}

![**Subtype-specific clustering and immune cell fractions, Related to Figure 5**. First two dimensions from UMAP of sample transcriptome data with points colored by `molecular_subtype` for medulloblastoma (A), ependymoma (B), low-grade glioma (C), and high-grade diffuse astrocytic tumors (D). (E) Box plots of quanTIseq estimates of immune cell fractions in histologies with more than one molecular subtype with N >=3. (F) Box plots of the ratio of immune cell fractions of CD8+ to CD4+ T cells in histologies with more than one molecular subtype with N >=3.](https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/0107be711b79f228f858f539fad1a4e8b2ced7c3/figures/pngs/figureS6.png?sanitize=true){#fig:S6 tag="S6" width="7in"}

[**Table S1. Related to Figure 1.**](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/0107be711b79f228f858f539fad1a4e8b2ced7c3/tables/results/TableS1-histologies.xlsx)
Table of specimens and associated metadata, clinical data, and histological data utilized in the OpenPBTA project.

[**Table S2. Related to Figures 2 and 3.**](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/0107be711b79f228f858f539fad1a4e8b2ced7c3/tables/results/TableS2-DNA-results-table.xlsx) 
Excel file with three sheets representing tables of TMB, eight CNS mutational signatures, and chromothripsis events per sample, respectively.

[**Table S3. Related to Figures 4 and 5.**](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/0107be711b79f228f858f539fad1a4e8b2ced7c3/tables/results/TableS3-RNA-results-table.xlsx)
Excel file with three sheets representing tables of _TP53_ scores, telomerase EXTEND scores, and quanTIseq immune scores, respectively.

[**Table S4. Related to Figures 4 and 5.**](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/0107be711b79f228f858f539fad1a4e8b2ced7c3/tables/results/TableS4-survival-results-table.xlsx)
Excel file with six sheets representing the survival analyses performed for this manuscript. 
See **Star Methods** for details.

[**Table S5. Related to Figure 1.**](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/0107be711b79f228f858f539fad1a4e8b2ced7c3/tables/results/TableS5-Key-Resources-table.xlsx)
Excel file with four sheets representing of all software and their respective versions used for the OpenPBTA project, including the R packages in the OpenPBTA Docker image, Python packages i the OpenPBTA Docker image, other command line tools in the OpenPBTA Docker image, and all software used in the OpenPBTA workflows, respectively. 
Note that all software in the OpenPBTA Docker image was utilized within the analysis repository, but not all software was used for the final manuscript.


## Consortia

The past and present members of the Children's Brain Tumor Network who contributed to the generation of specimens and data are Adam C. Resnick, Alexa Plisiewicz, Allison M. Morgan, Allison P. Heath, Alyssa Paul, Amanda Saratsis, Amy Smith, Ana Aguilar, Ana Guerreiro Stücklin, Anastasia Arynchyna, Andrea Franson, Angela J. Waanders, Angela N. Viaene, Anita Nirenberg, Anna Maria Buccoliero, Anna Yaffe, Anny Shai, Anthony Bet, Antoinette Price, Ashley Plant, Augustine Eze, Bailey C. Farrow, Baoli Hu, Beth Frenkel, Bo Zhang, Bobby Moulder, Bonnie Cole, Brian M. Ennis, Brian R. Rood, Brittany Lebert, Carina A. Leonard, Carl Koschmann, Caroline Caudill, Caroline Drinkwater, Cassie N. Kline, Catherine Sullivan, Chanel Keoni, Chiara Caporalini, Christine Bobick-Butcher, Christopher Mason, Chunde Li, Claire Carter, Claudia MaduroCoronado, Clayton Wiley, Cynthia Wong, David E. Kram, David Haussler, David Kram, David Pisapia, David Ziegler, Denise Morinigo, Derek Hanson, Donald W. Parsons, Elizabeth Appert, Emily Drake, Emily Golbeck, Ena Agbodza, Eric H. Raabe, Eric M. Jackson, Eric Raabe, Esteban Uceda, Eugene Hwang, Fausto Rodriquez, Gabrielle S. Stone, Gary Kohanbash, George Rafidi, Gerald Grant, Gerri Trooskin, Gilad Evrony, Graham Keyes, Hagop Boyajian, Holly B. Lindsay, Holly Beale, Ian F. Pollack, James Johnston, James Palmer, Jane Minturn, Jared Pisapia, Jason E. Cain, Javad Nazarian, Jeanette Haugh, Jeff Greenfield, Jeff Stevens, Jeffrey P. Greenfield, Jeffrey Rubens, Jena V. Lilly, Jennifer L. Mason, Jessica B. Foster, Jim Olson, Jo Lynne Rokita, Joanna J. Phillips, Jonathan Waller, Josh Rubin, Judy E. Palma, Justine Rizzo, Kaitlin Lehmann, Kamnaa Arya, Karlene Hall, Katherine Pehlivan, Kenneth Seidl, Kimberly Diamond, Kristen Harnett, Kristina Cole, Krutika S. Gaonkar, Lamiya Tauhid, Lina Lopez, Lionel Chow, Mahdi Sarmady, Margaret Nevins, Mari Groves, Mariarita Santi-Vicini, Marilyn M. Li, Marion Mateos, Mateusz Koptyra, Matija Snuderl, Matthew Miller, Matthew Sklar, Matthew Wood, Meghan Connors, Melissa Williams, Meredith Egan, Michael Fisher, Michael Koldobskiy, Michelle Monje, Migdalia Martinez, Miguel A. Brown, Mike Prados, Miriam Bornhorst, Mirko Scagnet, Mohamed AbdelBaki, Monique Carrero-Tagle, Nadia Dahmane, Nalin Gupta, Nathan Young, Nicholas A. Vitanza, Nicholas Tassone, Nicolas Gerber, Nithin D. Adappa, Nitin Wadhwani, Obi Obayashi, Olena M. Vaske, Olivier Elemento, Oren Becher, Philbert Oliveros, Phillip B. Storm, Pichai Raman, Rintaro Hashizume, Robert Keating, Robert M. Lober, Ron Firestein, Sabine Mueller, Sameer Agnihotri, Samuel G. Winebrake, Samuel Rivero-Hinojosa, Sarah Diane Black, Sarah Leary, Schuyler Stoller, Shannon Robins, Sharon Gardner, Sherri Mayans, Sherry Tutson, Shida Zhu, Sofie Salama, Sonia Partap, Sonika Dahiya, Sriram Venneti, Stacie Stapleton, Stephani Campion, Stephanie Stefankiewicz, Swetha Thambireddy, Tatiana Patton, Teresa Hidalgo, Theo Nicolaides, Thinh Q. Nguyen, Tiffany Walker, Valeria Lopez-Gil, Valerie Baubet, Xiao-Nan Li, Ximena Cuellar, Yiran Guo, Yuankun Zhu, and Zeinab Helil.

The past and present members of the Pacific Pediatric Neuro-Oncology Consortium who contributed to the generation of specimens and data are Adam C. Resnick, Alicia Lenzen, Alyssa Reddy, Amar Gajjar, Ana Guerreiro Stucklin, Anat Epstein, Andrea Franson, Angela Waanders, Anne Bendel, Anu Banerjee, Ashley Margol, Ashley Plant, Brian Rood, Carl Koschmann, Carol Bruggers, Caroline Hastings, Cassie N. Kline, Christina Coleman Abadi, Christopher Tinkle, Corey Raffel, Dan Runco, Daniel Landi, Daphne Adele Haas-Kogan, David Ashley, David Ziegler, Derek Hanson, Dong Anh Khuong Quang, Duane Mitchell, Elias Sayour, Eric Jackson, Eric Raabe, Eugene Hwang, Fatema Malbari, Geoffrey McCowage, Girish Dhall, Gregory Friedman, Hideho Okada, Ibrahim Qaddoumi, Iris Fried, Jae Cho, Jane Minturn, Jason Blatt, Javad Nazarian, Jeffrey Rubens, Jena Lilly, Jennifer Elster, Jennifer Mason, Jessica Schulte, Jonathan Schoenfeld, Josh Rubin, Karen Gauvain, Karen Wright, Katharine Offer, Katie Metrock, Kellie Haworth, Ken Cohen, Kristina A Cole, Lance Governale, Linda Stork, Lindsay Kilburn, Lissa Baird, Maggie Skrypek, Marcia Leonard, Margaret Shatara, Margot Lazow, Mariella Filbin, Maryam Fouladi, Matthew Miller, Megan Paul, Michael Fisher, Michael Koldobskiy, Michal Yalon Oren, Mimi Bandopadhayay, Miriam Bornhorst, Mohamed AbdelBaki, Nalin Gupta, Nathan Robison, Nicholas Whipple, Nick Gottardo, Nick Vitanza, Nicolas Gerber, Patricia Robertson, Payal Jain, Peter Sun, Prados, Michael, Priya Chan, Richard S Lemons, Robert Wechsler-Reya, Roger Packer, Russ Geyer, Sabine Mueller, Sahaja Acharya, Sam Cheshier, Sarah Leary, Scott Coven, Sharon Gardner, Sri Gururangan, Stewart Goldman, Susan Chi, Tab Cooney, Tatiana S. Patton, Theodore Nicolaides, and Tom Belle Davidson.

## References {.page_break_before}

<!-- Explicitly insert bibliography here -->
<div id="refs"></div>
