## Materials and Methods

### Children's Brain Tumor Tissue Consortium Data

### Pediatric Pacific Neuro-oncology Consortium Data

### DNA WGS Alignment

We used BWA-MEM [@arxiv:1303.3997] v0.7.17 for alignment of paired-end DNA-seq reads.
The alignment reference that we used was Homo Sapiens Human Genome (hg) version 38, patch release 12, fasta file obtained from UCSC [@url:http://hgdownload.soe.ucsc.edu/goldenPath/hg38/bigZips/].
Alignments were futher processed using following the Broad Institute's Best Practices [@url:https://software.broadinstitute.org/gatk/best-practices/workflow?id=11165] for processing BAMs in preparation for variant discovery.
Duplicates were marked using Samblaster[@doi:10/f6kft3] v0.1.24, BAMs merged and sorted using Sambamba [@doi:10/gfzsfw] v0.6.3.
Lastly, resultant BAMs were processing using Broad's Genome Analysis Tool Kit (GATK) [@url:https://software.broadinstitute.org/gatk/] v4.0.3.0, BaseRecalibrator submodule.

### Germ Line Single Nucleotide Variant Calling

### Germ Line Copy Number Variant Calling

### Germ Line Structural Variant Calling

### Somatic Single Nucleotide Variant Calling
#### SNV and INDEL calling

We used Strelka2 [@doi:10/gdwrp4] v2.9.3 and Mutect2 from GATK v4.1.1.0.
Strelka2 was run using default parameters on human genome reference hg38, canonical chromosomes only (chr1-22, X,Y,M), as recommended by the author.
Mutect2 was run following Broad best practices outlined from their Workflow Description Language (WDL) [@url:https://github.com/broadinstitute/gatk/blob/4.1.1.0/scripts/mutect2_wdl/mutect2.wdl].  

#### VCF annotation and MAF creation

We filtered outputs from both callers on the "PASS" filter, and annotated using The ENSEMBL Variant Effect Predictor [@doi:10/gdz75c], reference release 93, and created MAFs using MSKCC's vcf2maf [@url:https://github.com/mskcc/vcf2maf] v1.6.16.

### Somatic Copy Number Variant Calling

We used Control-FREEC [@doi:10/ckt4vz; @doi:10/c6bcps] v8.7 for copy number variant calls.

### Somatic Structural Variant Calling

We used Manta SV [@doi:10/gf3ggb] v1.4.0 for structural variant (SV) calls.
SV calling was also limited to regions using in Strelka2.

### Gene Expression Abundance Estimation
We used STAR [@doi:10/f4h523] v2.6.1d to align paired-end RNA-seq reads.
This output was used for all subsequent RNA analysis. The reference we used was that of ENSEMBL's GENCODE 27 [@url:https://www.gencodegenes.org/human/release_27.html], "Comprehensive gene annotation."
We used RSEM [@doi:10/cwg8n5] v1.3.1 for transcript- and gene-level quantification.
We also added a second method of quantification using kallisto [@doi:10.1038/nbt.3519] v0.43.1.
This method differs in that it uses pseudoaligments using fastq reads directly to the aforementioned GENCODE 27 reference.

### RNA Fusion Calling and Prioritization

### Clinical Data Harmonization
