## Materials and Methods

### Children's Brain Tumor Tissue Consortium Data
The deidentified patient's blood and tumor tissue were prospectively collected by the Children's Brain Tumor Tissue Consortium (CBTTC). 
The CBTTC [https://cbttc.org] is a collaborative, multi-institutional research program dedicated to the study of childhood brain tumors. 
The consortium currently includes 16 institutions worldwide. 
The CBTTC open source biorepository [https://eig.research.chop.edu/cbttc/] is available to member and non-member investigators through a submission request system [@doi:10.1186/s12864-016-2797-9]. 
All CBTTC data is available for viewing and download from the Gabriella Miller Kids First Data Resource Center (KF-DRC), [https://kidsfirstdrc.org]. 
Clinical reports can be downloaded as PDFs and Aperio SVS histology images. 
Pre- and post-operative MRI images and reports are also available for a subset of the CBTTC subjects.

The cell lines were generated by the CBTTC from either fresh tumor tissue obtained firectly from surgery performed at Children’s Hospital of Philadelphia (CHOP) or from prospectively collected tumor specimens stored in Recover Cell Culture Freezing media (cat# 12648010, Gibco).
The tissue was dissociatiated using enzymatic method with papain as described [@doi:10.1101/656587].
Briefly, tissue was washed with HBSS (cat# 14175095, Gibco), minced and icubated with activated papin solution (cat# LS003124, SciQuest) for up to 45 minutes. 
The papain was inactivated using ovomucoid solution (cat# 542000, SciQuest), tissue was briefly treated with DNase (cat#  10104159001, Sigma) and passed through the 100μm cell strainer (cat# 542000, Greiner Bio-One). 
Two cell culture conditions were initiated based on the number of cells avialble. 
For cultures utilizing the fetal bovine serum (FBS), a minimum density of 3×10^5 cells/ml were plated in DMEM/F-12 medium (cat# D8062, Sigma) supplemented with 20% FBS (cat# SH30910.03, Hyclone), 1% GlutaMAX (cat# 35050061, Gibco), Penicillin/Streptomycin-Amphotericin B Mixture (cat# 17-745E, Lonza) and 0.2% Normocin (cat# ant-nr-2, Invivogen). 
For the serum-free media conditions cells were plated at minimum density of 1×10^6 cells/ml in DMEM/F12 media supplemented with 1% GlutaMAX, 1x B-27 supplement minus vitamin A (cat# 12587-010, Gibco), 1x N-2 supplement (cat# 17502001, Gibco), 20 ng/ml epidermal growth factor (cat# PHG0311L, Gibco), 20 ng/ml basic fibroblast growth factor (cat# 100-18B, PeproTech), 2.5μg/ml heparin (cat# H3149, Sigma), Penicillin/Streptomycin-Amphotericin B Mixture and 0.2% Normocin. 

#### DNA/RNA extractions

Blood, tissue and cell line DNA/RNA extraction was performed at Biorepository Core (BioRC) at CHOP. 
Briefly, 10-20 mg frozen tissue, 0.4-1ml of blood or 2×10^6 cells pellet was used for extractions.
Tissues were lysed using a Qiagen TissueLyser II (Qiagen) with 2×30 sec at 18Hz settings using 5 mm steel beads (cat# 69989, Qiagen). 
Both tissue and cell pellets processes included a CHCl3 extraction and were run on the QiaCube automated platform (Qiagen) using the AllPrep DNA/RNA/miRNA Universal kit (cat# 80224, Qiagen). 
Blood was thawed and threated with RNase A (cat#, 19101, Qiagen); 0.4-1ml was processed using the Qiagen QIAsymphony automated platform (Qiagen) using the QIAsymphony DSP DNA Midi Kit (cat# 937255, Qiagen).

DNA and RNA quantity and quality was assessed by PerkinElmer DropletQuant UV-VIS spectrophotometer (PerkinElmer) and an Agilent 4200 TapeStation (Agilent, USA) for RINe and DINe (RNA Integrity Number equivalent and DNA Integrity Number equivalent respectively). 
Library preparation and sequencing was performed by the NantHealth sequencing center. 
Briefly, DNA sequencing libraries were prepared for tumor and matched-normal DNA using the KAPA Hyper prep kit (cat# KK8541, Roche); tumor RNA-Seq libraries were prepared using KAPA Stranded RNA-Seq with RiboErase kit(cat# KK8484, Roche). 
Whole genome sequencing (WGS) was performed at an average depth of coverage of 60X for tumor samples and 30X for germline. 
RNA samples were sequenced to an average of 200M reads. 
All samples were sequenced on the Illumina HiSeq platform (X/400) (Illumina) with 2 × 150bp read length.
For the cell line sequencing, samples labelled with “CL-adh” correspond to the adherent FBS cell lines and those labelled “CL-susp” are the serum-free lines.

### Pediatric Pacific Neuro-oncology Consortium Data

### DNA Alignment

### Germ Line Single Nucleotide Variant Calling

### Somatic Single Nucleotide Variant Calling

### Somatic Copy Number Variant Calling

### Somatic Structural Variant Calling

### Gene Expression Abundance Estimation

### RNA Fusion Calling and Prioritization
#### Gene fusion detection

We set up [Arriba v1.1.0](https://github.com/suhrig/arriba/) and STAR-Fusion 1.5.0 [@doi:10.1101/120295] fusion detection tools using CWL on CAVATICA.
For both these tools we used aligned BAM and chimeric SAM files from STAR as inputs and GRCh38_gencode_v27 GTF for gene annotation.
We ran STAR-Fusion with default parameters and annotated all fusion calls with GRCh38_v27_CTAT_lib_Feb092018.plug-n-play.tar.gz provided in the STAR-fusion release. 
For Arriba, we used a blacklist file (blacklist_hg38_GRCh38_2018-11-04.tsv.gz) from the Arriba release tarballs to remove recurrent fusion artifacts and transcripts present in healthy tissue.
We also provided Arriba with strandedness information or set it to auto-detection for polyA samples.

#### Fusion prioritization

We built a [fusion prioritization pipeline](https://github.com/d3b-center/fusion_filtering_pipeline) to filter and annotate fusions.
We considered all inframe and frameshift fusion calls with 1 or more junction reads and fused genes expressed with TPM greater than one to be true calls.
If a fusion call had large number of spanning fragment reads compared to junction reads (spanning fragment minus junction read greater than ten) or if either 5\` or 3\` genes fused to more than five different genes we removed these calls as a potential false positive.
We also removed fusions if the 5\` or 3\` ends were the same gene, and these were tagged as non-canonical splicing or duplication.
We used a list of curated fusion calls for each histology to capture each occurrence of the fusion as a putative driver fusion.
We prioritized a union of fusion calls as true calls if the fused genes were detected by both callers, the same fusion was recurrent in histology (>2 samples) or the fusion was specific to the broad histology. 
We annotated putative driver fusions and prioritized fusions lists with kinases, oncogenic, tumor suppressor, transcription factor, fused genes and known TCGA fusions from curated [datasheets](https://github.com/d3b-center/fusion_filtering_pipeline/tree/master/references).
We also added chimerDB [@doi:10.1093/nar/gkw1083] annotations to both driver and prioritized fusion list.

### Clinical Data Harmonization
