## Materials and Methods

### Children's Brain Tumor Tissue Consortium Data

### Pediatric Pacific Neuro-oncology Consortium Data

### Germ Line Single Nucleotide Variant Calling

### Germ Line Copy Number Variant Calling

### Germ Line Structural Variant Calling

### Somatic Single Nucleotide Variant Calling

### Somatic Copy Number Variant Calling

### Somatic Structural Variant Calling

### Gene Expression Abundance Estimation

### RNA Fusion Calling and Prioritization
**Gene Fusion Detection**
> Arriba v1.1.0 and STAR-Fusion 1.5.0 fusion detection tools were set up using CWL on CAVATICA.
> Aligned bam and chimeric sam files from STAR were used as inputs and GRCh38_gencode_v27 GTF was used for gene annotation in both tools.
> STAR-fusion was run with default paramters and fusion calls were annotated with GRCh38_v27_CTAT_lib_Feb092018.plug-n-play.tar.gz. 
> While running arriba, the additional blacklist file blacklist_hg38_GRCh38_2018-11-04.tsv.gz was provided in the arriba release tarballs to remove recurrent fusion artifacts and transcripts present in healthy tissue.
> Additionally strandedness information was provided to arriba or set to auto-detection for polyA samples.

**Fusion prioritization**
> All inframe and frameshift fusion calls with 1 or more junction reads and fused genes expressed at TPM >1 were used as true calls.
> If a fusion call had large number of spanning fragment reads compared to junction reads (if spanning fragment - junction read > 10) or if either 5\` or 3\` genes multi-fused > 5 different genes these were removed as a potential false positive.
> A list of curated list of fusion calls for each histology was used to capture each occurence of the fusion as a putative driver fusion.
> A union of fusion calls which were either detected by both callers, or the same fusion was recurrent in histology(>2 samples)  or the fusion was specific to the broad histology were prioritized as potential true calls. 
> The putative driver fusions and prioritized fusions lists were annotated to identify kinases,oncogenic,tumor suppresor fused genes and known TCGA fusions. 

### Clinical Data Harmonization
