## Materials and Methods

### Children's Brain Tumor Tissue Consortium Data

### Pediatric Pacific Neuro-oncology Consortium Data

### DNA WGS Alignment

We used BWA-MEM v0.7.17 [@arxiv:1303.3997] for alignment of paired-end DNA-seq reads.

### Germ Line Single Nucleotide Variant Calling

### Germ Line Copy Number Variant Calling

### Germ Line Structural Variant Calling

### Somatic Single Nucleotide Variant Calling
**SNV and INDEL Calling**

We used [Strelka2](doi:10.1038/s41592-018-0051-x) v2.9.3 and Mutect2 from [Broad's GATK](https://software.broadinstitute.org/gatk/) v4.1.1.0. Strelka2 was run using default parameters on human genome reference hg38, canonical chromosomes only (chr1-22, X,Y,M), as recommended hy the author.  Mutect2 was run following Braid best practices outlined from their [WDL](https://github.com/broadinstitute/gatk/blob/4.1.1.0/scripts/mutect2_wdl/mutect2.wdl).  

**VCF Annotation and MAF Creation**

We filtered outputs from both callers on the "PASS" filter, and annotated using [The ENSEMBL Variant Effect Predictor](doi:10.1186/s13059-016-0974-4), reference release 93, and created mafs using [MSKCC's vcf2maf](https://github.com/mskcc/vcf2maf) v1.6.16.

### Somatic Copy Number Variant Calling

We used [Control-FREEC](https://doi.org/10.1093/bioinformatics/btr670) v8.7 for copy number variant calls.

### Somatic Structural Variant Calling

We used [Manta SV](doi:10.1093/bioinformatics/btv710) v1.4.0 for structural variant calls.

### Gene Expression Abundance Estimation
We used [STAR aligner](doi:10.1093/bioinformatics/bts635) v2.6.1d for alignment of paired-end RNAseq reads. This output was used for all subsequent RNA analysis. We used [RSEM](https://github.com/deweylab/RSEM) v1.3.1 for transcript- and gene-level quantification.

### RNA Fusion Calling and Prioritization

### Clinical Data Harmonization
