## Materials and Methods

### Children's Brain Tumor Tissue Consortium Data

### Pediatric Pacific Neuro-oncology Consortium Data

### DNA WGS Alignment

We used BWA-MEM v0.7.17 [@arxiv:1303.3997] for alignment of paired-end DNA-seq reads.
The alignment reference that we used was Homo Sapiens Human Genome (hg) hg38 fasta file obtanied from [UCSC](http://hgdownload.soe.ucsc.edu/goldenPath/hg38/bigZips/).
Alignments were futher processed using following [Broad's Best Practices](https://software.broadinstitute.org/gatk/best-practices/workflow?id=11165) for processing bams in preparation for variant discovery.
Duplicates were marked using [Samblaster]( doi:10.1093/bioinformatics/btu314) v0.1.24, bams merged and sorted using [Sambamba](https://lomereiter.github.io/sambamba/) v0.6.3.
Lastly, resultant bams were processing using [Broad's Genome Analysis Tool Kit (GATK)](https://software.broadinstitute.org/gatk/) v4.0.3.0, BaseRecalibrator submodule.

### Germ Line Single Nucleotide Variant Calling

### Germ Line Copy Number Variant Calling

### Germ Line Structural Variant Calling

### Somatic Single Nucleotide Variant Calling
**SNV and INDEL Calling**

We used [Strelka2](doi:10.1038/s41592-018-0051-x) v2.9.3 and Mutect2 from GATK v4.1.1.0.
Strelka2 was run using default parameters on human genome reference hg38, canonical chromosomes only (chr1-22, X,Y,M), as recommended hy the author.
Mutect2 was run following Broad best practices outlined from their [WDL](https://github.com/broadinstitute/gatk/blob/4.1.1.0/scripts/mutect2_wdl/mutect2.wdl).  

**VCF Annotation and MAF Creation**

We filtered outputs from both callers on the "PASS" filter, and annotated using [The ENSEMBL Variant Effect Predictor](doi:10.1186/s13059-016-0974-4), reference release 93, and created mafs using [MSKCC's vcf2maf](https://github.com/mskcc/vcf2maf) v1.6.16.

### Somatic Copy Number Variant Calling

We used [Control-FREEC](https://doi.org/10.1093/bioinformatics/btr670) v8.7 for copy number variant calls.

### Somatic Structural Variant Calling

We used [Manta SV](doi:10.1093/bioinformatics/btv710) v1.4.0 for structural variant (SV) calls.
SV calling was also limited to regions using in Strelka2.

### Gene Expression Abundance Estimation
We used [STAR](doi:10.1093/bioinformatics/bts635) v2.6.1d to align paired-end RNA-seq reads.
This output was used for all subsequent RNA analysis. The reference we used was that of [ENSEMBL's GENCODE 27](https://www.gencodegenes.org/human/release_27.html), "Comprehensive gene annotation."
We used [RSEM](doi10.1186/1471-2105-12-323) v1.3.1 for transcript- and gene-level quantification.

### RNA Fusion Calling and Prioritization

### Clinical Data Harmonization
