## Results

### Crowd-sourced Somatic Analysis to create an Open Pediatric Brain Tumor Atlas

We previously performed whole genome sequencing (WGS), whole exome sequencing (WXS), and RNA sequencing (RNA-Seq) on matched tumor and normal tissues as well as selected cell lines from 943 patient tumors from the Pediatric Brain Tumor Atlas (PBTA) [@doi:10.1093/neuonc/noz192], consisting of samples from the Children’s Brain Tumor Network (CBTN) [@url:https://CBTN.org] and the PNOC003 DMG clinical trial [@doi:10.1002/ijc.32258] of the Pacific Pediatric Neuro-Oncology Symposium (PNOC) (**Figure {@fig:Fig1}A**). 
We then harnessed the benchmarking efforts of the Kids First Data Resource Center to develop a robust and reproducible data analysis workflow within the CAVATICA platform to perform the following primary somatic analyses: variant calling of single nucleotide variants (SNVs), copy number variants (CNVs), structural variants (SVs), and fusions (**Figure S1** - red boxes and **STAR Methods**). 
Next, we created a public Github analysis repository (https://github.com/AlexsLemonade/OpenPBTA-analysis) with continuous integration to ensure analysis reproducibility.
We additionally created a GitHub manuscript repository (https://github.com/AlexsLemonade/OpenPBTA-manuscript) with ManuBot [@url:https://manubot.org/ ; @doi:10.1371/journal.pcbi.1007128] integration to enable manuscript creation using Markdown within GitHub.

We maintained a data release folder on Amazon S3 containing merged files for each analysis, downloadable from the GitHub repository or open access CAVATICA project.  

<!-- TODO: Stub for the process for contributing analytical code (**Figure {@fig:FigS1}** and **STAR Methods**) and to the manuscript. -->

![**Figure 1. Overview of the OpenPBTA Project.** A, The Children's Brain Tumor Network and the Pacific Pediatric Neuro-Oncology Consortium collected tumor samples from 943 patients. To date, 38 cell lines were created from tumor tissue and over 2000 specimens were sequenced (N = 1035 RNA-Seq, N = 940 WGS, and N = 32 WXS or Targeted panel). Data was harmonized by the Kids First Data Resource Center using an Amazon S3 framework within CAVATICA. B, Processed data was released through the OpenPBTA GitHub project, through which contributors filed issues and developed code for analyses within docker. Additional analysis details can be found in **Figure S1**. Simultaneously, this manuscript was written in GitHub using Manubot. All analysis and manuscript code was reviewed and continuously integrated using CircleCI. C, Stacked bar plot summary of the number of biospecimens per phase of therapy per broad histology.](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/figures/pngs/figure1.png){#fig:Fig1}

### Molecular Subtyping of OpenPBTA CNS Tumors

Over the past two decades, together with the World Health Organization (WHO), experts in neurooncology have iteratively redefined the classifications of central nervous system (CNS) tumors [@doi: 10.1093/jnen/61.3.215; @doi:10.1007/s00401-007-0243-4]. 
More recently, in 2016 and 2021 [@doi:10.1007/s00401-016-1545-1; @doi:10.1093/neuonc/noab106], molecular subtypes have been integrated into these entities. 
The Children’s Brain Tumor Tissue Consortium (CBTTC), currently the Children’s Brain Tumor Network (CBTN), opened its protocol for brain tumor and matched normal sample collection in 2011 and as such, the majority of the samples within the OpenPBTA dataset lack molecular subtype annotation. 
In the absence of methylation arrays for tumor classification, we utilized key genomic features of entities described by the WHO in 2016, as well as Ryall and colleagues [@doi:10.1016/j.ccell.2020.03.011], coupled with clinician and pathologist review to subtype 64% (1,281/2,007) of tumor biospecimens with high confidence (**Table S1**).
Importantly, this collaborative molecular subtyping process allowed us to identify data entry errors (e.g., an ETMR entered as a medulloblastoma) and mis-identified specimens (e.g., Ewing sarcoma sample labeled as a craniopharyngioma), update diagnoses using modern terms (e.g., primitive neuro-ectodermal tumor [PNET] diagnoses), and discover rarer tumor entities within the OpenPBTA (e.g., H3-mutant ependymoma, meningioma with _YAP1::FAM118B_ fusion).
**Table 1** {#tbl:subtypes} lists the subtypes we defined within the OpenPBTA, comprising of LGATs (N = 569), high-grade gliomas (N = 314), embryonal tumors (N = 229), ependymomas (N = 65), tumors of sellar region (N = 51), mesenchymal non-meningothelial tumors (N = 21), glialneuronal tumors (N = 20), and chordomas (N = 12). For methods, see **STAR Methods** and **Figure S1**. 

| Broad histology                               | molecular subtype                        | n    |
|-----------------------------------------------|------------------------------------------|------|
| Chordoma                                      | CHDM, conventional                       | 4    |
| Chordoma                                      | CHDM, poorly differentiated              | 8    |
| Diffuse astrocytic and oligodendroglial tumor | DMG, H3 K28                              | 54   |
| Diffuse astrocytic and oligodendroglial tumor | DMG, H3 K28, TP53 activated              | 26   |
| Diffuse astrocytic and oligodendroglial tumor | DMG, H3 K28, TP53 loss                   | 91   |
| Diffuse astrocytic and oligodendroglial tumor | HGG, H3 G35                              | 7    |
| Diffuse astrocytic and oligodendroglial tumor | HGG, H3 G35, TP53 loss                   | 2    |
| Diffuse astrocytic and oligodendroglial tumor | HGG, H3 wildtype                         | 74   |
| Diffuse astrocytic and oligodendroglial tumor | HGG, H3 wildtype, TP53 activated         | 10   |
| Diffuse astrocytic and oligodendroglial tumor | HGG, H3 wildtype, TP53 loss              | 45   |
| Diffuse astrocytic and oligodendroglial tumor | HGG, IDH, TP53 activated                 | 3    |
| Diffuse astrocytic and oligodendroglial tumor | HGG, IDH, TP53 loss                      | 2    |
| Embryonal tumor                               | CNS Embryonal, NOS                       | 24   |
| Embryonal tumor                               | CNS HGNET-MN1                            | 1    |
| Embryonal tumor                               | CNS NB-FOXR2                             | 5    |
| Embryonal tumor                               | ETMR, C19MC-altered                      | 8    |
| Embryonal tumor                               | ETMR, NOS                                | 1    |
| Embryonal tumor                               | MB, Group3                               | 24   |
| Embryonal tumor                               | MB, Group4                               | 91   |
| Embryonal tumor                               | MB, SHH                                  | 55   |
| Embryonal tumor                               | MB, WNT                                  | 20   |
| Ependymal tumor                               | EPN, H3 K28                              | 2    |
| Ependymal tumor                               | EPN, PF A                                | 6    |
| Ependymal tumor                               | EPN, ST RELA                             | 51   |
| Ependymal tumor                               | EPN, ST YAP1                             | 6    |
| Low-grade astrocytic tumor                    | GNG, BRAF V600E                          | 25   |
| Low-grade astrocytic tumor                    | GNG, BRAF V600E, CDKN2A/B                | 2    |
| Low-grade astrocytic tumor                    | GNG, FGFR                                | 2    |
| Low-grade astrocytic tumor                    | GNG, H3                                  | 2    |
| Low-grade astrocytic tumor                    | GNG, IDH                                 | 4    |
| Low-grade astrocytic tumor                    | GNG, KIAA1549-BRAF                       | 10   |
| Low-grade astrocytic tumor                    | GNG, MYB/MYBL1                           | 2    |
| Low-grade astrocytic tumor                    | GNG, NF1-germline                        | 2    |
| Low-grade astrocytic tumor                    | GNG, NF1-somatic, BRAF V600E             | 1    |
| Low-grade astrocytic tumor                    | GNG, other MAPK                          | 7    |
| Low-grade astrocytic tumor                    | GNG, other MAPK, IDH                     | 2    |
| Low-grade astrocytic tumor                    | GNG, RTK                                 | 6    |
| Low-grade astrocytic tumor                    | GNG, wildtype                            | 28   |
| Low-grade astrocytic tumor                    | LGG, BRAF V600E                          | 53   |
| Low-grade astrocytic tumor                    | LGG, BRAF V600E, CDKN2A/B                | 10   |
| Low-grade astrocytic tumor                    | LGG, FGFR                                | 16   |
| Low-grade astrocytic tumor                    | LGG, IDH                                 | 6    |
| Low-grade astrocytic tumor                    | LGG, KIAA1549-BRAF                       | 222  |
| Low-grade astrocytic tumor                    | LGG, KIAA1549-BRAF, other MAPK           | 2    |
| Low-grade astrocytic tumor                    | LGG, MYB/MYBL1                           | 4    |
| Low-grade astrocytic tumor                    | LGG, NF1-germline                        | 12   |
| Low-grade astrocytic tumor                    | LGG, NF1-germline, CDKN2A/B              | 2    |
| Low-grade astrocytic tumor                    | LGG, NF1-germline, FGFR                  | 4    |
| Low-grade astrocytic tumor                    | LGG, NF1-somatic                         | 4    |
| Low-grade astrocytic tumor                    | LGG, NF1-somatic, FGFR                   | 2    |
| Low-grade astrocytic tumor                    | LGG, NF1-somatic, NF1-germline, CDKN2A/B | 2    |
| Low-grade astrocytic tumor                    | LGG, other MAPK                          | 23   |
| Low-grade astrocytic tumor                    | LGG, RTK                                 | 22   |
| Low-grade astrocytic tumor                    | LGG, RTK, CDKN2A/B                       | 2    |
| Low-grade astrocytic tumor                    | LGG, wildtype                            | 84   |
| Low-grade astrocytic tumor                    | SEGA, wildtype                           | 6    |
| Mesenchymal non-meningothelial tumor          | EWS                                      | 21   |
| Neuronal and mixed neuronal-glial tumor       | CNC                                      | 4    |
| Neuronal and mixed neuronal-glial tumor       | EVN                                      | 2    |
| Neuronal and mixed neuronal-glial tumor       | GNT, BRAF V600E                          | 2    |
| Neuronal and mixed neuronal-glial tumor       | GNT, KIAA1549-BRAF                       | 4    |
| Neuronal and mixed neuronal-glial tumor       | GNT, other MAPK                          | 2    |
| Neuronal and mixed neuronal-glial tumor       | GNT, other MAPK, FGFR                    | 2    |
| Neuronal and mixed neuronal-glial tumor       | GNT, RTK                                 | 4    |
| Tumors of sellar region                       | CRANIO, ADAM                             | 51   |
|                                               | Total                                    | 1281 |


Table 1: Molecular subtypes determined across OpenPBTA samples. {#tbl:subtypes}


### Somatic Mutational Landscape of Pediatric Brain Tumors

We  performed a comprehensive genomic analysis of somatic SNVs, CNVs, SVs, and fusions across 1,969 tumors (N = 1,019 RNA-Seq, N = 1,719 WGS, N = 64 WXS/Panel) and 38 cell lines (N = 16 RNA-Seq, N = 22 WGS) from 943 patients.
Following SNV consensus calling (**Figure S1** and **Figure S2, panels A-G**), we observed lower expected tumor mutation burden (TMB) **Figure S2, panel H** in pediatric tumors compared to adult brain tumors from The Cancer Genome Atlas (TCGA), **Figure S2, panel I**, with hypermutant (> 10 Mut/Mb) and ultra-hypermutant (> 100 Mut/Mb) tumors [@doi:10.1016/j.cell.2017.09.048] only found within HGATs.  

#### Low-grade astrocytic tumors
**Figure {@fig:Fig2}A** depicts an oncoprint of driver genes for 227 primary low-grade astrocytic tumors. As expected, the majority (62%, 140/227) of these tumors harbor a somatic alteration in _BRAF_, with canonical _BRAF::KIAA1549_ fusions as the major oncogenic driver. 
We observed additional mutations in _FGFR1_ (2%), _PIK3CA_ (2%), _KRAS_ (2%), _TP53_ (1%), and _ATRX_ (1%) and fusions in _NTRK2_ (2%), _RAF1_ (2%), _MYB_ (1%), _QKI_ (1%), _ROS1_ (1%), and _FGFR2_ (1%), concordant with previous studies reporting the near universal upregulation of the RAS/MAPK pathway in these tumors resulting from activating mutations and/or oncogenic fusions [@doi:10.1186/s40478-020-00902-z; @doi:10.1016/j.ccell.2020.03.011]. 
Indeed, we observed significant upregulation (ANOVA p < 0.01) of the KRAS signaling pathway in LGATs (**Figure {@fig:Fig4}B**).

#### Embryonal tumors
**Figure {@fig:Fig2}B** shows the mutational landscape for 128 primary embryonal tumors. 
The majority (N = 95) are medulloblastomas and span the spectrum of molecular subtypes: WNT, SHH, Group3, and Group 4 (see **Molecular Subtyping of CNS Tumors**), with their canonical mutations. 
We detected canonical _SMARCB1/SMARCA4_ deletions or inactivating mutations in atypical teratoid rhabdoid tumors (ATRTs) and C19MC amplification in the embryonal tumors with multilayer rosettes (ETMRs) [@doi:10.1007/s00401-020-02182-2; @doi:10.1093/neuonc/noab178; @doi:10.1186/s40478-020-00984-9; @doi:10.1038/nature22973].

#### Diffuse astrocytic and oligodendroglial tumors
In **Figure {@fig:Fig2}C**, we show genomic alterations in diffuse midline gliomas (DMGs, N = 34) and non-midline high-grade gliomas (N = 26) biopsied at diagnosis. 
The single oligodendroglioma sample in the OpenPBTA does not contain mutations in the genes shown and is therefore not present in this oncoprint. 
Across HGATs, we found _TP53_ (57%, 35/61) and _H3F3A_ (52%, 32/61) to be the most mutated and co-occurring genes (**Figure {@fig:Fig2}A**), followed by frequent mutations in _ATRX_ (30% 18/61). 
We found recurrent amplifications and fusions in _EGFR_, _MET_, _PDGFRA_, and _KIT_, highlighting that these tumors utilize multiple oncogenic mechanisms to activate tyrosine kinases, as has been previously reported [@doi:10.1007/s12254-021-00680-x; @doi:10.1002/ijc.32258; @doi:10.1016/j.ccell.2017.08.017; @doi:10.1186/s40478-020-00905-w]. 
Gene set enrichment analysis showed upregulation (ANOVA p < 0.01) of DNA repair, G2M checkpoint, and MYC pathways as well as downregulation of the TP53 pathway (**Figure {@fig:Fig4}B**).
The two tumors with ultra-high tumor mutation burden (TMB) (> 100 Mutations/Mb) were from patients with known mismatch repair deficiency syndrome [@doi:10.1093/neuonc/noz192].

#### Other CNS tumors
**Figure {@fig:Fig2}D** depicts an oncoprint for the remaining primary CNS tumors (N = 195). 
We observed 25% (15/60) of ependymoma tumors to be _C11orf95::RELA_ (now, _ZFTA::RELA_) fusion positive ependymomas and 70% (21/30) of craniopharyngiomas to be driven by mutations in _CTNNB1_. 
Multiple histologies contained somatic mutations or fusions in _NF2_: 41% (7/17) of meningiomas, 5% (3/60) of ependymomas,  and 27% (3/11) schwannomas. 
Rare fusions in _ERBB4_, _YAP1_, _KRAS_, and _MAML2_ were observed in 10% (6/60) of ependymoma tumors. 
DNETs harbored alterations in MAPK/PI3K pathway genes as previously reported [@doi:10.1093/jnen/nlz101], including _FGFR1_ (21%, 4/19), _PDGFRA_ (10%, 2/19), and _BRAF_ (5%, 1/19).
Frequent mutations in rarer brain tumor histologies (N < 5) are depicted in **Figure {@fig:FigS3}A**.

![**Figure 2. Mutational landscape of PBTA tumors.** Shown are frequencies of canonical somatic gene mutations, CNVs, fusions, and TMB (top bar plot) for the top 20 genes mutated across primary tumors within the OpenPBTA dataset. A, Low-grade astrocytic tumors (N = 227): low-grade glioma astrocytoma (N = 187), ganglioglioma (N = 35), subependymal giant cell astrocytoma (N = 2), diffuse fibrillary astrocytoma (N = 1), pilocytic astrocytoma (N = 1), and pleomorphic xanthoastrocytoma (N = 1); B, Embryonal tumors (N = 128): medulloblastomas (N = 95), atypical teratoid rhabdoid tumors (N = 24), embryonal tumors with multilayer rosettes (N = 2), other CNS embryonal tumors (N = 5), ganglioneuroblastoma (N = 1), and CNS neuroblastoma (N = 1); C, Diffuse astrocytic and oligodendroglial tumors (N = 61): diffuse midline gliomas (N = 34) and non-midline high-grade gliomas (N = 26), oligodendroglioma (N = 1); D, Other CNS tumors (N = 195): ependymomas (N = 60), dysembryoplastic neuroepithelial tumors (N = 19), meningiomas (N = 17), schwannoma (N = 11), neurofibroma plexiform (N = 7). Other CNS tumors with N < 5 are displayed in **Figure {@fig:FigS3}A**. Patient sex (`germline_sex_estimate`) and tumor histology (`cancer_group`) are displayed as annotations at the bottom of each plot. Only samples with mutations in the listed genes are shown. Multiple CNVs are denoted as a complex event.](images/main/Figure2.png){#fig:Fig2}


### Mutational co-occurrence and signatures highlight key oncogenic drivers

The top 50 mutated genes in primary tumors are shown **Figure {@fig:Fig3}** by tumor type (**A**, bar plots), with co-occurrence scores illustrated in the heatmap (**B**). 
We observed _TP53_ to be the most frequently mutated gene across OpenPBTA tumors (8.4%, 56/666), significantly co-occurring with _H3F3A_ (OR = 32, 95% CI: 15.3 - 66.7, q = 8.46e-17), _ATRX_ (OR = 20, 95% CI: 8.4 - 47.7, q = 4.43e-8), _NF1_ (OR = 8.62, 95% CI: 3.7 - 20.2, q = 5.45e-5), and _EGFR_ (OR = 18.2, 95% CI: 5 - 66.5, q = 1.6e-4). 
Other canonical cancer driver genes were frequently mutated: _BRAF_, _H3F3A_, _CTNNB1_, _NF1_, _ATRX_, _FGFR1_, and _PIK3CA_. 
Although LGG and embryonal tumors make up the majority of tumor types within the OpenPBTA, most of the significant gene interactions stem from HGATs (N = 847/872). 
At the broad histology level, _CTNNB1_ significantly co-occurs with _TP53_ (OR = 42.9, 95% CI: 7 - 261.4, q = 1.63e-3) and _DDX3X_ (OR = 21.1, 95% CI: 4.6 - 96.3, q = 4.46e-3) in embryonal tumors, _FGFR1_ and _PIK3CA_ significantly co-occur in LGGs (OR = 76.1, 95% CI: 9.85 - 588.1, q = 3.26e-3), consistent with previous findings [@doi:10.1200/JCO.2010.31.1670; 10.1186/s40478-020-01027-z].
_TP53_ and _PPM1D_ mutations have been shown to be mutually exclusive in HGATs, and our data recapitulates that trend (52/54 or 96.3% of tumors have a mutation in either gene, OR = 0.188, 95% CI: 0.04 - 0.94, p = 4.13e-2, q = 5.87e-2) [@doi:10.1038/ng.2938].
We next assessed the contributions of eight previously identified adult CNS-specific mutational signatures [@doi:10.1038/s43018-020-0027-5] (RefSig) across cancer groups **Figure {@fig:Fig3}E** and samples **Figure S4A**.
Stage 0 and/or 1 tumors characterized by low TMBs **Figure S2G** such as LGGs, gangliogliomas, craniopharyngiomas, DNETs, and schwannomas are expectedly dominated by Signature 1, which results from the normal process of spontaneous deamination of 5-methylcytosine.
Signature N6 is CNS-specific signature which we observe nearly universally across samples.
Drivers of Signature 18, _TP53_, _APC_, _NOTCH1_ (https://signal.mutationalsignatures.com/explore/referenceCancerSignature/31/drivers), are also canonical drivers of medulloblastoma tumors, and indeed, we observe Signature 18 as the most common signature in medulloblastoma tumors.
Signatures 3, 8, 18, and MMR2 are prevalent in HGGs, including DMGs.
Finally, we observe that the weight of Signature 1 is higher at diagnosis (pre-treatment) and is almost always lower in tumors at later phases of therapy (progression, recurrence, post-mortem, secondary malignancy) **Figure S4B**.
This trend may be the result of therapy-induced mutations which produce additional signatures (e.g., temozolomide treatment drives Signature 11), subclonal expansion, and/or acquisition of additional driver mutations during tumor progression, leading to higher overall TMBs and additional signatures.

<!-- TODO: possibly explore gene mutations driving signature 8, specifically for meningiomas, and describe -->

### Genomic instability of pediatric brain tumors 

We developed a method for generating consensus copy number altered regions from MantaSV, CNVkit, and ControlFREEC (**STAR Methods**). 
This resulted in high-confidence gains, losses, amplifications, and deep deletion calls used as input for breakpoint density (**Table S2**) and chromothripsis analysis.
We observed that HGATs, followed by medulloblastomas, have the most unstable genomes (**Figure S3A**).
By contrast, craniopharyngiomas generally lack somatic copy number variation.
These patterns of copy number variation largely align with estimates of tumor mutational burden (**Figure S2H**). 
The number of SV and CNV breakpoints were significantly correlated across tumors (p = 1.08e-37) (**Figure {@fig:Fig3}C**) and as expected, the number of chromothripsis regions called increases as breakpoint density increases (**Figure S3B-C**).
Chromothripsis events were observed in 41% (N = 19/46) of non-midline high-grade gliomas and 28.2% (N = 11/39) of DMGs (**Figure {@fig:Fig3}D**). 
We also found evidence of chromothripsis in over 15% of embryonal tumors, ependymomas, meningiomas, germinomas, glial-neuronal tumors, chordomas, metastatic secondary tumors, and sarcomas, highlighting the genomic instability and complexity of pediatric brain tumors.

![**Figure 3. Mutational co-occurrence and signatures highlight key oncogenic drivers.** A, Bar plot of occurrence and co-occurrence of nonsynonymous mutations for the 50 most commonly mutated genes across all tumor types (annotated from `cancer_group` if N >= 10 or `Other` if N < 10); B, Co-occurrence and mutual exclusivity of nonsynonymous mutations between genes; The co-occurrence score is defined as $I(-\log_{10}(P))$ where $P$ is defined by Fisher's exact test and $I$ is 1 when mutations co-occur more often than expected and -1 when exclusivity is more common; C, The number of SV breaks significantly correlate with CNV breaks (Adjusted R = 0.436, p = 1.08e-37). D, Chromothripsis frequency across pediatric brain tumors shown by `cancer_group` with N >= 3. E, Sina plots of RefSig signature weights for signatures 1, 3, 8, 11, 18, 19, N6, MMR2, and Other across cancer groups. Box plot lines represent the first quartile, median, and third quartile. ](images/main/Figure3.png){#fig:Fig3}

### Transcriptomic Landscape of Pediatric Brain Tumors

#### Prediction of _TP53_ oncogenicity and telomerase activity
To understand the _TP53_ phenotype in each tumor, we ran a classifier previously trained on TCGA [@doi:10.1016/j.celrep.2018.03.076] to infer _TP53_ inactivation status.
Using high-confidence SNVs, CNVs, SVs, and fusions in _TP53_ as true positive alterations, we achieved a high accuracy (AUROC = 0.85) for rRNA-depleted, stranded samples compared to randomly shuffled _TP53_ scores (**Figure {@fig:Fig5}A**). 
The classifier did not perform well on the poly-A samples (**Figure S5A**), potentially due to the low number of _TP53_ altered (N = 29) and/or total poly-A samples in our dataset (N = 58) rather than library type, as a previous study demonstrated high accuracy of this classifier on another poly-A dataset [@doi:10.1016/j.celrep.2019.09.071].
We annotated _TP53_ alterations as "activated" if samples harbored one of p.R273C or p.R248W mutations [@doi:10.1038/ng0593-42], "loss" if the patient had a Li Fraumeni Syndrome (LFS) predisposition diagnosis or if any other SNV, CNV, SV, or fusion fell within a functional domain.
If the _TP53_ mutation did not reside within the DNA-binding domain or if an alteration was not detected in _TP53_, we annotated the tumor as "other".
Interestingly, we observed that samples annotated as either "activated" or "loss" had significantly higher _TP53_ scores than those annotated as "other" (**Figure {@fig:Fig5}B**, p<sub>adj</sub> loss vs. other < 2e-16, p<sub>adj</sub> activated vs. other = 4.0e-5), suggesting that the classifier detects an oncogenic, or altered, _TP53_ phenotype (scores > 0.5) rather than solely _TP53_ inactivation, as interpreted previously [@doi:10.1016/j.celrep.2018.03.076].
Moreover, tumors with "activating" _TP53_ mutations had evidence of higher _TP53_ expression than those with _TP53_ "loss" mutations (Wilcoxon p = 3.5e-3, **Figure {@fig:Fig5}C**.
To further validate the classifier's accuracy, we assessed _TP53_ scores for patients with LFS, hypothesizing all of these tumors would have high scores.
Indeed, we observed higher scores in LFS tumors (N = 8) for which we detected high-confidence _TP53_ somatic alterations. 
Although we were unable to detect canonical somatic _TP53_ mutations in two LFS patient tumors with low _TP53_ scores, we confirmed the LFS diagnosis from pathology reports and found each to have a germline variant in _TP53_.
The tumor purity of these samples was low (16% and 37%), suggesting the classifier requires a certain level of tumor purity to achieve good performance, as we expect _TP53_ to be intact in normal cells.
Tumors with the highest median _TP53_ scores were those known to harbor somatic _TP53_ alterations: choroid plexus tumors, embryonal tumors, HGATs, and pineal tumors (**Figure {@fig:Fig5}E**), while melanocytic tumors, meningiomas, and tumors of cranial and paraspinal nerves had the lowest scores.
	
We next used gene expression data to predict telomerase activity using EXpression-based Telomerase ENzymatic activity Detection (EXTEND) [@doi:10.1038/s41467-020-20474-9] as a surrogate measure of malignant potential [@doi:10.1038/s41467-020-20474-9; 10.1093/carcin/bgp268].
EXTEND scores significantly correlated with _TERT_ (R = 0.55) and _TERC_ (R = 0.58) expression (**Figure S5B-C**).  
We found aggressive tumors such as CNS lymphoma, ETMR, ATRT, DMG, and HGG had high EXTEND scores (**Figure {@fig:Fig4}D**), while benign lesions such as teratomas, dysplasias, and hemangioblastomas had the lowest scores (**Table S3**).

![**Figure 4. _TP53_ and telomerase activity** A, Receiver Operating Characteristic for _TP53_ classifier run on FPKM of stranded RNA-Seq samples. B, Violin and box plots of _TP53_ scores plotted by _TP53_ alteration type. C, Violin and box plots of _TP53_ RNA expression plotted by _TP53_ activation status. D, Box plots of _TP53_ and telomerase (EXTEND) scores grouped by `cancer_group`. Mutation status is highlighted in orange (hypermutant) or red (ultra-hypermutant). E, Heatmap of RefSig mutational signatures for patients who have least one tumor with a TMB >= 10 Mut/Mb.](images/main/Figure4.png){#fig:Fig4}


#### Histologic and oncogenic pathway clustering

UMAP visualization of gene expression variation across brain tumors (**Figure {@fig:Fig5}A**) shows the expected clustering of brain tumors by histology.
We observed medulloblastomas cluster by molecular subtype with WNT and SHH in distinct clusters and Groups 3 and 4 showing some overlap (**Figure S5D**), as expected.
Of note, two samples annotated as the SHH subtype do not cluster with the MB samples and one clusters with Group 3 and 4 samples, suggesting potential subtype misclassification or different underlying biology of these tumors.
Additionally, except for three outliers, _C11orf95::RELA_ (_ZFTA::RELA_) fusion positive ependymomas fall within distinct clusters (**Figure S5E**).
_BRAF_-driven low-grade gliomas (**Figure S5F**) were present in three separate clusters, suggesting that there might be distinct underlying biology within these tumors.
Histone H3 G35-mutant HGATs generally clustered together, away from K28-mutant tumors (**Figure S5G**).
Interestingly, although H3 K28-mutant tumors have different biological drivers than H3 wildtype tumors, they did not form distinct clusters, suggesting they may either be driven by common transcriptional programs or our sample size is too small to detect transcriptional differences.

We performed gene set variant analysis (GSVA) for Hallmark cancer gene sets, demonstrating activation of underlying oncogenic pathways (**Figure {@fig:Fig5}B**).	

<!-- TODO: Add immune deconvolution figure 4C information. -->

![**Figure 5.** A, First two dimensions from UMAP of sample transcriptome data. Points are colored by `cancer_group` of the samples they represent. B, Heatmap of GSVA scores for Hallmark gene sets with significant differences, with samples ordered by `cancer_group`. C, QuanTIseq estimates of immune cell proportions shown by tumor molecular subtype.](images/main/Figure5.png){#fig:Fig5}
