## Results

### Crowd-sourced Somatic Analysis to create an Open Pediatric Brain Tumor Atlas

We previously performed whole genome sequencing (WGS), whole exome sequencing (WXS), and RNA sequencing (RNA-Seq) on matched tumor and normal tissues as well as selected cell lines from 943 patient tumors from the Pediatric Brain Tumor Atlas (PBTA) [@doi:10.1093/neuonc/noz192], consisting of samples from the [Children’s Brain Tumor Network (CBTN)](https://CBTN.org) and the PNOC003 DMG clinical trial [@doi:10.1002/ijc.32258] of the Pacific Pediatric Neuro-Oncology Symposium (PNOC) (**Figure {@fig:Fig1}A**). 
We then harnessed the benchmarking efforts of the Kids First Data Resource Center to develop a robust and reproducible data analysis workflow within the CAVATICA platform to perform the following primary somatic analyses: variant calling of single nucleotide variants (SNVs), copy number variants (CNVs), structural variants (SVs), and fusions (**Figure {@fig:S1}**) - red boxes and **STAR Methods**). 
We maintained a data release folder on Amazon S3 containing merged files for each analysis, downloadable from the GitHub repository or the open-access CAVATICA project (See **Data and code availability** section).
As new analytical products (e.g., tumor mutation burden calculations) that we expected to be used across multiple analyses were produced, or issues with the data were identified, new releases were made available in a versioned manner.

A key innovative feature of this project was the open contribution model used for both analyses (i.e., analytical code) and scientific manuscript writing. 
We created a public Github analysis repository (https://github.com/AlexsLemonade/OpenPBTA-analysis) to hold all code associated with analyses downstream of the Kids First Data Resource Center workflows and a GitHub manuscript repository (https://github.com/AlexsLemonade/OpenPBTA-manuscript) with ManuBot [@doi:10.1371/journal.pcbi.1007128] integration to enable real-time manuscript creation using Markdown within GitHub.

Additions to the manuscript often started with proposing a specific analysis in the analysis repository (for example, immune cell deconvolution of bulk RNA-Seq data).
First, a potential contributor proposed an analysis by filing an issue on the GitHub analysis repository. 
Next, organizers for the project, or other contributors with expertise, had the opportunity to comment on the proposal (**Figure {@fig:Fig1}C**). 
The contributor then forked the analysis repository (or make a copy for the purpose of adding code changes) and added their proposed analysis to their fork. 
They then filed a pull request on GitHub, where they requested to add their proposed changes to the main repository. 
All pull requests to the analysis repository underwent peer review by organizers and/or other contributors. 
Analytical code reviews conducted by peer reviewers checked for analysis accuracy (**Figure {@fig:Fig1}C-D**) and maintainability as well as readability of code and documentation. 

The collaborative nature of the project required additional steps beyond peer review of analytical code (**Figure {@fig:Fig1}D**).
Specifically, since multiple contributors were working on the code base and months would sometimes pass between updates to specific analysis modules, it was important to maintain a consistent software development environment to ensure that code could be revised. 
We leveraged Docker® and the Rocker project [@arxiv:1710.03675] to create a monolithic image that contained all dependencies necessary for analyses in the analysis repository. 
To ensure that the image would build and that code would execute over time, we used the continuous integration (CI) service CircleCI® to run analytical code on a small subset of data for testing, which allowed us to detect any sensitivity to changes in the underlying data prior to code review (**Figure {@fig:Fig1}D**).

We followed a similar process in our Manubot-powered [@doi:10.1371/journal.pcbi.1007128] manuscript repository for additions to the manuscript (**Figure {@fig:Fig1}C**). 
Contributors forked the manuscript repository, added proposed content to their branch, and filed pull requests to the main manuscript repository with their changes.
Similarly, pull requests underwent a peer review process for clarity and correctness, agreement with interpretation, and spell checking via Manubot (**Figure {@fig:Fig1}C**).

![**Overview of the OpenPBTA Project.** A, The Children's Brain Tumor Network and the Pacific Pediatric Neuro-Oncology Consortium collected tumor samples from 943 patients. To date, 38 cell lines were created from tumor tissue, and over 2000 specimens were sequenced (N = 1035 RNA-Seq, N = 940 WGS, and N = 32 WXS or Targeted panel). Data was harmonized by the Kids First Data Resource Center using an Amazon S3 framework within CAVATICA. B, Stacked bar plot summary of the number of biospecimens per phase of therapy per broad histology (GNG = ganglioglioma, LGG = low-grade glioma, DIPG = diffuse intrinsic pontine glioma, DMG = diffuse midline glioma, EPN = ependymoma, HGG = high-grade glioma, ATRT = atypical teratoid rhabdoid tumor, CNS-ET = CNS embryonal tumor, MB = medulloblastoma, PNF = plexiform neurofibroma, DNET = dysembryoplastic neuroepithelial tumor, CRANIO = craniopharyngioma, EWS = Ewing sarcoma, CPP = choroid plexus papilloma). C, Overview of the open analysis and manuscript contribution model. In the analysis GitHub repository, a contributor would propose an analysis that other participants can comment on. Contributors would then implement the analysis and file a request to add their changes to the analysis repository ("pull request"). Pull requests underwent review for scientific rigor and correctness of implementation. Pull requests were additionally checked to ensure that all software dependencies were included and the code was not sensitive to underlying data changes using container and continuous integration technologies. Finally, a contributor would file a pull request documenting their methods and results to the Manubot-powered manuscript repository. Pull requests in the manuscript repository were also subject to review. D, A potential path for an analytical pull request. Arrows indicate revisions to a pull request. Prior to review, a pull request was tested for dependency installation and whether or not the code would execute. Pull requests also required approval by organizers and/or other contributors, who checked for scientific correctness. Panel A created with BioRender.com.](https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/7d0e1c8b60fe258d3d7a61645f3ba8ba57025d3c/figures/pngs/figure1.png?sanitize=true){#fig:Fig1 width="7in"}

### Molecular Subtyping of OpenPBTA CNS Tumors

Over the past two decades, together with the World Health Organization (WHO), experts in neurooncology have iteratively redefined the classifications of central nervous system (CNS) tumors [@doi:10.1093/jnen/61.3.215; @doi:10.1007/s00401-007-0243-4]. 
More recently, in 2016 and 2021 [@doi:10.1007/s00401-016-1545-1; @doi:10.1093/neuonc/noab106], molecular subtypes have been integrated into these entities. 
The Children’s Brain Tumor Tissue Consortium (CBTTC), currently the Children’s Brain Tumor Network (CBTN), opened its protocol for brain tumor and matched normal sample collection in 2011 and as such, the majority of the samples within the OpenPBTA dataset lack molecular subtype annotation. 
In the absence of methylation arrays commonly used for tumor classification, we coupled key genomic features of tumor entities described by the WHO in 2016 and low-grade glioma (LGG) subtypes described by Ryall and colleagues [@doi:10.1016/j.ccell.2020.03.011] with clinician and pathologist review to subtype 64% (1,281/2,007) of tumor biospecimens with high confidence (**Table S1**).
Importantly, this collaborative molecular subtyping process allowed us to identify data entry errors (e.g., an ETMR entered as a medulloblastoma) and mis-identified specimens (e.g., Ewing sarcoma sample labeled as a craniopharyngioma), update diagnoses using modern terms (e.g., primitive neuro-ectodermal tumor [PNET] diagnoses), and discover rarer tumor entities within the OpenPBTA (e.g., H3-mutant ependymoma, meningioma with _YAP1::FAM118B_ fusion).
**Table 1** lists the subtypes we defined within the OpenPBTA, comprising of LGATs (N = 569), high-grade gliomas (N = 314), embryonal tumors (N = 229), ependymomas (N = 65), tumors of sellar region (N = 51), mesenchymal non-meningothelial tumors (N = 21), glialneuronal tumors (N = 20), and chordomas (N = 12). 
or detailed methods, see **STAR Methods** and **Figure {@fig:S1}**. 

| Broad histology                               | OpenPBTA molecular subtype                        | n    |
|-----------------------------------------------|------------------------------------------|------|
| Chordoma                                      | CHDM, conventional                       | 4    |
| Chordoma                                      | CHDM, poorly differentiated              | 8    |
| Diffuse astrocytic and oligodendroglial tumor | DMG, H3 K28                              | 54   |
| Diffuse astrocytic and oligodendroglial tumor | DMG, H3 K28, TP53 activated              | 26   |
| Diffuse astrocytic and oligodendroglial tumor | DMG, H3 K28, TP53 loss                   | 91   |
| Diffuse astrocytic and oligodendroglial tumor | HGG, H3 G35                              | 7    |
| Diffuse astrocytic and oligodendroglial tumor | HGG, H3 G35, TP53 loss                   | 2    |
| Diffuse astrocytic and oligodendroglial tumor | HGG, H3 wildtype                         | 74   |
| Diffuse astrocytic and oligodendroglial tumor | HGG, H3 wildtype, TP53 activated         | 10   |
| Diffuse astrocytic and oligodendroglial tumor | HGG, H3 wildtype, TP53 loss              | 45   |
| Diffuse astrocytic and oligodendroglial tumor | HGG, IDH, TP53 activated                 | 3    |
| Diffuse astrocytic and oligodendroglial tumor | HGG, IDH, TP53 loss                      | 2    |
| Embryonal tumor                               | CNS Embryonal, NOS                       | 24   |
| Embryonal tumor                               | CNS HGNET-MN1                            | 1    |
| Embryonal tumor                               | CNS NB-FOXR2                             | 5    |
| Embryonal tumor                               | ETMR, C19MC-altered                      | 8    |
| Embryonal tumor                               | ETMR, NOS                                | 1    |
| Embryonal tumor                               | MB, Group3                               | 24   |
| Embryonal tumor                               | MB, Group4                               | 91   |
| Embryonal tumor                               | MB, SHH                                  | 55   |
| Embryonal tumor                               | MB, WNT                                  | 20   |
| Ependymal tumor                               | EPN, H3 K28                              | 2    |
| Ependymal tumor                               | EPN, PF A                                | 6    |
| Ependymal tumor                               | EPN, ST RELA                             | 51   |
| Ependymal tumor                               | EPN, ST YAP1                             | 6    |
| Low-grade astrocytic tumor                    | GNG, BRAF V600E                          | 25   |
| Low-grade astrocytic tumor                    | GNG, BRAF V600E, CDKN2A/B                | 2    |
| Low-grade astrocytic tumor                    | GNG, FGFR                                | 2    |
| Low-grade astrocytic tumor                    | GNG, H3                                  | 2    |
| Low-grade astrocytic tumor                    | GNG, IDH                                 | 4    |
| Low-grade astrocytic tumor                    | GNG, KIAA1549-BRAF                       | 10   |
| Low-grade astrocytic tumor                    | GNG, MYB/MYBL1                           | 2    |
| Low-grade astrocytic tumor                    | GNG, NF1-germline                        | 2    |
| Low-grade astrocytic tumor                    | GNG, NF1-somatic, BRAF V600E             | 1    |
| Low-grade astrocytic tumor                    | GNG, other MAPK                          | 7    |
| Low-grade astrocytic tumor                    | GNG, other MAPK, IDH                     | 2    |
| Low-grade astrocytic tumor                    | GNG, RTK                                 | 6    |
| Low-grade astrocytic tumor                    | GNG, wildtype                            | 28   |
| Low-grade astrocytic tumor                    | LGG, BRAF V600E                          | 53   |
| Low-grade astrocytic tumor                    | LGG, BRAF V600E, CDKN2A/B                | 10   |
| Low-grade astrocytic tumor                    | LGG, FGFR                                | 16   |
| Low-grade astrocytic tumor                    | LGG, IDH                                 | 6    |
| Low-grade astrocytic tumor                    | LGG, KIAA1549-BRAF                       | 222  |
| Low-grade astrocytic tumor                    | LGG, KIAA1549-BRAF, other MAPK           | 2    |
| Low-grade astrocytic tumor                    | LGG, MYB/MYBL1                           | 4    |
| Low-grade astrocytic tumor                    | LGG, NF1-germline                        | 12   |
| Low-grade astrocytic tumor                    | LGG, NF1-germline, CDKN2A/B              | 2    |
| Low-grade astrocytic tumor                    | LGG, NF1-germline, FGFR                  | 4    |
| Low-grade astrocytic tumor                    | LGG, NF1-somatic                         | 4    |
| Low-grade astrocytic tumor                    | LGG, NF1-somatic, FGFR                   | 2    |
| Low-grade astrocytic tumor                    | LGG, NF1-somatic, NF1-germline, CDKN2A/B | 2    |
| Low-grade astrocytic tumor                    | LGG, other MAPK                          | 23   |
| Low-grade astrocytic tumor                    | LGG, RTK                                 | 22   |
| Low-grade astrocytic tumor                    | LGG, RTK, CDKN2A/B                       | 2    |
| Low-grade astrocytic tumor                    | LGG, wildtype                            | 84   |
| Low-grade astrocytic tumor                    | SEGA, wildtype                           | 6    |
| Mesenchymal non-meningothelial tumor          | EWS                                      | 21   |
| Neuronal and mixed neuronal-glial tumor       | CNC                                      | 4    |
| Neuronal and mixed neuronal-glial tumor       | EVN                                      | 2    |
| Neuronal and mixed neuronal-glial tumor       | GNT, BRAF V600E                          | 2    |
| Neuronal and mixed neuronal-glial tumor       | GNT, KIAA1549-BRAF                       | 4    |
| Neuronal and mixed neuronal-glial tumor       | GNT, other MAPK                          | 2    |
| Neuronal and mixed neuronal-glial tumor       | GNT, other MAPK, FGFR                    | 2    |
| Neuronal and mixed neuronal-glial tumor       | GNT, RTK                                 | 4    |
| Tumors of sellar region                       | CRANIO, ADAM                             | 51   |
|                                               | Total                                    | 1281 |


Table 1: Molecular subtypes determined across OpenPBTA samples.


### Somatic Mutational Landscape of Pediatric Brain Tumors

We  performed a comprehensive genomic analysis of somatic SNVs, CNVs, SVs, and fusions across 1,969 tumors (N = 1,019 RNA-Seq, N = 1,719 WGS, N = 64 WXS/Panel) and 38 cell lines (N = 16 RNA-Seq, N = 22 WGS) from 943 patients.
Following SNV consensus calling (**Figure {@fig:S1}** and **Figure {@fig:S2}A-G**), we observed lower expected tumor mutation burden (TMB) **Figure {@fig:S2}H** in pediatric tumors compared to adult brain tumors from The Cancer Genome Atlas (TCGA), **Figure {@fig:S2}I**, with hypermutant (> 10 Mut/Mb) and ultra-hypermutant (> 100 Mut/Mb) tumors [@doi:10.1016/j.cell.2017.09.048] only found within HGGs.  

#### Low-grade astrocytic tumors
**Figure {@fig:Fig2}A** depicts an oncoprint of driver genes for 227 primary low-grade astrocytic tumors. As expected, the majority (62%, 140/227) of these tumors harbor a somatic alteration in _BRAF_, with canonical _BRAF::KIAA1549_ fusions as the major oncogenic driver. 
We observed additional mutations in _FGFR1_ (2%), _PIK3CA_ (2%), _KRAS_ (2%), _TP53_ (1%), and _ATRX_ (1%) and fusions in _NTRK2_ (2%), _RAF1_ (2%), _MYB_ (1%), _QKI_ (1%), _ROS1_ (1%), and _FGFR2_ (1%), concordant with previous studies reporting the near universal upregulation of the RAS/MAPK pathway in these tumors resulting from activating mutations and/or oncogenic fusions [@doi:10.1186/s40478-020-00902-z; @doi:10.1016/j.ccell.2020.03.011]. 
Indeed, we observed significant upregulation (ANOVA p < 0.01) of the KRAS signaling pathway in LGATs (**Figure {@fig:Fig5}B**).

#### Embryonal tumors
**Figure {@fig:Fig2}B** shows the mutational landscape for 128 primary embryonal tumors. 
The majority (N = 95) are medulloblastomas and span the spectrum of molecular subtypes: WNT, SHH, Group3, and Group 4 (see **Molecular Subtyping of CNS Tumors**), with their canonical mutations. 
We detected canonical _SMARCB1/SMARCA4_ deletions or inactivating mutations in atypical teratoid rhabdoid tumors (ATRTs) and C19MC amplification in the embryonal tumors with multilayer rosettes (ETMRs) [@doi:10.1007/s00401-020-02182-2; @doi:10.1093/neuonc/noab178; @doi:10.1186/s40478-020-00984-9; @doi:10.1038/nature22973].

#### Diffuse astrocytic and oligodendroglial tumors
In **Figure {@fig:Fig2}C**, we show genomic alterations in diffuse midline gliomas (DMGs, N = 34) and non-midline high-grade gliomas (N = 26) biopsied at diagnosis. 
The single oligodendroglioma sample in the OpenPBTA does not contain mutations in the genes shown and is therefore not present in this oncoprint. 
Across HGGs, we found _TP53_ (57%, 35/61) and _H3F3A_ (52%, 32/61) to be the most mutated and co-occurring genes (**Figure {@fig:Fig2}A**), followed by frequent mutations in _ATRX_ (30% 18/61). 
We found recurrent amplifications and fusions in _EGFR_, _MET_, _PDGFRA_, and _KIT_, highlighting that these tumors utilize multiple oncogenic mechanisms to activate tyrosine kinases, as has been previously reported [@doi:10.1002/ijc.32258; @doi:10.1016/j.ccell.2017.08.017; @doi:10.1186/s40478-020-00905-w]. 
Gene set enrichment analysis showed upregulation (ANOVA p < 0.01) of DNA repair, G2M checkpoint, and MYC pathways as well as downregulation of the TP53 pathway (**Figure {@fig:Fig5}B**).
The two tumors with ultra-high tumor mutation burden (TMB) (> 100 Mutations/Mb) were from patients with known mismatch repair deficiency syndrome [@doi:10.1093/neuonc/noz192].

#### Other CNS tumors
**Figure {@fig:Fig2}D** depicts an oncoprint for the remaining primary CNS tumors (N = 195). 
We observed 25% (15/60) of ependymoma tumors to be _C11orf95::RELA_ (now, _ZFTA::RELA_) fusion positive ependymomas and 70% (21/30) of craniopharyngiomas to be driven by mutations in _CTNNB1_. 
Multiple histologies contained somatic mutations or fusions in _NF2_: 41% (7/17) of meningiomas, 5% (3/60) of ependymomas,  and 27% (3/11) schwannomas. 
Rare fusions in _ERBB4_, _YAP1_, _KRAS_, and _MAML2_ were observed in 10% (6/60) of ependymoma tumors. 
DNETs harbored alterations in MAPK/PI3K pathway genes as previously reported [@doi:10.1093/jnen/nlz101], including _FGFR1_ (21%, 4/19), _PDGFRA_ (10%, 2/19), and _BRAF_ (5%, 1/19).
Frequent mutations in rarer brain tumor histologies (N < 5) are depicted in **Figure {@fig:S3}A**.

![**Figure 2. Mutational landscape of PBTA tumors.** Shown are frequencies of canonical somatic gene mutations, CNVs, fusions, and TMB (top bar plot) for the top 20 genes mutated across primary tumors within the OpenPBTA dataset. A, Low-grade astrocytic tumors (N = 227): low-grade glioma astrocytoma (N = 187), ganglioglioma (N = 35), subependymal giant cell astrocytoma (N = 2), diffuse fibrillary astrocytoma (N = 1), pilocytic astrocytoma (N = 1), and pleomorphic xanthoastrocytoma (N = 1); B, Embryonal tumors (N = 128): medulloblastomas (N = 95), atypical teratoid rhabdoid tumors (N = 24), embryonal tumors with multilayer rosettes (N = 2), other CNS embryonal tumors (N = 5), ganglioneuroblastoma (N = 1), and CNS neuroblastoma (N = 1); C, Diffuse astrocytic and oligodendroglial tumors (N = 61): diffuse midline gliomas (N = 34) and non-midline high-grade gliomas (N = 26), oligodendroglioma (N = 1); D, Other CNS tumors (N = 195): ependymomas (N = 60), dysembryoplastic neuroepithelial tumors (N = 19), meningiomas (N = 17), schwannoma (N = 11), neurofibroma plexiform (N = 7). Other CNS tumors with N < 5 are displayed in **Figure {@fig:S3}A**. Patient sex (`germline_sex_estimate`) and tumor histology (`cancer_group`) are displayed as annotations at the bottom of each plot. Only samples with mutations in the listed genes are shown. Multiple CNVs are denoted as a complex event.](https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/7d0e1c8b60fe258d3d7a61645f3ba8ba57025d3c/figures/pngs/figure3.png?sanitize=true){#fig:Fig3 width="7in"}


### Mutational co-occurrence, CNV and signatures highlight key oncogenic drivers

The top 50 mutated genes in primary tumors are shown **Figure {@fig:Fig3}** by tumor type (**A**, bar plots), with co-occurrence scores illustrated in the heatmap (**B**). 
We observed _TP53_ to be the most frequently mutated gene across OpenPBTA tumors (8.4%, 56/666), significantly co-occurring with _H3F3A_ (OR = 32, 95% CI: 15.3 - 66.7, q = 8.46e-17), _ATRX_ (OR = 20, 95% CI: 8.4 - 47.7, q = 4.43e-8), _NF1_ (OR = 8.62, 95% CI: 3.7 - 20.2, q = 5.45e-5), and _EGFR_ (OR = 18.2, 95% CI: 5 - 66.5, q = 1.6e-4). 
Other canonical cancer driver genes were frequently mutated: _BRAF_, _H3F3A_, _CTNNB1_, _NF1_, _ATRX_, _FGFR1_, and _PIK3CA_. 
Although LGG and embryonal tumors make up the majority of tumor types within the OpenPBTA, most of the significant gene interactions stem from HGGs (N = 847/872). 
At the broad histology level, _CTNNB1_ significantly co-occurs with _TP53_ (OR = 42.9, 95% CI: 7 - 261.4, q = 1.63e-3) and _DDX3X_ (OR = 21.1, 95% CI: 4.6 - 96.3, q = 4.46e-3) in embryonal tumors, _FGFR1_ and _PIK3CA_ significantly co-occur in LGGs (OR = 76.1, 95% CI: 9.85 - 588.1, q = 3.26e-3), consistent with previous findings [@doi:10.1200/JCO.2010.31.1670; 10.1186/s40478-020-01027-z].
_TP53_ and _PPM1D_ mutations have been shown to be mutually exclusive in HGGs, and our data recapitulates that trend (52/54 or 96.3% of tumors have a mutation in either gene, OR = 0.188, 95% CI: 0.04 - 0.94, p = 4.13e-2, q = 5.87e-2) [@doi:10.1038/ng.2938].
We summarized broad copy number and structural variation and observed that HGGs and DMGs, followed by medulloblastomas, have the most unstable genomes (**Figure {@fig:S3}A**).
By contrast, craniopharyngiomas and schwannomas generally lack somatic copy number variation.
These patterns of copy number variation largely align with estimates of tumor mutational burden (**Figure {@fig:S2}H**). 
The number of SV and CNV breakpoints were significantly correlated across tumors (p = 1.08e-37) (**Figure {@fig:Fig3}C**) and as expected, the number of chromothripsis regions called increases as breakpoint density increases (**Figure {@fig:S3}B-C**).
Chromothripsis events were observed in 41% (N = 19/46) of non-midline high-grade gliomas and 28.2% (N = 11/39) of DMGs (**Figure {@fig:Fig3}D**). 
We also found evidence of chromothripsis in over 15% of embryonal tumors, ependymomas, meningiomas, germinomas, glial-neuronal tumors, chordomas, metastatic secondary tumors, and sarcomas, highlighting the genomic instability and complexity of pediatric brain tumors.
We next assessed the contributions of eight previously identified adult CNS-specific mutational signatures [@doi:10.1038/s43018-020-0027-5] (RefSig) across cancer groups **Figure {@fig:Fig3}E** and samples **Figure {@fig:S4}A**.
Stage 0 and/or 1 tumors characterized by low TMBs **Figure {@fig:S2}G** such as LGGs, gangliogliomas, craniopharyngiomas, DNETs, and schwannomas are expectedly dominated by Signature 1, which results from the normal process of spontaneous deamination of 5-methylcytosine.
Signature N6 is CNS-specific signature which we observe nearly universally across samples.
Drivers of Signature 18, _TP53_, _APC_, _NOTCH1_ (found at https://signal.mutationalsignatures.com/explore/referenceCancerSignature/31/drivers), are also canonical drivers of medulloblastoma, and indeed, we observe Signature 18 as the most common signature in medulloblastoma tumors.
Signatures 3, 8, 18, and MMR2 are prevalent in HGGs, including DMGs.
Finally, we observe that the weight of Signature 1 is higher at diagnosis (pre-treatment) and is almost always lower in tumors at later phases of therapy (progression, recurrence, post-mortem, secondary malignancy) **Figure {@fig:S4}B**.
This trend may be the result of therapy-induced mutations which produce additional signatures (e.g., temozolomide treatment drives Signature 11), subclonal expansion, and/or acquisition of additional driver mutations during tumor progression, leading to higher overall TMBs and additional signatures.

<!-- TODO: possibly explore gene mutations driving signature 8, specifically for meningiomas, and describe -->

![**Figure 3. Mutational co-occurrence and signatures highlight key oncogenic drivers.** A, Bar plot of occurrence and co-occurrence of nonsynonymous mutations for the 50 most commonly mutated genes across all tumor types (annotated from `cancer_group` if N >= 10 or `Other` if N < 10); B, Co-occurrence and mutual exclusivity of nonsynonymous mutations between genes; The co-occurrence score is defined as $I(-\log_{10}(P))$ where $P$ is defined by Fisher's exact test and $I$ is 1 when mutations co-occur more often than expected and -1 when exclusivity is more common; C, The number of SV breaks significantly correlate with CNV breaks (Adjusted R = 0.436, p = 1.08e-37). D, Chromothripsis frequency across pediatric brain tumors shown by `cancer_group` with N >= 3. E, Sina plots of RefSig signature weights for signatures 1, 3, 8, 11, 18, 19, N6, MMR2, and Other across cancer groups. Box plot lines represent the first quartile, median, and third quartile.](7d0e1c8b60fe258d3d7a61645f3ba8ba57025d3c){#fig:Fig3}

### Transcriptomic Landscape of Pediatric Brain Tumors

#### Prediction of _TP53_ oncogenicity and telomerase activity

To understand the _TP53_ phenotype in each tumor, we ran a classifier previously trained on TCGA [@doi:10.1016/j.celrep.2018.03.076] to infer _TP53_ inactivation status.
Using high-confidence SNVs, CNVs, SVs, and fusions in _TP53_ as true positive alterations, we achieved a high accuracy (AUROC = 0.85) for rRNA-depleted, stranded samples compared to randomly shuffled _TP53_ scores (**Figure {@fig:Fig4}A**). 
The classifier did not perform well on the poly-A samples (**Figure {@fig:S5}A**), potentially due to the low number of _TP53_ altered (N = 29) and/or total poly-A samples in our dataset (N = 58) rather than library type, as a previous study demonstrated high accuracy of this classifier on another poly-A dataset [@doi:10.1016/j.celrep.2019.09.071].
We annotated _TP53_ alterations as "activated" if samples harbored one of p.R273C or p.R248W mutations [@doi:10.1038/ng0593-42], "loss" if the patient had a Li Fraumeni Syndrome (LFS) predisposition diagnosis, tumor harbors a known hotspot mutation, or contains two hits (e.g.: SNV and CNV), suggesting both alleles are affected.
If the _TP53_ mutation did not reside within the DNA-binding domain or if an alteration was not detected in _TP53_, we annotated the tumor as "other".
While we expected that samples annotated as "loss" had higher _TP53_ scores than samples annotated as "other", we observed that those annotated as "activated" had similar _TP53_ scores to those annotated as "loss" (**Figure {@fig:Fig4}B**, Wilcoxon p = 0.23), suggesting that the classifier detects an oncogenic, or altered, _TP53_ phenotype (scores > 0.5) rather than solely _TP53_ inactivation, as interpreted previously [@doi:10.1016/j.celrep.2018.03.076].
Moreover, tumors with "activating" _TP53_ mutations had evidence of higher _TP53_ expression than those with _TP53_ "loss" mutations (Wilcoxon p = 3.5e-3, **Figure {@fig:Fig4}C**.
To further validate the classifier's accuracy, we assessed _TP53_ scores for patients with LFS, hypothesizing that all of these tumors would have high scores.
Indeed, we observed higher scores in LFS tumors (N = 8) for which we detected high-confidence _TP53_ somatic alterations (**Tables S1 and S3**). 
Although we were unable to detect canonical somatic _TP53_ mutations in two LFS patient tumors with low _TP53_ scores (BS_DEHJF4C7 with a score of 0.09 and BS_ZD5HN296 with a score of 0.28, we confirmed the LFS diagnosis from pathology reports and found each to have a pathogenic germline variant in _TP53_.
The tumor purity of these two LFS samples was low (16% and 37%), suggesting the classifier requires a certain level of tumor purity to achieve good performance, as we expect _TP53_ to be intact in normal cells.
Tumors with the highest median _TP53_ scores were those known to harbor somatic _TP53_ alterations: DMGs, medulloblastomas, HGGs, DNETs, ependymomas, and craniopharyngiomas (**Figure {@fig:Fig4}D**), while gangliogliomas, LGGs, meningiomas, and schwannomas had the lowest scores.
We next used gene expression data to predict telomerase activity using EXpression-based Telomerase ENzymatic activity Detection (EXTEND) [@doi:10.1038/s41467-020-20474-9] as a surrogate measure of malignant potential [@doi:10.1038/s41467-020-20474-9; 10.1093/carcin/bgp268].
As expected, EXTEND scores significantly correlated with _TERC_ (R = 0.619, p < 0.01) and _TERT_ (R = 0.491, p < 0.01) expression (**Figure {@fig:S5}B-C**).  
We found aggressive tumors such as CNS lymphoma, ETMR, ATRT, DMG, and HGG had high EXTEND scores (**Figure {@fig:Fig4}D**), while benign lesions such as teratomas, dysplasias, and hemangioblastomas had the lowest scores (**Table S3**).
This supports previous reports of a more aggressive phenotype in tumors with higher telomerase activity	[@doi:10.1007/s13277-016-5045-7;10.1038/labinvest.3700710;10.1007/s12032-016-0736-x;10.1111/j.1750-3639.2010.00372.x].

#### Hypermutant tumors share mutational signatures and have dysregulated **_TP53_**

We further investigated the mutational signature profiles of the hypermutant (N = 2) and ultra-hypermutant (N = 4) HGG tumors and/or derived cell lines from five patients in the OpenPBTA cohort.
The relative contributions of eight RefSig mutational signatures are shown in the heatmap in **Figure {@fig:Fig4}E** for all tumors from patients with at least one tumor or cell line having a TMB >= 10 Mut/Mb.
Signature 11, a suggested exposure to temozolomide, was indeed present in tumors with previous exposure to the drug (**Table 2**).
We found the MMR2 signature in tumors of four patients (PT_0SPKM4S8, PT_3CHB9PK5, PT_JNEV57VK, and PT_VTM2STE3) diagnosed with Lynch syndrome (**Table 2**), a genetic predisposition syndrome caused by a variant in a mismatch repair gene such as _PMS2_, _MLH1_, _MSH2_, _MSH6_, or others [@doi:10.3390/genes5030497].
Interestingly, the cell line derived from patient PT_VTM2STE3's tumor at progression was not hypermutated (TMB = 5.74) but shows exposure to _only_ the MMR2 signature, suggesting there was selective pressure to maintain an MMR phenotype _in vitro_.
From patient PT_JNEV57VK, only one of the two cell lines derived from the progressive tumor was hypermutant (TMB of 35.9 Mut/Mb).
The hypermutated cell line had a strong signature 11, while the non-hypermutated cell line from the same tumor showed a number of lesser exposures (1, 11, 18, 19, MMR2), highlighting the importance of carefully genomically characterizing and selecting models for preclinical studies based on research objectives.
We observed that signature 18, which has been associated with high genomic instability and can lead to a hypermutator phenotype [@doi:10.1038/s43018-020-0027-5], is uniformly represented among hypermutant solid tumors. 
Additionally, we found that all of these tumor or cell lines have dysfunctional _TP53_ (**Table 2**), consistent with a previous report showing dysregulation of _TP53_ is a dependency in tumors with high genomic instability [@doi:10.1038/s43018-020-0027-5].
With one exception, hypermutant and ultra-hypermutant tumors had high _TP53_ scores and telomerase activity **Figure {@fig:Fig4}D**.
Interestingly, none of the samples were exposed to signatures 3 (present in homologous recombination deficient tumors), signature 8 (arises from double nucleotide substitutions/unknown etiology), or signature N6 (a universal CNS tumor signature).
The mutual exclusivity of signatures 3 and MMR corroborates a previous report suggesting tumors do not have both deficient homologous repair and mismatch repair [@doi:10.1016/j.celrep.2018.03.076].

| Kids First Participant ID | Kids First Biospecimen ID | CBTN ID   | Phase of therapy  | Composition       | Therapy post-biopsy                              | Cancer predisposition(s)              | TMB                | OpenPBTA molecular subtype       |
|---------------------------|---------------------------|-----------|-------------------|-------------------|--------------------------------------------------|---------------------------------------|--------------------|----------------------------------|
| PT_0SPKM4S8               | BS_VW4XN9Y7               | 7316-2640 | Initial CNS Tumor | Solid Tissue      | Temozolomide, CCNU, Radiation                    | None documented                       | 187.38765454966892 | HGG, H3 wildtype, TP53 activated |
| PT_3CHB9PK5               | BS_20TBZG09               | 7316-515  | Initial CNS Tumor | Solid Tissue      | Temozolomide, Irinotecan, Bevacizumab, Radiation | NF-1,Other inherited conditions NOS   | 307.0492167109247  | HGG, H3 wildtype, TP53 loss      |
| PT_3CHB9PK5               | BS_8AY2GM4G               | 7316-2085 | Progressive       | Solid Tissue      | Unknown                                          | NF-1, Lynch syndrome, pathogenic MSH6 | 321.6079467820181  | HGG, H3 wildtype, TP53 loss      |
| PT_JNEV57VK               | BS_85Q5P8GF               | 7316-2594 | Initial CNS Tumor | Solid Tissue      | Temozolomide, Radiation                          | Lynch syndrome                        | 4.731587273105341  | DMG, H3 K28, TP53 loss           |
| PT_JNEV57VK               | BS_HM5GFJN8               | 7316-3058 | Progressive       | Derived Cell Line | Nivolumab                                        | Lynch Syndrome, pathogenic MSH2       | 35.89286913681093  | DMG, H3 K28, TP53 loss           |
| PT_JNEV57VK               | BS_QWM9BPDY               | 7316-3058 | Progressive       | Derived Cell Line | Nivolumab                                        | Lynch Syndrome, pathogenic MSH2       | 7.363357709033756  | DMG, H3 K28, TP53 loss           |
| PT_JNEV57VK               | BS_P0QJ1QAH               | 7316-3058 | Progressive       | Solid Tissue      | Nivolumab                                        | Lynch Syndrome, pathogenic MSH2       | 6.271452953701754  | DMG, H3 K28, TP53 activated      |
| PT_S0Q27J13               | BS_P3PF53V8               | 7316-2307 | Initial CNS Tumor | Solid Tissue      | Temozolomide, Irinotecan, Radiation              | None documented                       | 15.482649479451206 | HGG, H3 wildtype, TP53 activated |
| PT_VTM2STE3               | BS_ERFMPQN3               | 7316-2189 | Progressive       | Derived Cell Line | Unknown                                          | Lynch Syndrome                        | 5.739499354950266  | HGG, H3 wildtype, TP53 loss      |
| PT_VTM2STE3               | BS_02YBZSBY               | 7316-2189 | Progressive       | Solid Tissue      | Unknown                                          | Lynch Syndrome                        | 274.48805695576783 | HGG, H3 wildtype, TP53 activated |


Table 2. Patients with hypermutant tumors.	

Next, we asked whether transcriptomic classification of _TP53_ dysregulation and/or telomerase activity recapitulate the known prognostic influence of these oncogenic biomarkers.
We used an additive model to perform multivariate cox regression on overall survival (**STAR Methods**), controlling for extent of tumor resection and whether a tumor was low-grade (LGG group) or high-grade (HGG group). 
Depicted in the forest plot (**Figure {@fig:Fig4}F**), we show an expected significant overall survival benefit if the tumor was fully resected (HR = 0.37, CI = 0.2-0.69, p = 0.002) or if the tumor belongs to the LGG group (HR = 0.065,, CI = 0.0088-0.48, p = 0.007).
In contrast, we show an expected significant risk if the tumor belongs to the HGG group (HR = 5.8, CI = 3.6-9.1, p < 0.001).
High telomerase scores were a poor prognostic indicator across brain tumor histologies (HR = 20, CI = 6.1-65, p < 0.001), demonstrating that EXTEND scores calculated from RNA-Seq can be used as a rapid surrogate measure of telomerase activity.
Higher _TP53_ scores, indicative of dysregulation of the _TP53_ gene or pathway, while not significant across the entire OpenPBTA cohort, resulted in a significant overall survival risk for both DMGs (HR = 1.79e6, CI = 2-1.45e12, p = 0.04) and ependymomas (HR = 1612, CI = 9.1-3e5, p = 0.005).

Since we observed the negative prognostic effect of _TP53_ for HGGs, we assessed the effect of molecular subtypes with and without _TP53_ dysregulation, as determined by our _TP53_ classification combined with molecular evidence.
Using log rank univariate survival modeling (**STAR Methods**), we recapitulate the known increased risk of the H3 K28 mutation compared to H3 wildtype (BH-adjusted p = 0.0027) [@doi:10.1093/neuonc/nox149].
Furthermore, cox regression analysis revealed that DMG H3 K28 tumors with _TP53_ loss have significantly worse prognosis (HR = 3.2, CI = 1.5-6.6, p = 0.002) than DMG H3 K28 tumors with wildtype _TP53_ **Figure {@fig:Fig4}G**), which is visualized by Kaplan-Meier in **Figure {@fig:Fig4}H**. 

![**Figure 4. _TP53_ and telomerase activity** A, Receiver Operating Characteristic for _TP53_ classifier run on FPKM of stranded RNA-Seq samples. B, Violin and box plots of _TP53_ scores plotted by _TP53_ alteration type (N<sub>activated</sub> = 27, N<sub>lost</sub> = 140, N<sub>other</sub> = 999). C, Violin and box plots of _TP53_ RNA expression plotted by _TP53_ activation status (N<sub>activated</sub> = 11, N<sub>lost</sub> = 96, N<sub>other</sub> = 862). D, Box plots of _TP53_ and telomerase (EXTEND) scores grouped by `cancer_group`. Mutation status is highlighted in orange (hypermutant) or red (ultra-hypermutant). E, Heatmap of RefSig mutational signatures for patients who have least one tumor or cell line with a TMB >= 10 Mut/Mb. F, Forest plot depicting the prognostic effects of _TP53_ and telomerase scores on overall survival, controlling for extent of tumor resection, LGG group, and HGG group. G, Forest plot depicting the effect of molecular subtype on overall survival of HGGs. For F and G, hazard ratios (HR) with 95% confidence intervals and p-values are listed. Significant p-values are denoted with black diamonds. Reference groups are denoted by grey diamonds. H, Kaplan-Meier curve of HGG tumors by molecular subtype.](images/main/Figure4.png){#fig:Fig4}


#### Histologic and oncogenic pathway clustering

UMAP visualization of gene expression variation across brain tumors (**Figure {@fig:Fig5}A**) shows the expected clustering of brain tumors by histology.
We observed medulloblastomas cluster by molecular subtype with WNT and SHH in distinct clusters and Groups 3 and 4 showing some overlap (**Figure {@fig:S5}D**), as expected.
Of note, two samples annotated as the SHH subtype do not cluster with the MB samples and one clusters with Group 3 and 4 samples, suggesting potential subtype misclassification or different underlying biology of these tumors.

Additionally, except for three outliers, _C11orf95::RELA_ (_ZFTA::RELA_) fusion positive ependymomas fall within distinct clusters (**Figure {@fig:S5}E**).
_BRAF_-driven low-grade gliomas (**Figure {@fig:S5}F**) were present in three separate clusters, suggesting that there might be additional shared biology within each cluster.
Histone H3 G35-mutant HGGs generally clustered together and away from K28-mutant tumors (**Figure {@fig:S5}G**).
Interestingly, although H3 K28-mutant tumors have different biological drivers than H3 wildtype tumors, they did not form distinct clusters. 
This suggests they may be driven by common transcriptional programs, have other much stronger biological drivers such as their known distinct epigenetic drivers, or our sample size is too small to detect transcriptional differences.

We next performed gene set variant analysis (GSVA) for Hallmark cancer gene sets to demonstrate activation of underlying oncogenic pathways (**Figure {@fig:Fig5}B**) and characterized the immune cell landscape across OpenPBTA tumors by running quanTIseq on RNA-Seq data (**Figure {@fig:Fig5}C** and **Figure {@fig:S5}H**).
For example, HGG, DMG, MB, and ATRT tumors are known to upregulate _MYC_ [@doi:10.3390/genes8040107] which in turn activates _E2F_ and S phase [@doi:10.1016/s1097-2765(01)00275-1].
Indeed, we detected significant (Bonferroni p < 0.05) upregulation of _MYC_ and _E2F_ targets, as well as G2M (cell cycle phase following S phase) in both embryonal and diffuse astrocytic and oligodendroglial tumors compared to several other broad histologies.
In contrast, low-grade astrocytic tumors show significant downregulation (Bonferroni p < 0.05) of these pathways.
Schwannomas and neurofibromas, which have a documented inflammatory immune microenvironment of T and B lymphocytes, as well as tumor-associated macrophages (TAMs) driven by upregulation of cytokines such as IFN$\gamma$, IL-1, and IL-6, and TNF$\alpha$ [@doi:10.1093/noajnl/vdaa023].
This was recapitulated in GSVA hallmark pathways (Bonferroni p < 0.05) (**Figure {@fig:Fig5}B**) and immune cell deconvolution, showing that monocytes make up the majority of immune cells in these tumors (**Figure {@fig:Fig5}C**).
Complete GSVA results can be found within the `gene-set-enrichment-analysis` in the [OpenPBTA analysis repository](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/gene-set-enrichment-analysis).

Although CD8+ T-cell infiltration across all cancer groups is quite low (**Figure {@fig:Fig5}C** and **Figure {@fig:S5}H**), we observed some signal in specific cancer molecular subtypes (Groups 3 and 4 medulloblastoma) as well as outlier tumors (BRAF-driven LGG, BRAF-driven and wildtype ganglioglioma, H3 wildtype HGG).   
Surprisingly, the classically immunologically cold HGG and DMG tumors [@doi:10.1186/s40478-018-0553-x; 10.1093/brain/awab155] contained higher overall fractions of immune cells, with monocytes, dendritic cells, and NK cells being the most prevalent (**Figure {@fig:Fig5}C**).
Thus, we suspect that quanTIseq might actually be capturing microglia within these immune cell fractions.
While we did not detect significant prognostic effects of immune cell types in HGG or DMG, we did find that high levels of macrophage M1 and monocytes were associated with poorer overall survival (macrophage M1 HR = 749, p = 0.002, monocyte HR = 7.3e17, p = 0.006) in medulloblastoma tumors (**Figure {@fig:Fig5}D**).
Medulloblastomas typically have low expression of _CD274_ (PD-L1) [@doi:10.18632/oncotarget.24951] a result we reproduced here (**Figure {@fig:Fig5}E**).
However, we also found that higher expression of _CD274_ is significantly associated with better overall prognosis (HR = 0.00058, CI = 3.1e-6 to 0.11, p = 0.005).
Consistent with a previous study, we found higher expression of _CD274_ in the WNT subgroup, which has the best prognosis of all medulloblastoma subgroups [@doi:10.1080/2162402X.2018.1462430] (**Figure {@fig:Fig5}E**).
Finally, we asked whether any molecular subtypes might show an "immune hot" phenotype, as roughly defined by a greater proportion of CD8+ to CD4+ T cells and while we do see the highest ratios in Group 3 and Group 4 medulloblastomas (**Figure {@fig:Fig5}I**), very few tumors had ratios > 1, highlighting an urgent need to identify novel therapeutics for these "immune cold" pediatric brain tumors with poor prognosis.

![**Figure 5. Transcriptomic and Immune Landscape** A, First two dimensions from UMAP of sample transcriptome data. Points are colored by `broad_histology` of the samples they represent. B, Heatmap of GSVA scores for Hallmark gene sets with significant differences, with samples ordered by `cancer_group`. C, Box plots of quanTIseq estimates of immune cell proportions in histologies with N > 15. D, Forest plot depicting the effect of immune cell proportion or _CD274_ expression on overall survival of medulloblastomas. Hazard ratio (HR) with 95% confidence intervals and p-values are listed and significant p-values are denoted with colored diamonds. Of note, the upper CI for macrophage M1 HR was too large to plot. E, Box plot of _CD274_ expression (log<sub>2</sub> FPKM) for medulloblastoma samples grouped by molecular subtype. Adjusted p-values are shown.](images/main/Figure5.png){#fig:Fig5}