## Discussion

Through OpenPBTA, we used an open, reproducible analysis framework within GitHub to bring together researchers and clinicians across the nation to genomically characterize 943 pediatric brain tumors across 59 distinct histologies.
To our knowledge, this was both the first large-scale, collaborative open analysis of genomic data coupled with open manuscript writing and the largest cohort of pediatric brain tumors comprehensively analyzed to date.
We used available WGS, WXS, and RNA-Seq data to generate high-confidence consensus SNV and CNV calls, prioritize putative oncogenic fusions, and create over 40 scalable modules for common downstream cancer genomics analyses which have been used to create new knowledge as well as define new questions.
We detect and show expected frequencies of genomic lesions, mutational signatures, and aberrantly regulated signaling pathways.
Molecular subtyping information was largely not collected nor available for this cohort, and if available (eg: sparse medulloblastoma subtypes), it had to be manually curated from pathology reports and/or free text clinical data fields.
For WHO molecularly-defined diagnoses, we created subtyping modules and by working closely with pathologists and clinicians, were able to determine a research-grade integrated diagnosis for 44% (1260/2840) of biospecimens, thus enabling more targeted analyses to interrogate the biology of these tumors.
Through the subtyping modules, we were also able to determine mis-diagnosed or mis-identified samples. 
We applied multiple previously validated classifiers [@doi:10.1186/s13029-016-0053-y; @doi:10.1371/journal.pcbi.1008263; @doi:10.1016/j.celrep.2018.03.076; @doi:10.1038/s41467-020-20474-9] to RNA expression data to predict medulloblastoma subtypes, whether tumors have dysfunctional _TP53_ and/or whether telomerase is active.    

OpenPBTA has become a foundational layer to which we will add other genomic modalities and analyses, such as methylation, single cell, epigenomic, mRNA splicing, and model drug response data.	
The most significant innovations are twofold. 
First, the OpenPBTA has already enabled a framework to support N+1 subjects in an ongoing PNOC clinical trial to link tumor biology to clinical decision support.
Second, the OpenPBTA project has been rapidly expanded into a pan-pediatric cancer effort (https://github.com/PediatricOpenTargets/OpenPedCan-analysis).
We created the OpenPBTA project to provide robust code and data to the pediatric oncology community, as well as attract additional collaborators and new analyses to accelerate clinical translation.
All code and processed data are openly available through GitHub, CAVATICA, and PedcBioPortal (see **STAR METHODS**).
