## Discussion

Through the OpenPBTA, we used an open, reproducible analysis framework within GitHub to bring together researchers and clinicians from across the globe to genomically characterize 1,253 pediatric brain tumors across 59 distinct histologies.
We created the OpenPBTA project to provide robust code and data resources to the pediatric oncology community, as well as attract interdisciplinary scientists to collaborate on new analyses in order to accelerate therapeutic translation for children with cancer, goals we are seeing play out in real-time.
To our knowledge, this initiative represents the first large-scale, collaborative, open analysis of genomic data coupled with open manuscript writing, in which we comprehensively analyzed the largest cohort of pediatric brain tumors to date.
We used available WGS, WXS, and RNA-Seq data to generate high-confidence consensus SNV and CNV calls, prioritize putative oncogenic fusions, and establish over 40 scalable modules to perform common downstream cancer genomics analyses, all of which have undergone rigorous scientific and analytical code review. 
We detected and showed expected frequencies of genomic lesions, mutational signatures, and aberrantly regulated signaling pathways across multiple pediatric brain tumor histologies.
Molecular subtyping information from pathology reports was largely not collected nor available for this cohort, and if available (e.g.: sparse medulloblastoma subtypes), it had to be manually curated from pathology reports and/or free text clinical data fields.
Furthermore, in the absence of DNA methylation data, rapid classification to derive molecular subtypes could not be performed.
Thus, to enable biological interrogation of specific tumor subtypes, we created RNA- and DNA-based subtyping modules aligned with WHO molecularly-defined diagnoses.
We worked closely with pathologists and clinicians to build modules from which we determined a research-grade integrated diagnosis for 44% (1260/2840) of biospecimens as well as discovered incorrectly diagnosed or mis-identified samples in the OpenPBTA cohort. 

We harnessed RNA expression data for a number of analyses, including application of multiple previously validated classifiers.
From 43/122 (35.2%) medulloblastoma tumors whose pathology reports contained subtype information, we accurately predicted subtypes using MM2S (90.7%) or medulloPackage (95.4%) [@doi:10.1186/s13029-016-0053-y; @doi:10.1371/journal.pcbi.1008263].
We used additional classifiers to determine whether tumors have dysfunctional _TP53_ [@doi:10.1016/j.celrep.2018.03.076] and their degree of telomerase activity from EXTEND scores [@doi:10.1038/s41467-020-20474-9].
We found that hypermutant tumors universally display dysregulation of _TP53_ and that EXTEND scores are a robust surrogate measure for telomerase activity, as tumors with high scores have a significant adverse risk for overall survival. 
Additionally, high _TP53_ scores are a significant negative prognostic indicator for overall survival for patients with certain tumor types, such as H3 K28-altered DMGs and ependymomas. 
By assessing _TP53_ and telomerase activity prospectively from expression data, information previously only attainable with DNA sequencing and/or qPCR, we can quickly add oncogenic biomarker and prognostic knowledge to our existing biological understanding of these tumors.
We identified enrichment of hallmark cancer pathways and characterized the immune cell landscape across pediatric brain tumors, demonstrating some histologies, such as schwannomas, high-grade gliomas, and diffuse midline gliomas, may have a inflammatory tumor microenvironment.
Finally, our study reproduces the overall known poor infiltration of CD8+ T cells and general low expression of _CD274_ (PD-L1) in pediatric brain tumors, further highlighting the urgent need to identify novel therapeutic strategies for these "immune cold" tumors. 

OpenPBTA has rapidly become a foundational layer for a number of discovery research and translational projects which will continue to add other genomic modalities and analyses, such as germline, methylation, single cell, epigenomic, mRNA splicing, imaging, and model drug response data.
For example, the RNA fusion filtering module created within OpenPBTA set the stage for development of the R package _annoFuse_ [@doi:10.1186/s12859-020-03922-7] and an R Shiny application [_shinyFuse_](http://shiny.imbei.uni-mainz.de:3838/shinyFuse/).
Using medulloblastoma subtyping and immune deconvolution analyses performed herein, Dang and colleagues showed enrichment of monocyte and microglia-derived macrophages within the SHH subgroup which they suggest may accumulate following radiation therapy [@doi:10.1016/j.celrep.2021.108917].
Expression and copy number analyses were used to demonstrate that _GPC2_ is a highly expressed and copy number gained immunotherapeutic target in ETMRs, medulloblastomas, choroid plexus carcinomas, H3 wildtype high-grade gliomas, as well as DMGs [@doi:10.1101/2021.07.06.451385].
This led Foster and colleagues to subsequently develop a chimeric antigen receptor (CAR) directed against _GPC2_, for which they show preclinical efficacy in mouse models [@doi:10.1101/2021.07.06.451385]. 
Moreover, the OpenPBTA has enabled a framework to support real-time integration of subjects as each was enrolled on the PNOC008 high-grade glioma clinical trial [@url:https://pnoc.us/clinical-trial/pnoc008/], allowing researchers and clinicians to link tumor biology to translational impact through clinical decision support during tumor board discussions.
Finally, the OpenPBTA project was recently expanded into a pan-pediatric cancer effort to support the RACE Act (https://github.com/PediatricOpenTargets/OpenPedCan-analysis) as part of the the NCI's Childhood Cancer Data Initiative.
The OpenPBTA project has paved the way for collaborative, open, reproducible, and scalable analyses and we anticipate this foundational work will have limitless impact within the pediatric oncology community, ultimately leading to improved outcomes for children with cancer.

All code and processed data are openly available through GitHub, CAVATICA, and PedcBioPortal (see **STAR METHODS**).
