## Results

Results section stub.

### Crowd-sourced Somatic Analysis to create an Open Pediatric Brain Tumor Atlas

We previously performed whole genome sequencing (WGS), whole exome sequencing (WXS), and RNA sequencing (RNA-Seq) on matched tumor and normal tissues as well as selected cell lines from 943 patient tumors from the Pediatric Brain Tumor Atlas (PBTA) [@doi:10.1093/neuonc/noz192], samples sourced from the Childrenâ€™s Brain Tumor Network (CBTN) and the PNOC003 DMG clinical trial [@doi:10.1002/ijc.32258] of the Pacific Pediatric Neuro-oncology Symposium (PNOC) (*Figure 1*). 
We then harnessed the benchmarking efforts of the KidsFirst Data Resource Center to develop a robust and reproducible data analysis workflow within the CAVATICA platform to perform primary somatic analyses: variant calling of single nucleotide variants (SNVs), copy number variants (CNVs), structural variants (SVs), and fusions (*Figure S1* - red boxes and *STAR Methods*). 
Next, we created a Github analysis repository (https://github.com/AlexsLemonade/OpenPBTA-analysis) with CircleCI (Continuous Integration) to ensure analysis reproducibility and a GitHub manuscript repository (https://github.com/AlexsLemonade/OpenPBTA-manuscript) with ManuBot (@https://manubot.org/) integration to enable manuscript creation using Markdown within GitHub. 
We maintained a data release folder on Amazon S3 containing merged files for each analysis, downloadable from the GitHub repository (https://github.com/AlexsLemonade/OpenPBTA-analysis/#data-access-via-download-script) or open access CAVATICA project (https://cavatica.sbgenomics.com/u/cavatica/openpbta/).  

Stub for the process for contributing analytical code (*Figure S1* and *STAR Methods*) and to the manuscript


### Landscape of Genomic Alterations

The OncoPrint will provide a visualization of the genomic alterations found in the analyses implemented throughout the OpenPBTA project.

![OncoPrint displaying genes most frequently altered across the OpenPBTA dataset.
Genes include the top 50 most frequently mutated genes and the top 20 genes with copy number alterations.
Samples were filtered to primary samples or, when no primary sample from an individual participant was available, a randomly selected sample with whole genome sequencing (WGS) data.
Sample histology (`short_histology`) is displayed in the annotation bar at the bottom of the plot.](https://github.com/AlexsLemonade/OpenPBTA-analysis/raw/master/figures/pngs/fig3-oncoprint-landscape.png){#fig:oncoprint-landscape}

### Landscape of Mutational Processes

This section summarizes the mutational landscape of the pediatric brain tumor samples of this dataset.
Figure {@fig:openpbta-overview}A shows the tumor mutation burden as compared to adult TCGA brain-related tumors.
Figure {@fig:mutational-landscape-overview}B-C show concordance of these samples with mutational signatures from COSMIC [@url:https://cancer.sanger.ac.uk/cosmic] and Alexandrov et al, 2013 [@doi:10.1038/nature12477] signature sets.

![Mutational Landscape](https://github.com/AlexsLemonade/OpenPBTA-analysis/raw/master/figures/pngs/fig2-mutational-landscapes.png){#fig:mutational-landscape-overview}

### Copy Number Variant Overview

![This figure shows dominant statuses for small copy number consensus segments (< 10 Mb) across the genome, where each square represents a ~1 Mb binned section of the genome.
A dominant status is declared if one status is >75% coverage.
Unstable indicates multiple non-neutral statuses totaling coverage >75%.
Copy number segments longer than 10 Mb have been removed from the figure for easier interpretability.](https://github.com/AlexsLemonade/OpenPBTA-analysis/raw/master/figures/pngs/cn_status_heatmap.png){#fig:copy-number-status-heatmap width="7in"}

### Recurrence and co-occurrence of mutations

This section will discuss the genes and regions that are repeatedly mutated within and between cancer types.
The occurrence of mutations in affecting particular genes, separated by tumor type is shown in Figure {@fig:occurrence-plots}A, with significant co-occurrence across all types illustrated in Figure {@fig:occurrence-plots}B.

![Occurrence and co-occurrence of nonsynonymous mutations for the 50 most commonly mutated genes across all tumor types.
A) Counts of nonsynonymous mutations, colored by tumor type (as defined in `integrated_diagnosis`).
B) Co-occurrence and mutual exclusivity of nonsynonymous mutations between genes.
The co-occurrence score is defined as $I(-\log_{10}(P))$ where $P$ is defined by Fisher's exact test and $I$ is 1 when mutations co-occur more often than expected and -1 when exclusivity is more common.
](https://github.com/AlexsLemonade/OpenPBTA-analysis/raw/master/figures/pngs/mutation_cooccurrence_figure.png){#fig:occurrence-plots}

### Transcriptomic landscape

This section will discuss the overall structure of the transcriptome data, pathway analysis, and immune deconvolution.
A dimension reduction plot, a heatmap of GSVA scores [@doi:10.1186/1471-2105-14-7], and a heatmap of xCell fraction values [@doi:10.1186/s13059-017-1349-1] are shown in Figure {@fig:transcriptomic-overview}.

![Transcriptomic overview (stranded data only)
A) First two dimensions from Uniform Manifold Approximation and Projection (UMAP) of sample transcriptome data.
Points are colored by `short_histology` of the samples they represent.
B) Heatmap of GSVA scores for Hallmark gene sets with significant differences, with samples ordered by `short_histology`.
C) Heatmap of xCell fraction values for scores and cell types with high variance, with samples ordered by `short_histology`.
](https://github.com/AlexsLemonade/OpenPBTA-analysis/raw/master/figures/pngs/transcriptomic-overview.png){#fig:transcriptomic-overview}

### Quantify Telomerase Activity using Gene Expression Data

We used gene expression data sets from both poly-A and stranded RNA-Seq protocols to predict telomerase activities using EXTEND [@doi:10.1101/2020.05.21.109249] as shown in [Supplementary Tables](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/telomerase-activity-prediction/results).

For both platforms, EXTEND scores were highly consistent between FPKM versus counts data (Supplementary Figure) {@fig:comparison-plots}A and Figure {@fig:comparison-plots}B.
  
![Scatter plots comparing telomerase activities via Spearman correlation for counts with FPKM format.
A) poly-A platform .
B) stranded platform.
](https://github.com/AlexsLemonade/OpenPBTA-analysis/raw/master/figures/pngs/SuppTelomerase_Activities.png){#fig:comparison-plots}

We next examined telomerase activity across different brain tumor histological subtypes using data from the stranded FPKM protocol. 
Across the short histological groups, we found germinomas had the highest EXTEND scores. 
However, teratomas, another type of germ cell tumor, had much lower scores (Figure) {@fig:distribution-plots}A.
Teratomas were previously reported to have negligible or low telomerase activities [@doi:10.1093/jnci/91.15.1321]. 
Teratomas originating from the tissue of three dermal layers of an embryo, are benign germ cell neoplasms [@doi: 10.1097/MOP.0b013e32832b41ee].
These tumors are divided into mature and immature (malignant) groups, where pediatric tumors are mostly classified as mature ones [@doi: 10.1016/j.jpedsurg.2004.03.045]. 
Mature tumors are deficient of malignant potential thereby reducing the risk of malignancy [@doi: 10.1016/j.jpedsurg.2004.03.045], hence bearing low telomerase activities.
Benign neoplasia including dysplasia and hemangioblastomas had the lowest scores. 
Among common pediatric brain tumors, medulloblastomas had higher scores than high grade astrocytic tumors, ependymomas and low-grade astrocytomas. 
Across broad groups, benign tumors and precancerous lesions were among those with the lowest scores, whereas pineal region and embryonal tumors have the highest scores (Figure) {@fig:distribution-plots}B. 
Considering medulloblastoma tumors, high risk Group3 and Group4 subtypes had higher scores than WNT and SHH subtype (Figure) {@fig:distribution-plots}C. 
A similar pattern has been speculated with respect to cell of origin by one of the earlier studies [@doi:10.1007/s00401-013-1198-2] which mainly focused TERT promoter mutations for medulloblastoma subtypes comparisons rather than telomerase activity.

![Box plots representing distribution of telomerase activities for stranded FPKM data.
A) short histological classifications.
B) broad histological classifications.
C) Comparison of telomerase activities across molecular subtypes of medulloblastoma tumors. P-values have been calculated using series of two-sided t-tests and corrected via Bonferroni correction.
](https://github.com/AlexsLemonade/OpenPBTA-analysis/raw/master/figures/pngs/Telomerase_Activities.png){#fig:distribution-plots}
