## Limitations of Study

Notably, PBTA brain tumor samples were collected over decades, and RNA samples were prepared using two distinct library preparations (stranded or poly-A, **Figure {@fig:S7}A**) by multiple sequencing centers.
While we noted a strong library preparation batch effect (**Figure {@fig:S7}B**) and a possible sequencing center batch effect (**Figure {@fig:S7}C**), cancer groups are highly unbalanced across library preparations (**Figure {@fig:S7}A**).
We did not perform batch correction because removing batch effects across unbalanced groups may induce false differences among groups [@doi:10.1093/biostatistics/kxv027; @doi:10.1016/j.tibtech.2017.02.012].
Instead, we circumvent batch effects by grouping only stranded RNA-Seq expression data, which comprises the vast majority of the PBTA cohort, for transcriptomic analyses presented in **Figure {@fig:Fig4}** and **Figure {@fig:Fig5}** .
As batch correction strategy depends highly on research goals [@doi:10.1016/j.tibtech.2017.02.012], we provide library preparation-specific expression matrices in the OpenPBTA data release for others to adapt to their needs.
A second potential limitation is that performing analyses with all samples, rather than samples with high tumor purity, might result in loss of information, such as subclonal variants or low-level oncogenic pathway expression.
To this end, we re-performed transcriptomic analyses using only samples with high tumor purity (see **Methods** for details), and indeed, results were broadly consistent with those derived from the full cohort (**Figure {@fig:S7}D-I**).
To enable more robust statistical analysis and presentation of results, we randomly selected one independent specimen from patients with duplicate sequenced samples per tumor event rather than combining the data.
This practice did not induce notable differences if the selected specimen changed over time, e.g., with a new data release.
Finally, because this initial PBTA cohort mostly contains samples collected at diagnosis from one tumor section/punch, we could not reliably perform systematic intratumoral and/or longitudinal analyses, though we expect nearly 100 paired longitudinal tumors from the ([NIH X01 CA267587-01 pediatric brain tumor cohort](https://commonfund.nih.gov/kidsfirst/2021X01projects#FY21_Resnick)) to be released through [OpenPedCan](https://github.com/PediatricOpenTargets/OpenPedCan-analysis) for future exploration.
