<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <meta name="author" content="Joshua A. Shapiro" />
  <meta name="author" content="Krutika S. Gaonkar" />
  <meta name="author" content="Candace L. Savonen" />
  <meta name="author" content="Stephanie J. Spielman" />
  <meta name="author" content="Chante J. Bethell" />
  <meta name="author" content="Run Jin" />
  <meta name="author" content="Komal S. Rathi" />
  <meta name="author" content="Yuankun Zhu" />
  <meta name="author" content="Laura E. Egolf" />
  <meta name="author" content="Bailey K. Farrow" />
  <meta name="author" content="Daniel P. Miller" />
  <meta name="author" content="Yang Yang" />
  <meta name="author" content="Tejaswi Koganti" />
  <meta name="author" content="Nighat Noureen" />
  <meta name="author" content="Mateusz P. Koptyra" />
  <meta name="author" content="Nhat Duong" />
  <meta name="author" content="Adam A. Kraya" />
  <meta name="author" content="Cassie Kline" />
  <meta name="author" content="Yiran Guo" />
  <meta name="author" content="Phillip B. Storm" />
  <meta name="author" content="Javad Nazarian" />
  <meta name="author" content="Stephen C. Mack" />
  <meta name="author" content="Pichai Raman" />
  <meta name="author" content="Siyuan Zheng" />
  <meta name="author" content="Peter J. Madsen" />
  <meta name="author" content="Nicholas Van Kuren" />
  <meta name="author" content="Shannon Robins" />
  <meta name="author" content="Xiaoyan Huang" />
  <meta name="author" content="Angela J. Waanders" />
  <meta name="author" content="Jung Kim" />
  <meta name="author" content="Gregory P. Way" />
  <meta name="author" content="Christopher Blackden" />
  <meta name="author" content="Eric Wafula" />
  <meta name="author" content="Noel Coleman" />
  <meta name="author" content="Derek Hanson" />
  <meta name="author" content="Carl J. Koschmann" />
  <meta name="author" content="Sharon J. Diskin" />
  <meta name="author" content="Rishi R. Lulla" />
  <meta name="author" content="Miguel A. Brown" />
  <meta name="author" content="Jessica Wong" />
  <meta name="author" content="Jennifer L. Mason" />
  <meta name="author" content="Laura Scolaro" />
  <meta name="author" content="Meen Chul Kim" />
  <meta name="author" content="Hongbo M. Xie" />
  <meta name="author" content="Brian R. Rood" />
  <meta name="author" content="Kristina A. Cole" />
  <meta name="author" content="Ashley S. Margol" />
  <meta name="author" content="Zalman Vaksman" />
  <meta name="author" content="Rebecca S. Kaufman" />
  <meta name="author" content="Allison P. Heath" />
  <meta name="author" content="Brian M. Ennis" />
  <meta name="author" content="Mariarita Santi" />
  <meta name="author" content="Anna R. Poetsch" />
  <meta name="author" content="Angela N. Viaene" />
  <meta name="author" content="Douglas R. Stewart" />
  <meta name="author" content="Steven M. Foltz" />
  <meta name="author" content="Payal Jain" />
  <meta name="author" content="Bo Zhang" />
  <meta name="author" content="Shrivats Kannan" />
  <meta name="author" content="Michael Prados" />
  <meta name="author" content="Jena V. Lilly" />
  <meta name="author" content="Sabine Mueller" />
  <meta name="author" content="Adam C. Resnick" />
  <meta name="author" content="Casey S. Greene" />
  <meta name="author" content="Jo Lynne Rokita" />
  <meta name="author" content="Jaclyn N. Taroni" />
  <meta name="author" content="Children’s Brain Tumor Network" />
  <meta name="author" content="Pacific Pediatric Neuro-Oncology Consortium" />
  <meta name="dcterms.date" content="2023-02-24" />
  <meta name="keywords" content="pediatric cancer, brain tumor, tumor atlas, reproducibility, open science, classification, somatic variation" />
  <title>OpenPBTA: An Open Pediatric Brain Tumor Atlas</title>
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    span.underline{text-decoration: underline;}
    div.column{display: inline-block; vertical-align: top; width: 50%;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
    }
    .hanging div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }
  </style>
  <!--
  Manubot generated metadata rendered from header-includes-template.html.
  Suggest improvements at https://github.com/manubot/manubot/blob/main/manubot/process/header-includes-template.html
  -->
  <meta name="dc.format" content="text/html" />
  <meta name="dc.title" content="OpenPBTA: An Open Pediatric Brain Tumor Atlas" />
  <meta name="citation_title" content="OpenPBTA: An Open Pediatric Brain Tumor Atlas" />
  <meta property="og:title" content="OpenPBTA: An Open Pediatric Brain Tumor Atlas" />
  <meta property="twitter:title" content="OpenPBTA: An Open Pediatric Brain Tumor Atlas" />
  <meta name="dc.date" content="2023-02-24" />
  <meta name="citation_publication_date" content="2023-02-24" />
  <meta name="dc.language" content="en-US" />
  <meta name="citation_language" content="en-US" />
  <meta name="dc.relation.ispartof" content="Manubot" />
  <meta name="dc.publisher" content="Manubot" />
  <meta name="citation_journal_title" content="Manubot" />
  <meta name="citation_technical_report_institution" content="Manubot" />
  <meta name="citation_author" content="Joshua A. Shapiro" />
  <meta name="citation_author_institution" content="Childhood Cancer Data Lab, Alex&#39;s Lemonade Stand Foundation, Bala Cynwyd, PA, USA" />
  <meta name="citation_author_orcid" content="0000-0002-6224-0347" />
  <meta name="twitter:creator" content="@jashapiro" />
  <meta name="citation_author" content="Krutika S. Gaonkar" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery in Biomedicine, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Department of Bioinformatics and Health Informatics, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0003-0838-2405" />
  <meta name="twitter:creator" content="@aggokittu" />
  <meta name="citation_author" content="Candace L. Savonen" />
  <meta name="citation_author_institution" content="Childhood Cancer Data Lab, Alex&#39;s Lemonade Stand Foundation, Bala Cynwyd, PA, USA" />
  <meta name="citation_author_institution" content="Fred Hutchinson Cancer Center, Seattle, WA, USA" />
  <meta name="citation_author_orcid" content="0000-0001-6331-7070" />
  <meta name="twitter:creator" content="@cansavvy" />
  <meta name="citation_author" content="Stephanie J. Spielman" />
  <meta name="citation_author_institution" content="Childhood Cancer Data Lab, Alex&#39;s Lemonade Stand Foundation, Bala Cynwyd, PA, USA&lt;sup&gt;+&lt;/sup&gt;" />
  <meta name="citation_author_institution" content="Rowan University, Glassboro, NJ, USA" />
  <meta name="citation_author_orcid" content="0000-0002-9090-4788" />
  <meta name="twitter:creator" content="@stephspiel" />
  <meta name="citation_author" content="Chante J. Bethell" />
  <meta name="citation_author_institution" content="Childhood Cancer Data Lab, Alex&#39;s Lemonade Stand Foundation, Bala Cynwyd, PA, USA" />
  <meta name="citation_author_orcid" content="0000-0001-9653-8128" />
  <meta name="twitter:creator" content="@cjbethell" />
  <meta name="citation_author" content="Run Jin" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery in Biomedicine, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0002-8958-9266" />
  <meta name="twitter:creator" content="@runjin" />
  <meta name="citation_author" content="Komal S. Rathi" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery in Biomedicine, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Department of Bioinformatics and Health Informatics, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0001-5534-6904" />
  <meta name="twitter:creator" content="@komalsrathi" />
  <meta name="citation_author" content="Yuankun Zhu" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery in Biomedicine, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0002-2455-9525" />
  <meta name="twitter:creator" content="@zhuyuankun" />
  <meta name="citation_author" content="Laura E. Egolf" />
  <meta name="citation_author_institution" content="Cell and Molecular Biology Graduate Group, Perelman School of Medicine at the University of Pennsylvania" />
  <meta name="citation_author_institution" content="Division of Oncology, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0002-7103-4801" />
  <meta name="twitter:creator" content="@LauraEgolf" />
  <meta name="citation_author" content="Bailey K. Farrow" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery in Biomedicine, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0001-6727-6333" />
  <meta name="citation_author" content="Daniel P. Miller" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery in Biomedicine, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0002-2032-4358" />
  <meta name="citation_author" content="Yang Yang" />
  <meta name="citation_author_institution" content="Ben May Department for Cancer Research, University of Chicago, Chicago IL, USA" />
  <meta name="citation_author" content="Tejaswi Koganti" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery in Biomedicine, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0002-7733-6480" />
  <meta name="citation_author" content="Nighat Noureen" />
  <meta name="citation_author_institution" content="Greehey Children&#39;s Cancer Research Institute, UT Health San Antonio" />
  <meta name="citation_author_orcid" content="0000-0001-7495-8201" />
  <meta name="citation_author" content="Mateusz P. Koptyra" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery in Biomedicine, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0002-3857-6633" />
  <meta name="twitter:creator" content="@koptyram" />
  <meta name="citation_author" content="Nhat Duong" />
  <meta name="citation_author_institution" content="Department of Bioinformatics and Health Informatics, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0003-2852-4263" />
  <meta name="twitter:creator" content="@asiannhat" />
  <meta name="citation_author" content="Adam A. Kraya" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery in Biomedicine, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0002-8526-5694" />
  <meta name="citation_author" content="Cassie Kline" />
  <meta name="citation_author_institution" content="Division of Oncology, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0001-7765-7690" />
  <meta name="twitter:creator" content="@cassiekmd" />
  <meta name="citation_author" content="Yiran Guo" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery in Biomedicine, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0002-6549-8589" />
  <meta name="twitter:creator" content="@YiranGuo3" />
  <meta name="citation_author" content="Phillip B. Storm" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery in Biomedicine, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0002-7964-2449" />
  <meta name="citation_author" content="Javad Nazarian" />
  <meta name="citation_author_institution" content="Children&#39;s National Research Institute, Washington, D.C." />
  <meta name="citation_author_institution" content="George Washington University School of Medicine and Health Sciences, Washington, D.C." />
  <meta name="citation_author_orcid" content="0000-0002-1951-9828" />
  <meta name="citation_author" content="Stephen C. Mack" />
  <meta name="citation_author_institution" content="Department of Developmental Neurobiology, St. Jude Children’s Research Hospital" />
  <meta name="citation_author_orcid" content="0000-0001-9620-4742" />
  <meta name="citation_author" content="Pichai Raman" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery in Biomedicine, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Department of Bioinformatics and Health Informatics, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0001-6948-2157" />
  <meta name="twitter:creator" content="@PichaiRaman" />
  <meta name="citation_author" content="Siyuan Zheng" />
  <meta name="citation_author_institution" content="Greehey Children&#39;s Cancer Research Institute, UT Health San Antonio" />
  <meta name="citation_author_orcid" content="0000-0002-1031-9424" />
  <meta name="twitter:creator" content="@zhengsiyuan" />
  <meta name="citation_author" content="Peter J. Madsen" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery in Biomedicine, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0001-9266-3685" />
  <meta name="twitter:creator" content="@petermadsenmd" />
  <meta name="citation_author" content="Nicholas Van Kuren" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery in Biomedicine, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0002-7414-9516" />
  <meta name="citation_author" content="Shannon Robins" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery in Biomedicine, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0003-0594-1953" />
  <meta name="citation_author" content="Xiaoyan Huang" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery in Biomedicine, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0001-7267-4512" />
  <meta name="citation_author" content="Angela J. Waanders" />
  <meta name="citation_author_institution" content="Division of Hematology, Oncology, Neuro-Oncology, and Stem Cell Transplant, Ann &amp; Robert H Lurie Children’s Hospital of Chicago" />
  <meta name="citation_author_institution" content="Department of Pediatrics, Northwestern University Feinberg School of Medicine" />
  <meta name="citation_author_orcid" content="0000-0002-0571-2889" />
  <meta name="citation_author" content="Jung Kim" />
  <meta name="citation_author_institution" content="Clinical Genetics Branch, Division of Cancer Epidemiology and Genetics, National Cancer Institute" />
  <meta name="citation_author_orcid" content="0000-0001-6274-2841" />
  <meta name="citation_author" content="Gregory P. Way" />
  <meta name="citation_author_institution" content="Department of Biomedical Informatics, University of Colorado School of Medicine, Aurora, CO, USA" />
  <meta name="citation_author_orcid" content="0000-0002-0503-9348" />
  <meta name="twitter:creator" content="@gwaybio" />
  <meta name="citation_author" content="Christopher Blackden" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery in Biomedicine, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author" content="Eric Wafula" />
  <meta name="citation_author_institution" content="Department of Bioinformatics and Health Informatics, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0001-8073-3797" />
  <meta name="citation_author" content="Noel Coleman" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery in Biomedicine, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0001-6454-1285" />
  <meta name="citation_author" content="Derek Hanson" />
  <meta name="citation_author_institution" content="Hackensack Meridian School of Medicine" />
  <meta name="citation_author_institution" content="Hackensack University Medical Center" />
  <meta name="citation_author_orcid" content="0000-0002-0024-5142" />
  <meta name="citation_author" content="Carl J. Koschmann" />
  <meta name="citation_author_institution" content="Department of Pediatrics, University of Michigan Health, Ann Arbor, MI" />
  <meta name="citation_author_institution" content="Pediatric Hematology Oncology, Mott Children&#39;s Hospital, Ann Arbor, MI" />
  <meta name="citation_author_orcid" content="0000-0002-0825-7615" />
  <meta name="citation_author" content="Sharon J. Diskin" />
  <meta name="citation_author_institution" content="Division of Oncology, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Department of Pediatrics, University of Pennsylvania" />
  <meta name="citation_author_orcid" content="0000-0002-7200-8939" />
  <meta name="twitter:creator" content="@sjdiskin" />
  <meta name="citation_author" content="Rishi R. Lulla" />
  <meta name="citation_author_institution" content="Division of Hematology/Oncology, Hasbro Children&#39;s Hospital" />
  <meta name="citation_author_institution" content="Department of Pediatrics, The Warren Alpert School of Brown University, Providence, Rhode Island" />
  <meta name="citation_author_orcid" content="0000-0003-4109-2207" />
  <meta name="citation_author" content="Miguel A. Brown" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery in Biomedicine, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0001-6782-1442" />
  <meta name="twitter:creator" content="@migbro" />
  <meta name="citation_author" content="Jessica Wong" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery in Biomedicine, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0003-1508-7631" />
  <meta name="twitter:creator" content="@jessicawongbfx" />
  <meta name="citation_author" content="Jennifer L. Mason" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery in Biomedicine, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="twitter:creator" content="@jenn0307" />
  <meta name="citation_author" content="Laura Scolaro" />
  <meta name="citation_author_institution" content="Division of Oncology, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author" content="Meen Chul Kim" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery in Biomedicine, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0002-0308-783X" />
  <meta name="citation_author" content="Hongbo M. Xie" />
  <meta name="citation_author_institution" content="Department of Bioinformatics and Health Informatics, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0003-2223-0029" />
  <meta name="twitter:creator" content="@xiehb" />
  <meta name="citation_author" content="Brian R. Rood" />
  <meta name="citation_author_institution" content="Children&#39;s National Research Institute, Washington, D.C." />
  <meta name="citation_author_institution" content="George Washington University School of Medicine and Health Sciences, Washington, D.C." />
  <meta name="citation_author" content="Kristina A. Cole" />
  <meta name="citation_author_institution" content="Division of Oncology, Children&#39;s Hospital of Philadelphia, Philadelphia, PA" />
  <meta name="citation_author_institution" content="Department of Pediatrics, University of Pennsylvania, Philadelphia, PA" />
  <meta name="citation_author_institution" content="Abramson Family Cancer Research Institute, Perelman School of Medicine at the University of Pennsylvania, Philadelphia, PA" />
  <meta name="citation_author_orcid" content="0000-0003-0064-2882" />
  <meta name="citation_author" content="Ashley S. Margol" />
  <meta name="citation_author_institution" content="Division of Hematology and Oncology, Children&#39;s Hospital Los Angeles" />
  <meta name="citation_author_institution" content="Department of Pediatrics, Keck School of Medicine of University of Southern California" />
  <meta name="citation_author_orcid" content="0000-0002-3038-8005" />
  <meta name="citation_author" content="Zalman Vaksman" />
  <meta name="citation_author_institution" content="Division of Oncology, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author" content="Rebecca S. Kaufman" />
  <meta name="citation_author_institution" content="Division of Oncology, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Department of Bioinformatics and Health Informatics, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0001-8535-9730" />
  <meta name="citation_author" content="Allison P. Heath" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery in Biomedicine, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0002-2583-9668" />
  <meta name="twitter:creator" content="@allig8r" />
  <meta name="citation_author" content="Brian M. Ennis" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery in Biomedicine, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0002-2653-5009" />
  <meta name="citation_author" content="Mariarita Santi" />
  <meta name="citation_author_institution" content="Department of Pathology and Laboratory Medicine, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Department of Pathology and Laboratory Medicine, University of Pennsylvania Perelman School of Medicine" />
  <meta name="citation_author_orcid" content="0000-0002-6728-3450" />
  <meta name="citation_author" content="Anna R. Poetsch" />
  <meta name="citation_author_institution" content="Biotechnology Center, Technical University Dresden, Germany" />
  <meta name="citation_author_institution" content="National Center for Tumor Diseases, Dresden, Germany" />
  <meta name="citation_author_orcid" content="0000-0003-3056-4360" />
  <meta name="twitter:creator" content="@APoetsch" />
  <meta name="citation_author" content="Angela N. Viaene" />
  <meta name="citation_author_institution" content="Department of Pathology and Laboratory Medicine, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Department of Pathology and Laboratory Medicine, University of Pennsylvania Perelman School of Medicine" />
  <meta name="citation_author_orcid" content="0000-0001-6430-8360" />
  <meta name="citation_author" content="Douglas R. Stewart" />
  <meta name="citation_author_institution" content="Clinical Genetics Branch, Division of Cancer Epidemiology and Genetics, National Cancer Institute" />
  <meta name="citation_author_orcid" content="0000-0001-8193-1488" />
  <meta name="citation_author" content="Steven M. Foltz" />
  <meta name="citation_author_institution" content="Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania" />
  <meta name="citation_author_institution" content="Childhood Cancer Data Lab, Alex&#39;s Lemonade Stand Foundation, Bala Cynwyd, PA, USA" />
  <meta name="citation_author_orcid" content="0000-0002-9526-8194" />
  <meta name="citation_author" content="Payal Jain" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery in Biomedicine, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0002-5914-9083" />
  <meta name="twitter:creator" content="@jainpayal022" />
  <meta name="citation_author" content="Bo Zhang" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery in Biomedicine, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0002-0743-5379" />
  <meta name="citation_author" content="Shrivats Kannan" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery in Biomedicine, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0002-1460-920X" />
  <meta name="twitter:creator" content="@kshrivats" />
  <meta name="citation_author" content="Michael Prados" />
  <meta name="citation_author_institution" content="University of California, San Francisco, San Francisco, CA, USA" />
  <meta name="citation_author_orcid" content="0000-0002-9630-2075" />
  <meta name="citation_author" content="Jena V. Lilly" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery in Biomedicine, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0003-1439-6045" />
  <meta name="twitter:creator" content="@jvlilly" />
  <meta name="citation_author" content="Sabine Mueller" />
  <meta name="citation_author_institution" content="University of California, San Francisco, San Francisco, CA" />
  <meta name="citation_author_orcid" content="0000-0002-3452-5150" />
  <meta name="citation_author" content="Adam C. Resnick" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery in Biomedicine, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0003-0436-4189" />
  <meta name="twitter:creator" content="@adamcresnick" />
  <meta name="citation_author" content="Casey S. Greene" />
  <meta name="citation_author_institution" content="Department of Systems Pharmacology and Translational Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA, USA" />
  <meta name="citation_author_institution" content="Childhood Cancer Data Lab, Alex&#39;s Lemonade Stand Foundation, Bala Cynwyd, PA, USA" />
  <meta name="citation_author_institution" content="Center for Health AI, University of Colorado School of Medicine, Aurora, CO, USA" />
  <meta name="citation_author_institution" content="Department of Biomedical Informatics, University of Colorado School of Medicine, Aurora, CO, USA" />
  <meta name="citation_author_orcid" content="0000-0001-8713-9213" />
  <meta name="twitter:creator" content="@greenescientist" />
  <meta name="citation_author" content="Jo Lynne Rokita" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery in Biomedicine, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Department of Bioinformatics and Health Informatics, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0003-2171-3627" />
  <meta name="twitter:creator" content="@jolynnerokita" />
  <meta name="citation_author" content="Jaclyn N. Taroni" />
  <meta name="citation_author_institution" content="Childhood Cancer Data Lab, Alex&#39;s Lemonade Stand Foundation, Bala Cynwyd, PA, USA" />
  <meta name="citation_author_orcid" content="0000-0003-4734-4508" />
  <meta name="twitter:creator" content="@jaclyn_taroni" />
  <meta name="citation_author" content="Children&#39;s Brain Tumor Network" />
  <meta name="citation_author" content="Pacific Pediatric Neuro-Oncology Consortium" />
  <link rel="canonical" href="https://AlexsLemonade.github.io/OpenPBTA-manuscript/" />
  <meta property="og:url" content="https://AlexsLemonade.github.io/OpenPBTA-manuscript/" />
  <meta property="twitter:url" content="https://AlexsLemonade.github.io/OpenPBTA-manuscript/" />
  <meta name="citation_fulltext_html_url" content="https://AlexsLemonade.github.io/OpenPBTA-manuscript/" />
  <meta name="citation_pdf_url" content="https://AlexsLemonade.github.io/OpenPBTA-manuscript/manuscript.pdf" />
  <link rel="alternate" type="application/pdf" href="https://AlexsLemonade.github.io/OpenPBTA-manuscript/manuscript.pdf" />
  <link rel="alternate" type="text/html" href="https://AlexsLemonade.github.io/OpenPBTA-manuscript/v/1a64757484bac87c8f5ebf667e2c85030720273a/" />
  <meta name="manubot_html_url_versioned" content="https://AlexsLemonade.github.io/OpenPBTA-manuscript/v/1a64757484bac87c8f5ebf667e2c85030720273a/" />
  <meta name="manubot_pdf_url_versioned" content="https://AlexsLemonade.github.io/OpenPBTA-manuscript/v/1a64757484bac87c8f5ebf667e2c85030720273a/manuscript.pdf" />
  <meta property="og:type" content="article" />
  <meta property="twitter:card" content="summary_large_image" />
  <link rel="icon" type="image/png" sizes="192x192" href="https://manubot.org/favicon-192x192.png" />
  <link rel="mask-icon" href="https://manubot.org/safari-pinned-tab.svg" color="#ad1457" />
  <meta name="theme-color" content="#ad1457" />
  <!-- end Manubot generated metadata -->
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
</head>
<body>
<header id="title-block-header">
<h1 class="title">OpenPBTA: An Open Pediatric Brain Tumor Atlas</h1>
</header>
<p><small><em>
This manuscript
(<a href="https://AlexsLemonade.github.io/OpenPBTA-manuscript/v/1a64757484bac87c8f5ebf667e2c85030720273a/">permalink</a>)
was automatically generated
from <a href="https://github.com/AlexsLemonade/OpenPBTA-manuscript/tree/1a64757484bac87c8f5ebf667e2c85030720273a">AlexsLemonade/OpenPBTA-manuscript@1a64757</a>
on February 24, 2023.
</em></small></p>
<h2 id="authors">Authors</h2>
<ul>
<li><p><strong>Joshua A. Shapiro</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-6224-0347">0000-0002-6224-0347</a>
· <img src="images/github.png" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/jashapiro">jashapiro</a>
· <img src="images/twitter.png" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/jashapiro">jashapiro</a><br>
<small>
Childhood Cancer Data Lab, Alex’s Lemonade Stand Foundation, Bala Cynwyd, PA, USA
· Funded by Alex’s Lemonade Stand Foundation Childhood Cancer Data Lab (CCDL)
</small></p></li>
<li><p><strong>Krutika S. Gaonkar</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-0838-2405">0000-0003-0838-2405</a>
· <img src="images/github.png" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/kgaonkar6">kgaonkar6</a>
· <img src="images/twitter.png" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/aggokittu">aggokittu</a><br>
<small>
Center for Data-Driven Discovery in Biomedicine, Children’s Hospital of Philadelphia; Division of Neurosurgery, Children’s Hospital of Philadelphia; Department of Bioinformatics and Health Informatics, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Candace L. Savonen</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-6331-7070">0000-0001-6331-7070</a>
· <img src="images/github.png" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/cansavvy">cansavvy</a>
· <img src="images/twitter.png" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/cansavvy">cansavvy</a><br>
<small>
Childhood Cancer Data Lab, Alex’s Lemonade Stand Foundation, Bala Cynwyd, PA, USA; Fred Hutchinson Cancer Center, Seattle, WA, USA
· Funded by Alex’s Lemonade Stand Foundation Childhood Cancer Data Lab (CCDL)
</small></p></li>
<li><p><strong>Stephanie J. Spielman</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-9090-4788">0000-0002-9090-4788</a>
· <img src="images/github.png" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/sjspielman">sjspielman</a>
· <img src="images/twitter.png" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/stephspiel">stephspiel</a><br>
<small>
Childhood Cancer Data Lab, Alex’s Lemonade Stand Foundation, Bala Cynwyd, PA, USA<sup>+</sup>; Rowan University, Glassboro, NJ, USA
· Funded by Alex’s Lemonade Stand Foundation Childhood Cancer Data Lab (CCDL)
<br>
<sup>+</sup>Current affiliation
</small></p></li>
<li><p><strong>Chante J. Bethell</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-9653-8128">0000-0001-9653-8128</a>
· <img src="images/github.png" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/cbethell">cbethell</a>
· <img src="images/twitter.png" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/cjbethell">cjbethell</a><br>
<small>
Childhood Cancer Data Lab, Alex’s Lemonade Stand Foundation, Bala Cynwyd, PA, USA
· Funded by Alex’s Lemonade Stand Foundation Childhood Cancer Data Lab (CCDL)
</small></p></li>
<li><p><strong>Run Jin</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-8958-9266">0000-0002-8958-9266</a>
· <img src="images/github.png" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/runjin326">runjin326</a>
· <img src="images/twitter.png" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/runjin">runjin</a><br>
<small>
Center for Data-Driven Discovery in Biomedicine, Children’s Hospital of Philadelphia; Division of Neurosurgery, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Komal S. Rathi</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-5534-6904">0000-0001-5534-6904</a>
· <img src="images/github.png" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/komalsrathi">komalsrathi</a>
· <img src="images/twitter.png" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/komalsrathi">komalsrathi</a><br>
<small>
Center for Data-Driven Discovery in Biomedicine, Children’s Hospital of Philadelphia; Department of Bioinformatics and Health Informatics, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Yuankun Zhu</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-2455-9525">0000-0002-2455-9525</a>
· <img src="images/github.png" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/yuankunzhu">yuankunzhu</a>
· <img src="images/twitter.png" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/zhuyuankun">zhuyuankun</a><br>
<small>
Center for Data-Driven Discovery in Biomedicine, Children’s Hospital of Philadelphia; Division of Neurosurgery, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Laura E. Egolf</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-7103-4801">0000-0002-7103-4801</a>
· <img src="images/github.png" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/LauraEgolf">LauraEgolf</a>
· <img src="images/twitter.png" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/LauraEgolf">LauraEgolf</a><br>
<small>
Cell and Molecular Biology Graduate Group, Perelman School of Medicine at the University of Pennsylvania; Division of Oncology, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Bailey K. Farrow</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-6727-6333">0000-0001-6727-6333</a>
· <img src="images/github.png" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/baileyckelly">baileyckelly</a><br>
<small>
Center for Data-Driven Discovery in Biomedicine, Children’s Hospital of Philadelphia; Division of Neurosurgery, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Daniel P. Miller</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-2032-4358">0000-0002-2032-4358</a>
· <img src="images/github.png" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/dmiller15">dmiller15</a><br>
<small>
Center for Data-Driven Discovery in Biomedicine, Children’s Hospital of Philadelphia; Division of Neurosurgery, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Yang Yang</strong><br>
· <img src="images/github.png" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/yangyangclover">yangyangclover</a><br>
<small>
Ben May Department for Cancer Research, University of Chicago, Chicago IL, USA
</small></p></li>
<li><p><strong>Tejaswi Koganti</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-7733-6480">0000-0002-7733-6480</a>
· <img src="images/github.png" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/tkoganti">tkoganti</a><br>
<small>
Center for Data-Driven Discovery in Biomedicine, Children’s Hospital of Philadelphia; Division of Neurosurgery, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Nighat Noureen</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-7495-8201">0000-0001-7495-8201</a>
· <img src="images/github.png" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/NNoureen">NNoureen</a><br>
<small>
Greehey Children’s Cancer Research Institute, UT Health San Antonio
</small></p></li>
<li><p><strong>Mateusz P. Koptyra</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-3857-6633">0000-0002-3857-6633</a>
· <img src="images/github.png" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/mkoptyra">mkoptyra</a>
· <img src="images/twitter.png" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/koptyram">koptyram</a><br>
<small>
Center for Data-Driven Discovery in Biomedicine, Children’s Hospital of Philadelphia; Division of Neurosurgery, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Nhat Duong</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-2852-4263">0000-0003-2852-4263</a>
· <img src="images/github.png" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/fingerfen">fingerfen</a>
· <img src="images/twitter.png" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/asiannhat">asiannhat</a><br>
<small>
Department of Bioinformatics and Health Informatics, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Adam A. Kraya</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-8526-5694">0000-0002-8526-5694</a>
· <img src="images/github.png" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/aadamk">aadamk</a><br>
<small>
Center for Data-Driven Discovery in Biomedicine, Children’s Hospital of Philadelphia; Division of Neurosurgery, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Cassie Kline</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-7765-7690">0000-0001-7765-7690</a>
· <img src="images/twitter.png" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/cassiekmd">cassiekmd</a><br>
<small>
Division of Oncology, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Yiran Guo</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-6549-8589">0000-0002-6549-8589</a>
· <img src="images/github.png" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/Yiran-Guo">Yiran-Guo</a>
· <img src="images/twitter.png" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/YiranGuo3">YiranGuo3</a><br>
<small>
Center for Data-Driven Discovery in Biomedicine, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Phillip B. Storm</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-7964-2449">0000-0002-7964-2449</a><br>
<small>
Center for Data-Driven Discovery in Biomedicine, Children’s Hospital of Philadelphia; Division of Neurosurgery, Children’s Hospital of Philadelphia
· Funded by Alex’s Lemonade Stand Foundation (Catalyst); Children’s Hospital of Philadelphia Division of Neurosurgery
</small></p></li>
<li><p><strong>Javad Nazarian</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-1951-9828">0000-0002-1951-9828</a><br>
<small>
Children’s National Research Institute, Washington, D.C.; George Washington University School of Medicine and Health Sciences, Washington, D.C.
</small></p></li>
<li><p><strong>Stephen C. Mack</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-9620-4742">0000-0001-9620-4742</a><br>
<small>
Department of Developmental Neurobiology, St. Jude Children’s Research Hospital
</small></p></li>
<li><p><strong>Pichai Raman</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-6948-2157">0000-0001-6948-2157</a>
· <img src="images/github.png" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/pichairaman">pichairaman</a>
· <img src="images/twitter.png" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/PichaiRaman">PichaiRaman</a><br>
<small>
Center for Data-Driven Discovery in Biomedicine, Children’s Hospital of Philadelphia; Department of Bioinformatics and Health Informatics, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Siyuan Zheng</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-1031-9424">0000-0002-1031-9424</a>
· <img src="images/github.png" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/syzheng">syzheng</a>
· <img src="images/twitter.png" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/zhengsiyuan">zhengsiyuan</a><br>
<small>
Greehey Children’s Cancer Research Institute, UT Health San Antonio
</small></p></li>
<li><p><strong>Peter J. Madsen</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-9266-3685">0000-0001-9266-3685</a>
· <img src="images/twitter.png" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/petermadsenmd">petermadsenmd</a><br>
<small>
Division of Neurosurgery, Children’s Hospital of Philadelphia; Center for Data-Driven Discovery in Biomedicine, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Nicholas Van Kuren</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-7414-9516">0000-0002-7414-9516</a>
· <img src="images/github.png" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/nicholasvk">nicholasvk</a><br>
<small>
Center for Data-Driven Discovery in Biomedicine, Children’s Hospital of Philadelphia; Division of Neurosurgery, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Shannon Robins</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-0594-1953">0000-0003-0594-1953</a><br>
<small>
Center for Data-Driven Discovery in Biomedicine, Children’s Hospital of Philadelphia; Division of Neurosurgery, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Xiaoyan Huang</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-7267-4512">0000-0001-7267-4512</a>
· <img src="images/github.png" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/HuangXiaoyan0106">HuangXiaoyan0106</a><br>
<small>
Center for Data-Driven Discovery in Biomedicine, Children’s Hospital of Philadelphia; Division of Neurosurgery, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Angela J. Waanders</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-0571-2889">0000-0002-0571-2889</a>
· <img src="images/github.png" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/awaanders">awaanders</a><br>
<small>
Division of Hematology, Oncology, Neuro-Oncology, and Stem Cell Transplant, Ann &amp; Robert H Lurie Children’s Hospital of Chicago; Department of Pediatrics, Northwestern University Feinberg School of Medicine
</small></p></li>
<li><p><strong>Jung Kim</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-6274-2841">0000-0001-6274-2841</a><br>
<small>
Clinical Genetics Branch, Division of Cancer Epidemiology and Genetics, National Cancer Institute
</small></p></li>
<li><p><strong>Gregory P. Way</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-0503-9348">0000-0002-0503-9348</a>
· <img src="images/github.png" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/gwaybio">gwaybio</a>
· <img src="images/twitter.png" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/gwaybio">gwaybio</a><br>
<small>
Department of Biomedical Informatics, University of Colorado School of Medicine, Aurora, CO, USA
</small></p></li>
<li><p><strong>Christopher Blackden</strong><br>
· <img src="images/github.png" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/devbyaccident">devbyaccident</a><br>
<small>
Center for Data-Driven Discovery in Biomedicine, Children’s Hospital of Philadelphia; Division of Neurosurgery, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Eric Wafula</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-8073-3797">0000-0001-8073-3797</a>
· <img src="images/github.png" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/ewafula">ewafula</a><br>
<small>
Department of Bioinformatics and Health Informatics, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Noel Coleman</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-6454-1285">0000-0001-6454-1285</a><br>
<small>
Center for Data-Driven Discovery in Biomedicine, Children’s Hospital of Philadelphia; Division of Neurosurgery, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Derek Hanson</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-0024-5142">0000-0002-0024-5142</a><br>
<small>
Hackensack Meridian School of Medicine; Hackensack University Medical Center
</small></p></li>
<li><p><strong>Carl J. Koschmann</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-0825-7615">0000-0002-0825-7615</a><br>
<small>
Department of Pediatrics, University of Michigan Health, Ann Arbor, MI; Pediatric Hematology Oncology, Mott Children’s Hospital, Ann Arbor, MI
</small></p></li>
<li><p><strong>Sharon J. Diskin</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-7200-8939">0000-0002-7200-8939</a>
· <img src="images/github.png" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/sdiskin">sdiskin</a>
· <img src="images/twitter.png" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/sjdiskin">sjdiskin</a><br>
<small>
Division of Oncology, Children’s Hospital of Philadelphia; Department of Pediatrics, University of Pennsylvania
</small></p></li>
<li><p><strong>Rishi R. Lulla</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-4109-2207">0000-0003-4109-2207</a><br>
<small>
Division of Hematology/Oncology, Hasbro Children’s Hospital; Department of Pediatrics, The Warren Alpert School of Brown University, Providence, Rhode Island
</small></p></li>
<li><p><strong>Miguel A. Brown</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-6782-1442">0000-0001-6782-1442</a>
· <img src="images/github.png" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/migbro">migbro</a>
· <img src="images/twitter.png" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/migbro">migbro</a><br>
<small>
Center for Data-Driven Discovery in Biomedicine, Children’s Hospital of Philadelphia; Division of Neurosurgery, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Jessica Wong</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-1508-7631">0000-0003-1508-7631</a>
· <img src="images/github.png" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/wongjessica93">wongjessica93</a>
· <img src="images/twitter.png" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/jessicawongbfx">jessicawongbfx</a><br>
<small>
Center for Data-Driven Discovery in Biomedicine, Children’s Hospital of Philadelphia; Division of Neurosurgery, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Jennifer L. Mason</strong><br>
· <img src="images/github.png" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/jenn0307">jenn0307</a>
· <img src="images/twitter.png" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/jenn0307">jenn0307</a><br>
<small>
Center for Data-Driven Discovery in Biomedicine, Children’s Hospital of Philadelphia; Division of Neurosurgery, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Laura Scolaro</strong><br><br>
<small>
Division of Oncology, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Meen Chul Kim</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-0308-783X">0000-0002-0308-783X</a>
· <img src="images/github.png" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/liberaliscomputing">liberaliscomputing</a><br>
<small>
Center for Data-Driven Discovery in Biomedicine, Children’s Hospital of Philadelphia; Division of Neurosurgery, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Hongbo M. Xie</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-2223-0029">0000-0003-2223-0029</a>
· <img src="images/github.png" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/xiehongbo">xiehongbo</a>
· <img src="images/twitter.png" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/xiehb">xiehb</a><br>
<small>
Department of Bioinformatics and Health Informatics, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Brian R. Rood</strong><br><br>
<small>
Children’s National Research Institute, Washington, D.C.; George Washington University School of Medicine and Health Sciences, Washington, D.C.
</small></p></li>
<li><p><strong>Kristina A. Cole</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-0064-2882">0000-0003-0064-2882</a><br>
<small>
Division of Oncology, Children’s Hospital of Philadelphia, Philadelphia, PA; Department of Pediatrics, University of Pennsylvania, Philadelphia, PA; Abramson Family Cancer Research Institute, Perelman School of Medicine at the University of Pennsylvania, Philadelphia, PA
</small></p></li>
<li><p><strong>Ashley S. Margol</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-3038-8005">0000-0002-3038-8005</a><br>
<small>
Division of Hematology and Oncology, Children’s Hospital Los Angeles; Department of Pediatrics, Keck School of Medicine of University of Southern California
</small></p></li>
<li><p><strong>Zalman Vaksman</strong><br><br>
<small>
Division of Oncology, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Rebecca S. Kaufman</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-8535-9730">0000-0001-8535-9730</a>
· <img src="images/github.png" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/rebkau">rebkau</a><br>
<small>
Division of Oncology, Children’s Hospital of Philadelphia; Department of Bioinformatics and Health Informatics, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Allison P. Heath</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-2583-9668">0000-0002-2583-9668</a>
· <img src="images/github.png" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/allisonheath">allisonheath</a>
· <img src="images/twitter.png" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/allig8r">allig8r</a><br>
<small>
Center for Data-Driven Discovery in Biomedicine, Children’s Hospital of Philadelphia; Division of Neurosurgery, Children’s Hospital of Philadelphia
· Funded by NIH U2C HL138346-03; NCI/NIH Contract No. 75N91019D00024, Task Order No. 75N91020F00003; Australian Government, Department of Education
</small></p></li>
<li><p><strong>Brian M. Ennis</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-2653-5009">0000-0002-2653-5009</a>
· <img src="images/github.png" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/bmennis">bmennis</a><br>
<small>
Center for Data-Driven Discovery in Biomedicine, Children’s Hospital of Philadelphia; Division of Neurosurgery, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Mariarita Santi</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-6728-3450">0000-0002-6728-3450</a><br>
<small>
Department of Pathology and Laboratory Medicine, Children’s Hospital of Philadelphia; Department of Pathology and Laboratory Medicine, University of Pennsylvania Perelman School of Medicine
</small></p></li>
<li><p><strong>Anna R. Poetsch</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-3056-4360">0000-0003-3056-4360</a>
· <img src="images/github.png" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/arpoe">arpoe</a>
· <img src="images/twitter.png" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/APoetsch">APoetsch</a><br>
<small>
Biotechnology Center, Technical University Dresden, Germany; National Center for Tumor Diseases, Dresden, Germany
· Funded by The St. Anna Kinderkrebsforschung, Austria; The Mildred Scheel Early Career Center Dresden P2, funded by the German Cancer Aid
</small></p></li>
<li><p><strong>Angela N. Viaene</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-6430-8360">0000-0001-6430-8360</a><br>
<small>
Department of Pathology and Laboratory Medicine, Children’s Hospital of Philadelphia; Department of Pathology and Laboratory Medicine, University of Pennsylvania Perelman School of Medicine
</small></p></li>
<li><p><strong>Douglas R. Stewart</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-8193-1488">0000-0001-8193-1488</a><br>
<small>
Clinical Genetics Branch, Division of Cancer Epidemiology and Genetics, National Cancer Institute
</small></p></li>
<li><p><strong>Steven M. Foltz</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-9526-8194">0000-0002-9526-8194</a>
· <img src="images/github.png" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/envest">envest</a><br>
<small>
Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania; Childhood Cancer Data Lab, Alex’s Lemonade Stand Foundation, Bala Cynwyd, PA, USA
· Funded by Alex’s Lemonade Stand Foundation GR-000002471; National Institutes of Health K12GM081259
</small></p></li>
<li><p><strong>Payal Jain</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-5914-9083">0000-0002-5914-9083</a>
· <img src="images/github.png" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/jainpayal022">jainpayal022</a>
· <img src="images/twitter.png" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/jainpayal022">jainpayal022</a><br>
<small>
Center for Data-Driven Discovery in Biomedicine, Children’s Hospital of Philadelphia; Division of Neurosurgery, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Bo Zhang</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-0743-5379">0000-0002-0743-5379</a>
· <img src="images/github.png" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/zhangb1">zhangb1</a><br>
<small>
Center for Data-Driven Discovery in Biomedicine, Children’s Hospital of Philadelphia; Division of Neurosurgery, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Shrivats Kannan</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-1460-920X">0000-0002-1460-920X</a>
· <img src="images/github.png" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/shrivatsk">shrivatsk</a>
· <img src="images/twitter.png" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/kshrivats">kshrivats</a><br>
<small>
Center for Data-Driven Discovery in Biomedicine, Children’s Hospital of Philadelphia; Division of Neurosurgery, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Michael Prados</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-9630-2075">0000-0002-9630-2075</a><br>
<small>
University of California, San Francisco, San Francisco, CA, USA
</small></p></li>
<li><p><strong>Jena V. Lilly</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-1439-6045">0000-0003-1439-6045</a>
· <img src="images/github.png" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/jvlilly">jvlilly</a>
· <img src="images/twitter.png" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/jvlilly">jvlilly</a><br>
<small>
Center for Data-Driven Discovery in Biomedicine, Children’s Hospital of Philadelphia; Division of Neurosurgery, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Sabine Mueller</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-3452-5150">0000-0002-3452-5150</a><br>
<small>
University of California, San Francisco, San Francisco, CA
</small></p></li>
<li><p><strong>Adam C. Resnick</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-0436-4189">0000-0003-0436-4189</a>
· <img src="images/github.png" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/adamcresnick">adamcresnick</a>
· <img src="images/twitter.png" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/adamcresnick">adamcresnick</a><br>
<small>
Center for Data-Driven Discovery in Biomedicine, Children’s Hospital of Philadelphia; Division of Neurosurgery, Children’s Hospital of Philadelphia
· Funded by Alex’s Lemonade Stand Foundation (Catalyst); Children’s Brain Tumor Network; NIH 3P30 CA016520-44S5, U2C HL138346-03, U24 CA220457-03; NCI/NIH Contract No. 75N91019D00024, Task Order No. 75N91020F00003; Children’s Hospital of Philadelphia Division of Neurosurgery
</small></p></li>
<li><p><strong>Casey S. Greene</strong><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-8713-9213">0000-0001-8713-9213</a>
· <img src="images/github.png" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/cgreene">cgreene</a>
· <img src="images/twitter.png" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/greenescientist">greenescientist</a><br>
<small>
Department of Systems Pharmacology and Translational Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA, USA; Childhood Cancer Data Lab, Alex’s Lemonade Stand Foundation, Bala Cynwyd, PA, USA; Center for Health AI, University of Colorado School of Medicine, Aurora, CO, USA; Department of Biomedical Informatics, University of Colorado School of Medicine, Aurora, CO, USA
· Funded by Alex’s Lemonade Stand Foundation Childhood Cancer Data Lab (CCDL)
</small></p></li>
<li><p><strong>Jo Lynne Rokita</strong>
<sup><a href="#correspondence">✉</a></sup><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-2171-3627">0000-0003-2171-3627</a>
· <img src="images/github.png" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/jharenza">jharenza</a>
· <img src="images/twitter.png" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/jolynnerokita">jolynnerokita</a><br>
<small>
Center for Data-Driven Discovery in Biomedicine, Children’s Hospital of Philadelphia; Division of Neurosurgery, Children’s Hospital of Philadelphia; Department of Bioinformatics and Health Informatics, Children’s Hospital of Philadelphia
· Funded by Alex’s Lemonade Stand Foundation (Young Investigator, Catalyst); NCI/NIH Contract No. 75N91019D00024, Task Order No. 75N91020F00003
</small></p></li>
<li><p><strong>Jaclyn N. Taroni</strong>
<sup><a href="#correspondence">✉</a></sup><br>
<img src="images/orcid.png" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-4734-4508">0000-0003-4734-4508</a>
· <img src="images/github.png" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/jaclyn-taroni">jaclyn-taroni</a>
· <img src="images/twitter.png" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/jaclyn_taroni">jaclyn_taroni</a><br>
<small>
Childhood Cancer Data Lab, Alex’s Lemonade Stand Foundation, Bala Cynwyd, PA, USA
· Funded by Alex’s Lemonade Stand Foundation Childhood Cancer Data Lab (CCDL)
</small></p></li>
<li><p><strong>Children’s Brain Tumor Network</strong><br><br>
<small>
</small></p></li>
<li><p><strong>Pacific Pediatric Neuro-Oncology Consortium</strong><br><br>
<small>
</small></p></li>
</ul>
<h2 id="contact-information">Contact information</h2>
<div id="correspondence">
<p>✉Correspondence:
Jo Lynne Rokita &lt;rokita@chop.edu&gt;,
Jaclyn N. Taroni &lt;jaclyn.taroni@ccdatalab.org&gt;.</p>
</div>
<!-- Cell Genomics author instructions: https://marlin-prod.literatumonline.com/pb-assets/journals/research/cell-genomics/cell-genomics_IfA.pdf -->
<h2 id="in-brief">In Brief</h2>
<!-- aka the eTOC blurb. A short summary that describes the context and significance of the findings for the broader readership in 80 words or less. -->
<p>The OpenPBTA is a global, collaborative open-science initiative which brought together researchers and clinicians to genomically characterize 1,074 pediatric brain tumors and 22 patient-derived cell lines.
Shapiro, et. al create over 40 open-source, scalable modules to perform cancer genomics analyses and provide a richly-annotated somatic dataset across 58 brain tumor histologies.
The OpenPBTA framework can be used as a model for large-scale data integration to inform basic research, therapeutic target identification, and clinical translation.</p>
<h2 id="highlights">Highlights</h2>
<!-- Highlights are bullet points that convey the core findings of your paper. You may include up to four highlights. The length of each highlight cannot exceed 85 characters (including spaces).-->
<p>OpenPBTA collaborative analyses establish resource for 1,074 pediatric brain tumors <!--SAMPLECOUNT--></p>
<p>NGS-based WHO-aligned integrated diagnoses generated for 644 of 1,074 tumors <!--SAMPLECOUNT--></p>
<p>RNA-Seq analysis infers medulloblastoma subtypes, <em>TP53</em> status, and telomerase activity</p>
<p>OpenPBTA will accelerate therapeutic translation of genomic insights</p>
<h2 id="summary">Summary</h2>
<!-- The summary is a single paragraph no longer than 150 words. An effective summary includes the following elements: (1) a brief background of the question that avoids statements about how a process is not well understood; (2) a description of the results and approaches/model systems framed in the context of their conceptual interest; and (3) an indication of the broader significance of the work. We discourage novelty claims (e.g., use of the word “novel”) because they are overused, tend not to add meaning, and are difficult to verify. Please do not include references in the summary. CURRENT WORD COUNT 145 -->
<p>Pediatric brain and spinal cancer are the leading disease-related cause of death in children, thus we urgently need curative therapeutic strategies for these tumors.
To accelerate such discoveries, the Children’s Brain Tumor Network (CBTN) and Pacific Pediatric Neuro-Oncology Consortium (PNOC) created a systematic process for tumor biobanking, model generation, and sequencing with immediate access to harmonized data.
We leverage these data to create OpenPBTA, an open collaborative project which establishes over 40 scalable analysis modules to genomically characterize 1,074 pediatric brain tumors.
Transcriptomic classification reveals that <em>TP53</em> loss is a significant marker for poor overall survival in ependymomas and H3 K28-mutant diffuse midline gliomas and further identifies universal <em>TP53</em> dysregulation in mismatch repair-deficient hypermutant high-grade gliomas.
OpenPBTA is a foundational analysis platform actively being applied to other pediatric cancers and PNOC molecular tumor board decision-making, making it an invaluable resource to the pediatric oncology community.</p>
<h2 id="keywords">Keywords</h2>
<!-- We encourage you to include up to ten keywords with your paper. These keywords will be associated with your paper on Cell Press platforms and on PubMed. These keywords should be listed in the manuscript after the summary, separated by commas. -->
<p>pediatric cancer, brain tumors, somatic variation, open science, reproducibility, classification, tumor atlas</p>
<h2 id="introduction">Introduction</h2>
<p>Pediatric brain and spinal cord tumors are collectively the second most common malignancy in children after leukemia, and they represent the leading disease-related cause of death in children<span class="citation" data-cites="nAMOAA9j"><sup><a href="#ref-nAMOAA9j" role="doc-biblioref">1</a></sup></span>.
Five-year survival rates vary widely across different histologic and molecular classifications of brain tumors.
For example, most high-grade gliomas carry a universally fatal prognosis, while children with pilocytic astrocytoma have an estimated 10-year survival rate of 92%<span class="citation" data-cites="1H6hugOV3"><sup><a href="#ref-1H6hugOV3" role="doc-biblioref">2</a></sup></span>.
Moreover, estimates from 2009 suggest that children and adolescents aged 0-19 with brain tumors in the United States have lost an average of 47,631 years of potential life<span class="citation" data-cites="7p58e5Wj"><sup><a href="#ref-7p58e5Wj" role="doc-biblioref">3</a></sup></span>.</p>
<p>The low survival rates for some pediatric tumors are clearly multifactorial, explained partly by our lack of comprehensive understanding of the ever-evolving array of brain tumor molecular subtypes, difficulty drugging these tumors, and the shortage of drugs specifically labeled for pediatric malignancies.
Historically, some of the most fatal, inoperable brain tumors, such as diffuse intrinsic pontine gliomas (DIPGs), were not routinely biopsied due to perceived risks of biopsy and the paucity of therapeutic options that would require tissue.
Limited access to tissue to develop patient-derived cell lines and mouse models has been a barrier to research.
Furthermore, the incidence of any single brain tumor molecular subtype is relatively low due to the rarity of pediatric tumors in general.</p>
<p>To address these long-standing barriers, multiple national and international consortia have come together to uniformly collect clinically-annotated surgical biosamples and associated germline materials as part of both observational and interventional clinical trials.<br />
Such accessible, centralized resources enable collaborative sharing of specimens and data across rare cancer subtypes to accelerate breakthroughs and clinical translation.
The creation of the Pediatric Brain Tumor Atlas (PBTA) in 2018, led by the Children’s Brain Tumor Network (CBTN, cbtn.org)<span class="citation" data-cites="U5WisTG5"><sup><a href="#ref-U5WisTG5" role="doc-biblioref">4</a></sup></span> and the Pacific Pediatric Neuro-Oncology Consortium (PNOC, pnoc.us) is one such effort that builds on nearly 10 years of multi-institutional enrollment, sample collection, and clinical followup across more than 30 institutions.
Just as cooperation is required to share specimens and data, rigorous cancer genomic analysis requires collaboration among researchers with distinct expertise, such as computational scientists, bench scientists, clinicians, and pathologists.</p>
<p>Although there has been significant progress in recent years to elucidate the landscape of somatic variation responsible for pediatric brain tumor formation and progression, translation of therapeutic agents to phase II or III clinical trials and subsequent FDA approvals have not kept pace.
Within the last 20 years, the FDA has approved only seven targeted agents which can be used to treat pediatric brain tumors: mTOR inhibitor everolimus, for subependymal giant cell astrocytoma; anti-PD-1 immunotherapy pembrolizumab, for microsatellite instability–high or mismatch repair–deficient tumors; NTRK inhibitors larotrectinib and entrectinib, for tumors with an NTRK 1/2/3 gene fusions; MEK1/2 inhibitor selumetinib, for neurofibromatosis type 1 (NF1) and symptomatic, inoperable plexiform neurofibromas, and combination therapy MEK1/2 inhibitor trametinib and BRAF/CRAF inhibitor dabrafenib for unresectable or metastatic progressive tumors with BRAF V600E mutations<span class="citation" data-cites="12pkJWML4"><sup><a href="#ref-12pkJWML4" role="doc-biblioref">5</a></sup></span>.</p>
<p>This is, in part, due to pharmaceutical company priorities and concerns regarding toxicity, making it challenging for researchers to obtain to new therapeutic agents for pediatric clinical trials.
Critically, as of August 18, 2020, an amendment to the Pediatric Research Equity Act called the “Research to Accelerate Cures and Equity (RACE) for Children Act” mandates that all new adult oncology drugs also be tested in children when the molecular targets are relevant to a particular childhood cancer.
The regulatory change introduced by the RACE Act, coupled with the identification of putative molecular targets in pediatric cancers through genomic characterization, is poised to accelerate identification of novel and effective therapeutic for pediatric diseases that have otherwise been overlooked.</p>
<p>To leverage diverse scientific and analytical expertise to analyze the PBTA data, we created an open science model and incorporated features such as analytical code review<span class="citation" data-cites="1ApRLzuh0 rAFELHQ7"><sup><a href="#ref-1ApRLzuh0" role="doc-biblioref">6</a>,<a href="#ref-rAFELHQ7" role="doc-biblioref">7</a></sup></span> and continuous integration to test data and code<span class="citation" data-cites="Qh7xTLwz rAFELHQ7"><sup><a href="#ref-rAFELHQ7" role="doc-biblioref">7</a>,<a href="#ref-Qh7xTLwz" role="doc-biblioref">8</a></sup></span> to improve reproducibility throughout the life cycle of our project, termed OpenPBTA.</p>
<p>We anticipated that a model of open collaboration would enhance the value of our effort to the pediatric brain tumor research community and provide a framework for continuous, accelerated translation of pediatric brain tumor datasets.
Openly sharing data and code in real time allows others to build upon the work more rapidly, and publications that include data and code sharing are poised for greater impact<span class="citation" data-cites="1CcAUn3Lu Vj0v0GeK"><sup><a href="#ref-1CcAUn3Lu" role="doc-biblioref">9</a>,<a href="#ref-Vj0v0GeK" role="doc-biblioref">10</a></sup></span>.
Here, we present a comprehensive, collaborative, open genomic analysis of 1,074 tumors and 22 cell lines, comprised of 58 distinct brain tumor histologies from 943 patients.
The data and containerized infrastructure of OpenPBTA have been instrumental for discovery and translational research studies<span class="citation" data-cites="XYoocEyx 9mO4Op62 kIFuKyuK lNLcNX6n"><sup><a href="#ref-XYoocEyx" role="doc-biblioref">11</a>–<a href="#ref-lNLcNX6n" role="doc-biblioref">14</a></sup></span>, are actively integrated into PNOC molecular tumor board decision-making, and are a foundational layer for the NCI’s Childhood Cancer Data Initiative’s (CCDI) pediatric Molecular Targets Platform (<a href="https://moleculartargets.ccdi.cancer.gov/">https://moleculartargets.ccdi.cancer.gov/</a>) recently built in support of the RACE Act<span class="citation" data-cites="15d8OT6MC"><sup><a href="#ref-15d8OT6MC" role="doc-biblioref">15</a></sup></span>.
We anticipate OpenPBTA will be an invaluable resource to the pediatric oncology community.</p>
<h2 id="results">Results</h2>
<h3 id="crowd-sourced-somatic-analyses-to-create-an-open-pediatric-brain-tumor-atlas">Crowd-sourced Somatic Analyses to Create an Open Pediatric Brain Tumor Atlas</h3>
<p>We previously performed whole genome sequencing (WGS), whole exome sequencing (WXS), and RNA sequencing (RNA-Seq) on matched tumor and normal tissues as well as selected cell lines<span class="citation" data-cites="DwCzMoBJ"><sup><a href="#ref-DwCzMoBJ" role="doc-biblioref">16</a></sup></span> from 943 patients from the Pediatric Brain Tumor Atlas (PBTA), consisting of 911 patients from the <a href="https://CBTN.org">Children’s Brain Tumor Network (CBTN)</a><span class="citation" data-cites="U5WisTG5"><sup><a href="#ref-U5WisTG5" role="doc-biblioref">4</a></sup></span> and 32 patients from the <a href="https://pnoc.us/">Pacific Pediatric Neuro-Oncology Consortium (PNOC)</a><span class="citation" data-cites="BLNV1DbL 9mO4Op62"><sup><a href="#ref-9mO4Op62" role="doc-biblioref">12</a>,<a href="#ref-BLNV1DbL" role="doc-biblioref">17</a></sup></span> (<strong>Figure <a href="#fig:Fig1">1</a>A</strong>).
We then harnessed, and built upon, the benchmarking efforts of the <a href="https://kidsfirstdrc.org/">Gabriella Miller Kids First Data Resource Center</a> to develop robust and reproducible data analysis workflows within the CAVATICA platform to perform primary somatic analyses including calling of single nucleotide variants (SNVs), copy number variants (CNVs), structural variants (SVs), and gene fusions, often implementing multiple complementary methods (<strong>Figure <a href="#fig:S1">S1</a></strong>) and <strong>STAR Methods</strong>).</p>
<p>To facilitate analysis and visualization of this large, diverse cohort, we further categorized 2016 World Health Organization (WHO) broad tumor histologies<span class="citation" data-cites="7sgrMadR"><sup><a href="#ref-7sgrMadR" role="doc-biblioref">18</a></sup></span> into smaller groupings we denote “cancer groups.”
A summarized view of the number of biospecimens per phase of therapy across different broad histologies and cancer groups is shown in (<strong>Figure <a href="#fig:Fig1">1</a>B</strong>).
We maintained a data release folder on Amazon S3, downloadable directly from S3 or through the open-access <a href="https://www.cavatica.org/">CAVATICA project</a>, with merged files for each analysis (See <strong>Data and code availability</strong> section).
As we produced new results (e.g., tumor mutation burden calculations) that we expected to be used across multiple analyses, or identified data issues, we created new data releases in a versioned manner.
We reran all manuscript-specific analysis modules with the latest data release (v23) prior to submission and subsequently created a GitHub repository-tagged release to ensure reproducibility.</p>
<p>A key innovative feature of this project has been its open contribution model used for both analyses (i.e., analytical code) and scientific manuscript writing, a model which can be utilized by code contributors within individual scientific groups and/or through collaboration.
We created a public Github analysis repository (<a href="https://github.com/AlexsLemonade/OpenPBTA-analysis">https://github.com/AlexsLemonade/OpenPBTA-analysis</a>) to hold all code associated with analyses downstream of the Kids First Data Resource Center workflows and a GitHub manuscript repository (<a href="https://github.com/AlexsLemonade/OpenPBTA-manuscript">https://github.com/AlexsLemonade/OpenPBTA-manuscript</a>) with Manubot<span class="citation" data-cites="YuJbg3zO"><sup><a href="#ref-YuJbg3zO" role="doc-biblioref">19</a></sup></span> integration to enable real-time manuscript creation using Markdown within GitHub.
With very few exceptions noted in their respective analysis module READMEs, most modules can be run locally, and since they are run in the project Docker®<span class="citation" data-cites="1CJTmbOqr"><sup><a href="#ref-1CJTmbOqr" role="doc-biblioref">20</a></sup></span> container, they can easily be scaled and/or used on an Amazon EC2 instance.
Importantly, all analyses and manuscript writing were conducted openly throughout the research project, allowing any researcher in the world the opportunity to contribute.</p>
<p>The process for analysis and manuscript contributions is outlined in <strong>Figure <a href="#fig:Fig1">1</a>C</strong>.
First, a potential contributor would propose an analysis by filing an issue in the GitHub analysis repository.
Next, organizers for the project, or other contributors with expertise, had the opportunity to provide feedback about the proposed analysis (<strong>Figure <a href="#fig:Fig1">1</a>C</strong>).
The contributor then made a copy (fork) of the analysis repository and added their proposed analysis code and results to their fork.
The contributor would formally request to include their analytical code and results to the main OpenPBTA analysis repository by filing a pull request on GitHub.
All pull requests to the analysis repository underwent peer review by organizers and/or other contributors to ensure scientific accuracy, maintainability, and readability of code and documentation (<strong>Figure <a href="#fig:Fig1">1</a>C-D</strong>).
Importantly, this peer review process entailed two or more analysts running the same code within the same Docker®<span class="citation" data-cites="1CJTmbOqr"><sup><a href="#ref-1CJTmbOqr" role="doc-biblioref">20</a></sup></span> container to ensure reproducibility of results that were derived from a specific data release.</p>
<p>The collaborative nature of the project required additional steps beyond peer review of analytical code to ensure consistent results for all collaborators and over time (<strong>Figure <a href="#fig:Fig1">1</a>D</strong>).
We leveraged Docker®<span class="citation" data-cites="1CJTmbOqr"><sup><a href="#ref-1CJTmbOqr" role="doc-biblioref">20</a></sup></span> and the Rocker project<span class="citation" data-cites="NazMcQpM"><sup><a href="#ref-NazMcQpM" role="doc-biblioref">21</a></sup></span> to maintain a consistent software development environment, creating a monolithic image that contained all dependencies necessary for analyses.
To ensure that new code would execute in the development environment, we used the continuous integration (CI) service CircleCI® to run analytical code on a small subset of data for testing before formal code review, allowing us to detect code bugs or sensitivity to changes in the underlying data.</p>
<p>We followed a similar process in our Manubot-powered<span class="citation" data-cites="YuJbg3zO"><sup><a href="#ref-YuJbg3zO" role="doc-biblioref">19</a></sup></span> manuscript repository for additions to the manuscript (<strong>Figure <a href="#fig:Fig1">1</a>C</strong>).
Contributors forked the manuscript repository, added proposed content to their branch, and filed pull requests to the main manuscript repository with their changes.
Similarly, pull requests underwent a peer review process for clarity and correctness, agreement with interpretation, and spell checking via Manubot.</p>
<div id="fig:Fig1" class="fignos">
<figure>
<img src="https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/361fef09c574942d3d5ef0668461b37e3ee6cba6/figures/pngs/figure1.png?sanitize=true" style="width:7in" alt="Figure 1: Overview of the OpenPBTA Project. A, The Children’s Brain Tumor Network and the Pacific Pediatric Neuro-Oncology Consortium collected tumors from 943 patients. To date, 22 cell lines were created from tumor tissue, and over 2000 specimens were sequenced (N = 1035 RNA-Seq, N = 940 WGS, and N = 32 WXS or targeted panel). Data was harmonized by the Kids First Data Resource Center using an Amazon S3 framework within CAVATICA. B, Stacked bar plot summary of the number of biospecimens per phase of therapy. Each panel denotes a broad histology and each bar denotes a cancer group. (Abbreviations: GNG = ganglioglioma, Other LGG = other low-grade glioma, PA = pilocytic astrocytoma, PXA = pleomorphic xanthoastrocytoma, SEGA = subependymal giant cell astrocytoma, DIPG = diffuse intrinsic pontine glioma, DMG = diffuse midline glioma, Other HGG = other high-grade glioma, ATRT = atypical teratoid rhabdoid tumor, MB = medulloblastoma, Other ET = other embryonal tumor, EPN = ependymoma, PNF = plexiform neurofibroma, DNET = dysembryoplastic neuroepithelial tumor, CRANIO = craniopharyngioma, EWS = Ewing sarcoma, CPP = choroid plexus papilloma). Only tumors with available descriptors were included. C, Overview of the open analysis and manuscript contribution model. In the analysis GitHub repository, a contributor would propose an analysis that other participants can comment on. Contributors would then implement the analysis and file a request to add their changes to the analysis repository (“pull request”). Pull requests underwent review for scientific rigor and correctness of implementation. Pull requests were additionally checked to ensure that all software dependencies were included and the code was not sensitive to underlying data changes using container and continuous integration technologies. Finally, a contributor would file a pull request documenting their methods and results to the Manubot-powered manuscript repository. Pull requests in the manuscript repository were also subject to review. D, A potential path for an analytical pull request. Arrows indicate revisions to a pull request. Prior to review, a pull request was tested for dependency installation and whether or not the code would execute. Pull requests also required approval by organizers and/or other contributors, who checked for scientific correctness. Panel A created with BioRender.com." />
<figcaption aria-hidden="true"><span>Figure 1:</span> <strong>Overview of the OpenPBTA Project.</strong> A, The Children’s Brain Tumor Network and the Pacific Pediatric Neuro-Oncology Consortium collected tumors from 943 patients. To date, 22 cell lines were created from tumor tissue, and over 2000 specimens were sequenced (N = 1035 RNA-Seq, N = 940 WGS, and N = 32 WXS or targeted panel). Data was harmonized by the Kids First Data Resource Center using an Amazon S3 framework within CAVATICA. B, Stacked bar plot summary of the number of biospecimens per phase of therapy. Each panel denotes a broad histology and each bar denotes a cancer group. (Abbreviations: GNG = ganglioglioma, Other LGG = other low-grade glioma, PA = pilocytic astrocytoma, PXA = pleomorphic xanthoastrocytoma, SEGA = subependymal giant cell astrocytoma, DIPG = diffuse intrinsic pontine glioma, DMG = diffuse midline glioma, Other HGG = other high-grade glioma, ATRT = atypical teratoid rhabdoid tumor, MB = medulloblastoma, Other ET = other embryonal tumor, EPN = ependymoma, PNF = plexiform neurofibroma, DNET = dysembryoplastic neuroepithelial tumor, CRANIO = craniopharyngioma, EWS = Ewing sarcoma, CPP = choroid plexus papilloma). Only tumors with available descriptors were included. C, Overview of the open analysis and manuscript contribution model. In the analysis GitHub repository, a contributor would propose an analysis that other participants can comment on. Contributors would then implement the analysis and file a request to add their changes to the analysis repository (“pull request”). Pull requests underwent review for scientific rigor and correctness of implementation. Pull requests were additionally checked to ensure that all software dependencies were included and the code was not sensitive to underlying data changes using container and continuous integration technologies. Finally, a contributor would file a pull request documenting their methods and results to the Manubot-powered manuscript repository. Pull requests in the manuscript repository were also subject to review. D, A potential path for an analytical pull request. Arrows indicate revisions to a pull request. Prior to review, a pull request was tested for dependency installation and whether or not the code would execute. Pull requests also required approval by organizers and/or other contributors, who checked for scientific correctness. Panel A created with BioRender.com.</figcaption>
</figure>
</div>
<h3 id="molecular-subtyping-of-openpbta-cns-tumors">Molecular Subtyping of OpenPBTA CNS Tumors</h3>
<p>Over the past two decades, experts in neuro-oncology have worked with the WHO to iteratively redefine the classifications of central nervous system (CNS) tumors<span class="citation" data-cites="pmid:11895036 14TDMX5t7"><sup><a href="#ref-14TDMX5t7" role="doc-biblioref">22</a>,<a href="#ref-pmid:11895036" role="doc-biblioref"><strong>pmid:11895036?</strong></a></sup></span>.
More recently, in 2016<span class="citation" data-cites="7sgrMadR"><sup><a href="#ref-7sgrMadR" role="doc-biblioref">18</a></sup></span>, molecular subtypes driven by genetic alterations have been integrated into these classifications.
In 2011, the Children’s Brain Tumor Tissue Consortium, now known as the Children’s Brain Tumor Network (CBTN), opened its protocol for brain tumor and matched normal sample collection.
Since the CBTN opened its collection protocol in 2011, before molecular data were integrated into classifications, the majority of the tumors within the OpenPBTA lacked molecular subtype annotations at the time of tissue collection.
Moreover, the OpenPBTA data does not yet feature methylation arrays which are increasingly used to inform molecular subtyping and cancer diagnosis.
Therefore, we created analysis modules to systematically consider key genomic features of tumor entities described by the WHO in 2016 or Ryall and colleagues<span class="citation" data-cites="ULPByXgG"><sup><a href="#ref-ULPByXgG" role="doc-biblioref">23</a></sup></span> for low-grade gliomas (LGGs).
Coupled with clinician and pathologist review, we generated research-grade integrated diagnoses for 60% (644/1074) of tumors with high confidence (<strong>Table S1</strong>) without the requirement for methylation, a major innovation of this project. <!--SAMPLECOUNT-->
This allowed us to update cancer diagnoses to align with WHO classifications (e.g., tumors formerly ascribed primitive neuro-ectodermal tumor [PNET] diagnoses), discover rarer tumor entities within the OpenPBTA (e.g., H3-mutant ependymoma, meningioma with <em>YAP1::FAM118B</em> fusion), as well as identify and correct data entry errors (e.g., an ETMR incorrectly entered as a medulloblastoma) and histologically mis-identified specimens (e.g., Ewing sarcoma sample labeled as a craniopharyngioma).
Uniquely, we used transcriptomic classification to subtype 122 medulloblastomas into SHH, WNT, Group 3, or Group 4 with <code>MedulloClassifier</code><span class="citation" data-cites="12DKhuCiy"><sup><a href="#ref-12DKhuCiy" role="doc-biblioref">24</a></sup></span> and <code>MM2S</code><span class="citation" data-cites="RmzXK4Cw"><sup><a href="#ref-RmzXK4Cw" role="doc-biblioref">25</a></sup></span>, achieving accuracies of 95% (41/43) and 91% (39/43), respectively.
These 43 “true positive” subtypes were manually curated from pathology reports by two independent reviewers.</p>
<p><strong>Table <a href="#tbl:Table1">1</a></strong> lists the number of tumors we subtyped within OpenPBTA, comprising low-grade gliomas (N = 290), high-grade gliomas (N = 141), embryonal tumors (N = 126), ependymomas (N = 33), tumors of sellar region (N = 27), mesenchymal non-meningothelial tumors (N = 11), glialneuronal tumors (N = 10), and chordomas (N = 6), where Ns represent unique tumors. <!--SAMPLECOUNT-->
For detailed methods, see <strong>STAR Methods</strong> and <strong>Figure <a href="#fig:S1">S1</a></strong>.</p>
<div id="tbl:Table1" class="tablenos">
<table id="tbl:Table1">
<caption><span>Table 1:</span> <strong>Molecular subtypes generated through the OpenPBTA project.</strong> Listed are broad tumor histologies, molecular subtypes generated, and number of patients and tumors subtyped within the OpenPBTA project. </caption>
<colgroup>
<col style="width: 40%" />
<col style="width: 41%" />
<col style="width: 9%" />
<col style="width: 7%" />
</colgroup>
<thead>
<tr class="header">
<th>Broad histology group</th>
<th>OpenPBTA molecular subtype</th>
<th>Patients</th>
<th>Tumors</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Chordoma</td>
<td>CHDM, conventional</td>
<td>2</td>
<td>2</td>
</tr>
<tr class="even">
<td>Chordoma</td>
<td>CHDM, poorly differentiated</td>
<td>2</td>
<td>4</td>
</tr>
<tr class="odd">
<td>Embryonal tumor</td>
<td>CNS Embryonal, NOS</td>
<td>13</td>
<td>13</td>
</tr>
<tr class="even">
<td>Embryonal tumor</td>
<td>CNS HGNET-MN1</td>
<td>1</td>
<td>1</td>
</tr>
<tr class="odd">
<td>Embryonal tumor</td>
<td>CNS NB-FOXR2</td>
<td>2</td>
<td>3</td>
</tr>
<tr class="even">
<td>Embryonal tumor</td>
<td>ETMR, C19MC-altered</td>
<td>5</td>
<td>5</td>
</tr>
<tr class="odd">
<td>Embryonal tumor</td>
<td>ETMR, NOS</td>
<td>1</td>
<td>1</td>
</tr>
<tr class="even">
<td>Embryonal tumor</td>
<td>MB, Group3</td>
<td>14</td>
<td>14</td>
</tr>
<tr class="odd">
<td>Embryonal tumor</td>
<td>MB, Group4</td>
<td>48</td>
<td>49</td>
</tr>
<tr class="even">
<td>Embryonal tumor</td>
<td>MB, SHH</td>
<td>24</td>
<td>30</td>
</tr>
<tr class="odd">
<td>Embryonal tumor</td>
<td>MB, WNT</td>
<td>10</td>
<td>10</td>
</tr>
<tr class="even">
<td>Ependymoma</td>
<td>EPN, H3 K28</td>
<td>1</td>
<td>1</td>
</tr>
<tr class="odd">
<td>Ependymoma</td>
<td>EPN, ST RELA</td>
<td>25</td>
<td>28</td>
</tr>
<tr class="even">
<td>Ependymoma</td>
<td>EPN, ST YAP1</td>
<td>3</td>
<td>4</td>
</tr>
<tr class="odd">
<td>High-grade glioma</td>
<td>DMG, H3 K28</td>
<td>18</td>
<td>24</td>
</tr>
<tr class="even">
<td>High-grade glioma</td>
<td>DMG, H3 K28, TP53 activated</td>
<td>10</td>
<td>13</td>
</tr>
<tr class="odd">
<td>High-grade glioma</td>
<td>DMG, H3 K28, TP53 loss</td>
<td>30</td>
<td>40</td>
</tr>
<tr class="even">
<td>High-grade glioma</td>
<td>HGG, H3 G35</td>
<td>3</td>
<td>3</td>
</tr>
<tr class="odd">
<td>High-grade glioma</td>
<td>HGG, H3 G35, TP53 loss</td>
<td>1</td>
<td>1</td>
</tr>
<tr class="even">
<td>High-grade glioma</td>
<td>HGG, H3 wildtype</td>
<td>26</td>
<td>31</td>
</tr>
<tr class="odd">
<td>High-grade glioma</td>
<td>HGG, H3 wildtype, TP53 activated</td>
<td>5</td>
<td>5</td>
</tr>
<tr class="even">
<td>High-grade glioma</td>
<td>HGG, H3 wildtype, TP53 loss</td>
<td>14</td>
<td>21</td>
</tr>
<tr class="odd">
<td>High-grade glioma</td>
<td>HGG, IDH, TP53 activated</td>
<td>1</td>
<td>2</td>
</tr>
<tr class="even">
<td>High-grade glioma</td>
<td>HGG, IDH, TP53 loss</td>
<td>1</td>
<td>1</td>
</tr>
<tr class="odd">
<td>Low-grade glioma</td>
<td>GNG, BRAF V600E</td>
<td>13</td>
<td>13</td>
</tr>
<tr class="even">
<td>Low-grade glioma</td>
<td>GNG, BRAF V600E, CDKN2A/B</td>
<td>1</td>
<td>1</td>
</tr>
<tr class="odd">
<td>Low-grade glioma</td>
<td>GNG, FGFR</td>
<td>1</td>
<td>1</td>
</tr>
<tr class="even">
<td>Low-grade glioma</td>
<td>GNG, H3</td>
<td>1</td>
<td>1</td>
</tr>
<tr class="odd">
<td>Low-grade glioma</td>
<td>GNG, IDH</td>
<td>1</td>
<td>2</td>
</tr>
<tr class="even">
<td>Low-grade glioma</td>
<td>GNG, KIAA1549-BRAF</td>
<td>5</td>
<td>5</td>
</tr>
<tr class="odd">
<td>Low-grade glioma</td>
<td>GNG, MYB/MYBL1</td>
<td>1</td>
<td>1</td>
</tr>
<tr class="even">
<td>Low-grade glioma</td>
<td>GNG, NF1-germline</td>
<td>1</td>
<td>1</td>
</tr>
<tr class="odd">
<td>Low-grade glioma</td>
<td>GNG, NF1-somatic, BRAF V600E</td>
<td>1</td>
<td>1</td>
</tr>
<tr class="even">
<td>Low-grade glioma</td>
<td>GNG, other MAPK</td>
<td>4</td>
<td>4</td>
</tr>
<tr class="odd">
<td>Low-grade glioma</td>
<td>GNG, other MAPK, IDH</td>
<td>1</td>
<td>1</td>
</tr>
<tr class="even">
<td>Low-grade glioma</td>
<td>GNG, RTK</td>
<td>2</td>
<td>3</td>
</tr>
<tr class="odd">
<td>Low-grade glioma</td>
<td>GNG, wildtype</td>
<td>14</td>
<td>14</td>
</tr>
<tr class="even">
<td>Low-grade glioma</td>
<td>LGG, BRAF V600E</td>
<td>25</td>
<td>27</td>
</tr>
<tr class="odd">
<td>Low-grade glioma</td>
<td>LGG, BRAF V600E, CDKN2A/B</td>
<td>5</td>
<td>5</td>
</tr>
<tr class="even">
<td>Low-grade glioma</td>
<td>LGG, FGFR</td>
<td>8</td>
<td>8</td>
</tr>
<tr class="odd">
<td>Low-grade glioma</td>
<td>LGG, IDH</td>
<td>3</td>
<td>3</td>
</tr>
<tr class="even">
<td>Low-grade glioma</td>
<td>LGG, KIAA1549-BRAF</td>
<td>106</td>
<td>113</td>
</tr>
<tr class="odd">
<td>Low-grade glioma</td>
<td>LGG, KIAA1549-BRAF, NF1-germline</td>
<td>1</td>
<td>1</td>
</tr>
<tr class="even">
<td>Low-grade glioma</td>
<td>LGG, KIAA1549-BRAF, other MAPK</td>
<td>1</td>
<td>1</td>
</tr>
<tr class="odd">
<td>Low-grade glioma</td>
<td>LGG, MYB/MYBL1</td>
<td>2</td>
<td>2</td>
</tr>
<tr class="even">
<td>Low-grade glioma</td>
<td>LGG, NF1-germline</td>
<td>6</td>
<td>6</td>
</tr>
<tr class="odd">
<td>Low-grade glioma</td>
<td>LGG, NF1-germline, CDKN2A/B</td>
<td>1</td>
<td>1</td>
</tr>
<tr class="even">
<td>Low-grade glioma</td>
<td>LGG, NF1-germline, FGFR</td>
<td>1</td>
<td>2</td>
</tr>
<tr class="odd">
<td>Low-grade glioma</td>
<td>LGG, NF1-somatic</td>
<td>2</td>
<td>2</td>
</tr>
<tr class="even">
<td>Low-grade glioma</td>
<td>LGG, NF1-somatic, FGFR</td>
<td>1</td>
<td>1</td>
</tr>
<tr class="odd">
<td>Low-grade glioma</td>
<td>LGG, NF1-somatic, NF1-germline, CDKN2A/B</td>
<td>1</td>
<td>1</td>
</tr>
<tr class="even">
<td>Low-grade glioma</td>
<td>LGG, other MAPK</td>
<td>11</td>
<td>12</td>
</tr>
<tr class="odd">
<td>Low-grade glioma</td>
<td>LGG, RTK</td>
<td>8</td>
<td>10</td>
</tr>
<tr class="even">
<td>Low-grade glioma</td>
<td>LGG, RTK, CDKN2A/B</td>
<td>1</td>
<td>1</td>
</tr>
<tr class="odd">
<td>Low-grade glioma</td>
<td>LGG, wildtype</td>
<td>33</td>
<td>34</td>
</tr>
<tr class="even">
<td>Low-grade glioma</td>
<td>SEGA, RTK</td>
<td>1</td>
<td>1</td>
</tr>
<tr class="odd">
<td>Low-grade glioma</td>
<td>SEGA, wildtype</td>
<td>10</td>
<td>11</td>
</tr>
<tr class="even">
<td>Mesenchymal non-meningothelial tumor</td>
<td>EWS</td>
<td>9</td>
<td>11</td>
</tr>
<tr class="odd">
<td>Neuronal and mixed neuronal-glial tumor</td>
<td>CNC</td>
<td>2</td>
<td>2</td>
</tr>
<tr class="even">
<td>Neuronal and mixed neuronal-glial tumor</td>
<td>EVN</td>
<td>1</td>
<td>1</td>
</tr>
<tr class="odd">
<td>Neuronal and mixed neuronal-glial tumor</td>
<td>GNT, BRAF V600E</td>
<td>1</td>
<td>1</td>
</tr>
<tr class="even">
<td>Neuronal and mixed neuronal-glial tumor</td>
<td>GNT, KIAA1549-BRAF</td>
<td>1</td>
<td>2</td>
</tr>
<tr class="odd">
<td>Neuronal and mixed neuronal-glial tumor</td>
<td>GNT, other MAPK</td>
<td>1</td>
<td>1</td>
</tr>
<tr class="even">
<td>Neuronal and mixed neuronal-glial tumor</td>
<td>GNT, other MAPK, FGFR</td>
<td>1</td>
<td>1</td>
</tr>
<tr class="odd">
<td>Neuronal and mixed neuronal-glial tumor</td>
<td>GNT, RTK</td>
<td>1</td>
<td>2</td>
</tr>
<tr class="even">
<td>Tumor of sellar region</td>
<td>CRANIO, ADAM</td>
<td>27</td>
<td>27</td>
</tr>
<tr class="odd">
<td></td>
<td>Total</td>
<td>577</td>
<td>644</td>
</tr>
</tbody>
</table>
</div>
<h3 id="somatic-mutational-landscape-of-pediatric-brain-tumors">Somatic Mutational Landscape of Pediatric Brain Tumors</h3>
<p>We performed a comprehensive genomic analysis of somatic SNVs, CNVs, SVs, and fusions across 1,074 tumors (N = 1,019 RNA-Seq, N = 918 WGS, N = 32 WXS/Panel) and 22 cell lines (N = 16 RNA-Seq, N = 22 WGS), from 943 patients, 833 with paired normal specimens (N = 801 WGS, N = 32 WXS/Panel).
Tumor purity across the PBTA was high (median of 76%) and is plotted by cancer group in <strong>Figure <a href="#fig:S3">S3</a>A</strong>.
Unless otherwise noted, each analysis was performed for primary tumors using one tumor per patient.</p>
<!--SAMPLECOUNT-->
<p>Following SNV consensus calling (<strong>Figure <a href="#fig:S1">S1</a></strong> and <strong>Figure <a href="#fig:S2">S2</a>A-G</strong>), we observed as expected lower tumor mutation burden (TMB) <strong>Figure <a href="#fig:S2">S2</a>H</strong> in pediatric tumors compared to adult brain tumors from The Cancer Genome Atlas (TCGA), <strong>Figure <a href="#fig:S2">S2</a>I</strong>, with hypermutant (&gt; 10 Mut/Mb) and ultra-hypermutant (&gt; 100 Mut/Mb) tumors<span class="citation" data-cites="pmid:29056344"><sup><a href="#ref-pmid:29056344" role="doc-biblioref"><strong>pmid:29056344?</strong></a></sup></span> only found within HGGs.
<strong>Figure <a href="#fig:Fig2">2</a></strong> and <strong>Figure <a href="#fig:S3">S3</a>A</strong> depict oncoprints recapitulating known histology-specific driver genes in primary tumors across PBTA histologies.</p>
<!-- TODO: Update group-specific count numbers below with figure update -->
<h4 id="low-grade-gliomas">Low-grade gliomas</h4>
<p>As expected, the majority (62%, 140/227) of LGGs harbored a somatic alteration in <em>BRAF</em>, with canonical <em>BRAF::KIAA1549</em> fusions as the major oncogenic driver<span class="citation" data-cites="IamVuiNU"><sup><a href="#ref-IamVuiNU" role="doc-biblioref">26</a></sup></span> (<strong>Figure <a href="#fig:Fig2">2</a>A</strong>). <!--SAMPLECOUNT-->
We observed additional mutations in <em>FGFR1</em> (2%), <em>PIK3CA</em> (2%), <em>KRAS</em> (2%), <em>TP53</em> (1%), and <em>ATRX</em> (1%) and fusions in <em>NTRK2</em> (2%), <em>RAF1</em> (2%), <em>MYB</em> (1%), <em>QKI</em> (1%), <em>ROS1</em> (1%), and <em>FGFR2</em> (1%), concordant with previous studies reporting the near universal upregulation of the RAS/MAPK pathway in these tumors resulting from activating mutations and/or oncogenic fusions<span class="citation" data-cites="IamVuiNU ULPByXgG"><sup><a href="#ref-ULPByXgG" role="doc-biblioref">23</a>,<a href="#ref-IamVuiNU" role="doc-biblioref">26</a></sup></span>.
Indeed, we observed significant upregulation (ANOVA Bonferroni-corrected p &lt; 0.01) of the KRAS signaling pathway in LGGs (<strong>Figure <a href="#fig:Fig5">5</a>B</strong>).</p>
<h4 id="embryonal-tumors">Embryonal tumors</h4>
<p><!--SAMPLECOUNT-->
The majority (N = 95) of embryonal tumors were medulloblastomas that spanned the spectrum of molecular subtypes (WNT, SHH, Group3, and Group 4; see <strong>Molecular Subtyping of CNS Tumors</strong>), as identified by subtype-specific canonical mutations (<strong>Figure <a href="#fig:Fig2">2</a>B</strong>).
We detected canonical <em>SMARCB1/SMARCA4</em> deletions or inactivating mutations in atypical teratoid rhabdoid tumors (ATRTs) and C19MC amplification in the embryonal tumors with multilayer rosettes (ETMRs, displayed as other embryonal tumors)<span class="citation" data-cites="1Geb70JK9 1EzqPr2Ef j11VxJIm UuzX8B9C"><sup><a href="#ref-1Geb70JK9" role="doc-biblioref">27</a>–<a href="#ref-UuzX8B9C" role="doc-biblioref">30</a></sup></span>.</p>
<h4 id="high-grade-gliomas">High-grade gliomas</h4>
<!--SAMPLECOUNT-->
<p>Across HGGs, we found that <em>TP53</em> (57%, 35/61) and <em>H3F3A</em> (52%, 32/61) were both most mutated and co-occurring genes (<strong>Figure <a href="#fig:Fig2">2</a>A and C</strong>), followed by frequent mutations in <em>ATRX</em> (30%, 18/61) which is commonly mutated in gliomas<span class="citation" data-cites="UWkO4UwG"><sup><a href="#ref-UWkO4UwG" role="doc-biblioref">31</a></sup></span>.
We observed recurrent amplifications and fusions in <em>EGFR</em>, <em>MET</em>, <em>PDGFRA</em>, and <em>KIT</em>, highlighting that these tumors utilize multiple oncogenic mechanisms to activate tyrosine kinases, as has been previously reported<span class="citation" data-cites="BLNV1DbL vd5Uh9gY KU8wVb8T"><sup><a href="#ref-BLNV1DbL" role="doc-biblioref">17</a>,<a href="#ref-vd5Uh9gY" role="doc-biblioref">32</a>,<a href="#ref-KU8wVb8T" role="doc-biblioref">33</a></sup></span>.
Gene set enrichment analysis showed upregulation (ANOVA Bonferroni-corrected p &lt; 0.01) of DNA repair, G2M checkpoint, and MYC pathways as well as downregulation of the TP53 pathway (<strong>Figure <a href="#fig:Fig5">5</a>B</strong>).
The two tumors with ultra-high TMB (&gt; 100 Mutations/Mb) were from patients with known mismatch repair deficiency syndrome<span class="citation" data-cites="DwCzMoBJ"><sup><a href="#ref-DwCzMoBJ" role="doc-biblioref">16</a></sup></span>.</p>
<h4 id="other-cns-tumors">Other CNS tumors</h4>
<!--SAMPLECOUNT-->
<p>We observed that 25% (15/60) of ependymoma tumors were <em>C11orf95::RELA</em> (now, <em>ZFTA::RELA</em>) fusion-positive ependymomas [@:doi.org/10.1038/nature13109] and that 68% (21/31) of craniopharyngiomas were driven by mutations in <em>CTNNB1</em> (<strong>Figure <a href="#fig:Fig2">2</a>D</strong>).
Multiple histologies contained somatic mutations or fusions in <em>NF2</em>, including 41% (7/17) of meningiomas, 5% (3/60) of ependymomas, and 27% (3/11) schwannomas.
We observed rare fusions in <em>ERBB4</em>, <em>YAP1</em>, <em>KRAS</em>, and <em>MAML2</em> in 10% (6/60) of ependymoma tumors.
DNETs harbored alterations in MAPK/PI3K pathway genes, as has been previously reported<span class="citation" data-cites="eDtbAO3v"><sup><a href="#ref-eDtbAO3v" role="doc-biblioref">34</a></sup></span>, including <em>FGFR1</em> (21%, 4/19), <em>PDGFRA</em> (10%, 2/19), and <em>BRAF</em> (5%, 1/19).
Frequent mutations in additional rare brain tumor histologies are depicted in <strong>Figure <a href="#fig:S3">S3</a>A</strong>.</p>
<div id="fig:Fig2" class="fignos">
<figure>
<img src="https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/361fef09c574942d3d5ef0668461b37e3ee6cba6/figures/pngs/figure2.png?sanitize=true" style="width:9in" alt="Figure 2: Mutational landscape of PBTA tumors. Shown are frequencies of canonical somatic gene mutations, CNVs, fusions, and TMB (top bar plot) for the top 20 genes mutated across primary tumors within the OpenPBTA dataset. A, Low-grade astrocytic tumors (N = 227): pilocytic astrocytoma (N = 104), other low-grade glioma (N = 69), ganglioglioma (N = 35), pleomorphic xanthoastrocytoma (N = 9), subependymal giant cell astrocytoma (N = 10); B, Embryonal tumors (N = 128): medulloblastomas (N = 95), atypical teratoid rhabdoid tumors (N = 24), other embryonal tumors (N = 9); C, Diffuse astrocytic and oligodendroglial tumors (N = 61): diffuse midline gliomas (N = 34) and other high-grade gliomas (N = 27); D, Other CNS tumors (N = 194): ependymomas (N = 60), craniopharyngiomas (N = 31), meningiomas (N = 17), dysembryoplastic neuroepithelial tumors (N = 19), Ewing sarcomas (N = 7), schwannomas (N = 11), and neurofibroma plexiforms (N = 7). Additional, rare CNS tumors are displayed in Figure S3B. Tumor histology (Cancer Group) and patient sex (Germline sex estimate) are displayed as annotations at the bottom of each plot. Only tumors with mutations in the listed genes are shown. Multiple CNVs are denoted as a complex event. N denotes the number of patients and tumors." />
<figcaption aria-hidden="true"><span>Figure 2:</span> <strong>Mutational landscape of PBTA tumors.</strong> Shown are frequencies of canonical somatic gene mutations, CNVs, fusions, and TMB (top bar plot) for the top 20 genes mutated across primary tumors within the OpenPBTA dataset. A, Low-grade astrocytic tumors (N = 227): pilocytic astrocytoma (N = 104), other low-grade glioma (N = 69), ganglioglioma (N = 35), pleomorphic xanthoastrocytoma (N = 9), subependymal giant cell astrocytoma (N = 10); B, Embryonal tumors (N = 128): medulloblastomas (N = 95), atypical teratoid rhabdoid tumors (N = 24), other embryonal tumors (N = 9); C, Diffuse astrocytic and oligodendroglial tumors (N = 61): diffuse midline gliomas (N = 34) and other high-grade gliomas (N = 27); D, Other CNS tumors (N = 194): ependymomas (N = 60), craniopharyngiomas (N = 31), meningiomas (N = 17), dysembryoplastic neuroepithelial tumors (N = 19), Ewing sarcomas (N = 7), schwannomas (N = 11), and neurofibroma plexiforms (N = 7). Additional, rare CNS tumors are displayed in <strong>Figure <a href="#fig:S3">S3</a>B</strong>. Tumor histology (<code>Cancer Group</code>) and patient sex (<code>Germline sex estimate</code>) are displayed as annotations at the bottom of each plot. Only tumors with mutations in the listed genes are shown. Multiple CNVs are denoted as a complex event. N denotes the number of patients and tumors.</figcaption>
</figure>
</div>
<!--SAMPLECOUNT-->
<h3 id="mutational-co-occurrence-cnv-and-signatures-highlight-key-oncogenic-drivers">Mutational co-occurrence, CNV, and signatures highlight key oncogenic drivers</h3>
<p>We analyzed mutational co-occurrence across the OpenPBTA, using a single tumor from each patient with available WGS (N = 666 patients). <!--SAMPLECOUNT-->
The top 50 mutated genes (see <strong>STAR Methods</strong> for details) in primary tumors are shown in <strong>Figure <a href="#fig:Fig3">3</a></strong> by tumor type (<strong>A</strong>, bar plots), with co-occurrence scores illustrated in the heatmap (<strong>B</strong>).
As expected, <em>TP53</em> was the most frequently mutated gene across the OpenPBTA (8.4%, 56/666), significantly co-occurring with <em>H3F3A</em> (OR = 32, 95% CI: 15.3 - 66.7, q = 8.46e-17), <em>ATRX</em> (OR = 20, 95% CI: 8.4 - 47.7, q = 4.43e-8), <em>NF1</em> (OR = 8.62, 95% CI: 3.7 - 20.2, q = 5.45e-5), and <em>EGFR</em> (OR = 18.2, 95% CI: 5 - 66.5, q = 1.6e-4), with all of these driven by HGGs and consistent with previous reports<span class="citation" data-cites="vd5Uh9gY MsKFoF50 zrYR4Swt"><sup><a href="#ref-vd5Uh9gY" role="doc-biblioref">32</a>,<a href="#ref-MsKFoF50" role="doc-biblioref">35</a>,<a href="#ref-zrYR4Swt" role="doc-biblioref">36</a></sup></span>.</p>
<p>In medulloblastoma tumors, mutations in <em>CTNNB1</em> significantly co-occurred with mutations in <em>TP53</em> (OR = 42.9, 95% CI: 7 - 261.4, q = 1.63e-3), as has been previously observed<span class="citation" data-cites="ZINVcBaH"><sup><a href="#ref-ZINVcBaH" role="doc-biblioref">37</a></sup></span>, as well as in <em>DDX3X</em> (OR = 21.1, 95% CI: 4.6 - 96.3, q = 4.46e-3), which has also been previously identified as a commonly mutated gene in medulloblastomas<span class="citation" data-cites="ynFGewrs"><sup><a href="#ref-ynFGewrs" role="doc-biblioref">38</a></sup></span>.
Mutations in <em>FGFR1</em> and <em>PIK3CA</em> significantly co-occurred in LGGs (OR = 76.1, 95% CI: 9.85 - 588.1, q = 3.26e-3), consistent with previous findings<span class="citation" data-cites="ZINVcBaH 5k4QKyTN"><sup><a href="#ref-ZINVcBaH" role="doc-biblioref">37</a>,<a href="#ref-5k4QKyTN" role="doc-biblioref">39</a></sup></span>.
Of HGG tumors with mutations in <em>TP53</em> or <em>PPM1D</em>, 52/54 (96.3%) had mutations in only one of these genes (OR = 0.188, 95% CI: 0.04 - 0.94, p = 0.0413, q = 0.0587).
This trend recapitulates previous observations that <em>TP53</em> and <em>PPM1D</em> mutations tend to be mutually exclusive in HGGs<span class="citation" data-cites="1CpnMVmIN"><sup><a href="#ref-1CpnMVmIN" role="doc-biblioref">40</a></sup></span>.</p>
<p>We summarized broad CNV and SV and observed that HGGs and DMGs, followed by medulloblastomas, had the most unstable genomes (<strong>Figure <a href="#fig:S3">S3</a>C</strong>).
By contrast, craniopharyngiomas and schwannomas generally lacked somatic CNV.
Together, these CNV patterns largely aligned with our estimates of tumor mutational burden (<strong>Figure <a href="#fig:S2">S2</a>H</strong>).
The breakpoint density estimated from SV and CNV data was significantly correlated across tumors (linear regression p = 1.08e-37) (<strong>Figure <a href="#fig:Fig3">3</a>C</strong>) and as expected, the number of chromothripsis regions called increased as breakpoint density increased (<strong>Figure <a href="#fig:S3">S3</a>D-E</strong>).
We identified chromothripsis events in 28% (N = 11/39) of diffuse midline gliomas and in 40% (N = 19/48) of other HGGs (non-midline HGGs) (<strong>Figure <a href="#fig:Fig3">3</a>D</strong>).
We also found evidence of chromothripsis in over 15% of sarcomas, PXAs, metastatic secondary tumors, chordomas, glial-neuronal tumors, germinomas, meningiomas, ependymomas, medulloblastomas, ATRTs, and other embryonal tumors, highlighting the genomic instability and complexity of these pediatric brain tumors.</p>
<p>We next assessed the contributions of eight previously identified adult CNS-specific mutational signatures from the RefSig database<span class="citation" data-cites="Cc8jt6lX"><sup><a href="#ref-Cc8jt6lX" role="doc-biblioref">41</a></sup></span> across tumors (<strong>Figure <a href="#fig:Fig3">3</a>E</strong> and <strong>Figure <a href="#fig:S4">S4</a>A</strong>).
Stage 0 and/or 1 tumors characterized by low TMBs (<strong>Figure <a href="#fig:S2">S2</a>H</strong>) such as pilocytic astrocytomas, gangliogliomas, other LGGs, and craniopharyngiomas, were dominated by Signature 1 (<strong>Figure <a href="#fig:S4">S4</a>A</strong>), which results from the normal process of spontaneous deamination of 5-methylcytosine.
Signature N6 is a CNS-specific signature which we observed nearly universally across tumors.
Drivers of Signature 18, <em>TP53</em>, <em>APC</em>, <em>NOTCH1</em> (found at https://signal.mutationalsignatures.com/explore/referenceCancerSignature/31/drivers), are also canonical drivers of medulloblastoma, and indeed, we observed Signature 18 as the signature with the highest weight in medulloblastoma tumors.
Signatures 3, 8, 18, and MMR2 were prevalent in HGGs, including DMGs.
Finally, we found that the Signature 1 weight was higher at diagnosis (pre-treatment) and was almost always lower in tumors at later phases of therapy (progression, recurrence, post-mortem, secondary malignancy; <strong>Figure <a href="#fig:S4">S4</a>B</strong>).
This trend may have resulted from therapy-induced mutations that produced additional signatures (e.g., temozolomide treatment has been suggested to drive Signature 11<span class="citation" data-cites="OaMJfIBC"><sup><a href="#ref-OaMJfIBC" role="doc-biblioref">42</a></sup></span>), subclonal expansion, and/or acquisition of additional driver mutations during tumor progression, leading to higher overall TMBs and additional signatures.</p>
<!-- To do: possibly explore gene mutations driving signature 8, specifically for meningiomas, and describe -->
<div id="fig:Fig3" class="fignos">
<figure>
<img src="https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/361fef09c574942d3d5ef0668461b37e3ee6cba6/figures/pngs/figure3.png?sanitize=true" style="width:7in" alt="Figure 3: Mutational co-occurrence and signatures highlight key oncogenic drivers. A, Bar plot of occurrence and co-occurrence of nonsynonymous mutations for the 50 most commonly mutated genes across all tumor types, which are denoted as “Other” when there are fewer than 10 tumors per grouping; B, Co-occurrence and mutual exclusivity of nonsynonymous mutations between genes; The co-occurrence score is defined as I(-\log_{10}(P)) where P is defined by Fisher’s exact test and I is 1 when mutations co-occur more often than expected and -1 when exclusivity is more common; C, The number of SV breaks significantly correlate with CNV breaks (Adjusted R = 0.436, p = 1.08e-37). D, Chromothripsis frequency across pediatric brain tumors for all cancer groups with N &gt;= 3 tumors. E, Sina plots of RefSig signature weights for signatures 1, 11, 18, 19, 3, 8, N6, MMR2, and Other across cancer groups. Box plot represents 5% (lower whisker), 25% (lower box), 50% (median), 75% (upper box), and 95% (upper whisker) quantiles." />
<figcaption aria-hidden="true"><span>Figure 3:</span> <strong>Mutational co-occurrence and signatures highlight key oncogenic drivers.</strong> A, Bar plot of occurrence and co-occurrence of nonsynonymous mutations for the 50 most commonly mutated genes across all tumor types, which are denoted as “Other” when there are fewer than 10 tumors per grouping; B, Co-occurrence and mutual exclusivity of nonsynonymous mutations between genes; The co-occurrence score is defined as <span class="math inline">\(I(-\log_{10}(P))\)</span> where <span class="math inline">\(P\)</span> is defined by Fisher’s exact test and <span class="math inline">\(I\)</span> is 1 when mutations co-occur more often than expected and -1 when exclusivity is more common; C, The number of SV breaks significantly correlate with CNV breaks (Adjusted R = 0.436, p = 1.08e-37). D, Chromothripsis frequency across pediatric brain tumors for all cancer groups with N &gt;= 3 tumors. E, Sina plots of RefSig signature weights for signatures 1, 11, 18, 19, 3, 8, N6, MMR2, and Other across cancer groups. Box plot represents 5% (lower whisker), 25% (lower box), 50% (median), 75% (upper box), and 95% (upper whisker) quantiles.</figcaption>
</figure>
</div>
<h3 id="transcriptomic-landscape-of-pediatric-brain-tumors">Transcriptomic Landscape of Pediatric Brain Tumors</h3>
<h4 id="prediction-of-tp53-oncogenicity-and-telomerase-activity">Prediction of <em>TP53</em> oncogenicity and telomerase activity</h4>
<p>To understand the <em>TP53</em> phenotype in each tumor, we ran a classifier previously trained on TCGA<span class="citation" data-cites="X6oQhIf8"><sup><a href="#ref-X6oQhIf8" role="doc-biblioref">43</a></sup></span> to calculate a <em>TP53</em> score and infer <em>TP53</em> inactivation status.
We compared results of this classifier to “true positive” alterations derived using high-confidence SNVs, CNVs, SVs, and fusions in <em>TP53</em>.
Specifically, we annotated <em>TP53</em> alterations as “activated” if tumors harbored one of p.R273C or p.R248W gain-of-function mutations<span class="citation" data-cites="14asizOhY"><sup><a href="#ref-14asizOhY" role="doc-biblioref">44</a></sup></span>, or “lost” if the given patient either had a Li Fraumeni Syndrome (LFS) predisposition diagnosis, the tumor harbored a known hotspot mutation, or the tumor contained two hits (e.g. both SNV and CNV), which would suggest both alleles had been affected.
If the <em>TP53</em> mutation did not reside within the DNA-binding domain or we did not detect any alteration in <em>TP53</em>, we annotate the tumor as “other,” reflecting its unknown <em>TP53</em> alteration status.
The classifier achieved a high accuracy (AUROC = 0.85) for rRNA-depleted, stranded tumors compared to randomly shuffled <em>TP53</em> scores (<strong>Figure <a href="#fig:Fig4">4</a>A</strong>).
By contrast, while this classifier has previously shown strong performance on poly-A data from both adult<span class="citation" data-cites="X6oQhIf8"><sup><a href="#ref-X6oQhIf8" role="doc-biblioref">43</a></sup></span> tumors and pediatric patient-derived xenografts<span class="citation" data-cites="SDwYl8uA"><sup><a href="#ref-SDwYl8uA" role="doc-biblioref">45</a></sup></span>, it did not perform as well on the poly-A tumors in this cohort (AUROC = 0.62; <strong>Figure <a href="#fig:S5">S5</a>A</strong>).</p>
<p>While we expected that tumors annotated as “lost” would have higher <em>TP53</em> scores than would tumors annotated as “other,” we observed that tumors annotated as “activated” had similar <em>TP53</em> scores to those annotated as “lost” (<strong>Figure <a href="#fig:Fig4">4</a>B</strong>, Wilcoxon p = 0.92).
This result suggests that the classifier actually detects an oncogenic, or altered, <em>TP53</em> phenotype (scores &gt; 0.5) rather than solely <em>TP53</em> inactivation, as interpreted previously<span class="citation" data-cites="X6oQhIf8"><sup><a href="#ref-X6oQhIf8" role="doc-biblioref">43</a></sup></span>.
Moreover, tumors with “activating” <em>TP53</em> mutations showed higher <em>TP53</em> expression compared to those with <em>TP53</em> “loss” mutations (Wilcoxon p = 0.006, <strong>Figure <a href="#fig:Fig4">4</a>C</strong>).
Tumor types with the highest median <em>TP53</em> scores were those known to harbor somatic <em>TP53</em> alterations and included DMGs, medulloblastomas, HGGs, DNETs, ependymomas, and craniopharyngiomas (<strong>Figure <a href="#fig:Fig4">4</a>D</strong>), while gangliogliomas, LGGs, meningiomas, and schwannomas had the lowest median scores.</p>
<p>To further validate the classifier’s accuracy, we assessed <em>TP53</em> scores for patients with LFS, hypothesizing that all of these tumors would have high scores.
Indeed, we observed higher scores in 8/10 tumors from LFS patients (N = 8 patients)(<strong>Table S3</strong>).
Although we observed low <em>TP53</em> scores in two tumors from LFS patients (BS_DEHJF4C7 with a score of 0.09 and BS_ZD5HN296 with a score of 0.28), we confirmed from pathology reports that both patients were diagnosed with LFS and had a pathogenic germline variant in <em>TP53</em>.
In addition, the tumor purity of these two LFS tumors was low (16% and 37%, respectively), suggesting the classifier may require a certain level of tumor purity to achieve good performance, as we expect <em>TP53</em> to be intact in normal cells.
These transcriptomic scores can be utilized to infer <em>TP53</em> function in the absence of a predicted oncogenic <em>TP53</em> alteration or DNA sequencing in general.</p>
<p>We used gene expression data to predict telomerase activity using EXpression-based Telomerase ENzymatic activity Detection (<code>EXTEND</code>)<span class="citation" data-cites="RnBZNbdx"><sup><a href="#ref-RnBZNbdx" role="doc-biblioref">46</a></sup></span> as a surrogate measure of malignant potential<span class="citation" data-cites="RnBZNbdx 15inmTb6T"><sup><a href="#ref-RnBZNbdx" role="doc-biblioref">46</a>,<a href="#ref-15inmTb6T" role="doc-biblioref">47</a></sup></span>, such that higher <code>EXTEND</code> scores suggest increased malignant potential.
While we did not find that tumors with <em>TERT</em> promoter (TERTp) mutations (N = 6) had significantly higher telomerase activity scores than non-mutated tumors (Wilcoxon p-value = 0.1196), we observed that <code>EXTEND</code> scores significantly correlated with <em>TERC</em> (R = 0.619, p &lt; 0.01) and <em>TERT</em> (R = 0.491, p &lt; 0.01) expression (<strong>Figure <a href="#fig:S5">S5</a>B-C</strong>).
Since catalytically-active telomerase requires a combination of full-length <em>TERT</em>, <em>TERC</em>, as well as accessory proteins<span class="citation" data-cites="pmid:9751630"><sup><a href="#ref-pmid:9751630" role="doc-biblioref"><strong>pmid:9751630?</strong></a></sup></span>, we expect that <code>EXTEND</code> scores may not be exclusively correlated with <em>TERT</em> alterations and expression.
Next, we found aggressive tumors such as HGGs (DMGs and other high-grade gliomas) and MB had high <code>EXTEND</code> scores (<strong>Figure <a href="#fig:Fig4">4</a>D</strong>), while low-grade lesions such as schwannomas, GNGs, DNETs, and other low-grade gliomas had among the lowest scores (<strong>Table S3</strong>).
These findings support previous reports of a more aggressive phenotype in tumors with higher telomerase activity<span class="citation" data-cites="TJj4hOO wXW7qGqv dhzTU0Xu 5I97mcf1"><sup><a href="#ref-TJj4hOO" role="doc-biblioref">48</a>–<a href="#ref-5I97mcf1" role="doc-biblioref">51</a></sup></span>.</p>
<h4 id="hypermutant-tumors-share-mutational-signatures-and-have-dysregulated-tp53">Hypermutant tumors share mutational signatures and have dysregulated <strong><em>TP53</em></strong></h4>
<p>We further investigated the mutational signature profiles of the hypermutant (TMB &gt; 10 Mut/Mb; N = 3) and ultra-hypermutant (TMB &gt; 100 Mut/Mb; N = 4) tumors and/or derived cell lines from six patients in the OpenPBTA cohort (<strong>Figure <a href="#fig:Fig4">4</a>E</strong>).
Five of six tumors were diagnosed as HGGs and one was a brain metastasis of a MYCN non-amplified neuroblastoma tumor.
Signature 11, which is associated with exposure to temozolomide plus <em>MGMT</em> promoter and/or mismatch repair deficiency<span class="citation" data-cites="td36JJNb"><sup><a href="#ref-td36JJNb" role="doc-biblioref">52</a></sup></span>, was indeed present in tumors with previous exposure to the drug (<strong>Table <a href="#tbl:Table2">2</a></strong>).
We detected the MMR2 signature in tumors of four patients (PT_0SPKM4S8, PT_3CHB9PK5, PT_JNEV57VK, and PT_VTM2STE3) diagnosed with either constitutional mismatch repair deficiency (CMMRD) or Lynch syndrome (<strong>Table <a href="#tbl:Table2">2</a></strong>), genetic predisposition syndromes caused by a variant in a mismatch repair gene such as <em>PMS2</em>, <em>MLH1</em>, <em>MSH2</em>, <em>MSH6</em>, or others<span class="citation" data-cites="1DeOIIYxr"><sup><a href="#ref-1DeOIIYxr" role="doc-biblioref">53</a></sup></span>.
Three of these patients harbored pathogenic germline variants in one of the aforementioned genes.
While we did not find a <em>known</em> pathogenic variant in the germline of PT_VTM2STE3, this patient had a self-reported <em>PMS2</em> variant noted in their pathology report and we did find 19 intronic variants of unknown significance (VUS) in <em>PMS2</em>.
This is not surprising since an estimated 49% of germline <em>PMS2</em> variants in patients with CMMRD and/or Lynch syndrome are VUS<span class="citation" data-cites="1DeOIIYxr"><sup><a href="#ref-1DeOIIYxr" role="doc-biblioref">53</a></sup></span>.
Interestingly, while the cell line derived from patient PT_VTM2STE3’s tumor at progression was not hypermutated (TMB = 5.7 Mut/Mb), it solely showed the MMR2 signature of the eight CNS signatures examined, suggesting selective pressure to maintain a mismatch repair (MMR) phenotype <em>in vitro</em>.
From patient PT_JNEV57VK, only one of the two cell lines derived from the progressive tumor was hypermutated (TMB = 35.9 Mut/Mb).
This hypermutated cell line was strongly weighted towards signature 11, while this patient’s non-hypermutated cell line showed a number of lesser signature weights (1, 11, 18, 19, MMR2; Table S2), highlighting the plasticity of mutational processes and the need to carefully genomically characterize and select models for preclinical studies based on research objectives.</p>
<p>We observed that signature 18, which has been associated with high genomic instability and can lead to a hypermutator phenotype<span class="citation" data-cites="Cc8jt6lX"><sup><a href="#ref-Cc8jt6lX" role="doc-biblioref">41</a></sup></span>, was uniformly represented among hypermutant solid tumors.
Additionally, we found that all of the HGG tumors or cell lines had dysfunctional <em>TP53</em> (<strong>Table <a href="#tbl:Table2">2</a></strong>), consistent with a previous report showing <em>TP53</em> dysregulation is a dependency in tumors with high genomic instability<span class="citation" data-cites="Cc8jt6lX"><sup><a href="#ref-Cc8jt6lX" role="doc-biblioref">41</a></sup></span>.
With one exception, hypermutant and ultra-hypermutant tumors had high <em>TP53</em> scores (&gt; 0.5) and telomerase activity.
Interestingly, none of the hypermutant tumors showed evidence of signature 3 (present in homologous recombination deficient tumors), signature 8 (arises from double nucleotide substitutions/unknown etiology), or signature N6 (a universal CNS tumor signature).
The mutual exclusivity of signatures 3 and MMR2 corroborates a previous report suggesting tumors do not tend to feature both deficient homologous repair and mismatch repair<span class="citation" data-cites="X6oQhIf8"><sup><a href="#ref-X6oQhIf8" role="doc-biblioref">43</a></sup></span>.</p>
<div id="tbl:Table2" class="tablenos">
<table id="tbl:Table2" style="width:100%;">
<caption><span>Table 2:</span> <strong>Patients with hypermutant tumors.</strong> Listed are patients with at least one hypermutant or ultra-hypermutant tumor or cell line. Pathogenic (P) or likely pathogenic (LP) germline variants, coding region TMB, phase of therapy, therapeutic interventions, cancer predisposition (CMMRD = Constitutional mismatch repair deficiency), and molecular subtypes are included. </caption>
<colgroup>
<col style="width: 10%" />
<col style="width: 10%" />
<col style="width: 4%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 18%" />
<col style="width: 8%" />
<col style="width: 17%" />
<col style="width: 2%" />
<col style="width: 13%" />
</colgroup>
<thead>
<tr class="header">
<th>Kids First Participant ID</th>
<th>Kids First Biospecimen ID</th>
<th>CBTN ID</th>
<th>Phase of therapy</th>
<th>Composition</th>
<th>Therapy post-biopsy</th>
<th>Cancer predisposition</th>
<th>Pathogenic germline variant</th>
<th>TMB</th>
<th>OpenPBTA molecular subtype</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>PT_0SPKM4S8</td>
<td>BS_VW4XN9Y7</td>
<td>7316-2640</td>
<td>Initial CNS Tumor</td>
<td>Solid Tissue</td>
<td>Radiation, Temozolomide, CCNU</td>
<td>None documented</td>
<td>NM_000535.7(PMS2):c.137G&gt;T (p.Ser46Ile) (LP)</td>
<td>187.4</td>
<td>HGG, H3 wildtype, TP53 activated</td>
</tr>
<tr class="even">
<td>PT_3CHB9PK5</td>
<td>BS_20TBZG09</td>
<td>7316-515</td>
<td>Initial CNS Tumor</td>
<td>Solid Tissue</td>
<td>Radiation, Temozolomide, Irinotecan, Bevacizumab</td>
<td>CMMRD</td>
<td>NM_000179.3(MSH6):c.3439-2A&gt;G (LP)</td>
<td>307</td>
<td>HGG, H3 wildtype, TP53 loss</td>
</tr>
<tr class="odd">
<td>PT_3CHB9PK5</td>
<td>BS_8AY2GM4G</td>
<td>7316-2085</td>
<td>Progressive</td>
<td>Solid Tissue</td>
<td>Radiation, Temozolomide, Irinotecan, Bevacizumab</td>
<td>CMMRD</td>
<td>NM_000179.3(MSH6):c.3439-2A&gt;G (LP)</td>
<td>321.6</td>
<td>HGG, H3 wildtype, TP53 loss</td>
</tr>
<tr class="even">
<td>PT_EB0D3BXG</td>
<td>BS_F0GNWEJJ</td>
<td>7316-3311</td>
<td>Progressive</td>
<td>Solid Tissue</td>
<td>Radiation, Nivolumab</td>
<td>None documented</td>
<td>None detected</td>
<td>26.3</td>
<td>Metastatic NBL, MYCN non-amplified</td>
</tr>
<tr class="odd">
<td>PT_JNEV57VK</td>
<td>BS_85Q5P8GF</td>
<td>7316-2594</td>
<td>Initial CNS Tumor</td>
<td>Solid Tissue</td>
<td>Radiation, Temozolomide</td>
<td>Lynch Syndrome</td>
<td>NM_000251.3(MSH2):c.1906G&gt;C (p.Ala636Pro) (P)</td>
<td>4.7</td>
<td>DMG, H3 K28, TP53 loss</td>
</tr>
<tr class="even">
<td>PT_JNEV57VK</td>
<td>BS_HM5GFJN8</td>
<td>7316-3058</td>
<td>Progressive</td>
<td>Derived Cell Line</td>
<td>Radiation, Temozolomide, Nivolumab</td>
<td>Lynch Syndrome</td>
<td>NM_000251.3(MSH2):c.1906G&gt;C (p.Ala636Pro) (P)</td>
<td>35.9</td>
<td>DMG, H3 K28, TP53 loss</td>
</tr>
<tr class="odd">
<td>PT_JNEV57VK</td>
<td>BS_QWM9BPDY</td>
<td>7316-3058</td>
<td>Progressive</td>
<td>Derived Cell Line</td>
<td>Radiation, Temozolomide, Nivolumab</td>
<td>Lynch Syndrome</td>
<td>NM_000251.3(MSH2):c.1906G&gt;C (p.Ala636Pro) (P)</td>
<td>7.4</td>
<td>DMG, H3 K28, TP53 loss</td>
</tr>
<tr class="even">
<td>PT_JNEV57VK</td>
<td>BS_P0QJ1QAH</td>
<td>7316-3058</td>
<td>Progressive</td>
<td>Solid Tissue</td>
<td>Radiation, Temozolomide, Nivolumab</td>
<td>Lynch Syndrome</td>
<td>NM_000251.3(MSH2):c.1906G&gt;C (p.Ala636Pro) (P)</td>
<td>6.3</td>
<td>DMG, H3 K28, TP53 activated</td>
</tr>
<tr class="odd">
<td>PT_S0Q27J13</td>
<td>BS_P3PF53V8</td>
<td>7316-2307</td>
<td>Initial CNS Tumor</td>
<td>Solid Tissue</td>
<td>Radiation, Temozolomide, Irinotecan</td>
<td>None documented</td>
<td>None detected</td>
<td>15.5</td>
<td>HGG, H3 wildtype, TP53 activated</td>
</tr>
<tr class="even">
<td>PT_VTM2STE3</td>
<td>BS_ERFMPQN3</td>
<td>7316-2189</td>
<td>Progressive</td>
<td>Derived Cell Line</td>
<td>Unknown</td>
<td>Lynch Syndrome</td>
<td>None detected</td>
<td>5.7</td>
<td>HGG, H3 wildtype, TP53 loss</td>
</tr>
<tr class="odd">
<td>PT_VTM2STE3</td>
<td>BS_02YBZSBY</td>
<td>7316-2189</td>
<td>Progressive</td>
<td>Solid Tissue</td>
<td>Unknown</td>
<td>Lynch Syndrome</td>
<td>None detected</td>
<td>274.5</td>
<td>HGG, H3 wildtype, TP53 activated</td>
</tr>
</tbody>
</table>
</div>
<p>Next, we asked whether transcriptomic classification of <em>TP53</em> dysregulation and/or telomerase activity recapitulate the known prognostic influence of these oncogenic biomarkers.
To this end, we conducted a multivariate Cox regression on overall survival (<strong>Figure <a href="#fig:Fig4">4</a>F</strong>; <strong>STAR Methods</strong>), controlling for extent of tumor resection and whether a tumor was low-grade (LGG group) or high-grade glioma (HGG group).
We identified several expected trends, including a significant overall survival benefit if the tumor had been fully resected (HR = 0.35, 95% CI = 0.2 - 0.62, p &lt; 0.001) or if the tumor belonged to the LGG group (HR = 0.046, 95% CI = 0.0062 - 0.34, p = 0.003) as well as a significant risk if the tumor belonged to the HGG group (HR = 6.2, 95% CI = 4.0 - 9.5, p &lt; 0.001).
High telomerase scores were associated with poor prognosis across brain tumor histologies (HR = 20, 95% CI = 6.4 - 62, p &lt; 0.001), demonstrating that <code>EXTEND</code> scores calculated from RNA-Seq are an effective rapid surrogate measure for telomerase activity.
Although higher <em>TP53</em> scores, which predict <em>TP53</em> gene or pathway dysregulation, were not a significant predictor of risk across the entire OpenPBTA cohort (<strong>Table S4</strong>), we did find a significant survival risk associated with higher <em>TP53</em> scores within DMGs (HR = 6436, 95% CI = 2.67 - 1.55e7, p = 0.03) and ependymomas (HR = 2003, 95% CI = 9.9 - 4.05e5, p = 0.005).
Since we observed the negative prognostic effect of <em>TP53</em> scores for HGGs, we assessed the effect of molecular subtypes within HGGs on survival risk.
We found that DMG H3 K28 tumors with <em>TP53</em> loss had significantly worse prognosis (HR = 2.8, CI = 1.4-5.6, p = 0.003) than did DMG H3 K28 tumors with wildtype <em>TP53</em> (<strong>Figure <a href="#fig:Fig4">4</a>G</strong> and <strong>Figure <a href="#fig:Fig4">4</a>H</strong>).
This finding was also recently reported in two recent restrospective analyses of DIPG tumors<span class="citation" data-cites="9mO4Op62 1CbmKZRjS"><sup><a href="#ref-9mO4Op62" role="doc-biblioref">12</a>,<a href="#ref-1CbmKZRjS" role="doc-biblioref">54</a></sup></span>.</p>
<div id="fig:Fig4" class="fignos">
<figure>
<img src="https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/361fef09c574942d3d5ef0668461b37e3ee6cba6/figures/pngs/figure4.png?sanitize=true" style="width:7in" alt="Figure 4: TP53 and telomerase activity A, Receiver Operating Characteristic for TP53 classifier run on FPKM of stranded RNA-Seq tumors. B, Violin and strip plots of TP53 scores from stranded RNA-Seq tumors plotted by TP53 alteration type (Nactivated = 11, Nlost = 100, Nother = 866). C, Violin and strip plots of TP53 RNA expression from stranded RNA-Seq tumors plotted by TP53 activation status (Nactivated = 11, Nlost = 100, Nother = 866). D, Box plots of TP53 and telomerase (EXTEND) scores across cancer groups. Mutation status is highlighted in orange (hypermutant) or red (ultra-hypermutant). E, Heatmap of RefSig mutational signatures for patients who have least one tumor or cell line with a hypermutant phenotype. F, Forest plot depicting the prognostic effects of TP53 and telomerase scores on overall survival, controlling for extent of tumor resection, LGG group, and HGG group. G, Forest plot depicting the effect of molecular subtype on overall survival of HGGs. For F and G, hazard ratios (HR) with 95% confidence intervals and p-values (multivariate Cox) are listed. Significant p-values are denoted with black diamonds. Reference groups are denoted by grey diamonds. H, Kaplan-Meier curve of HGG tumors by molecular subtype. Box plot represents 5% (lower whisker), 25% (lower box), 50% (median), 75% (upper box), and 95% (upper whisker) quantiles." />
<figcaption aria-hidden="true"><span>Figure 4:</span> <strong><em>TP53</em> and telomerase activity</strong> A, Receiver Operating Characteristic for <em>TP53</em> classifier run on FPKM of stranded RNA-Seq tumors. B, Violin and strip plots of <em>TP53</em> scores from stranded RNA-Seq tumors plotted by <em>TP53</em> alteration type (N<sub>activated</sub> = 11, N<sub>lost</sub> = 100, N<sub>other</sub> = 866). C, Violin and strip plots of <em>TP53</em> RNA expression from stranded RNA-Seq tumors plotted by <em>TP53</em> activation status (N<sub>activated</sub> = 11, N<sub>lost</sub> = 100, N<sub>other</sub> = 866). D, Box plots of <em>TP53</em> and telomerase (EXTEND) scores across cancer groups. Mutation status is highlighted in orange (hypermutant) or red (ultra-hypermutant). E, Heatmap of RefSig mutational signatures for patients who have least one tumor or cell line with a hypermutant phenotype. F, Forest plot depicting the prognostic effects of <em>TP53</em> and telomerase scores on overall survival, controlling for extent of tumor resection, LGG group, and HGG group. G, Forest plot depicting the effect of molecular subtype on overall survival of HGGs. For F and G, hazard ratios (HR) with 95% confidence intervals and p-values (multivariate Cox) are listed. Significant p-values are denoted with black diamonds. Reference groups are denoted by grey diamonds. H, Kaplan-Meier curve of HGG tumors by molecular subtype. Box plot represents 5% (lower whisker), 25% (lower box), 50% (median), 75% (upper box), and 95% (upper whisker) quantiles.</figcaption>
</figure>
</div>
<h4 id="histologic-and-oncogenic-pathway-clustering">Histologic and oncogenic pathway clustering</h4>
<p>UMAP visualization of gene expression variation across brain tumors (<strong>Figure <a href="#fig:Fig5">5</a>A</strong>) showed the expected clustering of brain tumors by histology.
We additionally explored UMAP projections of gene expression within molecular subtypes for certain cancer groups.
We observed that, except for three outliers, <em>C11orf95::RELA</em> (<em>ZFTA::RELA</em>) fusion-positive ependymomas fell within distinct clusters (<strong>Figure <a href="#fig:S6">S6</a>A</strong>).
Medulloblastoma (MB) tumors cluster by molecular subtype, with WNT and SHH in distinct clusters and Groups 3 and 4 showing some overlap (<strong>Figure <a href="#fig:S6">S6</a>B</strong>), as expected.
Of note, two MB tumors annotated as the SHH subtype did not cluster with the other MB tumors, and one clustered with Group 3 and 4 tumors, suggesting potential subtype misclassification or different underlying biology of these two tumors.
<em>BRAF</em>-driven low-grade gliomas (<strong>Figure <a href="#fig:S6">S6</a>C</strong>) were present in three separate clusters, suggesting that there might be additional shared biology within each cluster.
Histone H3 G35-mutant high-grade gliomas generally clustered together and away from K28-mutant tumors (<strong>Figure <a href="#fig:S6">S6</a>D</strong>).
Interestingly, although H3 K28-mutant tumors have different biological drivers than do H3 wildtype tumors<span class="citation" data-cites="MeiYPRi0"><sup><a href="#ref-MeiYPRi0" role="doc-biblioref">55</a></sup></span>, they did not form distinct clusters.
This pattern suggests these subtypes may be driven by common transcriptional programs, have other much stronger biological drivers than their known distinct epigenetic drivers, or our sample size is too small to detect transcriptional differences.</p>
<p>We next performed gene set variant analysis (GSVA) for Hallmark cancer gene sets to demonstrate activation of underlying oncogenic pathways (<strong>Figure <a href="#fig:Fig5">5</a>B</strong> and quantified immune cell fractions across OpenPBTA tumors using quanTIseq (<strong>Figure <a href="#fig:Fig5">5</a>C</strong> and <strong>Figure <a href="#fig:S6">S6</a>E</strong>).
Through these analyses, we were able to recapitulate previously-described tumor biology.
For example, HGG, DMG, MB, and ATRT tumors are known to upregulate <em>MYC</em><span class="citation" data-cites="mzBdyDm0"><sup><a href="#ref-mzBdyDm0" role="doc-biblioref">56</a></sup></span> which in turn activates <em>E2F</em> and S phase<span class="citation" data-cites="pmid:11511364"><sup><a href="#ref-pmid:11511364" role="doc-biblioref"><strong>pmid:11511364?</strong></a></sup></span>.
Indeed, we detected significant (Bonferroni-corrected p &lt; 0.05) upregulation of <em>MYC</em> and <em>E2F</em> targets, as well as G2M (cell cycle phase following S phase) in MBs, ATRTs, and HGGs compared to several other cancer groups.
In contrast, LGGs showed significant downregulation (Bonferroni-corrected p &lt; 0.05) of these pathways.
Schwannomas and neurofibromas, which have a documented inflammatory immune microenvironment of T and B lymphocytes as well as tumor-associated macrophages (TAMs), are driven by upregulation of cytokines such as IFN<span class="math inline">\(\gamma\)</span>, IL-1, and IL-6, and TNF<span class="math inline">\(\alpha\)</span><span class="citation" data-cites="887hkpnS"><sup><a href="#ref-887hkpnS" role="doc-biblioref">57</a></sup></span>.
Indeed, we observed significant upregulation of these cytokines in GSVA hallmark pathways (Bonferroni-corrected p &lt; 0.05) (<strong>Figure <a href="#fig:Fig5">5</a>B</strong>) and found immune cell types dominated by monocytes in these tumors (<strong>Figure <a href="#fig:Fig5">5</a>C</strong>).
We also observed significant upregulation of pro-inflammatory cytokines IFN<span class="math inline">\(\alpha\)</span> and IFN<span class="math inline">\(\gamma\)</span> in both LGGs and craniopharyngiomas when compared to either medulloblastoma or ependymoma tumors (Bonferroni-corrected p &lt; 0.05) (<strong>Figure <a href="#fig:Fig5">5</a>B</strong>).
Together, these results supported previous proteogenomic findings of lower immune infiltration in aggressive medulloblastomas and ependymomas versus higher immune infiltration in <em>BRAF</em>-driven LGG and craniopharyngiomas<span class="citation" data-cites="meH98mKZ"><sup><a href="#ref-meH98mKZ" role="doc-biblioref">58</a></sup></span>.</p>
<p>Although CD8+ T-cell infiltration across all cancer groups was quite low (<strong>Figure <a href="#fig:Fig5">5</a>C</strong>), we observed some signal in specific cancer molecular subtypes (Groups 3 and 4 medulloblastoma) as well as outlier tumors (BRAF-driven LGG, BRAF-driven and wildtype ganglioglioma, and CNS embryonal NOS; <strong>Figure <a href="#fig:S6">S6</a>E</strong>)
Surprisingly, the classically immunologically-cold HGG and DMG tumors<span class="citation" data-cites="1B1SO39W1 RzxfvDRe"><sup><a href="#ref-1B1SO39W1" role="doc-biblioref">59</a>,<a href="#ref-RzxfvDRe" role="doc-biblioref">60</a></sup></span> contained higher overall fractions of immune cells, where monocytes, dendritic cells, and NK cells were the most prevalent (<strong>Figure <a href="#fig:Fig5">5</a>C</strong>).
Thus, we suspect that quanTIseq might actually have captured microglia within these immune cell fractions.</p>
<p>While we did not detect notable prognostic effects of immune cell infiltration on overall survival in HGG or DMG tumors, we did find that high levels of macrophage M1 and monocytes were associated with poorer overall survival (monocyte HR = 2.1e18, 95% CI = 3.80e5 - 1.2e31, p = 0.005, multivariate Cox) in medulloblastoma tumors (<strong>Figure <a href="#fig:Fig5">5</a>D</strong>).
We further reproduced previous findings (<strong>Figure <a href="#fig:Fig5">5</a>E</strong>) that medulloblastomas typically have low expression of <em>CD274</em> (PD-L1)<span class="citation" data-cites="IQLWvQPD"><sup><a href="#ref-IQLWvQPD" role="doc-biblioref">61</a></sup></span>.
However, we also found that higher expression of <em>CD274</em> was significantly associated with improved overall prognosis for medulloblastoma tumors, although with a marginal effect size (HR = 0.0012, 95% CI = 7.5e−06 - 0.18, p = 0.008, multivariate Cox) (<strong>Figure <a href="#fig:Fig5">5</a>D</strong>).
This result may be explained by the higher expression of <em>CD274</em> found in WNT subtype tumors by us and others<span class="citation" data-cites="TNNRIa7y"><sup><a href="#ref-TNNRIa7y" role="doc-biblioref">62</a></sup></span>, as this diagnosis carries the best prognosis of all medulloblastoma subgroups (<strong>Figure <a href="#fig:Fig5">5</a>E</strong>).</p>
<p>Finally, we asked whether any molecular subtypes might have a high ratio CD8+ to CD4+ T cells, a metric which has been associated with better immunotherapy response and prognosis following PD-L1 inhibition in non-small cell lung cancer or adoptive T cell therapy in multiple stage III or IV cancers<span class="citation" data-cites="ic0BrsYM MAFXjHGX"><sup><a href="#ref-ic0BrsYM" role="doc-biblioref">63</a>,<a href="#ref-MAFXjHGX" role="doc-biblioref">64</a></sup></span>.
While adamantinomatous craniopharyngiomas and Group 3 and Group 4 medulloblastomas had the highest CD8+ to CD4+ T cell ratios (<strong>Figure <a href="#fig:S6">S6</a>F</strong>), very few tumors had ratios greater than 1, highlighting an urgent need to identify novel therapeutics for pediatric brain tumors with poor prognosis.</p>
<div id="fig:Fig5" class="fignos">
<figure>
<img src="https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/361fef09c574942d3d5ef0668461b37e3ee6cba6/figures/pngs/figure5.png?sanitize=true" style="width:7in" alt="Figure 5: Transcriptomic and immune landscape of pediatric brain tumors A, First two dimensions from UMAP of transcriptome data for samples with stranded library preparation. Points are colored by the broad histology of the tumors they represent. B, Heatmap of GSVA scores for Hallmark gene sets with significant differences, with tumors ordered by cancer group (only scores for samples with stranded library preparation are shown). C, Box plots of quanTIseq estimates of immune cell proportions in select cancer groups with N &gt; 15 tumors. Note: Other HGGs and other LGGs have immune cell proportions similar to DMG and pilocytic astrocytoma, respectively, and are not shown. D, Forest plot depicting the additive effects of CD274 expression, immune cell proportion, and extent of tumor resection on overall survival of medulloblastoma patients. Hazard ratios (HR) with 95% confidence intervals and p-values (multivariate Cox) are listed. Significant p-values are denoted with black diamonds. Reference groups are denoted by grey diamonds. Of note, the Macrophage M1 HR was 0 (coefficient = -9.90e+4) with infinite upper and lower CIs, and thus it was not included in the figure. E, Box plot of CD274 expression (log2 FPKM) for medulloblastoma tumors grouped by molecular subtype. Bonferroni-corrected p-values from Wilcoxon tests are shown. Box plot represents 5% (lower whisker), 25% (lower box), 50% (median), 75% (upper box), and 95% (upper whisker) quantiles." />
<figcaption aria-hidden="true"><span>Figure 5:</span> <strong>Transcriptomic and immune landscape of pediatric brain tumors</strong> A, First two dimensions from UMAP of transcriptome data for samples with stranded library preparation. Points are colored by the broad histology of the tumors they represent. B, Heatmap of GSVA scores for Hallmark gene sets with significant differences, with tumors ordered by cancer group (only scores for samples with stranded library preparation are shown). C, Box plots of quanTIseq estimates of immune cell proportions in select cancer groups with N &gt; 15 tumors. Note: Other HGGs and other LGGs have immune cell proportions similar to DMG and pilocytic astrocytoma, respectively, and are not shown. D, Forest plot depicting the additive effects of <em>CD274</em> expression, immune cell proportion, and extent of tumor resection on overall survival of medulloblastoma patients. Hazard ratios (HR) with 95% confidence intervals and p-values (multivariate Cox) are listed. Significant p-values are denoted with black diamonds. Reference groups are denoted by grey diamonds. Of note, the Macrophage M1 HR was 0 (coefficient = -9.90e+4) with infinite upper and lower CIs, and thus it was not included in the figure. E, Box plot of <em>CD274</em> expression (log<sub>2</sub> FPKM) for medulloblastoma tumors grouped by molecular subtype. Bonferroni-corrected p-values from Wilcoxon tests are shown. Box plot represents 5% (lower whisker), 25% (lower box), 50% (median), 75% (upper box), and 95% (upper whisker) quantiles.</figcaption>
</figure>
</div>
<h2 id="discussion">Discussion</h2>
<p>The CBTN released the raw genomic data for the PBTA in September 2018 without embargo to allow researchers immediate access to begin making discoveries on behalf of children with CNS tumors everywhere.
Since the release of the raw data, the CBTN has approved nearly 200 data research projects<span class="citation" data-cites="U5WisTG5"><sup><a href="#ref-U5WisTG5" role="doc-biblioref">4</a></sup></span> from 69 different institutions, with 60% from non-CBTN sites.
We created OpenPBTA to define an open, real-time, reproducible analysis framework to genomically characterize pediatric brain tumors that brings together basic and translational researchers, clinicians, and data scientists on behalf of accelerated discovery and clinical impact.
We provide robust reusable code and data resources, paired with cloud-based availability of source and derived data resources, to the pediatric oncology community, encouraging interdisciplinary scientists to collaborate on new analyses in order to accelerate therapeutic translation for children with cancer, goals we are seeing play out in real-time.
To our knowledge, this initiative represents the first large-scale, collaborative, open analysis of genomic data coupled with open manuscript writing, in which we comprehensively analyzed the largest cohort of pediatric brain tumors to date, comprising 1,074 tumors across 58 distinct histologies.
We used available WGS, WXS, and RNA-Seq data to generate high-confidence consensus SNV and CNV calls, prioritize putative oncogenic fusions, and establish over 40 scalable modules to perform common downstream cancer genomics analyses, all of which have undergone rigorous scientific and analytical code review.
We detected and showed expected patterns of genomic lesions, mutational signatures, and aberrantly regulated signaling pathways across multiple pediatric brain tumor histologies.</p>
<p>Assembling large, pan-histology cohorts of fresh frozen samples and associated clinical phenotypes and outcomes requires a multi-year, multi-institutional framework, like those provided by CBTN and PNOC.
As such, uniform clinical molecular subtyping was largely not performed for most of this cohort at the time of diagnosis and/or at surgery, and when available (e.g., sparse medulloblastoma subtypes), it required manual curation from pathology reports and/or free text clinical data fields.
Furthermore, rapid classification to derive molecular subtypes could not be immediately performed since research-based DNA methylation data for these samples are not yet available.
Thus, to enable biological interrogation of specific tumor subtypes, we created RNA- and DNA-based subtyping modules aligned with WHO molecularly-defined diagnoses.
We worked closely with pathologists and clinicians to build modules from which we determined a research-grade integrated diagnosis for 60% of tumors while discovering incorrectly diagnosed or mis-identified samples in the OpenPBTA cohort. <!--SAMPLECOUNT--></p>
<p>We harnessed RNA expression data for a number of analyses, yielding important biological insights across multiple brain tumor histologies.
For example, we performed subtyping of medulloblastoma tumors, for which only 35% (43/122) had subtype information from pathology reports. <!--SAMPLECOUNT-->
Among the subtyped tumors, we accurately recapitulated subtypes using MM2S (91%; 39/43) or medulloPackage (95%; 41/43)<span class="citation" data-cites="RmzXK4Cw 12DKhuCiy"><sup><a href="#ref-12DKhuCiy" role="doc-biblioref">24</a>,<a href="#ref-RmzXK4Cw" role="doc-biblioref">25</a></sup></span>. <!--SAMPLECOUNT-->
We then applied the consensus of these methods to subtype all medulloblastoma tumors lacking pathology-based subtypes.</p>
<p>We advanced the integrative analyses and cross-cohort comparison via a number of validated modules.
We used an expression classifier to determine whether tumors have dysfunctional <em>TP53</em><span class="citation" data-cites="X6oQhIf8"><sup><a href="#ref-X6oQhIf8" role="doc-biblioref">43</a></sup></span> and the EXTEND algorithm to determine their degree of telomerase activity using a 13-gene signature<span class="citation" data-cites="RnBZNbdx"><sup><a href="#ref-RnBZNbdx" role="doc-biblioref">46</a></sup></span>.
Interestingly, in contrast to adult colorectal cancer and gastric adenocarcinoma, in which <em>TP53</em> loss of function is less frequent in hypermutated tumors<span class="citation" data-cites="sOb4Wh4q 11x4p1w23"><sup><a href="#ref-sOb4Wh4q" role="doc-biblioref">65</a>,<a href="#ref-11x4p1w23" role="doc-biblioref">66</a></sup></span>, we found that hypermutant HGG tumors universally displayed dysregulation of <em>TP53</em>.
Furthermore, high <em>TP53</em> scores were a significant prognostic marker for poor overall survival for patients with certain tumor types, such as H3 K28-mutant DMGs and ependymomas.
We also show that EXTEND scores are a robust surrogate measure for telomerase activity in pediatric brain tumors.
By assessing <em>TP53</em> and telomerase activity prospectively from expression data, information usually only attainable with DNA sequencing and/or qPCR, we can quickly incorporate oncogenic biomarker and prognostic knowledge and expand our biological understanding of these tumors.</p>
<p>We identified enrichment of hallmark cancer pathways and characterized the immune cell landscape across pediatric brain tumors, demonstrating tumors in some histologies, such as schwannomas, craniopharyngiomas, and low-grade gliomas, may have a inflammatory tumor microenvironment.
Of note, we observed upregulation of IFN<span class="math inline">\(\gamma\)</span>, IL-1, and IL-6, and TNF<span class="math inline">\(\alpha\)</span> in craniopharyngiomas, tumors difficult to resect due to their anatomical location and critical surrounding structures.
Neurotoxic side effects have been reported when interferon alpha immunotherapy is administered to reduce cystic craniopharyngioma tumor size and/or delay progression<span class="citation" data-cites="nCoKt2Vg rfGUdNhB"><sup><a href="#ref-nCoKt2Vg" role="doc-biblioref">67</a>,<a href="#ref-rfGUdNhB" role="doc-biblioref">68</a></sup></span>.
Thus, additional immune vulnerabilities, such as IL-6 inhibition and immune checkpoint blockade, have recently been proposed as therapies for cystic adamantinomatous craniopharyngiomas<span class="citation" data-cites="i9VLAGYQ pmid:34966342 pmid:32075140 15tuSDJZh oyGGylFL"><sup><a href="#ref-i9VLAGYQ" role="doc-biblioref">69</a>–<a href="#ref-oyGGylFL" role="doc-biblioref">71</a>,<a href="#ref-pmid:34966342" role="doc-biblioref"><strong>pmid:34966342?</strong></a>,<a href="#ref-pmid:32075140" role="doc-biblioref"><strong>pmid:32075140?</strong></a></sup></span> and our results noted above support this approach.
Finally, our study reproduced the overall known poor infiltration of CD8+ T cells and general low expression of <em>CD274</em> (PD-L1) in pediatric brain tumors, further highlighting the urgent need to identify novel therapeutic strategies for tumors unlikely to respond to immune checkpoint blockade therapy.</p>
<p>We note that while large-scale collaborative efforts tend to take a longer time to complete, our adoption of an open science framework for OpenPBTA substantially mitigated this concern.
By maintaining all data, analytical code, and results in public repositories, we ensured that such logistics did not hinder progress in the pediatric cancer research space.
Indeed, OpenPBTA has rapidly become a foundational data analysis and processing layer for a number of discovery research and translational projects which will continue to add other genomic modalities and analyses, such as germline, methylation, single cell, epigenomic, mRNA splicing, imaging, and model drug response data.
For example, the RNA fusion filtering module created within OpenPBTA set the stage for development of the R package <em>annoFuse</em><span class="citation" data-cites="10LfPTs0c"><sup><a href="#ref-10LfPTs0c" role="doc-biblioref">72</a></sup></span> and an R Shiny application <a href="http://shiny.imbei.uni-mainz.de:3838/shinyFuse/"><em>shinyFuse</em></a>.
Using medulloblastoma subtyping and immune deconvolution analyses performed herein, Dang and colleagues showed enrichment of monocyte and microglia-derived macrophages within the SHH subgroup which they suggest may accumulate following radiation therapy<span class="citation" data-cites="XYoocEyx"><sup><a href="#ref-XYoocEyx" role="doc-biblioref">11</a></sup></span>.
Expression and copy number analyses were used to demonstrate that <em>GPC2</em> is a highly expressed and copy number gained immunotherapeutic target in ETMRs, medulloblastomas, choroid plexus carcinomas, H3 wildtype high-grade gliomas, as well as DMGs.
This led Foster and colleagues to subsequently develop a chimeric antigen receptor (CAR) directed against <em>GPC2</em>, for which they show preclinical efficacy in mouse models<span class="citation" data-cites="kIFuKyuK"><sup><a href="#ref-kIFuKyuK" role="doc-biblioref">13</a></sup></span>.
Another recent study harnessed OpenPBTA to integrate germline analyses and discovered that pediatric HGG patients whose tumors undergo alternative lengthening of telomeres have enrichment of predicted pathogenic or likely pathogenic variants in genes in the mismatch repair pathway, oncogenic <em>ATRX</em> mutations, and increased TMB<span class="citation" data-cites="lNLcNX6n"><sup><a href="#ref-lNLcNX6n" role="doc-biblioref">14</a></sup></span>.
Moreover, OpenPBTA has enabled a framework to support real-time integration of clinical trial subjects as each was enrolled on the PNOC008 high-grade glioma clinical trial<span class="citation" data-cites="eQQpBw1T"><sup><a href="#ref-eQQpBw1T" role="doc-biblioref">73</a></sup></span> or PNOC027 medulloblastoma clinical trial<span class="citation" data-cites="8YqNPLmu"><sup><a href="#ref-8YqNPLmu" role="doc-biblioref">74</a></sup></span>, allowing researchers and clinicians to link tumor biology to translational impact through clinical decision support during tumor board discussions.
Finally, as part of the the NCI’s Childhood Cancer Data Initiative (CCDI), the OpenPBTA project was recently expanded into a pan-pediatric cancer effort (<a href="https://github.com/PediatricOpenTargets/OpenPedCan-analysis">https://github.com/PediatricOpenTargets/OpenPedCan-analysis</a>) to build the Molecular Targets Platform (<a href="https://moleculartargets.ccdi.cancer.gov/">https://moleculartargets.ccdi.cancer.gov/</a>) in support of the RACE Act.
An additional, large-scale cohort of &gt;1,500 tumor samples and associated germline DNA is in the process of undergoing sequence data generation as part of CBTN CCDI-Kids First NCI and Common Fund project (<a href="https://commonfund.nih.gov/kidsfirst/2021X01projects#FY21_Resnick">https://commonfund.nih.gov/kidsfirst/2021X01projects#FY21_Resnick</a>).
Like the original OpenPBTA cohort, data will be processed and released in near real-time via the Kids First Data Resource and integrated with OpenPBTA.
The OpenPBTA project has paved the way for new modes of collaborative data-driven discovery, open, reproducible, and scalable analyses that will extend beyond the current research described herein, and we anticipate this foundational work will continue to have a long-term impact within the pediatric brain tumor translational research community and beyond, ultimately leading to accelerated impact and improved outcomes for children with cancer.</p>
<p>All code and processed data are openly available through GitHub, CAVATICA, and PedcBioPortal (see <strong>STAR METHODS</strong>).</p>
<h2 id="acknowledgments">Acknowledgments</h2>
<p>We graciously thank the patients and families who have donated their tumors to the Children’s Brain Tumor Network (CBTN) and/or the Pacific Pediatric Neuro-oncology Consortium, without which this research would not be possible.</p>
<p>Philanthropic support has ensured the CBTN’s ability to collect, store, manage, and distribute specimen and data since it was founded in 2013.
In addition to the support from the CBTN Executive Council members and Brain Tumor Board of Visitors, the following donors have provided leadership level support for CBTN: Children’s Brain Tumor Foundation, Easie Family Foundation, Kortney Rose Foundation, Lilabean Foundation, Minnick Family Charitable Fund, Perricelli Family, Psalm 103 Foundation, and Swifty Foundation.</p>
<p>This work was funded through the Alex’s Lemonade Stand Foundation (ALSF) Childhood Cancer Data Lab (CSG), ALSF Young Investigator Award (JLR), ALSF Catalyst Award (JLR, ACR, PBS), ALSF Catalyst Award (SJS), ALSF CCDL Postdoctoral Training Grant (SMF), Children’s Hospital of Philadelphia Division of Neurosurgery (PBS and ACR), the Australian Government, Department of Education (APH), the St. Anna Kinderkrebsforschung, Austria (ARP), the Mildred Scheel Early Career Center Dresden P2, funded by the German Cancer Aid (ARP), and NIH Grants 3P30 CA016520-44S5 (ACR), U2C HL138346-03 (ACR, APH), U24 CA220457-03 (ACR), K12GM081259 (SMF), R03-CA23036 (SJD), and NIH Contract No. HHSN261200800001E (SJD).
This project has been funded in part with Federal funds from the National Cancer Institute, National Institutes of Health, under Contract No. 75N91019D00024, Task Order No. 75N91020F00003 (JLR, ACR, APH).
Additionally, this work was supported by the Intramural Research Program of the Division of Cancer Epidemiology and Genetics of the National Cancer Institute.
The content of this publication does not necessarily reflect the views or policies of the Department of Health and Human Services, nor does mention of trade names, commercial products or organizations imply endorsement by the U.S. Government.</p>
<p>The authors would like to thank the following collaborators who contributed or supervised analyses present in the analysis repository that were not included in the manuscript: William Amadio, Holly C. Beale, Ellen T. Kephart, A. Geoffrey Lyle, and Olena M. Vaske.
Finally, we would like to thank Yuanchao Zhang and Eric Wafula for adding to the project codebase, Jessica B. Foster for helpful discussions while drafting the manuscript, and Gina D. Mawla for identifying and reporting issues in OpenPBTA data.</p>
<h2 id="author-contributions">Author Contributions</h2>
<table>
<colgroup>
<col style="width: 50%" />
<col style="width: 50%" />
</colgroup>
<thead>
<tr class="header">
<th>Author</th>
<th>Contributions</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Joshua A. Shapiro</td>
<td>Methodology, Software, Validation, Formal analysis, Investigation, Writing - Original draft, Writing - Review and editing, Visualization, Supervision</td>
</tr>
<tr class="even">
<td>Krutika S. Gaonkar</td>
<td>Data curation, Formal analysis, Investigation, Methodology, Software, Writing – Original draft, Writing - Review and editing</td>
</tr>
<tr class="odd">
<td>Candace L. Savonen</td>
<td>Methodology, Software, Validation, Formal analysis, Investigation, Writing - Original draft, Writing - Review and editing, Visualization</td>
</tr>
<tr class="even">
<td>Stephanie J. Spielman</td>
<td>Validation, Formal analysis, Writing - Original draft, Writing - Review and editing, Investigation, Software, Visualization, Supervision, Funding acquisition</td>
</tr>
<tr class="odd">
<td>Chante J. Bethell</td>
<td>Methodology, Validation, Formal analysis, Investigation, Writing - Original draft, Visualization</td>
</tr>
<tr class="even">
<td>Run Jin</td>
<td>Data curation, Formal analysis, Visualization, Writing – Original draft, Writing - Review and editing</td>
</tr>
<tr class="odd">
<td>Komal S. Rathi</td>
<td>Formal analysis, Investigation, Methodology, Writing – Original draft</td>
</tr>
<tr class="even">
<td>Yuankun Zhu</td>
<td>Data curation, Formal analysis, Investigation, Methodology, Supervision</td>
</tr>
<tr class="odd">
<td>Laura E. Egolf</td>
<td>Formal analysis, Writing - Original draft</td>
</tr>
<tr class="even">
<td>Bailey K. Farrow</td>
<td>Data curation, Software</td>
</tr>
<tr class="odd">
<td>Daniel P. Miller</td>
<td>Formal analysis</td>
</tr>
<tr class="even">
<td>Yang Yang</td>
<td>Formal analysis, Software</td>
</tr>
<tr class="odd">
<td>Tejaswi Koganti</td>
<td>Formal analysis, Investigation</td>
</tr>
<tr class="even">
<td>Nighat Noureen</td>
<td>Formal analysis, Visualization, Writing - Original draft</td>
</tr>
<tr class="odd">
<td>Mateusz P. Koptyra</td>
<td>Formal analysis, Writing – Original draft</td>
</tr>
<tr class="even">
<td>Nhat Duong</td>
<td>Formal analysis, Investigation, Methodology</td>
</tr>
<tr class="odd">
<td>Adam A. Kraya</td>
<td>Methodology</td>
</tr>
<tr class="even">
<td>Cassie Kline</td>
<td>Supervision, Investigation, Writing - Review and editing</td>
</tr>
<tr class="odd">
<td>Yiran Guo</td>
<td>Formal analysis, Writing - Review and editing</td>
</tr>
<tr class="even">
<td>Phillip B. Storm</td>
<td>Conceptualization, Funding acquisition, Resources</td>
</tr>
<tr class="odd">
<td>Javad Nazarian</td>
<td>Conceptualization</td>
</tr>
<tr class="even">
<td>Stephen C. Mack</td>
<td>Writing - Review and editing</td>
</tr>
<tr class="odd">
<td>Pichai Raman</td>
<td>Conceptualization, Formal analysis, Methodology</td>
</tr>
<tr class="even">
<td>Siyuan Zheng</td>
<td>Formal analysis, Visualization, Writing - Original draft, Supervision, Writing - Review and editing</td>
</tr>
<tr class="odd">
<td>Peter J. Madsen</td>
<td>Writing – Review &amp; editing</td>
</tr>
<tr class="even">
<td>Nicholas Van Kuren</td>
<td>Data curation, Software</td>
</tr>
<tr class="odd">
<td>Shannon Robins</td>
<td>Data curation</td>
</tr>
<tr class="even">
<td>Xiaoyan Huang</td>
<td>Formal analysis</td>
</tr>
<tr class="odd">
<td>Angela J. Waanders</td>
<td>Supervision, Conceptualization</td>
</tr>
<tr class="even">
<td>Jung Kim</td>
<td>Investigation, Writing - Review and editing</td>
</tr>
<tr class="odd">
<td>Gregory P. Way</td>
<td>Investigation, Writing - Review and editing</td>
</tr>
<tr class="even">
<td>Christopher Blackden</td>
<td>Resources</td>
</tr>
<tr class="odd">
<td>Eric Wafula</td>
<td>Formal analysis, Software</td>
</tr>
<tr class="even">
<td>Noel Coleman</td>
<td>Data curation</td>
</tr>
<tr class="odd">
<td>Derek Hanson</td>
<td>Validation</td>
</tr>
<tr class="even">
<td>Carl J. Koschmann</td>
<td>Conceptualization</td>
</tr>
<tr class="odd">
<td>Sharon J. Diskin</td>
<td>Investigation, Supervision, Validation, Funding acquisition, Writing - Review and editing</td>
</tr>
<tr class="even">
<td>Rishi R. Lulla</td>
<td>Conceptualization</td>
</tr>
<tr class="odd">
<td>Miguel A. Brown</td>
<td>Data curation, Methodology, Formal analysis</td>
</tr>
<tr class="even">
<td>Jessica Wong</td>
<td>Writing – Original draft</td>
</tr>
<tr class="odd">
<td>Jennifer L. Mason</td>
<td>Supervision</td>
</tr>
<tr class="even">
<td>Laura Scolaro</td>
<td>Data curation</td>
</tr>
<tr class="odd">
<td>Meen Chul Kim</td>
<td>Data curation</td>
</tr>
<tr class="even">
<td>Hongbo M. Xie</td>
<td>Methodology, Supervision</td>
</tr>
<tr class="odd">
<td>Brian R. Rood</td>
<td>Conceptualization</td>
</tr>
<tr class="even">
<td>Kristina A. Cole</td>
<td>Writing - Review and editing</td>
</tr>
<tr class="odd">
<td>Ashley S. Margol</td>
<td>Writing - Review and editing</td>
</tr>
<tr class="even">
<td>Zalman Vaksman</td>
<td>Formal analysis, Investigation</td>
</tr>
<tr class="odd">
<td>Rebecca S. Kaufman</td>
<td>Formal analysis, Investigation, Validation</td>
</tr>
<tr class="even">
<td>Allison P. Heath</td>
<td>Project administration, Funding acquisition</td>
</tr>
<tr class="odd">
<td>Brian M. Ennis</td>
<td>Data curation, Formal analysis</td>
</tr>
<tr class="even">
<td>Mariarita Santi</td>
<td>Investigation, Validation, Writing - Review and editing</td>
</tr>
<tr class="odd">
<td>Anna R. Poetsch</td>
<td>Formal analysis, Funding acquisition, Writing – Review and editing</td>
</tr>
<tr class="even">
<td>Angela N. Viaene</td>
<td>Investigation, Validation</td>
</tr>
<tr class="odd">
<td>Douglas R. Stewart</td>
<td>Supervision, Writing - Review and editing</td>
</tr>
<tr class="even">
<td>Steven M. Foltz</td>
<td>Validation, Funding acquisition</td>
</tr>
<tr class="odd">
<td>Payal Jain</td>
<td>Data curation, Investigation, Validation</td>
</tr>
<tr class="even">
<td>Bo Zhang</td>
<td>Data curation, Formal analysis</td>
</tr>
<tr class="odd">
<td>Shrivats Kannan</td>
<td>Formal analysis, Methodology, Writing – Original draft</td>
</tr>
<tr class="even">
<td>Michael Prados</td>
<td>Conceptualization</td>
</tr>
<tr class="odd">
<td>Jena V. Lilly</td>
<td>Conceptualization, Funding acquisition, Project administration</td>
</tr>
<tr class="even">
<td>Sabine Mueller</td>
<td>Conceptualization</td>
</tr>
<tr class="odd">
<td>Adam C. Resnick</td>
<td>Conceptualization, Funding acquisition, Resources, Supervision</td>
</tr>
<tr class="even">
<td>Casey S. Greene</td>
<td>Conceptualization, Funding acquisition, Methodology, Project administration, Software, Supervision, Writing – Review &amp; editing</td>
</tr>
<tr class="odd">
<td>Jo Lynne Rokita</td>
<td>Conceptualization, Data curation, Formal analysis, Funding acquisition, Investigation, Methodology, Software, Supervision, Writing – Original draft, Writing - Review and editing</td>
</tr>
<tr class="even">
<td>Jaclyn N. Taroni</td>
<td>Methodology, Software, Validation, Formal analysis, Investigation, Data curation, Writing - Review and editing, Visualization, Supervision, Project administration</td>
</tr>
<tr class="odd">
<td>Children’s Brain Tumor Network</td>
<td>Conceptualization</td>
</tr>
<tr class="even">
<td>Pacific Pediatric Neuro-Oncology Consortium</td>
<td>Conceptualization</td>
</tr>
</tbody>
</table>
<p>Except for the first and last four authors, authorship order was determined as follows: Authors who contributed to the OpenPBTA code base are listed based on number of modules included in the manuscript to which that individual contributed and, in the case of ties, a random order is used. All remaining authors are then listed in a random order.</p>
<p>Code for determining authorship order can be found in the <code>count-contributions</code> module of the OpenPBTA analysis repository.</p>
<h2 id="declarations-of-interest">Declarations of Interest</h2>
<p>CSG’s spouse was an employee of Alex’s Lemonade Stand Foundation, which was a sponsor of this research.
JAS, CLS, CJB, SJS, and JNT are or were employees of Alex’s Lemonade Stand Foundation, a sponsor of this research.
AJW is a member of the Scientific Advisory boards for Alexion and DayOne Biopharmaceuticals.</p>
<h2 id="figure-titles-and-legends">Figure Titles and Legends</h2>
<p><strong>Figure 1. Overview of the OpenPBTA Project.</strong>
A, The Children’s Brain Tumor Network and the Pacific Pediatric Neuro-Oncology Consortium collected tumors from 943 patients. To date, 22 cell lines were created from tumor tissue, and over 2000 specimens were sequenced (N = 1035 RNA-Seq, N = 940 WGS, and N = 32 WXS or targeted panel). Data was harmonized by the Kids First Data Resource Center using an Amazon S3 framework within CAVATICA. B, Stacked bar plot summary of the number of biospecimens per phase of therapy. Each panel denotes a broad histology and each bar denotes a cancer group. (Abbreviations: GNG = ganglioglioma, Other LGG = other low-grade glioma, PA = pilocytic astrocytoma, PXA = pleomorphic xanthoastrocytoma, SEGA = subependymal giant cell astrocytoma, DIPG = diffuse intrinsic pontine glioma, DMG = diffuse midline glioma, Other HGG = other high-grade glioma, ATRT = atypical teratoid rhabdoid tumor, MB = medulloblastoma, Other ET = other embryonal tumor, EPN = ependymoma, PNF = plexiform neurofibroma, DNET = dysembryoplastic neuroepithelial tumor, CRANIO = craniopharyngioma, EWS = Ewing sarcoma, CPP = choroid plexus papilloma). Only tumors with available descriptors were included. C, Overview of the open analysis and manuscript contribution model. In the analysis GitHub repository, a contributor would propose an analysis that other participants can comment on. Contributors would then implement the analysis and file a request to add their changes to the analysis repository (“pull request”). Pull requests underwent review for scientific rigor and correctness of implementation. Pull requests were additionally checked to ensure that all software dependencies were included and the code was not sensitive to underlying data changes using container and continuous integration technologies. Finally, a contributor would file a pull request documenting their methods and results to the Manubot-powered manuscript repository. Pull requests in the manuscript repository were also subject to review. D, A potential path for an analytical pull request. Arrows indicate revisions to a pull request. Prior to review, a pull request was tested for dependency installation and whether or not the code would execute. Pull requests also required approval by organizers and/or other contributors, who checked for scientific correctness. Panel A created with <a href="biorender.com">BioRender.com</a>.</p>
<p><strong>Figure 2. Mutational landscape of PBTA tumors.</strong>
Shown are frequencies of canonical somatic gene mutations, CNVs, fusions, and TMB (top bar plot) for the top 20 genes mutated across primary tumors within the OpenPBTA dataset. A, Low-grade astrocytic tumors (N = 227): pilocytic astrocytoma (N = 104), other low-grade glioma (N = 69), ganglioglioma (N = 35), pleomorphic xanthoastrocytoma (N = 9), subependymal giant cell astrocytoma (N = 10); B, Embryonal tumors (N = 128): medulloblastomas (N = 95), atypical teratoid rhabdoid tumors (N = 24), other embryonal tumors (N = 9); C, Diffuse astrocytic and oligodendroglial tumors (N = 61): diffuse midline gliomas (N = 34) and other high-grade gliomas (N = 27); D, Other CNS tumors (N = 194): ependymomas (N = 60), craniopharyngiomas (N = 31), meningiomas (N = 17), dysembryoplastic neuroepithelial tumors (N = 19), Ewing sarcomas (N = 7), schwannomas (N = 11), and neurofibroma plexiforms (N = 7). Additional, rare CNS tumors are displayed in <strong>Figure <a href="#fig:S3">S3</a>A</strong>. Tumor histology (<code>Cancer Group</code>) and patient sex (<code>Germline sex estimate</code>) are displayed as annotations at the bottom of each plot. Only samples with mutations in the listed genes are shown. Multiple CNVs are denoted as a complex event.</p>
<p><strong>Figure 3. Mutational co-occurrence and signatures highlight key oncogenic drivers.</strong>
A, Bar plot of occurrence and co-occurrence of nonsynonymous mutations for the 50 most commonly mutated genes across all tumor types (annotated from <code>cancer_group</code> if N &gt;= 10 or <code>Other</code> if N &lt; 10); B, Co-occurrence and mutual exclusivity of nonsynonymous mutations between genes; The co-occurrence score is defined as <span class="math inline">\(I(-\log_{10}(P))\)</span> where <span class="math inline">\(P\)</span> is defined by Fisher’s exact test and <span class="math inline">\(I\)</span> is 1 when mutations co-occur more often than expected and -1 when exclusivity is more common; C, The number of SV breaks significantly correlates with the number of CNV breaks (Adjusted R = 0.438, p = 1.08e-37). D, Chromothripsis frequency across pediatric brain tumors shown by <code>cancer_group</code> with N &gt;= 3. E, Sina plots of RefSig signature weights for signatures 1, 11, 18, 19, 3, 8, N6, MMR2, and Other across cancer groups. Box plot lines represent the first quartile, median, and third quartile.</p>
<p><strong>Figure 4. <em>TP53</em> and telomerase activity</strong>
A, Receiver Operating Characteristic for <em>TP53</em> classifier run on FPKM of stranded RNA-Seq samples. B, Violin and box plots of <em>TP53</em> scores plotted by <em>TP53</em> alteration type (N<sub>activated</sub> = 11, N<sub>lost</sub> = 100, N<sub>other</sub> = 866). C, Violin and box plots of <em>TP53</em> RNA expression plotted by <em>TP53</em> activation status (N<sub>activated</sub> = 11, N<sub>lost</sub> = 100, N<sub>other</sub> = 866). D, Box plots of <em>TP53</em> and telomerase (EXTEND) scores grouped by <code>cancer_group</code>. Mutation status is highlighted in orange (hypermutant) or red (ultra-hypermutant). E, Heatmap of RefSig mutational signatures for patients who have least one tumor or cell line with a TMB &gt;= 10 Mut/Mb. F, Forest plot depicting the prognostic effects of <em>TP53</em> and telomerase scores on overall survival, controlling for extent of tumor resection, LGG group, and HGG group. G, Forest plot depicting the effect of molecular subtype on overall survival of HGGs. For F and G, hazard ratios (HR) with 95% confidence intervals and p-values are listed. Significant p-values are denoted with black diamonds. Reference groups are denoted by grey diamonds. H, Kaplan-Meier curve of HGG tumors by molecular subtype.</p>
<p><strong>Figure 5. Transcriptomic and immune landscape of pediatric brain tumors</strong>
A, First two dimensions from UMAP of sample transcriptome data. Points are colored by the broad histology of the samples they represent. B, Heatmap of GSVA scores for Hallmark gene sets with significant differences, with samples ordered by cancer group. C, Box plots of quanTIseq estimates of immune cell proportions in select cancer groups with N &gt; 15 samples. Note: Other HGGs and other LGGs have immune cell proportions similar to DMG and pilocytic astrocytoma, respectively, and are not shown. D, Forest plot depicting the additive effects of <em>CD274</em> expression, immune cell proportion, and extent of tumor resection on overall survival of medulloblastoma patients. Hazard ratios (HR) with 95% confidence intervals and p-values are listed. Significant p-values are denoted with black diamonds. Reference groups are denoted by grey diamonds. Of note, the Macrophage M1 HR was 0 (coefficient = -9.90e+4) with infinite upper and lower CIs, and thus it was not included in the figure. E, Box plot of <em>CD274</em> expression (log<sub>2</sub> FPKM) for medulloblastoma samples grouped by molecular subtype. Bonferroni-corrected p-values from Wilcoxon tests are shown.</p>
<h2 id="table-titles-and-legends">Table Titles and Legends</h2>
<p><strong>Table 1. Molecular subtypes generated through the OpenPBTA project.</strong>
Listed are broad tumor histologies, molecular subtypes generated, and number of patients and tumors subtyped within the OpenPBTA project.</p>
<p><strong>Table 2. Patients with hypermutant tumors.</strong>
Listed are patients with at least one hypermutant or ultra-hypermutant tumor or cell line. Pathogenic (P) or likely pathogenic (LP) germline variants, coding region TMB, phase of therapy, therapeutic interventions, cancer predisposition (CMMRD = Constitutional mismatch repair deficiency), and molecular subtypes are included.</p>
<h2 id="star-methods">STAR METHODS</h2>
<h3 id="resource-availability">RESOURCE AVAILABILITY</h3>
<h4 id="lead-contact">Lead contact</h4>
<p>Requests for access to OpenPBTA raw data and/or specimens may be directed to, and will be fulfilled by Jo Lynne Rokita (rokita@chop.edu).</p>
<h4 id="materials-availability">Materials availability</h4>
<p>This study did not create new, unique reagents.</p>
<h4 id="data-and-code-availability">Data and code availability</h4>
<p>Raw and harmonized WGS, WXS, and RNA-Seq data derived from human samples are available within the KidsFirst Portal<span class="citation" data-cites="THjIzNXb"><sup><a href="#ref-THjIzNXb" role="doc-biblioref">75</a></sup></span> upon access request to the CBTN (<a href="https://cbtn.org/">https://cbtn.org/</a>) as of the date of the publication.
In addition, merged summary files are openly accessible at <a href="https://cavatica.sbgenomics.com/u/cavatica/openpbta">https://cavatica.sbgenomics.com/u/cavatica/openpbta</a> or via download script in the <a href="https://github.com/AlexsLemonade/OpenPBTA-analysis">https://github.com/AlexsLemonade/OpenPBTA-analysis</a> repository.
Summary data are visible within PedcBioPortal at <a href="https://pedcbioportal.kidsfirstdrc.org/study/summary?id=openpbta">https://pedcbioportal.kidsfirstdrc.org/study/summary?id=openpbta</a>.
Associated DOIs are listed in the <strong>Key Resources Table</strong>.
<!-- TODO: create data dois --></p>
<p>All original code was developed within the following repositories and is publicly available as follows.
Primary data analyses can be found at <a href="https://github.com/d3b-center/OpenPBTA-workflows">https://github.com/d3b-center/OpenPBTA-workflows</a>.
Downstream data analyses can be found at <a href="https://github.com/AlexsLemonade/OpenPBTA-analysis">https://github.com/AlexsLemonade/OpenPBTA-analysis</a>.
Manuscript code can be found at <a href="https://github.com/AlexsLemonade/OpenPBTA-manuscript">https://github.com/AlexsLemonade/OpenPBTA-manuscript</a>.
Associated DOIs are listed in the <strong>Key Resources Table</strong>.
Software versions are documented in <strong>Table S5</strong> as an appendix to the <strong>Key Resources Table</strong>.
<!-- TODO: create code dois --></p>
<p>Any additional information required to reanalyze the data reported in this paper is available
from the lead contact upon request.</p>
<h3 id="method-details">METHOD DETAILS</h3>
<h4 id="biospecimen-collection">Biospecimen Collection</h4>
<p>The Pediatric Brain Tumor Atlas specimens are comprised of samples from Children’s Brain Tumor Network (CBTN) and the Pediatric Pacific Neuro-Oncology Consortium (PNOC).
The <a href="https://CBTN.org">CBTN</a> is a collaborative, multi-institutional (32 institutions worldwide) research program dedicated to the study of childhood brain tumors.
<a href="https://www.pnoc.us/">PNOC</a> is an international consortium dedicated to bringing new therapies to children and young adults with brain tumors.
We also include blood and tumor biospecimens from newly-diagnosed diffuse intrinsic pontine glioma (DIPG) patients as part of the PNOC003 clinical trial <a href="https://clinicaltrials.gov/ct2/show/NCT02274987">PNOC003/NCT02274987</a><span class="citation" data-cites="BLNV1DbL"><sup><a href="#ref-BLNV1DbL" role="doc-biblioref">17</a></sup></span>.</p>
<p>The CBTN-generated cell lines were derived from either fresh tumor tissue directly obtained from surgery performed at Children’s Hospital of Philadelphia (CHOP) or from prospectively collected tumor specimens stored in Recover Cell Culture Freezing medium (cat# 12648010, Gibco).
We dissociated tumor tissue using enzymatic method with papain as described<span class="citation" data-cites="DwCzMoBJ"><sup><a href="#ref-DwCzMoBJ" role="doc-biblioref">16</a></sup></span>.
Briefly, we washed tissue with HBSS (cat# 14175095, Gibco), and we minced and incubated the tissue with activated papain solution (cat# LS003124, SciQuest) for up to 45 minutes.
We used ovomucoid solution (cat# 542000, SciQuest) to inactivate the papain, briefly treated tissue with DNase (cat# 10104159001, Roche), passed it through the 100μm cell strainer (cat# 542000, Greiner Bio-One).
We initiated two cell culture conditions based on the number of cells available.
For cultures utilizing the fetal bovine serum (FBS), we plated a minimum density of 3×10<sup>5</sup> cells/mL in DMEM/F-12 medium (cat# D8062, Sigma) supplemented with 20% FBS (cat# SH30910.03, Hyclone), 1% GlutaMAX (cat# 35050061, Gibco), Penicillin/Streptomycin-Amphotericin B Mixture (cat# 17-745E, Lonza), and 0.2% Normocin (cat# ant-nr-2, Invivogen).
For serum-free media conditions, we plated cells at minimum density of 1×10<sup>6</sup> cells/mL in DMEM/F12 medium supplemented with 1% GlutaMAX, 1X B-27 supplement minus vitamin A (cat# 12587-010, Gibco), 1x N-2 supplement (cat# 17502001, Gibco), 20 ng/ml epidermal growth factor (cat# PHG0311L, Gibco), 20 ng/mL basic fibroblast growth factor (cat# 100-18B, PeproTech), 2.5μg/mL heparin (cat# H3149, Sigma), Penicillin/Streptomycin-Amphotericin B Mixture, and 0.2% Normocin.</p>
<h4 id="nucleic-acids-extraction-and-library-preparation">Nucleic acids extraction and library preparation</h4>
<h5 id="pnoc-samples">PNOC samples</h5>
<p>The Translational Genomic Research Institute (TGEN; Phoenix, AZ) performed DNA and RNA extractions on tumor biopsies using a DNA/RNA AllPrep Kit (Qiagen, #80204).
All RNA used for library prep had a minimum RIN of seven, but no QC thresholds were implemented for the DNA.
For library preparation, 500 ng of nucleic acids were used as input for RNA-Seq, WXS, and targeted DNA panel (panel) sequencing.
RNA library preparation was performed using the TruSeq RNA Sample Prep Kit (Illumina, #FC-122-1001) and the exome prep was performed using KAPA Library Preparation Kit (Roche, #KK8201) using Agilent’s SureSelect Human All Exon V5 backbone with custom probes.
The targeted DNA panel developed by Ashion Analytics (formerly known as the GEM Cancer panel) consisted of exonic probes against 541 cancer genes.
Both panel and WXS assays contained 44,000 probes across evenly spaced genomic loci used for genome-wide copy number analysis.
For the panel, additional probes tiled across intronic regions of 22 known tumor suppressor genes and 22 genes involved in common cancer translocations for structural analysis.
All extractions and library preparations were performed according to manufacturer’s instructions.</p>
<h5 id="cbtn-samples">CBTN samples</h5>
<p>Blood, tissue, and cell line DNA/RNA extractions were performed at the Biorepository Core at CHOP.
Briefly, 10-20 mg frozen tissue, 0.4-1ml of blood, or 2e6 cells pellet was used for extractions.
Tissues were lysed using a Qiagen TissueLyser II (Qiagen) with 2×30 sec at 18Hz settings using 5 mm steel beads (cat# 69989, Qiagen).
Both tissue and cell pellets processes included a CHCl<sub>3<sub> extraction and were run on the QIACube automated platform (Qiagen) using the AllPrep DNA/RNA/miRNA Universal kit (cat# 80224, Qiagen).
Blood was thawed and treated with RNase A (cat#, 19101, Qiagen); 0.4-1ml was processed using the Qiagen QIAsymphony automated platform (Qiagen) using the QIAsymphony DSP DNA Midi Kit (cat# 937255, Qiagen).
DNA and RNA quantity and quality was assessed by PerkinElmer DropletQuant UV-VIS spectrophotometer (PerkinElmer) and an Agilent 4200 TapeStation (Agilent, USA) for RIN and DIN (RNA Integrity Number and DNA Integrity Number, respectively).
The NantHealth Sequencing Center, BGI at CHOP, or the Genomic Clinical Core at Sidra Medical and Research Center performed library preparation and sequencing.
BGI at CHOP and Sidra Medical and Research Center used in house, center-specific workflows for sample preparation.
At NantHealth Sequencing Center, DNA sequencing libraries were prepared for tumor and matched-normal DNA using the KAPA HyperPrep kit (cat# 08098107702, Roche), and tumor RNA-Seq libraries were prepared using KAPA Stranded RNA-Seq with RiboErase kit (cat# 07962304001, Roche).</p>
<h4 id="data-generation">Data generation</h4>
<p>NantHealth and Sidra performed 2x150 bp WGS on paired tumor (~60X) and constitutive DNA (~30X) samples on an Illumina X/400.
BGI at CHOP performed 2x100 bp WGS sequenced at 60X depth for both tumor and normal samples.
NantHealth performed ribosomal-depleted whole transcriptome stranded RNA-Seq to an average depth of 200M.
BGI at CHOP performed poly-A or ribosomal-depleted whole transcriptome stranded RNA-Seq to an average depth of 100M.
The Translational Genomic Research Institute (TGEN; Phoenix, AZ) performed paired tumor (~200X) and constitutive whole exome sequencing (WXS) or targeted DNA panel (panel) and poly-A selected RNA-Seq (~200M reads) for PNOC tumor samples.
The panel tumor sample was sequenced to 470X, and the normal panel sample was sequenced to 308X.
PNOC 2x100 bp WXS and RNA-Seq libraries were sequenced on an Illumina HiSeq 2500.</p>
<h4 id="dna-wgs-alignment">DNA WGS Alignment</h4>
<p>We used <code>BWA-MEM</code><span class="citation" data-cites="Wql2Djro"><sup><a href="#ref-Wql2Djro" role="doc-biblioref">76</a></sup></span> to align paired-end DNA-seq reads to the version 38 patch release 12 of the <em>Homo sapiens</em> genome reference, obtained as a FASTA file from UCSC (see <strong>Key Resources Table</strong>).
Next, we used the Broad Institute’s Best Practices<span class="citation" data-cites="NCr4QkOg"><sup><a href="#ref-NCr4QkOg" role="doc-biblioref">77</a></sup></span> to process Binary Alignment/Map files (BAMs) in preparation for variant discovery.
We marked duplicates using <code>SAMBLASTER</code><span class="citation" data-cites="IzUbOv92"><sup><a href="#ref-IzUbOv92" role="doc-biblioref">78</a></sup></span>, and we merged and sorted BAMs using <code>Sambamba</code><span class="citation" data-cites="PcJwlZj3"><sup><a href="#ref-PcJwlZj3" role="doc-biblioref">79</a></sup></span>
We used the <code>BaseRecalibrator</code> submodule of the Broad’s Genome Analysis Tool Kit <code>GATK</code><span class="citation" data-cites="14nSa8zB9"><sup><a href="#ref-14nSa8zB9" role="doc-biblioref">80</a></sup></span> to process BAM files.
Lastly, for normal/germline input, we used the <code>GATK</code> <code>HaplotypeCaller</code><span class="citation" data-cites="36OiSGiq"><sup><a href="#ref-36OiSGiq" role="doc-biblioref">81</a></sup></span> submodule on the recalibrated BAM to generate a genomic variant call format (GVCF) file.
This file is used as the basis for germline calling, described in the <strong>SNV calling for B-allele Frequency (BAF) generation</strong> section.</p>
<p>We obtained references from the <a href="https://s3.console.aws.amazon.com/s3/buckets/broad-references/hg38/v0/">Broad Genome References on AWS</a> bucket with a general description of references at <a href="https://s3.amazonaws.com/broad-references/broad-references-readme.html" class="uri">https://s3.amazonaws.com/broad-references/broad-references-readme.html</a>.</p>
<h4 id="quality-control-of-sequencing-data">Quality Control of Sequencing Data</h4>
<p>To confirm sample matches and remove mis-matched samples from the dataset, we performed <code>NGSCheckMate</code><span class="citation" data-cites="A4FXW005"><sup><a href="#ref-A4FXW005" role="doc-biblioref">82</a></sup></span> on matched tumor/normal CRAM files.
Briefly, we processed CRAMs using <code>BCFtools</code> to filter and call 20k common single nucleotide polymorphisms (SNPs) using default parameters.
We used the resulting VCFs to run <code>NGSCheckMate</code>.
Per <code>NGSCheckMate</code> author recommendations, we used &lt;= 0.61 as a correlation coefficient cutoff at sequencing depths &gt; 10 to predict mis-matched samples.
We determined RNA-Seq read strandedness by running the <code>infer_experiment.py</code> script from <code>RNA-SeQC</code><span class="citation" data-cites="unXWCD9A"><sup><a href="#ref-unXWCD9A" role="doc-biblioref">83</a></sup></span> on the first 200k mapped reads.
We removed any samples whose calculated strandedness did not match strandedness information provided by the sequencing center.
We required that at least 60% of RNA-Seq reads mapped to the human reference for samples to be included in analysis.</p>
<h4 id="germline-variant-calling">Germline Variant Calling</h4>
<h5 id="snp-calling-for-b-allele-frequency-baf-generation">SNP calling for B-allele Frequency (BAF) generation</h5>
<p>We performed germline haplotype calls using the <code>GATK</code> Joint Genotyping Workflow on individual GVCFs from the normal sample alignment workflow.
Using only SNPs, we applied the <code>GATK</code> generic hard filter suggestions to the VCF, with an additional requirement of 10 reads minimum depth per SNP.
We used the filtered VCF as input to <code>Control-FREEC</code> and <code>CNVkit</code> (below) to generate B-allele frequency (BAF) files.
This single-sample workflow is available in the <a href="https://github.com/d3b-center/OpenPBTA-workflows">D3b GitHub repository</a>.
References can be obtained from the <a href="https://s3.console.aws.amazon.com/s3/buckets/broad-references/hg38/v0/">Broad Genome References on AWS</a> bucket, and a general description of references can be found at https://s3.amazonaws.com/broad-references/broad-references-readme.html.</p>
<h5 id="assessment-of-germline-variant-pathogenicity">Assessment of germline variant pathogenicity</h5>
<p>For patients with hypermutant samples, we first added population frequency of germline variants using <code>ANNOVAR</code><span class="citation" data-cites="Fv382g9q"><sup><a href="#ref-Fv382g9q" role="doc-biblioref">84</a></sup></span> and pathogenicity scoring from ClinVar<span class="citation" data-cites="16jeSVhEx"><sup><a href="#ref-16jeSVhEx" role="doc-biblioref">85</a></sup></span> using <code>SnpSift</code><span class="citation" data-cites="vOcCb1WG"><sup><a href="#ref-vOcCb1WG" role="doc-biblioref">86</a></sup></span>.
We then filtered for variants with read depth &gt;= 15, variant allele fraction &gt;= 0.20, and which were observed at &lt; 0.1% allele frequency across each population in the Genome Aggregation Database (see <strong>Key Resources Table</strong>).
Finally, we retained variants in genes included in the KEGG MMR gene set (see <strong>Key Resources Table</strong>), <em>POLE</em>, and/or <em>TP53</em> which were ClinVar-annotated as pathogenic (P) or likely pathogenic (LP) with review status of &gt;= 2 stars.
All P/LP variants were manually reviewed by an interdisciplinary team of scientists, clinicians, and genetic counselors.
This workflow is available in the <a href="https://github.com/d3b-center/OpenPBTA-workflows">D3b GitHub repository</a>.</p>
<h4 id="somatic-mutation-calling">Somatic Mutation Calling</h4>
<h5 id="snv-and-indel-calling">SNV and indel calling</h5>
<p>We used four variant callers to call SNVs and indels from paired tumor/normal samples with Targeted Panel, WXS, and/or WGS data: <code>Strelka2</code><span class="citation" data-cites="REfkDUtE"><sup><a href="#ref-REfkDUtE" role="doc-biblioref">87</a></sup></span>, <code>Mutect2</code><span class="citation" data-cites="9ZgwE4bQ"><sup><a href="#ref-9ZgwE4bQ" role="doc-biblioref">88</a></sup></span>, <code>Lancet</code><span class="citation" data-cites="V6KdWVYi"><sup><a href="#ref-V6KdWVYi" role="doc-biblioref">89</a></sup></span>, and <code>VarDictJava</code><span class="citation" data-cites="trQRR8fs"><sup><a href="#ref-trQRR8fs" role="doc-biblioref">90</a></sup></span>.
<code>VarDictJava</code>-only calls were not retained since ~ 39M calls with low VAF were uniquely called and may be potential false positives. (~1.2M calls were called by <code>Mutect2</code>, <code>Strelka2</code>, and <code>Lancet</code> and included consensus CNV calling as described below.)
We used only <code>Strelka2</code>, <code>Mutect2</code> and <code>Lancet</code> to analyze WXS samples from TCGA.
TCGA samples were captured using various WXS target capture kits and we downloaded the BED files from the <a href="https://api.gdc.cancer.gov/files"><code>GDC portal</code></a>.
The manufacturers provided the input interval BED files for both panel and WXS data for PBTA samples.
We padded all panel and WXS BED files were by 100 bp on each side for <code>Strelka2</code>, <code>Mutect2</code>, and <code>VarDictJava</code> runs and by 400 bp for the <code>Lancet</code> run.
For WGS calling, we utilized the non-padded BROAD Institute interval calling list <a href="https://console.cloud.google.com/storage/browser/_details/genomics-public-data/resources/broad/hg38/v0/wgs_calling_regions.hg38.interval_list"><code>wgs_calling_regions.hg38.interval_list</code></a>, comprised of the full genome minus N bases, unless otherwise noted below.
We ran <code>Strelka2</code><span class="citation" data-cites="REfkDUtE"><sup><a href="#ref-REfkDUtE" role="doc-biblioref">87</a></sup></span> using default parameters for canonical chromosomes (chr1-22, X,Y,M), as recommended by the authors, and we filtered the final <code>Strelka2</code> VCF for PASS variants.
We ran <code>Mutect2</code> from <code>GATK</code> according to Broad best practices outlined from their Workflow Description Language (WDL), and we filtered the final <code>Mutect2</code> VCF for PASS variants.
To manage memory issues, we ran <code>VarDictJava</code><span class="citation" data-cites="trQRR8fs"><sup><a href="#ref-trQRR8fs" role="doc-biblioref">90</a></sup></span> using 20 Kb interval chunks of the input BED, padded by 100 bp on each side, such that if an indel occurred in between intervals, it would be captured.
Parameters and filtering followed <a href="https://github.com/bcbio/bcbio-nextgen">BCBIO standards</a> except that variants with a variant allele frequency (VAF) &gt;= 0.05 (instead of &gt;= 0.10) were retained.
The 0.05 VAF increased the true positive rate for indels and decreased the false positive rate for SNVs when using <code>VarDictJava</code> in consensus calling.
We filtered the final <code>VarDictJava</code> VCF for PASS variants with <code>TYPE=StronglySomatic</code>.
We ran <code>Lancet</code> using default parameters, except for those noted below.
For input intervals to <code>Lancet</code> WGS, we created a reference BED from only the UTR, exome, and start/stop codon features of the GENCODE 31 reference, augmented as recommended with PASS variant calls from <code>Strelka2</code> and <code>Mutect2</code>.
We then padded these intervals by 300 bp on each side during <code>Lancet</code> variant calling.
Per recommendations for WGS samples, we augmented the Lancet input intervals described above with PASS variant calls from <code>Strelka2</code> and <code>Mutect2</code> as validation<span class="citation" data-cites="gG4xEIkq"><sup><a href="#ref-gG4xEIkq" role="doc-biblioref">91</a></sup></span>.</p>
<h5 id="vcf-annotation-and-maf-creation">VCF annotation and MAF creation</h5>
<p>We normalized INDELs with <code>bcftools norm</code> on all PASS VCFs using the <code>kfdrc_annot_vcf_sub_wf.cwl</code> subworkflow, release v3 (See <strong>Table S5</strong>).
The Ensembl Variant Effect Predictor (<code>VEP</code>)<span class="citation" data-cites="p1f5DxRQ"><sup><a href="#ref-p1f5DxRQ" role="doc-biblioref">92</a></sup></span>, reference release 93, was used to annotate variants and bcftools was used to add population allele frequency (AF) from gnomAD<span class="citation" data-cites="55r9iVwk"><sup><a href="#ref-55r9iVwk" role="doc-biblioref">93</a></sup></span>.
We annotated SNV and INDEL hotspots from v2 of Memorial Sloan Kettering Cancer Center’s (MSKCC) database (See <strong>Key Resources Table</strong>) as well as the <em>TERT</em> promoter mutations C228T and C250T<span class="citation" data-cites="59DKOl44"><sup><a href="#ref-59DKOl44" role="doc-biblioref">94</a></sup></span>.
We annotated SNVs by matching amino acid position (<code>Protein_position</code> column in MAF file) with SNVs in the MSKCC database, we matched splice sites to <code>HGVSp_Short</code> values in the MSKCC database, and we matched INDELs based on amino acid present within the range of INDEL hotspots values in the MSKCC database.
We removed non-hotspot annotated variants with a normal depth less than or equal to 7 and/or gnomAD allele frequency (AF) greater than 0.001 as potential germline variants.
We matched <em>TERT</em> promoter mutations using hg38 coordinates as indicated in ref.<span class="citation" data-cites="59DKOl44"><sup><a href="#ref-59DKOl44" role="doc-biblioref">94</a></sup></span>: C228T occurs at 5:1295113 is annotated as existing variant <code>s1242535815</code>, <code>COSM1716563</code>, or <code>COSM1716558</code>, and is 66 bp away from the TSS; C250T occurs at Chr5:1295135, is annotated as existing variant <code>COSM1716559</code>, and is 88 bp away from the TSS.
We retained variants annotated as <code>PASS</code> or <code>HotSpotAllele=1</code> in the final set, and we created MAFs using MSKCC’s <code>vcf2maf</code> tool.</p>
<h5 id="gather-snv-and-indel-hotspots">Gather SNV and INDEL Hotspots</h5>
<p>We retained all variant calls from <code>Strelka2</code>, <code>Mutect2</code>, or <code>Lancet</code> that overlapped with an SNV or INDEL hotspot in a hotspot-specific MAF file, which we then used for select analyses as described below.</p>
<h5 id="consensus-snv-calling">Consensus SNV Calling</h5>
<p>Our SNV calling process led to separate sets of predicted mutations for each caller.
We considered mutations to describe the same change if they were identical for the following MAF fields: <code>Chromosome</code>, <code>Start_Position</code>, <code>Reference_Allele</code>, <code>Allele</code>, and <code>Tumor_Sample_Barcode</code>.
<code>Strelka2</code> does not call multinucleotide variants (MNV), but instead calls each component SNV as a separate mutation, so we separated MNV calls from <code>Mutect2</code> and <code>Lancet</code> into consecutive SNVs before comparing them to <code>Strelka2</code> calls.
We examined VAFs produced by each caller and compared their overlap with each other (<strong>Figure S2</strong>).
<code>VarDictJava</code> calls included many variants that were not identified by other callers (<strong>Figure S2C</strong>), while the other callers produced results that were relatively consistent with one another.
Many of these <code>VarDictJava</code>-specific calls were variants with low allele frequency (<strong>Figure S2B</strong>).
We therefore derived consensus mutation calls as those shared among the other three callers (<code>Strelka2</code>, <code>Mutect2</code>, and <code>Lancet</code>), and we did not further consider <code>VarDictJava</code> calls due to concerns it called a large number of false positives.
This decision had minimal impact on results because <code>VarDictJava</code> also identified nearly every mutation that the other three callers identified, in addition to many unique mutations.</p>
<h4 id="somatic-copy-number-variant-calling-wgs-samples-only">Somatic Copy Number Variant Calling (WGS samples only)</h4>
<p>We used <code>Control-FREEC</code><span class="citation" data-cites="ZQ0L3o1q 1F3i4BvCt"><sup><a href="#ref-ZQ0L3o1q" role="doc-biblioref">95</a>,<a href="#ref-1F3i4BvCt" role="doc-biblioref">96</a></sup></span> and <code>CNVkit</code><span class="citation" data-cites="UTxRcYIQ"><sup><a href="#ref-UTxRcYIQ" role="doc-biblioref">97</a></sup></span> for copy number variant calls.
For both algorithms, the <code>germline_sex_estimate</code> (described below) was used as input for sample sex and germline variant calls (above) were used as input for BAF estimation.
<code>Control-FREEC</code> was run on human genome reference hg38 using the optional parameters of a 0.05 coefficient of variation, ploidy choice of 2-4, and BAF adjustment for tumor-normal pairs.
<code>Theta2</code><span class="citation" data-cites="V4PckbrH"><sup><a href="#ref-V4PckbrH" role="doc-biblioref">98</a></sup></span> used <code>VarDictJava</code> germline and somatic calls, filtered on PASS and strongly somatic, to infer tumor purity.
<code>Theta2</code> purity was added as an optional parameter to <code>CNVkit</code> to adjust copy number calls.
<code>CNVkit</code> was run on human genome reference hg38 using the optional parameters of Theta2 purity and BAF adjustment for tumor-normal pairs.
We used <code>GISTIC</code><span class="citation" data-cites="rDV1hhfF"><sup><a href="#ref-rDV1hhfF" role="doc-biblioref">99</a></sup></span> on the <code>CNVkit</code> and the consensus CNV segmentation files to generate gene-level copy number abundance (Log R Ratio) as well as chromosomal arm copy number alterations using the parameters specified in the (<code>run-gistic</code> analysis module in the OpenPBTA Analysis repository).</p>
<h5 id="consensus-cnv-calling">Consensus CNV Calling</h5>
<p>For each caller and sample, we called CNVs based on consensus among <code>Control-FREEC</code><span class="citation" data-cites="ZQ0L3o1q 1F3i4BvCt"><sup><a href="#ref-ZQ0L3o1q" role="doc-biblioref">95</a>,<a href="#ref-1F3i4BvCt" role="doc-biblioref">96</a></sup></span>, <code>CNVkit</code><span class="citation" data-cites="UTxRcYIQ"><sup><a href="#ref-UTxRcYIQ" role="doc-biblioref">97</a></sup></span>, and <code>Manta</code><span class="citation" data-cites="kTn1PIj5"><sup><a href="#ref-kTn1PIj5" role="doc-biblioref">100</a></sup></span>.
We specifically included CNVs called significant by <code>Control-FREEC</code> (p-value &lt; 0.01) and <code>Manta</code> calls that passed all filters in consensus calling.
We removed sample and consensus caller files with more than 2,500 CNVs because we expected these to be noisy and derive poor quality samples based on cutoffs used in <code>GISTIC</code><span class="citation" data-cites="rDV1hhfF"><sup><a href="#ref-rDV1hhfF" role="doc-biblioref">99</a></sup></span>.
For each sample, we included the regions in the final consensus set: 1) regions with reciprocal overlap of 50% or more between at least two of the callers; 2) smaller CNV regions in which more than 90% of regions are covered by another caller.
We did not include any copy number alteration called by a single algorithm in the consensus file.
We defined copy number as <code>NA</code> for any regions that had a neutral call for the samples included in the consensus file.
We merged CNV regions within 10,000 bp of each other with the same direction of gain or loss into single region.
We filtered out any CNVs that overlapped 50% or more with immunoglobulin, telomeric, centromeric, segment duplicated regions, or that were shorter than 3000 bp.</p>
<h4 id="somatic-structural-variant-calling-wgs-samples-only">Somatic Structural Variant Calling (WGS samples only)</h4>
<p>We used <code>Manta</code><span class="citation" data-cites="kTn1PIj5"><sup><a href="#ref-kTn1PIj5" role="doc-biblioref">100</a></sup></span> for structural variant (SV) calls, and we limited to regions used in <code>Strelka2</code>.
The hg38 reference for SV calling used was limited to canonical chromosome regions.
We used <code>AnnotSV</code><span class="citation" data-cites="7Ull5aU5"><sup><a href="#ref-7Ull5aU5" role="doc-biblioref">101</a></sup></span> to annotate <code>Manta</code> output.
All associated workflows are available in the <a href="https://github.com/d3b-center/OpenPBTA-workflows/">workflows GitHub repository</a>.</p>
<h4 id="gene-expression">Gene Expression</h4>
<h5 id="abundance-estimation">Abundance Estimation</h5>
<p>We used <code>STAR</code><span class="citation" data-cites="tTu8Ds9Z"><sup><a href="#ref-tTu8Ds9Z" role="doc-biblioref">102</a></sup></span> to align paired-end RNA-seq reads, and we used the associated alignment for all subsequent RNA analysis.
We used Ensembl GENCODE 27 “Comprehensive gene annotation” (see <strong>Key Resources Table</strong>) as a reference.
We used <code>RSEM</code><span class="citation" data-cites="Dh2n1EV3"><sup><a href="#ref-Dh2n1EV3" role="doc-biblioref">103</a></sup></span> for both FPKM and TPM transcript- and gene-level quantification.</p>
<h5 id="gene-expression-matrices-with-unique-hugo-symbols">Gene Expression Matrices with Unique HUGO Symbols</h5>
<p>To enable downstream analyses, we next identified gene symbols that map to multiple Ensembl gene identifiers (in GENCODE v27, 212 gene symbols map to 1866 Ensembl gene identifiers), known as multi-mapped gene symbols, and ensured unique mappings (<code>collapse-rnaseq</code> analysis module in the OpenPBTA Analysis repository).
To this end, we first removed genes with no expression from the <code>RSEM</code> abundance data by requiring an FPKM &gt; 0 in at least 1 sample across the PBTA cohort.
We computed the mean FPKM across all samples per gene.
For each multi-mapped gene symbol, we chose the Ensembl identifier corresponding to the maximum mean FPKM, using the assumption that the gene identifier with the highest expression best represented the expression of the gene.
After collapsing gene identifiers, 46,400 uniquely-expressed genes remained in the poly-A dataset, and 53,011 uniquely-expressed genes remained in the stranded dataset.</p>
<h5 id="gene-fusion-detection">Gene fusion detection</h5>
<p>We set up <code>Arriba</code><span class="citation" data-cites="DuQpY5dg"><sup><a href="#ref-DuQpY5dg" role="doc-biblioref">104</a></sup></span> and <code>STAR-Fusion</code><span class="citation" data-cites="dwCkClIW"><sup><a href="#ref-dwCkClIW" role="doc-biblioref">105</a></sup></span> fusion detection tools using CWL on CAVATICA.
For both of these tools, we used aligned BAM and chimeric SAM files from <code>STAR</code> as inputs and <code>GRCh38_gencode_v27</code> GTF for gene annotation.
We ran <code>STAR-Fusion</code> with default parameters and annotated all fusion calls with the <code>GRCh38_v27_CTAT_lib_Feb092018.plug-n-play.tar.gz</code> file from the <code>STAR-Fusion</code> release.
For <code>Arriba</code>, we used a blacklist file <code>blacklist_hg38_GRCh38_2018-11-04.tsv.gz</code> from the <code>Arriba</code> release to remove recurrent fusion artifacts and transcripts present in healthy tissue.
We provided <code>Arriba</code> with strandedness information for stranded samples, or we set it to auto-detection for poly-A samples.
We used <code>FusionAnnotator</code> on <code>Arriba</code> fusion calls to harmonize annotations with those of <code>STAR-Fusion</code>.
The RNA expression and fusion workflows can be found in the <a href="https://github.com/d3b-center/OpenPBTA-workflows/blob/master/cwl/kfdrc_RNAseq_workflow.cwl">D3b GitHub repository</a>.
The <code>FusionAnnotator</code> workflow we used for this analysis can be found in the <a href="https://github.com/d3b-center/FusionAnnotator">D3b GitHub repository</a>.</p>
<h3 id="quantification-and-statistical-analysis">QUANTIFICATION AND STATISTICAL ANALYSIS</h3>
<p>All p-values are two-sided unless otherwise stated.
Z-scores were calculated using the formula <span class="math inline">\(z=(x –\mu)/\sigma\)</span> where <span class="math inline">\(x\)</span> is the value of interest, <span class="math inline">\(\mu\)</span> is the mean, and <span class="math inline">\(\sigma\)</span> is the standard deviation.</p>
<h4 id="tumor-purity-tumor-purity-exploration-module">Tumor purity (<code>tumor-purity-exploration</code> module)</h4>
<p>We used Theta2 (as described in the “Somatic Copy Number Variant Calling section” Methods section) to infer tumor purity of WGS samples and explored purity values across histologies.
We assumed that co-extracted RNA and DNA samples had the same tumor purity.</p>
<h5 id="recurrently-mutated-genes-and-co-occurrence-of-gene-mutations-interaction-plots-analysis-module">Recurrently mutated genes and co-occurrence of gene mutations (<code>interaction-plots</code> analysis module)</h5>
<p>Using the consensus SNV calls, we identified genes that were recurrently mutated in the OpenPBTA cohort, including nonsynonymous mutations with a VAF &gt; 5% among the set of independent samples.
We used <code>VEP</code><span class="citation" data-cites="p1f5DxRQ"><sup><a href="#ref-p1f5DxRQ" role="doc-biblioref">92</a></sup></span> annotations, including “High” and “Moderate” consequence types as defined in the R package <code>Maftools</code><span class="citation" data-cites="83Prx3Xc"><sup><a href="#ref-83Prx3Xc" role="doc-biblioref">106</a></sup></span>, to determine the set of nonsynonymous mutations.
For each gene, we then tallied the number of samples that had at least one nonsynonymous mutation.</p>
<p>For genes that contained nonsynonymous mutations in multiple samples, we calculated pairwise mutation co-occurrence scores.
This score was defined as <span class="math inline">\(I(-\log_{10}(P))\)</span> where <span class="math inline">\(I\)</span> is 1 when the odds ratio is &gt; 1 (indicating co-occurrence), and -1 when the odds ratio is &lt; 1 (indicating mutual exclusivity), with <span class="math inline">\(P\)</span> defined by Fisher’s Exact Test.</p>
<h5 id="focal-copy-number-calling-focal-cn-file-preparation-analysis-module">Focal Copy Number Calling (<code>focal-cn-file-preparation</code> analysis module)</h5>
<p>We added the ploidy inferred via <code>Control-FREEC</code> to the consensus CNV segmentation file and used the ploidy and copy number values to define gain and loss values broadly at the chromosome level.
We used <code>bedtools coverage</code><span class="citation" data-cites="1HWiAHnIw"><sup><a href="#ref-1HWiAHnIw" role="doc-biblioref">107</a></sup></span> to add cytoband status using the UCSC cytoband file<span class="citation" data-cites="1DWggwY7R"><sup><a href="#ref-1DWggwY7R" role="doc-biblioref">108</a></sup></span> (See <strong>Key Resources Table</strong>).
The output status call fractions, which are values of the loss, gain, and callable fractions of each cytoband region, were used to define dominant status at the cytoband-level.
We calculated the weighted means of each status call fraction using band length.
We used the weighted means to define the dominant status at the chromosome arm-level.</p>
<p>A status was considered dominant if more than half of the region was callable and the status call fraction was greater than 0.9 for that region.
We adopted this 0.9 threshold to ensure that the dominant status fraction call was greater than the remaining status fraction calls in a region.</p>
<p>We aimed to define focal copy number units to avoid calling adjacent genes in the same cytoband or arm as copy number losses or gains where it would be more appropriate to call the broader region a loss or gain.
To determine the most focal units, we first considered the dominant status calls at the chromosome arm-level.
If the chromosome arm dominant status was callable but not clearly defined as a gain or loss, we instead included the cytoband-level status call.
Similarly, if a cytoband dominant status call was callable but not clearly defined as a gain or loss, we instead included gene-level status call.
To obtain the gene-level data, we used the <code>IRanges</code> package in R<span class="citation" data-cites="14d6Jzwus"><sup><a href="#ref-14d6Jzwus" role="doc-biblioref">109</a></sup></span> to find overlaps between the segments in the consensus CNV file and the exons in the GENCODE v27 annotation file (See <strong>Key Resources Table</strong>) .
If the copy number value was 0, we set the status to “deep deletion”.
For autosomes only, we set the status to “amplification” when the copy number value was greater than two times the ploidy value.
We plotted genome-wide gains and losses in (<strong>Figure <a href="#fig:S3">S3</a>C</strong>) using the R package <code>ComplexHeatmap</code><span class="citation" data-cites="oESNuFI5"><sup><a href="#ref-oESNuFI5" role="doc-biblioref">110</a></sup></span>.</p>
<h4 id="breakpoint-density-wgs-samples-only-chromosomal-instability-analysis-module">Breakpoint Density (WGS samples only; <code>chromosomal-instability</code> analysis module)</h4>
<p>We defined breakpoint density as the number of breaks per genome or exome per sample.
For Manta SV calls, we filtered to retain “PASS” variants and used breakpoints from the algorithm.
For consensus CNV calls, if |log<sub>2</sub> ratio| &gt; log<sub>2</sub>(1), we annotated the segment as a break.
We then calculated breakpoint density as:</p>
<p><span class="math display">\[\textrm{breakpoint density} = \frac{\textrm{N breaks}}{\textrm{Size in Mb of }\textit{effectively surveyed} \textrm{ genome}}\]</span></p>
<h4 id="chromothripsis-analysis-wgs-samples-only-chromothripsis-analysis-module">Chromothripsis Analysis (WGS samples only; <code>chromothripsis</code> analysis module)</h4>
<p>Considering only chromosomes 1-22 and X, we identified candidate chromothripsis regions in the set of independent tumor WGS samples with ShatterSeek<span class="citation" data-cites="17wG7AMwM"><sup><a href="#ref-17wG7AMwM" role="doc-biblioref">111</a></sup></span>, using Manta SV calls that passed all filters and consensus CNV calls.
We modified the consensus CNV data to fit <code>ShatterSeek</code> input requirements as follows: we set CNV-neutral or excluded regions as the respective sample’s ploidy value from <code>Control-FREEC</code>, and we then merged consecutive segments with the same copy number value.
We classified candidate chromothripsis regions as high- or low-confidence using the statistical criteria described by the <code>ShatterSeek</code> authors.</p>
<h5 id="immune-profiling-and-deconvolution-immune-deconv-analysis-module">Immune Profiling and Deconvolution (<code>immune-deconv</code> analysis module)</h5>
<p>We used the R package <code>immunedeconv</code><span class="citation" data-cites="pmid:31510660"><sup><a href="#ref-pmid:31510660" role="doc-biblioref"><strong>pmid:31510660?</strong></a></sup></span> with the method <code>quanTIseq</code><span class="citation" data-cites="RLEOBWiR"><sup><a href="#ref-RLEOBWiR" role="doc-biblioref">112</a></sup></span> to deconvolute various immune cell types in tumors using collapsed FPKM RNA-seq, with samples batched by library type and then combined.
The <code>quanTIseq</code> deconvolution method directly estimates absolute fractions of 10 immune cell types that represent inferred proportions of the cell types in the mixture.
Therefore, we utilized <code>quanTIseq</code> for inter-sample, intra-sample, and inter-histology score comparisons.</p>
<h5 id="gene-set-variation-analysis-gene-set-enrichment-analysis-analysis-module">Gene Set Variation Analysis (<code>gene-set-enrichment-analysis</code> analysis module)</h5>
<p>We performed Gene Set Variation Analysis (GSVA) on collapsed, log2-transformed RSEM FPKM data for stranded RNA-Seq samples using the <code>GSVA</code> Bioconductor package<span class="citation" data-cites="1F5zJEMaD"><sup><a href="#ref-1F5zJEMaD" role="doc-biblioref">113</a></sup></span>.
We specified the parameter <code>mx.diff=TRUE</code> to obtain Gaussian-distributed scores for each of the MSigDB hallmark gene sets<span class="citation" data-cites="1CbVoEpNJ"><sup><a href="#ref-1CbVoEpNJ" role="doc-biblioref">114</a></sup></span>.
We compared GSVA scores among histology groups using ANOVA and subsequent Tukey tests; p-values were Bonferroni-corrected for multiple hypothesis testing.
We plotted scores by cancer group using the <code>ComplexHeatmap</code> R package (<strong>Figure <a href="#fig:Fig5">5</a>B</strong>)<span class="citation" data-cites="oESNuFI5"><sup><a href="#ref-oESNuFI5" role="doc-biblioref">110</a></sup></span>.</p>
<h5 id="transcriptomic-dimension-reduction-transcriptomic-dimension-reduction-analysis-module">Transcriptomic Dimension Reduction (<code>transcriptomic-dimension-reduction</code> analysis module)</h5>
<p>We applied Uniform Manifold Approximation and Projection (UMAP)<span class="citation" data-cites="jErrUFsf"><sup><a href="#ref-jErrUFsf" role="doc-biblioref">115</a></sup></span> to log2-transformed FPKM data for stranded RNA-Seq samples using the <code>umap</code> R package (See <strong>Key Resources Table</strong>).
We considered all stranded RNA-Seq samples for this analysis, but we removed genes whose FPKM sum across samples was less than 100.
We set the UMAP number of neighbors parameter to 15.</p>
<h5 id="fusion-prioritization-fusion_filtering-analysis-module">Fusion prioritization (<code>fusion_filtering</code> analysis module)</h5>
<p>We performed artifact filtering and additional annotation on fusion calls to prioritize putative oncogenic fusions.
Briefly, we considered all in-frame and frameshift fusion calls with at least one junction read and at least one gene partner expressed (TPM &gt; 1) to be true calls.
If a fusion call had a large number of spanning fragment reads compared to junction reads (spanning fragment minus junction read greater than ten), we removed these calls as potential false positives.
We prioritized a union of fusion calls as true calls if the fused genes were detected by both callers, the same fusion was recurrent within a broad histology grouping (&gt; 2 samples), or the fusion was specific to the given broad histology.
If either 5’ or 3’ genes fused to more than five different genes within a sample, we removed these calls as potential false positives.
We annotated putative driver fusions and prioritized fusions based on partners containing known <a href="http://kinase.com/human/kinome/tables/Kincat_Hsap.08.02.xls">kinases</a>, <a href="http://www.bushmanlab.org/assets/doc/allOnco_Feb2017.tsv">oncogenes</a>, <a href="https://bioinfo.uth.edu/TSGene/Human_TSGs.txt?csrt=5027697123997809089">tumor suppressors</a>, curated transcription factors<span class="citation" data-cites="9vS8HBL6"><sup><a href="#ref-9vS8HBL6" role="doc-biblioref">116</a></sup></span>, <a href="https://cancer.sanger.ac.uk/census">COSMIC genes</a>, and/or known <a href="https://tumorfusions.org/PanCanFusV2/downloads/pancanfus.txt.gz">TCGA fusions</a> from curated references.
Based on pediatric cancer literature review, we added <em>MYBL1</em><span class="citation" data-cites="jLWV5IWB"><sup><a href="#ref-jLWV5IWB" role="doc-biblioref">117</a></sup></span>, <em>SNCAIP</em><span class="citation" data-cites="4wYR62jK"><sup><a href="#ref-4wYR62jK" role="doc-biblioref">118</a></sup></span>, <em>FOXR2</em><span class="citation" data-cites="kfmK8vm"><sup><a href="#ref-kfmK8vm" role="doc-biblioref">119</a></sup></span>, <em>TTYH1</em><span class="citation" data-cites="5ueZBnsJ"><sup><a href="#ref-5ueZBnsJ" role="doc-biblioref">120</a></sup></span>, and <em>TERT</em><span class="citation" data-cites="ASmwGlFp YfG9EVSk lWMOs28t 1B3tdZcAl"><sup><a href="#ref-ASmwGlFp" role="doc-biblioref">121</a>–<a href="#ref-1B3tdZcAl" role="doc-biblioref">124</a></sup></span> to the oncogene list, and we added <em>BCOR</em><span class="citation" data-cites="kfmK8vm"><sup><a href="#ref-kfmK8vm" role="doc-biblioref">119</a></sup></span> and <em>QKI</em><span class="citation" data-cites="1foRpfch"><sup><a href="#ref-1foRpfch" role="doc-biblioref">125</a></sup></span> to the tumor suppressor gene list.</p>
<h4 id="oncoprint-figure-generation-oncoprint-landscape-analysis-module">Oncoprint figure generation (<code>oncoprint-landscape</code> analysis module)</h4>
<p>We used <code>Maftools</code><span class="citation" data-cites="83Prx3Xc"><sup><a href="#ref-83Prx3Xc" role="doc-biblioref">106</a></sup></span> to generate oncoprints depicting the frequencies of canonical somatic gene mutations, CNVs, and fusions for the top 20 genes mutated across primary tumors within broad histologies of the OpenPBTA dataset.
We collated canonical genes from the literature for low-grade astrocytic tumors<span class="citation" data-cites="IamVuiNU"><sup><a href="#ref-IamVuiNU" role="doc-biblioref">26</a></sup></span>, embryonal tumors<span class="citation" data-cites="UuzX8B9C 1Geb70JK9 j11VxJIm CeB34T1V 1FGkQ9Wa7"><sup><a href="#ref-1Geb70JK9" role="doc-biblioref">27</a>,<a href="#ref-j11VxJIm" role="doc-biblioref">29</a>,<a href="#ref-UuzX8B9C" role="doc-biblioref">30</a>,<a href="#ref-CeB34T1V" role="doc-biblioref">126</a>,<a href="#ref-1FGkQ9Wa7" role="doc-biblioref">127</a></sup></span>, diffuse astrocytic and oligodendroglial tumors<span class="citation" data-cites="vd5Uh9gY BLNV1DbL 6F5vK3sT KU8wVb8T"><sup><a href="#ref-BLNV1DbL" role="doc-biblioref">17</a>,<a href="#ref-vd5Uh9gY" role="doc-biblioref">32</a>,<a href="#ref-KU8wVb8T" role="doc-biblioref">33</a>,<a href="#ref-6F5vK3sT" role="doc-biblioref">128</a></sup></span>, and other tumors: ependymomas, craniopharyngiomas, neuronal-glial mixed tumors, histiocytic tumors, chordoma, meningioma, and choroid plexus tumors<span class="citation" data-cites="pmid:28623522 f7P6U50v DGVkAk1W ovoneiG3 uDYz8KKh 107ZjoLxM F73HShWn pmid:12466115 BBuQnfgj m7hgzrvh"><sup><a href="#ref-f7P6U50v" role="doc-biblioref">129</a>–<a href="#ref-m7hgzrvh" role="doc-biblioref">136</a>,<a href="#ref-pmid:28623522" role="doc-biblioref"><strong>pmid:28623522?</strong></a>,<a href="#ref-pmid:12466115" role="doc-biblioref"><strong>pmid:12466115?</strong></a></sup></span>.</p>
<h4 id="mutational-signatures-mutational-signatures-analysis-module">Mutational Signatures (<code>mutational-signatures</code> analysis module)</h4>
<p>We obtained weights (i.e., exposures) for signature sets using the <code>deconstructSigs</code> R package function <code>whichSignatures()</code><span class="citation" data-cites="KhxTOfIb"><sup><a href="#ref-KhxTOfIb" role="doc-biblioref">137</a></sup></span> from consensus SNVs with the BSgenome.Hsapiens.UCSC.hg38 annotations (see <strong>Key Resources Table</strong>).
Specifically, we estimated signature weights across samples for eight signatures previously identified in the Signal reference set of signatures (“RefSig”) as associated with adult central nervous system (CNS) tumors<span class="citation" data-cites="Cc8jt6lX"><sup><a href="#ref-Cc8jt6lX" role="doc-biblioref">41</a></sup></span>.
These eight RefSig signatures are 1, 3, 8, 11, 18, 19, N6, and MMR2.
Weights for signatures fall in the range zero to one inclusive.
<code>deconstructSigs</code> estimates the weights for each signature across samples and allows for a proportion of unassigned weights referred to as “Other” in the text.
These results do not include signatures with small contributions; <code>deconstructSigs</code> drops signature weights that are less than 6%<span class="citation" data-cites="KhxTOfIb"><sup><a href="#ref-KhxTOfIb" role="doc-biblioref">137</a></sup></span>.
We plotted mutational signatures for patients with hypermutant tumors (<strong>Figure <a href="#fig:Fig4">4</a>E</strong>) using the R package <code>ComplexHeatmap</code><span class="citation" data-cites="oESNuFI5"><sup><a href="#ref-oESNuFI5" role="doc-biblioref">110</a></sup></span>.</p>
<h4 id="tumor-mutation-burden-snv-callers-analysis-module">Tumor Mutation Burden (<code>snv-callers</code> analysis module)</h4>
<p>We consider tumor mutation burden (TMB) to be the number of consensus SNVs per effectively surveyed base of the genome.
We considered base pairs to be effectively surveyed if they were in the intersection of the genomic ranges considered by the callers used to generate the consensus and where appropriate, regions of interest, such as coding sequences.
We calculated TMB as:</p>
<p><span class="math display">\[\textrm{TMB} = \frac{\textrm{# of coding sequence SNVs}}{\textrm{Size in Mb of }\textit{effectively surveyed} \textrm{ genome} }\]</span></p>
<p>We used the total number coding sequence consensus SNVs for the numerator and the size of the intersection of the regions considered by <code>Strelka2</code> and <code>Mutect2</code> with coding regions (CDS from GENCODE v27 annotation, see <strong>Key Resources Table</strong>) as the denominator.</p>
<h4 id="clinical-data-harmonization">Clinical Data Harmonization</h4>
<h5 id="who-classification-of-disease-types">WHO Classification of Disease Types</h5>
<p><strong>Table S1</strong> contains a README, along with sample technical, clinical, and additional metadata used for this study.</p>
<h5 id="molecular-subtyping">Molecular Subtyping</h5>
<p>We performed molecular subtyping on tumors in the OpenPBTA to the extent possible.
The <code>molecular_subtype</code> field in <code>pbta-histologies.tsv</code> contains molecular subtypes for tumor types selected from <code>pathology_diagnosis</code> and <code>pathology_free_text_diagnosis</code> fields as described below, following World Health Organization 2016 classification criteria<span class="citation" data-cites="7sgrMadR"><sup><a href="#ref-7sgrMadR" role="doc-biblioref">18</a></sup></span>.</p>
<p>Medulloblastoma (MB) subtypes SHH, WNT, Group 3, and Group 4 were predicted using the consensus of two RNA expression classifiers: <code>MedulloClassifier</code><span class="citation" data-cites="12DKhuCiy"><sup><a href="#ref-12DKhuCiy" role="doc-biblioref">24</a></sup></span> and <code>MM2S</code><span class="citation" data-cites="RmzXK4Cw"><sup><a href="#ref-RmzXK4Cw" role="doc-biblioref">25</a></sup></span> on the RSEM FPKM data (<code>molecular-subtyping-MB</code> analysis module).</p>
<p>High-grade glioma (HGG) subtypes were derived (<code>molecular-subtyping-HGG</code> analysis module) using the following criteria:</p>
<ol type="1">
<li>If any sample contained an <em>H3F3A</em> p.K28M, <em>HIST1H3B</em> p.K28M, <em>HIST1H3C</em> p.K28M, or <em>HIST2H3C</em> p.K28M mutation and no <em>BRAF</em> p.V600E mutation, it was subtyped as <code>DMG, H3 K28</code>.</li>
<li>If any sample contained an <em>HIST1H3B</em> p.K28M, <em>HIST1H3C</em> p.K28M, or <em>HIST2H3C</em> p.K28M mutation and a <em>BRAF</em> p.V600E mutation, it was subtyped as <code>DMG, H3 K28, BRAF V600E</code>.</li>
<li>If any sample contained an <em>H3F3A</em> p.G35V or p.G35R mutation, it was subtyped as<code>HGG, H3 G35</code>.</li>
<li>If any high-grade glioma sample contained an <em>IDH1</em> p.R132 mutation, it was subtyped as <code>HGG, IDH</code>.</li>
<li>If a sample was initially classified as HGG, had no defining histone mutations, and a BRAF p.V600E mutation, it was subtyped as <code>BRAF V600E</code>.</li>
<li>All other high-grade glioma samples that did not meet any of these criteria were subtyped as <code>HGG, H3 wildtype</code>.</li>
</ol>
<p>Embryonal tumors were included in non-MB and non-ATRT embryonal tumor subtyping (<code>molecular-subtyping-embryonal</code> analysis module) if they met any of the following criteria:</p>
<ol type="1">
<li>A <em>TTYH1</em> (5’ partner) fusion was detected.</li>
<li>A <em>MN1</em> (5’ partner) fusion was detected, with the exception of <code>MN1::PATZ1</code> since it is an entity separate of CNS HGNET-MN1 tumors<span class="citation" data-cites="6bmmWiU5"><sup><a href="#ref-6bmmWiU5" role="doc-biblioref">138</a></sup></span>.</li>
<li>Pathology diagnoses included “Supratentorial or Spinal Cord PNET” or “Embryonal Tumor with Multilayered Rosettes”.</li>
<li>A pathology diagnosis of “Neuroblastoma”, where the tumor was not indicated to be peripheral or metastatic and was located in the CNS.</li>
<li>Any sample with “embryonal tumor with multilayer rosettes, ros (who grade iv)”, “embryonal tumor, nos, congenital type”, “ependymoblastoma” or “medulloepithelioma” in pathology free text.</li>
</ol>
<p>Non-MB and non-ATRT embryonal tumors identified with the above criteria were further subtyped (<code>molecular-subtyping-embryonal</code> analysis module) using the criteria below<span class="citation" data-cites="pmid:30249036 t40FioEk WBoOEAK1 Cibjccyk"><sup><a href="#ref-t40FioEk" role="doc-biblioref">139</a>–<a href="#ref-Cibjccyk" role="doc-biblioref">141</a>,<a href="#ref-pmid:30249036" role="doc-biblioref"><strong>pmid:30249036?</strong></a></sup></span>.</p>
<ol type="1">
<li>Any RNA-seq biospecimen with <em>LIN28A</em> overexpression, plus a <em>TYH1</em> fusion (5’ partner) with a gene adjacent or within the C19MC miRNA cluster and/or copy number amplification of the C19MC region was subtyped as <code>ETMR, C19MC-altered</code> (Embryonal tumor with multilayer rosettes, chromosome 19 miRNA cluster altered)<span class="citation" data-cites="7UxjGOxW 5ueZBnsJ"><sup><a href="#ref-5ueZBnsJ" role="doc-biblioref">120</a>,<a href="#ref-7UxjGOxW" role="doc-biblioref">142</a></sup></span>.</li>
<li>Any RNA-seq biospecimen with <em>LIN28A</em> overexpression, a <em>TTYH1</em> fusion (5’ partner) with a gene adjacent or within the C19MC miRNA cluster but no evidence of copy number amplification of the C19MC region was subtyped as <code>ETMR, NOS</code> (Embryonal tumor with multilayer rosettes, not otherwise specified)<span class="citation" data-cites="7UxjGOxW 5ueZBnsJ"><sup><a href="#ref-5ueZBnsJ" role="doc-biblioref">120</a>,<a href="#ref-7UxjGOxW" role="doc-biblioref">142</a></sup></span>.</li>
<li>Any RNA-seq biospecimen with a fusion having a 5’ <em>MN1</em> and 3’ <em>BEND2</em> or <em>CXXC5</em> partner were subtyped as <code>CNS HGNET-MN1</code> [Central nervous system (CNS) high-grade neuroepithelial tumor with <em>MN1</em> alteration].</li>
<li>Non-MB and non-ATRT embryonal tumors with internal tandem duplication (as defined in<span class="citation" data-cites="DEDtXX7K"><sup><a href="#ref-DEDtXX7K" role="doc-biblioref">143</a></sup></span>) of <em>BCOR</em> were subtyped as <code>CNS HGNET-BCOR</code> (CNS high-grade neuroepithelial tumor with <em>BCOR</em> alteration).</li>
<li>Non-MB and non-ATRT embryonal tumors with over-expression and/or gene fusions in <em>FOXR2</em> were subtyped as <code>CNS NB-FOXR2</code> (CNS neuroblastoma with <em>FOXR2</em> activation).</li>
<li>Non-MB and non-ATRT embryonal tumors with <em>CIC::NUTM1</em> or other <em>CIC</em> fusions, were subtyped as <code>CNS EFT-CIC</code> (CNS Ewing sarcoma family tumor with <em>CIC</em> alteration)<span class="citation" data-cites="kfmK8vm"><sup><a href="#ref-kfmK8vm" role="doc-biblioref">119</a></sup></span></li>
<li>Non-MB and non-ATRT embryonal tumors that did not fit any of the above categories were subtyped as <code>CNS Embryonal, NOS</code> (CNS Embryonal tumor, not otherwise specified).</li>
</ol>
<p>Neurocytoma subtypes central neurocytoma (CNC) and extraventricular neurocytoma (EVN) were assigned (<code>molecular-subtyping-neurocytoma</code> analysis module) based on the primary site of the tumor<span class="citation" data-cites="ueHlof7U"><sup><a href="#ref-ueHlof7U" role="doc-biblioref">144</a></sup></span>.
If the tumor’s primary site was “ventricles,” we assigned the subtype as CNC; otherwise, we assigned the subtype as EVN.</p>
<p>Craniopharyngiomas (CRANIO) were subtyped (<code>molecular-subtyping-CRANIO</code> analysis module) into adamantinomatous (<code>CRANIO, ADAM</code>), papillary (<code>CRANIO, PAP</code>) or undetermined (<code>CRANIO, To be classified</code>) based on the following criteria<span class="citation" data-cites="WOSiJ6V2 xnZIq0gr"><sup><a href="#ref-WOSiJ6V2" role="doc-biblioref">145</a>,<a href="#ref-xnZIq0gr" role="doc-biblioref">146</a></sup></span>:</p>
<ol type="1">
<li>Craniopharyngiomas from patients over 40 years old with a <em>BRAF</em> p.V600E mutation were subtyped as <code>CRANIO, PAP</code>.</li>
<li>Craniopharyngiomas from patients younger than 40 years old with mutations in exon 3 of <em>CTNNB1</em> were subtyped as <code>CRANIO, ADAM</code>.</li>
<li>Craniopharyngiomas that did not fall into the above two categories were subtyped as <code>CRANIO, To be classified</code>.</li>
</ol>
<p>A molecular subtype of <code>EWS</code> was assigned to any tumor with a <em>EWSR1</em> fusion or with a <code>pathology_diagnosis</code> of <code>Ewings Sarcoma</code> (<code>molecular-subtyping-EWS</code> analysis module).</p>
<p>Low-grade gliomas (LGG) or glialneuronal tumors (GNT) were subtyped (<code>molecular-subtyping-LGAT</code> analysis module). based on SNV, fusion and CNV status based on<span class="citation" data-cites="ULPByXgG"><sup><a href="#ref-ULPByXgG" role="doc-biblioref">23</a></sup></span>, and as described below.</p>
<ol type="1">
<li>If a sample contained a <em>NF1</em> somatic mutation, either nonsense or missense, it was subtyped as <code>LGG, NF1-somatic</code>.</li>
<li>If a sample contained <em>NF1</em> germline mutation, as indicated by a patient having the neurofibromatosis cancer predisposition, it was subtyped as <code>LGG, NF1-germline</code>.</li>
<li>If a sample contained the <em>IDH</em> p.R132 mutation, it was subtyped as <code>LGG, IDH</code>.</li>
<li>If a sample contained a histone p.K28M mutation in either <em>H3F3A</em>, <em>H3F3B</em>, <em>HIST1H3B</em>, <em>HIST1H3C</em>, or <em>HIST2H3C</em>, or if it contained a p.G35R or p.G35V mutation in <em>H3F3A</em>, it was subtyped as <code>LGG, H3</code>.</li>
<li>If a sample contained <em>BRAF</em> p.V600E or any other non-canonical <em>BRAF</em> mutations in the kinase (PK_Tyr_Ser-Thr) domain PF07714 (see <strong>Key Resources Table</strong>), it was subtyped as <code>LGG, BRAF V600E</code>.</li>
<li>If a sample contained <code>KIAA1549::BRAF</code> fusion, it was subtyped as <code>LGG, KIAA1549::BRAF</code>.</li>
<li>If a sample contained SNV or indel in either <em>KRAS</em>, <em>NRAS</em>, <em>HRAS</em>, <em>MAP2K1</em>, <em>MAP2K2</em>, <em>MAP2K1</em>, <em>ARAF</em>, <em>RAF1</em>, or non-kinase domain of <em>BRAF</em>, or if it contained <em>RAF1</em> fusion, or <em>BRAF</em> fusion that was not <code>KIAA1549::BRAF</code>, it was subtyped as <code>LGG, other MAPK</code>.</li>
<li>If a sample contained SNV in either <em>MET</em>, <em>KIT</em> or <em>PDGFRA</em>, or if it contained fusion in <em>ALK</em>, <em>ROS1</em>, <em>NTRK1</em>, <em>NTRK2</em>, <em>NTRK3</em> or <em>PDGFRA</em>, it was subtyped as <code>LGG, RTK</code>.</li>
<li>If a sample contained <em>FGFR1</em> p.N546K, p.K656E, p.N577, or p. K687 hotspot mutations, or tyrosine kinase domain tandem duplication (See <strong>Key Resources Table</strong>), or <em>FGFR1</em> or <em>FGFR2</em> fusions, it was subtyped as <code>LGG, FGFR</code>.</li>
<li>If a sample contained <em>MYB</em> or <em>MYBL1</em> fusion, it was subtyped as <code>LGG, MYB/MYBL1</code>.</li>
<li>If a sample contained focal CDKN2A and/or CDKN2B deletion, it was subtyped as <code>LGG, CDKN2A/B</code>.</li>
</ol>
<p>For LGG tumors that did not have any of the above molecular alterations, if both RNA and DNA samples were available, it was subtyped as <code>LGG, wildtype</code>.
Otherwise, if either RNA or DNA sample was unavailable, it was subtyped as <code>LGG, To be classified</code>.</p>
<p>If pathology diagnosis was <code>Subependymal Giant Cell Astrocytoma (SEGA)</code>, the <code>LGG</code> portion of molecular subtype was recoded to <code>SEGA</code>.</p>
<p>Lastly, for all LGG- and GNT- subtyped samples, if the tumors were glialneuronal in origin, based on <code>pathology_free_text_diagnosis</code> entries of <code>desmoplastic infantile</code>,<code>desmoplastic infantile ganglioglioma</code>, <code>desmoplastic infantile astrocytoma</code> or <code>glioneuronal</code>, each was recoded as follows:
If pathology diagnosis is <code>Low-grade glioma/astrocytoma (WHO grade I/II)</code> or <code>Ganglioglioma</code>, the <code>LGG</code> portion of the molecular subtype was recoded to <code>GNT</code>.</p>
<p>Ependymomas (EPN) were subtyped (<code>molecular-subtyping-EPN</code> analysis module) into <code>EPN, ST RELA</code>, <code>EPN, ST YAP1</code>, <code>EPN, PF A</code> and <code>EPN, PF B</code> based on evidence for these molecular subgroups as described in Pajtler et al.<span class="citation" data-cites="f7P6U50v"><sup><a href="#ref-f7P6U50v" role="doc-biblioref">129</a></sup></span>.
Briefly, fusion, CNV and gene expression data were used to subtype EPN as follows:</p>
<ol type="1">
<li>Any tumor with fusions containing <code>RELA</code> as fusion partner, e.g., <code>C11orf95::RELA</code>, <code>LTBP3::RELA</code>, was subtyped as <code>EPN, ST RELA</code>.</li>
<li>Any tumor with fusions containing <code>YAP1</code> as fusion partner, such as <code>C11orf95::YAP1</code>, <code>YAP1::MAMLD1</code> and <code>YAP1::FAM118B</code>, was subtyped as <code>EPN, ST YAP1</code>.</li>
<li>Any tumor with the following molecular characterization would be subtyped as <code>EPN, PF A</code>:</li>
</ol>
<ul>
<li><em>CXorf67</em> expression z-score of over 3</li>
<li><em>TKTL1</em> expression z-score of over 3 and 1q gain</li>
</ul>
<ol start="4" type="1">
<li>Any tumor with the following molecular characterization would be subtyped as <code>EPN, PF B</code>:</li>
</ol>
<ul>
<li><em>GPBP17</em> expression z-score of over 3 and loss of 6q or 6p</li>
<li><em>IFT46</em> expression z-score of over 3 and loss of 6q or 6p</li>
</ul>
<p>Any tumor with the above molecular characteristics would be exclusively subtyped to the designated group.</p>
<p>For all other remaining EPN tumors without above molecular characteristics, they would be subtyped to <code>EPN, ST RELA</code> and <code>EPN, ST YAP1</code> in a non-exclusive way (e.g., a tumor could have both <code>EPN, ST RELA</code> and <code>EPN, ST YAP1</code> subtypes) if any of the following alterations were present.</p>
<ol type="1">
<li>Any tumor with the following alterations was assigned <code>EPN, ST RELA</code>:</li>
</ol>
<ul>
<li><code>PTEN::TAS2R1</code> fusion</li>
<li>chromosome 9 arm (9p or 9q) loss</li>
<li><em>RELA</em> expression z-score of over 3</li>
<li><em>L1CAM</em> expression z-score of over 3</li>
</ul>
<ol start="2" type="1">
<li>Any tumor with the following alterations was assigned <code>EPN, ST YAP1</code>:</li>
</ol>
<ul>
<li><code>C11orf95::MAML2</code> fusion</li>
<li>chromosome 11 short arm (11p) loss</li>
<li>chromosome 11 long arm (11q) gain</li>
<li><em>ARL4D</em> expression z-score of over 3</li>
<li><em>CLDN1</em> expression z-score of over 3</li>
</ul>
<p>After all relevant tumor samples were subtyped by the above molecular subtyping modules, the results from these modules, along with other clinical information (such as pathology diagnosis free text), were compiled in the <code>molecular-subtyping-pathology</code> module and integrated into the OpenPBTA data in the <code>molecular-subtyping-integrate</code> module.</p>
<h4 id="tp53-alteration-annotation-tp53_nf1_score-analysis-module">TP53 Alteration Annotation (<code>tp53_nf1_score</code> analysis module)</h4>
<p>We annotated <em>TP53</em> altered HGG samples as either <code>TP53 lost</code> or <code>TP53 activated</code> and integrated this within the molecular subtype.
To this end, we applied a <em>TP53</em> inactivation classifier originally trained on TCGA pan-cancer data<span class="citation" data-cites="X6oQhIf8"><sup><a href="#ref-X6oQhIf8" role="doc-biblioref">43</a></sup></span> to the matched RNA expression data, with samples batched by library type.
Along with the <em>TP53</em> classifier scores, we collectively used consensus SNV and CNV, SV, and reference databases that list <em>TP53</em> hotspot mutations<span class="citation" data-cites="10jkUywit tfWVBF8o"><sup><a href="#ref-10jkUywit" role="doc-biblioref">147</a>,<a href="#ref-tfWVBF8o" role="doc-biblioref">148</a></sup></span> and functional domains<span class="citation" data-cites="KxZPoysm"><sup><a href="#ref-KxZPoysm" role="doc-biblioref">149</a></sup></span> to determine <em>TP53</em> alteration status for each sample.
We adopted the following rules for calling either <code>TP53 lost</code> or <code>TP53 activated</code>:</p>
<ol type="1">
<li><p>If a sample had either of the two well-characterized <em>TP53</em> gain-of-function mutations, p.R273C or p.R248W<span class="citation" data-cites="14asizOhY"><sup><a href="#ref-14asizOhY" role="doc-biblioref">44</a></sup></span>, we assigned <code>TP53 activated</code> status.</p></li>
<li><p>Samples were annotated as <code>TP53 lost</code> if they contained i) a <em>TP53</em> hotspot mutation as defined by IARC <em>TP53</em> database or the MSKCC cancer hotspots database<span class="citation" data-cites="10jkUywit tfWVBF8o"><sup><a href="#ref-10jkUywit" role="doc-biblioref">147</a>,<a href="#ref-tfWVBF8o" role="doc-biblioref">148</a></sup></span> (see also, <strong>Key Resources Table</strong>), ii) two <em>TP53</em> alterations, including SNV, CNV or SV, indicative of probable bi-allelic alterations; iii) one <em>TP53</em> somatic alteration, including SNV, CNV, or SV or a germline <em>TP53</em> mutation indicated by the diagnosis of Li-Fraumeni syndrome (LFS)<span class="citation" data-cites="JdxWaORm"><sup><a href="#ref-JdxWaORm" role="doc-biblioref">150</a></sup></span>, or iv) one germline <em>TP53</em> mutation indicated by LFS and the <em>TP53</em> classifier score for matched RNA-Seq was greater than 0.5.</p></li>
</ol>
<h4 id="prediction-of-participants-genetic-sex">Prediction of participants’ genetic sex</h4>
<p>Participant metadata included a reported gender.
We used WGS germline data, in concert with the reported gender, to predict participant genetic sex so that we could identify sexually dimorphic outcomes.
This analysis may also indicate samples that may have been contaminated.
We used the <code>idxstats</code> utility from <code>SAMtools</code><span class="citation" data-cites="pmid:19505943"><sup><a href="#ref-pmid:19505943" role="doc-biblioref"><strong>pmid:19505943?</strong></a></sup></span> to calculate read lengths, the number of mapped reads, and the corresponding chromosomal location for reads to the X and Y chromosomes.
We used the fraction of total normalized X and Y chromosome reads that were attributed to the Y chromosome as a summary statistic.
We manually reviewed this statistic in the context of reported gender and determined that a threshold of less than 0.2 clearly delineated female samples.
We marked fractions greater than 0.4 as predicted males, and we marked samples with values in the inclusive range 0.2-0.4 as unknown.
We performed this analysis through <a href="https://github.com/d3b-center/sex-determination-tool">CWL</a> on CAVATICA.
We added resulting calls to the histologies file under the column header <code>germline_sex_estimate</code>.</p>
<h4 id="selection-of-independent-samples-independent-samples-analysis-module">Selection of independent samples (<code>independent-samples</code> analysis module)</h4>
<p>Certain analyses required that we select only a single representative specimen for each individual.
In these cases, we identified a single specimen by prioritizing primary tumors and those with whole-genome sequencing available.
If this filtering still resulted in multiple specimens, we randomly selected a single specimen from the remaining set.</p>
<h4 id="quantification-of-telomerase-activity-using-gene-expression-data-telomerase-activity-prediction-analysis-module">Quantification of Telomerase Activity using Gene Expression Data (<code>telomerase-activity-prediction</code> analysis module)</h4>
<p>We predicted telomerase activity of tumor samples using the recently developed <code>EXTEND</code> method<span class="citation" data-cites="RnBZNbdx"><sup><a href="#ref-RnBZNbdx" role="doc-biblioref">46</a></sup></span>, with samples batched by library type.
Briefly, <code>EXTEND</code> estimates telomerase activity based on the expression of a 13-gene signature.
We derived this signature by comparing telomerase-positive tumors and tumors with activated alternative lengthening of telomeres pathway, a group presumably negative of telomerase activity.</p>
<h4 id="survival-models-survival-analysis-analysis-module">Survival models (<code>survival-analysis</code> analysis module)</h4>
<p>We calculated overall survival (OS) as days since initial diagnosis and performed several survival analyses on the OpenPBTA cohort using the <a href="https://cran.r-project.org/package=survival"><code>survival</code> R package</a>.
We performed survival analysis for patients by HGG subtype using the Kaplan-Meier estimator<span class="citation" data-cites="6RHepB1T"><sup><a href="#ref-6RHepB1T" role="doc-biblioref">151</a></sup></span> and a log-rank test (Mantel-Cox test)<span class="citation" data-cites="pmid:5910392"><sup><a href="#ref-pmid:5910392" role="doc-biblioref"><strong>pmid:5910392?</strong></a></sup></span> on the different HGG subtypes.
Next, we used multivariate Cox (proportional hazards) regression analysis<span class="citation" data-cites="y0J19Ubd"><sup><a href="#ref-y0J19Ubd" role="doc-biblioref">152</a></sup></span> to model the following: a) <code>tp53 scores + telomerase scores + extent of tumor resection + LGG group + HGG group</code>, in which <code>tp53 scores</code> and <code>telomerase scores</code> are numeric, <code>extent of tumor resection</code> is categorical, and <code>LGG group</code> and <code>HGG group</code> are binary variables indicating whether the sample is in either broad histology grouping, b) <code>tp53 scores + telomerase scores + extent of tumor resection</code> for each <code>cancer_group</code> with an N&gt;=3 deceased patients (DIPG, DMG, HGG, MB, and EPN), and c) <code>quantiseq cell type fractions + CD274 expression + extent of tumor resection</code> for each <code>cancer_group</code> with an N&gt;=3 deceased patients (DIPG, DMG, HGG, MB, and EPN), in which <code>quantiseq cell type fractions</code> and <code>CD274 expression</code> are numeric.</p>
<h3 id="key-resources-table">KEY RESOURCES TABLE</h3>
<table>
<colgroup>
<col style="width: 21%" />
<col style="width: 28%" />
<col style="width: 50%" />
</colgroup>
<thead>
<tr class="header">
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Critical commercial assays</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>Recover Cell Culture Freezing media</td>
<td>Gibco</td>
<td>12648010</td>
</tr>
<tr class="odd">
<td>Hank’s Balanced Salt Solution (HBSS)</td>
<td>Gibco</td>
<td>14175095</td>
</tr>
<tr class="even">
<td>Papain</td>
<td>SciQuest</td>
<td>LS003124</td>
</tr>
<tr class="odd">
<td>Ovomucoid</td>
<td>SciQuest</td>
<td>542000</td>
</tr>
<tr class="even">
<td>DNase</td>
<td>Roche</td>
<td>10104159001</td>
</tr>
<tr class="odd">
<td>100μm cell strainer</td>
<td>Greiner Bio-One</td>
<td>542000</td>
</tr>
<tr class="even">
<td>DMEM/F-12 medium</td>
<td>Sigma</td>
<td>D8062</td>
</tr>
<tr class="odd">
<td>Fetal Bovine Serum (FBS)</td>
<td>Hyclone</td>
<td>SH30910.03</td>
</tr>
<tr class="even">
<td>GlutaMAX</td>
<td>Gibco</td>
<td>35050061</td>
</tr>
<tr class="odd">
<td>Penicillin/Streptomycin-Amphotericin B</td>
<td>Lonza</td>
<td>17-745E</td>
</tr>
<tr class="even">
<td>Normocin</td>
<td>Invivogen</td>
<td>ant-nr-2</td>
</tr>
<tr class="odd">
<td>B-27 supplement minus vitamin A</td>
<td>Gibco</td>
<td>12587-010</td>
</tr>
<tr class="even">
<td>N-2 supplement</td>
<td>Gibco</td>
<td>17502001</td>
</tr>
<tr class="odd">
<td>Epidermal growth factor</td>
<td>Gibco</td>
<td>PHG0311L</td>
</tr>
<tr class="even">
<td>Basic fibroblast growth factor</td>
<td>PeproTech</td>
<td>100-18B</td>
</tr>
<tr class="odd">
<td>Heparin</td>
<td>Sigma</td>
<td>H3149</td>
</tr>
<tr class="even">
<td>DNA/RNA AllPrep Kit</td>
<td>Qiagen</td>
<td>80204</td>
</tr>
<tr class="odd">
<td>TruSeq RNA Sample Prep Kit</td>
<td>Illumina</td>
<td>FC-122-1001</td>
</tr>
<tr class="even">
<td>KAPA Library Preparation Kit</td>
<td>Roche</td>
<td>KK8201</td>
</tr>
<tr class="odd">
<td>AllPrep DNA/RNA/miRNA Universal kit</td>
<td>Qiagen</td>
<td>80224</td>
</tr>
<tr class="even">
<td>RNase A</td>
<td>Qiagen</td>
<td>19101</td>
</tr>
<tr class="odd">
<td>QIAsymphony DSP DNA Midi Kit</td>
<td>Qiagen</td>
<td>937255</td>
</tr>
<tr class="even">
<td>KAPA HyperPrep kit</td>
<td>Roche</td>
<td>08098107702</td>
</tr>
<tr class="odd">
<td>RiboErase kit</td>
<td>Roche</td>
<td>07962304001</td>
</tr>
<tr class="even">
<td>Raw and harmonized WGS, WXS, Panel, RNA-Seq</td>
<td>KidsFirst Data Resource Center, this project</td>
<td><span class="citation" data-cites="THjIzNXb"><sup><a href="#ref-THjIzNXb" role="doc-biblioref">75</a></sup></span></td>
</tr>
<tr class="odd">
<td>Merged summary files</td>
<td>this project</td>
<td>https://cavatica.sbgenomics.com/u/cavatica/openpbta</td>
</tr>
<tr class="even">
<td>Merged summary files and downstream analyses</td>
<td>this project</td>
<td>https://github.com/AlexsLemonade/OpenPBTA-analysis/</td>
</tr>
<tr class="odd">
<td>Processed data</td>
<td>this project</td>
<td>https://pedcbioportal.kidsfirstdrc.org/study/summary?id=openpbta</td>
</tr>
<tr class="even">
<td>Experimental models: Cell lines</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>CBTN pediatric brain tumor-derived cell lines</td>
<td><span class="citation" data-cites="DwCzMoBJ"><sup><a href="#ref-DwCzMoBJ" role="doc-biblioref">16</a></sup></span></td>
<td>See <strong>Table S1</strong> for identifiers</td>
</tr>
<tr class="even">
<td>Software and algorithms</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>Data processing and analysis software</td>
<td>Multiple</td>
<td>See <strong>Table S5</strong> for identifiers</td>
</tr>
<tr class="even">
<td>OpenPBTA workflows repository</td>
<td>this project</td>
<td><span class="citation" data-cites="t5yrhQPI"><sup><a href="#ref-t5yrhQPI" role="doc-biblioref">153</a></sup></span></td>
</tr>
<tr class="odd">
<td>OpenPBTA analysis repository</td>
<td>this project</td>
<td><span class="citation" data-cites="SZviDnAS"><sup><a href="#ref-SZviDnAS" role="doc-biblioref">154</a></sup></span></td>
</tr>
<tr class="even">
<td>OpenPBTA manuscript repository</td>
<td>this project</td>
<td></td>
</tr>
<tr class="odd">
<td></td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>Other</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>TCGA WXS dataset</td>
<td>National Institutes of Health The Cancer Genome Atlas (TCGA)</td>
<td>dbGAP phs000178.v11.p8</td>
</tr>
<tr class="even">
<td>Cancer hotspots</td>
<td>MSKCC</td>
<td>https://www.cancerhotspots.org/#/download (v2)</td>
</tr>
<tr class="odd">
<td>Reference genomes</td>
<td>Broad</td>
<td>https://s3.console.aws.amazon.com/s3/buckets/broad-references/hg38/v0/</td>
</tr>
<tr class="even">
<td>Reference genome hg38, patch release 12</td>
<td>UCSC</td>
<td>http://hgdownload.soe.ucsc.edu/goldenPath/hg38/bigZips/</td>
</tr>
<tr class="odd">
<td>Human Cytoband file</td>
<td>UCSC</td>
<td>http://hgdownload.cse.ucsc.edu/goldenpath/hg38/database/cytoBand.txt.gz</td>
</tr>
<tr class="even">
<td>CDS from GENCODE v27 annotation</td>
<td>GENCODE</td>
<td>https://www.gencodegenes.org/human/release_27.html</td>
</tr>
<tr class="odd">
<td>PFAM domains and locations</td>
<td>UCSC</td>
<td>http://hgdownload.soe.ucsc.edu/goldenPath/hg38/database/pfamDesc.txt.gz; https://pfam.xfam.org/family/PF07714</td>
</tr>
<tr class="even">
<td>BSgenome.Hsapiens.UCSC.hg38 annotations</td>
<td>Bioconductor</td>
<td>https://bioconductor.org/packages/release/data/annotation/html/BSgenome.Hsapiens.UCSC.hg38.html</td>
</tr>
<tr class="odd">
<td>gnomAD v2.1.1 (exome and genome)</td>
<td>Genome Aggregation Database</td>
<td>https://gnomad.broadinstitute.org/downloads#v2-liftover-variants</td>
</tr>
<tr class="even">
<td>KEGG MMR gene set v7.5.1</td>
<td>BROAD Institute</td>
<td>https://www.gsea-msigdb.org/gsea/msigdb/download_geneset.jsp?geneSetName=KEGG_MISMATCH_REPAIR</td>
</tr>
<tr class="odd">
<td>ClinVar Database (2022-05-07)</td>
<td>NCBI</td>
<td>https://ftp.ncbi.nlm.nih.gov/pub/clinvar/vcf_GRCh38/archive_2.0/2022/clinvar_20220507.vcf.gz</td>
</tr>
</tbody>
</table>
<h2 id="supplemental-information-titles-and-legends">Supplemental Information Titles and Legends</h2>
<div id="fig:S1" class="fignos">
<figure>
<img src="https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/361fef09c574942d3d5ef0668461b37e3ee6cba6/figures/pngs/figureS1.png?sanitize=true" data-tag="S1" style="width:7in" alt="Figure S1: OpenPBTA Project Workflow, Related to Figure 1. Biospecimens and data were collected by CBTN and PNOC. Genomic sequencing and harmonization (orange boxes) were performed by the Kids First Data Resource Center (KFDRC). Analyses in the green boxes were performed by contributors of the OpenPBTA project. Output files are denoted in blue. Figure created with BioRender.com." />
<figcaption aria-hidden="true"><span>Figure S1:</span> <strong>OpenPBTA Project Workflow, Related to Figure 1.</strong> Biospecimens and data were collected by CBTN and PNOC. Genomic sequencing and harmonization (orange boxes) were performed by the Kids First Data Resource Center (KFDRC). Analyses in the green boxes were performed by contributors of the OpenPBTA project. Output files are denoted in blue. Figure created with <a href="biorender.com">BioRender.com</a>.</figcaption>
</figure>
</div>
<div id="fig:S2" class="fignos">
<figure>
<img src="https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/361fef09c574942d3d5ef0668461b37e3ee6cba6/figures/pngs/figureS2.png?sanitize=true" data-tag="S2" style="width:7in" alt="Figure S2: Validation of Consensus SNV calls and Tumor Mutation Burden, Related to Figures 2 and 3. Correlation (A) and violin (B) plots of mutation variant allele frequencies (VAFs) comparing the variant callers (Lancet, Strelka2, Mutect2, and VarDict) used for PBTA samples. Upset plot (C) showing overlap of variant calls. Correlation (D) and violin (E) plots of mutation variant allele frequencies (VAFs) comparing the variant callers (Lancet, Strelka2, and Mutect2) used for TCGA samples. Upset plot (F) showing overlap of variant calls. Violin plots (G) showing VAFs for Lancet calls performed on WGS and WXS from the same tumor (N = 52 samples from 13 patients). Cumulative distribution TMB plots for PBTA (H) and TCGA (I) tumors using consensus SNV calls." />
<figcaption aria-hidden="true"><span>Figure S2:</span> <strong>Validation of Consensus SNV calls and Tumor Mutation Burden, Related to Figures 2 and 3.</strong> Correlation (A) and violin (B) plots of mutation variant allele frequencies (VAFs) comparing the variant callers (Lancet, Strelka2, Mutect2, and VarDict) used for PBTA samples. Upset plot (C) showing overlap of variant calls. Correlation (D) and violin (E) plots of mutation variant allele frequencies (VAFs) comparing the variant callers (Lancet, Strelka2, and Mutect2) used for TCGA samples. Upset plot (F) showing overlap of variant calls. Violin plots (G) showing VAFs for Lancet calls performed on WGS and WXS from the same tumor (N = 52 samples from 13 patients). Cumulative distribution TMB plots for PBTA (H) and TCGA (I) tumors using consensus SNV calls.</figcaption>
</figure>
</div>
<div id="fig:S3" class="fignos">
<figure>
<img src="https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/361fef09c574942d3d5ef0668461b37e3ee6cba6/figures/pngs/figureS3.png?sanitize=true" data-tag="S3" style="width:7in" alt="Figure S3: Genomic instability of pediatric brain tumors, Related to Figures 2 and 3. (A) Violin plots of tumor purity by cancer group. Dots represent the group median. (B) Oncoprint of canonical somatic gene mutations, CNVs, fusions, and TMB (top bar plot) for the top 20 genes mutated across rare CNS tumors: desmoplastic infantile astrocytoma and ganglioglioma (N = 2), germinoma (N = 4), glial-neuronal NOS (N = 8), metastatic secondary tumors (N = 2), neurocytoma (N = 2), pineoblastoma (N = 4), Rosai-Dorfman disease (N = 2), and sarcomas (N = 4). Patient sex (Germline sex estimate) and tumor histology (Cancer Group) are displayed as annotations at the bottom of each plot. Only primary tumors with mutations in the listed genes are shown. Multiple CNVs are denoted as a complex event. (C) Genome-wide plot of CNV alterations by broad histology. Each row represents one sample. Box and whisker plots of number of CNV breaks (D) or SV breaks (E) by number of chromothripsis regions. Box plot represents 5% (lower whisker), 25% (lower box), 50% (median), 75% (upper box), and 95% (upper whisker) quantiles." />
<figcaption aria-hidden="true"><span>Figure S3:</span> <strong>Genomic instability of pediatric brain tumors, Related to Figures 2 and 3.</strong> (A) Violin plots of tumor purity by cancer group. Dots represent the group median. (B) Oncoprint of canonical somatic gene mutations, CNVs, fusions, and TMB (top bar plot) for the top 20 genes mutated across rare CNS tumors: desmoplastic infantile astrocytoma and ganglioglioma (N = 2), germinoma (N = 4), glial-neuronal NOS (N = 8), metastatic secondary tumors (N = 2), neurocytoma (N = 2), pineoblastoma (N = 4), Rosai-Dorfman disease (N = 2), and sarcomas (N = 4). Patient sex (<code>Germline sex estimate</code>) and tumor histology (<code>Cancer Group</code>) are displayed as annotations at the bottom of each plot. Only primary tumors with mutations in the listed genes are shown. Multiple CNVs are denoted as a complex event. (C) Genome-wide plot of CNV alterations by broad histology. Each row represents one sample. Box and whisker plots of number of CNV breaks (D) or SV breaks (E) by number of chromothripsis regions. Box plot represents 5% (lower whisker), 25% (lower box), 50% (median), 75% (upper box), and 95% (upper whisker) quantiles.</figcaption>
</figure>
</div>
<div id="fig:S4" class="fignos">
<figure>
<img src="https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/361fef09c574942d3d5ef0668461b37e3ee6cba6/figures/pngs/figureS4.png?sanitize=true" data-tag="S4" style="width:7in" alt="Figure S4: Mutational signatures in pediatric brain tumors, Related to Figure 3. (A) Sample-specific RefSig signature weights across cancer groups ordered by decreasing Signature 1 exposure. (B) Proportion of Signature 1 plotted by phase of therapy for each cancer group." />
<figcaption aria-hidden="true"><span>Figure S4:</span> <strong>Mutational signatures in pediatric brain tumors, Related to Figure 3.</strong> (A) Sample-specific RefSig signature weights across cancer groups ordered by decreasing Signature 1 exposure. (B) Proportion of Signature 1 plotted by phase of therapy for each cancer group.</figcaption>
</figure>
</div>
<div id="fig:S5" class="fignos">
<figure>
<img src="https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/361fef09c574942d3d5ef0668461b37e3ee6cba6/figures/pngs/figureS5.png?sanitize=true" data-tag="S5" style="width:7in" alt="Figure S5: Quality control metrics for TP53 and EXTEND scores, Related to Figure 4. (A) Receiver Operating Characteristic for TP53 classifier run on FPKM of poly-A RNA-Seq samples. Correlation plots for telomerase scores (EXTEND) with RNA expression of TERT (B) and TERC (C). Red dots in B and C denote samples with known TERT promoter (TERTp) mutations." />
<figcaption aria-hidden="true"><span>Figure S5:</span> <strong>Quality control metrics for <em>TP53</em> and EXTEND scores, Related to Figure 4</strong>. (A) Receiver Operating Characteristic for <em>TP53</em> classifier run on FPKM of poly-A RNA-Seq samples. Correlation plots for telomerase scores (EXTEND) with RNA expression of <em>TERT</em> (B) and <em>TERC</em> (C). Red dots in B and C denote samples with known <em>TERT</em> promoter (TERTp) mutations.</figcaption>
</figure>
</div>
<div id="fig:S6" class="fignos">
<figure>
<img src="https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/361fef09c574942d3d5ef0668461b37e3ee6cba6/figures/pngs/figureS6.png?sanitize=true" data-tag="S6" style="width:7in" alt="Figure S6: Subtype-specific clustering and immune cell fractions, Related to Figure 5. First two dimensions from UMAP of sample transcriptome data with points colored by molecular_subtype for medulloblastoma (A), ependymoma (B), low-grade glioma (C), and high-grade glioma (D). (E) Box plots of quanTIseq estimates of immune cell fractions in histologies with more than one molecular subtype with N &gt;=3. (F) Box plots of the ratio of immune cell fractions of CD8+ to CD4+ T cells in histologies with more than one molecular subtype with N &gt;=3. Box plot represents 5% (lower whisker), 25% (lower box), 50% (median), 75% (upper box), and 95% (upper whisker) quantiles." />
<figcaption aria-hidden="true"><span>Figure S6:</span> <strong>Subtype-specific clustering and immune cell fractions, Related to Figure 5</strong>. First two dimensions from UMAP of sample transcriptome data with points colored by <code>molecular_subtype</code> for medulloblastoma (A), ependymoma (B), low-grade glioma (C), and high-grade glioma (D). (E) Box plots of quanTIseq estimates of immune cell fractions in histologies with more than one molecular subtype with N &gt;=3. (F) Box plots of the ratio of immune cell fractions of CD8+ to CD4+ T cells in histologies with more than one molecular subtype with N &gt;=3. Box plot represents 5% (lower whisker), 25% (lower box), 50% (median), 75% (upper box), and 95% (upper whisker) quantiles.</figcaption>
</figure>
</div>
<p><a href="https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/361fef09c574942d3d5ef0668461b37e3ee6cba6/tables/results/TableS1-histologies.xlsx"><strong>Table S1. Related to Figure 1.</strong></a>
Table of specimens and associated metadata, clinical data, and histological data utilized in the OpenPBTA project.</p>
<p><a href="https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/361fef09c574942d3d5ef0668461b37e3ee6cba6/tables/results/TableS2-DNA-results-table.xlsx"><strong>Table S2. Related to Figures 2 and 3.</strong></a>
Excel file with three sheets representing tables of TMB, eight CNS mutational signatures, and chromothripsis events per sample, respectively.</p>
<p><a href="https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/361fef09c574942d3d5ef0668461b37e3ee6cba6/tables/results/TableS3-RNA-results-table.xlsx"><strong>Table S3. Related to Figures 4 and 5.</strong></a>
Excel file with three sheets representing tables of <em>TP53</em> scores, telomerase EXTEND scores, and quanTIseq immune scores, respectively.</p>
<p><a href="https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/361fef09c574942d3d5ef0668461b37e3ee6cba6/tables/results/TableS4-survival-results-table.xlsx"><strong>Table S4. Related to Figures 4 and 5.</strong></a>
Excel file with six sheets representing the survival analyses performed for this manuscript.
See <strong>Star Methods</strong> for details.</p>
<p><a href="https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/361fef09c574942d3d5ef0668461b37e3ee6cba6/tables/results/TableS5-Key-Resources-table.xlsx"><strong>Table S5. Related to Figure 1.</strong></a>
Excel file with four sheets representing of all software and their respective versions used for the OpenPBTA project, including the R packages in the OpenPBTA Docker image, Python packages i the OpenPBTA Docker image, other command line tools in the OpenPBTA Docker image, and all software used in the OpenPBTA workflows, respectively.
Note that all software in the OpenPBTA Docker image was utilized within the analysis repository, but not all software was used for the final manuscript.</p>
<h2 id="consortia">Consortia</h2>
<p>The past and present members of the Children’s Brain Tumor Network who contributed to the generation of specimens and data are Adam C. Resnick, Alexa Plisiewicz, Allison M. Morgan, Allison P. Heath, Alyssa Paul, Amanda Saratsis, Amy Smith, Ana Aguilar, Ana Guerreiro Stücklin, Anastasia Arynchyna, Andrea Franson, Angela J. Waanders, Angela N. Viaene, Anita Nirenberg, Anna Maria Buccoliero, Anna Yaffe, Anny Shai, Anthony Bet, Antoinette Price, Arlene Luther, Ashley Plant, Augustine Eze, Bailey K. Farrow, Baoli Hu, Beth Frenkel, Bo Zhang, Bobby Moulder, Bonnie Cole, Brian M. Ennis, Brian R. Rood, Brittany Lebert, Carina A. Leonard, Carl Koschmann, Caroline Caudill, Caroline Drinkwater, Cassie N. Kline, Catherine Sullivan, Chanel Keoni, Chiara Caporalini, Christine Bobick-Butcher, Christopher Mason, Chunde Li, Claire Carter, Claudia MaduroCoronado, Clayton Wiley, Cynthia Wong, David E. Kram, David Haussler, David Kram, David Pisapia, David Ziegler, Denise Morinigo, Derek Hanson, Donald W. Parsons, Elizabeth Appert, Emily Drake, Emily Golbeck, Ena Agbodza, Eric H. Raabe, Eric M. Jackson, Erin Alexander, Esteban Uceda, Eugene Hwang, Fausto Rodriquez, Gabrielle S. Stone, Gary Kohanbash, Gavriella Silverman, George Rafidi, Gerald Grant, Gerri Trooskin, Gilad Evrony, Graham Keyes, Hagop Boyajian, Holly B. Lindsay, Holly C. Beale, Ian F. Pollack, James Johnston, James Palmer, Jane Minturn, Jared Pisapia, Jason E. Cain, Jason R. Fangusaro, Javad Nazarian, Jeanette Haugh, Jeff Stevens, Jeffrey P. Greenfield, Jeffrey Rubens, Jena V. Lilly, Jennifer L. Mason, Jessica B. Foster, Jim Olson, Jo Lynne Rokita, Joanna J. Phillips, Jonathan Waller, Josh Rubin, Judy E. Palma, Justin McCroskey, Justine Rizzo, Kaitlin Lehmann, Kamnaa Arya, Karlene Hall, Katherine Pehlivan, Kenneth Seidl, Kimberly Diamond, Kristen Harnett, Kristina A. Cole, Krutika S. Gaonkar, Lamiya Tauhid, Laura Prolo, Leah Holloway, Leslie Brosig, Lina Lopez, Lionel Chow, Madhuri Kambhampati, Mahdi Sarmady, Margaret Nevins, Mari Groves, Mariarita Santi-Vicini, Marilyn M. Li, Marion Mateos, Mateusz Koptyra, Matija Snuderl, Matthew Miller, Matthew Sklar, Matthew Wood, Meghan Connors, Melissa Williams, Meredith Egan, Michael Fisher, Michael Koldobskiy, Michelle Monje, Migdalia Martinez, Miguel A. Brown, Mike Prados, Miriam Bornhorst, Mirko Scagnet, Mohamed AbdelBaki, Monique Carrero-Tagle, Nadia Dahmane, Nalin Gupta, Nathan Young, Nicholas A. Vitanza, Nicholas Tassone, Nicholas Van Kuren, Nicolas Gerber, Nithin D. Adappa, Nitin Wadhwani, Noel Coleman, Obi Obayashi, Olena M. Vaske, Olivier Elemento, Oren Becher, Philbert Oliveros, Phillip B. Storm, Pichai Raman, Prajwal Rajappa, Rintaro Hashizume, Rishi R. Lulla, Robert Keating, Robert M. Lober, Ron Firestein, Sabine Mueller, Sameer Agnihotri, Samuel G. Winebrake, Samuel Rivero-Hinojosa, Sarah Diane Black, Sarah Leary, Schuyler Stoller, Shannon Robins, Sharon Gardner, Shelly Wang, Sherri Mayans, Sherry Tutson, Shida Zhu, Sofie R. Salama, Sonia Partap, Sonika Dahiya, Sriram Venneti, Stacie Stapleton, Stephani Campion, Stephanie Stefankiewicz, Stewart Goldman, Swetha Thambireddy, Tatiana S. Patton, Teresa Hidalgo, Theo Nicolaides, Thinh Q. Nguyen, Thomas W. McLean, Tiffany Walker, Toba Niazi, Tobey MacDonald, Valeria Lopez-Gil, Valerie Baubet, Whitney Rife, Xiao-Nan Li, Ximena Cuellar, Yiran Guo, Yuankun Zhu, and Zeinab Helil.</p>
<p>The past and present members of the Pacific Pediatric Neuro-Oncology Consortium who contributed to the generation of specimens and data are Adam C. Resnick, Alicia Lenzen, Alyssa Reddy, Amar Gajjar, Ana Guerreiro Stucklin, Anat Epstein, Andrea Franson, Angela Waanders, Anne Bendel, Anu Banerjee, Ashley Margol, Ashley Plant, Brian Rood, Carl Koschmann, Carol Bruggers, Caroline Hastings, Cassie N. Kline, Christina Coleman Abadi, Christopher Tinkle, Corey Raffel, Dan Runco, Daniel Landi, Daphne Adele Haas-Kogan, David Ashley, David Ziegler, Derek Hanson, Dong Anh Khuong Quang, Duane Mitchell, Elias Sayour, Eric Jackson, Eric Raabe, Eugene Hwang, Fatema Malbari, Geoffrey McCowage, Girish Dhall, Gregory Friedman, Hideho Okada, Ibrahim Qaddoumi, Iris Fried, Jae Cho, Jane Minturn, Jason Blatt, Javad Nazarian, Jeffrey Rubens, Jena V. Lilly, Jennifer Elster, Jennifer L. Mason, Jessica Schulte, Jonathan Schoenfeld, Josh Rubin, Karen Gauvain, Karen Wright, Katharine Offer, Katie Metrock, Kellie Haworth, Ken Cohen, Kristina A. Cole, Lance Governale, Linda Stork, Lindsay Kilburn, Lissa Baird, Maggie Skrypek, Marcia Leonard, Margaret Shatara, Margot Lazow, Mariella Filbin, Maryam Fouladi, Matthew Miller, Megan Paul, Michael Fisher, Michael Koldobskiy, Michael Prados, Michal Yalon Oren, Mimi Bandopadhayay, Miriam Bornhorst, Mohamed AbdelBaki, Nalin Gupta, Nathan Robison, Nicholas Whipple, Nick Gottardo, Nicholas A. Vitanza, Nicolas Gerber, Patricia Robertson, Payal Jain, Peter Sun, Priya Chan, Richard S Lemons, Robert Wechsler-Reya, Roger Packer, Russ Geyer, Ryan Velasco, Sabine Mueller, Sahaja Acharya, Sam Cheshier, Sarah Leary, Scott Coven, Sebastian M. Waszak, Sharon Gardner, Sri Gururangan, Stewart Goldman, Susan Chi, Tab Cooney, Tatiana S. Patton, Theodore Nicolaides, and Tom Belle Davidson.</p>
<h2 class="page_break_before" id="references">References</h2>
<!-- Explicitly insert bibliography here -->
<div id="refs" class="references csl-bib-body" role="doc-bibliography">
<div id="ref-nAMOAA9j" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1. </div><div class="csl-right-inline">Ostrom, Q.T., Cioffi, G., Gittleman, H., Patil, N., Waite, K., Kruchko, C., and Barnholtz-Sloan, J.S. (2019). CBTRUS Statistical Report: Primary Brain and Other Central Nervous System Tumors Diagnosed in the United States in 2012–2016. Neuro-Oncology <em>21</em>, v1–v100. <a href="https://doi.org/10.1093/neuonc/noz150">10.1093/neuonc/noz150</a>.</div>
</div>
<div id="ref-1H6hugOV3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2. </div><div class="csl-right-inline">Ostrom, Q.T., Gittleman, H., Xu, J., Kromer, C., Wolinsky, Y., Kruchko, C., and Barnholtz-Sloan, J.S. (2016). CBTRUS Statistical Report: Primary Brain and Other Central Nervous System Tumors Diagnosed in the United States in 2009–2013. Neuro-Oncology <em>18</em>, v1–v75. <a href="https://doi.org/10.1093/neuonc/now207">10.1093/neuonc/now207</a>.</div>
</div>
<div id="ref-7p58e5Wj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">3. </div><div class="csl-right-inline">Blank, P.M., Ostrom, Q.T., Rouse, C., Wolinsky, Y., Kruchko, C., Salcido, J., and Barnholtz‐Sloan, J.S. (2015). Years of life lived with disease and years of potential life lost in children who die of cancer in the United States, 2009. Cancer Med <em>4</em>, 608–619. <a href="https://doi.org/10.1002/cam4.410">10.1002/cam4.410</a>.</div>
</div>
<div id="ref-U5WisTG5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">4. </div><div class="csl-right-inline">Lilly, J.V., Rokita, J.L., Mason, J.L., Patton, T., Stefankiewiz, S., Higgins, D., Trooskin, G., Larouci, C.A., Arya, K., Appert, E., et al. (2023). The children's brain tumor network (CBTN) - Accelerating research in pediatric central nervous system tumors through collaboration and open science. Neoplasia <em>35</em>, 100846. <a href="https://doi.org/10.1016/j.neo.2022.100846">10.1016/j.neo.2022.100846</a>.</div>
</div>
<div id="ref-12pkJWML4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">5. </div><div class="csl-right-inline">Oncology Center of Excellence, U.S. Food and Drug Administration (2022). <a href="https://www.fda.gov/about-fda/oncology-center-excellence/pediatric-oncology-drug-approvals">Pediatric Oncology Drug Approvals</a>.</div>
</div>
<div id="ref-1ApRLzuh0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">6. </div><div class="csl-right-inline">Vable, A.M., Diehl, S.F., and Glymour, M.M. (2021). Code Review as a Simple Trick to Enhance Reproducibility, Accelerate Learning, and Improve the Quality of Your Team’s Research. American Journal of Epidemiology <em>190</em>, 2172–2177. <a href="https://doi.org/10.1093/aje/kwab092">10.1093/aje/kwab092</a>.</div>
</div>
<div id="ref-rAFELHQ7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">7. </div><div class="csl-right-inline">Parker, H. (2017). Opinionated analysis development. PeerJ Preprints, e3210v1. <a href="https://doi.org/10.7287/peerj.preprints.3210v1">10.7287/peerj.preprints.3210v1</a>.</div>
</div>
<div id="ref-Qh7xTLwz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">8. </div><div class="csl-right-inline">Beaulieu-Jones, B.K., and Greene, C.S. (2017). Reproducibility of computational workflows is automated using continuous analysis. Nat Biotechnol <em>35</em>, 342–346. <a href="https://doi.org/10.1038/nbt.3780">10.1038/nbt.3780</a>.</div>
</div>
<div id="ref-1CcAUn3Lu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">9. </div><div class="csl-right-inline">Piwowar, H.A., Day, R.S., and Fridsma, D.B. (2007). Sharing Detailed Research Data Is Associated with Increased Citation Rate. PLoS ONE <em>2</em>, e308. <a href="https://doi.org/10.1371/journal.pone.0000308">10.1371/journal.pone.0000308</a>.</div>
</div>
<div id="ref-Vj0v0GeK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">10. </div><div class="csl-right-inline">Cadwallader, L., Papin, J.A., Mac Gabhann, F., and Kirk, R. (2021). Collaborating with our community to increase code sharing. PLoS Comput Biol <em>17</em>, e1008867. <a href="https://doi.org/10.1371/journal.pcbi.1008867">10.1371/journal.pcbi.1008867</a>.</div>
</div>
<div id="ref-XYoocEyx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">11. </div><div class="csl-right-inline">Dang, M.T., Gonzalez, M.V., Gaonkar, K.S., Rathi, K.S., Young, P., Arif, S., Zhai, L., Alam, Z., Devalaraja, S., To, T.K.J., et al. (2021). Macrophages in SHH subgroup medulloblastoma display dynamic heterogeneity that varies with treatment modality. Cell Reports <em>34</em>, 108917. <a href="https://doi.org/10.1016/j.celrep.2021.108917">10.1016/j.celrep.2021.108917</a>.</div>
</div>
<div id="ref-9mO4Op62" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">12. </div><div class="csl-right-inline">Kline, C., Jain, P., Kilburn, L., Bonner, E.R., Gupta, N., Crawford, J.R., Banerjee, A., Packer, R.J., Villanueva-Meyer, J., Luks, T., et al. (2022). Upfront Biology-Guided Therapy in Diffuse Intrinsic Pontine Glioma: Therapeutic, Molecular, and Biomarker Outcomes from PNOC003. Clinical Cancer Research <em>28</em>, 3965–3978. <a href="https://doi.org/10.1158/1078-0432.ccr-22-0803">10.1158/1078-0432.ccr-22-0803</a>.</div>
</div>
<div id="ref-kIFuKyuK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">13. </div><div class="csl-right-inline">Foster, J.B., Griffin, C., Rokita, J.L., Stern, A., Brimley, C., Rathi, K., Lane, M.V., Buongervino, S.N., Smith, T., Madsen, P.J., et al. (2022). Development of GPC2-directed chimeric antigen receptors using mRNA for pediatric brain tumors. J Immunother Cancer <em>10</em>, e004450. <a href="https://doi.org/10.1136/jitc-2021-004450">10.1136/jitc-2021-004450</a>.</div>
</div>
<div id="ref-lNLcNX6n" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">14. </div><div class="csl-right-inline">Stundon, J.L., Ijaz, H., Gaonkar, K.S., Kaufman, R.S., Jin, R., Karras, A., Vaksman, Z., Kim, J., Corbett, R.J., Lueder, M.R., et al. (2022). Alternative lengthening of telomeres (ALT) in pediatric high-grade gliomas can occur without ATRX mutation and is enriched in patients with pathogenic germline mismatch repair (MMR) variants. Neuro-Oncology. <a href="https://doi.org/10.1093/neuonc/noac278">10.1093/neuonc/noac278</a>.</div>
</div>
<div id="ref-15d8OT6MC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">15. </div><div class="csl-right-inline">RACE Act Poised to Advance Pediatric Cancer Research (2020). Cancer Discovery <em>10</em>, 1434–1434. <a href="https://doi.org/10.1158/2159-8290.cd-nb2020-081">10.1158/2159-8290.cd-nb2020-081</a>.</div>
</div>
<div id="ref-DwCzMoBJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">16. </div><div class="csl-right-inline">Ijaz, H., Koptyra, M., Gaonkar, K.S., Rokita, J.L., Baubet, V.P., Tauhid, L., Zhu, Y., Brown, M., Lopez, G., Zhang, B., et al. (2019). Pediatric high-grade glioma resources from the Children’s Brain Tumor Tissue Consortium. Neuro-Oncology <em>22</em>, 163–165. <a href="https://doi.org/10.1093/neuonc/noz192">10.1093/neuonc/noz192</a>.</div>
</div>
<div id="ref-BLNV1DbL" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">17. </div><div class="csl-right-inline">Mueller, S., Jain, P., Liang, W.S., Kilburn, L., Kline, C., Gupta, N., Panditharatna, E., Magge, S.N., Zhang, B., Zhu, Y., et al. (2019). A pilot precision medicine trial for children with diffuse intrinsic pontine glioma—PNOC003: A report from the Pacific Pediatric Neuro‐Oncology Consortium. Int. J. Cancer. <a href="https://doi.org/10.1002/ijc.32258">10.1002/ijc.32258</a>.</div>
</div>
<div id="ref-7sgrMadR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">18. </div><div class="csl-right-inline">Louis, D.N., Perry, A., Reifenberger, G., von Deimling, A., Figarella-Branger, D., Cavenee, W.K., Ohgaki, H., Wiestler, O.D., Kleihues, P., and Ellison, D.W. (2016). The 2016 World Health Organization Classification of Tumors of the Central Nervous System: a summary. Acta Neuropathol <em>131</em>, 803–820. <a href="https://doi.org/10.1007/s00401-016-1545-1">10.1007/s00401-016-1545-1</a>.</div>
</div>
<div id="ref-YuJbg3zO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">19. </div><div class="csl-right-inline">Himmelstein, D.S., Rubinetti, V., Slochower, D.R., Hu, D., Malladi, V.S., Greene, C.S., and Gitter, A. (2019). Open collaborative writing with Manubot. PLoS Comput Biol <em>15</em>, e1007128. <a href="https://doi.org/10.1371/journal.pcbi.1007128">10.1371/journal.pcbi.1007128</a>.</div>
</div>
<div id="ref-1CJTmbOqr" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">20. </div><div class="csl-right-inline">Merkel, D. (2014). <a href="https://dl.acm.org/doi/10.5555/2600239.2600241">Docker: lightweight Linux containers for consistent development and deployment</a>. Linux Journal <em>2014</em>, 2:2.</div>
</div>
<div id="ref-NazMcQpM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">21. </div><div class="csl-right-inline">Boettiger, C., and Eddelbuettel, D. (2017). An Introduction to Rocker: Docker Containers for R. <a href="https://doi.org/10.48550/arXiv.1710.03675">10.48550/arXiv.1710.03675</a>.</div>
</div>
<div id="ref-14TDMX5t7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">22. </div><div class="csl-right-inline">Louis, D.N., Ohgaki, H., Wiestler, O.D., Cavenee, W.K., Burger, P.C., Jouvet, A., Scheithauer, B.W., and Kleihues, P. (2007). The 2007 WHO Classification of Tumours of the Central Nervous System. Acta Neuropathol <em>114</em>, 97–109. <a href="https://doi.org/10.1007/s00401-007-0243-4">10.1007/s00401-007-0243-4</a>.</div>
</div>
<div id="ref-ULPByXgG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">23. </div><div class="csl-right-inline">Ryall, S., Zapotocky, M., Fukuoka, K., Nobre, L., Guerreiro Stucklin, A., Bennett, J., Siddaway, R., Li, C., Pajovic, S., Arnoldo, A., et al. (2020). Integrated Molecular and Clinical Analysis of 1,000 Pediatric Low-Grade Gliomas. Cancer Cell <em>37</em>, 569–583.e5. <a href="https://doi.org/10.1016/j.ccell.2020.03.011">10.1016/j.ccell.2020.03.011</a>.</div>
</div>
<div id="ref-12DKhuCiy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">24. </div><div class="csl-right-inline">Rathi, K.S., Arif, S., Koptyra, M., Naqvi, A.S., Taylor, D.M., Storm, P.B., Resnick, A.C., Rokita, J.L., and Raman, P. (2020). A transcriptome-based classifier to determine molecular subtypes in medulloblastoma. PLoS Comput Biol <em>16</em>, e1008263. <a href="https://doi.org/10.1371/journal.pcbi.1008263">10.1371/journal.pcbi.1008263</a>.</div>
</div>
<div id="ref-RmzXK4Cw" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">25. </div><div class="csl-right-inline">Gendoo, D.M.A., and Haibe-Kains, B. (2016). MM2S: personalized diagnosis of medulloblastoma patients and model systems. Source Code Biol Med <em>11</em>. <a href="https://doi.org/10.1186/s13029-016-0053-y">10.1186/s13029-016-0053-y</a>.</div>
</div>
<div id="ref-IamVuiNU" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">26. </div><div class="csl-right-inline">Ryall, S., Tabori, U., and Hawkins, C. (2020). Pediatric low-grade glioma in the era of molecular diagnostics. acta neuropathol commun <em>8</em>. <a href="https://doi.org/10.1186/s40478-020-00902-z">10.1186/s40478-020-00902-z</a>.</div>
</div>
<div id="ref-1Geb70JK9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">27. </div><div class="csl-right-inline">Lambo, S., von Hoff, K., Korshunov, A., Pfister, S.M., and Kool, M. (2020). ETMR: a tumor entity in its infancy. Acta Neuropathol <em>140</em>, 249–266. <a href="https://doi.org/10.1007/s00401-020-02182-2">10.1007/s00401-020-02182-2</a>.</div>
</div>
<div id="ref-1EzqPr2Ef" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">28. </div><div class="csl-right-inline">Richardson, S., Hill, R.M., Kui, C., Lindsey, J.C., Grabovksa, Y., Keeling, C., Pease, L., Bashton, M., Crosier, S., Vinci, M., et al. (2021). Emergence and maintenance of actionable genetic drivers at medulloblastoma relapse. Neuro-Oncology <em>24</em>, 153–165. <a href="https://doi.org/10.1093/neuonc/noab178">10.1093/neuonc/noab178</a>.</div>
</div>
<div id="ref-j11VxJIm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">29. </div><div class="csl-right-inline">Łastowska, M., Trubicka, J., Sobocińska, A., Wojtas, B., Niemira, M., Szałkowska, A., Krętowski, A., Karkucińska-Więckowska, A., Kaleta, M., Ejmont, M., et al. (2020). Molecular identification of CNS NB-FOXR2, CNS EFT-CIC, CNS HGNET-MN1 and CNS HGNET-BCOR pediatric brain tumors using tumor-specific signature genes. acta neuropathol commun <em>8</em>. <a href="https://doi.org/10.1186/s40478-020-00984-9">10.1186/s40478-020-00984-9</a>.</div>
</div>
<div id="ref-UuzX8B9C" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">30. </div><div class="csl-right-inline">Northcott, P.A., Buchhalter, I., Morrissy, A.S., Hovestadt, V., Weischenfeldt, J., Ehrenberger, T., Gröbner, S., Segura-Wang, M., Zichner, T., Rudneva, V.A., et al. (2017). The whole-genome landscape of medulloblastoma subtypes. Nature <em>547</em>, 311–317. <a href="https://doi.org/10.1038/nature22973">10.1038/nature22973</a>.</div>
</div>
<div id="ref-UWkO4UwG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">31. </div><div class="csl-right-inline">Haase, S., Garcia-Fabiani, M.B., Carney, S., Altshuler, D., Núñez, F.J., Méndez, F.M., Núñez, F., Lowenstein, P.R., and Castro, M.G. (2018). Mutant ATRX: uncovering a new therapeutic target for glioma. Expert Opinion on Therapeutic Targets <em>22</em>, 599–613. <a href="https://doi.org/10.1080/14728222.2018.1487953">10.1080/14728222.2018.1487953</a>.</div>
</div>
<div id="ref-vd5Uh9gY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">32. </div><div class="csl-right-inline">Mackay, A., Burford, A., Carvalho, D., Izquierdo, E., Fazal-Salom, J., Taylor, K.R., Bjerke, L., Clarke, M., Vinci, M., Nandhabalan, M., et al. (2017). Integrated Molecular Meta-Analysis of 1,000 Pediatric High-Grade and Diffuse Intrinsic Pontine Glioma. Cancer Cell <em>32</em>, 520–537.e5. <a href="https://doi.org/10.1016/j.ccell.2017.08.017">10.1016/j.ccell.2017.08.017</a>.</div>
</div>
<div id="ref-KU8wVb8T" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">33. </div><div class="csl-right-inline">Pratt, D., Quezado, M., Abdullaev, Z., Hawes, D., Yang, F., Garton, H.J.L., Judkins, A.R., Mody, R., Chinnaiyan, A., Aldape, K., et al. (2020). Diffuse intrinsic pontine glioma-like tumor with EZHIP expression and molecular features of PFA ependymoma. acta neuropathol commun <em>8</em>. <a href="https://doi.org/10.1186/s40478-020-00905-w">10.1186/s40478-020-00905-w</a>.</div>
</div>
<div id="ref-eDtbAO3v" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">34. </div><div class="csl-right-inline">Surrey, L.F., Jain, P., Zhang, B., Straka, J., Zhao, X., Harding, B.N., Resnick, A.C., Storm, P.B., Buccoliero, A.M., Genitori, L., et al. (2019). Genomic Analysis of Dysembryoplastic Neuroepithelial Tumor Spectrum Reveals a Diversity of Molecular Alterations Dysregulating the MAPK and PI3K/mTOR Pathways. Journal of Neuropathology &amp;amp; Experimental Neurology <em>78</em>, 1100–1111. <a href="https://doi.org/10.1093/jnen/nlz101">10.1093/jnen/nlz101</a>.</div>
</div>
<div id="ref-MsKFoF50" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">35. </div><div class="csl-right-inline">Sievers, P., Sill, M., Schrimpf, D., Stichel, D., Reuss, D.E., Sturm, D., Hench, J., Frank, S., Krskova, L., Vicha, A., et al. (2020). A subset of pediatric-type thalamic gliomas share a distinct DNA methylation profile, H3K27me3 loss and frequent alteration of &lt;i&gt;EGFR&lt;/i&gt;. Neuro-Oncology <em>23</em>, 34–43. <a href="https://doi.org/10.1093/neuonc/noaa251">10.1093/neuonc/noaa251</a>.</div>
</div>
<div id="ref-zrYR4Swt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">36. </div><div class="csl-right-inline">(2014). The genomic landscape of diffuse intrinsic pontine glioma and pediatric non-brainstem high-grade glioma. Nat Genet <em>46</em>, 444–450. <a href="https://doi.org/10.1038/ng.2938">10.1038/ng.2938</a>.</div>
</div>
<div id="ref-ZINVcBaH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">37. </div><div class="csl-right-inline">Pfaff, E., Remke, M., Sturm, D., Benner, A., Witt, H., Milde, T., von Bueren, A.O., Wittmann, A., Schöttler, A., Jorch, N., et al. (2010). &lt;i&gt;TP53&lt;/i&gt; Mutation Is Frequently Associated With &lt;i&gt;CTNNB1&lt;/i&gt; Mutation or &lt;i&gt;MYCN&lt;/i&gt; Amplification and Is Compatible With Long-Term Survival in Medulloblastoma. JCO <em>28</em>, 5188–5196. <a href="https://doi.org/10.1200/jco.2010.31.1670">10.1200/jco.2010.31.1670</a>.</div>
</div>
<div id="ref-ynFGewrs" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">38. </div><div class="csl-right-inline">Northcott, P.A., Jones, D.T.W., Kool, M., Robinson, G.W., Gilbertson, R.J., Cho, Y.-J., Pomeroy, S.L., Korshunov, A., Lichter, P., Taylor, M.D., et al. (2012). Medulloblastomics: the end of the beginning. Nat Rev Cancer <em>12</em>, 818–834. <a href="https://doi.org/10.1038/nrc3410">10.1038/nrc3410</a>.</div>
</div>
<div id="ref-5k4QKyTN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">39. </div><div class="csl-right-inline">Lucas, C.-H.G., Gupta, R., Doo, P., Lee, J.C., Cadwell, C.R., Ramani, B., Hofmann, J.W., Sloan, E.A., Kleinschmidt-DeMasters, B.K., Lee, H.S., et al. (2020). Comprehensive analysis of diverse low-grade neuroepithelial tumors with FGFR1 alterations reveals a distinct molecular signature of rosette-forming glioneuronal tumor. acta neuropathol commun <em>8</em>. <a href="https://doi.org/10.1186/s40478-020-01027-z">10.1186/s40478-020-01027-z</a>.</div>
</div>
<div id="ref-1CpnMVmIN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">40. </div><div class="csl-right-inline">Wu, G., Diaz, A.K., Paugh, B.S., Rankin, S.L., Ju, B., Li, Y., Zhu, X., Qu, C., Chen, X., Zhang, J., et al. (2014). The genomic landscape of diffuse intrinsic pontine glioma and pediatric non-brainstem high-grade glioma. Nature Genetics <em>46</em>, 444–450. <a href="https://doi.org/10.1038/ng.2938">10.1038/ng.2938</a>.</div>
</div>
<div id="ref-Cc8jt6lX" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">41. </div><div class="csl-right-inline">Degasperi, A., Amarante, T.D., Czarnecki, J., Shooter, S., Zou, X., Glodzik, D., Morganella, S., Nanda, A.S., Badja, C., Koh, G., et al. (2020). A practical framework and online tool for mutational signature analyses show intertissue variation and driver dependencies. Nat Cancer <em>1</em>, 249–263. <a href="https://doi.org/10.1038/s43018-020-0027-5">10.1038/s43018-020-0027-5</a>.</div>
</div>
<div id="ref-OaMJfIBC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">42. </div><div class="csl-right-inline">Wojciechowicz, K., Cantelli, E., Van Gerwen, B., Plug, M., Van Der Wal, A., Delzenne-Goette, E., Song, J.-Y., De Vries, S., Dekker, M., and Riele, H.T. (2014). Temozolomide Increases the Number of Mismatch Repair–Deficient Intestinal Crypts and Accelerates Tumorigenesis in a Mouse Model of Lynch Syndrome. Gastroenterology <em>147</em>, 1064–1072.e5. <a href="https://doi.org/10.1053/j.gastro.2014.07.052">10.1053/j.gastro.2014.07.052</a>.</div>
</div>
<div id="ref-X6oQhIf8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">43. </div><div class="csl-right-inline">Knijnenburg, T.A., Wang, L., Zimmermann, M.T., Chambwe, N., Gao, G.F., Cherniack, A.D., Fan, H., Shen, H., Way, G.P., Greene, C.S., et al. (2018). Genomic and Molecular Landscape of DNA Damage Repair Deficiency across The Cancer Genome Atlas. Cell Reports <em>23</em>, 239–254.e6. <a href="https://doi.org/10.1016/j.celrep.2018.03.076">10.1016/j.celrep.2018.03.076</a>.</div>
</div>
<div id="ref-14asizOhY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">44. </div><div class="csl-right-inline">Dittmer, D., Pati, S., Zambetti, G., Chu, S., Teresky, A.K., Moore, M., Finlay, C., and Levine, A.J. (1993). Gain of function mutations in p53. Nat Genet <em>4</em>, 42–46. <a href="https://doi.org/10.1038/ng0593-42">10.1038/ng0593-42</a>.</div>
</div>
<div id="ref-SDwYl8uA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">45. </div><div class="csl-right-inline">Rokita, J.L., Rathi, K.S., Cardenas, M.F., Upton, K.A., Jayaseelan, J., Cross, K.L., Pfeil, J., Egolf, L.E., Way, G.P., Farrel, A., et al. (2019). Genomic Profiling of Childhood Tumor Patient-Derived Xenograft Models to Enable Rational Clinical Trial Design. Cell Reports <em>29</em>, 1675–1689.e9. <a href="https://doi.org/10.1016/j.celrep.2019.09.071">10.1016/j.celrep.2019.09.071</a>.</div>
</div>
<div id="ref-RnBZNbdx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">46. </div><div class="csl-right-inline">Noureen, N., Wu, S., Lv, Y., Yang, J., Alfred Yung, W.K., Gelfond, J., Wang, X., Koul, D., Ludlow, A., and Zheng, S. (2021). Integrated analysis of telomerase enzymatic activity unravels an association with cancer stemness and proliferation. Nat Commun <em>12</em>. <a href="https://doi.org/10.1038/s41467-020-20474-9">10.1038/s41467-020-20474-9</a>.</div>
</div>
<div id="ref-15inmTb6T" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">47. </div><div class="csl-right-inline">Artandi, S.E., and DePinho, R.A. (2009). Telomeres and telomerase in cancer. Carcinogenesis <em>31</em>, 9–18. <a href="https://doi.org/10.1093/carcin/bgp268">10.1093/carcin/bgp268</a>.</div>
</div>
<div id="ref-TJj4hOO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">48. </div><div class="csl-right-inline">Ceja-Rangel, H.A., Sánchez-Suárez, P., Castellanos-Juárez, E., Peñaroja-Flores, R., Arenas-Aranda, D.J., Gariglio, P., and Benítez-Bribiesca, L. (2016). Shorter telomeres and high telomerase activity correlate with a highly aggressive phenotype in breast cancer cell lines. Tumor Biol. <em>37</em>, 11917–11926. <a href="https://doi.org/10.1007/s13277-016-5045-7">10.1007/s13277-016-5045-7</a>.</div>
</div>
<div id="ref-wXW7qGqv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">49. </div><div class="csl-right-inline">Oh, B.-K., Kim, H., Park, Y.N., Yoo, J.E., Choi, J., Kim, K.-S., Lee, J.J., and Park, C. (2008). High telomerase activity and long telomeres in advanced hepatocellular carcinomas with poor prognosis. Laboratory Investigation <em>88</em>, 144–152. <a href="https://doi.org/10.1038/labinvest.3700710">10.1038/labinvest.3700710</a>.</div>
</div>
<div id="ref-dhzTU0Xu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">50. </div><div class="csl-right-inline">Kulić, A., Plavetić, N.D., Gamulin, S., Jakić-Razumović, J., Vrbanec, D., and Sirotković-Skerlev, M. (2016). Telomerase activity in breast cancer patients: association with poor prognosis and more aggressive phenotype. Med Oncol <em>33</em>. <a href="https://doi.org/10.1007/s12032-016-0736-x">10.1007/s12032-016-0736-x</a>.</div>
</div>
<div id="ref-5I97mcf1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">51. </div><div class="csl-right-inline">Wong, V.C.H., Morrison, A., Tabori, U., and Hawkins, C.E. (2010). Telomerase Inhibition as a Novel Therapy for Pediatric Ependymoma. Brain Pathology <em>20</em>, 780–786. <a href="https://doi.org/10.1111/j.1750-3639.2010.00372.x">10.1111/j.1750-3639.2010.00372.x</a>.</div>
</div>
<div id="ref-td36JJNb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">52. </div><div class="csl-right-inline">Pich, O., Muiños, F., Lolkema, M.P., Steeghs, N., Gonzalez-Perez, A., and Lopez-Bigas, N. (2019). The mutational footprints of cancer therapies. Nat Genet <em>51</em>, 1732–1740. <a href="https://doi.org/10.1038/s41588-019-0525-5">10.1038/s41588-019-0525-5</a>.</div>
</div>
<div id="ref-1DeOIIYxr" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">53. </div><div class="csl-right-inline">Aronson, M., Colas, C., Shuen, A., Hampel, H., Foulkes, W.D., Baris Feldman, H., Goldberg, Y., Muleris, M., Wolfe Schneider, K., McGee, R.B., et al. (2021). Diagnostic criteria for constitutional mismatch repair deficiency (CMMRD): recommendations from the international consensus working group. J Med Genet <em>59</em>, 318–327. <a href="https://doi.org/10.1136/jmedgenet-2020-107627">10.1136/jmedgenet-2020-107627</a>.</div>
</div>
<div id="ref-1CbmKZRjS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">54. </div><div class="csl-right-inline">Vuong, H.G., Le, H.T., Ngo, T.N.M., Fung, K.-M., Battiste, J.D., McNall-Knapp, R., and Dunn, I.F. (2021). H3K27M-mutant diffuse midline gliomas should be further molecularly stratified: an integrated analysis of 669 patients. J Neurooncol <em>155</em>, 225–234. <a href="https://doi.org/10.1007/s11060-021-03890-9">10.1007/s11060-021-03890-9</a>.</div>
</div>
<div id="ref-MeiYPRi0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">55. </div><div class="csl-right-inline">Lewis, P.W., Müller, M.M., Koletsky, M.S., Cordero, F., Lin, S., Banaszynski, L.A., Garcia, B.A., Muir, T.W., Becher, O.J., and Allis, C.D. (2013). Inhibition of PRC2 Activity by a Gain-of-Function H3 Mutation Found in Pediatric Glioblastoma. Science <em>340</em>, 857–861. <a href="https://doi.org/10.1126/science.1232245">10.1126/science.1232245</a>.</div>
</div>
<div id="ref-mzBdyDm0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">56. </div><div class="csl-right-inline">Hutter, S., Bolin, S., Weishaupt, H., and Swartling, F. (2017). Modeling and Targeting MYC Genes in Childhood Brain Tumors. Genes <em>8</em>, 107. <a href="https://doi.org/10.3390/genes8040107">10.3390/genes8040107</a>.</div>
</div>
<div id="ref-887hkpnS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">57. </div><div class="csl-right-inline">Hannan, C.J., Lewis, D., O’Leary, C., Donofrio, C.A., Evans, D.G., Roncaroli, F., Brough, D., King, A.T., Coope, D., and Pathmanaban, O.N. (2020). The inflammatory microenvironment in vestibular schwannoma. Neuro-Oncology Advances <em>2</em>. <a href="https://doi.org/10.1093/noajnl/vdaa023">10.1093/noajnl/vdaa023</a>.</div>
</div>
<div id="ref-meH98mKZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">58. </div><div class="csl-right-inline">Petralia, F., Tignor, N., Reva, B., Koptyra, M., Chowdhury, S., Rykunov, D., Krek, A., Ma, W., Zhu, Y., Ji, J., et al. (2020). Integrated Proteogenomic Characterization across Major Histological Types of Pediatric Brain Cancer. Cell <em>183</em>, 1962–1985.e31. <a href="https://doi.org/10.1016/j.cell.2020.10.044">10.1016/j.cell.2020.10.044</a>.</div>
</div>
<div id="ref-1B1SO39W1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">59. </div><div class="csl-right-inline">Lin, G.L., Nagaraja, S., Filbin, M.G., Suvà, M.L., Vogel, H., and Monje, M. (2018). Non-inflammatory tumor microenvironment of diffuse intrinsic pontine glioma. acta neuropathol commun <em>6</em>. <a href="https://doi.org/10.1186/s40478-018-0553-x">10.1186/s40478-018-0553-x</a>.</div>
</div>
<div id="ref-RzxfvDRe" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">60. </div><div class="csl-right-inline">Ross, J.L., Velazquez Vega, J., Plant, A., MacDonald, T.J., Becher, O.J., and Hambardzumyan, D. (2021). Tumour immune landscape of paediatric high-grade gliomas. Brain <em>144</em>, 2594–2609. <a href="https://doi.org/10.1093/brain/awab155">10.1093/brain/awab155</a>.</div>
</div>
<div id="ref-IQLWvQPD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">61. </div><div class="csl-right-inline">Martin, A.M., Nirschl, C.J., Polanczyk, M.J., Bell, W.R., Nirschl, T.R., Harris-Bookman, S., Phallen, J., Hicks, J., Martinez, D., Ogurtsova, A., et al. (2018). PD-L1 expression in medulloblastoma: an evaluation by subgroup. Oncotarget <em>9</em>, 19177–19191. <a href="https://doi.org/10.18632/oncotarget.24951">10.18632/oncotarget.24951</a>.</div>
</div>
<div id="ref-TNNRIa7y" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">62. </div><div class="csl-right-inline">Bockmayr, M., Mohme, M., Klauschen, F., Winkler, B., Budczies, J., Rutkowski, S., and Schüller, U. (2018). Subgroup-specific immune and stromal microenvironment in medulloblastoma. OncoImmunology <em>7</em>, e1462430. <a href="https://doi.org/10.1080/2162402x.2018.1462430">10.1080/2162402x.2018.1462430</a>.</div>
</div>
<div id="ref-ic0BrsYM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">63. </div><div class="csl-right-inline">Duchemann, B., Naigeon, M., Auclin, E., Ferrara, R., Cassard, L., Jouniaux, J.-M., Boselli, L., Grivel, J., Desnoyer, A., Danlos, F.-X., et al. (2022). CD8&lt;sup&gt;+&lt;/sup&gt;PD-1&lt;sup&gt;+&lt;/sup&gt; to CD4&lt;sup&gt;+&lt;/sup&gt;PD-1&lt;sup&gt;+&lt;/sup&gt; ratio (PERLS) is associated with prognosis of patients with advanced NSCLC treated with PD-(L)1 blockers. J Immunother Cancer <em>10</em>, e004012. <a href="https://doi.org/10.1136/jitc-2021-004012">10.1136/jitc-2021-004012</a>.</div>
</div>
<div id="ref-MAFXjHGX" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">64. </div><div class="csl-right-inline">Shindo, G., Endo, T., Onda, M., Goto, S., Miyamoto, Y., and Kaneko, T. (2013). Is the CD4/CD8 Ratio an Effective Indicator for Clinical Estimation of Adoptive Immunotherapy for Cancer Treatment? JCT <em>04</em>, 1382–1390. <a href="https://doi.org/10.4236/jct.2013.48164">10.4236/jct.2013.48164</a>.</div>
</div>
<div id="ref-sOb4Wh4q" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">65. </div><div class="csl-right-inline">Yuza, K., Nagahashi, M., Watanabe, S., Takabe, K., and Wakai, T. (2017). Hypermutation and microsatellite instability in gastrointestinal cancers. Oncotarget <em>8</em>, 112103–112115. <a href="https://doi.org/10.18632/oncotarget.22783">10.18632/oncotarget.22783</a>.</div>
</div>
<div id="ref-11x4p1w23" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">66. </div><div class="csl-right-inline">Bass, A.J., Thorsson, V., Shmulevich, I., Reynolds, S.M., Miller, M., Bernard, B., Hinoue, T., Laird, P.W., Curtis, C., Shen, H., et al. (2014). Comprehensive molecular characterization of gastric adenocarcinoma. Nature <em>513</em>, 202–209. <a href="https://doi.org/10.1038/nature13480">10.1038/nature13480</a>.</div>
</div>
<div id="ref-nCoKt2Vg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">67. </div><div class="csl-right-inline">Sharma, J., Bonfield, C.M., Singhal, A., Hukin, J., and Steinbok, P. (2015). Intracystic interferon-α treatment leads to neurotoxicity in craniopharyngioma: case report. PED <em>16</em>, 301–304. <a href="https://doi.org/10.3171/2015.2.peds14656">10.3171/2015.2.peds14656</a>.</div>
</div>
<div id="ref-rfGUdNhB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">68. </div><div class="csl-right-inline">Mohammed, K.E.A., Mike, K.R.A., and Parkes, J. (2013). Unexpected brain atrophy following administration of intratumoral interferon alpha-2b for cystic craniopharyngioma: A case report. IJCRI <em>4</em>, 719. <a href="https://doi.org/10.5348/ijcri-2013-12-419-cr-13">10.5348/ijcri-2013-12-419-cr-13</a>.</div>
</div>
<div id="ref-i9VLAGYQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">69. </div><div class="csl-right-inline">Coy, S., Rashid, R., Lin, J.-R., Du, Z., Donson, A.M., Hankinson, T.C., Foreman, N.K., Manley, P.E., Kieran, M.W., Reardon, D.A., et al. (2018). Multiplexed immunofluorescence reveals potential PD-1/PD-L1 pathway vulnerabilities in craniopharyngioma. Neuro-Oncology <em>20</em>, 1101–1112. <a href="https://doi.org/10.1093/neuonc/noy035">10.1093/neuonc/noy035</a>.</div>
</div>
<div id="ref-15tuSDJZh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">70. </div><div class="csl-right-inline">Apps, J.R., Carreno, G., Gonzalez-Meljem, J.M., Haston, S., Guiho, R., Cooper, J.E., Manshaei, S., Jani, N., Hölsken, A., Pettorini, B., et al. (2018). Tumour compartment transcriptomics demonstrates the activation of inflammatory and odontogenic programmes in human adamantinomatous craniopharyngioma and identifies the MAPK/ERK pathway as a novel therapeutic target. Acta Neuropathol <em>135</em>, 757–777. <a href="https://doi.org/10.1007/s00401-018-1830-2">10.1007/s00401-018-1830-2</a>.</div>
</div>
<div id="ref-oyGGylFL" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">71. </div><div class="csl-right-inline">Grob, S., Mirsky, D.M., Donson, A.M., Dahl, N., Foreman, N.K., Hoffman, L.M., Hankinson, T.C., and Mulcahy Levy, J.M. (2019). Targeting IL-6 Is a Potential Treatment for Primary Cystic Craniopharyngioma. Front. Oncol. <em>9</em>. <a href="https://doi.org/10.3389/fonc.2019.00791">10.3389/fonc.2019.00791</a>.</div>
</div>
<div id="ref-10LfPTs0c" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">72. </div><div class="csl-right-inline">Gaonkar, K.S., Marini, F., Rathi, K.S., Jain, P., Zhu, Y., Chimicles, N.A., Brown, M.A., Naqvi, A.S., Zhang, B., Storm, P.B., et al. (2020). annoFuse: an R Package to annotate, prioritize, and interactively explore putative oncogenic RNA fusions. BMC Bioinformatics <em>21</em>. <a href="https://doi.org/10.1186/s12859-020-03922-7">10.1186/s12859-020-03922-7</a>.</div>
</div>
<div id="ref-eQQpBw1T" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">73. </div><div class="csl-right-inline">University of California, San Francisco (2023). <a href="https://clinicaltrials.gov/ct2/show/NCT03739372">A Pilot Trial Testing the Clinical Benefit of Using Molecular Profiling to Determine an Individualized Treatment Plan in Children and Young Adults With High Grade Glioma (Excluding Diffuse Intrinsic Pontine Glioma)</a> (clinicaltrials.gov).</div>
</div>
<div id="ref-8YqNPLmu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">74. </div><div class="csl-right-inline">University of California, San Francisco (2023). <a href="https://clinicaltrials.gov/ct2/show/NCT05057702">A Pilot Trial of Real Time Drug Screening and Genomic Testing to Determine an Individualized Treatment Plan in Children and Young Adults With Relapsed Medulloblastoma</a> (clinicaltrials.gov).</div>
</div>
<div id="ref-THjIzNXb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">75. </div><div class="csl-right-inline">Open Pediatric Brain Tumor Atlas, C.B.T.N., Pediatric Neuro Oncology Consortium (2022). Open Pediatric Brain Tumor Atlas. <a href="https://doi.org/10.24370/openpbta">10.24370/openpbta</a>.</div>
</div>
<div id="ref-Wql2Djro" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">76. </div><div class="csl-right-inline">Li, H. (2013). Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM. <a href="https://doi.org/10.48550/arXiv.1303.3997">10.48550/arXiv.1303.3997</a>.</div>
</div>
<div id="ref-NCr4QkOg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">77. </div><div class="csl-right-inline">DePristo, M.A., Banks, E., Poplin, R., Garimella, K.V., Maguire, J.R., Hartl, C., Philippakis, A.A., del Angel, G., Rivas, M.A., Hanna, M., et al. (2011). A framework for variation discovery and genotyping using next-generation DNA sequencing data. Nat Genet <em>43</em>, 491–498. <a href="https://doi.org/10.1038/ng.806">10.1038/ng.806</a>.</div>
</div>
<div id="ref-IzUbOv92" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">78. </div><div class="csl-right-inline">Faust, G.G., and Hall, I.M. (2014). &lt;i&gt;SAMBLASTER&lt;/i&gt;: fast duplicate marking and structural variant read extraction. Bioinformatics <em>30</em>, 2503–2505. <a href="https://doi.org/10.1093/bioinformatics/btu314">10.1093/bioinformatics/btu314</a>.</div>
</div>
<div id="ref-PcJwlZj3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">79. </div><div class="csl-right-inline">Tarasov, A., Vilella, A.J., Cuppen, E., Nijman, I.J., and Prins, P. (2015). Sambamba: fast processing of NGS alignment formats. Bioinformatics <em>31</em>, 2032–2034. <a href="https://doi.org/10.1093/bioinformatics/btv098">10.1093/bioinformatics/btv098</a>.</div>
</div>
<div id="ref-14nSa8zB9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">80. </div><div class="csl-right-inline">McKenna, A., Hanna, M., Banks, E., Sivachenko, A., Cibulskis, K., Kernytsky, A., Garimella, K., Altshuler, D., Gabriel, S., Daly, M., et al. (2010). The Genome Analysis Toolkit: A MapReduce framework for analyzing next-generation DNA sequencing data. Genome Res. <em>20</em>, 1297–1303. <a href="https://doi.org/10.1101/gr.107524.110">10.1101/gr.107524.110</a>.</div>
</div>
<div id="ref-36OiSGiq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">81. </div><div class="csl-right-inline">Poplin, R., Ruano-Rubio, V., DePristo, M.A., Fennell, T.J., Carneiro, M.O., Auwera, G.A.V. der, Kling, D.E., Gauthier, L.D., Levy-Moonshine, A., Roazen, D., et al. (2018). Scaling accurate genetic variant discovery to tens of thousands of samples. bioRxiv, 201178. <a href="https://doi.org/10.1101/201178">10.1101/201178</a>.</div>
</div>
<div id="ref-A4FXW005" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">82. </div><div class="csl-right-inline">Lee, S., Lee, S., Ouellette, S., Park, W.-Y., Lee, E.A., and Park, P.J. (2017). NGSCheckMate: software for validating sample identity in next-generation sequencing studies within and across data types. Nucleic Acids Research <em>45</em>, e103–e103. <a href="https://doi.org/10.1093/nar/gkx193">10.1093/nar/gkx193</a>.</div>
</div>
<div id="ref-unXWCD9A" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">83. </div><div class="csl-right-inline">DeLuca, D.S., Levin, J.Z., Sivachenko, A., Fennell, T., Nazaire, M.-D., Williams, C., Reich, M., Winckler, W., and Getz, G. (2012). RNA-SeQC: RNA-seq metrics for quality control and process optimization. Bioinformatics <em>28</em>, 1530–1532. <a href="https://doi.org/10.1093/bioinformatics/bts196">10.1093/bioinformatics/bts196</a>.</div>
</div>
<div id="ref-Fv382g9q" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">84. </div><div class="csl-right-inline">Wang, K., Li, M., and Hakonarson, H. (2010). ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data. Nucleic Acids Research <em>38</em>, e164–e164. <a href="https://doi.org/10.1093/nar/gkq603">10.1093/nar/gkq603</a>.</div>
</div>
<div id="ref-16jeSVhEx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">85. </div><div class="csl-right-inline">Landrum, M.J., Lee, J.M., Riley, G.R., Jang, W., Rubinstein, W.S., Church, D.M., and Maglott, D.R. (2013). ClinVar: public archive of relationships among sequence variation and human phenotype. Nucl. Acids Res. <em>42</em>, D980–D985. <a href="https://doi.org/10.1093/nar/gkt1113">10.1093/nar/gkt1113</a>.</div>
</div>
<div id="ref-vOcCb1WG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">86. </div><div class="csl-right-inline">Cingolani, P., Patel, V.M., Coon, M., Nguyen, T., Land, S.J., Ruden, D.M., and Lu, X. (2012). Using Drosophila melanogaster as a Model for Genotoxic Chemical Mutational Studies with a New Program, SnpSift. Front. Gene. <em>3</em>. <a href="https://doi.org/10.3389/fgene.2012.00035">10.3389/fgene.2012.00035</a>.</div>
</div>
<div id="ref-REfkDUtE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">87. </div><div class="csl-right-inline">Kim, S., Scheffler, K., Halpern, A.L., Bekritsky, M.A., Noh, E., Källberg, M., Chen, X., Kim, Y., Beyter, D., Krusche, P., et al. (2018). Strelka2: fast and accurate calling of germline and somatic variants. Nat Methods <em>15</em>, 591–594. <a href="https://doi.org/10.1038/s41592-018-0051-x">10.1038/s41592-018-0051-x</a>.</div>
</div>
<div id="ref-9ZgwE4bQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">88. </div><div class="csl-right-inline">Benjamin, D., Sato, T., Cibulskis, K., Getz, G., Stewart, C., and Lichtenstein, L. (2019). Calling Somatic SNVs and Indels with Mutect2. bioRxiv, 861054. <a href="https://doi.org/10.1101/861054">10.1101/861054</a>.</div>
</div>
<div id="ref-V6KdWVYi" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">89. </div><div class="csl-right-inline">Narzisi, G., Corvelo, A., Arora, K., Bergmann, E.A., Shah, M., Musunuri, R., Emde, A.-K., Robine, N., Vacic, V., and Zody, M.C. (2018). Genome-wide somatic variant calling using localized colored de Bruijn graphs. Commun Biol <em>1</em>. <a href="https://doi.org/10.1038/s42003-018-0023-9">10.1038/s42003-018-0023-9</a>.</div>
</div>
<div id="ref-trQRR8fs" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">90. </div><div class="csl-right-inline">Lai, Z., Markovets, A., Ahdesmaki, M., Chapman, B., Hofmann, O., McEwen, R., Johnson, J., Dougherty, B., Barrett, J.C., and Dry, J.R. (2016). VarDict: a novel and versatile variant caller for next-generation sequencing in cancer research. Nucleic Acids Res <em>44</em>, e108–e108. <a href="https://doi.org/10.1093/nar/gkw227">10.1093/nar/gkw227</a>.</div>
</div>
<div id="ref-gG4xEIkq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">91. </div><div class="csl-right-inline">Arora, K., Shah, M., Johnson, M., Sanghvi, R., Shelton, J., Nagulapalli, K., Oschwald, D.M., Zody, M.C., Germer, S., Jobanputra, V., et al. (2019). Deep whole-genome sequencing of 3 cancer cell lines on 2 sequencing platforms. Sci Rep <em>9</em>. <a href="https://doi.org/10.1038/s41598-019-55636-3">10.1038/s41598-019-55636-3</a>.</div>
</div>
<div id="ref-p1f5DxRQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">92. </div><div class="csl-right-inline">McLaren, W., Gil, L., Hunt, S.E., Riat, H.S., Ritchie, G.R.S., Thormann, A., Flicek, P., and Cunningham, F. (2016). The Ensembl Variant Effect Predictor. Genome Biol <em>17</em>. <a href="https://doi.org/10.1186/s13059-016-0974-4">10.1186/s13059-016-0974-4</a>.</div>
</div>
<div id="ref-55r9iVwk" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">93. </div><div class="csl-right-inline">Karczewski, K.J., Francioli, L.C., Tiao, G., Cummings, B.B., Alföldi, J., Wang, Q., Collins, R.L., Laricchia, K.M., Ganna, A., Birnbaum, D.P., et al. (2020). The mutational constraint spectrum quantified from variation in 141,456 humans. Nature <em>581</em>, 434–443. <a href="https://doi.org/10.1038/s41586-020-2308-7">10.1038/s41586-020-2308-7</a>.</div>
</div>
<div id="ref-59DKOl44" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">94. </div><div class="csl-right-inline">Zvereva, M., Pisarev, E., Hosen, I., Kisil, O., Matskeplishvili, S., Kubareva, E., Kamalov, D., Tivtikyan, A., Manel, A., Vian, E., et al. (2020). Activating Telomerase TERT Promoter Mutations and Their Application for the Detection of Bladder Cancer. IJMS <em>21</em>, 6034. <a href="https://doi.org/10.3390/ijms21176034">10.3390/ijms21176034</a>.</div>
</div>
<div id="ref-ZQ0L3o1q" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">95. </div><div class="csl-right-inline">Boeva, V., Popova, T., Bleakley, K., Chiche, P., Cappo, J., Schleiermacher, G., Janoueix-Lerosey, I., Delattre, O., and Barillot, E. (2011). Control-FREEC: a tool for assessing copy number and allelic content using next-generation sequencing data. Bioinformatics <em>28</em>, 423–425. <a href="https://doi.org/10.1093/bioinformatics/btr670">10.1093/bioinformatics/btr670</a>.</div>
</div>
<div id="ref-1F3i4BvCt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">96. </div><div class="csl-right-inline">Boeva, V., Zinovyev, A., Bleakley, K., Vert, J.-P., Janoueix-Lerosey, I., Delattre, O., and Barillot, E. (2010). Control-free calling of copy number alterations in deep-sequencing data using GC-content normalization. Bioinformatics <em>27</em>, 268–269. <a href="https://doi.org/10.1093/bioinformatics/btq635">10.1093/bioinformatics/btq635</a>.</div>
</div>
<div id="ref-UTxRcYIQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">97. </div><div class="csl-right-inline">Talevich, E., Shain, A.H., Botton, T., and Bastian, B.C. (2016). CNVkit: Genome-Wide Copy Number Detection and Visualization from Targeted DNA Sequencing. PLoS Comput Biol <em>12</em>, e1004873. <a href="https://doi.org/10.1371/journal.pcbi.1004873">10.1371/journal.pcbi.1004873</a>.</div>
</div>
<div id="ref-V4PckbrH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">98. </div><div class="csl-right-inline">Oesper, L., Satas, G., and Raphael, B.J. (2014). Quantifying tumor heterogeneity in whole-genome and whole-exome sequencing data. Bioinformatics <em>30</em>, 3532–3540. <a href="https://doi.org/10.1093/bioinformatics/btu651">10.1093/bioinformatics/btu651</a>.</div>
</div>
<div id="ref-rDV1hhfF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">99. </div><div class="csl-right-inline">Mermel, C.H., Schumacher, S.E., Hill, B., Meyerson, M.L., Beroukhim, R., and Getz, G. (2011). GISTIC2.0 facilitates sensitive and confident localization of the targets of focal somatic copy-number alteration in human cancers. Genome Biol <em>12</em>. <a href="https://doi.org/10.1186/gb-2011-12-4-r41">10.1186/gb-2011-12-4-r41</a>.</div>
</div>
<div id="ref-kTn1PIj5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">100. </div><div class="csl-right-inline">Chen, X., Schulz-Trieglaff, O., Shaw, R., Barnes, B., Schlesinger, F., Källberg, M., Cox, A.J., Kruglyak, S., and Saunders, C.T. (2015). Manta: rapid detection of structural variants and indels for germline and cancer sequencing applications. Bioinformatics <em>32</em>, 1220–1222. <a href="https://doi.org/10.1093/bioinformatics/btv710">10.1093/bioinformatics/btv710</a>.</div>
</div>
<div id="ref-7Ull5aU5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">101. </div><div class="csl-right-inline">Geoffroy, V., Herenger, Y., Kress, A., Stoetzel, C., Piton, A., Dollfus, H., and Muller, J. (2018). AnnotSV: an integrated tool for structural variations annotation. Bioinformatics <em>34</em>, 3572–3574. <a href="https://doi.org/10.1093/bioinformatics/bty304">10.1093/bioinformatics/bty304</a>.</div>
</div>
<div id="ref-tTu8Ds9Z" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">102. </div><div class="csl-right-inline">Dobin, A., Davis, C.A., Schlesinger, F., Drenkow, J., Zaleski, C., Jha, S., Batut, P., Chaisson, M., and Gingeras, T.R. (2012). STAR: ultrafast universal RNA-seq aligner. Bioinformatics <em>29</em>, 15–21. <a href="https://doi.org/10.1093/bioinformatics/bts635">10.1093/bioinformatics/bts635</a>.</div>
</div>
<div id="ref-Dh2n1EV3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">103. </div><div class="csl-right-inline">Li, B., and Dewey, C.N. (2011). RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome. BMC Bioinformatics <em>12</em>. <a href="https://doi.org/10.1186/1471-2105-12-323">10.1186/1471-2105-12-323</a>.</div>
</div>
<div id="ref-DuQpY5dg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">104. </div><div class="csl-right-inline">Uhrig, S., Ellermann, J., Walther, T., Burkhardt, P., Fröhlich, M., Hutter, B., Toprak, U.H., Neumann, O., Stenzinger, A., Scholl, C., et al. (2021). Accurate and efficient detection of gene fusions from RNA sequencing data. Genome Res. <em>31</em>, 448–460. <a href="https://doi.org/10.1101/gr.257246.119">10.1101/gr.257246.119</a>.</div>
</div>
<div id="ref-dwCkClIW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">105. </div><div class="csl-right-inline">Haas, B.J., Dobin, A., Stransky, N., Li, B., Yang, X., Tickle, T., Bankapur, A., Ganote, C., Doak, T.G., Pochet, N., et al. (2017). STAR-Fusion: Fast and Accurate Fusion Transcript Detection from RNA-Seq. bioRxiv, 120295. <a href="https://doi.org/10.1101/120295">10.1101/120295</a>.</div>
</div>
<div id="ref-83Prx3Xc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">106. </div><div class="csl-right-inline">Mayakonda, A., Lin, D.-C., Assenov, Y., Plass, C., and Koeffler, H.P. (2018). Maftools: efficient and comprehensive analysis of somatic variants in cancer. Genome Res. <em>28</em>, 1747–1756. <a href="https://doi.org/10.1101/gr.239244.118">10.1101/gr.239244.118</a>.</div>
</div>
<div id="ref-1HWiAHnIw" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">107. </div><div class="csl-right-inline">Quinlan, A.R., and Hall, I.M. (2010). BEDTools: a flexible suite of utilities for comparing genomic features. Bioinformatics <em>26</em>, 841–842. <a href="https://doi.org/10.1093/bioinformatics/btq033">10.1093/bioinformatics/btq033</a>.</div>
</div>
<div id="ref-1DWggwY7R" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">108. </div><div class="csl-right-inline">Meyer, L.R., Zweig, A.S., Hinrichs, A.S., Karolchik, D., Kuhn, R.M., Wong, M., Sloan, C.A., Rosenbloom, K.R., Roe, G., Rhead, B., et al. (2012). The UCSC Genome Browser database: extensions and updates 2013. Nucleic Acids Research <em>41</em>, D64–D69. <a href="https://doi.org/10.1093/nar/gks1048">10.1093/nar/gks1048</a>.</div>
</div>
<div id="ref-14d6Jzwus" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">109. </div><div class="csl-right-inline">Lawrence, M., Huber, W., Pagès, H., Aboyoun, P., Carlson, M., Gentleman, R., Morgan, M.T., and Carey, V.J. (2013). Software for Computing and Annotating Genomic Ranges. PLoS Comput Biol <em>9</em>, e1003118. <a href="https://doi.org/10.1371/journal.pcbi.1003118">10.1371/journal.pcbi.1003118</a>.</div>
</div>
<div id="ref-oESNuFI5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">110. </div><div class="csl-right-inline">Gu, Z., Eils, R., and Schlesner, M. (2016). Complex heatmaps reveal patterns and correlations in multidimensional genomic data. Bioinformatics <em>32</em>, 2847–2849. <a href="https://doi.org/10.1093/bioinformatics/btw313">10.1093/bioinformatics/btw313</a>.</div>
</div>
<div id="ref-17wG7AMwM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">111. </div><div class="csl-right-inline">Cortés-Ciriano, I., Lee, J.J.-K., Xi, R., Jain, D., Jung, Y.L., Yang, L., Gordenin, D., Klimczak, L.J., Zhang, C.-Z., Pellman, D.S., et al. (2020). Comprehensive analysis of chromothripsis in 2,658 human cancers using whole-genome sequencing. Nat Genet <em>52</em>, 331–341. <a href="https://doi.org/10.1038/s41588-019-0576-7">10.1038/s41588-019-0576-7</a>.</div>
</div>
<div id="ref-RLEOBWiR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">112. </div><div class="csl-right-inline">Finotello, F., Mayer, C., Plattner, C., Laschober, G., Rieder, D., Hackl, H., Krogsdam, A., Loncova, Z., Posch, W., Wilflingseder, D., et al. (2019). Molecular and pharmacological modulators of the tumor immune contexture revealed by deconvolution of RNA-seq data. Genome Med <em>11</em>. <a href="https://doi.org/10.1186/s13073-019-0638-6">10.1186/s13073-019-0638-6</a>.</div>
</div>
<div id="ref-1F5zJEMaD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">113. </div><div class="csl-right-inline">Hänzelmann, S., Castelo, R., and Guinney, J. (2013). GSVA: gene set variation analysis for microarray and RNA-Seq data. BMC Bioinformatics <em>14</em>. <a href="https://doi.org/10.1186/1471-2105-14-7">10.1186/1471-2105-14-7</a>.</div>
</div>
<div id="ref-1CbVoEpNJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">114. </div><div class="csl-right-inline">Liberzon, A., Birger, C., Thorvaldsdóttir, H., Ghandi, M., Mesirov, Jill P., and Tamayo, P. (2015). The Molecular Signatures Database Hallmark Gene Set Collection. Cell Systems <em>1</em>, 417–425. <a href="https://doi.org/10.1016/j.cels.2015.12.004">10.1016/j.cels.2015.12.004</a>.</div>
</div>
<div id="ref-jErrUFsf" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">115. </div><div class="csl-right-inline">McInnes, L., Healy, J., and Melville, J. (2020). UMAP: Uniform Manifold Approximation and Projection for Dimension Reduction. <a href="https://doi.org/10.48550/arXiv.1802.03426">10.48550/arXiv.1802.03426</a>.</div>
</div>
<div id="ref-9vS8HBL6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">116. </div><div class="csl-right-inline">Lambert, S.A., Jolma, A., Campitelli, L.F., Das, P.K., Yin, Y., Albu, M., Chen, X., Taipale, J., Hughes, T.R., and Weirauch, M.T. (2018). The Human Transcription Factors. Cell <em>172</em>, 650–665. <a href="https://doi.org/10.1016/j.cell.2018.01.029">10.1016/j.cell.2018.01.029</a>.</div>
</div>
<div id="ref-jLWV5IWB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">117. </div><div class="csl-right-inline">Ramkissoon, L.A., Horowitz, P.M., Craig, J.M., Ramkissoon, S.H., Rich, B.E., Schumacher, S.E., McKenna, A., Lawrence, M.S., Bergthold, G., Brastianos, P.K., et al. (2013). Genomic analysis of diffuse pediatric low-grade gliomas identifies recurrent oncogenic truncating rearrangements in the transcription factor
&lt;i&gt;MYBL1&lt;/i&gt;. Proc. Natl. Acad. Sci. U.S.A. <em>110</em>, 8188–8193. <a href="https://doi.org/10.1073/pnas.1300252110">10.1073/pnas.1300252110</a>.</div>
</div>
<div id="ref-4wYR62jK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">118. </div><div class="csl-right-inline">Northcott, P.A., Shih, D.J.H., Peacock, J., Garzia, L., Sorana Morrissy, A., Zichner, T., Stütz, A.M., Korshunov, A., Reimand, J., Schumacher, S.E., et al. (2012). Subgroup-specific structural variation across 1,000 medulloblastoma genomes. Nature <em>488</em>, 49–56. <a href="https://doi.org/10.1038/nature11327">10.1038/nature11327</a>.</div>
</div>
<div id="ref-kfmK8vm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">119. </div><div class="csl-right-inline">Sturm, D., Orr, Brent A., Toprak, Umut H., Hovestadt, V., Jones, David T.W., Capper, D., Sill, M., Buchhalter, I., Northcott, Paul A., Leis, I., et al. (2016). New Brain Tumor Entities Emerge from Molecular Classification of CNS-PNETs. Cell <em>164</em>, 1060–1072. <a href="https://doi.org/10.1016/j.cell.2016.01.015">10.1016/j.cell.2016.01.015</a>.</div>
</div>
<div id="ref-5ueZBnsJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">120. </div><div class="csl-right-inline">Kleinman, C.L., Gerges, N., Papillon-Cavanagh, S., Sin-Chan, P., Pramatarova, A., Quang, D.-A.K., Adoue, V., Busche, S., Caron, M., Djambazian, H., et al. (2013). Fusion of TTYH1 with the C19MC microRNA cluster drives expression of a brain-specific DNMT3B isoform in the embryonal brain tumor ETMR. Nat Genet <em>46</em>, 39–44. <a href="https://doi.org/10.1038/ng.2849">10.1038/ng.2849</a>.</div>
</div>
<div id="ref-ASmwGlFp" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">121. </div><div class="csl-right-inline">Valentijn, L.J., Koster, J., Zwijnenburg, D.A., Hasselt, N.E., van Sluis, P., Volckmann, R., van Noesel, M.M., George, R.E., Tytgat, G.A.M., Molenaar, J.J., et al. (2015). TERT rearrangements are frequent in neuroblastoma and identify aggressive tumors. Nat Genet <em>47</em>, 1411–1414. <a href="https://doi.org/10.1038/ng.3438">10.1038/ng.3438</a>.</div>
</div>
<div id="ref-YfG9EVSk" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">122. </div><div class="csl-right-inline">Cobrinik, D., Ostrovnaya, I., Hassimi, M., Tickoo, S.K., Cheung, I.Y., and Cheung, N.-K.V. (2013). Recurrent pre-existing and acquired DNA copy number alterations, including focal&lt;i&gt;TERT&lt;/i&gt;gains, in neuroblastoma central nervous system metastases. Genes Chromosomes Cancer <em>52</em>, 1150–1166. <a href="https://doi.org/10.1002/gcc.22110">10.1002/gcc.22110</a>.</div>
</div>
<div id="ref-lWMOs28t" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">123. </div><div class="csl-right-inline">Karlsson, J., Lilljebjörn, H., Holmquist Mengelbier, L., Valind, A., Rissler, M., Øra, I., Fioretos, T., and Gisselsson, D. (2015). Activation of human telomerase reverse transcriptase through gene fusion in clear cell sarcoma of the kidney. Cancer Letters <em>357</em>, 498–501. <a href="https://doi.org/10.1016/j.canlet.2014.11.057">10.1016/j.canlet.2014.11.057</a>.</div>
</div>
<div id="ref-1B3tdZcAl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">124. </div><div class="csl-right-inline">Karsy, M., Guan, J., Cohen, A.L., Jensen, R.L., and Colman, H. (2017). New Molecular Considerations for Glioma: IDH, ATRX, BRAF, TERT, H3 K27M. Curr Neurol Neurosci Rep <em>17</em>. <a href="https://doi.org/10.1007/s11910-017-0722-5">10.1007/s11910-017-0722-5</a>.</div>
</div>
<div id="ref-1foRpfch" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">125. </div><div class="csl-right-inline">Bandopadhayay, P., Ramkissoon, L.A., Jain, P., Bergthold, G., Wala, J., Zeid, R., Schumacher, S.E., Urbanski, L., O'Rourke, R., Gibson, W.J., et al. (2016). MYB-QKI rearrangements in angiocentric glioma drive tumorigenicity through a tripartite mechanism. Nat Genet <em>48</em>, 273–282. <a href="https://doi.org/10.1038/ng.3500">10.1038/ng.3500</a>.</div>
</div>
<div id="ref-CeB34T1V" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">126. </div><div class="csl-right-inline">Johann, P.D., Erkek, S., Zapatka, M., Kerl, K., Buchhalter, I., Hovestadt, V., Jones, D.T.W., Sturm, D., Hermann, C., Segura Wang, M., et al. (2016). Atypical Teratoid/Rhabdoid Tumors Are Comprised of Three Epigenetic Subgroups with Distinct Enhancer Landscapes. Cancer Cell <em>29</em>, 379–393. <a href="https://doi.org/10.1016/j.ccell.2016.02.001">10.1016/j.ccell.2016.02.001</a>.</div>
</div>
<div id="ref-1FGkQ9Wa7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">127. </div><div class="csl-right-inline">Mong, E.F., Yang, Y., Akat, K.M., Canfield, J., VanWye, J., Lockhart, J., Tsibris, J.C.M., Schatz, F., Lockwood, C.J., Tuschl, T., et al. (2020). Chromosome 19 microRNA cluster enhances cell reprogramming by inhibiting epithelial-to-mesenchymal transition. Sci Rep <em>10</em>. <a href="https://doi.org/10.1038/s41598-020-59812-8">10.1038/s41598-020-59812-8</a>.</div>
</div>
<div id="ref-6F5vK3sT" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">128. </div><div class="csl-right-inline">Louis, D.N., Perry, A., Wesseling, P., Brat, D.J., Cree, I.A., Figarella-Branger, D., Hawkins, C., Ng, H.K., Pfister, S.M., Reifenberger, G., et al. (2021). The 2021 WHO Classification of Tumors of the Central Nervous System: a summary. Neuro-Oncology <em>23</em>, 1231–1251. <a href="https://doi.org/10.1093/neuonc/noab106">10.1093/neuonc/noab106</a>.</div>
</div>
<div id="ref-f7P6U50v" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">129. </div><div class="csl-right-inline">Pajtler, Kristian W., Witt, H., Sill, M., Jones, David T.W., Hovestadt, V., Kratochwil, F., Wani, K., Tatevossian, R., Punchihewa, C., Johann, P., et al. (2015). Molecular Classification of Ependymal Tumors across All CNS Compartments, Histopathological Grades, and Age Groups. Cancer Cell <em>27</em>, 728–743. <a href="https://doi.org/10.1016/j.ccell.2015.04.002">10.1016/j.ccell.2015.04.002</a>.</div>
</div>
<div id="ref-DGVkAk1W" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">130. </div><div class="csl-right-inline">Parker, M., Mohankumar, K.M., Punchihewa, C., Weinlich, R., Dalton, J.D., Li, Y., Lee, R., Tatevossian, R.G., Phoenix, T.N., Thiruvenkatam, R., et al. (2014). C11orf95–RELA fusions drive oncogenic NF-κB signalling in ependymoma. Nature <em>506</em>, 451–455. <a href="https://doi.org/10.1038/nature13109">10.1038/nature13109</a>.</div>
</div>
<div id="ref-ovoneiG3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">131. </div><div class="csl-right-inline">Bi, W.L., Greenwald, N.F., Abedalthagafi, M., Wala, J., Gibson, W.J., Agarwalla, P.K., Horowitz, P., Schumacher, S.E., Esaulova, E., Mei, Y., et al. (2017). Genomic landscape of high-grade meningiomas. npj Genomic Med <em>2</em>. <a href="https://doi.org/10.1038/s41525-017-0014-7">10.1038/s41525-017-0014-7</a>.</div>
</div>
<div id="ref-uDYz8KKh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">132. </div><div class="csl-right-inline">Youngblood, M.W., Duran, D., Montejo, J.D., Li, C., Omay, S.B., Özduman, K., Sheth, A.H., Zhao, A.Y., Tyrtova, E., Miyagishima, D.F., et al. (2020). Correlations between genomic subgroup and clinical features in a cohort of more than 3000 meningiomas. Journal of Neurosurgery <em>133</em>, 1345–1354. <a href="https://doi.org/10.3171/2019.8.jns191266">10.3171/2019.8.jns191266</a>.</div>
</div>
<div id="ref-107ZjoLxM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">133. </div><div class="csl-right-inline">Qaddoumi, I., Orisme, W., Wen, J., Santiago, T., Gupta, K., Dalton, J.D., Tang, B., Haupfear, K., Punchihewa, C., Easton, J., et al. (2016). Genetic alterations in uncommon low-grade neuroepithelial tumors: BRAF, FGFR1, and MYB mutations occur at high frequency and align with morphology. Acta Neuropathol <em>131</em>, 833–845. <a href="https://doi.org/10.1007/s00401-016-1539-z">10.1007/s00401-016-1539-z</a>.</div>
</div>
<div id="ref-F73HShWn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">134. </div><div class="csl-right-inline">Thomas, C., Soschinski, P., Zwaig, M., Oikonomopoulos, S., Okonechnikov, K., Pajtler, K.W., Sill, M., Schweizer, L., Koch, A., Neumann, J., et al. (2020). The genetic landscape of choroid plexus tumors in children and adults. Neuro-Oncology <em>23</em>, 650–660. <a href="https://doi.org/10.1093/neuonc/noaa267">10.1093/neuonc/noaa267</a>.</div>
</div>
<div id="ref-BBuQnfgj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">135. </div><div class="csl-right-inline">Krooks, J., Minkov, M., and Weatherall, A.G. (2018). Langerhans cell histiocytosis in children. Journal of the American Academy of Dermatology <em>78</em>, 1035–1044. <a href="https://doi.org/10.1016/j.jaad.2017.05.059">10.1016/j.jaad.2017.05.059</a>.</div>
</div>
<div id="ref-m7hgzrvh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">136. </div><div class="csl-right-inline">Antin, C., Tauziède-Espariat, A., Debily, M.-A., Castel, D., Grill, J., Pagès, M., Ayrault, O., Chrétien, F., Gareton, A., Andreiuolo, F., et al. (2020). EZHIP is a specific diagnostic biomarker for posterior fossa ependymomas, group PFA and diffuse midline gliomas H3-WT with EZHIP overexpression. acta neuropathol commun <em>8</em>. <a href="https://doi.org/10.1186/s40478-020-01056-8">10.1186/s40478-020-01056-8</a>.</div>
</div>
<div id="ref-KhxTOfIb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">137. </div><div class="csl-right-inline">Rosenthal, R., McGranahan, N., Herrero, J., Taylor, B.S., and Swanton, C. (2016). deconstructSigs: delineating mutational processes in single tumors distinguishes DNA repair deficiencies and patterns of carcinoma evolution. Genome Biol <em>17</em>. <a href="https://doi.org/10.1186/s13059-016-0893-4">10.1186/s13059-016-0893-4</a>.</div>
</div>
<div id="ref-6bmmWiU5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">138. </div><div class="csl-right-inline">Burel‐Vandenbos, F., Pierron, G., Thomas, C., Reynaud, S., Gregoire, V., Duhil de Benaze, G., Croze, S., Chivoret, N., Honavar, M., Figarella‐Branger, D., et al. (2020). A polyphenotypic malignant paediatric brain tumour presenting a
&lt;i&gt;MN1‐PATZ1&lt;/i&gt;
fusion, no epigenetic similarities with CNS High‐Grade Neuroepithelial Tumour with
&lt;i&gt;MN1&lt;/i&gt;
Alteration (CNS HGNET‐MN1) and related to
&lt;i&gt;PATZ1&lt;/i&gt;
‐fused sarcomas. Neuropathol Appl Neurobiol <em>46</em>, 506–509. <a href="https://doi.org/10.1111/nan.12626">10.1111/nan.12626</a>.</div>
</div>
<div id="ref-t40FioEk" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">139. </div><div class="csl-right-inline">Rao, S., Rajeswarie, R.T., Chickabasaviah Yasha, T., Nandeesh, B.N., Arivazhagan, A., and Santosh, V. (2017). LIN28A, a sensitive immunohistochemical marker for Embryonal Tumor with Multilayered Rosettes (ETMR), is also positive in a subset of Atypical Teratoid/Rhabdoid Tumor (AT/RT). Childs Nerv Syst <em>33</em>, 1953–1959. <a href="https://doi.org/10.1007/s00381-017-3551-6">10.1007/s00381-017-3551-6</a>.</div>
</div>
<div id="ref-WBoOEAK1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">140. </div><div class="csl-right-inline">PDQ® Pediatric Treatment Editorial Board <a href="https://www.ncbi.nlm.nih.gov/pubmed/26389418">Childhood Medulloblastoma and Other Central Nervous System Embryonal Tumors Treatment (PDQ®): Health Professional Version</a>. In PDQ Cancer Information Summaries (National Cancer Institute).</div>
</div>
<div id="ref-Cibjccyk" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">141. </div><div class="csl-right-inline">Miele, E., De Vito, R., Ciolfi, A., Pedace, L., Russo, I., De Pasquale, M.D., Di Giannatale, A., Crocoli, A., Angelis, B.D., Tartaglia, M., et al. (2020). DNA Methylation Profiling for Diagnosing Undifferentiated Sarcoma with Capicua Transcriptional Receptor (CIC) Alterations. IJMS <em>21</em>, 1818. <a href="https://doi.org/10.3390/ijms21051818">10.3390/ijms21051818</a>.</div>
</div>
<div id="ref-7UxjGOxW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">142. </div><div class="csl-right-inline">Korshunov, A., Ryzhova, M., Jones, D.T.W., Northcott, P.A., van Sluis, P., Volckmann, R., Koster, J., Versteeg, R., Cowdrey, C., Perry, A., et al. (2012). LIN28A immunoreactivity is a potent diagnostic marker of embryonal tumor with multilayered rosettes (ETMR). Acta Neuropathol <em>124</em>, 875–881. <a href="https://doi.org/10.1007/s00401-012-1068-3">10.1007/s00401-012-1068-3</a>.</div>
</div>
<div id="ref-DEDtXX7K" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">143. </div><div class="csl-right-inline">Rustagi, N., Hampton, O.A., Li, J., Xi, L., Gibbs, R.A., Plon, S.E., Kimmel, M., and Wheeler, D.A. (2016). ITD assembler: an algorithm for internal tandem duplication discovery from short-read sequencing data. BMC Bioinformatics <em>17</em>. <a href="https://doi.org/10.1186/s12859-016-1031-8">10.1186/s12859-016-1031-8</a>.</div>
</div>
<div id="ref-ueHlof7U" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">144. </div><div class="csl-right-inline">Mohila, C.A., Rauch, R.A., and Adesina, A.M. (2016). Central Neurocytoma and Extraventricular Neurocytoma. In Atlas of Pediatric Brain Tumors (Springer International Publishing), pp. 195–199. <a href="https://doi.org/10.1007/978-3-319-33432-5_20">10.1007/978-3-319-33432-5_20</a>.</div>
</div>
<div id="ref-WOSiJ6V2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">145. </div><div class="csl-right-inline">Crotty, T.B., Scheithauer, B.W., Young, W.F., Davis, D.H., Shaw, E.G., Miller, G.M., and Burger, P.C. (1995). Papillary craniopharyngioma: a clinicopathological study of 48 cases. Journal of Neurosurgery <em>83</em>, 206–214. <a href="https://doi.org/10.3171/jns.1995.83.2.0206">10.3171/jns.1995.83.2.0206</a>.</div>
</div>
<div id="ref-xnZIq0gr" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">146. </div><div class="csl-right-inline">Bunin, G.R., Surawicz, T.S., Witman, P.A., Preston-Martin, S., Davis, F., and Bruner, J.M. (1998). The descriptive epidemiology of craniopharyngioma. Journal of Neurosurgery <em>89</em>, 547–551. <a href="https://doi.org/10.3171/jns.1998.89.4.0547">10.3171/jns.1998.89.4.0547</a>.</div>
</div>
<div id="ref-10jkUywit" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">147. </div><div class="csl-right-inline">Chang, M.T., Bhattarai, T.S., Schram, A.M., Bielski, C.M., Donoghue, M.T.A., Jonsson, P., Chakravarty, D., Phillips, S., Kandoth, C., Penson, A., et al. (2018). Accelerating Discovery of Functional Mutant Alleles in Cancer. Cancer Discovery <em>8</em>, 174–183. <a href="https://doi.org/10.1158/2159-8290.cd-17-0321">10.1158/2159-8290.cd-17-0321</a>.</div>
</div>
<div id="ref-tfWVBF8o" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">148. </div><div class="csl-right-inline">Chang, M.T., Asthana, S., Gao, S.P., Lee, B.H., Chapman, J.S., Kandoth, C., Gao, J., Socci, N.D., Solit, D.B., Olshen, A.B., et al. (2015). Identifying recurrent mutations in cancer reveals widespread lineage diversity and mutational specificity. Nat Biotechnol <em>34</em>, 155–163. <a href="https://doi.org/10.1038/nbt.3391">10.1038/nbt.3391</a>.</div>
</div>
<div id="ref-KxZPoysm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">149. </div><div class="csl-right-inline">Harms, K.L., and Chen, X. (2006). The functional domains in p53 family proteins exhibit both common and distinct properties. Cell Death Differ <em>13</em>, 890–897. <a href="https://doi.org/10.1038/sj.cdd.4401904">10.1038/sj.cdd.4401904</a>.</div>
</div>
<div id="ref-JdxWaORm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">150. </div><div class="csl-right-inline">Guha, T., and Malkin, D. (2017). Inherited&lt;i&gt;TP53&lt;/i&gt;Mutations and the Li–Fraumeni Syndrome. Cold Spring Harb Perspect Med <em>7</em>, a026187. <a href="https://doi.org/10.1101/cshperspect.a026187">10.1101/cshperspect.a026187</a>.</div>
</div>
<div id="ref-6RHepB1T" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">151. </div><div class="csl-right-inline">Kaplan, E.L., and Meier, P. (1958). Nonparametric Estimation from Incomplete Observations. Journal of the American Statistical Association <em>53</em>, 457–481. <a href="https://doi.org/10.2307/2281868">10.2307/2281868</a>.</div>
</div>
<div id="ref-y0J19Ubd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">152. </div><div class="csl-right-inline">Cox, D.R. (1972). Regression Models and Life-Tables. Journal of the Royal Statistical Society: Series B (Methodological) <em>34</em>, 187–202. <a href="https://doi.org/10.1111/j.2517-6161.1972.tb00899.x">10.1111/j.2517-6161.1972.tb00899.x</a>.</div>
</div>
<div id="ref-t5yrhQPI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">153. </div><div class="csl-right-inline">Rokita, J.L., and Brown, M. (2022). d3b-center/OpenPBTA-workflows: Release v1.0.4 (Zenodo) <a href="https://doi.org/10.5281/zenodo.6968175">10.5281/zenodo.6968175</a>.</div>
</div>
<div id="ref-SZviDnAS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">154. </div><div class="csl-right-inline">Taroni, J., Savonen, C., Krutika Gaonkar, Stephanie, Chante Bethell, Rokita, J.L., Shapiro, J., Greene, C., Yuankun Zhu, Komal Rathi, et al. (2022). AlexsLemonade/OpenPBTA-analysis: Initial Submission (Zenodo) <a href="https://doi.org/10.5281/zenodo.7044567">10.5281/zenodo.7044567</a>.</div>
</div>
</div>
<!-- default theme -->

<style>
  /* import google fonts */
  @import url("https://fonts.googleapis.com/css?family=Open+Sans:400,600,700");
  @import url("https://fonts.googleapis.com/css?family=Source+Code+Pro");

  /* -------------------------------------------------- */
  /* global */
  /* -------------------------------------------------- */

  /* all elements */
  * {
    /* force sans-serif font unless specified otherwise */
    font-family: "Open Sans", "Helvetica", sans-serif;

    /* prevent text inflation on some mobile browsers */
    -webkit-text-size-adjust: none !important;
    -moz-text-size-adjust: none !important;
    -o-text-size-adjust: none !important;
    text-size-adjust: none !important;
  }

  @media only screen {
    /* "page" element */
    body {
      position: relative;
      box-sizing: border-box;
      font-size: 12pt;
      line-height: 1.5;
      max-width: 8.5in;
      margin: 20px auto;
      padding: 40px;
      border-radius: 5px;
      border: solid 1px #bdbdbd;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
      background: #ffffff;
    }
  }

  /* when on screen < 8.5in wide */
  @media only screen and (max-width: 8.5in) {
    /* "page" element */
    body {
      padding: 20px;
      margin: 0;
      border-radius: 0;
      border: none;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05) inset;
      background: none;
    }
  }

  /* -------------------------------------------------- */
  /* headings */
  /* -------------------------------------------------- */

  /* all headings */
  h1,
  h2,
  h3,
  h4,
  h5,
  h6 {
    margin: 20px 0;
    padding: 0;
    font-weight: bold;
  }

  /* biggest heading */
  h1 {
    margin: 40px 0;
    text-align: center;
  }

  /* second biggest heading */
  h2 {
    margin-top: 30px;
    padding-bottom: 5px;
    border-bottom: solid 1px #bdbdbd;
  }

  /* heading font sizes */
  h1 {
    font-size: 2em;
  }
  h2 {
    font-size: 1.5em;
  }
  h3 {
    font-size: 1.35em;
  }
  h4 {
    font-size: 1.25em;
  }
  h5 {
    font-size: 1.15em;
  }
  h6 {
    font-size: 1em;
  }

  /* -------------------------------------------------- */
  /* manuscript header */
  /* -------------------------------------------------- */

  /* manuscript title */
  header > h1 {
    margin: 0;
  }

  /* manuscript title caption text (ie "automatically generated on") */
  header + p {
    text-align: center;
    margin-top: 10px;
  }

  /* -------------------------------------------------- */
  /* text elements */
  /* -------------------------------------------------- */

  /* links */
  a {
    color: #2196f3;
    overflow-wrap: break-word;
  }

  /* superscripts and subscripts */
  sub,
  sup {
    /* prevent from affecting line height */
    line-height: 0;
  }

  /* unordered and ordered lists*/
  ul,
  ol {
    padding-left: 20px;
  }

  /* class for styling text semibold */
  .semibold {
    font-weight: 600;
  }

  /* class for styling elements horizontally left aligned */
  .left {
    display: block;
    text-align: left;
    margin-left: auto;
    margin-right: 0;
    justify-content: left;
  }

  /* class for styling elements horizontally centered */
  .center {
    display: block;
    text-align: center;
    margin-left: auto;
    margin-right: auto;
    justify-content: center;
  }

  /* class for styling elements horizontally right aligned */
  .right {
    display: block;
    text-align: right;
    margin-left: 0;
    margin-right: auto;
    justify-content: right;
  }

  /* -------------------------------------------------- */
  /* section elements */
  /* -------------------------------------------------- */

  /* horizontal divider line */
  hr {
    border: none;
    height: 1px;
    background: #bdbdbd;
  }

  /* paragraphs, horizontal dividers, figures, tables, code */
  p,
  hr,
  figure,
  table,
  pre {
    /* treat all as "paragraphs", with consistent vertical margins */
    margin-top: 20px;
    margin-bottom: 20px;
  }

  /* -------------------------------------------------- */
  /* figures */
  /* -------------------------------------------------- */

  /* figure */
  figure {
    max-width: 100%;
    margin-left: auto;
    margin-right: auto;
  }

  /* figure caption */
  figcaption {
    padding: 0;
    padding-top: 10px;
  }

  /* figure image element */
  figure > img,
  figure > svg {
    max-width: 100%;
    display: block;
    margin-left: auto;
    margin-right: auto;
  }

  /* figure auto-number */
  img + figcaption > span:first-of-type,
  svg + figcaption > span:first-of-type {
    font-weight: bold;
    margin-right: 5px;
  }

  /* -------------------------------------------------- */
  /* tables */
  /* -------------------------------------------------- */

  /* table */
  table {
    border-collapse: collapse;
    border-spacing: 0;
    width: 100%;
    margin-left: auto;
    margin-right: auto;
  }

  /* table cells */
  th,
  td {
    border: solid 1px #bdbdbd;
    padding: 10px;
    /* squash table if too wide for page by forcing line breaks */
    overflow-wrap: break-word;
    word-break: break-word;
  }

  /* header row and even rows */
  th,
  tr:nth-child(2n) {
    background-color: #fafafa;
  }

  /* odd rows */
  tr:nth-child(2n + 1) {
    background-color: #ffffff;
  }

  /* table caption */
  caption {
    text-align: left;
    padding: 0;
    padding-bottom: 10px;
  }

  /* table auto-number */
  table > caption > span:first-of-type {
    font-weight: bold;
    margin-right: 5px;
  }

  /* -------------------------------------------------- */
  /* code */
  /* -------------------------------------------------- */

  /* multi-line code block */
  pre {
    padding: 10px;
    background-color: #eeeeee;
    color: #000000;
    border-radius: 5px;
    break-inside: avoid;
    text-align: left;
  }

  /* inline code, ie code within normal text */
  :not(pre) > code {
    padding: 0 4px;
    background-color: #eeeeee;
    color: #000000;
    border-radius: 5px;
  }

  /* code text */
  /* apply all children, to reach syntax highlighting sub-elements */
  code,
  code * {
    /* force monospace font */
    font-family: "Source Code Pro", "Courier New", monospace;
  }

  /* -------------------------------------------------- */
  /* quotes */
  /* -------------------------------------------------- */

  /* quoted text */
  blockquote {
    margin: 0;
    padding: 0;
    border-left: 4px solid #bdbdbd;
    padding-left: 16px;
    break-inside: avoid;
  }

  /* -------------------------------------------------- */
  /* banners */
  /* -------------------------------------------------- */

  /* info banners */
  .banner {
    box-sizing: border-box;
    display: block;
    position: relative;
    width: 100%;
    margin-top: 20px;
    margin-bottom: 20px;
    padding: 20px;
    text-align: center;
  }

  /* paragraph in banner */
  .banner > p {
    margin: 0;
  }

  /* -------------------------------------------------- */
  /* highlight colors */
  /* -------------------------------------------------- */

  .white {
    background: #ffffff;
  }
  .lightgrey {
    background: #eeeeee;
  }
  .grey {
    background: #757575;
  }
  .darkgrey {
    background: #424242;
  }
  .black {
    background: #000000;
  }
  .lightred {
    background: #ffcdd2;
  }
  .lightyellow {
    background: #ffecb3;
  }
  .lightgreen {
    background: #dcedc8;
  }
  .lightblue {
    background: #e3f2fd;
  }
  .lightpurple {
    background: #f3e5f5;
  }
  .red {
    background: #f44336;
  }
  .orange {
    background: #ff9800;
  }
  .yellow {
    background: #ffeb3b;
  }
  .green {
    background: #4caf50;
  }
  .blue {
    background: #2196f3;
  }
  .purple {
    background: #9c27b0;
  }
  .white,
  .lightgrey,
  .lightred,
  .lightyellow,
  .lightgreen,
  .lightblue,
  .lightpurple,
  .orange,
  .yellow,
  .white a,
  .lightgrey a,
  .lightred a,
  .lightyellow a,
  .lightgreen a,
  .lightblue a,
  .lightpurple a,
  .orange a,
  .yellow a {
    color: #000000;
  }
  .grey,
  .darkgrey,
  .black,
  .red,
  .green,
  .blue,
  .purple,
  .grey a,
  .darkgrey a,
  .black a,
  .red a,
  .green a,
  .blue a,
  .purple a {
    color: #ffffff;
  }

  /* -------------------------------------------------- */
  /* buttons */
  /* -------------------------------------------------- */

  /* class for styling links like buttons */
  .button {
    display: inline-flex;
    justify-content: center;
    align-items: center;
    margin: 5px;
    padding: 10px 20px;
    font-size: 0.75em;
    font-weight: 600;
    text-transform: uppercase;
    text-decoration: none;
    letter-spacing: 1px;
    background: none;
    color: #2196f3;
    border: solid 1px #bdbdbd;
    border-radius: 5px;
  }

  /* buttons when hovered */
  .button:hover:not([disabled]),
  .icon_button:hover:not([disabled]) {
    cursor: pointer;
    background: #f5f5f5;
  }

  /* buttons when disabled */
  .button[disabled],
  .icon_button[disabled] {
    opacity: 0.35;
    pointer-events: none;
  }

  /* class for styling buttons containg only single icon */
  .icon_button {
    display: inline-flex;
    justify-content: center;
    align-items: center;
    text-decoration: none;
    margin: 0;
    padding: 0;
    background: none;
    border-radius: 5px;
    border: none;
    width: 20px;
    height: 20px;
    min-width: 20px;
    min-height: 20px;
  }

  /* icon button inner svg image */
  .icon_button > svg {
    height: 16px;
  }

  /* -------------------------------------------------- */
  /* icons */
  /* -------------------------------------------------- */

  /* class for styling icons inline with text */
  .inline_icon {
    height: 1em;
    position: relative;
    top: 0.125em;
  }

  /* -------------------------------------------------- */
  /* references */
  /* -------------------------------------------------- */

  .csl-entry {
    margin-top: 15px;
    margin-bottom: 15px;
  }

  /* -------------------------------------------------- */
  /* print control */
  /* -------------------------------------------------- */

  @media print {
    @page {
      /* suggested printing margin */
      margin: 0.5in;
    }

    /* document and "page" elements */
    html,
    body {
      margin: 0;
      padding: 0;
      width: 100%;
      height: 100%;
    }

    /* "page" element */
    body {
      font-size: 11pt !important;
      line-height: 1.35;
    }

    /* all headings */
    h1,
    h2,
    h3,
    h4,
    h5,
    h6 {
      margin: 15px 0;
    }

    /* figures and tables */
    figure,
    table {
      font-size: 0.85em;
    }

    /* table cells */
    th,
    td {
      padding: 5px;
    }

    /* shrink font awesome icons */
    i.fas,
    i.fab,
    i.far,
    i.fal {
      transform: scale(0.85);
    }

    /* decrease banner margins */
    .banner {
      margin-top: 15px;
      margin-bottom: 15px;
      padding: 15px;
    }

    /* class for centering an element vertically on its own page */
    .page_center {
      margin: auto;
      width: 100%;
      height: 100%;
      display: flex;
      align-items: center;
      vertical-align: middle;
      break-before: page;
      break-after: page;
    }

    /* always insert a page break before the element */
    .page_break_before {
      break-before: page;
    }

    /* always insert a page break after the element */
    .page_break_after {
      break-after: page;
    }

    /* avoid page break before the element */
    .page_break_before_avoid {
      break-before: avoid;
    }

    /* avoid page break after the element */
    .page_break_after_avoid {
      break-after: avoid;
    }

    /* avoid page break inside the element */
    .page_break_inside_avoid {
      break-inside: avoid;
    }
  }

  /* -------------------------------------------------- */
  /* override pandoc css quirks */
  /* -------------------------------------------------- */

  .sourceCode {
    /* prevent unsightly overflow in wide code blocks */
    overflow: auto !important;
  }

  div.sourceCode {
    /* prevent background fill on top-most code block  container */
    background: none !important;
  }

  .sourceCode * {
    /* force consistent line spacing */
    line-height: 1.5 !important;
  }

  div.sourceCode {
    /* style code block margins same as <pre> element */
    margin-top: 20px;
    margin-bottom: 20px;
  }

  /* -------------------------------------------------- */
  /* tablenos */
  /* -------------------------------------------------- */

  /* tablenos wrapper */
  .tablenos {
    width: 100%;
    margin: 20px 0;
  }

  .tablenos > table {
    /* move margins from table to table_wrapper to allow margin collapsing */
    margin: 0;
  }

  @media only screen {
    /* tablenos wrapper */
    .tablenos {
      /* show scrollbar on tables if necessary to prevent overflow */
      overflow-x: auto !important;
    }

    .tablenos th,
    .tablenos td {
      overflow-wrap: unset !important;
      word-break: unset !important;
    }

    /* table in wrapper */
    .tablenos table,
    .tablenos table * {
      /* don't break table words */
      overflow-wrap: normal !important;
    }
  }
</style>
<!-- 
    Plugin Core

    Functions needed for and shared across all first-party plugins.
-->

<script>
  // get element that is target of hash (from link element or url)
  function getHashTarget(link) {
    const hash = link ? link.hash : window.location.hash;
    const id = hash.slice(1);
    let target = document.querySelector(`[id="${id}"]`);
    if (!target) return;

    // if figure or table, modify target to get expected element
    if (id.indexOf("fig:") === 0) target = target.querySelector("figure");
    if (id.indexOf("tbl:") === 0) target = target.querySelector("table");

    return target;
  }

  // get position/dimensions of element or viewport
  function getRectInView(element) {
    let rect = {};
    rect.left = 0;
    rect.top = 0;
    rect.right = document.documentElement.clientWidth;
    rect.bottom = document.documentElement.clientHeight;
    let style = {};

    if (element instanceof HTMLElement) {
      rect = element.getBoundingClientRect();
      style = window.getComputedStyle(element);
    }

    const margin = {};
    margin.left = parseFloat(style.marginLeftWidth) || 0;
    margin.top = parseFloat(style.marginTopWidth) || 0;
    margin.right = parseFloat(style.marginRightWidth) || 0;
    margin.bottom = parseFloat(style.marginBottomWidth) || 0;

    const border = {};
    border.left = parseFloat(style.borderLeftWidth) || 0;
    border.top = parseFloat(style.borderTopWidth) || 0;
    border.right = parseFloat(style.borderRightWidth) || 0;
    border.bottom = parseFloat(style.borderBottomWidth) || 0;

    const newRect = {};
    newRect.left = rect.left + margin.left + border.left;
    newRect.top = rect.top + margin.top + border.top;
    newRect.right = rect.right + margin.right + border.right;
    newRect.bottom = rect.bottom + margin.bottom + border.bottom;
    newRect.width = newRect.right - newRect.left;
    newRect.height = newRect.bottom - newRect.top;

    return newRect;
  }

  // get position of element relative to page
  function getRectInPage(element) {
    const rect = getRectInView(element);
    const body = getRectInView(document.body);

    const newRect = {};
    newRect.left = rect.left - body.left;
    newRect.top = rect.top - body.top;
    newRect.right = rect.right - body.left;
    newRect.bottom = rect.bottom - body.top;
    newRect.width = rect.width;
    newRect.height = rect.height;

    return newRect;
  }

  // get closest element before specified element that matches query
  function firstBefore(element, query) {
    while (element && element !== document.body && !element.matches(query))
      element = element.previousElementSibling || element.parentNode;

    return element;
  }

  // check if element is part of collapsed heading
  function isCollapsed(element) {
    while (element && element !== document.body) {
      if (element.dataset.collapsed === "true") return true;
      element = element.parentNode;
    }
    return false;
  }

  // expand any collapsed parent containers of element if necessary
  function expandElement(element) {
    if (isCollapsed(element)) {
      // accordion plugin
      const heading = firstBefore(element, "h2");
      if (heading) heading.click();
      // details/summary HTML element
      const summary = firstBefore(element, "summary");
      if (summary) summary.click();
    }
  }

  // scroll to and focus element
  function goToElement(element, offset) {
    // expand accordion section if collapsed
    expandElement(element);
    const y =
      getRectInView(element).top -
      getRectInView(document.documentElement).top -
      (offset || 0);

    // trigger any function listening for "onscroll" event
    window.dispatchEvent(new Event("scroll"));
    window.scrollTo(0, y);
    document.activeElement.blur();
    element.focus();
  }

  // get list of elements after a start element up to element matching query
  function nextUntil(element, query, exclude) {
    const elements = [];
    while (((element = element.nextElementSibling), element)) {
      if (element.matches(query)) break;
      if (!element.matches(exclude)) elements.push(element);
    }
    return elements;
  }
</script>
<!--
  Accordion Plugin

  Allows sections of content under h2 headings to be collapsible.
-->

<script type="module">
  // whether to always start expanded ('false'), always start collapsed
  // ('true'), or start collapsed when screen small ('auto')
  const startCollapsed = "auto";

  // start script
  function start() {
    // run through each <h2> heading
    const headings = document.querySelectorAll("h2");
    for (const heading of headings) {
      addArrow(heading);

      // start expanded/collapsed based on option
      if (
        startCollapsed === "true" ||
        (startCollapsed === "auto" && isSmallScreen()) ||
        heading.dataset.collapsed === "true"
      )
        collapseHeading(heading);
      else expandElement(heading);
    }

    // attach hash change listener to window
    window.addEventListener("hashchange", onHashChange);
  }

  // when hash (eg manuscript.html#introduction) changes
  function onHashChange() {
    const target = getHashTarget();
    if (target) goToElement(target);
  }

  // add arrow to heading
  function addArrow(heading) {
    // add arrow button
    const arrow = document.createElement("button");
    arrow.innerHTML = document.querySelector(".icon_angle_down").innerHTML;
    arrow.classList.add("icon_button", "accordion_arrow");
    heading.insertBefore(arrow, heading.firstChild);

    // attach click listener to heading and button
    heading.addEventListener("click", onHeadingClick);
    arrow.addEventListener("click", onArrowClick);
  }

  // determine if on mobile-like device with small screen
  function isSmallScreen() {
    return Math.min(window.innerWidth, window.innerHeight) < 480;
  }

  // when <h2> heading is clicked
  function onHeadingClick(event) {
    // only collapse if <h2> itself is target of click (eg, user did
    // not click on anchor within <h2>)
    if (event.target === this) toggleCollapse(this);
  }

  // when arrow button is clicked
  function onArrowClick() {
    toggleCollapse(this.parentNode);
  }

  // collapse section if expanded, expand if collapsed
  function toggleCollapse(heading) {
    if (heading.dataset.collapsed === "false") collapseHeading(heading);
    else expandElement(heading);
  }

  // elements to exclude from collapse, such as table of contents panel,
  // hypothesis panel, etc
  const exclude = "#toc_panel, div.annotator-frame, #lightbox_overlay";

  // collapse section
  function collapseHeading(heading) {
    heading.setAttribute("data-collapsed", "true");
    const children = getChildren(heading);
    for (const child of children) child.setAttribute("data-collapsed", "true");
  }

  // expand section
  function expandElement(heading) {
    heading.setAttribute("data-collapsed", "false");
    const children = getChildren(heading);
    for (const child of children) child.setAttribute("data-collapsed", "false");
  }

  // get list of elements between this <h2> and next <h2> or <h1>
  // ("children" of the <h2> section)
  function getChildren(heading) {
    return nextUntil(heading, "h2, h1", exclude);
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- angle down icon -->

<template class="icon_angle_down">
  <!-- modified from: https://fontawesome.com/icons/angle-down -->
  <svg width="16" height="16" viewBox="0 0 448 512">
    <path
      fill="currentColor"
      d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* accordion arrow button */
    .accordion_arrow {
      margin-right: 10px;
    }

    /* arrow icon when <h2> data-collapsed attribute true */
    h2[data-collapsed="true"] > .accordion_arrow > svg {
      transform: rotate(-90deg);
    }

    /* all elements (except <h2>'s) when data-collapsed attribute true */
    *:not(h2)[data-collapsed="true"] {
      display: none;
    }

    /* accordion arrow button when hovered and <h2>'s when hovered */
    .accordion_arrow:hover,
    h2[data-collapsed="true"]:hover,
    h2[data-collapsed="false"]:hover {
      cursor: pointer;
    }
  }

  /* always hide accordion arrow button on print */
  @media only print {
    .accordion_arrow {
      display: none;
    }
  }
</style>
<!--
  Anchors Plugin

  Adds an anchor next to each of a certain type of element that provides a
  human-readable url to that specific item/position in the document (e.g.
  "manuscript.html#abstract"). It also makes it such that scrolling out of view
  of a target removes its identifier from the url.
-->

<script type="module">
  // which types of elements to add anchors next to, in "document.querySelector"
  // format
  const typesQuery =
    'h1, h2, h3, div[id^="fig:"], div[id^="tbl:"], span[id^="eq:"]';

  // start script
  function start() {
    // add anchor to each element of specified types
    const elements = document.querySelectorAll(typesQuery);
    for (const element of elements) addAnchor(element);

    // attach scroll listener to window
    window.addEventListener("scroll", onScroll);
  }

  // when window is scrolled
  function onScroll() {
    // if url has hash and user has scrolled out of view of hash
    // target, remove hash from url
    const tolerance = 100;
    const target = getHashTarget();
    if (target) {
      if (
        target.getBoundingClientRect().top > window.innerHeight + tolerance ||
        target.getBoundingClientRect().bottom < 0 - tolerance
      )
        history.pushState(null, null, " ");
    }
  }

  // add anchor to element
  function addAnchor(element) {
    let addTo; // element to add anchor button to

    // if figure or table, modify withId and addTo to get expected
    // elements
    if (element.id.indexOf("fig:") === 0) {
      addTo = element.querySelector("figcaption");
    } else if (element.id.indexOf("tbl:") === 0) {
      addTo = element.querySelector("caption");
    } else if (element.id.indexOf("eq:") === 0) {
      addTo = element.querySelector(".eqnos-number");
    }

    addTo = addTo || element;
    const id = element.id || null;

    // do not add anchor if element doesn't have assigned id.
    // id is generated by pandoc and is assumed to be unique and
    // human-readable
    if (!id) return;

    // create anchor button
    const anchor = document.createElement("a");
    anchor.innerHTML = document.querySelector(".icon_link").innerHTML;
    anchor.title = "Link to this part of the document";
    anchor.classList.add("icon_button", "anchor");
    anchor.dataset.ignore = "true";
    anchor.href = "#" + id;
    addTo.appendChild(anchor);
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- link icon -->

<template class="icon_link">
  <!-- modified from: https://fontawesome.com/icons/link -->
  <svg width="16" height="16" viewBox="0 0 512 512">
    <path
      fill="currentColor"
      d="M326.612 185.391c59.747 59.809 58.927 155.698.36 214.59-.11.12-.24.25-.36.37l-67.2 67.2c-59.27 59.27-155.699 59.262-214.96 0-59.27-59.26-59.27-155.7 0-214.96l37.106-37.106c9.84-9.84 26.786-3.3 27.294 10.606.648 17.722 3.826 35.527 9.69 52.721 1.986 5.822.567 12.262-3.783 16.612l-13.087 13.087c-28.026 28.026-28.905 73.66-1.155 101.96 28.024 28.579 74.086 28.749 102.325.51l67.2-67.19c28.191-28.191 28.073-73.757 0-101.83-3.701-3.694-7.429-6.564-10.341-8.569a16.037 16.037 0 0 1-6.947-12.606c-.396-10.567 3.348-21.456 11.698-29.806l21.054-21.055c5.521-5.521 14.182-6.199 20.584-1.731a152.482 152.482 0 0 1 20.522 17.197zM467.547 44.449c-59.261-59.262-155.69-59.27-214.96 0l-67.2 67.2c-.12.12-.25.25-.36.37-58.566 58.892-59.387 154.781.36 214.59a152.454 152.454 0 0 0 20.521 17.196c6.402 4.468 15.064 3.789 20.584-1.731l21.054-21.055c8.35-8.35 12.094-19.239 11.698-29.806a16.037 16.037 0 0 0-6.947-12.606c-2.912-2.005-6.64-4.875-10.341-8.569-28.073-28.073-28.191-73.639 0-101.83l67.2-67.19c28.239-28.239 74.3-28.069 102.325.51 27.75 28.3 26.872 73.934-1.155 101.96l-13.087 13.087c-4.35 4.35-5.769 10.79-3.783 16.612 5.864 17.194 9.042 34.999 9.69 52.721.509 13.906 17.454 20.446 27.294 10.606l37.106-37.106c59.271-59.259 59.271-155.699.001-214.959z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* anchor button */
    .anchor {
      opacity: 0;
      margin-left: 5px;
    }

    /* anchor buttons within <h2>'s */
    h2 .anchor {
      margin-left: 10px;
    }

    /* anchor buttons when hovered/focused and anything containing an anchor button when hovered */
    *:hover > .anchor,
    .anchor:hover,
    .anchor:focus {
      opacity: 1;
    }

    /* anchor button when hovered */
    .anchor:hover {
      cursor: pointer;
    }
  }

  /* always show anchor button on devices with no mouse/hover ability */
  @media (hover: none) {
    .anchor {
      opacity: 1;
    }
  }

  /* always hide anchor button on print */
  @media only print {
    .anchor {
      display: none;
    }
  }
</style>
<!-- 
    Attributes Plugin

    Allows arbitrary HTML attributes to be attached to (almost) any element.
    Place an HTML comment inside or next to the desired element with the content:
    $attribute="value"
-->

<script type="module">
  // start script
  function start() {
    // get list of comments in document
    const comments = findComments();

    for (const comment of comments)
      if (comment.parentElement)
        addAttributes(comment.parentElement, comment.nodeValue.trim());
  }

  // add html attributes to specified element based on string of
  // html attributes and values
  function addAttributes(element, text) {
    // regex's for finding attribute/value pairs in the format of
    // attribute="value" or attribute='value
    const regex2 = /\$([a-zA-Z\-]+)?=\"(.+?)\"/;
    const regex1 = /\$([a-zA-Z\-]+)?=\'(.+?)\'/;

    // loop through attribute/value pairs
    let match;
    while ((match = text.match(regex2) || text.match(regex1))) {
      // get attribute and value from regex capture groups
      let attribute = match[1];
      let value = match[2];

      // remove from string
      text = text.substring(match.index + match[0].length);

      if (!attribute || !value) break;

      // set attribute of parent element
      try {
        element.setAttribute(attribute, value);
      } catch (error) {
        console.log(error);
      }

      // special case for colspan
      if (attribute === "colspan") removeTableCells(element, value);
    }
  }

  // get list of comment elements in document
  function findComments() {
    const comments = [];

    // iterate over comment nodes in document
    function acceptNode(node) {
      return NodeFilter.FILTER_ACCEPT;
    }
    const iterator = document.createNodeIterator(
      document.body,
      NodeFilter.SHOW_COMMENT,
      acceptNode
    );
    let node;
    while ((node = iterator.nextNode())) comments.push(node);

    return comments;
  }

  // remove certain number of cells after specified cell
  function removeTableCells(cell, number) {
    number = parseInt(number);
    if (!number) return;

    // remove elements
    for (; number > 1; number--) {
      if (cell.nextElementSibling) cell.nextElementSibling.remove();
    }
  }

  // start script on DOMContentLoaded instead of load to ensure this plugins
  // runs before other plugins
  window.addEventListener("DOMContentLoaded", start);
</script>
<!--
  Jump to First Plugin

  Adds a button next to each reference entry, figure, and table that jumps the
  page to the first occurrence of a link to that item in the manuscript.
-->

<script type="module">
  // whether to add buttons next to reference entries
  const references = "true";
  // whether to add buttons next to figures
  const figures = "true";
  // whether to add buttons next to tables
  const tables = "true";

  // start script
  function start() {
    if (references !== "false")
      makeButtons(`div[id^="ref-"]`, ".csl-left-margin", "reference");
    if (figures !== "false")
      makeButtons(`div[id^="fig:"]`, "figcaption", "figure");
    if (tables !== "false") makeButtons(`div[id^="tbl:"]`, "caption", "table");
  }

  // when jump button clicked
  function onButtonClick() {
    const first = getFirstOccurrence(this.dataset.id);
    if (!first) return;

    // update url hash so navigating "back" in history will return user to button
    window.location.hash = this.dataset.id;
    // scroll to link
    const timeout = function () {
      goToElement(first, window.innerHeight * 0.5);
    };
    window.setTimeout(timeout, 0);
  }

  // get first occurrence of link to item in document
  function getFirstOccurrence(id) {
    let query = "a";
    query += '[href="#' + id + '"]';
    // exclude buttons, anchor links, toc links, etc
    query += ":not(.button):not(.icon_button):not(.anchor):not(.toc_link)";
    return document.querySelector(query);
  }

  // add button next to each reference entry, figure, or table
  function makeButtons(query, containerQuery, subject) {
    const elements = document.querySelectorAll(query);
    for (const element of elements) {
      const id = element.id;
      const buttonContainer = element.querySelector(containerQuery);
      const first = getFirstOccurrence(id);

      // if can't find link to reference or place to put button, ignore
      if (!first || !buttonContainer) continue;

      // make jump button
      let button = document.createElement("button");
      button.classList.add("icon_button", "jump_arrow");
      button.title = `Jump to the first occurrence of this ${subject} in the document`;
      const icon = document.querySelector(".icon_angle_double_up");
      button.innerHTML = icon.innerHTML;
      button.dataset.id = id;
      button.dataset.ignore = "true";
      button.addEventListener("click", onButtonClick);
      buttonContainer.prepend(button);
    }
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- angle double up icon -->

<template class="icon_angle_double_up">
  <!-- modified from: https://fontawesome.com/icons/angle-double-up -->
  <svg width="16" height="16" viewBox="0 0 320 512">
    <path
      fill="currentColor"
      d="M177 255.7l136 136c9.4 9.4 9.4 24.6 0 33.9l-22.6 22.6c-9.4 9.4-24.6 9.4-33.9 0L160 351.9l-96.4 96.4c-9.4 9.4-24.6 9.4-33.9 0L7 425.7c-9.4-9.4-9.4-24.6 0-33.9l136-136c9.4-9.5 24.6-9.5 34-.1zm-34-192L7 199.7c-9.4 9.4-9.4 24.6 0 33.9l22.6 22.6c9.4 9.4 24.6 9.4 33.9 0l96.4-96.4 96.4 96.4c9.4 9.4 24.6 9.4 33.9 0l22.6-22.6c9.4-9.4 9.4-24.6 0-33.9l-136-136c-9.2-9.4-24.4-9.4-33.8 0z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* jump button */
    .jump_arrow {
      position: relative;
      top: 0.125em;
      margin-right: 5px;
    }
  }

  /* always hide jump button on print */
  @media only print {
    .jump_arrow {
      display: none;
    }
  }
</style>
<!-- 
    Lightbox Plugin

    Makes it such that when a user clicks on an image, the image fills the
    screen and the user can pan/drag/zoom the image and navigate between other
    images in the document.
-->

<script type="module">
  // list of possible zoom/scale factors
  const zooms =
    "0.1, 0.25, 0.333333, 0.5, 0.666666, 0.75, 1, 1.25, 1.5, 1.75, 2, 2.5, 3, 3.5, 4, 5, 6, 7, 8";
  // whether to fit image to view ('fit'), display at 100% and shrink if
  // necessary ('shrink'), or always display at 100% ('100')
  const defaultZoom = "fit";
  // whether to zoom in/out toward center of view ('true') or mouse ('false')
  const centerZoom = "false";

  // start script
  function start() {
    // run through each <img> element
    const imgs = document.querySelectorAll("figure > img");
    let count = 1;
    for (const img of imgs) {
      img.classList.add("lightbox_document_img");
      img.dataset.number = count;
      img.dataset.total = imgs.length;
      img.addEventListener("click", openLightbox);
      count++;
    }

    // attach mouse and key listeners to window
    window.addEventListener("mousemove", onWindowMouseMove);
    window.addEventListener("keyup", onKeyUp);
  }

  // when mouse is moved anywhere in window
  function onWindowMouseMove(event) {
    window.mouseX = event.clientX;
    window.mouseY = event.clientY;
  }

  // when key pressed
  function onKeyUp(event) {
    if (!event || !event.key) return;

    switch (event.key) {
      // trigger click of prev button
      case "ArrowLeft":
        const prevButton = document.getElementById("lightbox_prev_button");
        if (prevButton) prevButton.click();
        break;
      // trigger click of next button
      case "ArrowRight":
        const nextButton = document.getElementById("lightbox_next_button");
        if (nextButton) nextButton.click();
        break;
      // close on esc
      case "Escape":
        closeLightbox();
        break;
    }
  }

  // open lightbox
  function openLightbox() {
    const lightbox = makeLightbox(this);
    if (!lightbox) return;

    blurBody(lightbox);
    document.body.appendChild(lightbox);
  }

  // make lightbox
  function makeLightbox(img) {
    // delete lightbox if it exists, start fresh
    closeLightbox();

    // create screen overlay containing lightbox
    const overlay = document.createElement("div");
    overlay.id = "lightbox_overlay";

    // create image info boxes
    const numberInfo = document.createElement("div");
    const zoomInfo = document.createElement("div");
    numberInfo.id = "lightbox_number_info";
    zoomInfo.id = "lightbox_zoom_info";

    // create container for image
    const imageContainer = document.createElement("div");
    imageContainer.id = "lightbox_image_container";
    const lightboxImg = makeLightboxImg(
      img,
      imageContainer,
      numberInfo,
      zoomInfo
    );
    imageContainer.appendChild(lightboxImg);

    // create bottom container for caption and navigation buttons
    const bottomContainer = document.createElement("div");
    bottomContainer.id = "lightbox_bottom_container";
    const caption = makeCaption(img);
    const prevButton = makePrevButton(img);
    const nextButton = makeNextButton(img);
    bottomContainer.appendChild(prevButton);
    bottomContainer.appendChild(caption);
    bottomContainer.appendChild(nextButton);

    // attach top middle and bottom to overlay
    overlay.appendChild(numberInfo);
    overlay.appendChild(zoomInfo);
    overlay.appendChild(imageContainer);
    overlay.appendChild(bottomContainer);

    return overlay;
  }

  // make <img> object that is intuitively draggable and zoomable
  function makeLightboxImg(sourceImg, container, numberInfoBox, zoomInfoBox) {
    // create copy of source <img>
    const img = sourceImg.cloneNode(true);
    img.classList.remove("lightbox_document_img");
    img.removeAttribute("id");
    img.removeAttribute("width");
    img.removeAttribute("height");
    img.style.position = "unset";
    img.style.margin = "0";
    img.style.padding = "0";
    img.style.width = "";
    img.style.height = "";
    img.style.minWidth = "";
    img.style.minHeight = "";
    img.style.maxWidth = "";
    img.style.maxHeight = "";
    img.id = "lightbox_img";

    // build sorted list of zoomSteps
    const zoomSteps = zooms.split(/[^0-9.]/).map((step) => parseFloat(step));
    zoomSteps.sort((a, b) => a - b);

    // <img> object property variables
    let zoom = 1;
    let translateX = 0;
    let translateY = 0;
    let clickMouseX = undefined;
    let clickMouseY = undefined;
    let clickTranslateX = undefined;
    let clickTranslateY = undefined;

    updateNumberInfo();

    // update image numbers displayed in info box
    function updateNumberInfo() {
      numberInfoBox.innerHTML =
        sourceImg.dataset.number + " of " + sourceImg.dataset.total;
    }

    // update zoom displayed in info box
    function updateZoomInfo() {
      let zoomInfo = zoom * 100;
      if (!Number.isInteger(zoomInfo)) zoomInfo = zoomInfo.toFixed(2);
      zoomInfoBox.innerHTML = zoomInfo + "%";
    }

    // move to closest zoom step above current zoom
    const zoomIn = function () {
      for (const zoomStep of zoomSteps) {
        if (zoomStep > zoom) {
          zoom = zoomStep;
          break;
        }
      }
      updateTransform();
    };

    // move to closest zoom step above current zoom
    const zoomOut = function () {
      zoomSteps.reverse();
      for (const zoomStep of zoomSteps) {
        if (zoomStep < zoom) {
          zoom = zoomStep;
          break;
        }
      }
      zoomSteps.reverse();

      updateTransform();
    };

    // update display of <img> based on scale/translate properties
    const updateTransform = function () {
      // set transform
      img.style.transform =
        "translate(" +
        (translateX || 0) +
        "px," +
        (translateY || 0) +
        "px) scale(" +
        (zoom || 1) +
        ")";

      // get new width/height after scale
      const rect = img.getBoundingClientRect();
      // limit translate
      translateX = Math.max(translateX, -rect.width / 2);
      translateX = Math.min(translateX, rect.width / 2);
      translateY = Math.max(translateY, -rect.height / 2);
      translateY = Math.min(translateY, rect.height / 2);

      // set transform
      img.style.transform =
        "translate(" +
        (translateX || 0) +
        "px," +
        (translateY || 0) +
        "px) scale(" +
        (zoom || 1) +
        ")";

      updateZoomInfo();
    };

    // fit <img> to container
    const fit = function () {
      // no x/y offset, 100% zoom by default
      translateX = 0;
      translateY = 0;
      zoom = 1;

      // widths of <img> and container
      const imgWidth = img.naturalWidth;
      const imgHeight = img.naturalHeight;
      const containerWidth = parseFloat(
        window.getComputedStyle(container).width
      );
      const containerHeight = parseFloat(
        window.getComputedStyle(container).height
      );

      // how much zooming is needed to fit <img> to container
      const xRatio = imgWidth / containerWidth;
      const yRatio = imgHeight / containerHeight;
      const maxRatio = Math.max(xRatio, yRatio);
      const newZoom = 1 / maxRatio;

      // fit <img> to container according to option
      if (defaultZoom === "shrink") {
        if (maxRatio > 1) zoom = newZoom;
      } else if (defaultZoom === "fit") zoom = newZoom;

      updateTransform();
    };

    // when mouse wheel is rolled anywhere in container
    const onContainerWheel = function (event) {
      if (!event) return;

      // let ctrl + mouse wheel to zoom behave as normal
      if (event.ctrlKey) return;

      // prevent normal scroll behavior
      event.preventDefault();
      event.stopPropagation();

      // point around which to scale img
      const viewRect = container.getBoundingClientRect();
      const viewX = (viewRect.left + viewRect.right) / 2;
      const viewY = (viewRect.top + viewRect.bottom) / 2;
      const originX = centerZoom === "true" ? viewX : mouseX;
      const originY = centerZoom === "true" ? viewY : mouseY;

      // get point on image under origin
      const oldRect = img.getBoundingClientRect();
      const oldPercentX = (originX - oldRect.left) / oldRect.width;
      const oldPercentY = (originY - oldRect.top) / oldRect.height;

      // increment/decrement zoom
      if (event.deltaY < 0) zoomIn();
      if (event.deltaY > 0) zoomOut();

      // get offset between previous image point and origin
      const newRect = img.getBoundingClientRect();
      const offsetX = originX - (newRect.left + newRect.width * oldPercentX);
      const offsetY = originY - (newRect.top + newRect.height * oldPercentY);

      // translate image to keep image point under origin
      translateX += offsetX;
      translateY += offsetY;

      // perform translate
      updateTransform();
    };

    // when container is clicked
    function onContainerClick(event) {
      // if container itself is target of click, and not other
      // element above it
      if (event.target === this) closeLightbox();
    }

    // when mouse button is pressed on image
    const onImageMouseDown = function (event) {
      // store original mouse position relative to image
      clickMouseX = window.mouseX;
      clickMouseY = window.mouseY;
      clickTranslateX = translateX;
      clickTranslateY = translateY;
      event.stopPropagation();
      event.preventDefault();
    };

    // when mouse button is released anywhere in window
    const onWindowMouseUp = function (event) {
      // reset original mouse position
      clickMouseX = undefined;
      clickMouseY = undefined;
      clickTranslateX = undefined;
      clickTranslateY = undefined;

      // remove global listener if lightbox removed from document
      if (!document.body.contains(container))
        window.removeEventListener("mouseup", onWindowMouseUp);
    };

    // when mouse is moved anywhere in window
    const onWindowMouseMove = function (event) {
      if (
        clickMouseX === undefined ||
        clickMouseY === undefined ||
        clickTranslateX === undefined ||
        clickTranslateY === undefined
      )
        return;

      // offset image based on original and current mouse position
      translateX = clickTranslateX + window.mouseX - clickMouseX;
      translateY = clickTranslateY + window.mouseY - clickMouseY;
      updateTransform();
      event.preventDefault();

      // remove global listener if lightbox removed from document
      if (!document.body.contains(container))
        window.removeEventListener("mousemove", onWindowMouseMove);
    };

    // when window is resized
    const onWindowResize = function (event) {
      fit();

      // remove global listener if lightbox removed from document
      if (!document.body.contains(container))
        window.removeEventListener("resize", onWindowResize);
    };

    // attach the necessary event listeners
    img.addEventListener("dblclick", fit);
    img.addEventListener("mousedown", onImageMouseDown);
    container.addEventListener("wheel", onContainerWheel);
    container.addEventListener("mousedown", onContainerClick);
    container.addEventListener("touchstart", onContainerClick);
    window.addEventListener("mouseup", onWindowMouseUp);
    window.addEventListener("mousemove", onWindowMouseMove);
    window.addEventListener("resize", onWindowResize);

    // run fit() after lightbox atttached to document and <img> Loaded
    // so needed container and img dimensions available
    img.addEventListener("load", fit);

    return img;
  }

  // make caption
  function makeCaption(img) {
    const caption = document.createElement("div");
    caption.id = "lightbox_caption";
    const captionSource = img.nextElementSibling;
    if (captionSource.tagName.toLowerCase() === "figcaption") {
      const captionCopy = makeCopy(captionSource);
      caption.innerHTML = captionCopy.innerHTML;
    }

    caption.addEventListener("touchstart", function (event) {
      event.stopPropagation();
    });

    return caption;
  }

  // make carbon copy of html dom element
  function makeCopy(source) {
    const sourceCopy = source.cloneNode(true);

    // delete elements marked with ignore (eg anchor and jump buttons)
    const deleteFromCopy = sourceCopy.querySelectorAll('[data-ignore="true"]');
    for (const element of deleteFromCopy) element.remove();

    // delete certain element attributes
    const attributes = [
      "id",
      "data-collapsed",
      "data-selected",
      "data-highlighted",
      "data-glow",
    ];
    for (const attribute of attributes) {
      sourceCopy.removeAttribute(attribute);
      const elements = sourceCopy.querySelectorAll("[" + attribute + "]");
      for (const element of elements) element.removeAttribute(attribute);
    }

    return sourceCopy;
  }

  // make button to jump to previous image in document
  function makePrevButton(img) {
    const prevButton = document.createElement("button");
    prevButton.id = "lightbox_prev_button";
    prevButton.title = "Jump to the previous image in the document [←]";
    prevButton.classList.add("icon_button", "lightbox_button");
    prevButton.innerHTML = document.querySelector(".icon_caret_left").innerHTML;

    // attach click listeners to button
    prevButton.addEventListener("click", function () {
      getPrevImg(img).click();
    });

    return prevButton;
  }

  // make button to jump to next image in document
  function makeNextButton(img) {
    const nextButton = document.createElement("button");
    nextButton.id = "lightbox_next_button";
    nextButton.title = "Jump to the next image in the document [→]";
    nextButton.classList.add("icon_button", "lightbox_button");
    nextButton.innerHTML = document.querySelector(
      ".icon_caret_right"
    ).innerHTML;

    // attach click listeners to button
    nextButton.addEventListener("click", function () {
      getNextImg(img).click();
    });

    return nextButton;
  }

  // get previous image in document
  function getPrevImg(img) {
    const imgs = document.querySelectorAll(".lightbox_document_img");

    // find index of provided img
    let index;
    for (index = 0; index < imgs.length; index++) {
      if (imgs[index] === img) break;
    }

    // wrap index to other side if < 1
    if (index - 1 >= 0) index--;
    else index = imgs.length - 1;
    return imgs[index];
  }

  // get next image in document
  function getNextImg(img) {
    const imgs = document.querySelectorAll(".lightbox_document_img");

    // find index of provided img
    let index;
    for (index = 0; index < imgs.length; index++) {
      if (imgs[index] === img) break;
    }

    // wrap index to other side if > total
    if (index + 1 <= imgs.length - 1) index++;
    else index = 0;
    return imgs[index];
  }

  // close lightbox
  function closeLightbox() {
    focusBody();

    const lightbox = document.getElementById("lightbox_overlay");
    if (lightbox) lightbox.remove();
  }

  // make all elements behind lightbox non-focusable
  function blurBody(overlay) {
    const all = document.querySelectorAll("*");
    for (const element of all) element.tabIndex = -1;
    document.body.classList.add("body_no_scroll");
  }

  // make all elements focusable again
  function focusBody() {
    const all = document.querySelectorAll("*");
    for (const element of all) element.removeAttribute("tabIndex");
    document.body.classList.remove("body_no_scroll");
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- caret left icon -->

<template class="icon_caret_left">
  <!-- modified from: https://fontawesome.com/icons/caret-left -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M192 127.338v257.324c0 17.818-21.543 26.741-34.142 14.142L29.196 270.142c-7.81-7.81-7.81-20.474 0-28.284l128.662-128.662c12.599-12.6 34.142-3.676 34.142 14.142z"
    ></path>
  </svg>
</template>

<!-- caret right icon -->

<template class="icon_caret_right">
  <!-- modified from: https://fontawesome.com/icons/caret-right -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* regular <img> in document when hovered */
    img.lightbox_document_img:hover {
      cursor: pointer;
    }

    .body_no_scroll {
      overflow: hidden !important;
    }

    /* screen overlay */
    #lightbox_overlay {
      display: flex;
      flex-direction: column;
      position: fixed;
      left: 0;
      top: 0;
      right: 0;
      bottom: 0;
      background: rgba(0, 0, 0, 0.75);
      z-index: 3;
    }

    /* middle area containing lightbox image */
    #lightbox_image_container {
      flex-grow: 1;
      display: flex;
      justify-content: center;
      align-items: center;
      overflow: hidden;
      position: relative;
      padding: 20px;
    }

    /* bottom area containing caption */
    #lightbox_bottom_container {
      display: flex;
      justify-content: center;
      align-items: center;
      height: 100px;
      min-height: 100px;
      max-height: 100px;
      background: rgba(0, 0, 0, 0.5);
    }

    /* image number info text box */
    #lightbox_number_info {
      position: absolute;
      color: #ffffff;
      font-weight: 600;
      left: 2px;
      top: 0;
      z-index: 4;
    }

    /* zoom info text box */
    #lightbox_zoom_info {
      position: absolute;
      color: #ffffff;
      font-weight: 600;
      right: 2px;
      top: 0;
      z-index: 4;
    }

    /* copy of image caption */
    #lightbox_caption {
      box-sizing: border-box;
      display: inline-block;
      width: 100%;
      max-height: 100%;
      padding: 10px 0;
      text-align: center;
      overflow-y: auto;
      color: #ffffff;
    }

    /* navigation previous/next button */
    .lightbox_button {
      width: 100px;
      height: 100%;
      min-width: 100px;
      min-height: 100%;
      color: #ffffff;
    }

    /* navigation previous/next button when hovered */
    .lightbox_button:hover {
      background: none !important;
    }

    /* navigation button icon */
    .lightbox_button > svg {
      height: 25px;
    }

    /* figure auto-number */
    #lightbox_caption > span:first-of-type {
      font-weight: bold;
      margin-right: 5px;
    }

    /* lightbox image when hovered */
    #lightbox_img:hover {
      cursor: grab;
    }

    /* lightbox image when grabbed */
    #lightbox_img:active {
      cursor: grabbing;
    }
  }

  /* when on screen < 480px wide */
  @media only screen and (max-width: 480px) {
    /* make navigation buttons skinnier on small screens to make more room for caption text */
    .lightbox_button {
      width: 50px;
      min-width: 50px;
    }
  }

  /* always hide lightbox on print */
  @media only print {
    #lightbox_overlay {
      display: none;
    }
  }
</style>
<!-- 
  Link Highlight Plugin

  Makes it such that when a user hovers or focuses a link, other links that have
  the same target will be highlighted. It also makes it such that when clicking
  a link, the target of the link (eg reference, figure, table) is briefly
  highlighted.
-->

<script type="module">
  // whether to also highlight links that go to external urls
  const externalLinks = "false";
  // whether user must click off to unhighlight instead of just
  // un-hovering
  const clickUnhighlight = "false";
  // whether to also highlight links that are unique
  const highlightUnique = "true";

  // start script
  function start() {
    const links = getLinks();
    for (const link of links) {
      // attach mouse and focus listeners to link
      link.addEventListener("mouseenter", onLinkFocus);
      link.addEventListener("focus", onLinkFocus);
      link.addEventListener("mouseleave", onLinkUnhover);
    }

    // attach click and hash change listeners to window
    window.addEventListener("click", onClick);
    window.addEventListener("touchstart", onClick);
    window.addEventListener("hashchange", onHashChange);

    // run hash change on window load in case user has navigated
    // directly to hash
    onHashChange();
  }

  // when link is focused (tabbed to) or hovered
  function onLinkFocus() {
    highlight(this);
  }

  // when link is unhovered
  function onLinkUnhover() {
    if (clickUnhighlight !== "true") unhighlightAll();
  }

  // when the mouse is clicked anywhere in window
  function onClick(event) {
    unhighlightAll();
  }

  // when hash (eg manuscript.html#introduction) changes
  function onHashChange() {
    const target = getHashTarget();
    if (target) glowElement(target);
  }

  // start glow sequence on an element
  function glowElement(element) {
    const startGlow = function () {
      onGlowEnd();
      element.dataset.glow = "true";
      element.addEventListener("animationend", onGlowEnd);
    };
    const onGlowEnd = function () {
      element.removeAttribute("data-glow");
      element.removeEventListener("animationend", onGlowEnd);
    };
    startGlow();
  }

  // highlight link and all others with same target
  function highlight(link) {
    // force unhighlight all to start fresh
    unhighlightAll();

    // get links with same target
    if (!link) return;
    const sameLinks = getSameLinks(link);

    // if link unique and option is off, exit and don't highlight
    if (sameLinks.length <= 1 && highlightUnique !== "true") return;

    // highlight all same links, and "select" (special highlight) this
    // one
    for (const sameLink of sameLinks) {
      if (sameLink === link) sameLink.setAttribute("data-selected", "true");
      else sameLink.setAttribute("data-highlighted", "true");
    }
  }

  // unhighlight all links
  function unhighlightAll() {
    const links = getLinks();
    for (const link of links) {
      link.setAttribute("data-selected", "false");
      link.setAttribute("data-highlighted", "false");
    }
  }

  // get links with same target
  function getSameLinks(link) {
    const results = [];
    const links = getLinks();
    for (const otherLink of links) {
      if (otherLink.getAttribute("href") === link.getAttribute("href"))
        results.push(otherLink);
    }
    return results;
  }

  // get all links of types we wish to handle
  function getLinks() {
    let query = "a";
    if (externalLinks !== "true") query += '[href^="#"]';
    // exclude buttons, anchor links, toc links, etc
    query += ":not(.button):not(.icon_button):not(.anchor):not(.toc_link)";
    return document.querySelectorAll(query);
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<style>
  @media only screen {
    /* anything with data-highlighted attribute true */
    [data-highlighted="true"] {
      background: #ffeb3b;
    }

    /* anything with data-selected attribute true */
    [data-selected="true"] {
      background: #ff8a65 !important;
    }

    /* animation definition for glow */
    @keyframes highlight_glow {
      0% {
        background: none;
      }
      10% {
        background: #bbdefb;
      }
      100% {
        background: none;
      }
    }

    /* anything with data-glow attribute true */
    [data-glow="true"] {
      animation: highlight_glow 2s;
    }
  }
</style>
<!--
  Table of Contents Plugin

  Provides a "table of contents" (toc) panel on the side of the document that
  allows the user to conveniently navigate between sections of the document.
-->

<script type="module">
  // which types of elements to add links for, in "document.querySelector" format
  const typesQuery = "h1, h2, h3";
  // whether toc starts open. use 'true' or 'false', or 'auto' to
  // use 'true' behavior when screen wide enough and 'false' when not
  const startOpen = "false";
  // whether toc closes when clicking on toc link. use 'true' or
  // 'false', or 'auto' to use 'false' behavior when screen wide
  // enough and 'true' when not
  const clickClose = "auto";
  // if list item is more than this many characters, text will be
  // truncated
  const charLimit = "50";
  // whether or not to show bullets next to each toc item
  const bullets = "false";

  // start script
  function start() {
    // make toc panel and populate with entries (links to document
    // sections)
    const panel = makePanel();
    if (!panel) return;
    makeEntries(panel);
    // attach panel to document after making entries, so 'toc' heading
    // in panel isn't included in toc
    document.body.insertBefore(panel, document.body.firstChild);

    // initial panel state
    if (startOpen === "true" || (startOpen === "auto" && !isSmallScreen()))
      openPanel();
    else closePanel();

    // attach click, scroll, and hash change listeners to window
    window.addEventListener("click", onClick);
    window.addEventListener("scroll", onScroll);
    window.addEventListener("hashchange", onScroll);
    window.addEventListener("keyup", onKeyUp);
    onScroll();

    // add class to push document body down out of way of toc button
    document.body.classList.add("toc_body_nudge");
  }

  // determine if screen wide enough to fit toc panel
  function isSmallScreen() {
    // in default theme:
    // 816px = 8.5in = width of "page" (<body>) element
    // 260px = min width of toc panel (*2 for both sides of <body>)
    return window.innerWidth < 816 + 260 * 2;
  }

  // when mouse is clicked anywhere in window
  function onClick() {
    if (isSmallScreen()) closePanel();
  }

  // when window is scrolled or hash changed
  function onScroll() {
    highlightViewed();
  }

  // when key pressed
  function onKeyUp(event) {
    if (!event || !event.key) return;

    // close on esc
    if (event.key === "Escape") closePanel();
  }

  // find entry of currently viewed document section in toc and highlight
  function highlightViewed() {
    const firstId = getFirstInView(typesQuery);

    // get toc entries (links), unhighlight all, then highlight viewed
    const list = document.getElementById("toc_list");
    if (!firstId || !list) return;
    const links = list.querySelectorAll("a");
    for (const link of links) link.dataset.viewing = "false";
    const link = list.querySelector('a[href="#' + firstId + '"]');
    if (!link) return;
    link.dataset.viewing = "true";
  }

  // get first or previous toc listed element in top half of view
  function getFirstInView(query) {
    // get all elements matching query and with id
    const elements = document.querySelectorAll(query);
    const elementsWithIds = [];
    for (const element of elements) {
      if (element.id) elementsWithIds.push(element);
    }

    // get first or previous element in top half of view
    for (let i = 0; i < elementsWithIds.length; i++) {
      const element = elementsWithIds[i];
      const prevElement = elementsWithIds[Math.max(0, i - 1)];
      if (element.getBoundingClientRect().top >= 0) {
        if (element.getBoundingClientRect().top < window.innerHeight / 2)
          return element.id;
        else return prevElement.id;
      }
    }
  }

  // make panel
  function makePanel() {
    // create panel
    const panel = document.createElement("div");
    panel.id = "toc_panel";
    if (bullets === "true") panel.dataset.bullets = "true";

    // create header
    const header = document.createElement("div");
    header.id = "toc_header";

    // create toc button
    const button = document.createElement("button");
    button.id = "toc_button";
    button.innerHTML = document.querySelector(".icon_th_list").innerHTML;
    button.title = "Table of Contents";
    button.classList.add("icon_button");

    // create header text
    const text = document.createElement("h4");
    text.innerHTML = "Table of Contents";

    // create container for toc list
    const list = document.createElement("div");
    list.id = "toc_list";

    // attach click listeners
    panel.addEventListener("click", onPanelClick);
    header.addEventListener("click", onHeaderClick);
    button.addEventListener("click", onButtonClick);

    // attach elements
    header.appendChild(button);
    header.appendChild(text);
    panel.appendChild(header);
    panel.appendChild(list);

    return panel;
  }

  // create toc entries (links) to each element of the specified types
  function makeEntries(panel) {
    const elements = document.querySelectorAll(typesQuery);
    for (const element of elements) {
      // do not add link if element doesn't have assigned id
      if (!element.id) continue;

      // create link/list item
      const link = document.createElement("a");
      link.classList.add("toc_link");
      switch (element.tagName.toLowerCase()) {
        case "h1":
          link.dataset.level = "1";
          break;
        case "h2":
          link.dataset.level = "2";
          break;
        case "h3":
          link.dataset.level = "3";
          break;
        case "h4":
          link.dataset.level = "4";
          break;
      }
      link.title = element.innerText;
      let text = element.innerText;
      if (text.length > charLimit) text = text.slice(0, charLimit) + "...";
      link.innerHTML = text;
      link.href = "#" + element.id;
      link.addEventListener("click", onLinkClick);

      // attach link
      panel.querySelector("#toc_list").appendChild(link);
    }
  }

  // when panel is clicked
  function onPanelClick(event) {
    // stop click from propagating to window/document and closing panel
    event.stopPropagation();
  }

  // when header itself is clicked
  function onHeaderClick(event) {
    togglePanel();
  }

  // when button is clicked
  function onButtonClick(event) {
    togglePanel();
    // stop header underneath button from also being clicked
    event.stopPropagation();
  }

  // when link is clicked
  function onLinkClick(event) {
    if (clickClose === "true" || (clickClose === "auto" && isSmallScreen()))
      closePanel();
    else openPanel();
  }

  // open panel if closed, close if opened
  function togglePanel() {
    const panel = document.getElementById("toc_panel");
    if (!panel) return;

    if (panel.dataset.open === "true") closePanel();
    else openPanel();
  }

  // open panel
  function openPanel() {
    const panel = document.getElementById("toc_panel");
    if (panel) panel.dataset.open = "true";
  }

  // close panel
  function closePanel() {
    const panel = document.getElementById("toc_panel");
    if (panel) panel.dataset.open = "false";
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- th list icon -->

<template class="icon_th_list">
  <!-- modified from: https://fontawesome.com/icons/th-list -->
  <svg width="16" height="16" viewBox="0 0 512 512" tabindex="-1">
    <path
      fill="currentColor"
      d="M96 96c0 26.51-21.49 48-48 48S0 122.51 0 96s21.49-48 48-48 48 21.49 48 48zM48 208c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm0 160c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm96-236h352c8.837 0 16-7.163 16-16V76c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16z"
      tabindex="-1"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* toc panel */
    #toc_panel {
      box-sizing: border-box;
      position: fixed;
      top: 0;
      left: 0;
      background: #ffffff;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
      z-index: 2;
    }

    /* toc panel when closed */
    #toc_panel[data-open="false"] {
      min-width: 60px;
      width: 60px;
      height: 60px;
      border-right: solid 1px #bdbdbd;
      border-bottom: solid 1px #bdbdbd;
    }

    /* toc panel when open */
    #toc_panel[data-open="true"] {
      min-width: 260px;
      max-width: 480px;
      /* keep panel edge consistent distance away from "page" edge */
      width: calc(((100vw - 8.5in) / 2) - 30px - 40px);
      bottom: 0;
      border-right: solid 1px #bdbdbd;
    }

    /* toc panel header */
    #toc_header {
      box-sizing: border-box;
      display: flex;
      flex-direction: row;
      align-items: center;
      height: 60px;
      margin: 0;
      padding: 20px;
    }

    /* toc panel header when hovered */
    #toc_header:hover {
      cursor: pointer;
    }

    /* toc panel header when panel open */
    #toc_panel[data-open="true"] > #toc_header {
      border-bottom: solid 1px #bdbdbd;
    }

    /* toc open/close header button */
    #toc_button {
      margin-right: 20px;
    }

    /* hide toc list and header text when closed */
    #toc_panel[data-open="false"] > #toc_header > *:not(#toc_button),
    #toc_panel[data-open="false"] > #toc_list {
      display: none;
    }

    /* toc list of entries */
    #toc_list {
      box-sizing: border-box;
      width: 100%;
      padding: 20px;
      position: absolute;
      top: calc(60px + 1px);
      bottom: 0;
      overflow: auto;
    }

    /* toc entry, link to section in document */
    .toc_link {
      display: block;
      padding: 5px;
      position: relative;
      font-weight: 600;
      text-decoration: none;
    }

    /* toc entry when hovered or when "viewed" */
    .toc_link:hover,
    .toc_link[data-viewing="true"] {
      background: #f5f5f5;
    }

    /* toc entry, level 1 indentation */
    .toc_link[data-level="1"] {
      margin-left: 0;
    }

    /* toc entry, level 2 indentation */
    .toc_link[data-level="2"] {
      margin-left: 20px;
    }

    /* toc entry, level 3 indentation */
    .toc_link[data-level="3"] {
      margin-left: 40px;
    }

    /* toc entry, level 4 indentation */
    .toc_link[data-level="4"] {
      margin-left: 60px;
    }

    /* toc entry bullets */
    #toc_panel[data-bullets="true"] .toc_link[data-level]:before {
      position: absolute;
      left: -15px;
      top: -1px;
      font-size: 1.5em;
    }

    /* toc entry, level 2 bullet */
    #toc_panel[data-bullets="true"] .toc_link[data-level="2"]:before {
      content: "\2022";
    }

    /* toc entry, level 3 bullet */
    #toc_panel[data-bullets="true"] .toc_link[data-level="3"]:before {
      content: "\25AB";
    }

    /* toc entry, level 4 bullet */
    #toc_panel[data-bullets="true"] .toc_link[data-level="4"]:before {
      content: "-";
    }
  }

  /* when on screen < 8.5in wide */
  @media only screen and (max-width: 8.5in) {
    /* push <body> ("page") element down to make room for toc icon */
    .toc_body_nudge {
      padding-top: 60px;
    }

    /* toc icon when panel closed and not hovered */
    #toc_panel[data-open="false"]:not(:hover) {
      background: rgba(255, 255, 255, 0.75);
    }
  }

  /* always hide toc panel on print */
  @media only print {
    #toc_panel {
      display: none;
    }
  }
</style>
<!-- 
  Tooltips Plugin

  Makes it such that when the user hovers or focuses a link to a citation or
  figure, a tooltip appears with a preview of the reference content, along with
  arrows to navigate between instances of the same reference in the document.
-->

<script type="module">
  // whether user must click off to close tooltip instead of just un-hovering
  const clickClose = "false";
  // delay (in ms) between opening and closing tooltip
  const delay = "100";

  // start script
  function start() {
    const links = getLinks();
    for (const link of links) {
      // attach hover and focus listeners to link
      link.addEventListener("mouseover", onLinkHover);
      link.addEventListener("mouseleave", onLinkUnhover);
      link.addEventListener("focus", onLinkFocus);
      link.addEventListener("touchend", onLinkTouch);
    }

    // attach mouse, key, and resize listeners to window
    window.addEventListener("mousedown", onClick);
    window.addEventListener("touchstart", onClick);
    window.addEventListener("keyup", onKeyUp);
    window.addEventListener("resize", onResize);
  }

  // when link is hovered
  function onLinkHover() {
    // function to open tooltip
    const delayOpenTooltip = function () {
      openTooltip(this);
    }.bind(this);

    // run open function after delay
    this.openTooltipTimer = window.setTimeout(delayOpenTooltip, delay);
  }

  // when mouse leaves link
  function onLinkUnhover() {
    // cancel opening tooltip
    window.clearTimeout(this.openTooltipTimer);

    // don't close on unhover if option specifies
    if (clickClose === "true") return;

    // function to close tooltip
    const delayCloseTooltip = function () {
      // if tooltip open and if mouse isn't over tooltip, close
      const tooltip = document.getElementById("tooltip");
      if (tooltip && !tooltip.matches(":hover")) closeTooltip();
    };

    // run close function after delay
    this.closeTooltipTimer = window.setTimeout(delayCloseTooltip, delay);
  }

  // when link is focused (tabbed to)
  function onLinkFocus(event) {
    openTooltip(this);
  }

  // when link is touched on touch screen
  function onLinkTouch(event) {
    // attempt to force hover state on first tap always, and trigger
    // regular link click (and navigation) on second tap
    if (event.target === document.activeElement) event.target.click();
    else {
      document.activeElement.blur();
      event.target.focus();
    }
    if (event.cancelable) event.preventDefault();
    event.stopPropagation();
    return false;
  }

  // when mouse is clicked anywhere in window
  function onClick(event) {
    closeTooltip();
  }

  // when key pressed
  function onKeyUp(event) {
    if (!event || !event.key) return;

    switch (event.key) {
      // trigger click of prev button
      case "ArrowLeft":
        const prevButton = document.getElementById("tooltip_prev_button");
        if (prevButton) prevButton.click();
        break;
      // trigger click of next button
      case "ArrowRight":
        const nextButton = document.getElementById("tooltip_next_button");
        if (nextButton) nextButton.click();
        break;
      // close on esc
      case "Escape":
        closeTooltip();
        break;
    }
  }

  // when window is resized or zoomed
  function onResize() {
    closeTooltip();
  }

  // get all links of types we wish to handle
  function getLinks() {
    const queries = [];
    // exclude buttons, anchor links, toc links, etc
    const exclude =
      ":not(.button):not(.icon_button):not(.anchor):not(.toc_link)";
    queries.push('a[href^="#ref-"]' + exclude); // citation links
    queries.push('a[href^="#fig:"]' + exclude); // figure links
    const query = queries.join(", ");
    return document.querySelectorAll(query);
  }

  // get links with same target, get index of link in set, get total
  // same links
  function getSameLinks(link) {
    const sameLinks = [];
    const links = getLinks();
    for (const otherLink of links) {
      if (otherLink.getAttribute("href") === link.getAttribute("href"))
        sameLinks.push(otherLink);
    }

    return {
      elements: sameLinks,
      index: sameLinks.indexOf(link),
      total: sameLinks.length,
    };
  }

  // open tooltip
  function openTooltip(link) {
    // delete tooltip if it exists, start fresh
    closeTooltip();

    // make tooltip element
    const tooltip = makeTooltip(link);

    // if source couldn't be found and tooltip not made, exit
    if (!tooltip) return;

    // make navbar elements
    const navBar = makeNavBar(link);
    if (navBar) tooltip.firstElementChild.appendChild(navBar);

    // attach tooltip to page
    document.body.appendChild(tooltip);

    // position tooltip
    const position = function () {
      positionTooltip(link);
    };
    position();

    // if tooltip contains images, position again after they've loaded
    const imgs = tooltip.querySelectorAll("img");
    for (const img of imgs) img.addEventListener("load", position);
  }

  // close (delete) tooltip
  function closeTooltip() {
    const tooltip = document.getElementById("tooltip");
    if (tooltip) tooltip.remove();
  }

  // make tooltip
  function makeTooltip(link) {
    // get target element that link points to
    const source = getSource(link);

    // if source can't be found, exit
    if (!source) return;

    // create new tooltip
    const tooltip = document.createElement("div");
    tooltip.id = "tooltip";
    const tooltipContent = document.createElement("div");
    tooltipContent.id = "tooltip_content";
    tooltip.appendChild(tooltipContent);

    // make copy of source node and put in tooltip
    const sourceCopy = makeCopy(source);
    tooltipContent.appendChild(sourceCopy);

    // attach mouse event listeners
    tooltip.addEventListener("click", onTooltipClick);
    tooltip.addEventListener("mousedown", onTooltipClick);
    tooltip.addEventListener("touchstart", onTooltipClick);
    tooltip.addEventListener("mouseleave", onTooltipUnhover);

    // (for interaction with lightbox plugin)
    // transfer click on tooltip copied img to original img
    const sourceImg = source.querySelector("img");
    const sourceCopyImg = sourceCopy.querySelector("img");
    if (sourceImg && sourceCopyImg) {
      const clickImg = function () {
        sourceImg.click();
        closeTooltip();
      };
      sourceCopyImg.addEventListener("click", clickImg);
    }

    return tooltip;
  }

  // make carbon copy of html dom element
  function makeCopy(source) {
    const sourceCopy = source.cloneNode(true);

    // delete elements marked with ignore (eg anchor and jump buttons)
    const deleteFromCopy = sourceCopy.querySelectorAll('[data-ignore="true"]');
    for (const element of deleteFromCopy) element.remove();

    // delete certain element attributes
    const attributes = [
      "id",
      "data-collapsed",
      "data-selected",
      "data-highlighted",
      "data-glow",
      "class",
    ];
    for (const attribute of attributes) {
      sourceCopy.removeAttribute(attribute);
      const elements = sourceCopy.querySelectorAll("[" + attribute + "]");
      for (const element of elements) element.removeAttribute(attribute);
    }

    return sourceCopy;
  }

  // when tooltip is clicked
  function onTooltipClick(event) {
    // when user clicks on tooltip, stop click from transferring
    // outside of tooltip (eg, click off to close tooltip, or eg click
    // off to unhighlight same refs)
    event.stopPropagation();
  }

  // when tooltip is unhovered
  function onTooltipUnhover(event) {
    if (clickClose === "true") return;

    // make sure new mouse/touch/focus no longer over tooltip or any
    // element within it
    const tooltip = document.getElementById("tooltip");
    if (!tooltip) return;
    if (this.contains(event.relatedTarget)) return;

    closeTooltip();
  }

  // make nav bar to go betwen prev/next instances of same reference
  function makeNavBar(link) {
    // find other links to the same source
    const sameLinks = getSameLinks(link);

    // don't show nav bar when singular reference
    if (sameLinks.total <= 1) return;

    // find prev/next links with same target
    const prevLink = getPrevLink(link, sameLinks);
    const nextLink = getNextLink(link, sameLinks);

    // create nav bar
    const navBar = document.createElement("div");
    navBar.id = "tooltip_nav_bar";
    const text = sameLinks.index + 1 + " of " + sameLinks.total;

    // create nav bar prev/next buttons
    const prevButton = document.createElement("button");
    const nextButton = document.createElement("button");
    prevButton.id = "tooltip_prev_button";
    nextButton.id = "tooltip_next_button";
    prevButton.title =
      "Jump to the previous occurence of this item in the document [←]";
    nextButton.title =
      "Jump to the next occurence of this item in the document [→]";
    prevButton.classList.add("icon_button");
    nextButton.classList.add("icon_button");
    prevButton.innerHTML = document.querySelector(".icon_caret_left").innerHTML;
    nextButton.innerHTML =
      document.querySelector(".icon_caret_right").innerHTML;
    navBar.appendChild(prevButton);
    navBar.appendChild(document.createTextNode(text));
    navBar.appendChild(nextButton);

    // attach click listeners to buttons
    prevButton.addEventListener("click", function () {
      onPrevNextClick(link, prevLink);
    });
    nextButton.addEventListener("click", function () {
      onPrevNextClick(link, nextLink);
    });

    return navBar;
  }

  // get previous link with same target
  function getPrevLink(link, sameLinks) {
    if (!sameLinks) sameLinks = getSameLinks(link);
    // wrap index to other side if < 1
    let index;
    if (sameLinks.index - 1 >= 0) index = sameLinks.index - 1;
    else index = sameLinks.total - 1;
    return sameLinks.elements[index];
  }

  // get next link with same target
  function getNextLink(link, sameLinks) {
    if (!sameLinks) sameLinks = getSameLinks(link);
    // wrap index to other side if > total
    let index;
    if (sameLinks.index + 1 <= sameLinks.total - 1) index = sameLinks.index + 1;
    else index = 0;
    return sameLinks.elements[index];
  }

  // get element that is target of link or url hash
  function getSource(link) {
    const hash = link ? link.hash : window.location.hash;
    const id = hash.slice(1);
    let target = document.querySelector('[id="' + id + '"]');
    if (!target) return;

    // if ref or figure, modify target to get expected element
    if (id.indexOf("ref-") === 0) target = target.querySelector(":nth-child(2)");
    else if (id.indexOf("fig:") === 0) target = target.querySelector("figure");

    return target;
  }

  // when prev/next arrow button is clicked
  function onPrevNextClick(link, prevNextLink) {
    if (link && prevNextLink)
      goToElement(prevNextLink, window.innerHeight * 0.5);
  }

  // scroll to and focus element
  function goToElement(element, offset) {
    // expand accordion section if collapsed
    expandElement(element);
    const y =
      getRectInView(element).top -
      getRectInView(document.documentElement).top -
      (offset || 0);
    // trigger any function listening for "onscroll" event
    window.dispatchEvent(new Event("scroll"));
    window.scrollTo(0, y);
    document.activeElement.blur();
    element.focus();
  }

  // determine position to place tooltip based on link position in
  // viewport and tooltip size
  function positionTooltip(link, left, top) {
    const tooltipElement = document.getElementById("tooltip");
    if (!tooltipElement) return;

    // get convenient vars for position/dimensions of
    // link/tooltip/page/view
    link = getRectInPage(link);
    const tooltip = getRectInPage(tooltipElement);
    const view = getRectInPage();

    // horizontal positioning
    if (left)
      // use explicit value
      left = left;
    else if (link.left + tooltip.width < view.right)
      // fit tooltip to right of link
      left = link.left;
    else if (link.right - tooltip.width > view.left)
      // fit tooltip to left of link
      left = link.right - tooltip.width;
    // center tooltip in view
    else left = (view.right - view.left) / 2 - tooltip.width / 2;

    // vertical positioning
    if (top)
      // use explicit value
      top = top;
    else if (link.top - tooltip.height > view.top)
      // fit tooltip above link
      top = link.top - tooltip.height;
    else if (link.bottom + tooltip.height < view.bottom)
      // fit tooltip below link
      top = link.bottom;
    else {
      // center tooltip in view
      top = view.top + view.height / 2 - tooltip.height / 2;
      // nudge off of link to left/right if possible
      if (link.right + tooltip.width < view.right) left = link.right;
      else if (link.left - tooltip.width > view.left)
        left = link.left - tooltip.width;
    }

    tooltipElement.style.left = left + "px";
    tooltipElement.style.top = top + "px";
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- caret left icon -->

<template class="icon_caret_left">
  <!-- modified from: https://fontawesome.com/icons/caret-left -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M192 127.338v257.324c0 17.818-21.543 26.741-34.142 14.142L29.196 270.142c-7.81-7.81-7.81-20.474 0-28.284l128.662-128.662c12.599-12.6 34.142-3.676 34.142 14.142z"
    ></path>
  </svg>
</template>

<!-- caret right icon -->

<template class="icon_caret_right">
  <!-- modified from: https://fontawesome.com/icons/caret-right -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* tooltip container */
    #tooltip {
      position: absolute;
      width: 50%;
      min-width: 240px;
      max-width: 75%;
      z-index: 1;
    }

    /* tooltip content */
    #tooltip_content {
      margin-bottom: 5px;
      padding: 20px;
      border-radius: 5px;
      border: solid 1px #bdbdbd;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
      background: #ffffff;
      overflow-wrap: break-word;
    }

    /* tooltip copy of paragraphs and figures */
    #tooltip_content > p,
    #tooltip_content > figure {
      margin: 0;
      max-height: 320px;
      overflow-y: auto;
    }

    /* tooltip copy of <img> */
    #tooltip_content > figure > img,
    #tooltip_content > figure > svg {
      max-height: 260px;
    }

    /* navigation bar */
    #tooltip_nav_bar {
      margin-top: 10px;
      text-align: center;
    }

    /* navigation bar previous/next buton */
    #tooltip_nav_bar > .icon_button {
      position: relative;
      top: 3px;
    }

    /* navigation bar previous button */
    #tooltip_nav_bar > .icon_button:first-of-type {
      margin-right: 5px;
    }

    /* navigation bar next button */
    #tooltip_nav_bar > .icon_button:last-of-type {
      margin-left: 5px;
    }
  }

  /* always hide tooltip on print */
  @media only print {
    #tooltip {
      display: none;
    }
  }
</style>
<!--
  Analytics Plugin (third-party) 
  
  Copy and paste code from Google Analytics or similar service here.
-->
<!-- 
  Annotations Plugin

  Allows public annotation of the  manuscript. See https://web.hypothes.is/.
-->

<script type="module">
  // configuration
  window.hypothesisConfig = function () {
    return {
      branding: {
        accentColor: "#2196f3",
        appBackgroundColor: "#f8f8f8",
        ctaBackgroundColor: "#f8f8f8",
        ctaTextColor: "#000000",
        selectionFontFamily: "Open Sans, Helvetica, sans serif",
        annotationFontFamily: "Open Sans, Helvetica, sans serif",
      },
    };
  };

  // hypothesis client script
  const embed = "https://hypothes.is/embed.js";
  // hypothesis annotation count query url
  const query = "https://api.hypothes.is/api/search?limit=0&url=";

  // start script
  function start() {
    const button = makeButton();
    document.body.insertBefore(button, document.body.firstChild);
    insertCount(button);
  }

  // make button
  function makeButton() {
    // create button
    const button = document.createElement("button");
    button.id = "hypothesis_button";
    button.innerHTML = document.querySelector(".icon_hypothesis").innerHTML;
    button.title = "Hypothesis annotations";
    button.classList.add("icon_button");

    function onClick(event) {
      onButtonClick(event, button);
    }

    // attach click listeners
    button.addEventListener("click", onClick);

    return button;
  }

  // insert annotations count
  async function insertCount(button) {
    // get annotation count from Hypothesis based on url
    let count = "-";
    try {
      const canonical = document.querySelector('link[rel="canonical"]');
      const location = window.location;
      const url = encodeURIComponent((canonical || location).href);
      const response = await fetch(query + url);
      const json = await response.json();
      count = json.total || "-";
    } catch (error) {
      console.log(error);
    }

    // put count into button
    const counter = document.createElement("span");
    counter.id = "hypothesis_count";
    counter.innerHTML = count;
    button.title = "View " + count + " Hypothesis annotations";
    button.append(counter);
  }

  // when button is clicked
  function onButtonClick(event, button) {
    const script = document.createElement("script");
    script.src = embed;
    document.body.append(script);
    button.remove();
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- hypothesis icon -->

<template class="icon_hypothesis">
  <!-- modified from: https://simpleicons.org/icons/hypothesis.svg / https://git.io/Jf1VB -->
  <svg width="16" height="16" viewBox="0 0 24 24" tabindex="-1">
    <path
      fill="currentColor"
      d="M3.43 0C2.5 0 1.72 .768 1.72 1.72V18.86C1.72 19.8 2.5 20.57 3.43 20.57H9.38L12 24L14.62 20.57H20.57C21.5 20.57 22.29 19.8 22.29 18.86V1.72C22.29 .77 21.5 0 20.57 0H3.43M5.14 3.43H7.72V9.43S8.58 7.72 10.28 7.72C12 7.72 13.74 8.57 13.74 11.24V17.14H11.16V12C11.16 10.61 10.28 10.07 9.43 10.29C8.57 10.5 7.72 11.41 7.72 13.29V17.14H5.14V3.43M18 13.72C18.95 13.72 19.72 14.5 19.72 15.42A1.71 1.71 0 0 1 18 17.13A1.71 1.71 0 0 1 16.29 15.42C16.29 14.5 17.05 13.71 18 13.71Z"
      tabindex="-1"
    ></path>
  </svg>
</template>

<style>
  /* hypothesis activation button */
  #hypothesis_button {
    box-sizing: border-box;
    position: fixed;
    top: 0;
    right: 0;
    width: 60px;
    height: 60px;
    background: #ffffff;
    border-radius: 0;
    border-left: solid 1px #bdbdbd;
    border-bottom: solid 1px #bdbdbd;
    box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
    z-index: 2;
  }

  /* hypothesis button svg */
  #hypothesis_button > svg {
    position: relative;
    top: -4px;
  }

  /* hypothesis annotation count */
  #hypothesis_count {
    position: absolute;
    left: 0;
    right: 0;
    bottom: 5px;
  }

  /* side panel */
  .annotator-frame {
    width: 280px !important;
  }

  /* match highlight color to rest of theme */
  .annotator-highlights-always-on .annotator-hl {
    background-color: #ffeb3b !important;
  }

  /* match focused color to rest of theme */
  .annotator-hl.annotator-hl-focused {
    background-color: #ff8a65 !important;
  }

  /* match bucket bar color to rest of theme */
  .annotator-bucket-bar {
    background: #f5f5f5 !important;
  }

  /* always hide button, toolbar, and tooltip on print */
  @media only print {
    #hypothesis_button {
      display: none;
    }

    .annotator-frame {
      display: none !important;
    }

    hypothesis-adder {
      display: none !important;
    }
  }
</style>
<!-- 
  Mathjax Plugin (third-party) 

  Allows the proper rendering of math/equations written in LaTeX.
  See https://www.mathjax.org/.
-->

<script type="text/x-mathjax-config">
  // configuration
  MathJax.Hub.Config({
    "CommonHTML": { linebreaks: { automatic: true } },
    "HTML-CSS": { linebreaks: { automatic: true } },
    "SVG": { linebreaks: { automatic: true } },
    "fast-preview": { disabled: true }
  });
</script>

<script
  src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"
  integrity="sha512-0doc9hKxR3PYwso42RD1p5ySZpzzuDiOwMrdCEh2WdJZCjcmFKc/wEnL+z8fBQrnHoiNWbo+3fiGkOYXBdQp4A=="
  crossorigin="anonymous"
></script>

<style>
  /* mathjax containers */
  .math.display > span:not(.MathJax_Preview) {
    /* turn inline element (no dimensions) into block (allows fixed width and thus scrolling) */
    display: flex !important;
    overflow-x: auto !important;
    overflow-y: hidden !important;
    justify-content: center;
    align-items: center;
    margin: 0 !important;
  }

  /* right click menu */
  .MathJax_Menu {
    border-radius: 5px !important;
    border: solid 1px #bdbdbd !important;
    box-shadow: none !important;
  }

  /* equation auto-number */
  span[id^="eq:"] > span.math.display + span {
    font-weight: 600;
  }

  /* equation */
  span[id^="eq:"] > span.math.display > span {
    /* nudge to make room for equation auto-number and anchor */
    margin-right: 60px !important;
  }
</style>
</body>
</html>
