<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <meta name="author" content="Joshua A. Shapiro" />
  <meta name="author" content="Candace L. Savonen" />
  <meta name="author" content="Chante J. Bethell" />
  <meta name="author" content="Krutika S. Gaonkar" />
  <meta name="author" content="Run Jin" />
  <meta name="author" content="Yuankun Zhu" />
  <meta name="author" content="Miguel A. Brown" />
  <meta name="author" content="Nhat Duong" />
  <meta name="author" content="Komal S. Rathi" />
  <meta name="author" content="Nighat Noureen" />
  <meta name="author" content="Bo Zhang" />
  <meta name="author" content="Brian M. Ennis" />
  <meta name="author" content="Stephanie J. Spielman" />
  <meta name="author" content="Laura E. Egolf" />
  <meta name="author" content="Bailey Farrow" />
  <meta name="author" content="Nicolas Van Kuren" />
  <meta name="author" content="Meen Chul Kim" />
  <meta name="author" content="Tejaswi Koganti" />
  <meta name="author" content="Shrivats Kannan" />
  <meta name="author" content="Pichai Raman" />
  <meta name="author" content="Jennifer Mason" />
  <meta name="author" content="Daniel P. Miller" />
  <meta name="author" content="Anna R. Poetsch" />
  <meta name="author" content="Payal Jain" />
  <meta name="author" content="Adam A. Kraya" />
  <meta name="author" content="Allison P. Heath" />
  <meta name="author" content="Mateusz P. Koptyra" />
  <meta name="author" content="Shannon Robbins" />
  <meta name="author" content="Yiran Guo" />
  <meta name="author" content="Xiaoyan Huang" />
  <meta name="author" content="Jessica Wong" />
  <meta name="author" content="Mariarita Santi" />
  <meta name="author" content="Angela Viaene" />
  <meta name="author" content="Laura Scolaro" />
  <meta name="author" content="Angela Waanders" />
  <meta name="author" content="Derek Hanson" />
  <meta name="author" content="Steven M. Foltz" />
  <meta name="author" content="Hongbo M. Xie" />
  <meta name="author" content="Siyuan Zheng" />
  <meta name="author" content="Cassie N. Kline" />
  <meta name="author" content="Jena V. Lilly" />
  <meta name="author" content="Philip B. Storm" />
  <meta name="author" content="Adam C. Resnick" />
  <meta name="author" content="Casey S. Greene" />
  <meta name="author" content="Jo Lynne Rokita*" />
  <meta name="author" content="Jaclyn N. Taroni*" />
  <meta name="author" content="Children’s Brain Tumor Network" />
  <meta name="author" content="Pacific Pediatric Neurooncology Consortium" />
  <meta name="dcterms.date" content="2022-04-11" />
  <meta name="keywords" content="pediatric cancer, brain tumor, tumor atlas" />
  <title>An Open Pediatric Brain Tumor Atlas</title>
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    span.underline{text-decoration: underline;}
    div.column{display: inline-block; vertical-align: top; width: 50%;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
  </style>
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
  <!--
  Manubot generated metadata rendered from header-includes-template.html.
  Suggest improvements at https://github.com/manubot/manubot/blob/master/manubot/process/header-includes-template.html
  -->
  <meta name="dc.format" content="text/html" />
  <meta name="dc.title" content="An Open Pediatric Brain Tumor Atlas" />
  <meta name="citation_title" content="An Open Pediatric Brain Tumor Atlas" />
  <meta property="og:title" content="An Open Pediatric Brain Tumor Atlas" />
  <meta property="twitter:title" content="An Open Pediatric Brain Tumor Atlas" />
  <meta name="dc.date" content="2022-04-11" />
  <meta name="citation_publication_date" content="2022-04-11" />
  <meta name="dc.language" content="en-US" />
  <meta name="citation_language" content="en-US" />
  <meta name="dc.relation.ispartof" content="Manubot" />
  <meta name="dc.publisher" content="Manubot" />
  <meta name="citation_journal_title" content="Manubot" />
  <meta name="citation_technical_report_institution" content="Manubot" />
  <meta name="citation_author" content="Joshua A. Shapiro" />
  <meta name="citation_author_institution" content="Childhood Cancer Data Lab, Alex&#39;s Lemonade Stand Foundation, Philadelphia, PA, USA" />
  <meta name="citation_author_orcid" content="0000-0002-6224-0347" />
  <meta name="twitter:creator" content="@jashapiro" />
  <meta name="citation_author" content="Candace L. Savonen" />
  <meta name="citation_author_institution" content="Childhood Cancer Data Lab, Alex&#39;s Lemonade Stand Foundation, Philadelphia, PA, USA" />
  <meta name="citation_author_orcid" content="0000-0001-6331-7070" />
  <meta name="twitter:creator" content="@cansavvy" />
  <meta name="citation_author" content="Chante J. Bethell" />
  <meta name="citation_author_institution" content="Childhood Cancer Data Lab, Alex&#39;s Lemonade Stand Foundation, Philadelphia, PA, USA" />
  <meta name="citation_author_orcid" content="0000-0001-9653-8128" />
  <meta name="twitter:creator" content="@cjbethell" />
  <meta name="citation_author" content="Krutika S. Gaonkar" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Department of Bioinformatics and Health Informatics, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0003-0838-2405" />
  <meta name="twitter:creator" content="@aggokittu" />
  <meta name="citation_author" content="Run Jin" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0002-8958-9266" />
  <meta name="twitter:creator" content="@runjin" />
  <meta name="citation_author" content="Yuankun Zhu" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0002-2455-9525" />
  <meta name="twitter:creator" content="@zhuyuankun" />
  <meta name="citation_author" content="Miguel A. Brown" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0001-6782-1442" />
  <meta name="twitter:creator" content="@migbro" />
  <meta name="citation_author" content="Nhat Duong" />
  <meta name="citation_author_institution" content="Department of Bioinformatics and Health Informatics, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0003-2852-4263" />
  <meta name="twitter:creator" content="@asiannhat" />
  <meta name="citation_author" content="Komal S. Rathi" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Department of Bioinformatics and Health Informatics, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0001-5534-6904" />
  <meta name="twitter:creator" content="@komalsrathi" />
  <meta name="citation_author" content="Nighat Noureen" />
  <meta name="citation_author_institution" content="Greehey Children&#39;s Cancer Research Institute, UT Health San Antonio" />
  <meta name="citation_author_orcid" content="0000-0001-7495-8201" />
  <meta name="citation_author" content="Bo Zhang" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0002-0743-5379" />
  <meta name="citation_author" content="Brian M. Ennis" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0002-2653-5009" />
  <meta name="citation_author" content="Stephanie J. Spielman" />
  <meta name="citation_author_institution" content="Childhood Cancer Data Lab, Alex&#39;s Lemonade Stand Foundation, Philadelphia, PA, USA&lt;sup&gt;☯&lt;/sup&gt;" />
  <meta name="citation_author_institution" content="Rowan University, Glassboro, NJ, USA" />
  <meta name="citation_author_orcid" content="0000-0002-9090-4788" />
  <meta name="twitter:creator" content="@stephspiel" />
  <meta name="citation_author" content="Laura E. Egolf" />
  <meta name="citation_author_institution" content="Cell and Molecular Biology Graduate Group, Perelman School of Medicine at the University of Pennsylvania" />
  <meta name="citation_author_institution" content="Division of Oncology, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0002-7103-4801" />
  <meta name="twitter:creator" content="@LauraEgolf" />
  <meta name="citation_author" content="Bailey Farrow" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0001-6727-6333" />
  <meta name="citation_author" content="Nicolas Van Kuren" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0002-7414-9516" />
  <meta name="citation_author" content="Meen Chul Kim" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0002-0308-783X" />
  <meta name="citation_author" content="Tejaswi Koganti" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0002-7733-6480" />
  <meta name="citation_author" content="Shrivats Kannan" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0002-1460-920X" />
  <meta name="twitter:creator" content="@kshrivats" />
  <meta name="citation_author" content="Pichai Raman" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Department of Bioinformatics and Health Informatics, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0001-6948-2157" />
  <meta name="twitter:creator" content="@PichaiRaman" />
  <meta name="citation_author" content="Jennifer Mason" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="twitter:creator" content="@jenn0307" />
  <meta name="citation_author" content="Daniel P. Miller" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0002-2032-4358" />
  <meta name="citation_author" content="Anna R. Poetsch" />
  <meta name="citation_author_institution" content="Biotechnology Center, Technical University Dresden, Germany" />
  <meta name="citation_author_institution" content="National Center for Tumor Diseases, Dresden, Germany" />
  <meta name="citation_author_orcid" content="0000-0003-3056-4360" />
  <meta name="twitter:creator" content="@APoetsch" />
  <meta name="citation_author" content="Payal Jain" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0002-5914-9083" />
  <meta name="twitter:creator" content="@jainpayal022" />
  <meta name="citation_author" content="Adam A. Kraya" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0002-8526-5694" />
  <meta name="citation_author" content="Allison P. Heath" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0002-2583-9668" />
  <meta name="twitter:creator" content="@allig8r" />
  <meta name="citation_author" content="Mateusz P. Koptyra" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0002-3857-6633" />
  <meta name="twitter:creator" content="@koptyram" />
  <meta name="citation_author" content="Shannon Robbins" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author" content="Yiran Guo" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0002-6549-8589" />
  <meta name="twitter:creator" content="@YiranGuo3" />
  <meta name="citation_author" content="Xiaoyan Huang" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0001-7267-4512" />
  <meta name="citation_author" content="Jessica Wong" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0003-1508-7631" />
  <meta name="twitter:creator" content="@jessicawongbfx" />
  <meta name="citation_author" content="Mariarita Santi" />
  <meta name="citation_author_institution" content="Department of Pathology and Laboratory Medicine, Children’s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Department of Pathology and Laboratory Medicine, University of Pennsylvania Perelman School of Medicine" />
  <meta name="citation_author_orcid" content="0000-0002-6728-3450" />
  <meta name="citation_author" content="Angela Viaene" />
  <meta name="citation_author_institution" content="Department of Pathology and Laboratory Medicine, Children’s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Department of Pathology and Laboratory Medicine, University of Pennsylvania Perelman School of Medicine" />
  <meta name="citation_author_orcid" content="0000-0001-6430-8360" />
  <meta name="citation_author" content="Laura Scolaro" />
  <meta name="citation_author_institution" content="Division of Oncology, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author" content="Angela Waanders" />
  <meta name="citation_author_institution" content="Department of Oncology, Ann &amp; Robert H. Lurie Children’s Hospital of Chicago" />
  <meta name="citation_author_institution" content="Department of Pediatrics, Northwestern University Feinberg School of Medicine" />
  <meta name="citation_author_orcid" content="0000-0002-0571-2889" />
  <meta name="citation_author" content="Derek Hanson" />
  <meta name="citation_author_institution" content="Hackensack Meridian School of Medicine" />
  <meta name="citation_author_institution" content="Hackensack University Medical Center" />
  <meta name="citation_author_orcid" content="0000-0002-0024-5142" />
  <meta name="citation_author" content="Steven M. Foltz" />
  <meta name="citation_author_institution" content="Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania" />
  <meta name="citation_author_institution" content="Childhood Cancer Data Lab, Alex&#39;s Lemonade Stand Foundation, Philadelphia, PA, USA" />
  <meta name="citation_author_orcid" content="0000-0002-9526-8194" />
  <meta name="citation_author" content="Hongbo M. Xie" />
  <meta name="citation_author_institution" content="Department of Bioinformatics and Health Informatics, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0003-2223-0029" />
  <meta name="twitter:creator" content="@xiehb" />
  <meta name="citation_author" content="Siyuan Zheng" />
  <meta name="citation_author_institution" content="Greehey Children&#39;s Cancer Research Institute, UT Health San Antonio" />
  <meta name="citation_author_orcid" content="0000-0002-1031-9424" />
  <meta name="twitter:creator" content="@zhengsiyuan" />
  <meta name="citation_author" content="Cassie N. Kline" />
  <meta name="citation_author_institution" content="Division of Oncology, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0001-7765-7690" />
  <meta name="twitter:creator" content="@cnkline13" />
  <meta name="citation_author" content="Jena V. Lilly" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0003-1439-6045" />
  <meta name="twitter:creator" content="@jvlilly" />
  <meta name="citation_author" content="Philip B. Storm" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0002-7964-2449" />
  <meta name="citation_author" content="Adam C. Resnick" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0003-0436-4189" />
  <meta name="twitter:creator" content="@adamcresnick" />
  <meta name="citation_author" content="Casey S. Greene" />
  <meta name="citation_author_institution" content="Department of Systems Pharmacology and Translational Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA, USA" />
  <meta name="citation_author_institution" content="Childhood Cancer Data Lab, Alex&#39;s Lemonade Stand Foundation, Philadelphia, PA, USA" />
  <meta name="citation_author_institution" content="Center for Health AI, University of Colorado School of Medicine, Aurora, CO, USA" />
  <meta name="citation_author_institution" content="Department of Biochemistry and Molecular Genetics, University of Colorado School of Medicine, Aurora, CO, USA" />
  <meta name="citation_author_orcid" content="0000-0001-8713-9213" />
  <meta name="twitter:creator" content="@greenescientist" />
  <meta name="citation_author" content="Jo Lynne Rokita*" />
  <meta name="citation_author_institution" content="Center for Data-Driven Discovery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Division of Neurosurgery, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_institution" content="Department of Bioinformatics and Health Informatics, Children&#39;s Hospital of Philadelphia" />
  <meta name="citation_author_orcid" content="0000-0003-2171-3627" />
  <meta name="twitter:creator" content="@jolynnerokita" />
  <meta name="citation_author" content="Jaclyn N. Taroni*" />
  <meta name="citation_author_institution" content="Childhood Cancer Data Lab, Alex&#39;s Lemonade Stand Foundation, Philadelphia, PA, USA" />
  <meta name="citation_author_orcid" content="0000-0003-4734-4508" />
  <meta name="twitter:creator" content="@jaclyn_taroni" />
  <meta name="citation_author" content="Children&#39;s Brain Tumor Network" />
  <meta name="citation_author" content="Pacific Pediatric Neurooncology Consortium" />
  <link rel="canonical" href="https://AlexsLemonade.github.io/OpenPBTA-manuscript/" />
  <meta property="og:url" content="https://AlexsLemonade.github.io/OpenPBTA-manuscript/" />
  <meta property="twitter:url" content="https://AlexsLemonade.github.io/OpenPBTA-manuscript/" />
  <meta name="citation_fulltext_html_url" content="https://AlexsLemonade.github.io/OpenPBTA-manuscript/" />
  <meta name="citation_pdf_url" content="https://AlexsLemonade.github.io/OpenPBTA-manuscript/manuscript.pdf" />
  <link rel="alternate" type="application/pdf" href="https://AlexsLemonade.github.io/OpenPBTA-manuscript/manuscript.pdf" />
  <link rel="alternate" type="text/html" href="https://AlexsLemonade.github.io/OpenPBTA-manuscript/v/5f5fb1408fdbc35a89604a8bcc938d4e8221357d/" />
  <meta name="manubot_html_url_versioned" content="https://AlexsLemonade.github.io/OpenPBTA-manuscript/v/5f5fb1408fdbc35a89604a8bcc938d4e8221357d/" />
  <meta name="manubot_pdf_url_versioned" content="https://AlexsLemonade.github.io/OpenPBTA-manuscript/v/5f5fb1408fdbc35a89604a8bcc938d4e8221357d/manuscript.pdf" />
  <meta property="og:type" content="article" />
  <meta property="twitter:card" content="summary_large_image" />
  <link rel="icon" type="image/png" sizes="192x192" href="https://manubot.org/favicon-192x192.png" />
  <link rel="mask-icon" href="https://manubot.org/safari-pinned-tab.svg" color="#ad1457" />
  <meta name="theme-color" content="#ad1457" />
  <!-- end Manubot generated metadata -->
</head>
<body>
<header id="title-block-header">
<h1 class="title">An Open Pediatric Brain Tumor Atlas</h1>
</header>
<p><small><em>
This manuscript
(<a href="https://AlexsLemonade.github.io/OpenPBTA-manuscript/v/5f5fb1408fdbc35a89604a8bcc938d4e8221357d/">permalink</a>)
was automatically generated
from <a href="https://github.com/AlexsLemonade/OpenPBTA-manuscript/tree/5f5fb1408fdbc35a89604a8bcc938d4e8221357d">AlexsLemonade/OpenPBTA-manuscript@5f5fb14</a>
on April 11, 2022.
</em></small></p>
<h2 id="authors">Authors</h2>
<ul>
<li><p><strong>Joshua A. Shapiro</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-6224-0347">0000-0002-6224-0347</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/jashapiro">jashapiro</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/jashapiro">jashapiro</a><br>
<small><br />
Childhood Cancer Data Lab, Alex’s Lemonade Stand Foundation, Philadelphia, PA, USA
· Funded by Alex’s Lemonade Stand Foundation Childhood Cancer Data Lab (CCDL)
</small></p></li>
<li><p><strong>Candace L. Savonen</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-6331-7070">0000-0001-6331-7070</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/cansavvy">cansavvy</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/cansavvy">cansavvy</a><br>
<small><br />
Childhood Cancer Data Lab, Alex’s Lemonade Stand Foundation, Philadelphia, PA, USA
· Funded by Alex’s Lemonade Stand Foundation Childhood Cancer Data Lab (CCDL)
</small></p></li>
<li><p><strong>Chante J. Bethell</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-9653-8128">0000-0001-9653-8128</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/cbethell">cbethell</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/cjbethell">cjbethell</a><br>
<small><br />
Childhood Cancer Data Lab, Alex’s Lemonade Stand Foundation, Philadelphia, PA, USA
· Funded by Alex’s Lemonade Stand Foundation Childhood Cancer Data Lab (CCDL)
</small></p></li>
<li><p><strong>Krutika S. Gaonkar</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-0838-2405">0000-0003-0838-2405</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/kgaonkar6">kgaonkar6</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/aggokittu">aggokittu</a><br>
<small><br />
Center for Data-Driven Discovery, Children’s Hospital of Philadelphia; Division of Neurosurgery, Children’s Hospital of Philadelphia; Department of Bioinformatics and Health Informatics, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Run Jin</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-8958-9266">0000-0002-8958-9266</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/runjin326">runjin326</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/runjin">runjin</a><br>
<small><br />
Center for Data-Driven Discovery, Children’s Hospital of Philadelphia; Division of Neurosurgery, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Yuankun Zhu</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-2455-9525">0000-0002-2455-9525</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/yuankunzhu">yuankunzhu</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/zhuyuankun">zhuyuankun</a><br>
<small><br />
Center for Data-Driven Discovery, Children’s Hospital of Philadelphia; Division of Neurosurgery, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Miguel A. Brown</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-6782-1442">0000-0001-6782-1442</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/migbro">migbro</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/migbro">migbro</a><br>
<small><br />
Center for Data-Driven Discovery, Children’s Hospital of Philadelphia; Division of Neurosurgery, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Nhat Duong</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-2852-4263">0000-0003-2852-4263</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/fingerfen">fingerfen</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/asiannhat">asiannhat</a><br>
<small><br />
Department of Bioinformatics and Health Informatics, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Komal S. Rathi</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-5534-6904">0000-0001-5534-6904</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/komalsrathi">komalsrathi</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/komalsrathi">komalsrathi</a><br>
<small><br />
Center for Data-Driven Discovery, Children’s Hospital of Philadelphia; Department of Bioinformatics and Health Informatics, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Nighat Noureen</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-7495-8201">0000-0001-7495-8201</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/NNoureen">NNoureen</a><br>
<small><br />
Greehey Children’s Cancer Research Institute, UT Health San Antonio
</small></p></li>
<li><p><strong>Bo Zhang</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-0743-5379">0000-0002-0743-5379</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/zhangb1">zhangb1</a><br>
<small><br />
Center for Data-Driven Discovery, Children’s Hospital of Philadelphia; Division of Neurosurgery, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Brian M. Ennis</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-2653-5009">0000-0002-2653-5009</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/bmennis">bmennis</a><br>
<small><br />
Center for Data-Driven Discovery, Children’s Hospital of Philadelphia; Division of Neurosurgery, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Stephanie J. Spielman</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-9090-4788">0000-0002-9090-4788</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/sjspielman">sjspielman</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/stephspiel">stephspiel</a><br>
<small><br />
Childhood Cancer Data Lab, Alex’s Lemonade Stand Foundation, Philadelphia, PA, USA<sup>☯</sup>; Rowan University, Glassboro, NJ, USA
· Funded by Alex’s Lemonade Stand Foundation Childhood Cancer Data Lab (CCDL)
<br>
<sup>☯</sup>Current affiliation
</small></p></li>
<li><p><strong>Laura E. Egolf</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-7103-4801">0000-0002-7103-4801</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/LauraEgolf">LauraEgolf</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/LauraEgolf">LauraEgolf</a><br>
<small><br />
Cell and Molecular Biology Graduate Group, Perelman School of Medicine at the University of Pennsylvania; Division of Oncology, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Bailey Farrow</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-6727-6333">0000-0001-6727-6333</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/baileyckelly">baileyckelly</a><br>
<small><br />
Center for Data-Driven Discovery, Children’s Hospital of Philadelphia; Division of Neurosurgery, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Nicolas Van Kuren</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-7414-9516">0000-0002-7414-9516</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/nicholasvk">nicholasvk</a><br>
<small><br />
Center for Data-Driven Discovery, Children’s Hospital of Philadelphia; Division of Neurosurgery, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Meen Chul Kim</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-0308-783X">0000-0002-0308-783X</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/liberaliscomputing">liberaliscomputing</a><br>
<small><br />
Center for Data-Driven Discovery, Children’s Hospital of Philadelphia; Division of Neurosurgery, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Tejaswi Koganti</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-7733-6480">0000-0002-7733-6480</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/tkoganti">tkoganti</a><br>
<small><br />
Center for Data-Driven Discovery, Children’s Hospital of Philadelphia; Division of Neurosurgery, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Shrivats Kannan</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-1460-920X">0000-0002-1460-920X</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/shrivatsk">shrivatsk</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/kshrivats">kshrivats</a><br>
<small><br />
Center for Data-Driven Discovery, Children’s Hospital of Philadelphia; Division of Neurosurgery, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Pichai Raman</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-6948-2157">0000-0001-6948-2157</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/pichairaman">pichairaman</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/PichaiRaman">PichaiRaman</a><br>
<small><br />
Center for Data-Driven Discovery, Children’s Hospital of Philadelphia; Department of Bioinformatics and Health Informatics, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Jennifer Mason</strong><br>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/jenn0307">jenn0307</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/jenn0307">jenn0307</a><br>
<small><br />
Center for Data-Driven Discovery, Children’s Hospital of Philadelphia; Division of Neurosurgery, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Daniel P. Miller</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-2032-4358">0000-0002-2032-4358</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/dmiller15">dmiller15</a><br>
<small><br />
Center for Data-Driven Discovery, Children’s Hospital of Philadelphia; Division of Neurosurgery, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Anna R. Poetsch</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-3056-4360">0000-0003-3056-4360</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/arpoe">arpoe</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/APoetsch">APoetsch</a><br>
<small><br />
Biotechnology Center, Technical University Dresden, Germany; National Center for Tumor Diseases, Dresden, Germany
</small></p></li>
<li><p><strong>Payal Jain</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-5914-9083">0000-0002-5914-9083</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/jainpayal022">jainpayal022</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/jainpayal022">jainpayal022</a><br>
<small><br />
Center for Data-Driven Discovery, Children’s Hospital of Philadelphia; Division of Neurosurgery, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Adam A. Kraya</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-8526-5694">0000-0002-8526-5694</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/aadamk">aadamk</a><br>
<small><br />
Center for Data-Driven Discovery, Children’s Hospital of Philadelphia; Division of Neurosurgery, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Allison P. Heath</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-2583-9668">0000-0002-2583-9668</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/allisonheath">allisonheath</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/allig8r">allig8r</a><br>
<small><br />
Center for Data-Driven Discovery, Children’s Hospital of Philadelphia; Division of Neurosurgery, Children’s Hospital of Philadelphia
· Funded by NIH U2C HL138346-03; NCI/NIH Contract No. 75N91019D00024, Task Order No. 75N91020F00003; Australian Government, Department of Education
</small></p></li>
<li><p><strong>Mateusz P. Koptyra</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-3857-6633">0000-0002-3857-6633</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/mkoptyra">mkoptyra</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/koptyram">koptyram</a><br>
<small><br />
Center for Data-Driven Discovery, Children’s Hospital of Philadelphia; Division of Neurosurgery, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Shannon Robbins</strong><br><br>
<small><br />
Center for Data-Driven Discovery, Children’s Hospital of Philadelphia; Division of Neurosurgery, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Yiran Guo</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-6549-8589">0000-0002-6549-8589</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/Yiran-Guo">Yiran-Guo</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/YiranGuo3">YiranGuo3</a><br>
<small><br />
Center for Data-Driven Discovery, Children’s Hospital of Philadelphia; Division of Neurosurgery, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Xiaoyan Huang</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-7267-4512">0000-0001-7267-4512</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/HuangXiaoyan0106">HuangXiaoyan0106</a><br>
<small><br />
Center for Data-Driven Discovery, Children’s Hospital of Philadelphia; Division of Neurosurgery, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Jessica Wong</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-1508-7631">0000-0003-1508-7631</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/wongjessica93">wongjessica93</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/jessicawongbfx">jessicawongbfx</a><br>
<small><br />
Center for Data-Driven Discovery, Children’s Hospital of Philadelphia; Division of Neurosurgery, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Mariarita Santi</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-6728-3450">0000-0002-6728-3450</a><br>
<small><br />
Department of Pathology and Laboratory Medicine, Children’s Hospital of Philadelphia; Department of Pathology and Laboratory Medicine, University of Pennsylvania Perelman School of Medicine
</small></p></li>
<li><p><strong>Angela Viaene</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-6430-8360">0000-0001-6430-8360</a><br>
<small><br />
Department of Pathology and Laboratory Medicine, Children’s Hospital of Philadelphia; Department of Pathology and Laboratory Medicine, University of Pennsylvania Perelman School of Medicine
</small></p></li>
<li><p><strong>Laura Scolaro</strong><br><br>
<small><br />
Division of Oncology, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Angela Waanders</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-0571-2889">0000-0002-0571-2889</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/awaanders">awaanders</a><br>
<small><br />
Department of Oncology, Ann &amp; Robert H. Lurie Children’s Hospital of Chicago; Department of Pediatrics, Northwestern University Feinberg School of Medicine
</small></p></li>
<li><p><strong>Derek Hanson</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-0024-5142">0000-0002-0024-5142</a><br>
<small><br />
Hackensack Meridian School of Medicine; Hackensack University Medical Center
</small></p></li>
<li><p><strong>Steven M. Foltz</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-9526-8194">0000-0002-9526-8194</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/envest">envest</a><br>
<small><br />
Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania; Childhood Cancer Data Lab, Alex’s Lemonade Stand Foundation, Philadelphia, PA, USA
</small></p></li>
<li><p><strong>Hongbo M. Xie</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-2223-0029">0000-0003-2223-0029</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/xiehongbo">xiehongbo</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/xiehb">xiehb</a><br>
<small><br />
Department of Bioinformatics and Health Informatics, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Siyuan Zheng</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-1031-9424">0000-0002-1031-9424</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/syzheng">syzheng</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/zhengsiyuan">zhengsiyuan</a><br>
<small><br />
Greehey Children’s Cancer Research Institute, UT Health San Antonio
</small></p></li>
<li><p><strong>Cassie N. Kline</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-7765-7690">0000-0001-7765-7690</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/cnkline13">cnkline13</a><br>
<small><br />
Division of Oncology, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Jena V. Lilly</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-1439-6045">0000-0003-1439-6045</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/jvlilly">jvlilly</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/jvlilly">jvlilly</a><br>
<small><br />
Center for Data-Driven Discovery, Children’s Hospital of Philadelphia; Division of Neurosurgery, Children’s Hospital of Philadelphia
</small></p></li>
<li><p><strong>Philip B. Storm</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-7964-2449">0000-0002-7964-2449</a><br>
<small><br />
Center for Data-Driven Discovery, Children’s Hospital of Philadelphia; Division of Neurosurgery, Children’s Hospital of Philadelphia
· Funded by Alex’s Lemonade Stand Foundation (Catalyst); Children’s Hospital of Philadelphia Division of Neurosurgery
</small></p></li>
<li><p><strong>Adam C. Resnick</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-0436-4189">0000-0003-0436-4189</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/adamcresnick">adamcresnick</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/adamcresnick">adamcresnick</a><br>
<small><br />
Center for Data-Driven Discovery, Children’s Hospital of Philadelphia; Division of Neurosurgery, Children’s Hospital of Philadelphia
· Funded by Alex’s Lemonade Stand Foundation (Catalyst); Children’s Brain Tumor Network; NIH 3P30 CA016520-44S5, U2C HL138346-03, U24 CA220457-03; NCI/NIH Contract No. 75N91019D00024, Task Order No. 75N91020F00003; Children’s Hospital of Philadelphia Division of Neurosurgery
</small></p></li>
<li><p><strong>Casey S. Greene</strong><br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-8713-9213">0000-0001-8713-9213</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/cgreene">cgreene</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/greenescientist">greenescientist</a><br>
<small><br />
Department of Systems Pharmacology and Translational Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA, USA; Childhood Cancer Data Lab, Alex’s Lemonade Stand Foundation, Philadelphia, PA, USA; Center for Health AI, University of Colorado School of Medicine, Aurora, CO, USA; Department of Biochemistry and Molecular Genetics, University of Colorado School of Medicine, Aurora, CO, USA
· Funded by Alex’s Lemonade Stand Foundation Childhood Cancer Data Lab (CCDL)
</small></p></li>
<li><p><strong>Jo Lynne Rokita</strong>*<br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-2171-3627">0000-0003-2171-3627</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/jharenza">jharenza</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/jolynnerokita">jolynnerokita</a><br>
<small><br />
Center for Data-Driven Discovery, Children’s Hospital of Philadelphia; Division of Neurosurgery, Children’s Hospital of Philadelphia; Department of Bioinformatics and Health Informatics, Children’s Hospital of Philadelphia
· Funded by Alex’s Lemonade Stand Foundation (Young Investigator, Catalyst); NCI/NIH Contract No. 75N91019D00024, Task Order No. 75N91020F00003
</small></p></li>
<li><p><strong>Jaclyn N. Taroni</strong>*<br>
<img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-4734-4508">0000-0003-4734-4508</a>
· <img src="images/github.svg" class="inline_icon" alt="GitHub icon" />
<a href="https://github.com/jaclyn-taroni">jaclyn-taroni</a>
· <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" />
<a href="https://twitter.com/jaclyn_taroni">jaclyn_taroni</a><br>
<small><br />
Childhood Cancer Data Lab, Alex’s Lemonade Stand Foundation, Philadelphia, PA, USA
· Funded by Alex’s Lemonade Stand Foundation Childhood Cancer Data Lab (CCDL)
</small></p></li>
<li><p><strong>Children’s Brain Tumor Network</strong><br><br>
<small>
</small></p></li>
<li><p><strong>Pacific Pediatric Neurooncology Consortium</strong><br><br>
<small>
</small></p></li>
</ul>
<h2 id="contact-information">Contact information</h2>
<p>*Correspondence: jaclyn.taroni@ccdatalab.org, rokita@chop.edu</p>
<h2 id="in-brief">In Brief</h2>
<!-- to add -->
<h2 id="highlights">Highlights</h2>
<!-- Highlights are bullet points that convey the core findings of your paper. You may include up to four highlights. The length of each highlight cannot exceed 85 characters (including spaces).-->
<h2 id="summary">Summary</h2>
<!-- The summary is a single paragraph no longer than 150 words. An effective summary includes the following elements: (1) a brief background of the question that avoids statements about how a process is not well understood; (2) a description of the results and approaches/model systems framed in the context of their conceptual interest; and (3) an indication of the broader significance of the work. We discourage novelty claims (e.g., use of the word “novel”) because they are overused, tend not to add meaning, and are difficult to verify. Please do not include references in the summary. -->
<h2 id="keywords">Keywords</h2>
<!-- We encourage you to include up to ten keywords with your paper. These keywords will be associated with your paper on Cell Press platforms and on PubMed. These keywords should be listed in the manuscript after the summary, separated by commas. -->
<p>pediatric brain tumors, somatic variation, open science, classification</p>
<h2 id="introduction">Introduction</h2>
<p>Pediatric brain and spinal cord tumors are the second most common tumors in children after leukemia, yet they represent the leading disease-related cause of death in children <span class="citation" data-cites="nAMOAA9j">[<a href="#ref-nAMOAA9j" role="doc-biblioref">1</a>]</span>.
Five-year survival rates vary widely across different histologic and molecular classifications of brain tumors.
For example, most high-grade and embryonal tumors carry a universally fatal prognosis while children with pilocytic astrocytoma have an estimated 10-year survival rate of 92% <span class="citation" data-cites="1H6hugOV3">[<a href="#ref-1H6hugOV3" role="doc-biblioref">2</a>]</span>.
Despite their relative rarity, the years of potential life lost due to brain tumors in 2009 was estimated at 47,631 years for children and adolescents aged 0-19 in the United States <span class="citation" data-cites="doi:10.1159/00009389610.1159/000093896">[<span class="citeproc-not-found" data-reference-id="doi:10.1159/00009389610.1159/000093896"><strong>???</strong></span>]</span>.
The low survival rates for some tumors are clearly multifactorial but can be explained in part by our lack of understanding of the ever-evolving array of brain tumor molecular subtypes, difficulty drugging these entities, and the shortage of drugs specifically labeled for pediatric malignancies.
Historically, some of the most fatal, inoperable brain tumors, such as diffuse midline gliomas, were not routinely biopsied due to perceived risks of biopsy and the paucity of therapeutic options that would require tissue.
Limited access to tissue to develop patient-derived cell line and mouse models has been a barrier to research.
Furthermore, the incidence of any single molecular tumor entity is relatively low due to the rarity of pediatric tumors in general.
Together, these factors have hindered research progress and have led to multiple national and international center and consortia efforts to collaboratively share specimens and data to accelerate breakthroughs and clinical translation.</p>
<p>There has been significant progress in recent years to elucidate the landscape of somatic variation responsible for pediatric brain tumor formation and progression, however, translation of therapeutic agents to phase II or III clinical trials and subsequent FDA approval has been slow.
Within the last 20 years, the FDA has approved only five drugs for the treatment of pediatric brain tumors: mTOR inhibitor, everolimus, for subependymal giant cell astrocytoma; anti-PD-1 immunotherapy, pembrolizumab, for microsatellite instability–high or mismatch repair–deficient tumors; NTRK inhibitors larotrectinib and entrectinib for tumors with an NTRK 1/2/3 gene fusions; MEK1/2 inhibitor, selumetinib, for neurofibromatosis type 1 (NF1) and symptomatic, inoperable plexiform neurofibromas.
This is, in part, due to pharmaceutical company priorities and/or concerns regarding toxicity that have resulted in an inability to obtain drugs for pediatric clinical trials, ultimately delaying access to new agents.
An amendment to the Pediatric Research Equity Act called the Research to Accelerate Cures and Equity (RACE) for Children Act mandates that as of August 18, 2020 all new adult oncology drugs also be tested in children when the molecular targets are relevant to a particular childhood cancer.
Here, we present a comprehensive, collaborative, open genomic analysis of 943 patient tumors from 59 distinct brain tumor histologies which can be used to support the RACE Act and accelerate rational clinical trial design.</p>
<h2 id="results">Results</h2>
<h3 id="crowd-sourced-somatic-analysis-to-create-an-open-pediatric-brain-tumor-atlas">Crowd-sourced Somatic Analysis to create an Open Pediatric Brain Tumor Atlas</h3>
<p>We previously performed whole genome sequencing (WGS), whole exome sequencing (WXS), and RNA sequencing (RNA-Seq) on matched tumor and normal tissues as well as selected cell lines from 943 patient tumors from the Pediatric Brain Tumor Atlas (PBTA) <span class="citation" data-cites="DwCzMoBJ">[<a href="#ref-DwCzMoBJ" role="doc-biblioref">3</a>]</span>, consisting of samples from the Children’s Brain Tumor Network (CBTN) <span class="citation" data-cites="VT338cRv">[<a href="#ref-VT338cRv" role="doc-biblioref">4</a>]</span> and the PNOC003 DMG clinical trial <span class="citation" data-cites="BLNV1DbL">[<a href="#ref-BLNV1DbL" role="doc-biblioref">5</a>]</span> of the Pacific Pediatric Neuro-Oncology Symposium (PNOC) (<strong>Figure <a href="#fig:Fig1">1</a>A</strong>).
We then harnessed the benchmarking efforts of the Kids First Data Resource Center to develop a robust and reproducible data analysis workflow within the CAVATICA platform to perform the following primary somatic analyses: variant calling of single nucleotide variants (SNVs), copy number variants (CNVs), structural variants (SVs), and fusions (<strong>Figure S1</strong> - red boxes and <strong>STAR Methods</strong>).
Next, we created a public Github analysis repository (https://github.com/AlexsLemonade/OpenPBTA-analysis) with continuous integration to ensure analysis reproducibility.
We additionally created a GitHub manuscript repository (https://github.com/AlexsLemonade/OpenPBTA-manuscript) with ManuBot <span class="citation" data-cites="petulEc2 YuJbg3zO">[<a href="#ref-petulEc2" role="doc-biblioref">6</a>,<a href="#ref-YuJbg3zO" role="doc-biblioref">7</a>]</span> integration to enable manuscript creation using Markdown within GitHub.</p>
<p>We maintained a data release folder on Amazon S3 containing merged files for each analysis, downloadable from the GitHub repository or open access CAVATICA project.</p>
<!-- TODO: Stub for the process for contributing analytical code (**Figure {@fig:FigS1}** and **STAR Methods**) and to the manuscript. -->
<div id="fig:Fig1" class="fignos">
<figure>
<img src="images/main/Figure1.png" alt="" /><figcaption><span>Figure 1:</span> <strong>Figure 1. Overview of the OpenPBTA Project.</strong> A, The Children’s Brain Tumor Network and the Pacific Pediatric Neuro-Oncology Consortium collected tumor samples from 943 patients. To date, 38 cell lines were created from tumor tissue and over 2000 specimens were sequenced (N = 1035 RNA-Seq, N = 940 WGS, and N = 32 WXS or Targeted panel). Data was harmonized by the Kids First Data Resource Center using an Amazon S3 framework within CAVATICA. B, Processed data was released through the OpenPBTA GitHub project, through which contributors filed issues and developed code for analyses within docker. Additional analysis details can be found in <strong>Figure S1</strong>. Simultaneously, this manuscript was written in GitHub using Manubot. All analysis and manuscript code was reviewed and continuously integrated using CircleCI. C, Stacked bar plot summary of the number of biospecimens per phase of therapy per broad histology.</figcaption>
</figure>
</div>
<h3 id="molecular-subtyping-of-openpbta-cns-tumors">Molecular Subtyping of OpenPBTA CNS Tumors</h3>
<p>Over the past two decades, together with the World Health Organization (WHO), experts in neurooncology have iteratively redefined the classifications of central nervous system (CNS) tumors <span class="citation" data-cites="doi 14TDMX5t7">[<span class="citeproc-not-found" data-reference-id="doi"><strong>???</strong></span> 10.1093/jnen/61.3.215,<a href="#ref-14TDMX5t7" role="doc-biblioref">8</a>]</span>.
More recently, in 2016 and 2021 <span class="citation" data-cites="7sgrMadR 6F5vK3sT">[<a href="#ref-7sgrMadR" role="doc-biblioref">9</a>,<a href="#ref-6F5vK3sT" role="doc-biblioref">10</a>]</span>, molecular subtypes have been integrated into these entities.
The Children’s Brain Tumor Tissue Consortium (CBTTC), currently the Children’s Brain Tumor Network (CBTN), opened its protocol for brain tumor and matched normal sample collection in 2011 and as such, the majority of the samples within the OpenPBTA dataset lack molecular subtype annotation.
In the absence of methylation arrays for tumor classification, we utilized key genomic features of entities described by the WHO in 2016, as well as Ryall and colleagues <span class="citation" data-cites="ULPByXgG">[<a href="#ref-ULPByXgG" role="doc-biblioref">11</a>]</span>, coupled with clinician and pathologist review to subtype 64% (1,281/2,007) of tumor biospecimens with high confidence (<strong>Table S1</strong>).
Importantly, this collaborative molecular subtyping process allowed us to identify data entry errors (e.g., an ETMR entered as a medulloblastoma) and mis-identified specimens (e.g., Ewing sarcoma sample labeled as a craniopharyngioma), update diagnoses using modern terms (e.g., primitive neuro-ectodermal tumor [PNET] diagnoses), and discover rarer tumor entities within the OpenPBTA (e.g., H3-mutant ependymoma, meningioma with <em>YAP1::FAM118B</em> fusion).
<strong>Table 1</strong> {#tbl:subtypes} lists the subtypes we defined within the OpenPBTA, comprising of LGATs (N = 569), high-grade gliomas (N = 314), embryonal tumors (N = 229), ependymomas (N = 65), tumors of sellar region (N = 51), mesenchymal non-meningothelial tumors (N = 21), glialneuronal tumors (N = 20), and chordomas (N = 12). For methods, see <strong>STAR Methods</strong> and <strong>Figure S1</strong>.</p>
<table>
<colgroup>
<col style="width: 49%" />
<col style="width: 44%" />
<col style="width: 6%" />
</colgroup>
<thead>
<tr class="header">
<th>Broad histology</th>
<th>molecular subtype</th>
<th>n</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Chordoma</td>
<td>CHDM, conventional</td>
<td>4</td>
</tr>
<tr class="even">
<td>Chordoma</td>
<td>CHDM, poorly differentiated</td>
<td>8</td>
</tr>
<tr class="odd">
<td>Diffuse astrocytic and oligodendroglial tumor</td>
<td>DMG, H3 K28</td>
<td>54</td>
</tr>
<tr class="even">
<td>Diffuse astrocytic and oligodendroglial tumor</td>
<td>DMG, H3 K28, TP53 activated</td>
<td>26</td>
</tr>
<tr class="odd">
<td>Diffuse astrocytic and oligodendroglial tumor</td>
<td>DMG, H3 K28, TP53 loss</td>
<td>91</td>
</tr>
<tr class="even">
<td>Diffuse astrocytic and oligodendroglial tumor</td>
<td>HGG, H3 G35</td>
<td>7</td>
</tr>
<tr class="odd">
<td>Diffuse astrocytic and oligodendroglial tumor</td>
<td>HGG, H3 G35, TP53 loss</td>
<td>2</td>
</tr>
<tr class="even">
<td>Diffuse astrocytic and oligodendroglial tumor</td>
<td>HGG, H3 wildtype</td>
<td>74</td>
</tr>
<tr class="odd">
<td>Diffuse astrocytic and oligodendroglial tumor</td>
<td>HGG, H3 wildtype, TP53 activated</td>
<td>10</td>
</tr>
<tr class="even">
<td>Diffuse astrocytic and oligodendroglial tumor</td>
<td>HGG, H3 wildtype, TP53 loss</td>
<td>45</td>
</tr>
<tr class="odd">
<td>Diffuse astrocytic and oligodendroglial tumor</td>
<td>HGG, IDH, TP53 activated</td>
<td>3</td>
</tr>
<tr class="even">
<td>Diffuse astrocytic and oligodendroglial tumor</td>
<td>HGG, IDH, TP53 loss</td>
<td>2</td>
</tr>
<tr class="odd">
<td>Embryonal tumor</td>
<td>CNS Embryonal, NOS</td>
<td>24</td>
</tr>
<tr class="even">
<td>Embryonal tumor</td>
<td>CNS HGNET-MN1</td>
<td>1</td>
</tr>
<tr class="odd">
<td>Embryonal tumor</td>
<td>CNS NB-FOXR2</td>
<td>5</td>
</tr>
<tr class="even">
<td>Embryonal tumor</td>
<td>ETMR, C19MC-altered</td>
<td>8</td>
</tr>
<tr class="odd">
<td>Embryonal tumor</td>
<td>ETMR, NOS</td>
<td>1</td>
</tr>
<tr class="even">
<td>Embryonal tumor</td>
<td>MB, Group3</td>
<td>24</td>
</tr>
<tr class="odd">
<td>Embryonal tumor</td>
<td>MB, Group4</td>
<td>91</td>
</tr>
<tr class="even">
<td>Embryonal tumor</td>
<td>MB, SHH</td>
<td>55</td>
</tr>
<tr class="odd">
<td>Embryonal tumor</td>
<td>MB, WNT</td>
<td>20</td>
</tr>
<tr class="even">
<td>Ependymal tumor</td>
<td>EPN, H3 K28</td>
<td>2</td>
</tr>
<tr class="odd">
<td>Ependymal tumor</td>
<td>EPN, PF A</td>
<td>6</td>
</tr>
<tr class="even">
<td>Ependymal tumor</td>
<td>EPN, ST RELA</td>
<td>51</td>
</tr>
<tr class="odd">
<td>Ependymal tumor</td>
<td>EPN, ST YAP1</td>
<td>6</td>
</tr>
<tr class="even">
<td>Low-grade astrocytic tumor</td>
<td>GNG, BRAF V600E</td>
<td>25</td>
</tr>
<tr class="odd">
<td>Low-grade astrocytic tumor</td>
<td>GNG, BRAF V600E, CDKN2A/B</td>
<td>2</td>
</tr>
<tr class="even">
<td>Low-grade astrocytic tumor</td>
<td>GNG, FGFR</td>
<td>2</td>
</tr>
<tr class="odd">
<td>Low-grade astrocytic tumor</td>
<td>GNG, H3</td>
<td>2</td>
</tr>
<tr class="even">
<td>Low-grade astrocytic tumor</td>
<td>GNG, IDH</td>
<td>4</td>
</tr>
<tr class="odd">
<td>Low-grade astrocytic tumor</td>
<td>GNG, KIAA1549-BRAF</td>
<td>10</td>
</tr>
<tr class="even">
<td>Low-grade astrocytic tumor</td>
<td>GNG, MYB/MYBL1</td>
<td>2</td>
</tr>
<tr class="odd">
<td>Low-grade astrocytic tumor</td>
<td>GNG, NF1-germline</td>
<td>2</td>
</tr>
<tr class="even">
<td>Low-grade astrocytic tumor</td>
<td>GNG, NF1-somatic, BRAF V600E</td>
<td>1</td>
</tr>
<tr class="odd">
<td>Low-grade astrocytic tumor</td>
<td>GNG, other MAPK</td>
<td>7</td>
</tr>
<tr class="even">
<td>Low-grade astrocytic tumor</td>
<td>GNG, other MAPK, IDH</td>
<td>2</td>
</tr>
<tr class="odd">
<td>Low-grade astrocytic tumor</td>
<td>GNG, RTK</td>
<td>6</td>
</tr>
<tr class="even">
<td>Low-grade astrocytic tumor</td>
<td>GNG, wildtype</td>
<td>28</td>
</tr>
<tr class="odd">
<td>Low-grade astrocytic tumor</td>
<td>LGG, BRAF V600E</td>
<td>53</td>
</tr>
<tr class="even">
<td>Low-grade astrocytic tumor</td>
<td>LGG, BRAF V600E, CDKN2A/B</td>
<td>10</td>
</tr>
<tr class="odd">
<td>Low-grade astrocytic tumor</td>
<td>LGG, FGFR</td>
<td>16</td>
</tr>
<tr class="even">
<td>Low-grade astrocytic tumor</td>
<td>LGG, IDH</td>
<td>6</td>
</tr>
<tr class="odd">
<td>Low-grade astrocytic tumor</td>
<td>LGG, KIAA1549-BRAF</td>
<td>222</td>
</tr>
<tr class="even">
<td>Low-grade astrocytic tumor</td>
<td>LGG, KIAA1549-BRAF, other MAPK</td>
<td>2</td>
</tr>
<tr class="odd">
<td>Low-grade astrocytic tumor</td>
<td>LGG, MYB/MYBL1</td>
<td>4</td>
</tr>
<tr class="even">
<td>Low-grade astrocytic tumor</td>
<td>LGG, NF1-germline</td>
<td>12</td>
</tr>
<tr class="odd">
<td>Low-grade astrocytic tumor</td>
<td>LGG, NF1-germline, CDKN2A/B</td>
<td>2</td>
</tr>
<tr class="even">
<td>Low-grade astrocytic tumor</td>
<td>LGG, NF1-germline, FGFR</td>
<td>4</td>
</tr>
<tr class="odd">
<td>Low-grade astrocytic tumor</td>
<td>LGG, NF1-somatic</td>
<td>4</td>
</tr>
<tr class="even">
<td>Low-grade astrocytic tumor</td>
<td>LGG, NF1-somatic, FGFR</td>
<td>2</td>
</tr>
<tr class="odd">
<td>Low-grade astrocytic tumor</td>
<td>LGG, NF1-somatic, NF1-germline, CDKN2A/B</td>
<td>2</td>
</tr>
<tr class="even">
<td>Low-grade astrocytic tumor</td>
<td>LGG, other MAPK</td>
<td>23</td>
</tr>
<tr class="odd">
<td>Low-grade astrocytic tumor</td>
<td>LGG, RTK</td>
<td>22</td>
</tr>
<tr class="even">
<td>Low-grade astrocytic tumor</td>
<td>LGG, RTK, CDKN2A/B</td>
<td>2</td>
</tr>
<tr class="odd">
<td>Low-grade astrocytic tumor</td>
<td>LGG, wildtype</td>
<td>84</td>
</tr>
<tr class="even">
<td>Low-grade astrocytic tumor</td>
<td>SEGA, wildtype</td>
<td>6</td>
</tr>
<tr class="odd">
<td>Mesenchymal non-meningothelial tumor</td>
<td>EWS</td>
<td>21</td>
</tr>
<tr class="even">
<td>Neuronal and mixed neuronal-glial tumor</td>
<td>CNC</td>
<td>4</td>
</tr>
<tr class="odd">
<td>Neuronal and mixed neuronal-glial tumor</td>
<td>EVN</td>
<td>2</td>
</tr>
<tr class="even">
<td>Neuronal and mixed neuronal-glial tumor</td>
<td>GNT, BRAF V600E</td>
<td>2</td>
</tr>
<tr class="odd">
<td>Neuronal and mixed neuronal-glial tumor</td>
<td>GNT, KIAA1549-BRAF</td>
<td>4</td>
</tr>
<tr class="even">
<td>Neuronal and mixed neuronal-glial tumor</td>
<td>GNT, other MAPK</td>
<td>2</td>
</tr>
<tr class="odd">
<td>Neuronal and mixed neuronal-glial tumor</td>
<td>GNT, other MAPK, FGFR</td>
<td>2</td>
</tr>
<tr class="even">
<td>Neuronal and mixed neuronal-glial tumor</td>
<td>GNT, RTK</td>
<td>4</td>
</tr>
<tr class="odd">
<td>Tumors of sellar region</td>
<td>CRANIO, ADAM</td>
<td>51</td>
</tr>
<tr class="even">
<td></td>
<td>Total</td>
<td>1281</td>
</tr>
</tbody>
</table>
<p>Table 1: Molecular subtypes determined across OpenPBTA samples. {#tbl:subtypes}</p>
<h3 id="somatic-mutational-landscape-of-pediatric-brain-tumors">Somatic Mutational Landscape of Pediatric Brain Tumors</h3>
<p>We performed a comprehensive genomic analysis of somatic SNVs, CNVs, SVs, and fusions across 1,969 tumors (N = 1,019 RNA-Seq, N = 1,719 WGS, N = 64 WXS/Panel) and 38 cell lines (N = 16 RNA-Seq, N = 22 WGS) from 943 patients.
Following SNV consensus calling (<strong>Figure S1</strong> and <strong>Figure S2, panels A-F</strong>), we observed lower expected tumor mutation burden (TMB) <strong>Figure S2, panel G</strong> in pediatric tumors compared to adult brain tumors from The Cancer Genome Atlas (TCGA), with hypermutant (&gt; 10 Mut/Mb) and ultra-hypermutant (&gt; 100 Mut/Mb) tumors <span class="citation" data-cites="OgCAsGmn">[<a href="#ref-OgCAsGmn" role="doc-biblioref">12</a>]</span> only found within HGATs.</p>
<h4 id="low-grade-astrocytic-tumors">Low-grade astrocytic tumors</h4>
<p><strong>Figure <a href="#fig:Fig2">2</a>A</strong> depicts an oncoprint of driver genes for 227 primary low-grade astrocytic tumors. As expected, the majority (62%, 140/227) of these tumors harbor a somatic alteration in <em>BRAF</em>, with canonical <em>BRAF::KIAA1549</em> fusions as the major oncogenic driver.
We observed additional mutations in <em>FGFR1</em> (2%), <em>PIK3CA</em> (2%), <em>KRAS</em> (2%), <em>TP53</em> (1%), and <em>ATRX</em> (1%) and fusions in <em>NTRK2</em> (2%), <em>RAF1</em> (2%), <em>MYB</em> (1%), <em>QKI</em> (1%), <em>ROS1</em> (1%), and <em>FGFR2</em> (1%), concordant with previous studies reporting the near universal upregulation of the RAS/MAPK pathway in these tumors resulting from activating mutations and/or oncogenic fusions <span class="citation" data-cites="IamVuiNU ULPByXgG">[<a href="#ref-ULPByXgG" role="doc-biblioref">11</a>,<a href="#ref-IamVuiNU" role="doc-biblioref">13</a>]</span>.
Indeed, we observed significant upregulation (ANOVA p &lt; 0.01) of the KRAS signaling pathway in LGATs (<strong>Figure <a href="#fig:Fig4">4</a>B</strong>).</p>
<h4 id="embryonal-tumors">Embryonal tumors</h4>
<p><strong>Figure <a href="#fig:Fig2">2</a>B</strong> shows the mutational landscape for 128 primary embryonal tumors.
The majority (N = 95) are medulloblastomas and span the spectrum of molecular subtypes: WNT, SHH, Group3, and Group 4 (see <strong>Molecular Subtyping of CNS Tumors</strong>), with their canonical mutations.
We detected canonical <em>SMARCB1/SMARCA4</em> deletions or inactivating mutations in atypical teratoid rhabdoid tumors (ATRTs) and C19MC amplification in the embryonal tumors with multilayer rosettes (ETMRs) <span class="citation" data-cites="1Geb70JK9 1EzqPr2Ef j11VxJIm UuzX8B9C">[<a href="#ref-1Geb70JK9" role="doc-biblioref">14</a>,<a href="#ref-1EzqPr2Ef" role="doc-biblioref">15</a>,<a href="#ref-j11VxJIm" role="doc-biblioref">16</a>,<a href="#ref-UuzX8B9C" role="doc-biblioref">17</a>]</span>.</p>
<h4 id="diffuse-astrocytic-and-oligodendroglial-tumors">Diffuse astrocytic and oligodendroglial tumors</h4>
<p>In <strong>Figure <a href="#fig:Fig2">2</a>C</strong>, we show genomic alterations in diffuse midline gliomas (DMGs, N = 34) and non-midline high-grade gliomas (N = 26) biopsied at diagnosis.
The single oligodendroglioma sample in the OpenPBTA does not contain mutations in the genes shown and is therefore not present in this oncoprint.
Across HGATs, we found <em>TP53</em> (57%, 35/61) and <em>H3F3A</em> (52%, 32/61) to be the most mutated and co-occurring genes (<strong>Figure <a href="#fig:Fig2">2</a>A</strong>), followed by frequent mutations in <em>ATRX</em> (30% 18/61).
We found recurrent amplifications and fusions in <em>EGFR</em>, <em>MET</em>, <em>PDGFRA</em>, and <em>KIT</em>, highlighting that these tumors utilize multiple oncogenic mechanisms to activate tyrosine kinases, as has been previously reported <span class="citation" data-cites="QNvTzXiX BLNV1DbL vd5Uh9gY KU8wVb8T">[<a href="#ref-BLNV1DbL" role="doc-biblioref">5</a>,<a href="#ref-QNvTzXiX" role="doc-biblioref">18</a>,<a href="#ref-vd5Uh9gY" role="doc-biblioref">19</a>,<a href="#ref-KU8wVb8T" role="doc-biblioref">20</a>]</span>.
Gene set enrichment analysis showed upregulation (ANOVA p &lt; 0.01) of DNA repair, G2M checkpoint, and MYC pathways as well as downregulation of the TP53 pathway (<strong>Figure <a href="#fig:Fig4">4</a>B</strong>).
The two tumors with ultra-high tumor mutation burden (TMB) (&gt; 100 Mutations/Mb) were from patients with known mismatch repair deficiency syndrome <span class="citation" data-cites="DwCzMoBJ">[<a href="#ref-DwCzMoBJ" role="doc-biblioref">3</a>]</span>.</p>
<h4 id="other-cns-tumors">Other CNS tumors</h4>
<p><strong>Figure <a href="#fig:Fig2">2</a>D</strong> depicts an oncoprint for the remaining primary CNS tumors (N = 195).
We observed 25% (15/60) of ependymoma tumors to be <em>C11orf95::RELA</em> (now, <em>ZFTA::RELA</em>) fusion positive ependymomas and 70% (21/30) of craniopharyngiomas to be driven by mutations in <em>CTNNB1</em>.
Multiple histologies contained somatic mutations or fusions in <em>NF2</em>: 41% (7/17) of meningiomas, 5% (3/60) of ependymomas, and 27% (3/11) schwannomas.
Rare fusions in <em>ERBB4</em>, <em>YAP1</em>, <em>KRAS</em>, and <em>MAML2</em> were observed in 10% (6/60) of ependymoma tumors.
DNETs harbored alterations in MAPK/PI3K pathway genes as previously reported <span class="citation" data-cites="eDtbAO3v">[<a href="#ref-eDtbAO3v" role="doc-biblioref">21</a>]</span>, including <em>FGFR1</em> (21%, 4/19), <em>PDGFRA</em> (10%, 2/19), and <em>BRAF</em> (5%, 1/19).
Frequent mutations in rarer brain tumor histologies (N &lt; 5) are depicted in <strong>Figure ??A</strong>.</p>
<div id="fig:Fig2" class="fignos">
<figure>
<img src="https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/figures/pngs/primary_only_oncoprint_landscape.png" alt="" /><figcaption><span>Figure 2:</span> <strong>Figure 2. Mutational landscape of PBTA tumors.</strong> Shown are frequencies of canonical somatic gene mutations, CNVs, fusions, and TMB (top bar plot) for the top 20 genes mutated across primary tumors within the OpenPBTA dataset. A, Low-grade astrocytic tumors (N = 227): low-grade glioma astrocytoma (N = 187), ganglioglioma (N = 35), subependymal giant cell astrocytoma (N = 2), diffuse fibrillary astrocytoma (N = 1), pilocytic astrocytoma (N = 1), and pleomorphic xanthoastrocytoma (N = 1); B, Embryonal tumors (N = 128): medulloblastomas (N = 95), atypical teratoid rhabdoid tumors (N = 24), embryonal tumors with multilayer rosettes (N = 2), other CNS embryonal tumors (N = 5), ganglioneuroblastoma (N = 1), and CNS neuroblastoma (N = 1); C, Diffuse astrocytic and oligodendroglial tumors (N = 61): diffuse midline gliomas (N = 34) and non-midline high-grade gliomas (N = 26), oligodendroglioma (N = 1); D, Other CNS tumors (N = 195): ependymomas (N = 60), dysembryoplastic neuroepithelial tumors (N = 19), meningiomas (N = 17), schwannoma (N = 11), neurofibroma plexiform (N = 7). Other CNS tumors with N &lt; 5 are displayed in <strong>Figure ??A</strong>. Patient sex (<code>germline_sex_estimate</code>) and tumor histology (<code>cancer_group</code>) are displayed as annotations at the bottom of each plot. Only samples with mutations in the listed genes are shown. Multiple CNVs are denoted as a complex event.</figcaption>
</figure>
</div>
<h3 id="mutational-co-occurrence-and-signatures-highlight-key-oncogenic-drivers">Mutational co-occurrence and signatures highlight key oncogenic drivers</h3>
<p>The top 50 mutated genes in primary tumors are shown <strong>Figure <a href="#fig:Fig3">3</a></strong> by tumor type (<strong>A</strong>, bar plots), with co-occurrence scores illustrated in the heatmap (<strong>B</strong>).
We observed <em>TP53</em> to be the most frequently mutated gene across OpenPBTA tumors (8.4%, 56/666), significantly co-occurring with <em>H3F3A</em> (OR = 32, 95% CI: 15.3 - 66.7, q = 8.46e-17), <em>ATRX</em> (OR = 20, 95% CI: 8.4 - 47.7, q = 4.43e-8), <em>NF1</em> (OR = 8.62, 95% CI: 3.7 - 20.2, q = 5.45e-5), and <em>EGFR</em> (OR = 18.2, 95% CI: 5 - 66.5, q = 1.6e-4).
Other canonical cancer driver genes were frequently mutated: <em>BRAF</em>, <em>H3F3A</em>, <em>CTNNB1</em>, <em>NF1</em>, <em>ATRX</em>, <em>FGFR1</em>, and <em>PIK3CA</em>.
Although LGG and embryonal tumors make up the majority of tumor types within the OpenPBTA, most of the significant gene interactions stem from HGATs (N = 847/872).
At the broad histology level, <em>CTNNB1</em> significantly co-occurs with <em>TP53</em> (OR = 42.9, 95% CI: 7 - 261.4, q = 1.63e-3) and <em>DDX3X</em> (OR = 21.1, 95% CI: 4.6 - 96.3, q = 4.46e-3) in embryonal tumors, <em>FGFR1</em> and <em>PIK3CA</em> significantly co-occur in LGGs (OR = 76.1, 95% CI: 9.85 - 588.1, q = 3.26e-3), consistent with previous findings [<span class="citation" data-cites="ZINVcBaH"><a href="#ref-ZINVcBaH" role="doc-biblioref">22</a></span>; 10.1186/s40478-020-01027-z].
<em>TP53</em> and <em>PPM1D</em> mutations have been shown to be mutually exclusive in HGATs, and our data recapitulates that trend (52/54 or 96.3% of tumors have a mutation in either gene, OR = 0.188, 95% CI: 0.04 - 0.94, p = 4.13e-2, q = 5.87e-2) <span class="citation" data-cites="zrYR4Swt">[<a href="#ref-zrYR4Swt" role="doc-biblioref">23</a>]</span>.
We next assessed the contributions of eight previously identified adult CNS-specific mutational signatures <span class="citation" data-cites="Cc8jt6lX">[<a href="#ref-Cc8jt6lX" role="doc-biblioref">24</a>]</span> (RefSig) across cancer groups <strong>Figure <a href="#fig:Fig3">3</a>C</strong> and samples <strong>Figure ??A</strong>.
Stage 0 and/or 1 tumors characterized by low TMBs <strong>Figure ??G</strong> such as LGGs, gangliogliomas, craniopharyngiomas, DNETs, and schwannomas are expectedly dominated by Signature 1, which results from the normal process of spontaneous deamination of 5-methylcytosine.
Signature N6 is CNS-specific signature which we observe nearly universally across samples.
Drivers of Signature 18, <em>TP53</em>, <em>APC</em>, <em>NOTCH1</em> (https://signal.mutationalsignatures.com/explore/referenceCancerSignature/31/drivers), are also canonical drivers of medulloblastoma tumors, and indeed, we observe Signature 18 as the most common signature in medulloblastoma tumors.
Signatures 3, 8, 18, and MMR2 are prevalent in HGGs, including DMGs.
Finally, we observe that the weight of Signature 1 is higher at diagnosis (pre-treatment) and is almost always lower in tumors at later phases of therapy (progression, recurrence, post-mortem, secondary malignancy) <strong>Figure ??B</strong>.
This trend may be the result of therapy-induced mutations which produce additional signatures (e.g., temozolomide treatment drives Signature 11), subclonal expansion, and/or acquisition of additional driver mutations during tumor progression, leading to higher overall TMBs and additional signatures.</p>
<!-- TODO: possibly explore gene mutations driving signature 8, specifically for meningiomas, and describe -->
<h3 id="genomic-instability-of-pediatric-brain-tumors">Genomic instability of pediatric brain tumors</h3>
<p>We developed a method for generating consensus copy number altered regions from MantaSV, CNVkit, and ControlFREEC (<strong>STAR Methods</strong>).
This resulted in high-confidence gains, losses, amplifications, and deep deletion calls used as input for breakpoint density (<strong>Table S2</strong>) and chromothripsis analysis.
We observed that HGATs, followed by medulloblastomas, have the most unstable genomes (<strong>Figure S3A</strong>).
By contrast, craniopharyngiomas generally lack somatic copy number variation.
These patterns of copy number variation largely align with estimates of tumor mutational burden (<strong>Figure S2G</strong>).
The number of SV and CNV breakpoints were significantly correlated across tumors (p = 1.08e-37) (<strong>Figure <a href="#fig:Fig3">3</a>D</strong>) and as expected, the number of chromothripsis regions called increases as breakpoint density increases (<strong>Figure S3B-C</strong>).
Chromothripsis events were observed in 41% (N = 19/46) of non-midline high-grade gliomas and 28.2% (N = 11/39) of DMGs (<strong>Figure <a href="#fig:Fig3">3</a>E</strong>).
We also found evidence of chromothripsis in over 15% of embryonal tumors, ependymomas, meningiomas, germinomas, glial-neuronal tumors, chordomas, metastatic secondary tumors, and sarcomas, highlighting the genomic instability and complexity of pediatric brain tumors.</p>
<div id="fig:Fig3" class="fignos">
<figure>
<img src="images/main/Figure3.png" alt="" /><figcaption><span>Figure 3:</span> <strong>Figure 3. Mutational co-occurrence and signatures highlight key oncogenic drivers.</strong> A, Bar plot of occurrence and co-occurrence of nonsynonymous mutations for the 50 most commonly mutated genes across all tumor types (annotated from <code>cancer_group</code> if N &gt;= 10 or <code>Other</code> if N &lt; 10); B, Co-occurrence and mutual exclusivity of nonsynonymous mutations between genes; The co-occurrence score is defined as <span class="math inline">\(I(-\log_{10}(P))\)</span> where <span class="math inline">\(P\)</span> is defined by Fisher’s exact test and <span class="math inline">\(I\)</span> is 1 when mutations co-occur more often than expected and -1 when exclusivity is more common; C, Sina plots of RefSig signature weights for signatures 1, 3, 8, 11, 18, 19, N6, MMR2, and Other across cancer groups. Box plot lines represent the first quartile, median, and third quartile. D, The number of SV breaks significantly correlate with CNV breaks (Adjusted R = 0.436, p = 1.08e-37). E, Chromothripsis frequency across pediatric brain tumors shown by <code>cancer_group</code> with N &gt;= 3.</figcaption>
</figure>
</div>
<h3 id="transcriptomic-landscape-of-pediatric-brain-tumors">Transcriptomic Landscape of Pediatric Brain Tumors</h3>
<h4 id="histologic-and-oncogenic-pathway-clustering">Histologic and oncogenic pathway clustering</h4>
<p>UMAP visualization of gene expression variation across brain tumors (<strong>Figure <a href="#fig:Fig4">4</a>A</strong>) shows the expected clustering of brain tumors by histology.
We observed medulloblastomas cluster by molecular subtype with WNT and SHH in distinct clusters and Groups 3 and 4 showing some overlap (<strong>Figure ??A</strong>), as expected.
Of note, two samples annotated as the SHH subtype do not cluster with the MB samples and one clusters with Group 3 and 4 samples, suggesting potential subtype misclassification or different underlying biology of these tumors.
Additionally, except for three outliers, <em>C11orf95::RELA</em> (<em>ZFTA::RELA</em>) fusion positive ependymomas fall within distinct clusters (<strong>Figure ??B</strong>).
<em>BRAF</em>-driven low-grade gliomas (<strong>Figure ??C</strong>) were present in three separate clusters, suggesting that there might be distinct underlying biology within these tumors.
Histone H3 G35-mutant HGATs generally clustered together, away from K28-mutant tumors (<strong>Figure ??D</strong>).
Interestingly, although H3 K28-mutant tumors have different biological drivers than H3 wildtype tumors, they did not form distinct clusters, suggesting they may either be driven by common transcriptional programs or our sample size is too small to detect transcriptional differences.</p>
<p>We performed gene set variant analysis (GSVA) for Hallmark cancer gene sets, demonstrating activation of underlying oncogenic pathways (<strong>Figure <a href="#fig:Fig4">4</a>B</strong>).</p>
<!-- TODO: Add immune deconvolution figure 4C information. -->
<div id="fig:Fig4" class="fignos">
<figure>
<img src="images/main/Figure4.png" alt="" /><figcaption><span>Figure 4:</span> <strong>Figure 4. Transcriptomic overview</strong> A, First two dimensions from UMAP of sample transcriptome data.
Points are colored by <code>cancer_group</code> of the samples they represent. B, Heatmap of GSVA scores for Hallmark gene sets with significant differences, with samples ordered by <code>cancer_group</code>. C, <!-- TODO: Add description about immune deconvolution image we use as main. --></figcaption>
</figure>
</div>
<h4 id="prediction-of-tp53-oncogenicity-and-telomerase-activity">Prediction of <em>TP53</em> oncogenicity and telomerase activity</h4>
<p>To understand the <em>TP53</em> phenotype in each tumor, we ran a classifier previously trained on TCGA <span class="citation" data-cites="X6oQhIf8">[<a href="#ref-X6oQhIf8" role="doc-biblioref">25</a>]</span> to infer <em>TP53</em> inactivation status.
Using high-confidence SNVs, CNVs, SVs, and fusions in <em>TP53</em> as true positive alterations, we achieved a high accuracy (AUROC = 0.85) for rRNA-depleted, stranded samples compared to randomly shuffled <em>TP53</em> scores (<strong>Figure <a href="#fig:Fig5">5</a>A</strong>).
The classifier did not perform well on the poly-A samples (<strong>Figure ??E-F</strong>), potentially due to the low number of <em>TP53</em> altered (N = 29) and/or total poly-A samples in our dataset (N = 58) rather than library type, as a previous study demonstrated high accuracy of this classifier on another poly-A dataset <span class="citation" data-cites="SDwYl8uA">[<a href="#ref-SDwYl8uA" role="doc-biblioref">26</a>]</span>.
We annotated <em>TP53</em> alterations as “activated” if samples harbored one of p.R273C or p.R248W mutations <span class="citation" data-cites="14asizOhY">[<a href="#ref-14asizOhY" role="doc-biblioref">27</a>]</span>, “loss” if the patient had a Li Fraumeni Syndrome (LFS) predisposition diagnosis or if any other SNV, CNV, SV, or fusion fell within a functional domain.
If the <em>TP53</em> mutation did not reside within the DNA-binding domain or if an alteration was not detected in <em>TP53</em>, we annotated the tumor as “other”.
Interestingly, we observed that samples annotated as either “activated” or “loss” had significantly higher <em>TP53</em> scores than those annotated as “other” (<strong>Figure <a href="#fig:Fig5">5</a>B</strong>, p<sub>adj</sub> loss vs. other &lt; 2e-16, p<sub>adj</sub> activated vs. other = 4.0e-5), suggesting that the classifier detects an oncogenic, or altered, <em>TP53</em> phenotype (scores &gt; 0.5) rather than solely <em>TP53</em> inactivation, as interpreted previously <span class="citation" data-cites="X6oQhIf8">[<a href="#ref-X6oQhIf8" role="doc-biblioref">25</a>]</span>.
Moreover, tumors with “activating” <em>TP53</em> mutations had evidence of higher <em>TP53</em> expression than those with <em>TP53</em> “loss” mutations (Wilcoxon p = 3.5e-3, <strong>Figure <a href="#fig:Fig5">5</a>C</strong>.
To further validate the classifier’s accuracy, we assessed <em>TP53</em> scores for patients with LFS, hypothesizing all of these tumors would have high scores.
Indeed, we observed higher scores in LFS tumors (N = 8) for which we detected high-confidence <em>TP53</em> somatic alterations.
Although we were unable to detect canonical somatic <em>TP53</em> mutations in two LFS patient tumors with low <em>TP53</em> scores, we confirmed the LFS diagnosis from pathology reports and found each to have a germline variant in <em>TP53</em>.
The tumor purity of these samples was low (16% and 37%), suggesting the classifier requires a certain level of tumor purity to achieve good performance, as we expect <em>TP53</em> to be intact in normal cells.
Tumors with the highest median <em>TP53</em> scores were those known to harbor somatic <em>TP53</em> alterations: choroid plexus tumors, embryonal tumors, HGATs, and pineal tumors (<strong>Figure <a href="#fig:Fig5">5</a>E</strong>), while melanocytic tumors, meningiomas, and tumors of cranial and paraspinal nerves had the lowest scores.</p>
<p>We next used gene expression data to predict telomerase activity using EXpression-based Telomerase ENzymatic activity Detection (EXTEND) <span class="citation" data-cites="RnBZNbdx">[<a href="#ref-RnBZNbdx" role="doc-biblioref">28</a>]</span> as a surrogate measure of malignant potential [<span class="citation" data-cites="RnBZNbdx"><a href="#ref-RnBZNbdx" role="doc-biblioref">28</a></span>; 10.1093/carcin/bgp268].
EXTEND scores significantly correlated with <em>TERT</em> (R = 0.55) and <em>TERC</em> (R = 0.58) expression (<strong>Figure ??G-H</strong>).<br />
We found aggressive tumors such as CNS lymphoma, ETMR, ATRT, DMG, and HGG had high EXTEND scores (<strong>Figure <a href="#fig:Fig5">5</a>G</strong>), while benign lesions such as teratomas, dysplasias, and hemangioblastomas had the lowest scores (<strong>Table S3</strong>).</p>
<!-- TODO: Fig 5 will likely be changing and need an update. -->
<div id="fig:Fig5" class="fignos">
<figure>
<img src="images/main/Figure5.png" alt="" /><figcaption><span>Figure 5:</span> <strong>Figure 5. <em>TP53</em> and telomerase activity</strong> A, Receiver Operating Characteristic for <em>TP53</em> classifier run on FPKM of stranded RNA-Seq samples. B, Violin and box plots of <em>TP53</em> scores plotted by <em>TP53</em> alteration type. C, Violin and box plots of <em>TP53</em> RNA expression plotted by <em>TP53</em> activation status. D, Box plots of <em>TP53</em> scores grouped by <code>broad_histology</code>. E, Box plots of EXTEND scores grouped by <code>broad_histology</code>.</figcaption>
</figure>
</div>
<h2 id="discussion">Discussion</h2>
<p>Stub in discussion section</p>
<h2 id="acknowledgments">Acknowledgments</h2>
<p>We graciously thank the patients and families who have donated their tumors to the Children’s Brain Tumor Network and/or the Pacific Pediatric Neuro-oncology Consortium, without which, this research would not be possible.
This work was funded through the Alex’s Lemonade Stand Foundation (ALSF) Childhood Cancer Data Lab (JNT, CSG, JAS, CLS, CJB), ALSF Young Investigator Award (JLR), ALSF Catalyst Award (JLR, ACR, PBS), Children’s Hospital of Philadelphia Division of Neurosurgery (PBS and ACR), the Australian Government, Department of Education (APH), and NIH Grants 3P30 CA016520-44S5 (ACR), U2C HL138346-03 (ACR, APH), and U24 CA220457-03 (ACR).
This project has been funded in part with Federal funds from the National Cancer Institute, National Institutes of Health, under Contract No. 75N91019D00024, Task Order No. 75N91020F00003 (JLR, ACR, APH).
The content of this publication does not necessarily reflect the views or policies of the Department of Health and Human Services, nor does mention of trade names, commercial products or organizations imply endorsement by the U.S. Government.</p>
<h2 id="author-contributions">Author Contributions</h2>
<table>
<colgroup>
<col style="width: 50%" />
<col style="width: 50%" />
</colgroup>
<thead>
<tr class="header">
<th>Author</th>
<th>Contributions</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Joshua A. Shapiro</td>
<td>Methodology, Software, Validation, Formal analysis, Investigation, Writing - Original draft, Writing - Review and editing, Visualization, Supervision</td>
</tr>
<tr class="even">
<td>Candace L. Savonen</td>
<td>Methodology, Software, Validation, Formal analysis, Investigation, Writing - Original draft, Visualization</td>
</tr>
<tr class="odd">
<td>Chante J. Bethell</td>
<td>Methodology, Validation, Formal analysis, Investigation, Writing - Original draft, Visualization</td>
</tr>
<tr class="even">
<td>Krutika S. Gaonkar</td>
<td>Data curation, Formal Analysis, Investigation, Methodology, Software, Writing – original draft</td>
</tr>
<tr class="odd">
<td>Run Jin</td>
<td>Data curation, Formal Analysis, Visualization, Writing – original draft</td>
</tr>
<tr class="even">
<td>Yuankun Zhu</td>
<td>Data curation, Formal Analysis, Investigation, Methodology, Supervision</td>
</tr>
<tr class="odd">
<td>Miguel A. Brown</td>
<td>Data curation, Methodology</td>
</tr>
<tr class="even">
<td>Nhat Duong</td>
<td>Formal Analysis, Investigation, Methodology</td>
</tr>
<tr class="odd">
<td>Komal S. Rathi</td>
<td>Formal Analysis, Investigation, Methodology, Writing – original draft</td>
</tr>
<tr class="even">
<td>Nighat Noureen</td>
<td>Formal analysis, Visualization, Writing - Original draft</td>
</tr>
<tr class="odd">
<td>Bo Zhang</td>
<td>Data curation, Formal Analysis</td>
</tr>
<tr class="even">
<td>Brian M. Ennis</td>
<td>Data curation, Formal Analysis</td>
</tr>
<tr class="odd">
<td>Stephanie J. Spielman</td>
<td>Validation, Formal analysis, Writing - Review and editing, Visualization, Supervision</td>
</tr>
<tr class="even">
<td>Laura E. Egolf</td>
<td>Formal analysis, Writing - Original draft</td>
</tr>
<tr class="odd">
<td>Bailey Farrow</td>
<td>Data curation, Software</td>
</tr>
<tr class="even">
<td>Nicolas Van Kuren</td>
<td>Data curation, Software</td>
</tr>
<tr class="odd">
<td>Meen Chul Kim</td>
<td>Data curation</td>
</tr>
<tr class="even">
<td>Tejaswi Koganti</td>
<td>Formal Analysis, Investigation</td>
</tr>
<tr class="odd">
<td>Shrivats Kannan</td>
<td>Formal Analysis, Methodology, Writing – original draft</td>
</tr>
<tr class="even">
<td>Pichai Raman</td>
<td>Conceptualization, Formal Analysis, Methodology</td>
</tr>
<tr class="odd">
<td>Jennifer Mason</td>
<td>Supervision</td>
</tr>
<tr class="even">
<td>Daniel P. Miller</td>
<td>Formal Analysis</td>
</tr>
<tr class="odd">
<td>Anna R. Poetsch</td>
<td>Formal Analysis</td>
</tr>
<tr class="even">
<td>Payal Jain</td>
<td>Data curation, Investigation, Validation</td>
</tr>
<tr class="odd">
<td>Adam A. Kraya</td>
<td>Methodology</td>
</tr>
<tr class="even">
<td>Allison P. Heath</td>
<td>Project administration</td>
</tr>
<tr class="odd">
<td>Mateusz P. Koptyra</td>
<td>Formal Analysis, Writing – original draft</td>
</tr>
<tr class="even">
<td>Shannon Robbins</td>
<td>Data curation</td>
</tr>
<tr class="odd">
<td>Yiran Guo</td>
<td>Formal Analysis</td>
</tr>
<tr class="even">
<td>Xiaoyan Huang</td>
<td>Formal Analysis</td>
</tr>
<tr class="odd">
<td>Jessica Wong</td>
<td>Writing – original draft</td>
</tr>
<tr class="even">
<td>Mariarita Santi</td>
<td>Investigation, Validation</td>
</tr>
<tr class="odd">
<td>Angela Viaene</td>
<td>Investigation, Validation</td>
</tr>
<tr class="even">
<td>Laura Scolaro</td>
<td>Data Curation</td>
</tr>
<tr class="odd">
<td>Angela Waanders</td>
<td>Supervison</td>
</tr>
<tr class="even">
<td>Derek Hanson</td>
<td>Validation</td>
</tr>
<tr class="odd">
<td>Steven M. Foltz</td>
<td>Validation</td>
</tr>
<tr class="even">
<td>Hongbo M. Xie</td>
<td>Methodology, Supervision</td>
</tr>
<tr class="odd">
<td>Siyuan Zheng</td>
<td>Formal analysis, Visualization, Writing - Original draft, Supervision</td>
</tr>
<tr class="even">
<td>Cassie N. Kline</td>
<td>Supervision</td>
</tr>
<tr class="odd">
<td>Jena V. Lilly</td>
<td>Conceptualization, Funding acquisition, Project administration</td>
</tr>
<tr class="even">
<td>Philip B. Storm</td>
<td>Conceptualization, Funding acquisition, Resources</td>
</tr>
<tr class="odd">
<td>Adam C. Resnick</td>
<td>Conceptualization, Funding acquisition, Resources, Supervision</td>
</tr>
<tr class="even">
<td>Casey S. Greene</td>
<td>Conceptualization, Funding acquisition, Methodology, Project administration, Software, Supervision, Writing – review &amp; editing</td>
</tr>
<tr class="odd">
<td>Jo Lynne Rokita*</td>
<td>Conceptualization, Data curation, Formal Analysis, Funding acquisition, Investigation, Methodology, Software, Supervision, Writing – original draft</td>
</tr>
<tr class="even">
<td>Jaclyn N. Taroni*</td>
<td>Methodology, Software, Validation, Formal analysis, Investigation, Data curation, Writing - Review and editing, Visualization, Supervision, Project administration</td>
</tr>
<tr class="odd">
<td>Children’s Brain Tumor Network</td>
<td>Conceptualization</td>
</tr>
<tr class="even">
<td>Pacific Pediatric Neurooncology Consortium</td>
<td>Conceptualization</td>
</tr>
</tbody>
</table>
<h2 id="declarations-of-interest">Declarations of Interest</h2>
<p>CSG’s spouse was an employee of Alex’s Lemonade Stand Foundation, which was a sponsor of this research.
JAS, CLS, CJB, SJS, and JNT are or were employees of Alex’s Lemonade Stand Foundation, a sponsor of this research.</p>
<h2 id="figure-titles-and-legends">Figure Titles and Legends</h2>
<p><strong>Figure 1. Overview of the OpenPBTA Project.</strong></p>
<h2 id="star-methods">STAR METHODS</h2>
<h3 id="resource-availability">RESOURCE AVAILABILITY</h3>
<h4 id="lead-contact">Lead contact</h4>
<p>Requests for access to OpenPBTA raw data and/or specimens may be directed to, and will be fulfilled by Jo Lynne Rokita (rokita@chop.edu).</p>
<h4 id="materials-availability">Materials availability</h4>
<p>This study did not create new, unique reagents.</p>
<h4 id="data-and-code-availability">Data and code availability</h4>
<p>Raw and harmonized WGS, WXS, and RNA-Seq data derived from human samples are available within the KidsFirst Portal <span class="citation" data-cites="THjIzNXb">[<a href="#ref-THjIzNXb" role="doc-biblioref">29</a>]</span> upon access request to the CBTN (https://cbtn.org/) as of the date of the publication.
In addition, merged summary files are openly accessible at https://cavatica.sbgenomics.com/u/cavatica/openpbta or via download script from https://github.com/AlexsLemonade/OpenPBTA-analysis/.
Summary data are visible within Pedcbioportal at https://pedcbioportal.kidsfirstdrc.org/study/summary?id=openpbta.
Links or DOIs are listed in the <strong>Key Resources Table</strong>.
<!-- TODO: create data dois --></p>
<p>All original code has been deposited in the following repositories and is publicly available as of the date of the publication:
- Primary data analyses: https://github.com/d3b-center/OpenPBTA-workflows/
- Downstream data analyses: https://github.com/AlexsLemonade/OpenPBTA-analysis/
- Manuscript code: https://github.com/AlexsLemonade/OpenPBTA-manuscript
Links or DOIs are listed in the <strong>Key Resources Table</strong>.
<!-- TODO: create code dois --></p>
<p>Any additional information required to reanalyze the data reported in this paper is available
from the lead contact upon request.</p>
<h3 id="method-details">METHOD DETAILS</h3>
<h4 id="biospecimen-collection">Biospecimen Collection</h4>
<p>The Pediatric Brain Tumor Atlas specimens are comprised of samples from Children’s Brain Tumor Network (CBTN) and the Pediatric Pacific Neuro-Oncology Consortium (PNOC).
The CBTN <span class="citation" data-cites="VT338cRv">[<a href="#ref-VT338cRv" role="doc-biblioref">4</a>]</span> is a collaborative, multi-institutional (18 institutions worldwide) research program dedicated to the study of childhood brain tumors.
The Pacific Pediatric Neuro-Oncology Consortium (PNOC) <span class="citation" data-cites="cLFzPEr9">[<a href="#ref-cLFzPEr9" role="doc-biblioref">30</a>]</span> is an international consortium dedicated to bringing new therapies to children and young adults with brain tumors.
PNOC collected blood and tumor biospecimens from newly-diagnosed diffuse intrinsic pontine glioma (DIPG) patients as part of the clinical trial <a href="https://clinicaltrials.gov/ct2/show/NCT02274987">PNOC003/NCT02274987</a> <span class="citation" data-cites="BLNV1DbL">[<a href="#ref-BLNV1DbL" role="doc-biblioref">5</a>]</span>.</p>
<p>All CBTN data can be download from the Gabriella Miller Kids First Data Resource Center, <span class="citation" data-cites="JkpD6jMa">[<a href="#ref-JkpD6jMa" role="doc-biblioref">31</a>]</span>.
The de-identified patient’s blood and tumor tissue were prospectively collected by the consortium from patients enrolled within the CBTN.</p>
<p>The cell lines were generated by the CBTN from either fresh tumor tissue obtained directly from surgery performed at Children’s Hospital of Philadelphia (CHOP) or from prospectively collected tumor specimens stored in Recover Cell Culture Freezing medium (cat# 12648010, Gibco).
The tissue was dissociated using enzymatic method with papain as described <span class="citation" data-cites="v08ocfr0">[<a href="#ref-v08ocfr0" role="doc-biblioref">32</a>]</span>.
Briefly, tissue was washed with HBSS (cat# 14175095, Gibco), minced and incubated with activated papain solution (cat# LS003124, SciQuest) for up to 45 minutes.
The papain was inactivated using ovomucoid solution (cat# 542000, SciQuest), tissue was briefly treated with DNase (cat# 10104159001, Roche) and passed through the 100μm cell strainer (cat# 542000, Greiner Bio-One).
Two cell culture conditions were initiated based on the number of cells available.
For cultures utilizing the fetal bovine serum (FBS), a minimum density of 3×10^5 cells/ml were plated in DMEM/F-12 medium (cat# D8062, Sigma) supplemented with 20% FBS (cat# SH30910.03, Hyclone), 1% GlutaMAX (cat# 35050061, Gibco), Penicillin/Streptomycin-Amphotericin B Mixture (cat# 17-745E, Lonza) and 0.2% Normocin (cat# ant-nr-2, Invivogen).
For the serum-free media conditions cells were plated at minimum density of 1×10^6 cells/ml in DMEM/F12 medium supplemented with 1% GlutaMAX, 1x B-27 supplement minus vitamin A (cat# 12587-010, Gibco), 1x N-2 supplement (cat# 17502001, Gibco), 20 ng/ml epidermal growth factor (cat# PHG0311L, Gibco), 20 ng/ml basic fibroblast growth factor (cat# 100-18B, PeproTech), 2.5μg/ml heparin (cat# H3149, Sigma), Penicillin/Streptomycin-Amphotericin B Mixture and 0.2% Normocin.</p>
<h4 id="nucleic-acids-extraction-and-library-preparation">Nucleic acids extraction and library preparation</h4>
<h5 id="pnoc-samples">PNOC samples</h5>
<p>The Translational Genomic Research Institute (TGEN; Phoenix, AZ) performed DNA and RNA extractions on tumor biopsies using a DNA/RNA AllPrep Kit (Qiagen, #80204).
All RNA used for library prep had a minimum RIN of 7 but no QC thresholds were implemented for the DNA.
For library preparation, 500ng of nucleic acids were used as input for RNA-Seq, WXS, and targeted DNA panel (panel).
The RNA prep was performed using the TruSeq RNA Sample Prep Kit (Illumina, #FC-122-1001) and the exome prep was performed using KAPA Library Preparation Kit (Roche, #KK8201) using Agilent’s SureSelect Human All Exon V5 backbone with custom probes.
The targeted DNA panel developed by Ashion (formerly known as the GEM Cancer panel) consisted of exonic probes against 541 cancer genes.
Both panel and WXS assays contained 44,000 probes across evenly spaced genomic loci used for genome-wide copy number analysis.
For the panel, additional probes tiled across intronic regions of 22 known tumor suppressor genes and 22 genes involved in common cancer translocations for structural analysis.
All extractions and library preparations were performed according to manufacturer’s instructions.</p>
<h5 id="cbtn-samples">CBTN samples</h5>
<p>Blood, tissue, and cell line DNA/RNA extractions were performed at the Biorepository Core at CHOP.
Briefly, 10-20 mg frozen tissue, 0.4-1ml of blood or 2×10^6 cells pellet was used for extractions.
Tissues were lysed using a Qiagen TissueLyser II (Qiagen) with 2×30 sec at 18Hz settings using 5 mm steel beads (cat# 69989, Qiagen).
Both tissue and cell pellets processes included a CHCl3 extraction and were run on the QIACube automated platform (Qiagen) using the AllPrep DNA/RNA/miRNA Universal kit (cat# 80224, Qiagen).
Blood was thawed and treated with RNase A (cat#, 19101, Qiagen); 0.4-1ml was processed using the Qiagen QIAsymphony automated platform (Qiagen) using the QIAsymphony DSP DNA Midi Kit (cat# 937255, Qiagen).
DNA and RNA quantity and quality was assessed by PerkinElmer DropletQuant UV-VIS spectrophotometer (PerkinElmer) and an Agilent 4200 TapeStation (Agilent, USA) for RIN and DIN (RNA Integrity Number and DNA Integrity Number, respectively).
Library preparation and sequencing was performed by the NantHealth sequencing center.
Briefly, DNA sequencing libraries were prepared for tumor and matched-normal DNA using the KAPA HyperPrep kit (cat# 08098107702, Roche); tumor RNA-Seq libraries were prepared using KAPA Stranded RNA-Seq with RiboErase kit (cat# 07962304001, Roche).
Whole genome sequencing (WGS) was performed at an average depth of coverage of 60X for tumor samples and 30X for germline.
The panel tumor sample was sequenced to 470X and the normal panel sample was sequenced to 308X.
RNA samples were sequenced to an average of 200M reads.
All samples were sequenced on the Illumina HiSeq platform (X/400) (Illumina) with 2 × 150bp read length.</p>
<h4 id="data-generation">Data generation</h4>
<p>NantHealth Sequencing Center (Culver City, CA) performed whole genome sequencing (WGS) on all paired tumor (~60X) and constitutive (~30X) DNA samples.
WGS libraries were 2x150 bp and sequenced on an Illumina X/400.
NantHealth Sequencing Center performed ribosomal-depleted whole transcriptome stranded RNA-Seq to an average depth of 100M reads for CBTN tumor samples.
The Translational Genomic Research Institute (TGEN; Phoenix, AZ) performed paired tumor (~200X) and constitutive whole exome sequencing (WXS) or targeted DNA panel (panel) and poly-A selected RNA-Seq (~200M reads) for PNOC tumor samples.
PNOC WXS and RNA-Seq libraries 2x100 bp and sequenced on an Illumina HiSeq 2500.</p>
<h4 id="dna-wgs-alignment">DNA WGS Alignment</h4>
<p>We used BWA-MEM <span class="citation" data-cites="gmH6YDca">[<a href="#ref-gmH6YDca" role="doc-biblioref">33</a>]</span> v0.7.17 for alignment of paired-end DNA-seq reads.
We used version 38, patch release 12 of the <em>Homo sapiens</em> genome as our alignment reference, which we obtained as a FASTA file from UCSC <span class="citation" data-cites="19mY46wTE">[<a href="#ref-19mY46wTE" role="doc-biblioref">34</a>]</span>.
Alignments were further processed using following the Broad Institute’s Best Practices <span class="citation" data-cites="EHMzG34o">[<a href="#ref-EHMzG34o" role="doc-biblioref">35</a>]</span> for processing Binary Alignment/Map files (BAMs) in preparation for variant discovery.
Duplicates were marked using SAMBLASTER <span class="citation" data-cites="IzUbOv92">[<a href="#ref-IzUbOv92" role="doc-biblioref">36</a>]</span> v0.1.24, BAMs merged and sorted using Sambamba <span class="citation" data-cites="PcJwlZj3">[<a href="#ref-PcJwlZj3" role="doc-biblioref">37</a>]</span> v0.6.3.
Resultant BAMs were processing using Broad’s Genome Analysis Tool Kit [GATK] (https://software.broadinstitute.org/gatk/) v4.0.3.0, BaseRecalibrator submodule.
Lastly, for normal/germline input, we run the GATK HaplotypeCaller <span class="citation" data-cites="15Mgg4tfC">[<a href="#ref-15Mgg4tfC" role="doc-biblioref">38</a>]</span> submodule on the recalibrated BAM, generating a genomic variant call format (GVCF) file.
This file is used as the basis for germline calling, described in the “SNV calling for B-allele Frequency (BAF) generation” section.
References can be obtained from the <a href="https://s3.console.aws.amazon.com/s3/buckets/broad-references/hg38/v0/">Broad Genome References on AWS</a> bucket, with a general description of references here <span class="citation" data-cites="jkkXdcPS">[<a href="#ref-jkkXdcPS" role="doc-biblioref">39</a>]</span>.</p>
<h4 id="quality-control-of-sequencing-data">Quality Control of Sequencing Data</h4>
<p>NGSCheckmate <span class="citation" data-cites="A4FXW005">[<a href="#ref-A4FXW005" role="doc-biblioref">40</a>]</span> was performed on matched tumor/normal CRAM files to confirm sample matches and remove mismatched samples from the dataset.
CRAM inputs were preprocessed using BCFtools to filter and call 20k common single nucleotide polymorphisms (SNPs) using default parameters<span class="citation" data-cites="16tUBMk59">[<a href="#ref-16tUBMk59" role="doc-biblioref">41</a>]</span> and the resulting VCFs were used to run NGSCheckmate using <a href="https://github.com/d3b-center/ngs_checkmate_wf">this workflow</a> in the D3b GitHub repository.
Per author guidelines, &lt;= 0.61 was used as a correlation coefficient cutoff at sequencing depths &gt;10 to predict mismatched samples.
For RNA-Seq, read strandedness was determined by running the <a href="http://rseqc.sourceforge.net/#infer-experiment-py"><code>infer_experiment.py</code> script</a> on the first 200k mapped reads.
If calculated strandedness did not match strandedness information received from the sequencing center, samples were removed from analysis.
We required at least 60% of RNA-Seq reads mapped to the human reference or samples were removed from analysis.</p>
<h4 id="germline-variant-calling">Germline Variant Calling</h4>
<h5 id="snp-calling-for-b-allele-frequency-baf-generation">SNP calling for B-allele Frequency (BAF) generation</h5>
<p>Germline haplotype calls were performed following the <a href="https://software.broadinstitute.org/gatk/best-practices/workflow?id=11145">GATK Joint Genotyping Workflow</a>, except the workflow was run on an individual sample basis.
This workflow was applied to the GVCF output from the alignment workflow on normal/germline samples.
Using only SNPs, we applied the <a href="https://gatkforums.broadinstitute.org/gatk/discussion/2806/howto-apply-hard-filters-to-a-call-set">GATK generic hard filter suggestions</a> to the VCF, with an additional requirement of 10 reads minimum depth per SNP.
This filtered VCF was used as input to Control-FREEC and CNVkit (below) for generation of BAF files.
GATK v4.0.12.0 was used for all steps except <code>VariantFiltration</code>, which used 3.8.0 because as of GATK 4.0.12.0, this tool was beta and known to be unreliable for this purpose.
This single-sample workflow can be found in the <a href="https://github.com/kids-first/kf-jointgenotyping-workflow">Kids First GitHub repository</a>.
References can be obtained from the <a href="https://s3.console.aws.amazon.com/s3/buckets/broad-references/hg38/v0/">Broad Genome References on AWS</a> bucket, with a general description of references here <span class="citation" data-cites="jkkXdcPS">[<a href="#ref-jkkXdcPS" role="doc-biblioref">39</a>]</span>.</p>
<h4 id="somatic-mutation-calling">Somatic Mutation Calling</h4>
<h5 id="snv-and-indel-calling">SNV and indel calling</h5>
<p>For PBTA samples, we used four variant callers to call SNVs and indels from targeted DNA panel, WXS, and WGS data: Strelka2 <span class="citation" data-cites="REfkDUtE">[<a href="#ref-REfkDUtE" role="doc-biblioref">42</a>]</span>, Mutect2 <span class="citation" data-cites="149BEKISi">[<a href="#ref-149BEKISi" role="doc-biblioref">43</a>]</span>, Lancet <span class="citation" data-cites="V6KdWVYi">[<a href="#ref-V6KdWVYi" role="doc-biblioref">44</a>]</span>, and VarDict <span class="citation" data-cites="trQRR8fs">[<a href="#ref-trQRR8fs" role="doc-biblioref">45</a>]</span>.
WXS samples from TCGA were run using Strelka2, Mutect2 and Lancet.
TCGA samples were captured using different WXS target capture kits and all the BED files were downloaded from <a href="https://api.gdc.cancer.gov/files"><code>GDC portal</code></a>.
The input interval BED files for both panel and WXS data for PBTA samples were provided by the manufacturers.
For both PBTA and TCGA, all panel and WXS BED files were padded by 100 bp on each side during Strelka2, Mutect2, and VarDict runs and 400 bp for the Lancet run.<br />
For WGS calling, we utilized the non-padded BROAD Institute interval calling list <a href="https://console.cloud.google.com/storage/browser/_details/genomics-public-data/resources/broad/hg38/v0/wgs_calling_regions.hg38.interval_list"><code>wgs_calling_regions.hg38.interval_list</code></a>, comprised of the full genome minus N bases, unless otherwise noted below.
Strelka2 <span class="citation" data-cites="REfkDUtE">[<a href="#ref-REfkDUtE" role="doc-biblioref">42</a>]</span> v2.9.3 was run using default parameters for canonical chromosomes (chr1-22, X,Y,M), as recommended by the authors.
The final Strelka2 VCF was filtered for PASS variants.
Mutect2 from GATK v4.1.1.0 was run following Broad best practices outlined from their Workflow Description Language (WDL) <span class="citation" data-cites="E5aTvXmQ">[<a href="#ref-E5aTvXmQ" role="doc-biblioref">46</a>]</span>.
The final Mutect2 VCF was filtered for PASS variants.
To manage memory issues, VarDictJava <span class="citation" data-cites="trQRR8fs">[<a href="#ref-trQRR8fs" role="doc-biblioref">45</a>]</span> v1.58 <span class="citation" data-cites="1GMHnwH5p">[<a href="#ref-1GMHnwH5p" role="doc-biblioref">47</a>]</span> was run using 20Kb interval chunks of the input BED, padded by 100 bp on each side, such that if an indel occurred in between intervals, it would be captured.
Parameters and filtering followed <a href="https://github.com/bcbio/bcbio-nextgen">BCBIO standards</a> except that variants with a variant allele frequency (VAF) &gt;= 0.05 (instead of &gt;= 0.10) were retained.
The 0.05 VAF increased the true positive rate for indels and decreased the false positive rate for SNVs when using VarDict in consensus calling.
The final VCF was filtered for PASS variants with TYPE=StronglySomatic.
Lancet v1.0.7 was run using default parameters, except for those noted below.
For input intervals to Lancet WGS, a reference BED was created by using only the UTR, exome, and start/stop codon features of the GENCODE 31 reference, augmented as recommended with PASS variant calls from Strelka2 and Mutect2 <span class="citation" data-cites="bKthX7vJ">[<a href="#ref-bKthX7vJ" role="doc-biblioref">48</a>]</span>.
These intervals were then padded by 300 bp on each side during Lancet variant calling.
Per recommendations by the New York Genome Center <span class="citation" data-cites="bKthX7vJ">[<a href="#ref-bKthX7vJ" role="doc-biblioref">48</a>]</span>, for WGS samples, the Lancet input intervals described above were augmented with PASS variant calls from Strelka2 and Mutect2 as validation.</p>
<h5 id="vcf-annotation-and-maf-creation">VCF annotation and MAF creation</h5>
<p>Normalization of INDELs using <code>bcftools norm</code> <span class="citation" data-cites="SkVcTxDP">[<a href="#ref-SkVcTxDP" role="doc-biblioref">49</a>]</span> was performed on all PASS VCFs using the following subworkflow <span class="citation" data-cites="TeJdzIor">[<a href="#ref-TeJdzIor" role="doc-biblioref">50</a>]</span>, release v3.
The ENSEMBL Variant Effect Predictor <span class="citation" data-cites="p1f5DxRQ">[<a href="#ref-p1f5DxRQ" role="doc-biblioref">51</a>]</span>, reference release 93, was used to annotate variants and bcftools was used to add population allele frequency (AF) from gnomAD.
SNV and INDEL hotspots from v2 of MSKCC’s database <span class="citation" data-cites="qEpbKZ3t">[<a href="#ref-qEpbKZ3t" role="doc-biblioref">52</a>]</span> plus the C228T and C250T TERT promoter mutations <span class="citation" data-cites="59DKOl44">[<a href="#ref-59DKOl44" role="doc-biblioref">53</a>]</span> were annotated.
SNVs were annotated by matching amino acid position (<code>Protein_position</code> column in MAF file) with SNVs in the MSKCC database, splice sites were matched to <code>HGVSp_Short</code>values in the MSKCC database, and INDELs were matched based on amino acid present within the range of INDEL hotspots values in the MSKCC database.
Non-hotspot annotated variants with a normal depth of &lt;= 7 and/or gnomAD AF &gt; 0.001 were removed as potential germline variants.
TERT promoter mutations were matched using hg38 coordinates from <span class="citation" data-cites="59DKOl44">[<a href="#ref-59DKOl44" role="doc-biblioref">53</a>]</span>: C228T occurs at 5:1295113, is annotated as existing variant <code>s1242535815</code>, <code>COSM1716563</code>, <code>COSM1716558</code>, and is 66bp away from TSS and C250T occurs at Chr5:1295135, is annotated as existing variant <code>COSM1716559</code>, and is 88 bp away from TSS.</p>
<p>The final set of variants were retained if annotated as PASS or HotSpotAllele=1.
MAFs were created using MSKCC’s vcf2maf <span class="citation" data-cites="11Fn5jqUI">[<a href="#ref-11Fn5jqUI" role="doc-biblioref">54</a>]</span> v1.6.17.</p>
<h5 id="gather-snv-and-indel-hotspots">Gather SNV and INDEL Hotspots</h5>
<p>All variant calls from Strelka2, Mutect2, or Lancet that overlap with an SNV or INDEL hotspot from v2 of MSKCC’s database <span class="citation" data-cites="qEpbKZ3t">[<a href="#ref-qEpbKZ3t" role="doc-biblioref">52</a>]</span> or the C228T and C250T TERT promoter mutations <span class="citation" data-cites="59DKOl44">[<a href="#ref-59DKOl44" role="doc-biblioref">53</a>]</span> were retained in a hotspot-specific MAF file, which was used for select analyses as described in the methods below.
VarDict-only calls were not retained since ~ 39M calls with low VAF were uniquely called and may be potential false positives.</p>
<h5 id="consensus-snv-calling">Consensus SNV Calling</h5>
<p>Our SNV calling process led to separate sets of predicted mutations for each caller.
We considered mutations to describe the same change if they were identical for the following MAF fields: <code>Chromosome</code>, <code>Start_Position</code>, <code>Reference_Allele</code>, <code>Allele</code>, and <code>Tumor_Sample_Barcode</code>.
Strelka2 does not call multinucleotide variants (MNV), but instead calls each component SNV as a separate mutation, so we separated MNV calls from Mutect2 and Lancet into consecutive SNVs before comparing them with Strelka2.
We examined the variant allele frequencies produced by each caller and compared their overlap with each other <span class="citation" data-cites="Ob2wQ9wX">[<a href="#ref-Ob2wQ9wX" role="doc-biblioref">55</a>]</span>.
VarDict calls included many variants that were not identified by other callers <span class="citation" data-cites="Z1i0sjNH">[<a href="#ref-Z1i0sjNH" role="doc-biblioref">56</a>]</span>, while the other callers produced results that were relatively consistent with one another.
Many of these VarDict-specific calls were variants with low allele frequency <span class="citation" data-cites="tkwfe0u6">[<a href="#ref-tkwfe0u6" role="doc-biblioref">57</a>]</span>.
We termed mutations shared among the other three callers (Strelka2, Mutect2, and Lancet) to be consensus mutation calls and dropped VarDict due to concerns about it calling a large number of false positives.
In practice, because our filtered set was based on the intersection of these three sets and because VarDict called nearly every mutation from the other three callers plus many that were unique to it, the decision to not consider VarDict calls has little impact on the results.</p>
<p>For some downstream analyses, only coding sequence SNVs (based on GENCODE v27 <span class="citation" data-cites="9oRUnb2k">[<a href="#ref-9oRUnb2k" role="doc-biblioref">58</a>]</span>) are used, to enhance comparability to other studies.
We considered base pairs to be <em>effectively surveyed</em> if they were in the intersection of the genomic ranges considered by the callers used to generate the consensus and where appropriate, regions of interest, such as coding sequences.
This definition of <em>effectively surveyed</em> base pairs is what is used to calculate effective genome size for calculations for tumor mutation burden.</p>
<h4 id="somatic-copy-number-variant-calling-wgs-samples-only">Somatic Copy Number Variant Calling (WGS samples only)</h4>
<p>We used Control-FREEC <span class="citation" data-cites="ZQ0L3o1q 1F3i4BvCt">[<a href="#ref-ZQ0L3o1q" role="doc-biblioref">59</a>,<a href="#ref-1F3i4BvCt" role="doc-biblioref">60</a>]</span> v11.6 and CNVkit <span class="citation" data-cites="UTxRcYIQ">[<a href="#ref-UTxRcYIQ" role="doc-biblioref">61</a>]</span> v0.9.3 for copy number variant calls.
For both algorithms, the <code>germline_sex_estimate</code> (described below) was used as input for sample sex and germline variant calls (above) were used as input for BAF estimation.
Control-FREEC was run on human genome reference hg38 using the optional parameters of a 0.05 coefficient of variation, ploidy choice of 2-4, and BAF adjustment for tumor-normal pairs.
Theta2 <span class="citation" data-cites="V4PckbrH">[<a href="#ref-V4PckbrH" role="doc-biblioref">62</a>]</span> used VarDict germline and somatic calls, filtered on PASS and strongly somatic, to infer tumor purity.
Theta2 purity was added as an optional parameter to CNVkit to adjust copy number calls.
CNVkit was run on human genome reference hg38 using the optional parameters of Theta2 purity and BAF adjustment for tumor-normal pairs.
We used GISTIC <span class="citation" data-cites="rDV1hhfF">[<a href="#ref-rDV1hhfF" role="doc-biblioref">63</a>]</span> v.2.0.23 on the CNVkit and the consensus CNV segmentation files to generate gene-level copy number abundance (Log R Ratio) as well as chromosomal arm copy number alterations using the parameters specified in the <a href="https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/run-gistic/scripts/run-gistic-openpbta.sh">OpenPBTA Analysis repository</a>.</p>
<h5 id="consensus-cnv-calling">Consensus CNV Calling</h5>
<p>For each caller and sample, CNVs were called based on consensus among Control-FREEC <span class="citation" data-cites="ZQ0L3o1q 1F3i4BvCt">[<a href="#ref-ZQ0L3o1q" role="doc-biblioref">59</a>,<a href="#ref-1F3i4BvCt" role="doc-biblioref">60</a>]</span>, CNVkit <span class="citation" data-cites="UTxRcYIQ">[<a href="#ref-UTxRcYIQ" role="doc-biblioref">61</a>]</span>, and Manta <span class="citation" data-cites="kTn1PIj5">[<a href="#ref-kTn1PIj5" role="doc-biblioref">64</a>]</span>.
CNVs called significant by Control-FREEC (p-value &lt; 0.01) and Manta calls that passed all filters <span class="citation" data-cites="10RYb622Q">[<a href="#ref-10RYb622Q" role="doc-biblioref">65</a>]</span> were included in consensus calling.
Sample and caller combination files with more than 2500 CNVs called were removed from the set; we expect these to be noisy and poor quality samples based on cutoffs used in GISTIC <span class="citation" data-cites="rDV1hhfF">[<a href="#ref-rDV1hhfF" role="doc-biblioref">63</a>]</span>.
For each sample, the following regions are included in the final consensus set: 1) regions with reciprocal overlap of 50% or more between two of the three callers; 2) smaller CNV regions in which more than 90% of regions are covered by another caller.
Any copy number alteration that was not called by two or more callers was not included in the consensus file.
For the samples that are included in the consensus file, if a certain region has a neutral call, copy number of <code>NA</code> is defined for that region.
CNV regions within 10,000 bp of each other with the same direction of gain or loss were merged into single region.
We filtered out any CNVs that overlapped 50% or more with immunoglobulin, telomeric, centromeric, segment duplicated regions or were shorter than 3000 bp.</p>
<h4 id="somatic-structural-variant-calling-wgs-samples-only">Somatic Structural Variant Calling (WGS samples only)</h4>
<p>We used Manta SV <span class="citation" data-cites="kTn1PIj5">[<a href="#ref-kTn1PIj5" role="doc-biblioref">64</a>]</span> v1.4.0 for structural variant (SV) calls.
Manta SV calling was also limited to regions used in Strelka2.
The hg38 reference for SV calling used was limited to canonical chromosome regions.
The somatic DNA workflows for SNV, indel, copy number, and SV calling can be found in the <a href="https://github.com/d3b-center/OpenPBTA-workflows/">OpenPBTA workflows Github repository</a>.
Manta SV output was annotated using <a href="https://lbgi.fr/AnnotSV/">AnnotSV v2.1</a> <span class="citation" data-cites="7Ull5aU5">[<a href="#ref-7Ull5aU5" role="doc-biblioref">66</a>]</span> and the workflow can be found in the <a href="https://github.com/d3b-center/annotSV/tree/master">D3b GitHub repository</a>.</p>
<h4 id="gene-expression">Gene Expression</h4>
<h5 id="abundance-estimation">Abundance Estimation</h5>
<p>We used STAR <span class="citation" data-cites="tTu8Ds9Z">[<a href="#ref-tTu8Ds9Z" role="doc-biblioref">67</a>]</span> v2.6.1d to align paired-end RNA-seq reads.
This output was used for all subsequent RNA analysis.
We used Ensembl GENCODE 27 <span class="citation" data-cites="9oRUnb2k">[<a href="#ref-9oRUnb2k" role="doc-biblioref">58</a>]</span>, “Comprehensive gene annotation” as a reference.
We used RSEM <span class="citation" data-cites="Dh2n1EV3">[<a href="#ref-Dh2n1EV3" role="doc-biblioref">68</a>]</span> v1.3.1 for both FPKM and TPM transcript- and gene-level quantification.
We also added a second method of quantification using kallisto <span class="citation" data-cites="12KZMHMQl">[<a href="#ref-12KZMHMQl" role="doc-biblioref">69</a>]</span> v0.43.1.
This method differs in that it uses pseudoaligments using FASTQ reads directly to the aforementioned GENCODE 27 reference.</p>
<h5 id="gene-expression-matrices-with-unique-hugo-symbols">Gene Expression Matrices with Unique HUGO Symbols</h5>
<p>Algorithms that perform gene set enrichment, molecular subtyping, or immune-profiling, for example, require an RNA-seq gene expression matrix as input, with HUGO gene symbols as row names and sample names as column names.
There is a small proportion of gene symbols that map to multiple Ensembl gene identifiers (in GENCODE v27, 212 gene symbols map to 1866 Ensembl gene identifiers), termed multi-mapped gene symbols.</p>
<p>We first removed genes with no expression from the RSEM abundance data using a cut-off of FPKM &gt; 0 in at least 1 sample across the PBTA cohort.
We computed the mean FPKM across all samples per gene and for each multi-mapped gene symbol, we chose the Ensembl identifier corresponding to the maximum mean FPKM with the goal of choosing the identifier that best represented the expression of the gene.
After collapsing gene identifiers, there were a total of 46,400 unique expressed genes in the poly-A dataset and a total of 53,011 unique expressed genes remaining in the stranded dataset.
More detail can be found in the <a href="https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/collapse-rnaseq"><code>collapse-rnaseq</code> analysis module</a>.</p>
<h5 id="gene-fusion-detection">Gene fusion detection</h5>
<p>We set up <a href="https://github.com/suhrig/arriba/">Arriba v1.1.0</a> and STAR-Fusion 1.5.0 <span class="citation" data-cites="rqYlIiAi">[<a href="#ref-rqYlIiAi" role="doc-biblioref">70</a>]</span> fusion detection tools using CWL on CAVATICA.
For both of these tools, we used aligned BAM and chimeric SAM files from STAR as inputs and GRCh38_gencode_v27 GTF for gene annotation.
We ran STAR-Fusion with default parameters and annotated all fusion calls with GRCh38_v27_CTAT_lib_Feb092018.plug-n-play.tar.gz provided in the STAR-fusion release.
For Arriba, we used a blacklist file (blacklist_hg38_GRCh38_2018-11-04.tsv.gz) from the Arriba release to remove recurrent fusion artifacts and transcripts present in healthy tissue.
We also provided Arriba with strandedness information for stranded samples, or we set it to auto-detection for poly-A samples.
We used <a href="https://github.com/FusionAnnotator/FusionAnnotator">FusionAnnotator</a> on Arriba fusion calls in order to harmonize annotations with those of STAR-Fusion.
The RNA expression and fusion workflows can be found in the <a href="https://github.com/d3b-center/OpenPBTA-workflows/blob/master/cwl/kfdrc_RNAseq_workflow.cwl">OpenPBTA workflows GitHub repository</a>.
The FusionAnnotator workflow can be found in the <a href="https://github.com/d3b-center/FusionAnnotator">D3b GitHub repository</a>.</p>
<h3 id="quantification-and-statistical-analysis">QUANTIFICATION AND STATISTICAL ANALYSIS</h3>
<h5 id="recurrently-mutated-genes-and-co-occurrence-of-gene-mutations">Recurrently mutated genes and co-occurrence of gene mutations</h5>
<p>Using the consensus SNV calls, we identified genes that were recurrently mutated in the cohort, including nonsynonymous mutations with a variant allele frequency greater than 5% among the set of independent samples.
The set of nonsynonymous mutations was determined using ENSEMBL Variant Effect Predictor <span class="citation" data-cites="p1f5DxRQ">[<a href="#ref-p1f5DxRQ" role="doc-biblioref">51</a>]</span> annotations, including High and Moderate consequence types as defined in <code>maftools v. 2.2.10</code> <span class="citation" data-cites="83Prx3Xc">[<a href="#ref-83Prx3Xc" role="doc-biblioref">71</a>]</span>.
For each gene, we then tallied the number of samples that had at least one nonsynonymous mutation.</p>
<p>For genes that contained nonsynonymous mutations in multiple samples, we calculated pairwise mutation co-occurrence scores.
This score was defined as the <span class="math inline">\(I\times -\log_{10}(P)\)</span> where <span class="math inline">\(I\)</span> is 1 when the odds ratio is &gt; 1 (indicating co-occurrence), and -1 when the odds ratio is &lt; 1 (indicating mutual exclusivity), with <span class="math inline">\(P\)</span> defined by Fisher’s Exact Test.</p>
<h5 id="focal-copy-number-calling">Focal Copy Number Calling</h5>
<p>We added the ploidy inferred via Control-FREEC to the consensus CNV segmentation file and used the ploidy and copy number values to define gain and loss values broadly at the chromosome level.
We used bedtools coverage <span class="citation" data-cites="Ddy8jB2X 1HWiAHnIw">[<a href="#ref-Ddy8jB2X" role="doc-biblioref">72</a>,<a href="#ref-1HWiAHnIw" role="doc-biblioref">73</a>]</span> to add cytoband status using the UCSC cytoband file <span class="citation" data-cites="H6flbhE 1DWggwY7R">[<a href="#ref-H6flbhE" role="doc-biblioref">74</a>,<a href="#ref-1DWggwY7R" role="doc-biblioref">75</a>]</span>.
The output status call fractions, which are values of the loss, gain and callable fractions of each cytoband region, were used to define dominant status at the cytoband-level.
The weighted means of each status call fraction were calculated using band length.
We used the weighted means to define the dominant status at the chromosome arm-level.</p>
<p>A status was considered dominant if more than half of the region was callable and the status call fraction was greater than 0.9 for that region.
The 0.9 threshold was chosen to ensure that the dominant status fraction call is greater than the remaining status fraction calls in a region.</p>
<p>We also wanted to define focal copy number units to avoid calling adjacent genes in the same cytoband or arm as copy number losses or gains where it would be more appropriate to call the broader region a loss or gain.
For the determination of the most focal units, we first considered the dominant status calls at the chromosome arm-level.
If the chromosome arm dominant status was callable but not clearly defined as a gain or loss, we instead included the cytoband-level status call.
Similarly, if a cytoband dominant status call was callable but not clearly defined as a gain or loss, we instead included gene-level status call.
To obtain the gene-level data, we used the mergeByOverlaps function <span class="citation" data-cites="GAYU0k3L">[<a href="#ref-GAYU0k3L" role="doc-biblioref">76</a>]</span> from the IRanges package in R <span class="citation" data-cites="14d6Jzwus">[<a href="#ref-14d6Jzwus" role="doc-biblioref">77</a>]</span> to find overlaps between the segments in the consensus CNV file and the exons in the GENCODE v27 annotation file <span class="citation" data-cites="9oRUnb2k">[<a href="#ref-9oRUnb2k" role="doc-biblioref">58</a>]</span>.
If the copy number value was 0, we set the status to “deep deletion”.
For autosomes only, we set the status to “amplification” when the copy number value was greater than two times the ploidy value.</p>
<h4 id="chromothripsis-analysis-wgs-samples-only">Chromothripsis Analysis (WGS samples only)</h4>
<p>Candidate chromothripsis regions were identified in the set of independent tumor WGS samples with ShatterSeek <span class="citation" data-cites="17wG7AMwM">[<a href="#ref-17wG7AMwM" role="doc-biblioref">78</a>]</span>, using Manta SV calls that passed all filters and consensus CNV calls.
Only chromosomes 1-22 and X were considered.
The consensus CNV data were modified to fit ShatterSeek input requirements: CNV-neutral or excluded regions (both annotated as <code>NA</code> in the consensus data) were filled in with the respective sample’s ploidy value from Control-FREEC, and consecutive segments with the same copy number value were merged.
Candidate chromothripsis regions were classified as high- or low-confidence by applying the statistical criteria described by the ShatterSeek authors.</p>
<h5 id="immune-profiling-and-deconvolution">Immune Profiling and Deconvolution</h5>
<p>We used the R package <code>immunedeconv</code> <span class="citation" data-cites="HnrrHXPm klaJnG8x">[<a href="#ref-HnrrHXPm" role="doc-biblioref">79</a>,<a href="#ref-klaJnG8x" role="doc-biblioref">80</a>]</span> with the method <code>quanTIseq</code> <span class="citation" data-cites="10.1186/s13073-019-0638-6">[<span class="citeproc-not-found" data-reference-id="10.1186/s13073-019-0638-6"><strong>???</strong></span>]</span> to deconvolute various immune cell types across tumors from the PBTA cohort in the stranded and poly-A collapsed FPKM RNA-seq datasets (<a href="https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/immune-deconv"><code>immune-deconv</code> analysis module</a>).
The <code>quanTIseq</code> deconvolution method directly estimates absolute fractions of 10 immune cell types that represent inferred proportions of the cell types in the mixture.
Therefore, we utilized <code>quanTIseq</code> for inter-sample, intra-sample, and inter-histology score comparisons.</p>
<h5 id="gene-set-variation-analysis">Gene Set Variation Analysis</h5>
<p>We performed Gene Set Variation Analysis (GSVA) <span class="citation" data-cites="1F5zJEMaD">[<a href="#ref-1F5zJEMaD" role="doc-biblioref">81</a>]</span> on collapsed, log2-transformed RSEM FPKM data using the GSVA Bioconductor package <span class="citation" data-cites="w3jDgsn7">[<a href="#ref-w3jDgsn7" role="doc-biblioref">82</a>]</span> with setting <code>mx.diff=TRUE</code> to obtain Gaussian-distributed scores (<a href="https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/gene-set-enrichment-analysis"><code>gene-set-enrichment-analysis</code> analysis module</a>) for each of the MSigDB hallmark gene sets <span class="citation" data-cites="1CbVoEpNJ">[<a href="#ref-1CbVoEpNJ" role="doc-biblioref">83</a>]</span>.
We compared GSVA scores among histology groups (<code>short_histology</code>) using ANOVA and subsequent Tukey tests; p-values were Bonferroni-corrected for multiple hypothesis testing.</p>
<h5 id="transcriptomic-dimension-reduction">Transcriptomic Dimension reduction</h5>
<p>We applied Uniform Manifold Approximation and Projection (UMAP) <span class="citation" data-cites="14XP70sGJ">[<a href="#ref-14XP70sGJ" role="doc-biblioref">84</a>]</span> to log2-transformed FPKM data using the umap R package <span class="citation" data-cites="WsAj0wOM">[<a href="#ref-WsAj0wOM" role="doc-biblioref">85</a>]</span>.
We set the number of neighbors to 15 (<a href="https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/transcriptomic-dimension-reduction"><code>transcriptomic-dimension-reduction</code> analysis module</a>).</p>
<h5 id="fusion-prioritization">Fusion prioritization</h5>
<p>We performed artifact filtering and additional annotation on fusion calls to prioritize putative oncogenic fusions.
Briefly, we considered all in frame and frameshift fusion calls with a minimum of 1 junction reads and at least one gene partner expressed (TPM &gt; 1) to be true calls.
If a fusion call had large number of spanning fragment reads compared to junction reads (spanning fragment minus junction read greater than ten), we removed these calls as potential false positives.
We prioritized a union of fusion calls as true calls if the fused genes were detected by both callers, the same fusion was recurrent within a <code>broad_histology</code> (&gt;2 samples) or the fusion was specific to the <code>broad_histology</code>.
If either 5’ or 3’ genes fused to more than five different genes within a sample, we removed these calls as potential false positives.
We annotated putative driver fusions and prioritized fusions based on partners containing known <a href="http://kinase.com/human/kinome/tables/Kincat_Hsap.08.02.xls">kinases</a>, <a href="http://www.bushmanlab.org/assets/doc/allOnco_Feb2017.tsv">oncogenes</a>, <a href="https://bioinfo.uth.edu/TSGene/Human_TSGs.txt?csrt=5027697123997809089">tumor suppressors</a>, curated transcription factors <span class="citation" data-cites="9vS8HBL6">[<a href="#ref-9vS8HBL6" role="doc-biblioref">86</a>]</span>, <a href="https://cancer.sanger.ac.uk/census">COSMIC genes</a>, and/or known <a href="https://tumorfusions.org/PanCanFusV2/downloads/pancanfus.txt.gz">TCGA fusions</a> from curated <a href="https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/fusion_filtering/references">references</a>.
<em>MYBL1</em> <span class="citation" data-cites="jLWV5IWB">[<a href="#ref-jLWV5IWB" role="doc-biblioref">87</a>]</span>, <em>SNCAIP</em> <span class="citation" data-cites="4wYR62jK">[<a href="#ref-4wYR62jK" role="doc-biblioref">88</a>]</span>, <em>FOXR2</em> <span class="citation" data-cites="kfmK8vm">[<a href="#ref-kfmK8vm" role="doc-biblioref">89</a>]</span>, <em>TTYH1</em> <span class="citation" data-cites="5ueZBnsJ">[<a href="#ref-5ueZBnsJ" role="doc-biblioref">90</a>]</span>, and <em>TERT</em> <span class="citation" data-cites="ASmwGlFp YfG9EVSk lWMOs28t 1B3tdZcAl">[<a href="#ref-ASmwGlFp" role="doc-biblioref">91</a>,<a href="#ref-YfG9EVSk" role="doc-biblioref">92</a>,<a href="#ref-lWMOs28t" role="doc-biblioref">93</a>,<a href="#ref-1B3tdZcAl" role="doc-biblioref">94</a>]</span> were added to the oncogene list and <em>BCOR</em> <span class="citation" data-cites="kfmK8vm">[<a href="#ref-kfmK8vm" role="doc-biblioref">89</a>]</span> and <em>QKI</em> <span class="citation" data-cites="1foRpfch">[<a href="#ref-1foRpfch" role="doc-biblioref">95</a>]</span> were added to the tumor suppressor gene list based on pediatric cancer literature review.
The fusion filtering workflow can be found in the <a href="https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/fusion_filtering">OpenPBTA Analysis repository</a>.</p>
<h4 id="oncoprint-figure-generation">Oncoprint figure generation</h4>
<p>Maftools v. 2.2.10 <span class="citation" data-cites="83Prx3Xc">[<a href="#ref-83Prx3Xc" role="doc-biblioref">71</a>]</span> was used to generate oncoprints depicting the frequencies of canonical somatic gene mutations, CNVs, and fusions for the top 20 genes mutated across primary tumors within broad histologies of the OpenPBTA dataset.
Canonical genes were collated from review of the literature for low-grade astrocytic tumors <span class="citation" data-cites="IamVuiNU">[<a href="#ref-IamVuiNU" role="doc-biblioref">13</a>]</span>, embryonal tumors <span class="citation" data-cites="UuzX8B9C 1Geb70JK9 j11VxJIm CeB34T1V 1FGkQ9Wa7">[<a href="#ref-1Geb70JK9" role="doc-biblioref">14</a>,<a href="#ref-j11VxJIm" role="doc-biblioref">16</a>,<a href="#ref-UuzX8B9C" role="doc-biblioref">17</a>,<a href="#ref-CeB34T1V" role="doc-biblioref">96</a>,<a href="#ref-1FGkQ9Wa7" role="doc-biblioref">97</a>]</span>, diffuse astrocytic and oligodendroglial tumors <span class="citation" data-cites="vd5Uh9gY BLNV1DbL 6F5vK3sT KU8wVb8T">[<a href="#ref-BLNV1DbL" role="doc-biblioref">5</a>,<a href="#ref-6F5vK3sT" role="doc-biblioref">10</a>,<a href="#ref-vd5Uh9gY" role="doc-biblioref">19</a>,<a href="#ref-KU8wVb8T" role="doc-biblioref">20</a>]</span>, and other tumors (ependymal tumors, craniopharyngiomas, neuronal-glial mixed tumors, histiocytic tumors, chordoma, meningioma, and choroid plexus tumors)
<span class="citation" data-cites="m7hgzrvh ZXjw164D f7P6U50v DGVkAk1W ovoneiG3 uDYz8KKh 107ZjoLxM F73HShWn doi:10.1016/s0002-9440 BBuQnfgj">[<span class="citeproc-not-found" data-reference-id="doi:10.1016/s0002-9440"><strong>???</strong></span>,<a href="#ref-m7hgzrvh" role="doc-biblioref">98</a>,<a href="#ref-ZXjw164D" role="doc-biblioref">99</a>,<a href="#ref-f7P6U50v" role="doc-biblioref">100</a>,<a href="#ref-DGVkAk1W" role="doc-biblioref">101</a>,<a href="#ref-ovoneiG3" role="doc-biblioref">102</a>,<a href="#ref-uDYz8KKh" role="doc-biblioref">103</a>,<a href="#ref-107ZjoLxM" role="doc-biblioref">104</a>,<a href="#ref-F73HShWn" role="doc-biblioref">105</a>,<a href="#ref-BBuQnfgj" role="doc-biblioref">106</a>]</span>.</p>
<h4 id="mutational-signatures">Mutational Signatures</h4>
<p>We obtained weights (i.e., exposures) for signature sets using the <code>deconstructSigs</code> R package function <code>whichSignatures()</code> <span class="citation" data-cites="KhxTOfIb 1eIeMT1A">[<a href="#ref-KhxTOfIb" role="doc-biblioref">107</a>,<a href="#ref-1eIeMT1A" role="doc-biblioref">108</a>]</span> from consensus SNVs with the BSgenome.Hsapiens.UCSC.hg38 annotations <span class="citation" data-cites="GpcQXpz9">[<a href="#ref-GpcQXpz9" role="doc-biblioref">109</a>]</span>.
Specifically, we estimated signature weights across samples for eight signatures previously identified in the Signal reference set of signatures (“RefSig”) as associated with adult central nervous system (CNS) tumors <span class="citation" data-cites="Cc8jt6lX">[<a href="#ref-Cc8jt6lX" role="doc-biblioref">24</a>]</span>.
These eight RefSig signatures are 1, 3, 8, 11, 18, 19, N6, and MMR2.
Weights for signatures fall in the range zero to one inclusive.
<code>deconstructSigs</code> estimates the weights for each signature across samples and allows for a proportion of unassigned weights referred to as “Other” in the text.
These results do not include signatures with small contributions; <code>deconstructSigs</code> drops signature weights that are less than 6% <span class="citation" data-cites="KhxTOfIb">[<a href="#ref-KhxTOfIb" role="doc-biblioref">107</a>]</span>.</p>
<h3 id="tumor-mutation-burden">Tumor Mutation Burden</h3>
<p>We consider tumor mutation burden (TMB) to be the number of consensus SNVs per <em>effectively surveyed</em> base of the genome.</p>
<p><span class="math display">\[ \textrm{TMB} = \frac{\textrm{\# of coding sequence SNVs}}{\textrm{Size in Mb of {\em effectively surveyed} genome}}
\]</span></p>
<p>We used the total number coding sequence consensus SNVs for the numerator and the size of the intersection of the regions considered by Strelka2 and Mutect2 with coding regions (CDS from GENCODE v27 annotation <span class="citation" data-cites="9oRUnb2k">[<a href="#ref-9oRUnb2k" role="doc-biblioref">58</a>]</span>) as the denominator.</p>
<p>Lancet showed a bias for calling mutations with a much lower VAF for WXS than WGS data (<strong>Figure S2, panel G</strong>).
Since TCGA data was all WXS, we dropped Lancet from TMB calculations. <span class="citation" data-cites="url">[<span class="citeproc-not-found" data-reference-id="url"><strong>???</strong></span> https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/snv-callers/lancet-wxs-tests]</span>.</p>
<h4 id="clinical-data-harmonization">Clinical Data Harmonization</h4>
<!-- TODO: pull these "S1 and S2 tables out and append to actual Table S1" -->
<h5 id="who-classification-of-disease-types">WHO Classification of Disease Types</h5>
<p>The <code>pathology_diagnosis</code> field in the <code>pbta-histologies.tsv</code> file contains one or more diagnoses from on the patient’s pathology report.
The <code>pathology_free_text_diagnosis</code> field in the <code>pbta-histologies.tsv</code> file contains additional free text diagnosis information gathered from the patient’s pathology report.
The <code>broad_histology</code> denotes the broad 2016 WHO classification for each tumor.
The <code>short_histology</code> is an abbreviated version of either the <code>broad_histology</code> or <code>integrated_diagnosis</code> for plotting purposes.
Except for LGAT samples, the <code>integrated_diagnosis</code> field in the <code>pbta-histologies.tsv</code> file was derived to match a standardized 2016 WHO diagnosis <span class="citation" data-cites="7sgrMadR">[<a href="#ref-7sgrMadR" role="doc-biblioref">9</a>]</span> based on <code>pathology_diagnosis</code>, molecular subtyping, and in some cases, additional pathology review.
The <code>harmonized_diagnosis</code> is the final <code>integrated_diagnosis</code>, if one exists, or a diagnosis derived from the <code>pathology_diagnosis</code> and <code>pathology_free_text_diagnosis</code> in the absence of molecular data.
The <code>cancer_group</code> is a grouping narrower than <code>broad_histology</code> derived within the <a href="https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/molecular-subtyping-integrate">molecular subtyping integrate module</a>.
With clinician assistance, the <code>CNS_region</code> was categorized as hemispheric, midline, mixed, optic pathway, posterior fossa, spine, suprasellar, ventricles or other based on specimen location (see table below).</p>
<table>
<colgroup>
<col style="width: 10%" />
<col style="width: 44%" />
<col style="width: 45%" />
</colgroup>
<thead>
<tr class="header">
<th>Clinical and Histology Metadata</th>
<th>Definition</th>
<th>Possible values</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>age_at_diagnosis_days</td>
<td>Patient age at diagnosis in days</td>
<td>numeric</td>
</tr>
<tr class="even">
<td>age_last_update_days</td>
<td>Patient age at the last clinical event/update in days</td>
<td>numeric</td>
</tr>
<tr class="odd">
<td>aliquot_id</td>
<td>External aliquot identifier</td>
<td>variable</td>
</tr>
<tr class="even">
<td>broad_histology</td>
<td>Broad WHO 2016 classification of cancer type</td>
<td>text</td>
</tr>
<tr class="odd">
<td>cancer_group</td>
<td>Harmonized cancer groupings for plots</td>
<td>text</td>
</tr>
<tr class="even">
<td>cancer_predispositions</td>
<td>Reported cancer predisposition syndromes</td>
<td>text</td>
</tr>
<tr class="odd">
<td>CNS_region</td>
<td>Harmonized brain region based on <code>primary_site</code></td>
<td>Hemispheric;Midline;Mixed;Optic pathway;Other;Posterior fossa;Spine;Suprasellar;Ventricles</td>
</tr>
<tr class="even">
<td>cohort</td>
<td>Scientific cohort</td>
<td>CBTN;PNOC</td>
</tr>
<tr class="odd">
<td>cohort_participant_id</td>
<td>Scientific cohort participant ID</td>
<td>C#####-C######</td>
</tr>
<tr class="even">
<td>composition</td>
<td>Sample composition</td>
<td>Derived Cell Line;Not Reported;Peripheral Whole Blood;Saliva;Solid Tissue</td>
</tr>
<tr class="odd">
<td>ethnicity</td>
<td>Patient reported ethnicity</td>
<td>text</td>
</tr>
<tr class="even">
<td>experimental_strategy</td>
<td>Sequencing strategy</td>
<td>WGS;WXS;RNA-Seq;Panel</td>
</tr>
<tr class="odd">
<td>extent_of_tumor_resection</td>
<td>Amount of tumor resected at time of surgical event</td>
<td>Biopsy only;Partial resection;Gross/Near total resection;Not Reported;Unavailable</td>
</tr>
<tr class="even">
<td>germline_sex_estimate</td>
<td>Predicted sex of patient based on germline X and Y ratio calculation (described in methods)</td>
<td>Female;Male;Unknown</td>
</tr>
<tr class="odd">
<td>harmonized_diagnosis</td>
<td><code>integrated_diagnosis</code>, if exists, or updated and harmonized diagnosis using pathology_free_text_diagnosis information</td>
<td>text</td>
</tr>
<tr class="even">
<td>integrated_diagnosis</td>
<td>2016 WHO diagnosis integrated from pathology diagnosis and molecular subtyping</td>
<td>text</td>
</tr>
<tr class="odd">
<td>Kids_First_Biospecimen_ID</td>
<td>KidsFirst biospecimen identifier</td>
<td>BS_########</td>
</tr>
<tr class="even">
<td>Kids_First_Participant_ID</td>
<td>KidsFirst patient identifier</td>
<td>PT_########</td>
</tr>
<tr class="odd">
<td>molecular_subtype</td>
<td>Molecular subtype defined by WHO 2016 guidelines</td>
<td>text</td>
</tr>
<tr class="even">
<td>normal_fraction</td>
<td>Theta2 normal DNA fraction estimate</td>
<td>numeric</td>
</tr>
<tr class="odd">
<td>Notes</td>
<td>Free text field describing changes from <code>pathology_diagnosis</code> to <code>integrated_diagnosis</code> or manner in which molecular_subtype was determined</td>
<td>text</td>
</tr>
<tr class="even">
<td>OS_days</td>
<td>Overall survival in days</td>
<td>numeric</td>
</tr>
<tr class="odd">
<td>OS_status</td>
<td>Overall survival status</td>
<td>DECEASED;LIVING</td>
</tr>
<tr class="even">
<td>parent_aliquot_id</td>
<td>External identifier combining sample_id, sample_type, aliquot_id, and sequencing_strategy for some samples</td>
<td>text</td>
</tr>
<tr class="odd">
<td>pathology_diagnosis</td>
<td>Reported and/or harmonized patient diagnosis from pathology reports</td>
<td>text</td>
</tr>
<tr class="even">
<td>pathology_free_text_diagnosis</td>
<td>Free text patient diagnosis from pathology reports</td>
<td>text</td>
</tr>
<tr class="odd">
<td>PFS_days</td>
<td>Progression-free survival in days</td>
<td>numeric</td>
</tr>
<tr class="even">
<td>primary_site</td>
<td>Bodily site(s) from which specimen was derived</td>
<td>text</td>
</tr>
<tr class="odd">
<td>race</td>
<td>Patient reported race</td>
<td>text</td>
</tr>
<tr class="even">
<td>reported_gender</td>
<td>Patient reported gender</td>
<td>text</td>
</tr>
<tr class="odd">
<td>RNA_library</td>
<td>Type of RNA-Sequencing library preparation</td>
<td>stranded;poly-A</td>
</tr>
<tr class="even">
<td>sample_id</td>
<td>External biospecimen identifier</td>
<td>variable</td>
</tr>
<tr class="odd">
<td>sample_type</td>
<td>Broad sample type</td>
<td>Normal;Tumor</td>
</tr>
<tr class="even">
<td>seq_center</td>
<td>Sequencing center</td>
<td>BGI;BGI@CHOP Genome Center;Genomic Clinical Core at Sidra Medical and Research Center;NantOmics;The Translational Genomics Research Institute</td>
</tr>
<tr class="odd">
<td>short_histology</td>
<td>Abbreviated <code>integrated_diagnosis</code> or <code>broad_histology</code> for plotting purposes</td>
<td>text</td>
</tr>
<tr class="even">
<td>tumor_descriptor</td>
<td>Phase of therapy from which tumor was derived</td>
<td>Initial CNS Tumor;Progressive Progressive Disease Post-Mortem;Recurrence;Second Malignancy;Unavailable</td>
</tr>
<tr class="odd">
<td>tumor_fraction</td>
<td>Theta2 tumor DNA fraction estimate</td>
<td>numeric</td>
</tr>
<tr class="even">
<td>tumor_ploidy</td>
<td>Control-FREEC ploidy</td>
<td>numeric</td>
</tr>
</tbody>
</table>
<p>Table S1. Clinical metadata collected for OpenPBTA. {#tbl:S1}</p>
<table>
<colgroup>
<col style="width: 11%" />
<col style="width: 88%" />
</colgroup>
<thead>
<tr class="header">
<th>CNS_region</th>
<th>primary_site</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Hemispheric</td>
<td>Frontal Lobe,Temporal Lobe,Parietal Lobe,Occipital Lobe</td>
</tr>
<tr class="even">
<td>Midline</td>
<td>Pons/Brainstem,Brain Stem- Midbrain/Tectum,Brain Stem- Pons,Brain Stem-Medulla,Thalamus,Basal Ganglia,Hippocampus,Pineal Gland</td>
</tr>
<tr class="odd">
<td>Spine</td>
<td>Spinal Cord- Cervical,Spinal Cord- Thoracic,Spinal Cord- Lumbar/Thecal Sac,Spine NOS</td>
</tr>
<tr class="even">
<td>Ventricles</td>
<td>Ventricles</td>
</tr>
<tr class="odd">
<td>Posterior fossa</td>
<td>Cerebellum/Posterior Fossa</td>
</tr>
<tr class="even">
<td>Optic pathway</td>
<td>Optic Pathway</td>
</tr>
<tr class="odd">
<td>Suprasellar</td>
<td>Suprasellar/Hypothalamic/Pituitary</td>
</tr>
<tr class="even">
<td>Other</td>
<td>Meninges/Dura,Other locations NOS,Skull,Cranial Nerves NOS,Brain</td>
</tr>
</tbody>
</table>
<p>Table S2. Harmonized CNS brain regions derived from primary site values. {#tbl:S2}</p>
<h5 id="molecular-subtyping">Molecular Subtyping</h5>
<p>The <code>molecular_subtype</code> column in the <code>pbta-histologies.tsv</code> file contains molecular subtypes for tumor types selected from <code>pathology_diagnosis</code> and <code>pathology_free_text_diagnosis</code> fields as described below, following World Health Organization 2016 classification criteria <span class="citation" data-cites="7sgrMadR">[<a href="#ref-7sgrMadR" role="doc-biblioref">9</a>]</span>.</p>
<p>Medulloblastoma (MB) subtypes SHH, MYC, Group 3, and Group 4 were predicted using the consensus of two RNA expression classifiers: [Medulloblastoma Classifier]](https://github.com/d3b-center/medullo-classifier-package) and MM2S Classifier <span class="citation" data-cites="RmzXK4Cw">[<a href="#ref-RmzXK4Cw" role="doc-biblioref">110</a>]</span> on the RSEM FPKM data.</p>
<p>High-grade glioma (HGG) subtypes were derived using the criteria below (additional details in the <a href="https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/molecular-subtyping-HGG">analysis README</a>):</p>
<ol type="1">
<li>If any sample contained an <em>H3F3A</em> p.K28M, <em>HIST1H3B</em> p.K28M, <em>HIST1H3C</em> p.K28M, or <em>HIST2H3C</em> p.K28M mutation and no <em>BRAF</em> p.V600E mutation, it was subtyped as <code>DMG, H3K28</code>.</li>
<li>If any sample contained an <em>HIST1H3B</em> p.K28M, <em>HIST1H3C</em> p.K28M, or <em>HIST2H3C</em> p.K28M mutation and a <em>BRAF</em> p.V600E mutation, it was subtyped as <code>DMG, H3 K28, BRAF V600E</code>.</li>
<li>If any sample contained an <em>H3F3A</em> p.G35V or p.G35R mutation, it was subtyped as<code>HGG, H3 G35</code>.</li>
<li>If any high-grade glioma sample contained an <em>IDH1</em> p.R132 mutation, it was subtyped as <code>HGG, IDH</code>.</li>
<li>If a sample was initially classified as HGAT, had no defining histone mutations, and a BRAF p.V600E mutation, it was subtyped as <code>BRAF V600E</code>.</li>
<li>All other high-grade glioma samples that did not meet any of these criteria were subtyped as <code>HGG, H3 wildtype</code>.</li>
</ol>
<p>Embryonal tumors were included in non-MB and non-ATRT embryonal tumor subtyping if they met any of the following criteria:
1. A <em>TTYH1</em> (5’ partner) fusion was detected.
2. A <em>MN1</em> (5’ partner) fusion was detected, with the exception of <code>MN1--PATZ1</code> since it is an entity separate of CNS HGNET-MN1 tumors <span class="citation" data-cites="6bmmWiU5">[<a href="#ref-6bmmWiU5" role="doc-biblioref">111</a>]</span>.
3. Pathology diagnoses included “Supratentorial or Spinal Cord PNET” or “Embryonal Tumor with Multilayered Rosettes”.
4. A pathology diagnosis of “Neuroblastoma”, where the tumor was not indicated to be peripheral or metastatic and was located in the CNS.
5. Any sample with “embryonal tumor with multilayer rosettes, ros (who grade iv)”, “embryonal tumor, nos, congenital type”, “ependymoblastoma” or “medulloepithelioma” in pathology free text.</p>
<p>Non-MB and non-ATRT embryonal tumors identified with the above criteria were further subtyped using the criteria below <span class="citation" data-cites="ynt8AIJ4 t40FioEk 1AM78Fkrd Cibjccyk">[<a href="#ref-ynt8AIJ4" role="doc-biblioref">112</a>,<a href="#ref-t40FioEk" role="doc-biblioref">113</a>,<a href="#ref-1AM78Fkrd" role="doc-biblioref">114</a>,<a href="#ref-Cibjccyk" role="doc-biblioref">115</a>]</span>.
Additional details can be found in the analysis <a href="https://alexslemonade.github.io/OpenPBTA-analysis/analyses/molecular-subtyping-embryonal/04-table-prep.nb.html">notebook</a>.</p>
<ol type="1">
<li>Any RNA-seq biospecimen with <em>LIN28A</em> overexpression, plus a <em>TYH1</em> fusion (5’ partner) with a gene adjacent or within the C19MC miRNA cluster and/or copy number amplification of the C19MC region was subtyped as <code>ETMR, C19MC-altered</code> (Embryonal tumor with multilayer rosettes, chromosome 19 miRNA cluster altered) [<span class="citation" data-cites="7UxjGOxW"><a href="#ref-7UxjGOxW" role="doc-biblioref">116</a></span>; 10.1038/ng.2849].</li>
<li>Any RNA-seq biospecimen with <em>LIN28A</em> overexpression, a <em>TTYH1</em> fusion (5’ partner) with a gene adjacent or within the C19MC miRNA cluster but no evidence of copy number amplification of the C19MC region was subtyped as <code>ETMR, NOS</code> (Embryonal tumor with multilayer rosettes, not otherwise specified) <span class="citation" data-cites="7UxjGOxW 5ueZBnsJ">[<a href="#ref-5ueZBnsJ" role="doc-biblioref">90</a>,<a href="#ref-7UxjGOxW" role="doc-biblioref">116</a>]</span>.</li>
<li>Any RNA-seq biospecimen with a fusion having a 5’ <em>MN1</em> and 3’ <em>BEND2</em> or <em>CXXC5</em> partner were subtyped as <code>CNS HGNET-MN1</code> (Central nervous system (CNS) high-grade neuroepithelial tumor with <em>MN1</em> alteration).</li>
<li>Non-MB and non-ATRT embryonal tumors with internal tandem duplication (as defined in <span class="citation" data-cites="DEDtXX7K">[<a href="#ref-DEDtXX7K" role="doc-biblioref">117</a>]</span>) of <em>BCOR</em> were subtyped as <code>CNS HGNET-BCOR</code> (CNS high-grade neuroepithelial tumor with <em>BCOR</em> alteration).</li>
<li>Non-MB and non-ATRT embryonal tumors with over-expression and/or gene fusions in <em>FOXR2</em> were subtyped as <code>CNS NB-FOXR2</code> (CNS neuroblastoma with <em>FOXR2</em> activation).</li>
<li>Non-MB and non-ATRT embryonal tumors with <em>CIC-NUTM1</em> or other <em>CIC</em> fusions, were subtyped as <code>CNS EFT-CIC</code> (CNS Ewing sarcoma family tumor with <em>CIC</em> alteration) <span class="citation" data-cites="doi:/j.cell.2016.01.015">[<span class="citeproc-not-found" data-reference-id="doi:/j.cell.2016.01.015"><strong>???</strong></span>]</span></li>
<li>Non-MB and non-ATRT embryonal tumors that did not fit any of the above categories were subtyped as <code>CNS Embryonal, NOS</code> (CNS Embryonal tumor, not otherwise specified).</li>
</ol>
<p>Neurocytoma subtypes central neurocytoma (CNC) and extraventricular neurocytoma (EVN) were assigned based on the primary site of the tumor <span class="citation" data-cites="ueHlof7U">[<a href="#ref-ueHlof7U" role="doc-biblioref">118</a>]</span>.
If <code>primary_site</code> of the tumor was <code>Ventricles</code>, it was subtyped as <code>CNC</code>; otherwise, it was subtyped as <code>EVN</code>.</p>
<p>Craniopharyngiomas (CRANIO) were subtyped into adamantinomatous (<code>CRANIO, ADAM</code>), papillary (<code>CRANIO, PAP</code>) or undetermined (<code>CRANIO, To be classified</code>) based on the following criteria <span class="citation" data-cites="WOSiJ6V2 xnZIq0gr">[<a href="#ref-WOSiJ6V2" role="doc-biblioref">119</a>,<a href="#ref-xnZIq0gr" role="doc-biblioref">120</a>]</span>:
1. Craniopharyngiomas from patients over 40 years old with a <em>BRAF</em> p.V600E mutation were subtyped as <code>CRANIO, PAP</code>.
2. Craniopharyngiomas from patients younger than 40 years old with mutations in exon 3 of <em>CTNNB1</em> were subtyped as <code>CRANIO, ADAM</code>.
3. Craniopharyngiomas that do not fall into the above two categories were subtyped as <code>CRANIO, To be classified</code>.</p>
<p>A molecular subtype of <code>EWS</code> was assigned to any tumor with a <em>EWSR1</em> fusion or with a <code>pathology_diagnosis</code> of <code>Ewings Sarcoma</code>.</p>
<p>Low-grade astrocytic tumors (LGAT) or glialneuronal tumors (GNT) were subtyped based on SNV, fusion and CNV status based on <span class="citation" data-cites="ULPByXgG"><a href="#ref-ULPByXgG" role="doc-biblioref">11</a></span>, and as described below.
1. If a sample contained a <em>NF1</em> somatic mutation, either nonsense or missense, it was subtyped as <code>LGG, NF1-somatic</code>.
2. If a sample contained <em>NF1</em> germline mutation, as indicated by a patient having the neurofibromatosis cancer predisposition, it was subtyped as <code>LGG, NF1-germline</code>.
3. If a sample contained the <em>IDH</em> p.R132 mutation, it was subtyped as <code>LGG, IDH</code>.
4. If a sample contained a histone p.K28M mutation in either <em>H3F3A</em>, <em>H3F3B</em>, <em>HIST1H3B</em>, <em>HIST1H3C</em>, or <em>HIST2H3C</em>, or if it contained a p.G35R or p.G35V mutation in <em>H3F3A</em>, it was subtyped as <code>LGG, H3</code>.
5. If a sample contained <em>BRAF</em> p.V600E or any other non-canonical <em>BRAF</em> mutations in the kinase (PK_Tyr_Ser-Thr) domain <span class="citation" data-cites="1WxziFyg">[<a href="#ref-1WxziFyg" role="doc-biblioref">121</a>]</span>, it was subtyped as <code>LGG, BRAF V600E</code>.
6. If a sample contained <code>KIAA1549--BRAF</code> fusion, it was subtyped as <code>LGG, KIAA1549-BRAF</code>.
7. If a sample contained SNV or indel in either <em>KRAS</em>, <em>NRAS</em>, <em>HRAS</em>, <em>MAP2K1</em>, <em>MAP2K2</em>, <em>MAP2K1</em>, <em>ARAF</em>, <em>RAF1</em>, or non-kinase domain of <em>BRAF</em>, or if it contained <em>RAF1</em> fusion, or <em>BRAF</em> fusion that was not <code>KIAA1549--BRAF</code>, it was subtyped as <code>LGG, other MAPK</code>.
8. If a sample contained SNV in either <em>MET</em>, <em>KIT</em> or <em>PDGFRA</em>, or if it contained fusion in <em>ALK</em>, <em>ROS1</em>, <em>NTRK1</em>, <em>NTRK2</em>, <em>NTRK3</em> or <em>PDGFRA</em>, it was subtyped as <code>LGG, RTK</code>.
9. If a sample contained <em>FGFR1</em> p.N546K, p.K656E, p.N577, or p. K687 hotspot mutations, or tyrosine kinase domain tandem duplication <span class="citation" data-cites="p1ZTCLep">[<a href="#ref-p1ZTCLep" role="doc-biblioref">122</a>]</span>, or <em>FGFR1</em> or <em>FGFR2</em> fusions, it was subtyped as <code>LGG, FGFR</code>.
10. If a sample contained <em>MYB</em> or <em>MYBL1</em> fusion, it was subtyped as <code>LGG, MYB/MYBL1</code>.
11. If a sample contained focal CDKN2A and/or CDKN2B deletion, it was subtyped as <code>LGG, CDKN2A/B</code>.</p>
<p>For LGAT tumors that did not have any of the above molecular alterations, if both RNA and DNA samples were available, it was subtyped as <code>LGG, wildtype</code>.
Otherwise, if either RNA or DNA sample was unavailable, it was subtyped as <code>LGG, To be classified</code>.</p>
<p>If pathology diagnosis was <code>Subependymal Giant Cell Astrocytoma (SEGA)</code>, the <code>LGG</code> portion of molecular subtype was recoded to <code>SEGA</code>.</p>
<p>Lastly, for all subtyped samples, if the tumors were glialneuronal in origin, based on <code>pathology_free_text_diagnosis</code> entries of <code>desmoplastic infantile</code>,<code>desmoplastic infantile ganglioglioma</code>, <code>desmoplastic infantile astrocytoma</code> or <code>glioneuronal</code>, each was recoded as follows:
If pathology diagnosis is <code>Low-grade glioma/astrocytoma (WHO grade I/II)</code> or <code>Ganglioglioma</code>, the <code>LGG</code> portion of the molecular subtype was recoded to <code>GNT</code>.</p>
<p>Ependymoma (EPN) were subtyped into <code>EPN, ST RELA</code>, <code>EPN, ST YAP1</code>, <code>EPN, PF A</code> and <code>EPN, PF B</code> based on evidence for these molecular subgroups as described in Pajtler et al. <span class="citation" data-cites="f7P6U50v">[<a href="#ref-f7P6U50v" role="doc-biblioref">100</a>]</span>.
Briefly, fusion, CNV and gene expression data were used to subtype EPN as followed:
1. Any tumor with fusions containing <code>RELA</code> as fusion partner, e.g., <code>C11orf95--RELA</code>, <code>LTBP3--RELA</code>, was subtyped as <code>EPN, ST RELA</code>.
2. Any tumor with fusions containing <code>YAP1</code> as fusion partner, such as <code>C11orf95--YAP1</code>, <code>YAP1--MAMLD1</code> and <code>YAP1--FAM118B</code>, was subtyped as <code>EPN, ST YAP1</code>.
3. Any tumor with the following molecular characterization would be subtyped as <code>EPN, PF A</code>:
- <em>CXorf67</em> expression z-score of over 3
- <em>TKTL1</em> expression z-score of over 3 and 1q gain
4. Any tumor with the following molecular characterization would be subtyped as <code>EPN, PF B</code>:
- <em>GPBP17</em> expression z-score of over 3 and loss of 6q or 6p
- <em>IFT46</em> expression z-score of over 3 and loss of 6q or 6p</p>
<p>Any tumor with the above molecular characteristics would be exclusively subtyped to the designated group.</p>
<p>For all other remaining EPN tumors without above molecular characteristics, they would be subtyped to <code>EPN, ST RELA</code> and <code>EPN, ST YAP1</code> in a non-exclusive way (e.g., a tumor could have both <code>EPN, ST RELA</code> and <code>EPN, ST YAP1</code> subtypes) if any of the following alterations were present.
1. Any tumor with the following alterations was assigned <code>EPN, ST RELA</code>:
- <code>PTEN--TAS2R1</code> fusion
- chromosome 9 arm (9p or 9q) loss
- <em>RELA</em> expression z-score of over 3
- <em>L1CAM</em> expression z-score of over 3
2. Any tumor with the following alterations was assigned <code>EPN, ST YAP1</code>:
- <code>C11orf95--MAML2</code> fusion
- chromosome 11 short arm (11p) loss<br />
- chromosome 11 long arm (11q) gain
- <em>ARL4D</em> expression z-score of over 3
- <em>CLDN1</em> expression z-score of over 3</p>
<p>After all relevant tumor samples were subtyped by the above molecular subtyping modules, the results from these modules, along with other clinical information (such as pathology diagnosis free text), were compiled through <code>molecular-subtyping-pathology</code> module.
The compilation was executed by the following steps:</p>
<p>Firstly, <code>broad_histology</code>, <code>short_histology</code>, and <code>integrated_diagnosis</code> columns in the result files from the above subtyping modules (i.e., <code>CRANIO_molecular_subtype.tsv</code>, <code>EWS_results.tsv</code>, <code>EPN_all_data_withsubgroup.tsv</code>, <code>HGG_molecular_subtype.tsv</code> ,<code>lgat_subtyping.tsv</code>, <code>MB_molecular_subtype.tsv</code>, <code>embryonal_tumor_molecular_subtypes.tsv</code>, and <code>neurocytoma_subtyping.tsv</code>) were updated based on the molecular subtype of the tumor.
Detailed information about the updating procedure were included in the analysis <a href="https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/molecular-subtyping-pathology/01-compile-subtyping-results.Rmd">notebook</a>.
Notes were also added to indicated that the changes in <code>broad_histology</code>, <code>short_histology</code> and <code>integrated_diagnosis</code> were from OpenPBTA subtyping modules.</p>
<p>Subsequently, <code>broad_histology</code>, <code>short_histology</code> and <code>harmonized_diagnosis</code> columns of tumors with particular pathology diagnosis free text were updated as specified in the following table:</p>
<table style="width:100%;">
<colgroup>
<col style="width: 16%" />
<col style="width: 16%" />
<col style="width: 16%" />
<col style="width: 16%" />
<col style="width: 16%" />
<col style="width: 16%" />
</colgroup>
<thead>
<tr class="header">
<th>pathology_diagnosis</th>
<th>subtyping module</th>
<th>pathology_free_text_diagnosis</th>
<th>broad_histology</th>
<th>short_histology</th>
<th>harmonized_diagnosis</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Primary CNS lymphoma</td>
<td>NA</td>
<td>contains <code>burkitt's lymphoma</code></td>
<td>Lymphoma</td>
<td>CNS lymphoma</td>
<td>CNS Burkitt’s lymphoma</td>
</tr>
<tr class="even">
<td>Other</td>
<td>NA</td>
<td>contains <code>xanthogranuloma</code> or <code>jxg</code></td>
<td>Histiocytic tumor</td>
<td>JXG</td>
<td>Juvenile xanthogranuloma</td>
</tr>
<tr class="odd">
<td>Meningioma</td>
<td>NA</td>
<td>contains <code>atypical</code></td>
<td>Meningioma</td>
<td>Meningioma</td>
<td>Atypical meningioma</td>
</tr>
<tr class="even">
<td>Meningioma</td>
<td>NA</td>
<td>contains <code>anaplastic</code></td>
<td>Meningioma</td>
<td>Meningioma</td>
<td>Anaplastic (malignant) meningioma</td>
</tr>
<tr class="odd">
<td>Meningioma</td>
<td>NA</td>
<td>contains <code>clear cell meningioma</code></td>
<td>Meningioma</td>
<td>Meningioma</td>
<td>Clear cell meningioma</td>
</tr>
<tr class="even">
<td>Meningioma</td>
<td>NA</td>
<td>contains <code>meningothelial</code></td>
<td>Meningioma</td>
<td>Meningioma</td>
<td>Meningothelial meningioma</td>
</tr>
<tr class="odd">
<td>Meningioma</td>
<td>NA</td>
<td>does not contain <code>atypical</code>, <code>anaplastic</code>, <code>clear cell</code>, or <code>meningothelial</code></td>
<td>Meningioma</td>
<td>Meningioma</td>
<td>Meningioma</td>
</tr>
<tr class="even">
<td>Choroid plexus papilloma</td>
<td>NA</td>
<td>contains <code>atypical</code></td>
<td>Choroid plexus tumor</td>
<td>Choroid plexus tumor</td>
<td>Atypical choroid plexus papilloma</td>
</tr>
<tr class="odd">
<td>Craniopharyngioma</td>
<td>CRANIO</td>
<td>contains <code>adamantinomatous</code></td>
<td>Tumors of sellar region</td>
<td>Craniopharyngioma</td>
<td>Adamantinomatous craniopharyngioma</td>
</tr>
</tbody>
</table>
<p>Similarly, <code>broad_histology</code>, <code>short_histology</code>, <code>integrated_diagnosis</code> and <code>harmonized_diagnosis</code> columns of tumors with following pathology diagnosis free text were updated as specified in the table below:</p>
<table style="width:100%;">
<colgroup>
<col style="width: 16%" />
<col style="width: 16%" />
<col style="width: 16%" />
<col style="width: 16%" />
<col style="width: 16%" />
<col style="width: 16%" />
</colgroup>
<thead>
<tr class="header">
<th>pathology_diagnosis</th>
<th>subtyping module</th>
<th>pathology_free_text_diagnosis</th>
<th>broad_histology</th>
<th>short_histology</th>
<th>integrated_diagnosis</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Low-grade glioma/astrocytoma (WHO grade I/II)</td>
<td>LGAT</td>
<td>contains <code>sega or subependymal giant cell astrocytoma</code></td>
<td>Low grade astrocytic tumor</td>
<td>LGAT</td>
<td>Subependymal Giant Cell Astrocytoma, <subtype></td>
</tr>
<tr class="even">
<td>Low-grade glioma/astrocytoma (WHO grade I/II)</td>
<td>LGAT</td>
<td>contains <code>fibrillary</code></td>
<td>Low grade astrocytic tumor</td>
<td>LGAT</td>
<td>Diffuse fibrillary astrocytoma, <subtype></td>
</tr>
<tr class="odd">
<td>Low-grade glioma/astrocytoma (WHO grade I/II)</td>
<td>LGAT</td>
<td>contains <code>gliomatosis cerebri, type 1, ia</code></td>
<td>Low grade astrocytic tumor</td>
<td>LGAT</td>
<td>Gliomatosis cerebri, <subtype></td>
</tr>
<tr class="even">
<td>Low-grade glioma/astrocytoma (WHO grade I/II)</td>
<td>LGAT</td>
<td>contains <code>jpa</code> or <code>juvenile astrocytoma</code> or <code>pilocytic</code> or <code>pilocystic</code> (typo) or <code>pilomyxoid</code> but does not contain <code>fibrillary</code></td>
<td>Low grade astrocytic tumor</td>
<td>LGAT</td>
<td>Pilocytic astrocytoma, <subtype></td>
</tr>
<tr class="odd">
<td>Low-grade glioma/astrocytoma (WHO grade I/II)</td>
<td>LGAT</td>
<td>contains <code>oligodendroglioma who ii</code></td>
<td>Diffuse astrocytic and oligodendroglial tumor</td>
<td>Oligodendroglioma</td>
<td>Oligodendroglioma, <subtype></td>
</tr>
<tr class="even">
<td>Low-grade glioma/astrocytoma (WHO grade I/II)</td>
<td>LGAT</td>
<td>contains <code>pxa</code> or <code>pleomorphic xanthoastrocytoma</code></td>
<td>Low grade astrocytic tumor</td>
<td>LGAT</td>
<td>Pleomorphic xanthoastrocytoma, <subtype></td>
</tr>
</tbody>
</table>
<p>Additionally, <code>broad_histology</code>, <code>short_histology</code>, <code>integrated_diagnosis</code> and <code>harmonized_diagnosis</code> columns of tumors with following pathology diagnosis free text were updated as specified in the table below:</p>
<table style="width:100%;">
<colgroup>
<col style="width: 14%" />
<col style="width: 14%" />
<col style="width: 14%" />
<col style="width: 14%" />
<col style="width: 14%" />
<col style="width: 14%" />
<col style="width: 14%" />
</colgroup>
<thead>
<tr class="header">
<th>pathology_diagnosis</th>
<th>subtyping module</th>
<th>pathology_free_text_diagnosis</th>
<th>broad_histology</th>
<th>short_histology</th>
<th>integrated_diagnosis</th>
<th>harmonized_diagnosis</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Low-grade glioma/astrocytoma (WHO grade I/II)</td>
<td>NA, remove from LGAT module</td>
<td>contains <code>desmoplastic infantile astrocytoma</code></td>
<td>Neuronal and mixed neuronal-glial tumor</td>
<td>GNT</td>
<td>Desmoplastic infantile astrocytoma and ganglioglioma, <subtype></td>
<td>Desmoplastic infantile astrocytoma and ganglioglioma</td>
</tr>
<tr class="even">
<td>Low-grade glioma/astrocytoma (WHO grade I/II)</td>
<td>NA, remove from LGAT module</td>
<td>contains <code>diffuse leptomeningeal glioneuronal tumor</code></td>
<td>Neuronal and mixed neuronal-glial tumor</td>
<td>GNT</td>
<td>Diffuse leptomeningeal glioneuronal tumor, <subtype></td>
<td>Diffuse leptomeningeal glioneuronal tumor</td>
</tr>
<tr class="odd">
<td>Low-grade glioma/astrocytoma (WHO grade I/II)</td>
<td>NA, remove from LGAT module</td>
<td>contains <code>glioneuronal</code></td>
<td>Neuronal and mixed neuronal-glial tumor</td>
<td>GNT</td>
<td>Glial-neuronal tumor NOS, <subtype></td>
<td>Glial-neuronal tumor NOS</td>
</tr>
<tr class="even">
<td>Low-grade glioma/astrocytoma (WHO grade I/II)</td>
<td>NA, remove from LGAT module</td>
<td>contains <code>rosette forming glioneuronal tumor</code></td>
<td>Neuronal and mixed neuronal-glial tumor</td>
<td>GNT</td>
<td>Rosette-forming glioneuronal tumor, <subtype></td>
<td>Rosette-forming glioneuronal tumor</td>
</tr>
</tbody>
</table>
<p>Notes were also added to indicated that the changes in <code>broad_histology</code>, <code>short_histology</code>, <code>integrated_diagnosis</code> and <code>harmonized_diagnosis</code> were from pathology diagnosis free text.</p>
<p>For samples with subtype discrepancies, <code>molecular_subtype</code> and <code>integrated_diagnosis</code> were updated following pathology or clinical review.
Detailed information can be found in the analysis notebooks for <a href="https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/molecular-subtyping-pathology/03-incorporate-clinical-feedback.Rmd">clinical</a> and <a href="https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/molecular-subtyping-pathology/04-incorporate-pathology-feedback.Rmd">pathology</a> feedback.
Finally, the newly compiled subtypes were integrated into the <code>pbta-histologies.tsv</code> file in the <code>molecular-subtyping-integrate</code> module.</p>
<h4 id="tp53-alteration-annotation">TP53 Alteration Annotation</h4>
<p>In addition to tumor types mentioned above, TP53 altered status is also annotated for all samples and if a sample is determined to be either <code>TP53 loss</code> or <code>TP53 activated</code>, this annotation will be included in the <code>molecular_subtype</code> column.
We applied a <em>TP53</em> inactivation classifier originally trained on TCGA PanCan data <span class="citation" data-cites="X6oQhIf8">[<a href="#ref-X6oQhIf8" role="doc-biblioref">25</a>]</span> to the matched RNA expression data for each sample.
Along with the <em>TP53</em> classifier scores, consensus SNV and CNV, SV, and references databases that list TP53 hotspot mutations <span class="citation" data-cites="qEpbKZ3t 10jkUywit tfWVBF8o">[<a href="#ref-qEpbKZ3t" role="doc-biblioref">52</a>,<a href="#ref-10jkUywit" role="doc-biblioref">123</a>,<a href="#ref-tfWVBF8o" role="doc-biblioref">124</a>]</span> and functional domains <span class="citation" data-cites="KxZPoysm">[<a href="#ref-KxZPoysm" role="doc-biblioref">125</a>]</span> were used collectively to determine TP53 alteration status for each sample.
The rules for calling either <code>TP53 loss</code> or <code>TP53 activated</code> are as follows:
If a sample has any of the two well-characterized <em>TP53</em> gain-of-function mutations, p.R273C or p.R248W <span class="citation" data-cites="14asizOhY">[<a href="#ref-14asizOhY" role="doc-biblioref">27</a>]</span>, <code>TP53 activated</code> status will be assigned.
A sample will be annotated as <code>TP53 loss</code> if any of the following conditions is met:
1) It contains a <em>TP53</em> hotspot mutation as defined by IARC TP53 database <span class="citation" data-cites="qEpbKZ3t 10jkUywit tfWVBF8o">[<a href="#ref-qEpbKZ3t" role="doc-biblioref">52</a>,<a href="#ref-10jkUywit" role="doc-biblioref">123</a>,<a href="#ref-tfWVBF8o" role="doc-biblioref">124</a>]</span>
2) It contains two <em>TP53</em> alterations, including SNV, CNV or SV, which is indicative of probable bi-allelic alterations
3) It contains one <em>TP53</em> somatic alteration, including SNV, CNV, or SV and a germline <em>TP53</em> mutation indicated by the diagnosis of Li-Fraumeni syndrome <span class="citation" data-cites="1CnI07C7L">[<a href="#ref-1CnI07C7L" role="doc-biblioref">126</a>]</span>
4) It contains one germline <em>TP53</em> mutation indicated by Li-Fraumeni syndrome and the <em>TP53</em> classifier score for matched RNA-Seq is over 0.5.</p>
<h4 id="survival-analysis">Survival analysis</h4>
<p>Overall survival, denoted <code>OS_days</code>, was calculated as days since initial diagnosis.</p>
<h4 id="prediction-of-participants-genetic-sex">Prediction of participants’ genetic sex</h4>
<p>The clinical metadata provided included a reported gender.
We used DNA data, in concert with the reported gender, to predict participant genetic sex so that we could identify sexually dimorphic outcomes.
This analysis could also reveal samples that may have been contaminated in certain circumstances.
We used the idxstats utility from SAMtools <span class="citation" data-cites="pVqo7JQF">[<a href="#ref-pVqo7JQF" role="doc-biblioref">127</a>]</span> to calculate read lengths, the number of mapped reads, and the corresponding chromosomal location for reads to the X and Y chromosomes.
We used the fraction of total normalized X and Y chromosome reads that were attributed to the Y chromosome as a summary statistic.
We reviewed this statistic in the context of reported gender and determined that a threshold of less than 0.2 clearly delineated female samples.
Fractions greater than 0.4 were predicted to be males.
Samples with values in the range [0.2, 0.4] were marked as unknown.
We ran this analysis through <a href="https://github.com/d3b-center/sex-determination-tool">CWL</a> on CAVATICA.
Resulting calls were added to the clinical metadata as <code>germline_sex_estimate</code>.</p>
<h4 id="selection-of-independent-samples">Selection of independent samples</h4>
<p>Certain analyses required that we select only a single representative specimen for each individual.
In these cases, we prioritized primary tumors and those with whole-genome sequencing available.
If this filtering still resulted in multiple specimens, we selected from the remaining set randomly.</p>
<h4 id="quantification-of-telomerase-activity-using-gene-expression-data">Quantification of Telomerase Activity using Gene Expression Data</h4>
<p>We predicted telomerase activity of pediatric brain tumor samples using our recently developed method EXTEND.
In brief, EXTEND estimates telomerase activity based on the expression of a 13-gene signature.
This signature was derived by comparing telomerase positive tumors and tumors with activated alternative lengthening of telomeres pathway, a group presumably negative of telomerase activity.
More details about the algorithm can be found in reference <span class="citation" data-cites="16bqdaYmh">[<a href="#ref-16bqdaYmh" role="doc-biblioref">128</a>]</span>.
We calculated telomerase activity score for each sample of the PBTA cohort and examined the score distribution across both broad and specific disease histological subtypes.
We also compared EXTEND scores across the four molecular subgroups of medulloblastoma (Group3, Group4, SHH and WNT) using a series of pairwise two-sample t-tests with a Bonferroni correction for multiple testing.
EXTEND scores have been further compared, using Spearman rank correlation, between counts and FPKM gene expression values from poly-A and stranded protocols.</p>
<h3 id="additional-resources">ADDITIONAL RESOURCES</h3>
<!-- TODO: Add section -->
<h3 id="key-resources-table">KEY RESOURCES TABLE</h3>
<table>
<colgroup>
<col style="width: 12%" />
<col style="width: 35%" />
<col style="width: 52%" />
</colgroup>
<thead>
<tr class="header">
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Critical commercial assays</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>Recover Cell Culture Freezing media</td>
<td>Gibco</td>
<td>12648010</td>
</tr>
<tr class="odd">
<td>Hank’s Balanced Salt Solution (HBSS)</td>
<td>Gibco</td>
<td>14175095</td>
</tr>
<tr class="even">
<td>Papain</td>
<td>SciQuest</td>
<td>LS003124</td>
</tr>
<tr class="odd">
<td>Ovomucoid</td>
<td>SciQuest</td>
<td>542000</td>
</tr>
<tr class="even">
<td>DNase</td>
<td>Roche</td>
<td>10104159001</td>
</tr>
<tr class="odd">
<td>100μm cell strainer</td>
<td>Greiner Bio-One</td>
<td>542000</td>
</tr>
<tr class="even">
<td>DMEM/F-12 medium</td>
<td>Sigma</td>
<td>D8062</td>
</tr>
<tr class="odd">
<td>Fetal Bovine Serum (FBS)</td>
<td>Hyclone</td>
<td>SH30910.03</td>
</tr>
<tr class="even">
<td>GlutaMAX</td>
<td>Gibco</td>
<td>35050061</td>
</tr>
<tr class="odd">
<td>Penicillin/Streptomycin-Amphotericin B</td>
<td>Lonza</td>
<td>17-745E</td>
</tr>
<tr class="even">
<td>Normocin</td>
<td>Invivogen</td>
<td>ant-nr-2</td>
</tr>
<tr class="odd">
<td>B-27 supplement minus vitamin A</td>
<td>Gibco</td>
<td>12587-010</td>
</tr>
<tr class="even">
<td>N-2 supplement</td>
<td>Gibco</td>
<td>17502001</td>
</tr>
<tr class="odd">
<td>Epidermal growth factor</td>
<td>Gibco</td>
<td>PHG0311L</td>
</tr>
<tr class="even">
<td>Basic fibroblast growth factor</td>
<td>PeproTech</td>
<td>100-18B</td>
</tr>
<tr class="odd">
<td>Heparin</td>
<td>Sigma</td>
<td>H3149</td>
</tr>
<tr class="even">
<td>DNA/RNA AllPrep Kit</td>
<td>Qiagen</td>
<td>80204</td>
</tr>
<tr class="odd">
<td>TruSeq RNA Sample Prep Kit</td>
<td>Illumina</td>
<td>FC-122-1001</td>
</tr>
<tr class="even">
<td>KAPA Library Preparation Kit</td>
<td>Roche</td>
<td>KK8201</td>
</tr>
<tr class="odd">
<td>AllPrep DNA/RNA/miRNA Universal kit</td>
<td>Qiagen</td>
<td>80224</td>
</tr>
<tr class="even">
<td>RNase A</td>
<td>Qiagen</td>
<td>19101</td>
</tr>
<tr class="odd">
<td>QIAsymphony DSP DNA Midi Kit</td>
<td>Qiagen</td>
<td>937255</td>
</tr>
<tr class="even">
<td>KAPA HyperPrep kit</td>
<td>Roche</td>
<td>08098107702</td>
</tr>
<tr class="odd">
<td>RiboErase kit</td>
<td>Roche</td>
<td>07962304001</td>
</tr>
<tr class="even">
<td>Deposited data</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>Raw and harmonized WGS, WXS, Panel, RNA-Seq</td>
<td>KidsFirst Data Resource Center, this project</td>
<td><span class="citation" data-cites="THjIzNXb">[<a href="#ref-THjIzNXb" role="doc-biblioref">29</a>]</span></td>
</tr>
<tr class="even">
<td>Merged summary files</td>
<td>this project</td>
<td>https://cavatica.sbgenomics.com/u/cavatica/openpbta</td>
</tr>
<tr class="odd">
<td>Merged summary files and downstream analyses</td>
<td>this project</td>
<td>https://github.com/AlexsLemonade/OpenPBTA-analysis/</td>
</tr>
<tr class="even">
<td>Processed data</td>
<td>this project</td>
<td>https://pedcbioportal.kidsfirstdrc.org/study/summary?id=openpbta</td>
</tr>
<tr class="odd">
<td>Experimental models: Cell lines</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>CBTN pediatric brain tumor-derived cell lines</td>
<td><span class="citation" data-cites="DwCzMoBJ">[<a href="#ref-DwCzMoBJ" role="doc-biblioref">3</a>]</span></td>
<td>See <strong>Table S1</strong> for identifiers</td>
</tr>
<tr class="odd">
<td>Software and algorithms</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>OpenPBTA workflows repository</td>
<td>this project</td>
<td><span class="citation" data-cites="DbMaLwlv">[<a href="#ref-DbMaLwlv" role="doc-biblioref">129</a>]</span></td>
</tr>
<tr class="odd">
<td>OpenPBTA analysis repository</td>
<td>this project</td>
<td></td>
</tr>
<tr class="even">
<td>OpenPBTA manuscript repository</td>
<td>this project</td>
<td></td>
</tr>
<tr class="odd">
<td></td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>Other</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>TCGA WXS dataset</td>
<td>National Institutes of Health The Cancer Genome Atlas (TCGA)</td>
<td></td>
</tr>
<tr class="even">
<td>dbGAP phs000178.v11.p8</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td></td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td></td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td></td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td></td>
<td></td>
<td></td>
</tr>
</tbody>
</table>
<h2 id="supplemental-information-titles-and-legends">Supplemental Information Titles and Legends</h2>
<!-- TODO: Add complete legends once figures are finalized. -->
<p><strong>Figure S1. OpenPBTA Project Workflow, Related to Figure 1.</strong>
Biospecimens and data were collected by CBTN and PNOC.
Genomic sequencing and harmonization (orange boxes) were performed by the Kids First Data Resource Center (KFDRC).
Analyses in the green boxes were performed by contributors of the OpenPBTA project.
Output files are denoted in blue.
Figure created with biorender.com.</p>
<p><strong>Figure S2. Validation of Consensus SNV calls and Tumor Mutation Burden, Related to Figures 2 and 3.</strong><br />
Correlation (A) and violin (B) plots of mutation variant allele frequencies (VAFs) comparing the variant callers (Lancet, Strelka2, Mutect2, and VarDict) used for PBTA samples.
Upset plot (C) showing overlap of variant calls. Correlation (D) and violin (E) plots of mutation variant allele frequencies (VAFs) comparing the variant callers (Lancet, Strelka2, and Mutect2) used for TCGA samples.
Upset plot (F) showing overlap of variant calls.
Cumulative distribution TMB plots for PBTA (G) and TCGA (H)tumors using consensus SNV calls.</p>
<p><strong>Figure S3. Genomic instability of pediatric brain tumors, Related to Figures 2 and 3.</strong>
(A) Oncoprint of canonical somatic gene mutations, CNVs, fusions, and TMB (top bar plot) for the top 20 genes mutated across rare CNS tumors (N &lt; 5 each): desmoplastic infantile astrocytoma and ganglioglioma (N = 1), germinoma (N = 4), glial-neuronal NOS (N = 4), metastatic secondary tumors (N = 3), neurocytoma (N = 2), and pineoblastoma (N = 3).
Patient sex (<code>germline_sex_estimate</code>) and tumor histology (<code>cancer_group</code>) are displayed as annotations at the bottom of each plot.
Only samples with mutations in the listed genes are shown.
Multiple CNVs are denoted as a complex event.
(B) Genome-wide plot of CNV alterations by broad histology.
Each row represents one sample.
Box and whisker plots of number of CNV breaks (C) or SV breaks (D) by number of chromothripsis regions.</p>
<p><strong>Figure S4. Related to Figure 3.</strong>
(A) Sample-specific RefSig signature weights across cancer groups ordered by decreasing Signature 1 exposure.
(B) Proportion of Signature 1 plotted by phase of therapy for each cancer group.</p>
<p><strong>Figure S5. Related to Figure 4</strong>.
First two dimensions from UMAP of sample transcriptome data with points colored by <code>molecular_subtype</code> for medulloblastoma (A), ependymoma (B), low-grade glioma (C), and high-grade diffuse astrocytic tumors (D).
(E) Receiver Operating Characteristic for <em>TP53</em> classifier run on FPKM of poly-A RNA-Seq samples.
(F) Violin and box plots of <em>TP53</em> scores plotted by <em>TP53</em> alteration type in poly-A RNA-Seq samples.
Correlation plots for telomerase scores (EXTEND) with RNA expression of <em>TERT</em> (G) and <em>TERC</em> (H).</p>
<p><strong>Table S1. Related to Figure 1.</strong>
Table of specimens and associated metadata, clinical data, and histological data utilized in the OpenPBTA project.</p>
<p><strong>Table S2. Related to Figures 2 and 3.</strong>
Table of TMB, eight CNS mutational signatures, and chromothripsis events per sample.</p>
<p><strong>Table S3. Related to Figures 4 and 5.</strong> Table of <em>TP53</em> scores, telomerase EXTEND scores, and quanTIseq immune scores.</p>
<h2 id="consortia">Consortia</h2>
<p>The past and present members of the Children’s Brain Tumor Network who contributed to the generation of specimens and data are Adam C. Resnick, Alexa Plisiewicz, Allison M. Morgan, Allison P. Heath, Alyssa Paul, Amanda Saratsis, Amy Smith, Ana Aguilar, Ana Guerreiro Stücklin, Anastasia Arynchyna, Andrea Franson, Angela J. Waanders, Angela N. Viaene, Anita Nirenberg, Anna Maria Buccoliero, Anna Yaffe, Anny Shai, Anthony Bet, Antoinette Price, Ashley Plant, Augustine Eze, Bailey C. Farrow, Baoli Hu, Beth Frenkel, Bo Zhang, Bobby Moulder, Bonnie Cole, Brian M. Ennis, Brian R. Rood, Brittany Lebert, Carina A. Leonard, Carl Koschmann, Caroline Caudill, Caroline Drinkwater, Cassie N. Kline, Catherine Sullivan, Chanel Keoni, Chiara Caporalini, Christine Bobick-Butcher, Christopher Mason, Chunde Li, Claire Carter, Claudia MaduroCoronado, Clayton Wiley, Cynthia Wong, David E. Kram, David Haussler, David Kram, David Pisapia, David Ziegler, Denise Morinigo, Derek Hanson, Donald W. Parsons, Elizabeth Appert, Emily Drake, Emily Golbeck, Ena Agbodza, Eric H. Raabe, Eric M. Jackson, Eric Raabe, Esteban Uceda, Eugene Hwang, Fausto Rodriquez, Gabrielle S. Stone, Gary Kohanbash, George Rafidi, Gerald Grant, Gerri Trooskin, Gilad Evrony, Graham Keyes, Hagop Boyajian, Holly B. Lindsay, Holly Beale, Ian F. Pollack, James Johnston, James Palmer, Jane Minturn, Jared Pisapia, Jason E. Cain, Javad Nazarian, Jeanette Haugh, Jeff Greenfield, Jeff Stevens, Jeffrey P. Greenfield, Jeffrey Rubens, Jena V. Lilly, Jennifer L. Mason, Jessica B. Foster, Jim Olson, Jo Lynne Rokita, Joanna J. Phillips, Jonathan Waller, Josh Rubin, Judy E. Palma, Justine Rizzo, Kaitlin Lehmann, Kamnaa Arya, Karlene Hall, Katherine Pehlivan, Kenneth Seidl, Kimberly Diamond, Kristen Harnett, Kristina Cole, Krutika S. Gaonkar, Lamiya Tauhid, Lina Lopez, Lionel Chow, Mahdi Sarmady, Margaret Nevins, Mari Groves, Mariarita Santi-Vicini, Marilyn M. Li, Marion Mateos, Mateusz Koptyra, Matija Snuderl, Matthew Miller, Matthew Sklar, Matthew Wood, Meghan Connors, Melissa Williams, Meredith Egan, Michael Fisher, Michael Koldobskiy, Michelle Monje, Migdalia Martinez, Miguel A. Brown, Mike Prados, Miriam Bornhorst, Mirko Scagnet, Mohamed AbdelBaki, Monique Carrero-Tagle, Nadia Dahmane, Nalin Gupta, Nathan Young, Nicholas A. Vitanza, Nicholas Tassone, Nicolas Gerber, Nithin D. Adappa, Nitin Wadhwani, Obi Obayashi, Olena M. Vaske, Olivier Elemento, Oren Becher, Philbert Oliveros, Phillip B. Storm, Pichai Raman, Rintaro Hashizume, Robert Keating, Robert M. Lober, Ron Firestein, Sabine Mueller, Sameer Agnihotri, Samuel G. Winebrake, Samuel Rivero-Hinojosa, Sarah Diane Black, Sarah Leary, Schuyler Stoller, Shannon Robins, Sharon Gardner, Sherri Mayans, Sherry Tutson, Shida Zhu, Sofie Salama, Sonia Partap, Sonika Dahiya, Sriram Venneti, Stacie Stapleton, Stephani Campion, Stephanie Stefankiewicz, Swetha Thambireddy, Tatiana Patton, Teresa Hidalgo, Theo Nicolaides, Thinh Q. Nguyen, Tiffany Walker, Valeria Lopez-Gil, Valerie Baubet, Xiao-Nan Li, Ximena Cuellar, Yiran Guo, Yuankun Zhu, and Zeinab Helil.</p>
<p>The past and present members of the Pacific Pediatric Neuro-Oncology Consortium who contributed to the generation of specimens and data are Adam C. Resnick, Alicia Lenzen, Alyssa Reddy, Amar Gajjar, Ana Guerreiro Stucklin, Anat Epstein, Andrea Franson, Angela Waanders, Anne Bendel, Anu Banerjee, Ashley Margol, Ashley Plant, Brian Rood, Carl Koschmann, Carol Bruggers, Caroline Hastings, Cassie N. Kline, Christina Coleman Abadi, Christopher Tinkle, Corey Raffel, Dan Runco, Daniel Landi, Daphne Adele Haas-Kogan, David Ashley, David Ziegler, Derek Hanson, Dong Anh Khuong Quang, Duane Mitchell, Elias Sayour, Eric Jackson, Eric Raabe, Eugene Hwang, Fatema Malbari, Geoffrey McCowage, Girish Dhall, Gregory Friedman, Hideho Okada, Ibrahim Qaddoumi, Iris Fried, Jae Cho, Jane Minturn, Jason Blatt, Javad Nazarian, Jeffrey Rubens, Jena Lilly, Jennifer Elster, Jennifer Mason, Jessica Schulte, Jonathan Schoenfeld, Josh Rubin, Karen Gauvain, Karen Wright, Katharine Offer, Katie Metrock, Kellie Haworth, Ken Cohen, Kristina A Cole, Lance Governale, Linda Stork, Lindsay Kilburn, Lissa Baird, Maggie Skrypek, Marcia Leonard, Margaret Shatara, Margot Lazow, Mariella Filbin, Maryam Fouladi, Matthew Miller, Megan Paul, Michael Fisher, Michael Koldobskiy, Michal Yalon Oren, Mimi Bandopadhayay, Miriam Bornhorst, Mohamed AbdelBaki, Nalin Gupta, Nathan Robison, Nicholas Whipple, Nick Gottardo, Nick Vitanza, Nicolas Gerber, Patricia Robertson, Payal Jain, Peter Sun, Prados, Michael, Priya Chan, Richard S Lemons, Robert Wechsler-Reya, Roger Packer, Russ Geyer, Sabine Mueller, Sahaja Acharya, Sam Cheshier, Sarah Leary, Scott Coven, Sharon Gardner, Sri Gururangan, Stewart Goldman, Susan Chi, Tab Cooney, Tatiana S. Patton, Theodore Nicolaides, and Tom Belle Davidson.</p>
<h2 class="page_break_before" id="references">References</h2>
<!-- Explicitly insert bibliography here -->
<div id="refs" class="references hanging-indent" role="doc-bibliography">
<div id="ref-nAMOAA9j">
<p>1. <strong>CBTRUS Statistical Report: Primary Brain and Other Central Nervous System Tumors Diagnosed in the United States in 2012–2016</strong> <br />
Quinn T Ostrom, Gino Cioffi, Haley Gittleman, Nirav Patil, Kristin Waite, Carol Kruchko, Jill S Barnholtz-Sloan<br />
<em>Neuro-Oncology</em> (2019-11-01) <a href="https://doi.org/gg4d4k">https://doi.org/gg4d4k</a> <br />
DOI: <a href="https://doi.org/10.1093/neuonc/noz150">10.1093/neuonc/noz150</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31675094">31675094</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6823730">PMC6823730</a></p>
</div>
<div id="ref-1H6hugOV3">
<p>2. <strong>CBTRUS Statistical Report: Primary Brain and Other Central Nervous System Tumors Diagnosed in the United States in 2009–2013</strong> <br />
Quinn T. Ostrom, Haley Gittleman, Jordan Xu, Courtney Kromer, Yingli Wolinsky, Carol Kruchko, Jill S. Barnholtz-Sloan<br />
<em>Neuro-Oncology</em> (2016-10) <a href="https://doi.org/ggzh6m">https://doi.org/ggzh6m</a> <br />
DOI: <a href="https://doi.org/10.1093/neuonc/now207">10.1093/neuonc/now207</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28475809">28475809</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8483569">PMC8483569</a></p>
</div>
<div id="ref-DwCzMoBJ">
<p>3. <strong>Pediatric high-grade glioma resources from the Children’s Brain Tumor Tissue Consortium</strong> <br />
Heba Ijaz, Mateusz Koptyra, Krutika S Gaonkar, Jo Lynne Rokita, Valerie P Baubet, Lamiya Tauhid, Yuankun Zhu, Miguel Brown, Gonzalo Lopez, Bo Zhang, … Kristina A Cole<br />
<em>Neuro-Oncology</em> (2020-01-11) <a href="https://doi.org/gm3hpz">https://doi.org/gm3hpz</a> <br />
DOI: <a href="https://doi.org/10.1093/neuonc/noz192">10.1093/neuonc/noz192</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31613963">31613963</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6954395">PMC6954395</a></p>
</div>
<div id="ref-VT338cRv">
<p>4. <strong>Children’s Brain Tumor Network</strong> <a href="https://cbtn.org/">https://cbtn.org/</a></p>
</div>
<div id="ref-BLNV1DbL">
<p>5. <strong>A pilot precision medicine trial for children with diffuse intrinsic pontine glioma—PNOC003: A report from the Pacific Pediatric Neuro‐Oncology Consortium</strong> <br />
Sabine Mueller, Payal Jain, Winnie S. Liang, Lindsay Kilburn, Cassie Kline, Nalin Gupta, Eshini Panditharatna, Suresh N. Magge, Bo Zhang, Yuankun Zhu, … Adam C. Resnick<br />
<em>International Journal of Cancer</em> (2019-04-03) <a href="https://doi.org/gf6pfb">https://doi.org/gf6pfb</a> <br />
DOI: <a href="https://doi.org/10.1002/ijc.32258">10.1002/ijc.32258</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30861105">30861105</a></p>
</div>
<div id="ref-petulEc2">
<p>6. <strong>Manubot - Manuscripts, open and automated</strong> <a href="https://manubot.org">https://manubot.org</a></p>
</div>
<div id="ref-YuJbg3zO">
<p>7. <strong>Open collaborative writing with Manubot</strong> <br />
Daniel S. Himmelstein, Vincent Rubinetti, David R. Slochower, Dongbo Hu, Venkat S. Malladi, Casey S. Greene, Anthony Gitter<br />
<em>PLOS Computational Biology</em> (2019-06-24) <a href="https://doi.org/c7np">https://doi.org/c7np</a> <br />
DOI: <a href="https://doi.org/10.1371/journal.pcbi.1007128">10.1371/journal.pcbi.1007128</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31233491">31233491</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6611653">PMC6611653</a></p>
</div>
<div id="ref-14TDMX5t7">
<p>8. <strong>The 2007 WHO Classification of Tumours of the Central Nervous System</strong> <br />
David N. Louis, Hiroko Ohgaki, Otmar D. Wiestler, Webster K. Cavenee, Peter C. Burger, Anne Jouvet, Bernd W. Scheithauer, Paul Kleihues<br />
<em>Acta Neuropathologica</em> (2007-08) <a href="https://doi.org/cm372n">https://doi.org/cm372n</a> <br />
DOI: <a href="https://doi.org/10.1007/s00401-007-0243-4">10.1007/s00401-007-0243-4</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17618441">17618441</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1929165">PMC1929165</a></p>
</div>
<div id="ref-7sgrMadR">
<p>9. <strong>The 2016 World Health Organization Classification of Tumors of the Central Nervous System: a summary</strong> <br />
David N. Louis, Arie Perry, Guido Reifenberger, Andreas von Deimling, Dominique Figarella-Branger, Webster K. Cavenee, Hiroko Ohgaki, Otmar D. Wiestler, Paul Kleihues, David W. Ellison<br />
<em>Acta Neuropathologica</em> (2016-06) <a href="https://doi.org/f8mspx">https://doi.org/f8mspx</a> <br />
DOI: <a href="https://doi.org/10.1007/s00401-016-1545-1">10.1007/s00401-016-1545-1</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27157931">27157931</a></p>
</div>
<div id="ref-6F5vK3sT">
<p>10. <strong>The 2021 WHO Classification of Tumors of the Central Nervous System: a summary</strong> <br />
David N Louis, Arie Perry, Pieter Wesseling, Daniel J Brat, Ian A Cree, Dominique Figarella-Branger, Cynthia Hawkins, HK Ng, Stefan M Pfister, Guido Reifenberger, … David W Ellison<br />
<em>Neuro-Oncology</em> (2021-08-02) <a href="https://doi.org/gmqhbf">https://doi.org/gmqhbf</a> <br />
DOI: <a href="https://doi.org/10.1093/neuonc/noab106">10.1093/neuonc/noab106</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/34185076">34185076</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8328013">PMC8328013</a></p>
</div>
<div id="ref-ULPByXgG">
<p>11. <strong>Integrated Molecular and Clinical Analysis of 1,000 Pediatric Low-Grade Gliomas</strong> <br />
Scott Ryall, Michal Zapotocky, Kohei Fukuoka, Liana Nobre, Ana Guerreiro Stucklin, Julie Bennett, Robert Siddaway, Christopher Li, Sanja Pajovic, Anthony Arnoldo, … Cynthia Hawkins<br />
<em>Cancer Cell</em> (2020-04) <a href="https://doi.org/ggsmx2">https://doi.org/ggsmx2</a> <br />
DOI: <a href="https://doi.org/10.1016/j.ccell.2020.03.011">10.1016/j.ccell.2020.03.011</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32289278">32289278</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7169997">PMC7169997</a></p>
</div>
<div id="ref-OgCAsGmn">
<p>12. <strong>Comprehensive Analysis of Hypermutation in Human Cancer</strong> <br />
Brittany B. Campbell, Nicholas Light, David Fabrizio, Matthew Zatzman, Fabio Fuligni, Richard de Borja, Scott Davidson, Melissa Edwards, Julia A. Elvin, Karl P. Hodel, … Adam Shlien<br />
<em>Cell</em> (2017-11) <a href="https://doi.org/gcjk7b">https://doi.org/gcjk7b</a> <br />
DOI: <a href="https://doi.org/10.1016/j.cell.2017.09.048">10.1016/j.cell.2017.09.048</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29056344">29056344</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5849393">PMC5849393</a></p>
</div>
<div id="ref-IamVuiNU">
<p>13. <strong>Pediatric low-grade glioma in the era of molecular diagnostics</strong> <br />
Scott Ryall, Uri Tabori, Cynthia Hawkins<br />
<em>Acta Neuropathologica Communications</em> (2020-12) <a href="https://doi.org/gm3kst">https://doi.org/gm3kst</a> <br />
DOI: <a href="https://doi.org/10.1186/s40478-020-00902-z">10.1186/s40478-020-00902-z</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32164789">32164789</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7066826">PMC7066826</a></p>
</div>
<div id="ref-1Geb70JK9">
<p>14. <strong>ETMR: a tumor entity in its infancy</strong> <br />
Sander Lambo, Katja von Hoff, Andrey Korshunov, Stefan M. Pfister, Marcel Kool<br />
<em>Acta Neuropathologica</em> (2020-09) <a href="https://doi.org/gm3q25">https://doi.org/gm3q25</a> <br />
DOI: <a href="https://doi.org/10.1007/s00401-020-02182-2">10.1007/s00401-020-02182-2</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32601913">32601913</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7423804">PMC7423804</a></p>
</div>
<div id="ref-1EzqPr2Ef">
<p>15. <strong>Emergence and maintenance of actionable genetic drivers at medulloblastoma relapse</strong> <br />
Stacey Richardson, Rebecca M Hill, Christopher Kui, Janet C Lindsey, Yura Grabovksa, Claire Keeling, Louise Pease, Matthew Bashton, Stephen Crosier, Maria Vinci, … Steven C Clifford<br />
<em>Neuro-Oncology</em> (2022-01-05) <a href="https://doi.org/gm3q3n">https://doi.org/gm3q3n</a> <br />
DOI: <a href="https://doi.org/10.1093/neuonc/noab178">10.1093/neuonc/noab178</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/34272868">34272868</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8730763">PMC8730763</a></p>
</div>
<div id="ref-j11VxJIm">
<p>16. <strong>Molecular identification of CNS NB-FOXR2, CNS EFT-CIC, CNS HGNET-MN1 and CNS HGNET-BCOR pediatric brain tumors using tumor-specific signature genes</strong> <br />
Maria Łastowska, Joanna Trubicka, Anna Sobocińska, Bartosz Wojtas, Magdalena Niemira, Anna Szałkowska, Adam Krętowski, Agnieszka Karkucińska-Więckowska, Magdalena Kaleta, Maria Ejmont, … Ewa Matyja<br />
<em>Acta Neuropathologica Communications</em> (2020-12) <a href="https://doi.org/gm3q3p">https://doi.org/gm3q3p</a> <br />
DOI: <a href="https://doi.org/10.1186/s40478-020-00984-9">10.1186/s40478-020-00984-9</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32650833">32650833</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7350623">PMC7350623</a></p>
</div>
<div id="ref-UuzX8B9C">
<p>17. <strong>The whole-genome landscape of medulloblastoma subtypes</strong> <br />
Paul A. Northcott, Ivo Buchhalter, A. Sorana Morrissy, Volker Hovestadt, Joachim Weischenfeldt, Tobias Ehrenberger, Susanne Gröbner, Maia Segura-Wang, Thomas Zichner, Vasilisa A. Rudneva, … Peter Lichter<br />
<em>Nature</em> (2017-07) <a href="https://doi.org/gbnzc9">https://doi.org/gbnzc9</a> <br />
DOI: <a href="https://doi.org/10.1038/nature22973">10.1038/nature22973</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28726821">28726821</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5905700">PMC5905700</a></p>
</div>
<div id="ref-QNvTzXiX">
<p>18. <strong>WHO classification of tumors of the nervous system: preview of the upcoming 5th edition</strong> <br />
Elisabeth J. Rushing<br />
<em>memo - Magazine of European Medical Oncology</em> (2021-06) <a href="https://doi.org/gk2z97">https://doi.org/gk2z97</a> <br />
DOI: <a href="https://doi.org/10.1007/s12254-021-00680-x">10.1007/s12254-021-00680-x</a></p>
</div>
<div id="ref-vd5Uh9gY">
<p>19. <strong>Integrated Molecular Meta-Analysis of 1,000 Pediatric High-Grade and Diffuse Intrinsic Pontine Glioma</strong> <br />
Alan Mackay, Anna Burford, Diana Carvalho, Elisa Izquierdo, Janat Fazal-Salom, Kathryn R. Taylor, Lynn Bjerke, Matthew Clarke, Mara Vinci, Meera Nandhabalan, … Chris Jones<br />
<em>Cancer Cell</em> (2017-10) <a href="https://doi.org/gb4kpn">https://doi.org/gb4kpn</a> <br />
DOI: <a href="https://doi.org/10.1016/j.ccell.2017.08.017">10.1016/j.ccell.2017.08.017</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28966033">28966033</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5637314">PMC5637314</a></p>
</div>
<div id="ref-KU8wVb8T">
<p>20. <strong>Diffuse intrinsic pontine glioma-like tumor with EZHIP expression and molecular features of PFA ependymoma</strong> <br />
Drew Pratt, Martha Quezado, Zied Abdullaev, Debra Hawes, Fusheng Yang, Hugh J. L. Garton, Alexander R. Judkins, Rajen Mody, Arul Chinnaiyan, Kenneth Aldape, … Sriram Venneti<br />
<em>Acta Neuropathologica Communications</em> (2020-12) <a href="https://doi.org/gm3ksv">https://doi.org/gm3ksv</a> <br />
DOI: <a href="https://doi.org/10.1186/s40478-020-00905-w">10.1186/s40478-020-00905-w</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32197665">32197665</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7083001">PMC7083001</a></p>
</div>
<div id="ref-eDtbAO3v">
<p>21. <strong>Genomic Analysis of Dysembryoplastic Neuroepithelial Tumor Spectrum Reveals a Diversity of Molecular Alterations Dysregulating the MAPK and PI3K/mTOR Pathways</strong> <br />
Lea F Surrey, Payal Jain, Bo Zhang, Joshua Straka, Xiaonan Zhao, Brian N Harding, Adam C Resnick, Phillip B Storm, Anna Maria Buccoliero, Lorenzo Genitori, … Mariarita Santi<br />
<em>Journal of Neuropathology &amp; Experimental Neurology</em> (2019-12-01) <a href="https://doi.org/gpnm3p">https://doi.org/gpnm3p</a> <br />
DOI: <a href="https://doi.org/10.1093/jnen/nlz101">10.1093/jnen/nlz101</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31617914">31617914</a></p>
</div>
<div id="ref-ZINVcBaH">
<p>22. <strong><em>TP53</em> Mutation Is Frequently Associated With <em>CTNNB1</em> Mutation or <em>MYCN</em> Amplification and Is Compatible With Long-Term Survival in Medulloblastoma</strong> <br />
Elke Pfaff, Marc Remke, Dominik Sturm, Axel Benner, Hendrik Witt, Till Milde, André O. von Bueren, Andrea Wittmann, Anna Schöttler, Norbert Jorch, … Stefan M. Pfister<br />
<em>Journal of Clinical Oncology</em> (2010-12-10) <a href="https://doi.org/dpd6hh">https://doi.org/dpd6hh</a> <br />
DOI: <a href="https://doi.org/10.1200/jco.2010.31.1670">10.1200/jco.2010.31.1670</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21060032">21060032</a></p>
</div>
<div id="ref-zrYR4Swt">
<p>23. <strong>The genomic landscape of diffuse intrinsic pontine glioma and pediatric non-brainstem high-grade glioma</strong> <br />
the St. Jude Children’s Research Hospital–Washington University Pediatric Cancer Genome Project<br />
<em>Nature Genetics</em> (2014-05) <a href="https://doi.org/f526f8">https://doi.org/f526f8</a> <br />
DOI: <a href="https://doi.org/10.1038/ng.2938">10.1038/ng.2938</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24705251">24705251</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4056452">PMC4056452</a></p>
</div>
<div id="ref-Cc8jt6lX">
<p>24. <strong>A practical framework and online tool for mutational signature analyses show intertissue variation and driver dependencies</strong> <br />
Andrea Degasperi, Tauanne Dias Amarante, Jan Czarnecki, Scott Shooter, Xueqing Zou, Dominik Glodzik, Sandro Morganella, Arjun S. Nanda, Cherif Badja, Gene Koh, … Serena Nik-Zainal<br />
<em>Nature Cancer</em> (2020-02-14) <a href="https://doi.org/gjddc6">https://doi.org/gjddc6</a> <br />
DOI: <a href="https://doi.org/10.1038/s43018-020-0027-5">10.1038/s43018-020-0027-5</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32118208">32118208</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7048622">PMC7048622</a></p>
</div>
<div id="ref-X6oQhIf8">
<p>25. <strong>Genomic and Molecular Landscape of DNA Damage Repair Deficiency across The Cancer Genome Atlas</strong> <br />
Theo A. Knijnenburg, Linghua Wang, Michael T. Zimmermann, Nyasha Chambwe, Galen F. Gao, Andrew D. Cherniack, Huihui Fan, Hui Shen, Gregory P. Way, Casey S. Greene, … Armaz Mariamidze<br />
<em>Cell Reports</em> (2018-04) <a href="https://doi.org/gfspsc">https://doi.org/gfspsc</a> <br />
DOI: <a href="https://doi.org/10.1016/j.celrep.2018.03.076">10.1016/j.celrep.2018.03.076</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29617664">29617664</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5961503">PMC5961503</a></p>
</div>
<div id="ref-SDwYl8uA">
<p>26. <strong>Genomic Profiling of Childhood Tumor Patient-Derived Xenograft Models to Enable Rational Clinical Trial Design</strong> <br />
Jo Lynne Rokita, Komal S. Rathi, Maria F. Cardenas, Kristen A. Upton, Joy Jayaseelan, Katherine L. Cross, Jacob Pfeil, Laura E. Egolf, Gregory P. Way, Alvin Farrel, … John M. Maris<br />
<em>Cell Reports</em> (2019-11) <a href="https://doi.org/gg596n">https://doi.org/gg596n</a> <br />
DOI: <a href="https://doi.org/10.1016/j.celrep.2019.09.071">10.1016/j.celrep.2019.09.071</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31693904">31693904</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6880934">PMC6880934</a></p>
</div>
<div id="ref-14asizOhY">
<p>27. <strong>Gain of function mutations in p53</strong> <br />
Dirk Dittmer, Sibani Pati, Gerard Zambetti, Shelley Chu, Angelika K. Teresky, Mary Moore, Cathy Finlay, Arnold J. Levine<br />
<em>Nature Genetics</em> (1993-05) <a href="https://doi.org/crqst7">https://doi.org/crqst7</a> <br />
DOI: <a href="https://doi.org/10.1038/ng0593-42">10.1038/ng0593-42</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/8099841">8099841</a></p>
</div>
<div id="ref-RnBZNbdx">
<p>28. <strong>Integrated analysis of telomerase enzymatic activity unravels an association with cancer stemness and proliferation</strong> <br />
Nighat Noureen, Shaofang Wu, Yingli Lv, Juechen Yang, W. K. Alfred Yung, Jonathan Gelfond, Xiaojing Wang, Dimpy Koul, Andrew Ludlow, Siyuan Zheng<br />
<em>Nature Communications</em> (2021-12) <a href="https://doi.org/gmfxxd">https://doi.org/gmfxxd</a> <br />
DOI: <a href="https://doi.org/10.1038/s41467-020-20474-9">10.1038/s41467-020-20474-9</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33420056">33420056</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7794223">PMC7794223</a></p>
</div>
<div id="ref-THjIzNXb">
<p>29. <strong>Open Pediatric Brain Tumor Atlas</strong> <br />
Children’s Brain Tumor Network, Pediatric Neuro Oncology Consortium Open Pediatric Brain Tumor Atlas<br />
<em>Kids First Data Resource Center</em> (2022) <a href="https://doi.org/gpp5dv">https://doi.org/gpp5dv</a> <br />
DOI: <a href="https://doi.org/10.24370/openpbta">10.24370/openpbta</a></p>
</div>
<div id="ref-cLFzPEr9">
<p>30. <strong>Pacific Pediatric Neuro-Oncology Consortium</strong> <a href="https://pnoc.us/">https://pnoc.us/</a></p>
</div>
<div id="ref-JkpD6jMa">
<p>31. <strong>Working Together to Put Kids First</strong> <a href="https://kidsfirstdrc.org/">https://kidsfirstdrc.org/</a></p>
</div>
<div id="ref-v08ocfr0">
<p>32. <strong>Pediatric High Grade Glioma Resources From the Children’s Brain Tumor Tissue Consortium (CBTTC) and Pediatric Brain Tumor Atlas (PBTA)</strong> <br />
Heba Ijaz, Mateusz Koptyra, Krutika S. Gaonkar, Jo Lynne Rokita, Valerie P. Baubet, Lamiya Tauhid, Yankun Zhu, Miguel Brown, Gonzalo Lopez, Bo Zhang, … Kristina A. Cole<br />
<em>Cancer Biology</em> (2019-05-31) <a href="https://doi.org/gf66qt">https://doi.org/gf66qt</a> <br />
DOI: <a href="https://doi.org/10.1101/656587">10.1101/656587</a></p>
</div>
<div id="ref-gmH6YDca">
<p>33. <strong>Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM</strong> <br />
Heng Li<br />
<em>arXiv</em> (2013-05-28) <a href="https://arxiv.org/abs/1303.3997">https://arxiv.org/abs/1303.3997</a></p>
</div>
<div id="ref-19mY46wTE">
<p>34. <strong>Index of /goldenPath/hg38/bigZips</strong> <a href="http://hgdownload.soe.ucsc.edu/goldenPath/hg38/bigZips/">http://hgdownload.soe.ucsc.edu/goldenPath/hg38/bigZips/</a></p>
</div>
<div id="ref-EHMzG34o">
<p>35. <a href="https://software.broadinstitute.org/gatk/best-practices/workflow?id">https://software.broadinstitute.org/gatk/best-practices/workflow?id</a></p>
</div>
<div id="ref-IzUbOv92">
<p>36. <strong>SAMBLASTER: fast duplicate marking and structural variant read extraction</strong> <br />
G. G. Faust, I. M. Hall<br />
<em>Bioinformatics</em> (2014-09-01) <a href="https://doi.org/f6kft3">https://doi.org/f6kft3</a> <br />
DOI: <a href="https://doi.org/10.1093/bioinformatics/btu314">10.1093/bioinformatics/btu314</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24812344">24812344</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4147885">PMC4147885</a></p>
</div>
<div id="ref-PcJwlZj3">
<p>37. <strong>Sambamba: fast processing of NGS alignment formats</strong> <br />
Artem Tarasov, Albert J. Vilella, Edwin Cuppen, Isaac J. Nijman, Pjotr Prins<br />
<em>Bioinformatics</em> (2015-06-15) <a href="https://doi.org/gfzsfw">https://doi.org/gfzsfw</a> <br />
DOI: <a href="https://doi.org/10.1093/bioinformatics/btv098">10.1093/bioinformatics/btv098</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25697820">25697820</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4765878">PMC4765878</a></p>
</div>
<div id="ref-15Mgg4tfC">
<p>38. <strong>Scaling accurate genetic variant discovery to tens of thousands of samples</strong> <br />
Ryan Poplin, Valentin Ruano-Rubio, Mark A. DePristo, Tim J. Fennell, Mauricio O. Carneiro, Geraldine A. Van der Auwera, David E. Kling, Laura D. Gauthier, Ami Levy-Moonshine, David Roazen, … Eric Banks<br />
<em>Genomics</em> (2017-11-14) <a href="https://doi.org/ggmrvr">https://doi.org/ggmrvr</a> <br />
DOI: <a href="https://doi.org/10.1101/201178">10.1101/201178</a></p>
</div>
<div id="ref-jkkXdcPS">
<p>39. <strong>Broad Genome References</strong> <a href="https://s3.amazonaws.com/broad-references/broad-references-readme.html">https://s3.amazonaws.com/broad-references/broad-references-readme.html</a></p>
</div>
<div id="ref-A4FXW005">
<p>40. <strong>NGSCheckMate: software for validating sample identity in next-generation sequencing studies within and across data types</strong> <br />
Sejoon Lee, Soohyun Lee, Scott Ouellette, Woong-Yang Park, Eunjung A. Lee, Peter J. Park<br />
<em>Nucleic Acids Research</em> (2017-06-20) <a href="https://doi.org/f9xrq4">https://doi.org/f9xrq4</a> <br />
DOI: <a href="https://doi.org/10.1093/nar/gkx193">10.1093/nar/gkx193</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28369524">28369524</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5499645">PMC5499645</a></p>
</div>
<div id="ref-16tUBMk59">
<p>41. <strong>GitHub - parklab/NGSCheckMate: Software program for checking sample matching for NGS data</strong> <br />
GitHub<br />
<a href="https://github.com/parklab/NGSCheckMate">https://github.com/parklab/NGSCheckMate</a></p>
</div>
<div id="ref-REfkDUtE">
<p>42. <strong>Strelka2: fast and accurate calling of germline and somatic variants</strong> <br />
Sangtae Kim, Konrad Scheffler, Aaron L. Halpern, Mitchell A. Bekritsky, Eunho Noh, Morten Källberg, Xiaoyu Chen, Yeonbin Kim, Doruk Beyter, Peter Krusche, Christopher T. Saunders<br />
<em>Nature Methods</em> (2018-08) <a href="https://doi.org/gdwrp4">https://doi.org/gdwrp4</a> <br />
DOI: <a href="https://doi.org/10.1038/s41592-018-0051-x">10.1038/s41592-018-0051-x</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30013048">30013048</a></p>
</div>
<div id="ref-149BEKISi">
<p>43. <strong>Calling Somatic SNVs and Indels with Mutect2</strong> <br />
David Benjamin, Takuto Sato, Kristian Cibulskis, Gad Getz, Chip Stewart, Lee Lichtenstein<br />
<em>Bioinformatics</em> (2019-12-02) <a href="https://doi.org/ggntwv">https://doi.org/ggntwv</a> <br />
DOI: <a href="https://doi.org/10.1101/861054">10.1101/861054</a></p>
</div>
<div id="ref-V6KdWVYi">
<p>44. <strong>Genome-wide somatic variant calling using localized colored de Bruijn graphs</strong> <br />
Giuseppe Narzisi, André Corvelo, Kanika Arora, Ewa A. Bergmann, Minita Shah, Rajeeva Musunuri, Anne-Katrin Emde, Nicolas Robine, Vladimir Vacic, Michael C. Zody<br />
<em>Communications Biology</em> (2018-12) <a href="https://doi.org/gfcfr8">https://doi.org/gfcfr8</a> <br />
DOI: <a href="https://doi.org/10.1038/s42003-018-0023-9">10.1038/s42003-018-0023-9</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30271907">30271907</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6123722">PMC6123722</a></p>
</div>
<div id="ref-trQRR8fs">
<p>45. <strong>VarDict: a novel and versatile variant caller for next-generation sequencing in cancer research</strong> <br />
Zhongwu Lai, Aleksandra Markovets, Miika Ahdesmaki, Brad Chapman, Oliver Hofmann, Robert McEwen, Justin Johnson, Brian Dougherty, J. Carl Barrett, Jonathan R. Dry<br />
<em>Nucleic Acids Research</em> (2016-06-20) <a href="https://doi.org/f8v6qz">https://doi.org/f8v6qz</a> <br />
DOI: <a href="https://doi.org/10.1093/nar/gkw227">10.1093/nar/gkw227</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27060149">27060149</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4914105">PMC4914105</a></p>
</div>
<div id="ref-E5aTvXmQ">
<p>46. <strong>gatk/mutect2.wdl at 4.1.1.0 · broadinstitute/gatk</strong> <br />
GitHub<br />
<a href="https://github.com/broadinstitute/gatk">https://github.com/broadinstitute/gatk</a></p>
</div>
<div id="ref-1GMHnwH5p">
<p>47. <strong>GitHub - AstraZeneca-NGS/VarDictJava: VarDict Java port</strong> <br />
GitHub<br />
<a href="https://github.com/AstraZeneca-NGS/VarDictJava">https://github.com/AstraZeneca-NGS/VarDictJava</a></p>
</div>
<div id="ref-bKthX7vJ">
<p>48. <strong>Deep sequencing of 3 cancer cell lines on 2 sequencing platforms</strong> <br />
Kanika Arora, Minita Shah, Molly Johnson, Rashesh Sanghvi, Jennifer Shelton, Kshithija Nagulapalli, Dayna M. Oschwald, Michael C. Zody, Soren Germer, Vaidehi Jobanputra, … Nicolas Robine<br />
<em>Genomics</em> (2019-04-30) <a href="https://doi.org/ggc9vx">https://doi.org/ggc9vx</a> <br />
DOI: <a href="https://doi.org/10.1101/623702">10.1101/623702</a></p>
</div>
<div id="ref-SkVcTxDP">
<p>49. <strong>bcftools(1)</strong> <a href="http://samtools.github.io/bcftools/bcftools.html#norm">http://samtools.github.io/bcftools/bcftools.html#norm</a></p>
</div>
<div id="ref-TeJdzIor">
<p>50. <strong>OpenPBTA-workflows/kfdrc_annot_vcf_sub_wf.cwl at master · d3b-center/OpenPBTA-workflows</strong> <br />
GitHub<br />
<a href="https://github.com/d3b-center/OpenPBTA-workflows">https://github.com/d3b-center/OpenPBTA-workflows</a></p>
</div>
<div id="ref-p1f5DxRQ">
<p>51. <strong>The Ensembl Variant Effect Predictor</strong> <br />
William McLaren, Laurent Gil, Sarah E. Hunt, Harpreet Singh Riat, Graham R. S. Ritchie, Anja Thormann, Paul Flicek, Fiona Cunningham<br />
<em>Genome Biology</em> (2016-12) <a href="https://doi.org/gdz75c">https://doi.org/gdz75c</a> <br />
DOI: <a href="https://doi.org/10.1186/s13059-016-0974-4">10.1186/s13059-016-0974-4</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27268795">27268795</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4893825">PMC4893825</a></p>
</div>
<div id="ref-qEpbKZ3t">
<p>52. <strong>Cancer Hotspots</strong> <a href="https://www.cancerhotspots.org/">https://www.cancerhotspots.org/</a></p>
</div>
<div id="ref-59DKOl44">
<p>53. <strong>Activating Telomerase TERT Promoter Mutations and Their Application for the Detection of Bladder Cancer</strong> <br />
Maria Zvereva, Eduard Pisarev, Ismail Hosen, Olga Kisil, Simon Matskeplishvili, Elena Kubareva, David Kamalov, Alexander Tivtikyan, Arnaud Manel, Emmanuel Vian, … Florence Le Calvez-Kelm<br />
<em>International Journal of Molecular Sciences</em> (2020-08-21) <a href="https://doi.org/gmf45b">https://doi.org/gmf45b</a> <br />
DOI: <a href="https://doi.org/10.3390/ijms21176034">10.3390/ijms21176034</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32839402">32839402</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7503716">PMC7503716</a></p>
</div>
<div id="ref-11Fn5jqUI">
<p>54. <strong>GitHub - mskcc/vcf2maf: Convert a VCF into a MAF, where each variant is annotated to only one of all possible gene isoforms</strong> <br />
GitHub<br />
<a href="https://github.com/mskcc/vcf2maf">https://github.com/mskcc/vcf2maf</a></p>
</div>
<div id="ref-Ob2wQ9wX">
<p>55. <strong>OpenPBTA-analysis/compare_snv_callers_plots.Rmd at master · AlexsLemonade/OpenPBTA-analysis</strong> <br />
GitHub<br />
<a href="https://github.com/AlexsLemonade/OpenPBTA-analysis">https://github.com/AlexsLemonade/OpenPBTA-analysis</a></p>
</div>
<div id="ref-Z1i0sjNH">
<p>56. <a href="https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/snv-callers/plots/comparison/upset_plot.png">https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/snv-callers/plots/comparison/upset_plot.png</a></p>
</div>
<div id="ref-tkwfe0u6">
<p>57. <a href="https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/snv-callers/plots/comparison/vaf_violin_plot.png">https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/snv-callers/plots/comparison/vaf_violin_plot.png</a></p>
</div>
<div id="ref-9oRUnb2k">
<p>58. <strong>GENCODE - Human Release 27</strong> <a href="https://www.gencodegenes.org/human/release_27.html">https://www.gencodegenes.org/human/release_27.html</a></p>
</div>
<div id="ref-ZQ0L3o1q">
<p>59. <strong>Control-FREEC: a tool for assessing copy number and allelic content using next-generation sequencing data</strong> <br />
Valentina Boeva, Tatiana Popova, Kevin Bleakley, Pierre Chiche, Julie Cappo, Gudrun Schleiermacher, Isabelle Janoueix-Lerosey, Olivier Delattre, Emmanuel Barillot<br />
<em>Bioinformatics</em> (2012-02-01) <a href="https://doi.org/ckt4vz">https://doi.org/ckt4vz</a> <br />
DOI: <a href="https://doi.org/10.1093/bioinformatics/btr670">10.1093/bioinformatics/btr670</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22155870">22155870</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3268243">PMC3268243</a></p>
</div>
<div id="ref-1F3i4BvCt">
<p>60. <strong>Control-free calling of copy number alterations in deep-sequencing data using GC-content normalization</strong> <br />
Valentina Boeva, Andrei Zinovyev, Kevin Bleakley, Jean-Philippe Vert, Isabelle Janoueix-Lerosey, Olivier Delattre, Emmanuel Barillot<br />
<em>Bioinformatics</em> (2011-01-15) <a href="https://doi.org/c6bcps">https://doi.org/c6bcps</a> <br />
DOI: <a href="https://doi.org/10.1093/bioinformatics/btq635">10.1093/bioinformatics/btq635</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21081509">21081509</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3018818">PMC3018818</a></p>
</div>
<div id="ref-UTxRcYIQ">
<p>61. <strong>CNVkit: Genome-Wide Copy Number Detection and Visualization from Targeted DNA Sequencing</strong> <br />
Eric Talevich, A. Hunter Shain, Thomas Botton, Boris C. Bastian<br />
<em>PLOS Computational Biology</em> (2016-04-21) <a href="https://doi.org/c9pd">https://doi.org/c9pd</a> <br />
DOI: <a href="https://doi.org/10.1371/journal.pcbi.1004873">10.1371/journal.pcbi.1004873</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27100738">27100738</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4839673">PMC4839673</a></p>
</div>
<div id="ref-V4PckbrH">
<p>62. <strong>Quantifying tumor heterogeneity in whole-genome and whole-exome sequencing data</strong> <br />
Layla Oesper, Gryte Satas, Benjamin J. Raphael<br />
<em>Bioinformatics</em> (2014-12-15) <a href="https://doi.org/f6rmgt">https://doi.org/f6rmgt</a> <br />
DOI: <a href="https://doi.org/10.1093/bioinformatics/btu651">10.1093/bioinformatics/btu651</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25297070">25297070</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4253833">PMC4253833</a></p>
</div>
<div id="ref-rDV1hhfF">
<p>63. <strong>GISTIC2.0 facilitates sensitive and confident localization of the targets of focal somatic copy-number alteration in human cancers</strong> <br />
Craig H Mermel, Steven E Schumacher, Barbara Hill, Matthew L Meyerson, Rameen Beroukhim, Gad Getz<br />
<em>Genome Biology</em> (2011-04) <a href="https://doi.org/dzhjqh">https://doi.org/dzhjqh</a> <br />
DOI: <a href="https://doi.org/10.1186/gb-2011-12-4-r41">10.1186/gb-2011-12-4-r41</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21527027">21527027</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3218867">PMC3218867</a></p>
</div>
<div id="ref-kTn1PIj5">
<p>64. <strong>Manta: rapid detection of structural variants and indels for germline and cancer sequencing applications</strong> <br />
Xiaoyu Chen, Ole Schulz-Trieglaff, Richard Shaw, Bret Barnes, Felix Schlesinger, Morten Källberg, Anthony J. Cox, Semyon Kruglyak, Christopher T. Saunders<br />
<em>Bioinformatics</em> (2016-04-15) <a href="https://doi.org/gf3ggb">https://doi.org/gf3ggb</a> <br />
DOI: <a href="https://doi.org/10.1093/bioinformatics/btv710">10.1093/bioinformatics/btv710</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26647377">26647377</a></p>
</div>
<div id="ref-10RYb622Q">
<p>65. <strong>manta/README.md at 75b5c38d4fcd2f6961197b28a41eb61856f2d976 · Illumina/manta</strong> <br />
GitHub<br />
<a href="https://github.com/Illumina/manta">https://github.com/Illumina/manta</a></p>
</div>
<div id="ref-7Ull5aU5">
<p>66. <strong>AnnotSV: an integrated tool for structural variations annotation</strong> <br />
Véronique Geoffroy, Yvan Herenger, Arnaud Kress, Corinne Stoetzel, Amélie Piton, Hélène Dollfus, Jean Muller<br />
<em>Bioinformatics</em> (2018-10-15) <a href="https://doi.org/gdcsh3">https://doi.org/gdcsh3</a> <br />
DOI: <a href="https://doi.org/10.1093/bioinformatics/bty304">10.1093/bioinformatics/bty304</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29669011">29669011</a></p>
</div>
<div id="ref-tTu8Ds9Z">
<p>67. <strong>STAR: ultrafast universal RNA-seq aligner</strong> <br />
Alexander Dobin, Carrie A. Davis, Felix Schlesinger, Jorg Drenkow, Chris Zaleski, Sonali Jha, Philippe Batut, Mark Chaisson, Thomas R. Gingeras<br />
<em>Bioinformatics</em> (2013-01) <a href="https://doi.org/f4h523">https://doi.org/f4h523</a> <br />
DOI: <a href="https://doi.org/10.1093/bioinformatics/bts635">10.1093/bioinformatics/bts635</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23104886">23104886</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3530905">PMC3530905</a></p>
</div>
<div id="ref-Dh2n1EV3">
<p>68. <strong>RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome</strong> <br />
Bo Li, Colin N Dewey<br />
<em>BMC Bioinformatics</em> (2011-12) <a href="https://doi.org/cwg8n5">https://doi.org/cwg8n5</a> <br />
DOI: <a href="https://doi.org/10.1186/1471-2105-12-323">10.1186/1471-2105-12-323</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21816040">21816040</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3163565">PMC3163565</a></p>
</div>
<div id="ref-12KZMHMQl">
<p>69. <strong>Near-optimal probabilistic RNA-seq quantification</strong> <br />
Nicolas L Bray, Harold Pimentel, Páll Melsted, Lior Pachter<br />
<em>Nature Biotechnology</em> (2016-05) <a href="https://doi.org/f8nvsp">https://doi.org/f8nvsp</a> <br />
DOI: <a href="https://doi.org/10.1038/nbt.3519">10.1038/nbt.3519</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27043002">27043002</a></p>
</div>
<div id="ref-rqYlIiAi">
<p>70. <strong>STAR-Fusion: Fast and Accurate Fusion Transcript Detection from RNA-Seq</strong> <br />
Brian J. Haas, Alex Dobin, Nicolas Stransky, Bo Li, Xiao Yang, Timothy Tickle, Asma Bankapur, Carrie Ganote, Thomas G. Doak, Nathalie Pochet, … Aviv Regev<br />
<em>Bioinformatics</em> (2017-03-24) <a href="https://doi.org/gf5pc5">https://doi.org/gf5pc5</a> <br />
DOI: <a href="https://doi.org/10.1101/120295">10.1101/120295</a></p>
</div>
<div id="ref-83Prx3Xc">
<p>71. <strong>Maftools: efficient and comprehensive analysis of somatic variants in cancer</strong> <br />
Anand Mayakonda, De-Chen Lin, Yassen Assenov, Christoph Plass, H. Phillip Koeffler<br />
<em>Genome Research</em> (2018-11) <a href="https://doi.org/gfmnwf">https://doi.org/gfmnwf</a> <br />
DOI: <a href="https://doi.org/10.1101/gr.239244.118">10.1101/gr.239244.118</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30341162">30341162</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6211645">PMC6211645</a></p>
</div>
<div id="ref-Ddy8jB2X">
<p>72. <strong>coverage — bedtools 2.30.0 documentation</strong> <a href="https://bedtools.readthedocs.io/en/latest/content/tools/coverage.html">https://bedtools.readthedocs.io/en/latest/content/tools/coverage.html</a></p>
</div>
<div id="ref-1HWiAHnIw">
<p>73. <strong>BEDTools: a flexible suite of utilities for comparing genomic features</strong> <br />
Aaron R. Quinlan, Ira M. Hall<br />
<em>Bioinformatics</em> (2010-03-15) <a href="https://doi.org/cmrms3">https://doi.org/cmrms3</a> <br />
DOI: <a href="https://doi.org/10.1093/bioinformatics/btq033">10.1093/bioinformatics/btq033</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20110278">20110278</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2832824">PMC2832824</a></p>
</div>
<div id="ref-H6flbhE">
<p>74. <a href="http://hgdownload.cse.ucsc.edu/goldenpath/hg38/database/cytoBand.txt.gz">http://hgdownload.cse.ucsc.edu/goldenpath/hg38/database/cytoBand.txt.gz</a></p>
</div>
<div id="ref-1DWggwY7R">
<p>75. <strong>The UCSC Genome Browser database: extensions and updates 2013</strong> <br />
Laurence R. Meyer, Ann S. Zweig, Angie S. Hinrichs, Donna Karolchik, Robert M. Kuhn, Matthew Wong, Cricket A. Sloan, Kate R. Rosenbloom, Greg Roe, Brooke Rhead, … W. James Kent<br />
<em>Nucleic Acids Research</em> (2012-11-15) <a href="https://doi.org/f4jr4v">https://doi.org/f4jr4v</a> <br />
DOI: <a href="https://doi.org/10.1093/nar/gks1048">10.1093/nar/gks1048</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23155063">23155063</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3531082">PMC3531082</a></p>
</div>
<div id="ref-GAYU0k3L">
<p>76. <strong>findOverlaps-methods function - RDocumentation</strong> <a href="https://www.rdocumentation.org/packages/IRanges/versions/2.6.1/topics/findOverlaps-methods">https://www.rdocumentation.org/packages/IRanges/versions/2.6.1/topics/findOverlaps-methods</a></p>
</div>
<div id="ref-14d6Jzwus">
<p>77. <strong>Software for Computing and Annotating Genomic Ranges</strong> <br />
Michael Lawrence, Wolfgang Huber, Hervé Pagès, Patrick Aboyoun, Marc Carlson, Robert Gentleman, Martin T. Morgan, Vincent J. Carey<br />
<em>PLoS Computational Biology</em> (2013-08-08) <a href="https://doi.org/f5cmfg">https://doi.org/f5cmfg</a> <br />
DOI: <a href="https://doi.org/10.1371/journal.pcbi.1003118">10.1371/journal.pcbi.1003118</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23950696">23950696</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3738458">PMC3738458</a></p>
</div>
<div id="ref-17wG7AMwM">
<p>78. <strong>Comprehensive analysis of chromothripsis in 2,658 human cancers using whole-genome sequencing</strong> <br />
Isidro Cortés-Ciriano, Jake June-Koo Lee, Ruibin Xi, Dhawal Jain, Youngsook L. Jung, Lixing Yang, Dmitry Gordenin, Leszek J. Klimczak, Cheng-Zhong Zhang, David S. Pellman, … PCAWG Consortium<br />
<em>Nature Genetics</em> (2020-03-02) <a href="https://doi.org/ggkkpt">https://doi.org/ggkkpt</a> <br />
DOI: <a href="https://doi.org/10.1038/s41588-019-0576-7">10.1038/s41588-019-0576-7</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32025003">32025003</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7058534">PMC7058534</a></p>
</div>
<div id="ref-HnrrHXPm">
<p>79. <strong>Comprehensive evaluation of transcriptome-based cell-type quantification methods for immuno-oncology.</strong> <br />
Gregor Sturm, Francesca Finotello, Florent Petitprez, Jitao David Zhang, Jan Baumbach, Wolf H Fridman, Markus List, Tatsiana Aneichyk<br />
<em>Bioinformatics (Oxford, England)</em> (2019-07-15) <a href="https://www.ncbi.nlm.nih.gov/pubmed/31510660">https://www.ncbi.nlm.nih.gov/pubmed/31510660</a> <br />
DOI: <a href="https://doi.org/10.1093/bioinformatics/btz363">10.1093/bioinformatics/btz363</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31510660">31510660</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6612828">PMC6612828</a></p>
</div>
<div id="ref-klaJnG8x">
<p>80. <strong>GitHub - icbi-lab/immunedeconv: A unified interface to immune deconvolution methods (CIBERSORT, EPIC, quanTIseq, TIMER, xCell, MCPcounter)</strong> <br />
GitHub<br />
<a href="https://github.com/icbi-lab/immunedeconv">https://github.com/icbi-lab/immunedeconv</a></p>
</div>
<div id="ref-1F5zJEMaD">
<p>81. <strong>GSVA: gene set variation analysis for microarray and RNA-Seq data</strong> <br />
Sonja Hänzelmann, Robert Castelo, Justin Guinney<br />
<em>BMC Bioinformatics</em> (2013-12) <a href="https://doi.org/gb8vx5">https://doi.org/gb8vx5</a> <br />
DOI: <a href="https://doi.org/10.1186/1471-2105-14-7">10.1186/1471-2105-14-7</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23323831">23323831</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3618321">PMC3618321</a></p>
</div>
<div id="ref-w3jDgsn7">
<p>82. <strong>GSVA</strong> <br />
Justin Guinney [Aut, Cre], Robert Castelo [Aut], Joan Fernandez[Ctb]<br />
<em>Bioconductor</em> (2017) <a href="https://doi.org/ggxrqs">https://doi.org/ggxrqs</a> <br />
DOI: <a href="https://doi.org/10.18129/b9.bioc.gsva">10.18129/b9.bioc.gsva</a></p>
</div>
<div id="ref-1CbVoEpNJ">
<p>83. <strong>The Molecular Signatures Database Hallmark Gene Set Collection</strong> <br />
Arthur Liberzon, Chet Birger, Helga Thorvaldsdóttir, Mahmoud Ghandi, Jill P. Mesirov, Pablo Tamayo<br />
<em>Cell Systems</em> (2015-12) <a href="https://doi.org/gf78hq">https://doi.org/gf78hq</a> <br />
DOI: <a href="https://doi.org/10.1016/j.cels.2015.12.004">10.1016/j.cels.2015.12.004</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26771021">26771021</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4707969">PMC4707969</a></p>
</div>
<div id="ref-14XP70sGJ">
<p>84. <strong>UMAP: Uniform Manifold Approximation and Projection for Dimension
Reduction</strong> <br />
Leland McInnes, John Healy, James Melville<br />
<em>arXiv</em> (2018-02-09) <a href="https://arxiv.org/abs/1802.03426v2">https://arxiv.org/abs/1802.03426v2</a></p>
</div>
<div id="ref-WsAj0wOM">
<p>85. <a href="https://cran.r-project.org/package">https://cran.r-project.org/package</a></p>
</div>
<div id="ref-9vS8HBL6">
<p>86. <strong>The Human Transcription Factors</strong> <br />
Samuel A. Lambert, Arttu Jolma, Laura F. Campitelli, Pratyush K. Das, Yimeng Yin, Mihai Albu, Xiaoting Chen, Jussi Taipale, Timothy R. Hughes, Matthew T. Weirauch<br />
<em>Cell</em> (2018-02) <a href="https://doi.org/gcw8rb">https://doi.org/gcw8rb</a> <br />
DOI: <a href="https://doi.org/10.1016/j.cell.2018.01.029">10.1016/j.cell.2018.01.029</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29425488">29425488</a></p>
</div>
<div id="ref-jLWV5IWB">
<p>87. <strong>Genomic analysis of diffuse pediatric low-grade gliomas identifies recurrent oncogenic truncating rearrangements in the transcription factor <em>MYBL1</em></strong> <br />
Lori A. Ramkissoon, Peleg M. Horowitz, Justin M. Craig, Shakti H. Ramkissoon, Benjamin E. Rich, Steven E. Schumacher, Aaron McKenna, Michael S. Lawrence, Guillaume Bergthold, Priscilla K. Brastianos, … Keith L. Ligon<br />
<em>Proceedings of the National Academy of Sciences</em> (2013-05-14) <a href="https://doi.org/f42gg4">https://doi.org/f42gg4</a> <br />
DOI: <a href="https://doi.org/10.1073/pnas.1300252110">10.1073/pnas.1300252110</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23633565">23633565</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3657784">PMC3657784</a></p>
</div>
<div id="ref-4wYR62jK">
<p>88. <strong>Subgroup-specific structural variation across 1,000 medulloblastoma genomes</strong> <br />
Paul A. Northcott, David J. H. Shih, John Peacock, Livia Garzia, A. Sorana Morrissy, Thomas Zichner, Adrian M. Stütz, Andrey Korshunov, Jüri Reimand, Steven E. Schumacher, … Michael D. Taylor<br />
<em>Nature</em> (2012-08) <a href="https://doi.org/ggdhk3">https://doi.org/ggdhk3</a> <br />
DOI: <a href="https://doi.org/10.1038/nature11327">10.1038/nature11327</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22832581">22832581</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3683624">PMC3683624</a></p>
</div>
<div id="ref-kfmK8vm">
<p>89. <strong>New Brain Tumor Entities Emerge from Molecular Classification of CNS-PNETs</strong> <br />
Dominik Sturm, Brent A. Orr, Umut H. Toprak, Volker Hovestadt, David T.W. Jones, David Capper, Martin Sill, Ivo Buchhalter, Paul A. Northcott, Irina Leis, … Marcel Kool<br />
<em>Cell</em> (2016-02) <a href="https://doi.org/f3t869">https://doi.org/f3t869</a> <br />
DOI: <a href="https://doi.org/10.1016/j.cell.2016.01.015">10.1016/j.cell.2016.01.015</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26919435">26919435</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5139621">PMC5139621</a></p>
</div>
<div id="ref-5ueZBnsJ">
<p>90. <strong>Fusion of TTYH1 with the C19MC microRNA cluster drives expression of a brain-specific DNMT3B isoform in the embryonal brain tumor ETMR</strong> <br />
Claudia L Kleinman, Noha Gerges, Simon Papillon-Cavanagh, Patrick Sin-Chan, Albena Pramatarova, Dong-Anh Khuong Quang, Véronique Adoue, Stephan Busche, Maxime Caron, Haig Djambazian, … Nada Jabado<br />
<em>Nature Genetics</em> (2014-01) <a href="https://doi.org/ggdhk4">https://doi.org/ggdhk4</a> <br />
DOI: <a href="https://doi.org/10.1038/ng.2849">10.1038/ng.2849</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24316981">24316981</a></p>
</div>
<div id="ref-ASmwGlFp">
<p>91. <strong>TERT rearrangements are frequent in neuroblastoma and identify aggressive tumors</strong> <br />
Linda J Valentijn, Jan Koster, Danny A Zwijnenburg, Nancy E Hasselt, Peter van Sluis, Richard Volckmann, Max M van Noesel, Rani E George, Godelieve AM Tytgat, Jan J Molenaar, Rogier Versteeg<br />
<em>Nature Genetics</em> (2015-12) <a href="https://doi.org/ggdhk5">https://doi.org/ggdhk5</a> <br />
DOI: <a href="https://doi.org/10.1038/ng.3438">10.1038/ng.3438</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26523776">26523776</a></p>
</div>
<div id="ref-YfG9EVSk">
<p>92. <strong>Recurrent pre-existing and acquired DNA copy number alterations, including focal <em>TERT</em> gains, in neuroblastoma central nervous system metastases: Copy Number Changes in Neuroblastoma Cns Metastases</strong> <br />
David Cobrinik, Irina Ostrovnaya, Maryam Hassimi, Satish K. Tickoo, Irene Y. Cheung, Nai-Kong V. Cheung<br />
<em>Genes, Chromosomes and Cancer</em> (2013-12) <a href="https://doi.org/f5gd94">https://doi.org/f5gd94</a> <br />
DOI: <a href="https://doi.org/10.1002/gcc.22110">10.1002/gcc.22110</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24123354">24123354</a></p>
</div>
<div id="ref-lWMOs28t">
<p>93. <strong>Activation of human telomerase reverse transcriptase through gene fusion in clear cell sarcoma of the kidney</strong> <br />
Jenny Karlsson, Henrik Lilljebjörn, Linda Holmquist Mengelbier, Anders Valind, Marianne Rissler, Ingrid Øra, Thoas Fioretos, David Gisselsson<br />
<em>Cancer Letters</em> (2015-02) <a href="https://doi.org/f25ck5">https://doi.org/f25ck5</a> <br />
DOI: <a href="https://doi.org/10.1016/j.canlet.2014.11.057">10.1016/j.canlet.2014.11.057</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25481751">25481751</a></p>
</div>
<div id="ref-1B3tdZcAl">
<p>94. <strong>New Molecular Considerations for Glioma: IDH, ATRX, BRAF, TERT, H3 K27M</strong> <br />
Michael Karsy, Jian Guan, Adam L. Cohen, Randy L. Jensen, Howard Colman<br />
<em>Current Neurology and Neuroscience Reports</em> (2017-02) <a href="https://doi.org/ggdhk2">https://doi.org/ggdhk2</a> <br />
DOI: <a href="https://doi.org/10.1007/s11910-017-0722-5">10.1007/s11910-017-0722-5</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28271343">28271343</a></p>
</div>
<div id="ref-1foRpfch">
<p>95. <strong>MYB-QKI rearrangements in angiocentric glioma drive tumorigenicity through a tripartite mechanism</strong> <br />
Pratiti Bandopadhayay, Lori A Ramkissoon, Payal Jain, Guillaume Bergthold, Jeremiah Wala, Rhamy Zeid, Steven E Schumacher, Laura Urbanski, Ryan O’Rourke, William J Gibson, … Adam C Resnick<br />
<em>Nature Genetics</em> (2016-03) <a href="https://doi.org/f8bwqn">https://doi.org/f8bwqn</a> <br />
DOI: <a href="https://doi.org/10.1038/ng.3500">10.1038/ng.3500</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26829751">26829751</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4767685">PMC4767685</a></p>
</div>
<div id="ref-CeB34T1V">
<p>96. <strong>Atypical Teratoid/Rhabdoid Tumors Are Comprised of Three Epigenetic Subgroups with Distinct Enhancer Landscapes</strong> <br />
Pascal D. Johann, Serap Erkek, Marc Zapatka, Kornelius Kerl, Ivo Buchhalter, Volker Hovestadt, David T. W. Jones, Dominik Sturm, Carl Hermann, Maia Segura Wang, … Marcel Kool<br />
<em>Cancer Cell</em> (2016-03) <a href="https://doi.org/f8fmz4">https://doi.org/f8fmz4</a> <br />
DOI: <a href="https://doi.org/10.1016/j.ccell.2016.02.001">10.1016/j.ccell.2016.02.001</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26923874">26923874</a></p>
</div>
<div id="ref-1FGkQ9Wa7">
<p>97. <strong>Chromosome 19 microRNA cluster enhances cell reprogramming by inhibiting epithelial-to-mesenchymal transition</strong> <br />
Ezinne F. Mong, Ying Yang, Kemal M. Akat, John Canfield, Jeffrey VanWye, John Lockhart, John C. M. Tsibris, Frederick Schatz, Charles J. Lockwood, Thomas Tuschl, … Hana Totary-Jain<br />
<em>Scientific Reports</em> (2020-12) <a href="https://doi.org/gg329d">https://doi.org/gg329d</a> <br />
DOI: <a href="https://doi.org/10.1038/s41598-020-59812-8">10.1038/s41598-020-59812-8</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32080251">32080251</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7033247">PMC7033247</a></p>
</div>
<div id="ref-m7hgzrvh">
<p>98. <strong>EZHIP is a specific diagnostic biomarker for posterior fossa ependymomas, group PFA and diffuse midline gliomas H3-WT with EZHIP overexpression</strong> <br />
C. Antin, A. Tauziède-Espariat, M.-A. Debily, D. Castel, J. Grill, M. Pagès, O. Ayrault, F. Chrétien, A. Gareton, F. Andreiuolo, … P. Varlet<br />
<em>Acta Neuropathologica Communications</em> (2020-12) <a href="https://doi.org/gm3q3q">https://doi.org/gm3q3q</a> <br />
DOI: <a href="https://doi.org/10.1186/s40478-020-01056-8">10.1186/s40478-020-01056-8</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33153494">33153494</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7643397">PMC7643397</a></p>
</div>
<div id="ref-ZXjw164D">
<p>99. <strong>H3 K27M mutations are extremely rare in posterior fossa group A ependymoma</strong> <br />
Scott Ryall, Miguel Guzman, Samer K. Elbabaa, Betty Luu, Stephen C. Mack, Michal Zapotocky, Michael D. Taylor, Cynthia Hawkins, Vijay Ramaswamy<br />
<em>Child’s Nervous System</em> (2017-07) <a href="https://doi.org/gbn2dn">https://doi.org/gbn2dn</a> <br />
DOI: <a href="https://doi.org/10.1007/s00381-017-3481-3">10.1007/s00381-017-3481-3</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28623522">28623522</a></p>
</div>
<div id="ref-f7P6U50v">
<p>100. <strong>Molecular Classification of Ependymal Tumors across All CNS Compartments, Histopathological Grades, and Age Groups</strong> <br />
Kristian W. Pajtler, Hendrik Witt, Martin Sill, David T.W. Jones, Volker Hovestadt, Fabian Kratochwil, Khalida Wani, Ruth Tatevossian, Chandanamali Punchihewa, Pascal Johann, … Stefan M. Pfister<br />
<em>Cancer Cell</em> (2015-05) <a href="https://doi.org/f7ct8f">https://doi.org/f7ct8f</a> <br />
DOI: <a href="https://doi.org/10.1016/j.ccell.2015.04.002">10.1016/j.ccell.2015.04.002</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25965575">25965575</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4712639">PMC4712639</a></p>
</div>
<div id="ref-DGVkAk1W">
<p>101. <strong>C11orf95–RELA fusions drive oncogenic NF-κB signalling in ependymoma</strong> <br />
Matthew Parker, Kumarasamypet M. Mohankumar, Chandanamali Punchihewa, Ricardo Weinlich, James D. Dalton, Yongjin Li, Ryan Lee, Ruth G. Tatevossian, Timothy N. Phoenix, Radhika Thiruvenkatam, … Richard J. Gilbertson<br />
<em>Nature</em> (2014-02-27) <a href="https://doi.org/f5t87h">https://doi.org/f5t87h</a> <br />
DOI: <a href="https://doi.org/10.1038/nature13109">10.1038/nature13109</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24553141">24553141</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4050669">PMC4050669</a></p>
</div>
<div id="ref-ovoneiG3">
<p>102. <strong>Genomic landscape of high-grade meningiomas</strong> <br />
Wenya Linda Bi, Noah F. Greenwald, Malak Abedalthagafi, Jeremiah Wala, Will J. Gibson, Pankaj K. Agarwalla, Peleg Horowitz, Steven E. Schumacher, Ekaterina Esaulova, Yu Mei, … Rameen Beroukhim<br />
<em>npj Genomic Medicine</em> (2017-12) <a href="https://doi.org/gbnhbw">https://doi.org/gbnhbw</a> <br />
DOI: <a href="https://doi.org/10.1038/s41525-017-0014-7">10.1038/s41525-017-0014-7</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28713588">28713588</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5506858">PMC5506858</a></p>
</div>
<div id="ref-uDYz8KKh">
<p>103. <strong>Correlations between genomic subgroup and clinical features in a cohort of more than 3000 meningiomas</strong> <br />
Mark W. Youngblood, Daniel Duran, Julio D. Montejo, Chang Li, Sacit Bulent Omay, Koray Özduman, Amar H. Sheth, Amy Y. Zhao, Evgeniya Tyrtova, Danielle F. Miyagishima, … Murat Günel<br />
<em>Journal of Neurosurgery</em> (2020-11) <a href="https://doi.org/gm3q3r">https://doi.org/gm3q3r</a> <br />
DOI: <a href="https://doi.org/10.3171/2019.8.jns191266">10.3171/2019.8.jns191266</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31653806">31653806</a></p>
</div>
<div id="ref-107ZjoLxM">
<p>104. <strong>Genetic alterations in uncommon low-grade neuroepithelial tumors: BRAF, FGFR1, and MYB mutations occur at high frequency and align with morphology</strong> <br />
Ibrahim Qaddoumi, Wilda Orisme, Ji Wen, Teresa Santiago, Kirti Gupta, James D. Dalton, Bo Tang, Kelly Haupfear, Chandanamali Punchihewa, John Easton, … David W. Ellison<br />
<em>Acta Neuropathologica</em> (2016-06) <a href="https://doi.org/f8m3pn">https://doi.org/f8m3pn</a> <br />
DOI: <a href="https://doi.org/10.1007/s00401-016-1539-z">10.1007/s00401-016-1539-z</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26810070">26810070</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4866893">PMC4866893</a></p>
</div>
<div id="ref-F73HShWn">
<p>105. <strong>The genetic landscape of choroid plexus tumors in children and adults</strong> <br />
Christian Thomas, Patrick Soschinski, Melissa Zwaig, Spyridon Oikonomopoulos, Konstantin Okonechnikov, Kristian W Pajtler, Martin Sill, Leonille Schweizer, Arend Koch, Julia Neumann, … Martin Hasselblatt<br />
<em>Neuro-Oncology</em> (2021-04-12) <a href="https://doi.org/gppwxt">https://doi.org/gppwxt</a> <br />
DOI: <a href="https://doi.org/10.1093/neuonc/noaa267">10.1093/neuonc/noaa267</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33249490">33249490</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8041331">PMC8041331</a></p>
</div>
<div id="ref-BBuQnfgj">
<p>106. <strong>Langerhans cell histiocytosis in children</strong> <br />
Jolie Krooks, Milen Minkov, Angela G. Weatherall<br />
<em>Journal of the American Academy of Dermatology</em> (2018-06) <a href="https://doi.org/gdmdp2">https://doi.org/gdmdp2</a> <br />
DOI: <a href="https://doi.org/10.1016/j.jaad.2017.05.059">10.1016/j.jaad.2017.05.059</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29754885">29754885</a></p>
</div>
<div id="ref-KhxTOfIb">
<p>107. <strong>deconstructSigs: delineating mutational processes in single tumors distinguishes DNA repair deficiencies and patterns of carcinoma evolution</strong> <br />
Rachel Rosenthal, Nicholas McGranahan, Javier Herrero, Barry S. Taylor, Charles Swanton<br />
<em>Genome Biology</em> (2016-12) <a href="https://doi.org/f8bdsq">https://doi.org/f8bdsq</a> <br />
DOI: <a href="https://doi.org/10.1186/s13059-016-0893-4">10.1186/s13059-016-0893-4</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26899170">26899170</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4762164">PMC4762164</a></p>
</div>
<div id="ref-1eIeMT1A">
<p>108. <strong>GitHub - raerose01/deconstructSigs: deconstructSigs</strong> <br />
GitHub<br />
<a href="https://github.com/raerose01/deconstructSigs">https://github.com/raerose01/deconstructSigs</a></p>
</div>
<div id="ref-GpcQXpz9">
<p>109. <strong>BSgenome.Hsapiens.UCSC.hg38</strong> <br />
Bioconductor<br />
<a href="http://bioconductor.org/packages/BSgenome.Hsapiens.UCSC.hg38/">http://bioconductor.org/packages/BSgenome.Hsapiens.UCSC.hg38/</a></p>
</div>
<div id="ref-RmzXK4Cw">
<p>110. <strong>MM2S: personalized diagnosis of medulloblastoma patients and model systems</strong> <br />
Deena M. A. Gendoo, Benjamin Haibe-Kains<br />
<em>Source Code for Biology and Medicine</em> (2016-12) <a href="https://doi.org/ghcqf2">https://doi.org/ghcqf2</a> <br />
DOI: <a href="https://doi.org/10.1186/s13029-016-0053-y">10.1186/s13029-016-0053-y</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27069505">27069505</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4827218">PMC4827218</a></p>
</div>
<div id="ref-6bmmWiU5">
<p>111. <strong>A polyphenotypic malignant paediatric brain tumour presenting a <em>MN1‐PATZ1</em> fusion, no epigenetic similarities with CNS High‐Grade Neuroepithelial Tumour with <em>MN1</em> Alteration (CNS HGNET‐MN1) and related to <em>PATZ1</em> ‐fused sarcomas</strong> <br />
F. Burel‐Vandenbos, G. Pierron, C. Thomas, S. Reynaud, V. Gregoire, G. Duhil de Benaze, S. Croze, N. Chivoret, M. Honavar, D. Figarella‐Branger, … C. Godfraind<br />
<em>Neuropathology and Applied Neurobiology</em> (2020-08) <a href="https://doi.org/gmndph">https://doi.org/gmndph</a> <br />
DOI: <a href="https://doi.org/10.1111/nan.12626">10.1111/nan.12626</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32397004">32397004</a></p>
</div>
<div id="ref-ynt8AIJ4">
<p>112. <strong>Embryonal Tumors of the Central Nervous System in Children: The Era of Targeted Therapeutics.</strong> <br />
David E Kram, Jacob J Henderson, Muhammad Baig, Diya Chakraborty, Morgan A Gardner, Subhasree Biswas, Soumen Khatua<br />
<em>Bioengineering (Basel, Switzerland)</em> (2018-09-23) <a href="https://www.ncbi.nlm.nih.gov/pubmed/30249036">https://www.ncbi.nlm.nih.gov/pubmed/30249036</a> <br />
DOI: <a href="https://doi.org/10.3390/bioengineering5040078">10.3390/bioengineering5040078</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30249036">30249036</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6315657">PMC6315657</a></p>
</div>
<div id="ref-t40FioEk">
<p>113. <strong>LIN28A, a sensitive immunohistochemical marker for Embryonal Tumor with Multilayered Rosettes (ETMR), is also positive in a subset of Atypical Teratoid/Rhabdoid Tumor (AT/RT)</strong> <br />
Shilpa Rao, R. T. Rajeswarie, T. Chickabasaviah Yasha, Bevinahalli N. Nandeesh, Arimappamagan Arivazhagan, Vani Santosh<br />
<em>Child’s Nervous System</em> (2017-11) <a href="https://doi.org/ggnpkn">https://doi.org/ggnpkn</a> <br />
DOI: <a href="https://doi.org/10.1007/s00381-017-3551-6">10.1007/s00381-017-3551-6</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28744687">28744687</a></p>
</div>
<div id="ref-1AM78Fkrd">
<p>114. <strong>Childhood Medulloblastoma and Other Central Nervous System Embryonal Tumors Treatment (PDQ®)–Health Professional Version - National Cancer Institute</strong> (2008-02-13) <a href="https://www.cancer.gov/types/brain/hp/child-cns-embryonal-treatment-pdq">https://www.cancer.gov/types/brain/hp/child-cns-embryonal-treatment-pdq</a></p>
</div>
<div id="ref-Cibjccyk">
<p>115. <strong>DNA Methylation Profiling for Diagnosing Undifferentiated Sarcoma with Capicua Transcriptional Receptor (CIC) Alterations</strong> <br />
Evelina Miele, Rita De Vito, Andrea Ciolfi, Lucia Pedace, Ida Russo, Maria Debora De Pasquale, Angela Di Giannatale, Alessandro Crocoli, Biagio De Angelis, Marco Tartaglia, … Giuseppe Maria Milano<br />
<em>International Journal of Molecular Sciences</em> (2020-03-06) <a href="https://doi.org/ggnn7x">https://doi.org/ggnn7x</a> <br />
DOI: <a href="https://doi.org/10.3390/ijms21051818">10.3390/ijms21051818</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32155762">32155762</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7084764">PMC7084764</a></p>
</div>
<div id="ref-7UxjGOxW">
<p>116. <strong>LIN28A immunoreactivity is a potent diagnostic marker of embryonal tumor with multilayered rosettes (ETMR)</strong> <br />
Andrey Korshunov, Marina Ryzhova, David T. W. Jones, Paul A. Northcott, Peter van Sluis, Richard Volckmann, Jan Koster, Rogier Versteeg, Cynthia Cowdrey, Arie Perry, … Marcel Kool<br />
<em>Acta Neuropathologica</em> (2012-12) <a href="https://doi.org/f4dxvc">https://doi.org/f4dxvc</a> <br />
DOI: <a href="https://doi.org/10.1007/s00401-012-1068-3">10.1007/s00401-012-1068-3</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23161096">23161096</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3508282">PMC3508282</a></p>
</div>
<div id="ref-DEDtXX7K">
<p>117. <strong>ITD assembler: an algorithm for internal tandem duplication discovery from short-read sequencing data</strong> <br />
Navin Rustagi, Oliver A Hampton, Jie Li, Liu Xi, Richard A. Gibbs, Sharon E. Plon, Marek Kimmel, David A. Wheeler<br />
<em>BMC Bioinformatics</em> (2016-12) <a href="https://doi.org/gmndpj">https://doi.org/gmndpj</a> <br />
DOI: <a href="https://doi.org/10.1186/s12859-016-1031-8">10.1186/s12859-016-1031-8</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27121965">27121965</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4847212">PMC4847212</a></p>
</div>
<div id="ref-ueHlof7U">
<p>118. <strong>Central Neurocytoma and Extraventricular Neurocytoma</strong> <br />
Carrie A. Mohila, Ronald A. Rauch, Adekunle M. Adesina<br />
<em>Atlas of Pediatric Brain Tumors</em> (2016) <a href="https://doi.org/gmndn7">https://doi.org/gmndn7</a> <br />
DOI: <a href="https://doi.org/10.1007/978-3-319-33432-5_20">10.1007/978-3-319-33432-5_20</a> · ISBN: <a href="https://worldcat.org/isbn/9783319334301">9783319334301</a></p>
</div>
<div id="ref-WOSiJ6V2">
<p>119. <strong>Papillary craniopharyngioma: a clinicopathological study of 48 cases</strong> <br />
Thomas B. Crotty, Bernd W. Scheithauer, William F. Young, Dudley H. Davis, Edward G. Shaw, Gary M. Miller, Peter C. Burger<br />
<em>Journal of Neurosurgery</em> (1995-08) <a href="https://doi.org/ccpj22">https://doi.org/ccpj22</a> <br />
DOI: <a href="https://doi.org/10.3171/jns.1995.83.2.0206">10.3171/jns.1995.83.2.0206</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/7616262">7616262</a></p>
</div>
<div id="ref-xnZIq0gr">
<p>120. <strong>The descriptive epidemiology of craniopharyngioma</strong> <br />
Greta R. Bunin, Tanya S. Surawicz, Philip A. Witman, Susan Preston-Martin, Faith Davis, Janet M. Bruner<br />
<em>Journal of Neurosurgery</em> (1998-10) <a href="https://doi.org/fh9xsh">https://doi.org/fh9xsh</a> <br />
DOI: <a href="https://doi.org/10.3171/jns.1998.89.4.0547">10.3171/jns.1998.89.4.0547</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/9761047">9761047</a></p>
</div>
<div id="ref-1WxziFyg">
<p>121. <strong>Pfam: Family: PK_Tyr_Ser-Thr (PF07714)</strong> <a href="https://pfam.xfam.org/family/PF07714">https://pfam.xfam.org/family/PF07714</a></p>
</div>
<div id="ref-p1ZTCLep">
<p>122. <a href="http://hgdownload.soe.ucsc.edu/goldenPath/hg38/database/pfamDesc.txt.gz">http://hgdownload.soe.ucsc.edu/goldenPath/hg38/database/pfamDesc.txt.gz</a></p>
</div>
<div id="ref-10jkUywit">
<p>123. <strong>Accelerating Discovery of Functional Mutant Alleles in Cancer</strong> <br />
Matthew T. Chang, Tripti Shrestha Bhattarai, Alison M. Schram, Craig M. Bielski, Mark T. A. Donoghue, Philip Jonsson, Debyani Chakravarty, Sarah Phillips, Cyriac Kandoth, Alexander Penson, … Barry S. Taylor<br />
<em>Cancer Discovery</em> (2018-02) <a href="https://doi.org/gf9twp">https://doi.org/gf9twp</a> <br />
DOI: <a href="https://doi.org/10.1158/2159-8290.cd-17-0321">10.1158/2159-8290.cd-17-0321</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29247016">29247016</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5809279">PMC5809279</a></p>
</div>
<div id="ref-tfWVBF8o">
<p>124. <strong>Identifying recurrent mutations in cancer reveals widespread lineage diversity and mutational specificity</strong> <br />
Matthew T Chang, Saurabh Asthana, Sizhi Paul Gao, Byron H Lee, Jocelyn S Chapman, Cyriac Kandoth, JianJiong Gao, Nicholas D Socci, David B Solit, Adam B Olshen, … Barry S Taylor<br />
<em>Nature Biotechnology</em> (2016-02) <a href="https://doi.org/gf7vxg">https://doi.org/gf7vxg</a> <br />
DOI: <a href="https://doi.org/10.1038/nbt.3391">10.1038/nbt.3391</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26619011">26619011</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4744099">PMC4744099</a></p>
</div>
<div id="ref-KxZPoysm">
<p>125. <strong>The functional domains in p53 family proteins exhibit both common and distinct properties</strong> <br />
KL Harms, X Chen<br />
<em>Cell Death &amp; Differentiation</em> (2006-06) <a href="https://doi.org/fwgrrt">https://doi.org/fwgrrt</a> <br />
DOI: <a href="https://doi.org/10.1038/sj.cdd.4401904">10.1038/sj.cdd.4401904</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16543939">16543939</a></p>
</div>
<div id="ref-1CnI07C7L">
<p>126. <strong>Inherited</strong> <br />
Tanya Guha, David Malkin<br />
<em>Cold Spring Harbor perspectives in medicine</em> (2017-04-03) <a href="https://www.ncbi.nlm.nih.gov/pubmed/28270529">https://www.ncbi.nlm.nih.gov/pubmed/28270529</a> <br />
DOI: <a href="https://doi.org/10.1101/cshperspect.a026187">10.1101/cshperspect.a026187</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28270529">28270529</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5378014">PMC5378014</a></p>
</div>
<div id="ref-pVqo7JQF">
<p>127. <strong>The Sequence Alignment/Map format and SAMtools.</strong> <br />
Heng Li, Bob Handsaker, Alec Wysoker, Tim Fennell, Jue Ruan, Nils Homer, Gabor Marth, Goncalo Abecasis, Richard Durbin, <br />
<em>Bioinformatics (Oxford, England)</em> (2009-06-08) <a href="https://www.ncbi.nlm.nih.gov/pubmed/19505943">https://www.ncbi.nlm.nih.gov/pubmed/19505943</a> <br />
DOI: <a href="https://doi.org/10.1093/bioinformatics/btp352">10.1093/bioinformatics/btp352</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19505943">19505943</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2723002">PMC2723002</a></p>
</div>
<div id="ref-16bqdaYmh">
<p>128. <strong>Inferring Telomerase Enzymatic Activity from Expression Data</strong> <br />
Nighat Noureen, Shaofang Wu, Yingli Lyu, Juechen Yang, WK Alfred Yung, Jonathan Gelfond, Xiaojing Wang, Dimpy Koul, Andrew Ludlow, Siyuan Zheng<br />
<em>Bioinformatics</em> (2020-05-26) <a href="https://doi.org/ggxfhq">https://doi.org/ggxfhq</a> <br />
DOI: <a href="https://doi.org/https://doi.org/10.1101/2020.05.21.109249">https://doi.org/10.1101/2020.05.21.109249</a></p>
</div>
<div id="ref-DbMaLwlv">
<p>129. <strong>d3b-center/OpenPBTA-workflows: Release v1.0.0</strong> <br />
Jo Lynne Rokita, Miguel Brown<br />
<em>Zenodo</em> (2022-03-16) <a href="https://doi.org/gppqwv">https://doi.org/gppqwv</a> <br />
DOI: <a href="https://doi.org/10.5281/zenodo.6363520">10.5281/zenodo.6363520</a></p>
</div>
</div>
<!-- default theme -->

<style>
    /* import google fonts */
    @import url("https://fonts.googleapis.com/css?family=Open+Sans:400,600,700");
    @import url("https://fonts.googleapis.com/css?family=Source+Code+Pro");

    /* -------------------------------------------------- */
    /* global */
    /* -------------------------------------------------- */

    /* all elements */
    * {
        /* force sans-serif font unless specified otherwise */
        font-family: "Open Sans", "Helvetica", sans-serif;

        /* prevent text inflation on some mobile browsers */
        -webkit-text-size-adjust: none !important;
        -moz-text-size-adjust: none !important;
        -o-text-size-adjust: none !important;
        text-size-adjust: none !important;
    }

    @media only screen {
        /* "page" element */
        body {
            position: relative;
            box-sizing: border-box;
            font-size: 12pt;
            line-height: 1.5;
            max-width: 8.5in;
            margin: 20px auto;
            padding: 40px;
            border-radius: 5px;
            border: solid 1px #bdbdbd;
            box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
            background: #ffffff;
        }
    }

    /* when on screen < 8.5in wide */
    @media only screen and (max-width: 8.5in) {
        /* "page" element */
        body {
            padding: 20px;
            margin: 0;
            border-radius: 0;
            border: none;
            box-shadow: 0 0 20px rgba(0, 0, 0, 0.05) inset;
            background: none;
        }
    }

    /* -------------------------------------------------- */
    /* headings */
    /* -------------------------------------------------- */

    /* all headings */
    h1,
    h2,
    h3,
    h4,
    h5,
    h6 {
        margin: 20px 0;
        padding: 0;
        font-weight: bold;
    }

    /* biggest heading */
    h1 {
        margin: 40px 0;
        text-align: center;
    }

    /* second biggest heading */
    h2 {
        margin-top: 30px;
        padding-bottom: 5px;
        border-bottom: solid 1px #bdbdbd;
    }

    /* heading font sizes */
    h1 {
        font-size: 2em;
    }
    h2 {
        font-size: 1.5em;
    }
    h3{
        font-size: 1.35em;
    }
    h4 {
        font-size: 1.25em;
    }
    h5 {
        font-size: 1.15em;
    }
    h6 {
        font-size: 1em;
    }

    /* -------------------------------------------------- */
    /* manuscript header */
    /* -------------------------------------------------- */

    /* manuscript title */
    header > h1 {
        margin: 0;
    }

    /* manuscript title caption text (ie "automatically generated on") */
    header + p {
        text-align: center;
        margin-top: 10px;
    }

    /* -------------------------------------------------- */
    /* text elements */
    /* -------------------------------------------------- */

    /* links */
    a {
        color: #2196f3;
        overflow-wrap: break-word;
    }

    /* normal links (not empty, not button link, not syntax highlighting link) */
    a:not(:empty):not(.button):not(.sourceLine) {
        padding-left: 1px;
        padding-right: 1px;
    }

    /* superscripts and subscripts */
    sub,
    sup {
        /* prevent from affecting line height */
        line-height: 0;
    }

    /* unordered and ordered lists*/
    ul,
    ol {
        padding-left: 20px;
    }

    /* class for styling text semibold */
    .semibold {
        font-weight: 600;
    }

    /* class for styling elements horizontally left aligned */
    .left {
        display: block;
        text-align: left;
        margin-left: auto;
        margin-right: 0;
        justify-content: left;
    }

    /* class for styling elements horizontally centered */
    .center {
        display: block;
        text-align: center;
        margin-left: auto;
        margin-right: auto;
        justify-content: center;
    }

    /* class for styling elements horizontally right aligned */
    .right {
        display: block;
        text-align: right;
        margin-left: 0;
        margin-right: auto;
        justify-content: right;
    }

    /* -------------------------------------------------- */
    /* section elements */
    /* -------------------------------------------------- */

    /* horizontal divider line */
    hr {
        border: none;
        height: 1px;
        background: #bdbdbd;
    }

    /* paragraphs, horizontal dividers, figures, tables, code */
    p,
    hr,
    figure,
    table,
    pre {
        /* treat all as "paragraphs", with consistent vertical margins */
        margin-top: 20px;
        margin-bottom: 20px;
    }

    /* -------------------------------------------------- */
    /* figures */
    /* -------------------------------------------------- */

    /* figure */
    figure {
        max-width: 100%;
        margin-left: auto;
        margin-right: auto;
    }

    /* figure caption */
    figcaption {
        padding: 0;
        padding-top: 10px;
    }

    /* figure image element */
    figure > img,
    figure > svg {
        max-width: 100%;
        display: block;
        margin-left: auto;
        margin-right: auto;
    }

    /* figure auto-number */
    img + figcaption > span:first-of-type,
    svg + figcaption > span:first-of-type {
        font-weight: bold;
        margin-right: 5px;
    }

    /* -------------------------------------------------- */
    /* tables */
    /* -------------------------------------------------- */

    /* table */
    table {
        border-collapse: collapse;
        border-spacing: 0;
        width: 100%;
        margin-left: auto;
        margin-right: auto;
    }

    /* table cells */
    th,
    td {
        border: solid 1px #bdbdbd;
        padding: 10px;
        /* squash table if too wide for page by forcing line breaks */
        overflow-wrap: break-word;
        word-break: break-word;
    }

    /* header row and even rows */
    th,
    tr:nth-child(2n) {
        background-color: #fafafa;
    }

    /* odd rows */
    tr:nth-child(2n + 1) {
        background-color: #ffffff;
    }

    /* table caption */
    caption {
        text-align: left;
        padding: 0;
        padding-bottom: 10px;
    }

    /* table auto-number */
    table > caption > span:first-of-type,
    div.table_wrapper > table > caption > span:first-of-type {
        font-weight: bold;
        margin-right: 5px;
    }

    /* -------------------------------------------------- */
    /* code */
    /* -------------------------------------------------- */

    /* multi-line code block */
    pre {
        padding: 10px;
        background-color: #eeeeee;
        color: #000000;
        border-radius: 5px;
        break-inside: avoid;
        text-align: left;
    }

    /* inline code, ie code within normal text */
    :not(pre) > code {
        padding: 0 4px;
        background-color: #eeeeee;
        color: #000000;
        border-radius: 5px;
    }

    /* code text */
    /* apply all children, to reach syntax highlighting sub-elements */
    code,
    code * {
        /* force monospace font */
        font-family: "Source Code Pro", "Courier New", monospace;
    }

    /* -------------------------------------------------- */
    /* quotes */
    /* -------------------------------------------------- */

    /* quoted text */
    blockquote {
        margin: 0;
        padding: 0;
        border-left: 4px solid #bdbdbd;
        padding-left: 16px;
        break-inside: avoid;
    }

    /* -------------------------------------------------- */
    /* banners */
    /* -------------------------------------------------- */

    /* info banners */
    .banner {
        box-sizing: border-box;
        display: block;
        position: relative;
        width: 100%;
        margin-top: 20px;
        margin-bottom: 20px;
        padding: 20px;
        text-align: center;
    }

    /* paragraph in banner */
    .banner > p {
        margin: 0;
    }

    /* -------------------------------------------------- */
    /* highlight colors */
    /* -------------------------------------------------- */

    .white {
        background: #ffffff;
    }
    .lightgrey {
        background: #eeeeee;
    }
    .grey {
        background: #757575;
    }
    .darkgrey {
        background: #424242;
    }
    .black {
        background: #000000;
    }
    .lightred {
        background: #ffcdd2;
    }
    .lightyellow {
        background: #ffecb3;
    }
    .lightgreen {
        background: #dcedc8;
    }
    .lightblue {
        background: #e3f2fd;
    }
    .lightpurple {
        background: #f3e5f5;
    }
    .red {
        background: #f44336;
    }
    .orange {
        background: #ff9800;
    }
    .yellow {
        background: #ffeb3b;
    }
    .green {
        background: #4caf50;
    }
    .blue {
        background: #2196f3;
    }
    .purple {
        background: #9c27b0;
    }
    .white,
    .lightgrey,
    .lightred,
    .lightyellow,
    .lightgreen,
    .lightblue,
    .lightpurple,
    .orange,
    .yellow,
    .white a,
    .lightgrey a,
    .lightred a,
    .lightyellow a,
    .lightgreen a,
    .lightblue a,
    .lightpurple a,
    .orange a,
    .yellow a {
        color: #000000;
    }
    .grey,
    .darkgrey,
    .black,
    .red,
    .green,
    .blue,
    .purple,
    .grey a,
    .darkgrey a,
    .black a,
    .red a,
    .green a,
    .blue a,
    .purple a {
        color: #ffffff;
    }

    /* -------------------------------------------------- */
    /* buttons */
    /* -------------------------------------------------- */

    /* class for styling links like buttons */
    .button {
        display: inline-flex;
        justify-content: center;
        align-items: center;
        margin: 5px;
        padding: 10px 20px;
        font-size: 0.75em;
        font-weight: 600;
        text-transform: uppercase;
        text-decoration: none;
        letter-spacing: 1px;
        background: none;
        color: #2196f3;
        border: solid 1px #bdbdbd;
        border-radius: 5px;
    }

    /* buttons when hovered */
    .button:hover:not([disabled]),
    .icon_button:hover:not([disabled]) {
        cursor: pointer;
        background: #f5f5f5;
    }

    /* buttons when disabled */
    .button[disabled],
    .icon_button[disabled] {
        opacity: 0.35;
        pointer-events: none;
    }

    /* class for styling buttons containg only single icon */
    .icon_button {
        display: inline-flex;
        justify-content: center;
        align-items: center;
        text-decoration: none;
        margin: 0;
        padding: 0;
        background: none;
        border-radius: 5px;
        border: none;
        width: 20px;
        height: 20px;
        min-width: 20px;
        min-height: 20px;
    }

    /* icon button inner svg image */
    .icon_button > svg {
        height: 16px;
    }

    /* -------------------------------------------------- */
    /* icons */
    /* -------------------------------------------------- */

    /* class for styling icons inline with text */
    .inline_icon {
        height: 1em;
        position: relative;
        top: 0.125em;
    }

    /* -------------------------------------------------- */
    /* print control */
    /* -------------------------------------------------- */

    @media print {
        @page {
            /* suggested printing margin */
            margin: 0.5in;
        }

        /* document and "page" elements */
        html, body {
            margin: 0;
            padding: 0;
            width: 100%;
            height: 100%;
        }

        /* "page" element */
        body {
            font-size: 11pt !important;
            line-height: 1.35;
        }

        /* all headings */
        h1,
        h2,
        h3,
        h4,
        h5,
        h6 {
            margin: 15px 0;
        }

        /* figures and tables */
        figure, table {
            font-size: 0.85em;
        }

        /* table cells */
        th,
        td {
            padding: 5px;
        }

        /* shrink font awesome icons */
        i.fas,
        i.fab,
        i.far,
        i.fal {
            transform: scale(0.85);
        }

        /* decrease banner margins */
        .banner {
            margin-top: 15px;
            margin-bottom: 15px;
            padding: 15px;
        }

        /* class for centering an element vertically on its own page */
        .page_center {
            margin: auto;
            width: 100%;
            height: 100%;
            display: flex;
            align-items: center;
            vertical-align: middle;
            break-before: page;
            break-after: page;
        }

        /* always insert a page break before the element */
        .page_break_before {
            break-before: page;
        }

        /* always insert a page break after the element */
        .page_break_after {
            break-after: page;
        }

        /* avoid page break before the element */
        .page_break_before_avoid {
            break-before: avoid;
        }

        /* avoid page break after the element */
        .page_break_after_avoid {
            break-after: avoid;
        }

        /* avoid page break inside the element */
        .page_break_inside_avoid {
            break-inside: avoid;
        }
    }

    /* -------------------------------------------------- */
    /* override pandoc css quirks */
    /* -------------------------------------------------- */

    .sourceCode {
        /* prevent unsightly overflow in wide code blocks */
        overflow: auto !important;
    }

    div.sourceCode {
        /* prevent background fill on top-most code block  container */
        background: none !important;
    }

    .sourceCode * {
        /* force consistent line spacing */
        line-height: 1.5 !important;
    }

    div.sourceCode {
        /* style code block margins same as <pre> element */
        margin-top: 20px;
        margin-bottom: 20px;
    }

    /* -------------------------------------------------- */
    /* tablenos */
    /* -------------------------------------------------- */

    /* tablenos wrapper */
    .tablenos {
        /* show scrollbar on tables if necessary to prevent overflow */
        width: 100%;
        margin: 20px 0;
    }

    .tablenos > table {
        /* move margins from table to table_wrapper to allow margin collapsing */
        margin: 0;
    }

    @media only screen {
        /* tablenos wrapper */
        .tablenos {
            /* show scrollbar on tables if necessary to prevent overflow */
            overflow-x: auto !important;
        }

        .tablenos th,
        .tablenos td {
            overflow-wrap: unset !important;
            word-break: unset !important;
        }

        /* table in wrapper */
        .tablenos table,
        .tablenos table * {
            /* don't break table words */
            overflow-wrap: normal !important;
        }
    }

    /* -------------------------------------------------- */
    /* mathjax */
    /* -------------------------------------------------- */

    /* mathjax containers */
    .math.display > span:not(.MathJax_Preview) {
        /* turn inline element (no dimensions) into block (allows fixed width and thus scrolling) */
        display: flex !important;
        overflow-x: auto !important;
        overflow-y: hidden !important;
        justify-content: center;
        align-items: center;
        margin: 0 !important;
    }

    /* right click menu */
    .MathJax_Menu {
        border-radius: 5px !important;
        border: solid 1px #bdbdbd !important;
        box-shadow: none !important;
    }

    /* equation auto-number */
    span[id^="eq:"] > span.math.display + span {
        font-weight: 600;
    }

    /* equation */
    span[id^="eq:"] > span.math.display > span {
        /* nudge to make room for equation auto-number and anchor */
        margin-right: 60px !important;
    }

    /* -------------------------------------------------- */
    /* anchors plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* anchor button */
        .anchor {
            opacity: 0;
            margin-left: 5px;
        }

        /* anchor buttons within <h2>'s */
        h2 .anchor {
            margin-left: 10px;
        }

        /* anchor buttons when hovered/focused and anything containing an anchor button when hovered */
        *:hover > .anchor,
        .anchor:hover,
        .anchor:focus {
            opacity: 1;
        }

        /* anchor button when hovered */
        .anchor:hover {
            cursor: pointer;
        }
    }

    /* always show anchor button on devices with no mouse/hover ability */
    @media (hover: none) {
        .anchor {
            opacity: 1;
        }
    }

    /* always hide anchor button on print */
    @media only print {
        .anchor {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* accordion plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* accordion arrow button */
        .accordion_arrow {
            margin-right: 10px;
        }

        /* arrow icon when <h2> data-collapsed attribute true */
        h2[data-collapsed="true"] > .accordion_arrow > svg {
            transform: rotate(-90deg);
        }

        /* all elements (except <h2>'s) when data-collapsed attribute true */
        *:not(h2)[data-collapsed="true"] {
            display: none;
        }

        /* accordion arrow button when hovered and <h2>'s when hovered */
        .accordion_arrow:hover,
        h2[data-collapsed="true"]:hover,
        h2[data-collapsed="false"]:hover {
            cursor: pointer;
        }
    }

    /* always hide accordion arrow button on print */
    @media only print {
        .accordion_arrow {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* tooltips plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* tooltip container */
        #tooltip {
            position: absolute;
            width: 50%;
            min-width: 240px;
            max-width: 75%;
            z-index: 1;
        }

        /* tooltip content */
        #tooltip_content {
            margin-bottom: 5px;
            padding: 20px;
            border-radius: 5px;
            border: solid 1px #bdbdbd;
            box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
            background: #ffffff;
            overflow-wrap: break-word;
        }

        /* tooltip copy of paragraphs and figures */
        #tooltip_content > p,
        #tooltip_content > figure {
            margin: 0;
            max-height: 320px;
            overflow-y: auto;
        }

        /* tooltip copy of <img> */
        #tooltip_content > figure > img,
        #tooltip_content > figure > svg {
            max-height: 260px;
        }

        /* navigation bar */
        #tooltip_nav_bar {
            margin-top: 10px;
            text-align: center;
        }

        /* navigation bar previous/next buton */
        #tooltip_nav_bar > .icon_button {
            position: relative;
            top: 3px;
        }

        /* navigation bar previous button */
        #tooltip_nav_bar > .icon_button:first-of-type {
            margin-right: 5px;
        }

        /* navigation bar next button */
        #tooltip_nav_bar > .icon_button:last-of-type {
            margin-left: 5px;
        }
    }

    /* always hide tooltip on print */
    @media only print {
        #tooltip {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* jump to first plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* jump button */
        .jump_arrow {
            position: relative;
            top: 0.125em;
            margin-right: 5px;
        }
    }

    /* always hide jump button on print */
    @media only print {
        .jump_arrow {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* link highlight plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* anything with data-highlighted attribute true */
        [data-highlighted="true"] {
            background: #ffeb3b;
        }

        /* anything with data-selected attribute true */
        [data-selected="true"] {
            background: #ff8a65 !important;
        }

        /* animation definition for glow */
        @keyframes highlight_glow {
            0% {
                background: none;
            }
            10% {
                background: #bbdefb;
            }
            100% {
                background: none;
            }
        }

        /* anything with data-glow attribute true */
        [data-glow="true"] {
            animation: highlight_glow 2s;
        }
    }

    /* -------------------------------------------------- */
    /* table of contents plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* toc panel */
        #toc_panel {
            box-sizing: border-box;
            position: fixed;
            top: 0;
            left: 0;
            background: #ffffff;
            box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
            z-index: 2;
        }

        /* toc panel when closed */
        #toc_panel[data-open="false"] {
            min-width: 60px;
            width: 60px;
            height: 60px;
            border-right: solid 1px #bdbdbd;
            border-bottom: solid 1px #bdbdbd;
        }

        /* toc panel when open */
        #toc_panel[data-open="true"] {
            min-width: 260px;
            max-width: 480px;
            /* keep panel edge consistent distance away from "page" edge */
            width: calc(((100vw - 8.5in) / 2) - 30px - 40px);
            bottom: 0;
            border-right: solid 1px #bdbdbd;
        }

        /* toc panel header */
        #toc_header {
            box-sizing: border-box;
            display: flex;
            flex-direction: row;
            align-items: center;
            height: 60px;
            margin: 0;
            padding: 20px;
        }

        /* toc panel header when hovered */
        #toc_header:hover {
            cursor: pointer;
        }

        /* toc panel header when panel open */
        #toc_panel[data-open="true"] > #toc_header {
            border-bottom: solid 1px #bdbdbd;
        }

        /* toc open/close header button */
        #toc_button {
            margin-right: 20px;
        }

        /* hide toc list and header text when closed */
        #toc_panel[data-open="false"] > #toc_header > *:not(#toc_button),
        #toc_panel[data-open="false"] > #toc_list {
            display: none;
        }

        /* toc list of entries */
        #toc_list {
            box-sizing: border-box;
            width: 100%;
            padding: 20px;
            position: absolute;
            top: calc(60px + 1px);
            bottom: 0;
            overflow: auto;
        }

        /* toc entry, link to section in document */
        .toc_link {
            display: block;
            padding: 5px;
            position: relative;
            font-weight: 600;
            text-decoration: none;
        }

        /* toc entry when hovered or when "viewed" */
        .toc_link:hover,
        .toc_link[data-viewing="true"] {
            background: #f5f5f5;
        }

        /* toc entry, level 1 indentation */
        .toc_link[data-level="1"] {
            margin-left: 0;
        }

        /* toc entry, level 2 indentation */
        .toc_link[data-level="2"] {
            margin-left: 20px;
        }

        /* toc entry, level 3 indentation */
        .toc_link[data-level="3"] {
            margin-left: 40px;
        }

        /* toc entry, level 4 indentation */
        .toc_link[data-level="4"] {
            margin-left: 60px;
        }

        /* toc entry bullets */
        #toc_panel[data-bullets="true"] .toc_link[data-level]:before {
            position: absolute;
            left: -15px;
            top: -1px;
            font-size: 1.5em;
        }

        /* toc entry, level 2 bullet */
        #toc_panel[data-bullets="true"] .toc_link[data-level="2"]:before {
            content: "\2022";
        }

        /* toc entry, level 3 bullet */
        #toc_panel[data-bullets="true"] .toc_link[data-level="3"]:before {
            content: "\25AB";
        }

        /* toc entry, level 4 bullet */
        #toc_panel[data-bullets="true"] .toc_link[data-level="4"]:before {
            content: "-";
        }
    }

    /* when on screen < 8.5in wide */
    @media only screen and (max-width: 8.5in) {
        /* push <body> ("page") element down to make room for toc icon */
        .toc_body_nudge {
            padding-top: 60px;
        }

        /* toc icon when panel closed and not hovered */
        #toc_panel[data-open="false"]:not(:hover) {
            background: rgba(255, 255, 255, 0.75);
        }
    }

    /* always hide toc panel on print */
    @media only print {
        #toc_panel {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* lightbox plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* regular <img> in document when hovered */
        img.lightbox_document_img:hover {
            cursor: pointer;
        }

        .body_no_scroll {
            overflow: hidden !important;
        }

        /* screen overlay */
        #lightbox_overlay {
            display: flex;
            flex-direction: column;
            position: fixed;
            left: 0;
            top: 0;
            right: 0;
            bottom: 0;
            background: rgba(0, 0, 0, 0.75);
            z-index: 3;
        }

        /* middle area containing lightbox image */
        #lightbox_image_container {
            flex-grow: 1;
            display: flex;
            justify-content: center;
            align-items: center;
            overflow: hidden;
            position: relative;
            padding: 20px;
        }

        /* bottom area containing caption */
        #lightbox_bottom_container {
            display: flex;
            justify-content: center;
            align-items: center;
            height: 100px;
            min-height: 100px;
            max-height: 100px;
            background: rgba(0, 0, 0, 0.5);
        }

        /* image number info text box */
        #lightbox_number_info {
            position: absolute;
            color: #ffffff;
            font-weight: 600;
            left: 2px;
            top: 0;
            z-index: 4;
        }

        /* zoom info text box */
        #lightbox_zoom_info {
            position: absolute;
            color: #ffffff;
            font-weight: 600;
            right: 2px;
            top: 0;
            z-index: 4;
        }

        /* copy of image caption */
        #lightbox_caption {
            box-sizing: border-box;
            display: inline-block;
            width: 100%;
            max-height: 100%;
            padding: 10px 0;
            text-align: center;
            overflow-y: auto;
            color: #ffffff;
        }

        /* navigation previous/next button */
        .lightbox_button {
            width: 100px;
            height: 100%;
            min-width: 100px;
            min-height: 100%;
            color: #ffffff;
        }

        /* navigation previous/next button when hovered */
        .lightbox_button:hover {
            background: none !important;
        }

        /* navigation button icon */
        .lightbox_button > svg {
            height: 25px;
        }

        /* figure auto-number */
        #lightbox_caption > span:first-of-type {
            font-weight: bold;
            margin-right: 5px;
        }

        /* lightbox image when hovered */
        #lightbox_img:hover {
            cursor: grab;
        }

        /* lightbox image when grabbed */
        #lightbox_img:active {
            cursor: grabbing;
        }
    }

    /* when on screen < 480px wide */
    @media only screen and (max-width: 480px) {
        /* make navigation buttons skinnier on small screens to make more room for caption text */
        .lightbox_button {
            width: 50px;
            min-width: 50px;
        }
    }

    /* always hide lightbox on print */
    @media only print {
        #lightbox_overlay {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* hypothesis (annotations) plugin */
    /* -------------------------------------------------- */

    /* hypothesis activation button */
    #hypothesis_button {
        box-sizing: border-box;
        position: fixed;
        top: 0;
        right: 0;
        width: 60px;
        height: 60px;
        background: #ffffff;
        border-radius: 0;
        border-left: solid 1px #bdbdbd;
        border-bottom: solid 1px #bdbdbd;
        box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
        z-index: 2;
    }

    /* hypothesis button svg */
    #hypothesis_button > svg {
        position: relative;
        top: -4px;
    }

    /* hypothesis annotation count */
    #hypothesis_count {
        position: absolute;
        left: 0;
        right: 0;
        bottom: 5px;
    }

    /* side panel */
    .annotator-frame {
        width: 280px !important;
    }

    /* match highlight color to rest of theme */
    .annotator-highlights-always-on .annotator-hl {
        background-color: #ffeb3b !important;
    }

    /* match focused color to rest of theme */
    .annotator-hl.annotator-hl-focused {
        background-color: #ff8a65 !important;
    }

    /* match bucket bar color to rest of theme */
    .annotator-bucket-bar {
        background: #f5f5f5 !important;
    }

    /* always hide button, toolbar, and tooltip on print */
    @media only print {
        #hypothesis_button {
            display: none;
        }

        .annotator-frame {
            display: none !important;
        }

        hypothesis-adder {
            display: none !important;
        }
    }
</style>
<!-- anchors plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin adds an anchor next to each of a certain type
        // of element that provides a human-readable url to that specific
        // item/position in the document (eg "manuscript.html#abstract"). It
        // also makes it such that scrolling out of view of a target removes
        // its identifier from the url.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'anchors';

        // default plugin options
        const options = {
            // which types of elements to add anchors next to, in
            // "document.querySelector" format
            typesQuery: 'h1, h2, h3, [id^="fig:"], [id^="tbl:"], [id^="eq:"]',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            // add anchor to each element of specified types
            const elements = document.querySelectorAll(options.typesQuery);
            for (const element of elements)
                addAnchor(element);

            // attach scroll listener to window
            window.addEventListener('scroll', onScroll);
        }

        // when window is scrolled
        function onScroll() {
            // if url has hash and user has scrolled out of view of hash
            // target, remove hash from url
            const tolerance = 100;
            const target = getHashTarget();
            if (target) {
                if (
                    target.getBoundingClientRect().top >
                        window.innerHeight + tolerance ||
                    target.getBoundingClientRect().bottom < 0 - tolerance
                )
                    history.pushState(null, null, ' ');
            }
        }

        // add anchor to element
        function addAnchor(element) {
            let addTo; // element to add anchor button to

            // if figure or table, modify withId and addTo to get expected
            // elements
            if (element.id.indexOf('fig:') === 0) {
                addTo = element.querySelector('figcaption');
            } else if (element.id.indexOf('tbl:') === 0) {
                addTo = element.querySelector('caption');
            } else if (element.id.indexOf('eq:') === 0) {
                addTo = element.querySelector('.eqnos-number');
            }

            addTo = addTo || element;
            const id = element.id || null;

            // do not add anchor if element doesn't have assigned id.
            // id is generated by pandoc and is assumed to be unique and
            // human-readable
            if (!id)
                return;

            // create anchor button
            const anchor = document.createElement('a');
            anchor.innerHTML = document.querySelector('.icon_link').innerHTML;
            anchor.title = 'Link to this part of the document';
            anchor.classList.add('icon_button', 'anchor');
            anchor.dataset.ignore = 'true';
            anchor.href = '#' + id;
            addTo.appendChild(anchor);
        }

        // get element that is target of link or url hash
        function getHashTarget() {
            const hash = window.location.hash;
            const id = hash.slice(1);
            let target = document.querySelector('[id="' + id + '"]');
            if (!target)
                return;

            // if figure or table, modify target to get expected element
            if (id.indexOf('fig:') === 0)
                target = target.querySelector('figure');
            if (id.indexOf('tbl:') === 0)
                target = target.querySelector('table');

            return target;
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>

<!-- link icon -->

<template class="icon_link">
    <!-- modified from: https://fontawesome.com/icons/link -->
    <svg width="16" height="16" viewBox="0 0 512 512">
        <path
            fill="currentColor"
            d="M326.612 185.391c59.747 59.809 58.927 155.698.36 214.59-.11.12-.24.25-.36.37l-67.2 67.2c-59.27 59.27-155.699 59.262-214.96 0-59.27-59.26-59.27-155.7 0-214.96l37.106-37.106c9.84-9.84 26.786-3.3 27.294 10.606.648 17.722 3.826 35.527 9.69 52.721 1.986 5.822.567 12.262-3.783 16.612l-13.087 13.087c-28.026 28.026-28.905 73.66-1.155 101.96 28.024 28.579 74.086 28.749 102.325.51l67.2-67.19c28.191-28.191 28.073-73.757 0-101.83-3.701-3.694-7.429-6.564-10.341-8.569a16.037 16.037 0 0 1-6.947-12.606c-.396-10.567 3.348-21.456 11.698-29.806l21.054-21.055c5.521-5.521 14.182-6.199 20.584-1.731a152.482 152.482 0 0 1 20.522 17.197zM467.547 44.449c-59.261-59.262-155.69-59.27-214.96 0l-67.2 67.2c-.12.12-.25.25-.36.37-58.566 58.892-59.387 154.781.36 214.59a152.454 152.454 0 0 0 20.521 17.196c6.402 4.468 15.064 3.789 20.584-1.731l21.054-21.055c8.35-8.35 12.094-19.239 11.698-29.806a16.037 16.037 0 0 0-6.947-12.606c-2.912-2.005-6.64-4.875-10.341-8.569-28.073-28.073-28.191-73.639 0-101.83l67.2-67.19c28.239-28.239 74.3-28.069 102.325.51 27.75 28.3 26.872 73.934-1.155 101.96l-13.087 13.087c-4.35 4.35-5.769 10.79-3.783 16.612 5.864 17.194 9.042 34.999 9.69 52.721.509 13.906 17.454 20.446 27.294 10.606l37.106-37.106c59.271-59.259 59.271-155.699.001-214.959z"
        ></path>
    </svg>
</template>
<!-- accordion plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin allows sections of content under <h2> headings
        // to be collapsible.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'accordion';

        // default plugin options
        const options = {
            // whether to always start expanded ('false'), always start
            // collapsed ('true'), or start collapsed when screen small ('auto')
            startCollapsed: 'auto',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            // run through each <h2> heading
            const headings = document.querySelectorAll('h2');
            for (const heading of headings) {
                addArrow(heading);

                // start expanded/collapsed based on option
                if (
                    options.startCollapsed === 'true' ||
                    (options.startCollapsed === 'auto' && isSmallScreen())
                )
                    collapseHeading(heading);
                else
                    expandHeading(heading);
            }

            // attach hash change listener to window
            window.addEventListener('hashchange', onHashChange);
        }

        // when hash (eg manuscript.html#introduction) changes
        function onHashChange() {
            const target = getHashTarget();
            if (target)
                goToElement(target);
        }

        // add arrow to heading
        function addArrow(heading) {
            // add arrow button
            const arrow = document.createElement('button');
            arrow.innerHTML = document.querySelector(
                '.icon_angle_down'
            ).innerHTML;
            arrow.classList.add('icon_button', 'accordion_arrow');
            heading.insertBefore(arrow, heading.firstChild);

            // attach click listener to heading and button
            heading.addEventListener('click', onHeadingClick);
            arrow.addEventListener('click', onArrowClick);
        }

        // determine if on mobile-like device with small screen
        function isSmallScreen() {
            return Math.min(window.innerWidth, window.innerHeight) < 480;
        }

        // scroll to and focus element
        function goToElement(element, offset) {
            // expand accordion section if collapsed
            expandElement(element);
            const y =
                getRectInView(element).top -
                getRectInView(document.documentElement).top -
                (offset || 0);
            // trigger any function listening for "onscroll" event
            window.dispatchEvent(new Event('scroll'));
            window.scrollTo(0, y);
            document.activeElement.blur();
            element.focus();
        }

        // get element that is target of hash
        function getHashTarget(link) {
            const hash = link ? link.hash : window.location.hash;
            const id = hash.slice(1);
            let target = document.querySelector('[id="' + id + '"]');
            if (!target)
                return;

            // if figure or table, modify target to get expected element
            if (id.indexOf('fig:') === 0)
                target = target.querySelector('figure');
            if (id.indexOf('tbl:') === 0)
                target = target.querySelector('table');

            return target;
        }

        // when <h2> heading is clicked
        function onHeadingClick(event) {
            // only collapse if <h2> itself is target of click (eg, user did
            // not click on anchor within <h2>)
            if (event.target === this)
                toggleCollapse(this);
        }

        // when arrow button is clicked
        function onArrowClick() {
            toggleCollapse(this.parentNode);
        }

        // collapse section if expanded, expand if collapsed
        function toggleCollapse(heading) {
            if (heading.dataset.collapsed === 'false')
                collapseHeading(heading);
            else
                expandHeading(heading);
        }

        // elements to exclude from collapse, such as table of contents panel,
        // hypothesis panel, etc
        const exclude = '#toc_panel, div.annotator-frame, #lightbox_overlay';

        // collapse section
        function collapseHeading(heading) {
            heading.setAttribute('data-collapsed', 'true');
            const children = getChildren(heading);
            for (const child of children)
                child.setAttribute('data-collapsed', 'true');
        }

        // expand section
        function expandHeading(heading) {
            heading.setAttribute('data-collapsed', 'false');
            const children = getChildren(heading);
            for (const child of children)
                child.setAttribute('data-collapsed', 'false');
        }

        // get list of elements between this <h2> and next <h2> or <h1>
        // ("children" of the <h2> section)
        function getChildren(heading) {
            return nextUntil(heading, 'h2, h1', exclude);
        }

        // get position/dimensions of element or viewport
        function getRectInView(element) {
            let rect = {};
            rect.left = 0;
            rect.top = 0;
            rect.right = document.documentElement.clientWidth;
            rect.bottom = document.documentElement.clientHeight;
            let style = {};

            if (element instanceof HTMLElement) {
                rect = element.getBoundingClientRect();
                style = window.getComputedStyle(element);
            }

            const margin = {};
            margin.left = parseFloat(style.marginLeftWidth) || 0;
            margin.top = parseFloat(style.marginTopWidth) || 0;
            margin.right = parseFloat(style.marginRightWidth) || 0;
            margin.bottom = parseFloat(style.marginBottomWidth) || 0;

            const border = {};
            border.left = parseFloat(style.borderLeftWidth) || 0;
            border.top = parseFloat(style.borderTopWidth) || 0;
            border.right = parseFloat(style.borderRightWidth) || 0;
            border.bottom = parseFloat(style.borderBottomWidth) || 0;

            const newRect = {};
            newRect.left = rect.left + margin.left + border.left;
            newRect.top = rect.top + margin.top + border.top;
            newRect.right = rect.right + margin.right + border.right;
            newRect.bottom = rect.bottom + margin.bottom + border.bottom;
            newRect.width = newRect.right - newRect.left;
            newRect.height = newRect.bottom - newRect.top;

            return newRect;
        }

        // get list of elements after a start element up to element matching
        // query
        function nextUntil(element, query, exclude) {
            const elements = [];
            while (element = element.nextElementSibling, element) {
                if (element.matches(query))
                    break;
                if (!element.matches(exclude))
                    elements.push(element);
            }
            return elements;
        }

        // get closest element before specified element that matches query
        function firstBefore(element, query) {
            while (
                element &&
                element !== document.body &&
                !element.matches(query)
            )
                element = element.previousElementSibling || element.parentNode;

            return element;
        }

        // check if element is part of collapsed heading
        function isCollapsed(element) {
            while (element && element !== document.body) {
                if (element.dataset.collapsed === 'true')
                    return true;
                element = element.parentNode;
            }
            return false;
        }

        // expand heading containing element if necesary
        function expandElement(element) {
            if (isCollapsed(element)) {
                const heading = firstBefore(element, 'h2');
                if (heading)
                    heading.click();
            }
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>

<!-- angle down icon -->

<template class="icon_angle_down">
    <!-- modified from: https://fontawesome.com/icons/angle-down -->
    <svg width="16" height="16" viewBox="0 0 448 512">
        <path
            fill="currentColor"
            d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"
        ></path>
    </svg>
</template>
<!-- tooltips plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin makes it such that when the user hovers or
        // focuses a link to a citation or figure, a tooltip appears with a
        // preview of the reference content, along with arrows to navigate
        // between instances of the same reference in the document.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'tooltips';

        // default plugin options
        const options = {
            // whether user must click off to close tooltip instead of just
            // un-hovering
            clickClose: 'false',
            // delay (in ms) between opening and closing tooltip
            delay: '100',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            const links = getLinks();
            for (const link of links) {
                // attach hover and focus listeners to link
                link.addEventListener('mouseover', onLinkHover);
                link.addEventListener('mouseleave', onLinkUnhover);
                link.addEventListener('focus', onLinkFocus);
                link.addEventListener('touchend', onLinkTouch);
            }

            // attach mouse, key, and resize listeners to window
            window.addEventListener('mousedown', onClick);
            window.addEventListener('touchstart', onClick);
            window.addEventListener('keyup', onKeyUp);
            window.addEventListener('resize', onResize);
        }

        // when link is hovered
        function onLinkHover() {
            // function to open tooltip
            const delayOpenTooltip = function() {
                openTooltip(this);
            }.bind(this);

            // run open function after delay
            this.openTooltipTimer = window.setTimeout(
                delayOpenTooltip,
                options.delay
            );
        }

        // when mouse leaves link
        function onLinkUnhover() {
            // cancel opening tooltip
            window.clearTimeout(this.openTooltipTimer);

            // don't close on unhover if option specifies
            if (options.clickClose === 'true')
                return;

            // function to close tooltip
            const delayCloseTooltip = function() {
                // if tooltip open and if mouse isn't over tooltip, close
                const tooltip = document.getElementById('tooltip');
                if (tooltip && !tooltip.matches(':hover'))
                    closeTooltip();
            };

            // run close function after delay
            this.closeTooltipTimer = window.setTimeout(
                delayCloseTooltip,
                options.delay
            );
        }

        // when link is focused (tabbed to)
        function onLinkFocus(event) {
            openTooltip(this);
        }

        // when link is touched on touch screen
        function onLinkTouch(event) {
            // attempt to force hover state on first tap always, and trigger
            // regular link click (and navigation) on second tap
            if (event.target === document.activeElement)
                event.target.click();
            else {
                document.activeElement.blur();
                event.target.focus();
            }
            if (event.cancelable)
                event.preventDefault();
            event.stopPropagation();
            return false;
        }

        // when mouse is clicked anywhere in window
        function onClick(event) {
            closeTooltip();
        }

        // when key pressed
        function onKeyUp(event) {
            if (!event || !event.key)
                return;

            switch (event.key) {
                // trigger click of prev button
                case 'ArrowLeft':
                    const prevButton = document.getElementById(
                        'tooltip_prev_button'
                    );
                    if (prevButton)
                        prevButton.click();
                    break;
                // trigger click of next button
                case 'ArrowRight':
                    const nextButton = document.getElementById(
                        'tooltip_next_button'
                    );
                    if (nextButton)
                        nextButton.click();
                    break;
                // close on esc
                case 'Escape':
                    closeTooltip();
                    break;
            }
        }

        // when window is resized or zoomed
        function onResize() {
            closeTooltip();
        }

        // get all links of types we wish to handle
        function getLinks() {
            const queries = [];
            // exclude buttons, anchor links, toc links, etc
            const exclude =
                ':not(.button):not(.icon_button):not(.anchor):not(.toc_link)';
            queries.push('a[href^="#ref-"]' + exclude); // citation links
            queries.push('a[href^="#fig:"]' + exclude); // figure links
            const query = queries.join(', ');
            return document.querySelectorAll(query);
        }

        // get links with same target, get index of link in set, get total
        // same links
        function getSameLinks(link) {
            const sameLinks = [];
            const links = getLinks();
            for (const otherLink of links) {
                if (
                    otherLink.getAttribute('href') === link.getAttribute('href')
                )
                    sameLinks.push(otherLink);
            }

            return {
                elements: sameLinks,
                index: sameLinks.indexOf(link),
                total: sameLinks.length
            };
        }

        // open tooltip
        function openTooltip(link) {
            // delete tooltip if it exists, start fresh
            closeTooltip();

            // make tooltip element
            const tooltip = makeTooltip(link);

            // if source couldn't be found and tooltip not made, exit
            if (!tooltip)
                return;

            // make navbar elements
            const navBar = makeNavBar(link);
            if (navBar)
                tooltip.firstElementChild.appendChild(navBar);

            // attach tooltip to page
            document.body.appendChild(tooltip);

            // position tooltip
            const position = function() {
                positionTooltip(link);
            };
            position();

            // if tooltip contains images, position again after they've loaded
            const imgs = tooltip.querySelectorAll('img');
            for (const img of imgs)
                img.addEventListener('load', position);
        }

        // close (delete) tooltip
        function closeTooltip() {
            const tooltip = document.getElementById('tooltip');
            if (tooltip)
                tooltip.remove();
        }

        // make tooltip
        function makeTooltip(link) {
            // get target element that link points to
            const source = getSource(link);

            // if source can't be found, exit
            if (!source)
                return;

            // create new tooltip
            const tooltip = document.createElement('div');
            tooltip.id = 'tooltip';
            const tooltipContent = document.createElement('div');
            tooltipContent.id = 'tooltip_content';
            tooltip.appendChild(tooltipContent);

            // make copy of source node and put in tooltip
            const sourceCopy = makeCopy(source);
            tooltipContent.appendChild(sourceCopy);

            // attach mouse event listeners
            tooltip.addEventListener('click', onTooltipClick);
            tooltip.addEventListener('mousedown', onTooltipClick);
            tooltip.addEventListener('touchstart', onTooltipClick);
            tooltip.addEventListener('mouseleave', onTooltipUnhover);

            // (for interaction with lightbox plugin)
            // transfer click on tooltip copied img to original img
            const sourceImg = source.querySelector('img');
            const sourceCopyImg = sourceCopy.querySelector('img');
            if (sourceImg && sourceCopyImg) {
                const clickImg = function() {
                    sourceImg.click();
                    closeTooltip();
                };
                sourceCopyImg.addEventListener('click', clickImg);
            }

            return tooltip;
        }

        // make carbon copy of html dom element
        function makeCopy(source) {
            const sourceCopy = source.cloneNode(true);

            // delete elements marked with ignore (eg anchor and jump buttons)
            const deleteFromCopy = sourceCopy.querySelectorAll(
                '[data-ignore="true"]'
            );
            for (const element of deleteFromCopy)
                element.remove();

            // delete certain element attributes
            const attributes = [
                'id',
                'data-collapsed',
                'data-selected',
                'data-highlighted',
                'data-glow'
            ];
            for (const attribute of attributes) {
                sourceCopy.removeAttribute(attribute);
                const elements = sourceCopy.querySelectorAll(
                    '[' + attribute + ']'
                );
                for (const element of elements)
                    element.removeAttribute(attribute);
            }

            return sourceCopy;
        }

        // when tooltip is clicked
        function onTooltipClick(event) {
            // when user clicks on tooltip, stop click from transferring
            // outside of tooltip (eg, click off to close tooltip, or eg click
            // off to unhighlight same refs)
            event.stopPropagation();
        }

        // when tooltip is unhovered
        function onTooltipUnhover(event) {
            if (options.clickClose === 'true')
                return;

            // make sure new mouse/touch/focus no longer over tooltip or any
            // element within it
            const tooltip = document.getElementById('tooltip');
            if (!tooltip)
                return;
            if (this.contains(event.relatedTarget))
                return;

            closeTooltip();
        }

        // make nav bar to go betwen prev/next instances of same reference
        function makeNavBar(link) {
            // find other links to the same source
            const sameLinks = getSameLinks(link);

            // don't show nav bar when singular reference
            if (sameLinks.total <= 1)
                return;

            // find prev/next links with same target
            const prevLink = getPrevLink(link, sameLinks);
            const nextLink = getNextLink(link, sameLinks);

            // create nav bar
            const navBar = document.createElement('div');
            navBar.id = 'tooltip_nav_bar';
            const text = sameLinks.index + 1 + ' of ' + sameLinks.total;

            // create nav bar prev/next buttons
            const prevButton = document.createElement('button');
            const nextButton = document.createElement('button');
            prevButton.id = 'tooltip_prev_button';
            nextButton.id = 'tooltip_next_button';
            prevButton.title =
                'Jump to the previous occurence of this item in the document [←]';
            nextButton.title =
                'Jump to the next occurence of this item in the document [→]';
            prevButton.classList.add('icon_button');
            nextButton.classList.add('icon_button');
            prevButton.innerHTML = document.querySelector(
                '.icon_caret_left'
            ).innerHTML;
            nextButton.innerHTML = document.querySelector(
                '.icon_caret_right'
            ).innerHTML;
            navBar.appendChild(prevButton);
            navBar.appendChild(document.createTextNode(text));
            navBar.appendChild(nextButton);

            // attach click listeners to buttons
            prevButton.addEventListener('click', function() {
                onPrevNextClick(link, prevLink);
            });
            nextButton.addEventListener('click', function() {
                onPrevNextClick(link, nextLink);
            });

            return navBar;
        }

        // get previous link with same target
        function getPrevLink(link, sameLinks) {
            if (!sameLinks)
                sameLinks = getSameLinks(link);
            // wrap index to other side if < 1
            let index;
            if (sameLinks.index - 1 >= 0)
                index = sameLinks.index - 1;
            else
                index = sameLinks.total - 1;
            return sameLinks.elements[index];
        }

        // get next link with same target
        function getNextLink(link, sameLinks) {
            if (!sameLinks)
                sameLinks = getSameLinks(link);
            // wrap index to other side if > total
            let index;
            if (sameLinks.index + 1 <= sameLinks.total - 1)
                index = sameLinks.index + 1;
            else
                index = 0;
            return sameLinks.elements[index];
        }

        // get element that is target of link or url hash
        function getSource(link) {
            const hash = link ? link.hash : window.location.hash;
            const id = hash.slice(1);
            let target = document.querySelector('[id="' + id + '"]');
            if (!target)
                return;

            // if ref or figure, modify target to get expected element
            if (id.indexOf('ref-') === 0)
                target = target.querySelector('p');
            else if (id.indexOf('fig:') === 0)
                target = target.querySelector('figure');

            return target;
        }

        // when prev/next arrow button is clicked
        function onPrevNextClick(link, prevNextLink) {
            if (link && prevNextLink)
                goToElement(prevNextLink, window.innerHeight * 0.5);
        }

        // scroll to and focus element
        function goToElement(element, offset) {
            // expand accordion section if collapsed
            expandElement(element);
            const y =
                getRectInView(element).top -
                getRectInView(document.documentElement).top -
                (offset || 0);
            // trigger any function listening for "onscroll" event
            window.dispatchEvent(new Event('scroll'));
            window.scrollTo(0, y);
            document.activeElement.blur();
            element.focus();
        }

        // determine position to place tooltip based on link position in
        // viewport and tooltip size
        function positionTooltip(link, left, top) {
            const tooltipElement = document.getElementById('tooltip');
            if (!tooltipElement)
                return;

            // get convenient vars for position/dimensions of
            // link/tooltip/page/view
            link = getRectInPage(link);
            const tooltip = getRectInPage(tooltipElement);
            const view = getRectInPage();

            // horizontal positioning
            if (left)
                // use explicit value
                left = left;
            else if (link.left + tooltip.width < view.right)
                // fit tooltip to right of link
                left = link.left;
            else if (link.right - tooltip.width > view.left)
                // fit tooltip to left of link
                left = link.right - tooltip.width;
            // center tooltip in view
            else
                left = (view.right - view.left) / 2 - tooltip.width / 2;

            // vertical positioning
            if (top)
                // use explicit value
                top = top;
            else if (link.top - tooltip.height > view.top)
                // fit tooltip above link
                top = link.top - tooltip.height;
            else if (link.bottom + tooltip.height < view.bottom)
                // fit tooltip below link
                top = link.bottom;
            else {
                // center tooltip in view
                top = view.top + view.height / 2 - tooltip.height / 2;
                // nudge off of link to left/right if possible
                if (link.right + tooltip.width < view.right)
                    left = link.right;
                else if (link.left - tooltip.width > view.left)
                    left = link.left - tooltip.width;
            }

            tooltipElement.style.left = left + 'px';
            tooltipElement.style.top = top + 'px';
        }

        // get position/dimensions of element or viewport
        function getRectInView(element) {
            let rect = {};
            rect.left = 0;
            rect.top = 0;
            rect.right = document.documentElement.clientWidth;
            rect.bottom = document.documentElement.clientHeight;
            let style = {};

            if (element instanceof HTMLElement) {
                rect = element.getBoundingClientRect();
                style = window.getComputedStyle(element);
            }

            const margin = {};
            margin.left = parseFloat(style.marginLeftWidth) || 0;
            margin.top = parseFloat(style.marginTopWidth) || 0;
            margin.right = parseFloat(style.marginRightWidth) || 0;
            margin.bottom = parseFloat(style.marginBottomWidth) || 0;

            const border = {};
            border.left = parseFloat(style.borderLeftWidth) || 0;
            border.top = parseFloat(style.borderTopWidth) || 0;
            border.right = parseFloat(style.borderRightWidth) || 0;
            border.bottom = parseFloat(style.borderBottomWidth) || 0;

            const newRect = {};
            newRect.left = rect.left + margin.left + border.left;
            newRect.top = rect.top + margin.top + border.top;
            newRect.right = rect.right + margin.right + border.right;
            newRect.bottom = rect.bottom + margin.bottom + border.bottom;
            newRect.width = newRect.right - newRect.left;
            newRect.height = newRect.bottom - newRect.top;

            return newRect;
        }

        // get position of element relative to page
        function getRectInPage(element) {
            const rect = getRectInView(element);
            const body = getRectInView(document.body);

            const newRect = {};
            newRect.left = rect.left - body.left;
            newRect.top = rect.top - body.top;
            newRect.right = rect.right - body.left;
            newRect.bottom = rect.bottom - body.top;
            newRect.width = rect.width;
            newRect.height = rect.height;

            return newRect;
        }

        // (for interaction with accordion plugin)
        // get closest element before specified element that matches query
        function firstBefore(element, query) {
            while (
                element &&
                element !== document.body &&
                !element.matches(query)
            )
                element = element.previousElementSibling || element.parentNode;

            return element;
        }

        // (for interaction with accordion plugin)
        // check if element is part of collapsed heading
        function isCollapsed(element) {
            while (element && element !== document.body) {
                if (element.dataset.collapsed === 'true')
                    return true;
                element = element.parentNode;
            }
            return false;
        }

        // (for interaction with accordion plugin)
        // expand heading containing element if necesary
        function expandElement(element) {
            if (isCollapsed(element)) {
                const heading = firstBefore(element, 'h2');
                if (heading)
                    heading.click();
            }
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>

<!-- caret left icon -->

<template class="icon_caret_left">
    <!-- modified from: https://fontawesome.com/icons/caret-left -->
    <svg width="16" height="16" viewBox="0 0 192 512">
        <path
            fill="currentColor"
            d="M192 127.338v257.324c0 17.818-21.543 26.741-34.142 14.142L29.196 270.142c-7.81-7.81-7.81-20.474 0-28.284l128.662-128.662c12.599-12.6 34.142-3.676 34.142 14.142z"
        ></path>
    </svg>
</template>

<!-- caret right icon -->

<template class="icon_caret_right">
    <!-- modified from: https://fontawesome.com/icons/caret-right -->
    <svg width="16" height="16" viewBox="0 0 192 512">
        <path
            fill="currentColor"
            d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
        ></path>
    </svg>
</template>
<!-- jump to first plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin adds a button next to each reference entry,
        // figure, and table that jumps the page to the first occurrence of a
        // link to that item in the manuscript.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'jumpToFirst';

        // default plugin options
        const options = {
            // whether to add buttons next to reference entries
            references: 'true',
            // whether to add buttons next to figures
            figures: 'true',
            // whether to add buttons next to tables
            tables: 'true',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            if (options.references !== 'false')
                makeReferenceButtons();
            if (options.figures !== 'false')
                makeFigureButtons();
            if (options.tables !== 'false')
                makeTableButtons();
        }

        // when jump button clicked
        function onButtonClick() {
            const first = getFirstOccurrence(this.dataset.id);
            if (!first)
                return;

            // update url hash so navigating "back" in history will return
            // user to jump button
            window.location.hash = this.dataset.id;
            // scroll to link
            window.setTimeout(function() {
                goToElement(first, window.innerHeight * 0.5);
            }, 0);
        }

        // get first occurence of link to item in document
        function getFirstOccurrence(id) {
            let query = 'a';
            query += '[href="#' + id + '"]';
            // exclude buttons, anchor links, toc links, etc
            query +=
                ':not(.button):not(.icon_button):not(.anchor):not(.toc_link)';
            return document.querySelector(query);
        }

        // add button next to each reference entry
        function makeReferenceButtons() {
            const references = document.querySelectorAll('div[id^="ref-"]');
            for (const reference of references) {
                // get reference id and element to add button to
                const id = reference.id;
                const container = reference.firstElementChild;
                const first = getFirstOccurrence(id);

                // if can't find link to reference, ignore
                if (!first)
                    continue;

                // make jump button
                let button = document.createElement('button');
                button.classList.add('icon_button', 'jump_arrow');
                button.title =
                    'Jump to the first occurence of this reference in the document';
                button.innerHTML = document.querySelector(
                    '.icon_angle_double_up'
                ).innerHTML;
                button.dataset.id = id;
                button.dataset.ignore = 'true';
                container.innerHTML = button.outerHTML + container.innerHTML;
                button = container.firstElementChild;
                button.addEventListener('click', onButtonClick);
            }
        }

        // add button next to each figure
        function makeFigureButtons() {
            const figures = document.querySelectorAll('[id^="fig:"]');
            for (const figure of figures) {
                // get figure id and element to add button to
                const id = figure.id;
                const container = figure.querySelector('figcaption') || figure;
                const first = getFirstOccurrence(id);

                // if can't find link to figure, ignore
                if (!first)
                    continue;

                // make jump button
                const button = document.createElement('button');
                button.classList.add('icon_button', 'jump_arrow');
                button.title =
                    'Jump to the first occurence of this figure in the document';
                button.innerHTML = document.querySelector(
                    '.icon_angle_double_up'
                ).innerHTML;
                button.dataset.id = id;
                button.dataset.ignore = 'true';
                container.insertBefore(button, container.firstElementChild);
                button.addEventListener('click', onButtonClick);
            }
        }

        // add button next to each figure
        function makeTableButtons() {
            const tables = document.querySelectorAll('[id^="tbl:"]');
            for (const table of tables) {
                // get ref id and element to add button to
                const id = table.id;
                const container = table.querySelector('caption') || table;
                const first = getFirstOccurrence(id);

                // if can't find link to table, ignore
                if (!first)
                    continue;

                // make jump button
                const button = document.createElement('button');
                button.classList.add('icon_button', 'jump_arrow');
                button.title =
                    'Jump to the first occurence of this table in the document';
                button.innerHTML = document.querySelector(
                    '.icon_angle_double_up'
                ).innerHTML;
                button.dataset.id = id;
                button.dataset.ignore = 'true';
                container.insertBefore(button, container.firstElementChild);
                button.addEventListener('click', onButtonClick);
            }
        }

        // scroll to and focus element
        function goToElement(element, offset) {
            // expand accordion section if collapsed
            expandElement(element);
            const y =
                getRectInView(element).top -
                getRectInView(document.documentElement).top -
                (offset || 0);
            // trigger any function listening for "onscroll" event
            window.dispatchEvent(new Event('scroll'));
            window.scrollTo(0, y);
            document.activeElement.blur();
            element.focus();
        }

        // get position/dimensions of element or viewport
        function getRectInView(element) {
            let rect = {};
            rect.left = 0;
            rect.top = 0;
            rect.right = document.documentElement.clientWidth;
            rect.bottom = document.documentElement.clientHeight;
            let style = {};

            if (element instanceof HTMLElement) {
                rect = element.getBoundingClientRect();
                style = window.getComputedStyle(element);
            }

            const margin = {};
            margin.left = parseFloat(style.marginLeftWidth) || 0;
            margin.top = parseFloat(style.marginTopWidth) || 0;
            margin.right = parseFloat(style.marginRightWidth) || 0;
            margin.bottom = parseFloat(style.marginBottomWidth) || 0;

            const border = {};
            border.left = parseFloat(style.borderLeftWidth) || 0;
            border.top = parseFloat(style.borderTopWidth) || 0;
            border.right = parseFloat(style.borderRightWidth) || 0;
            border.bottom = parseFloat(style.borderBottomWidth) || 0;

            const newRect = {};
            newRect.left = rect.left + margin.left + border.left;
            newRect.top = rect.top + margin.top + border.top;
            newRect.right = rect.right + margin.right + border.right;
            newRect.bottom = rect.bottom + margin.bottom + border.bottom;
            newRect.width = newRect.right - newRect.left;
            newRect.height = newRect.bottom - newRect.top;

            return newRect;
        }

        // get closest element before specified element that matches query
        function firstBefore(element, query) {
            while (
                element &&
                element !== document.body &&
                !element.matches(query)
            )
                element = element.previousElementSibling || element.parentNode;

            return element;
        }

        // check if element is part of collapsed heading
        function isCollapsed(element) {
            while (element && element !== document.body) {
                if (element.dataset.collapsed === 'true')
                    return true;
                element = element.parentNode;
            }
            return false;
        }

        // (for interaction with accordion plugin)
        // expand heading containing element if necesary
        function expandElement(element) {
            if (isCollapsed(element)) {
                const heading = firstBefore(element, 'h2');
                if (heading)
                    heading.click();
            }
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>

<!-- angle double up icon -->

<template class="icon_angle_double_up">
    <!-- modified from: https://fontawesome.com/icons/angle-double-up -->
    <svg width="16" height="16" viewBox="0 0 320 512">
        <path
            fill="currentColor"
            d="M177 255.7l136 136c9.4 9.4 9.4 24.6 0 33.9l-22.6 22.6c-9.4 9.4-24.6 9.4-33.9 0L160 351.9l-96.4 96.4c-9.4 9.4-24.6 9.4-33.9 0L7 425.7c-9.4-9.4-9.4-24.6 0-33.9l136-136c9.4-9.5 24.6-9.5 34-.1zm-34-192L7 199.7c-9.4 9.4-9.4 24.6 0 33.9l22.6 22.6c9.4 9.4 24.6 9.4 33.9 0l96.4-96.4 96.4 96.4c9.4 9.4 24.6 9.4 33.9 0l22.6-22.6c9.4-9.4 9.4-24.6 0-33.9l-136-136c-9.2-9.4-24.4-9.4-33.8 0z"
        ></path>
    </svg>
</template>
<!-- link highlight plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin makes it such that when a user hovers or
        // focuses a link, other links that have the same target will be
        // highlighted. It also makes it such that when clicking a link, the
        // target of the link (eg reference, figure, table) is briefly
        // highlighted.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'linkHighlight';

        // default plugin options
        const options = {
            // whether to also highlight links that go to external urls
            externalLinks: 'false',
            // whether user must click off to unhighlight instead of just
            // un-hovering
            clickUnhighlight: 'false',
            // whether to also highlight links that are unique
            highlightUnique: 'true',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            const links = getLinks();
            for (const link of links) {
                // attach mouse and focus listeners to link
                link.addEventListener('mouseenter', onLinkFocus);
                link.addEventListener('focus', onLinkFocus);
                link.addEventListener('mouseleave', onLinkUnhover);
            }

            // attach click and hash change listeners to window
            window.addEventListener('click', onClick);
            window.addEventListener('touchstart', onClick);
            window.addEventListener('hashchange', onHashChange);

            // run hash change on window load in case user has navigated
            // directly to hash
            onHashChange();
        }

        // when link is focused (tabbed to) or hovered
        function onLinkFocus() {
            highlight(this);
        }

        // when link is unhovered
        function onLinkUnhover() {
            if (options.clickUnhighlight !== 'true')
                unhighlightAll();
        }

        // when the mouse is clicked anywhere in window
        function onClick(event) {
            unhighlightAll();
        }

        // when hash (eg manuscript.html#introduction) changes
        function onHashChange() {
            const target = getHashTarget();
            if (target)
                glowElement(target);
        }

        // get element that is target of link or url hash
        function getHashTarget(link) {
            const hash = link ? link.hash : window.location.hash;
            const id = hash.slice(1);
            let target = document.querySelector('[id="' + id + '"]');
            if (!target)
                return;

            return target;
        }

        // start glow sequence on an element
        function glowElement(element) {
            const startGlow = function() {
                onGlowEnd();
                element.dataset.glow = 'true';
                element.addEventListener('animationend', onGlowEnd);
            };
            const onGlowEnd = function() {
                element.removeAttribute('data-glow');
                element.removeEventListener('animationend', onGlowEnd);
            };
            startGlow();
        }

        // highlight link and all others with same target
        function highlight(link) {
            // force unhighlight all to start fresh
            unhighlightAll();

            // get links with same target
            if (!link)
                return;
            const sameLinks = getSameLinks(link);

            // if link unique and option is off, exit and don't highlight
            if (sameLinks.length <= 1 && options.highlightUnique !== 'true')
                return;

            // highlight all same links, and "select" (special highlight) this
            // one
            for (const sameLink of sameLinks) {
                if (sameLink === link)
                    sameLink.setAttribute('data-selected', 'true');
                else
                    sameLink.setAttribute('data-highlighted', 'true');
            }
        }

        // unhighlight all links
        function unhighlightAll() {
            const links = getLinks();
            for (const link of links) {
                link.setAttribute('data-selected', 'false');
                link.setAttribute('data-highlighted', 'false');
            }
        }

        // get links with same target
        function getSameLinks(link) {
            const results = [];
            const links = getLinks();
            for (const otherLink of links) {
                if (
                    otherLink.getAttribute('href') === link.getAttribute('href')
                )
                    results.push(otherLink);
            }
            return results;
        }

        // get all links of types we wish to handle
        function getLinks() {
            let query = 'a';
            if (options.externalLinks !== 'true')
                query += '[href^="#"]';
            // exclude buttons, anchor links, toc links, etc
            query +=
                ':not(.button):not(.icon_button):not(.anchor):not(.toc_link)';
            return document.querySelectorAll(query);
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>
<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin provides a "table of contents" (toc) panel on
        // the side of the document that allows the user to conveniently
        // navigate between sections of the document.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'tableOfContents';

        // default plugin options
        const options = {
            // which types of elements to add links for, in
            // "document.querySelector" format
            typesQuery: 'h1, h2, h3',
            // whether toc starts open. use 'true' or 'false', or 'auto' to
            // use 'true' behavior when screen wide enough and 'false' when not
            startOpen: 'false',
            // whether toc closes when clicking on toc link. use 'true' or
            // 'false', or 'auto' to use 'false' behavior when screen wide
            // enough and 'true' when not
            clickClose: 'auto',
            // if list item is more than this many characters, text will be
            // truncated
            charLimit: '50',
            // whether or not to show bullets next to each toc item
            bullets: 'false',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            // make toc panel and populate with entries (links to document
            // sections)
            const panel = makePanel();
            if (!panel)
                return;
            makeEntries(panel);
            // attach panel to document after making entries, so 'toc' heading
            // in panel isn't included in toc
            document.body.insertBefore(panel, document.body.firstChild);

            // initial panel state
            if (
                options.startOpen === 'true' ||
                (options.startOpen === 'auto' && !isSmallScreen())
            )
                openPanel();
            else
                closePanel();

            // attach click, scroll, and hash change listeners to window
            window.addEventListener('click', onClick);
            window.addEventListener('scroll', onScroll);
            window.addEventListener('hashchange', onScroll);
            window.addEventListener('keyup', onKeyUp);
            onScroll();

            // add class to push document body down out of way of toc button
            document.body.classList.add('toc_body_nudge');
        }

        // determine if screen wide enough to fit toc panel
        function isSmallScreen() {
            // in default theme:
            // 816px = 8.5in = width of "page" (<body>) element
            // 260px = min width of toc panel (*2 for both sides of <body>)
            return window.innerWidth < 816 + 260 * 2;
        }

        // when mouse is clicked anywhere in window
        function onClick() {
            if (isSmallScreen())
                closePanel();
        }

        // when window is scrolled or hash changed
        function onScroll() {
            highlightViewed();
        }

        // when key pressed
        function onKeyUp(event) {
            if (!event || !event.key)
                return;

            // close on esc
            if (event.key === 'Escape')
                closePanel();
        }

        // find entry of currently viewed document section in toc and highlight
        function highlightViewed() {
            const firstId = getFirstInView(options.typesQuery);

            // get toc entries (links), unhighlight all, then highlight viewed
            const list = document.getElementById('toc_list');
            if (!firstId || !list)
                return;
            const links = list.querySelectorAll('a');
            for (const link of links)
                link.dataset.viewing = 'false';
            const link = list.querySelector('a[href="#' + firstId + '"]');
            if (!link)
                return;
            link.dataset.viewing = 'true';
        }

        // get first or previous toc listed element in top half of view
        function getFirstInView(query) {
            // get all elements matching query and with id
            const elements = document.querySelectorAll(query);
            const elementsWithIds = [];
            for (const element of elements) {
                if (element.id)
                    elementsWithIds.push(element);
            }


            // get first or previous element in top half of view
            for (let i = 0; i < elementsWithIds.length; i++) {
                const element = elementsWithIds[i];
                const prevElement = elementsWithIds[Math.max(0, i - 1)];
                if (element.getBoundingClientRect().top >= 0) {
                    if (
                        element.getBoundingClientRect().top <
                        window.innerHeight / 2
                    )
                        return element.id;
                    else
                        return prevElement.id;
                }
            }
        }

        // make panel
        function makePanel() {
            // create panel
            const panel = document.createElement('div');
            panel.id = 'toc_panel';
            if (options.bullets === 'true')
                panel.dataset.bullets = 'true';

            // create header
            const header = document.createElement('div');
            header.id = 'toc_header';

            // create toc button
            const button = document.createElement('button');
            button.id = 'toc_button';
            button.innerHTML = document.querySelector('.icon_th_list').innerHTML;
            button.title = 'Table of Contents';
            button.classList.add('icon_button');

            // create header text
            const text = document.createElement('h4');
            text.innerHTML = 'Table of Contents';

            // create container for toc list
            const list = document.createElement('div');
            list.id = 'toc_list';

            // attach click listeners
            panel.addEventListener('click', onPanelClick);
            header.addEventListener('click', onHeaderClick);
            button.addEventListener('click', onButtonClick);

            // attach elements
            header.appendChild(button);
            header.appendChild(text);
            panel.appendChild(header);
            panel.appendChild(list);

            return panel;
        }

        // create toc entries (links) to each element of the specified types
        function makeEntries(panel) {
            const elements = document.querySelectorAll(options.typesQuery);
            for (const element of elements) {
                // do not add link if element doesn't have assigned id
                if (!element.id)
                    continue;

                // create link/list item
                const link = document.createElement('a');
                link.classList.add('toc_link');
                switch (element.tagName.toLowerCase()) {
                    case 'h1':
                        link.dataset.level = '1';
                        break;
                    case 'h2':
                        link.dataset.level = '2';
                        break;
                    case 'h3':
                        link.dataset.level = '3';
                        break;
                    case 'h4':
                        link.dataset.level = '4';
                        break;
                }
                link.title = element.innerText;
                let text = element.innerText;
                if (text.length > options.charLimit)
                    text = text.slice(0, options.charLimit) + '...';
                link.innerHTML = text;
                link.href = '#' + element.id;
                link.addEventListener('click', onLinkClick);

                // attach link
                panel.querySelector('#toc_list').appendChild(link);
            }
        }

        // when panel is clicked
        function onPanelClick(event) {
            // stop click from propagating to window/document and closing panel
            event.stopPropagation();
        }

        // when header itself is clicked
        function onHeaderClick(event) {
            togglePanel();
        }

        // when button is clicked
        function onButtonClick(event) {
            togglePanel();
            // stop header underneath button from also being clicked
            event.stopPropagation();
        }

        // when link is clicked
        function onLinkClick(event) {
            if (
                options.clickClose === 'true' ||
                (options.clickClose === 'auto' && isSmallScreen())
            )
                closePanel();
            else
                openPanel();
        }

        // open panel if closed, close if opened
        function togglePanel() {
            const panel = document.getElementById('toc_panel');
            if (!panel)
                return;

            if (panel.dataset.open === 'true')
                closePanel();
            else
                openPanel();
        }

        // open panel
        function openPanel() {
            const panel = document.getElementById('toc_panel');
            if (panel)
                panel.dataset.open = 'true';
        }

        // close panel
        function closePanel() {
            const panel = document.getElementById('toc_panel');
            if (panel)
                panel.dataset.open = 'false';
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>

<!-- th list icon -->

<template class="icon_th_list">
    <!-- modified from: https://fontawesome.com/icons/th-list -->
    <svg width="16" height="16" viewBox="0 0 512 512" tabindex="-1">
        <path
            fill="currentColor"
            d="M96 96c0 26.51-21.49 48-48 48S0 122.51 0 96s21.49-48 48-48 48 21.49 48 48zM48 208c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm0 160c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm96-236h352c8.837 0 16-7.163 16-16V76c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16z"
            tabindex="-1"
        ></path>
    </svg>
</template>
<!-- lightbox plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin makes it such that when a user clicks on an
        // image, the image fills the screen and the user can pan/drag/zoom
        // the image and navigate between other images in the document.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'lightbox';

        // default plugin options
        const options = {
            // list of possible zoom/scale factors
            zoomSteps:
                '0.1, 0.25, 0.333333, 0.5, 0.666666, 0.75, 1,' +
                '1.25, 1.5, 1.75, 2, 2.5, 3, 3.5, 4, 5, 6, 7, 8',
            // whether to fit image to view ('fit'), display at 100% and shrink
            // if necessary ('shrink'), or always display at 100% ('100')
            defaultZoom: 'fit',
            // whether to zoom in/out toward center of view ('true') or mouse
            // ('false')
            centerZoom: 'false',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            // run through each <img> element
            const imgs = document.querySelectorAll('figure > img');
            let count = 1;
            for (const img of imgs) {
                img.classList.add('lightbox_document_img');
                img.dataset.number = count;
                img.dataset.total = imgs.length;
                img.addEventListener('click', openLightbox);
                count++;
            }

            // attach mouse and key listeners to window
            window.addEventListener('mousemove', onWindowMouseMove);
            window.addEventListener('keyup', onKeyUp);
        }

        // when mouse is moved anywhere in window
        function onWindowMouseMove(event) {
            window.mouseX = event.clientX;
            window.mouseY = event.clientY;
        }

        // when key pressed
        function onKeyUp(event) {
            if (!event || !event.key)
                return;

            switch (event.key) {
                // trigger click of prev button
                case 'ArrowLeft':
                    const prevButton = document.getElementById(
                        'lightbox_prev_button'
                    );
                    if (prevButton)
                        prevButton.click();
                    break;
                // trigger click of next button
                case 'ArrowRight':
                    const nextButton = document.getElementById(
                        'lightbox_next_button'
                    );
                    if (nextButton)
                        nextButton.click();
                    break;
                // close on esc
                case 'Escape':
                    closeLightbox();
                    break;
            }
        }

        // open lightbox
        function openLightbox() {
            const lightbox = makeLightbox(this);
            if (!lightbox)
                return;

            blurBody(lightbox);
            document.body.appendChild(lightbox);
        }

        // make lightbox
        function makeLightbox(img) {
            // delete lightbox if it exists, start fresh
            closeLightbox();

            // create screen overlay containing lightbox
            const overlay = document.createElement('div');
            overlay.id = 'lightbox_overlay';

            // create image info boxes
            const numberInfo = document.createElement('div');
            const zoomInfo = document.createElement('div');
            numberInfo.id = 'lightbox_number_info';
            zoomInfo.id = 'lightbox_zoom_info';

            // create container for image
            const imageContainer = document.createElement('div');
            imageContainer.id = 'lightbox_image_container';
            const lightboxImg = makeLightboxImg(
                img,
                imageContainer,
                numberInfo,
                zoomInfo
            );
            imageContainer.appendChild(lightboxImg);

            // create bottom container for caption and navigation buttons
            const bottomContainer = document.createElement('div');
            bottomContainer.id = 'lightbox_bottom_container';
            const caption = makeCaption(img);
            const prevButton = makePrevButton(img);
            const nextButton = makeNextButton(img);
            bottomContainer.appendChild(prevButton);
            bottomContainer.appendChild(caption);
            bottomContainer.appendChild(nextButton);

            // attach top middle and bottom to overlay
            overlay.appendChild(numberInfo);
            overlay.appendChild(zoomInfo);
            overlay.appendChild(imageContainer);
            overlay.appendChild(bottomContainer);

            return overlay;
        }

        // make <img> object that is intuitively draggable and zoomable
        function makeLightboxImg(
            sourceImg,
            container,
            numberInfoBox,
            zoomInfoBox
        ) {
            // create copy of source <img>
            const img = sourceImg.cloneNode(true);
            img.classList.remove('lightbox_document_img');
            img.removeAttribute('id');
            img.removeAttribute('width');
            img.removeAttribute('height');
            img.style.position = 'unset';
            img.style.margin = '0';
            img.style.padding = '0';
            img.style.width = '';
            img.style.height = '';
            img.style.minWidth = '';
            img.style.minHeight = '';
            img.style.maxWidth = '';
            img.style.maxHeight = '';
            img.id = 'lightbox_img';

            // build sorted list of unique zoomSteps, always including a 100%
            let zoomSteps = [];
            const optionsZooms = options.zoomSteps.split(/[^0-9.]/);
            for (const optionZoom of optionsZooms) {
                const newZoom = parseFloat(optionZoom);
                if (newZoom && !zoomSteps.includes(newZoom))
                    zoomSteps.push(newZoom);
            }
            if (!zoomSteps.includes(1))
                zoomSteps.push(1);
            zoomSteps = zoomSteps.sort(function sortNumber(a, b) {
                return a - b;
            });

            // <img> object property variables
            let zoom = 1;
            let translateX = 0;
            let translateY = 0;
            let clickMouseX = undefined;
            let clickMouseY = undefined;
            let clickTranslateX = undefined;
            let clickTranslateY = undefined;

            updateNumberInfo();

            // update image numbers displayed in info box
            function updateNumberInfo() {
                numberInfoBox.innerHTML =
                    sourceImg.dataset.number + ' of ' + sourceImg.dataset.total;
            }

            // update zoom displayed in info box
            function updateZoomInfo() {
                let zoomInfo = zoom * 100;
                if (!Number.isInteger(zoomInfo))
                    zoomInfo = zoomInfo.toFixed(2);
                zoomInfoBox.innerHTML = zoomInfo + '%';
            }

            // move to closest zoom step above current zoom
            const zoomIn = function() {
                for (const zoomStep of zoomSteps) {
                    if (zoomStep > zoom) {
                        zoom = zoomStep;
                        break;
                    }
                }
                updateTransform();
            };

            // move to closest zoom step above current zoom
            const zoomOut = function() {
                zoomSteps.reverse();
                for (const zoomStep of zoomSteps) {
                    if (zoomStep < zoom) {
                        zoom = zoomStep;
                        break;
                    }
                }
                zoomSteps.reverse();

                updateTransform();
            };

            // update display of <img> based on scale/translate properties
            const updateTransform = function() {
                // set transform
                img.style.transform =
                    'translate(' +
                    (translateX || 0) +
                    'px,' +
                    (translateY || 0) +
                    'px) scale(' +
                    (zoom || 1) +
                    ')';

                // get new width/height after scale
                const rect = img.getBoundingClientRect();
                // limit translate
                translateX = Math.max(translateX, -rect.width / 2);
                translateX = Math.min(translateX, rect.width / 2);
                translateY = Math.max(translateY, -rect.height / 2);
                translateY = Math.min(translateY, rect.height / 2);

                // set transform
                img.style.transform =
                    'translate(' +
                    (translateX || 0) +
                    'px,' +
                    (translateY || 0) +
                    'px) scale(' +
                    (zoom || 1) +
                    ')';

                updateZoomInfo();
            };

            // fit <img> to container
            const fit = function() {
                // no x/y offset, 100% zoom by default
                translateX = 0;
                translateY = 0;
                zoom = 1;

                // widths of <img> and container
                const imgWidth = img.naturalWidth;
                const imgHeight = img.naturalHeight;
                const containerWidth = parseFloat(
                    window.getComputedStyle(container).width
                );
                const containerHeight = parseFloat(
                    window.getComputedStyle(container).height
                );

                // how much zooming is needed to fit <img> to container
                const xRatio = imgWidth / containerWidth;
                const yRatio = imgHeight / containerHeight;
                const maxRatio = Math.max(xRatio, yRatio);
                const newZoom = 1 / maxRatio;

                // fit <img> to container according to option
                if (options.defaultZoom === 'shrink') {
                    if (maxRatio > 1)
                        zoom = newZoom;
                } else if (options.defaultZoom === 'fit')
                    zoom = newZoom;

                updateTransform();
            };

            // when mouse wheel is rolled anywhere in container
            const onContainerWheel = function(event) {
                if (!event)
                    return;

                // let ctrl + mouse wheel to zoom behave as normal
                if (event.ctrlKey)
                    return;

                // prevent normal scroll behavior
                event.preventDefault();
                event.stopPropagation();

                // point around which to scale img
                const viewRect = container.getBoundingClientRect();
                const viewX = (viewRect.left + viewRect.right) / 2;
                const viewY = (viewRect.top + viewRect.bottom) / 2;
                const originX = options.centerZoom === 'true' ? viewX : mouseX;
                const originY = options.centerZoom === 'true' ? viewY : mouseY;

                // get point on image under origin
                const oldRect = img.getBoundingClientRect();
                const oldPercentX = (originX - oldRect.left) / oldRect.width;
                const oldPercentY = (originY - oldRect.top) / oldRect.height;

                // increment/decrement zoom
                if (event.deltaY < 0)
                    zoomIn();
                if (event.deltaY > 0)
                    zoomOut();

                // get offset between previous image point and origin
                const newRect = img.getBoundingClientRect();
                const offsetX =
                    originX - (newRect.left + newRect.width * oldPercentX);
                const offsetY =
                    originY - (newRect.top + newRect.height * oldPercentY);

                // translate image to keep image point under origin
                translateX += offsetX;
                translateY += offsetY;

                // perform translate
                updateTransform();
            };

            // when container is clicked
            function onContainerClick(event) {
                // if container itself is target of click, and not other
                // element above it
                if (event.target === this)
                    closeLightbox();
            }

            // when mouse button is pressed on image
            const onImageMouseDown = function(event) {
                // store original mouse position relative to image
                clickMouseX = window.mouseX;
                clickMouseY = window.mouseY;
                clickTranslateX = translateX;
                clickTranslateY = translateY;
                event.stopPropagation();
                event.preventDefault();
            };

            // when mouse button is released anywhere in window
            const onWindowMouseUp = function(event) {
                // reset original mouse position
                clickMouseX = undefined;
                clickMouseY = undefined;
                clickTranslateX = undefined;
                clickTranslateY = undefined;

                // remove global listener if lightbox removed from document
                if (!document.body.contains(container))
                    window.removeEventListener('mouseup', onWindowMouseUp);
            };

            // when mouse is moved anywhere in window
            const onWindowMouseMove = function(event) {
                if (
                    clickMouseX === undefined ||
                    clickMouseY === undefined ||
                    clickTranslateX === undefined ||
                    clickTranslateY === undefined
                )
                    return;

                // offset image based on original and current mouse position
                translateX = clickTranslateX + window.mouseX - clickMouseX;
                translateY = clickTranslateY + window.mouseY - clickMouseY;
                updateTransform();
                event.preventDefault();

                // remove global listener if lightbox removed from document
                if (!document.body.contains(container))
                    window.removeEventListener('mousemove', onWindowMouseMove);
            };

            // when window is resized
            const onWindowResize = function(event) {
                fit();

                // remove global listener if lightbox removed from document
                if (!document.body.contains(container))
                    window.removeEventListener('resize', onWindowResize);
            };

            // attach the necessary event listeners
            img.addEventListener('dblclick', fit);
            img.addEventListener('mousedown', onImageMouseDown);
            container.addEventListener('wheel', onContainerWheel);
            container.addEventListener('mousedown', onContainerClick);
            container.addEventListener('touchstart', onContainerClick);
            window.addEventListener('mouseup', onWindowMouseUp);
            window.addEventListener('mousemove', onWindowMouseMove);
            window.addEventListener('resize', onWindowResize);

            // run fit() after lightbox atttached to document and <img> Loaded
            // so needed container and img dimensions available
            img.addEventListener('load', fit);

            return img;
        }

        // make caption
        function makeCaption(img) {
            const caption = document.createElement('div');
            caption.id = 'lightbox_caption';
            const captionSource = img.nextElementSibling;
            if (captionSource.tagName.toLowerCase() === 'figcaption') {
                const captionCopy = makeCopy(captionSource);
                caption.innerHTML = captionCopy.innerHTML;
            }

            caption.addEventListener('touchstart', function(event) {
                event.stopPropagation();
            });

            return caption;
        }

        // make carbon copy of html dom element
        function makeCopy(source) {
            const sourceCopy = source.cloneNode(true);

            // delete elements marked with ignore (eg anchor and jump buttons)
            const deleteFromCopy = sourceCopy.querySelectorAll(
                '[data-ignore="true"]'
            );
            for (const element of deleteFromCopy)
                element.remove();

            // delete certain element attributes
            const attributes = [
                'id',
                'data-collapsed',
                'data-selected',
                'data-highlighted',
                'data-glow'
            ];
            for (const attribute of attributes) {
                sourceCopy.removeAttribute(attribute);
                const elements = sourceCopy.querySelectorAll(
                    '[' + attribute + ']'
                );
                for (const element of elements)
                    element.removeAttribute(attribute);
            }

            return sourceCopy;
        }

        // make button to jump to previous image in document
        function makePrevButton(img) {
            const prevButton = document.createElement('button');
            prevButton.id = 'lightbox_prev_button';
            prevButton.title = 'Jump to the previous image in the document [←]';
            prevButton.classList.add('icon_button', 'lightbox_button');
            prevButton.innerHTML = document.querySelector(
                '.icon_caret_left'
            ).innerHTML;

            // attach click listeners to button
            prevButton.addEventListener('click', function() {
                getPrevImg(img).click();
            });

            return prevButton;
        }

        // make button to jump to next image in document
        function makeNextButton(img) {
            const nextButton = document.createElement('button');
            nextButton.id = 'lightbox_next_button';
            nextButton.title = 'Jump to the next image in the document [→]';
            nextButton.classList.add('icon_button', 'lightbox_button');
            nextButton.innerHTML = document.querySelector(
                '.icon_caret_right'
            ).innerHTML;

            // attach click listeners to button
            nextButton.addEventListener('click', function() {
                getNextImg(img).click();
            });

            return nextButton;
        }

        // get previous image in document
        function getPrevImg(img) {
            const imgs = document.querySelectorAll('.lightbox_document_img');

            // find index of provided img
            let index;
            for (index = 0; index < imgs.length; index++) {
                if (imgs[index] === img)
                    break;
            }


            // wrap index to other side if < 1
            if (index - 1 >= 0)
                index--;
            else
                index = imgs.length - 1;
            return imgs[index];
        }

        // get next image in document
        function getNextImg(img) {
            const imgs = document.querySelectorAll('.lightbox_document_img');

            // find index of provided img
            let index;
            for (index = 0; index < imgs.length; index++) {
                if (imgs[index] === img)
                    break;
            }


            // wrap index to other side if > total
            if (index + 1 <= imgs.length - 1)
                index++;
            else
                index = 0;
            return imgs[index];
        }

        // close lightbox
        function closeLightbox() {
            focusBody();

            const lightbox = document.getElementById('lightbox_overlay');
            if (lightbox)
                lightbox.remove();
        }

        // make all elements behind lightbox non-focusable
        function blurBody(overlay) {
            const all = document.querySelectorAll('*');
            for (const element of all)
                element.tabIndex = -1;
            document.body.classList.add('body_no_scroll');
        }

        // make all elements focusable again
        function focusBody() {
            const all = document.querySelectorAll('*');
            for (const element of all)
                element.removeAttribute('tabIndex');
            document.body.classList.remove('body_no_scroll');
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>

<!-- caret left icon -->

<template class="icon_caret_left">
    <!-- modified from: https://fontawesome.com/icons/caret-left -->
    <svg width="16" height="16" viewBox="0 0 192 512">
        <path
            fill="currentColor"
            d="M192 127.338v257.324c0 17.818-21.543 26.741-34.142 14.142L29.196 270.142c-7.81-7.81-7.81-20.474 0-28.284l128.662-128.662c12.599-12.6 34.142-3.676 34.142 14.142z"
        ></path>
    </svg>
</template>

<!-- caret right icon -->

<template class="icon_caret_right">
    <!-- modified from: https://fontawesome.com/icons/caret-right -->
    <svg width="16" height="16" viewBox="0 0 192 512">
        <path
            fill="currentColor"
            d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
        ></path>
    </svg>
</template>
<!-- attributes plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin allows arbitrary HTML attributes to be attached
        // to (almost) any element. Place an HTML comment inside or next to the
        // desired element in the format <!-- $attribute="value" -->

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'attributes';

        // default plugin options
        const options = {
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            // get list of comments in document
            const comments = findComments();

            for(const comment of comments)
                if (comment.parentElement)
                    addAttributes(
                        comment.parentElement,
                        comment.nodeValue.trim()
                    );
        }

        // add html attributes to specified element based on string of 
        // html attributes and values
        function addAttributes(element, text) {
            // regex's for finding attribute/value pairs in the format of
            // attribute="value" or attribute='value
            const regex2 = /\$([a-zA-Z\-]+)?=\"(.+?)\"/;
            const regex1 = /\$([a-zA-Z\-]+)?=\'(.+?)\'/;

            // loop through attribute/value pairs
            let match;
            while(match = text.match(regex2) || text.match(regex1)) {
                // get attribute and value from regex capture groups
                let attribute = match[1];
                let value = match[2];

                // remove from string
                text = text.substring(match.index + match[0].length);

                if (!attribute || !value)
                    break;

                // set attribute of parent element
                try {
                    element.setAttribute(attribute, value);
                } catch(error) {
                    console.log(error);
                }

                // special case for colspan
                if (attribute === 'colspan')
                    removeTableCells(element, value);
            }
        }

        // get list of comment elements in document
        function findComments() {
            const comments = [];

            // iterate over comment nodes in document
            function acceptNode(node) {
                return NodeFilter.FILTER_ACCEPT;
            }
            const iterator = document.createNodeIterator(
                document.body,
                NodeFilter.SHOW_COMMENT,
                acceptNode
            );
            let node;
            while(node = iterator.nextNode())
                comments.push(node);

            return comments;
        }

        // remove certain number of cells after specified cell
        function removeTableCells(cell, number) {
            number = parseInt(number);
            if (!number)
                return;

            // remove elements
            for(; number > 1; number--) {
                if (cell.nextElementSibling)
                    cell.nextElementSibling.remove();
            }
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>
<!-- mathjax plugin configuration -->

<script type="text/x-mathjax-config">
    MathJax.Hub.Config({
        "CommonHTML": { linebreaks: { automatic: true } },
        "HTML-CSS": { linebreaks: { automatic: true } },
        "SVG": { linebreaks: { automatic: true } },
        "fast-preview": { disabled: true }
  });
</script>

<!-- mathjax plugin -->

<script
    src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"
    integrity="sha512-0doc9hKxR3PYwso42RD1p5ySZpzzuDiOwMrdCEh2WdJZCjcmFKc/wEnL+z8fBQrnHoiNWbo+3fiGkOYXBdQp4A=="
    crossorigin="anonymous"
>
    // /////////////////////////
    // DESCRIPTION
    // /////////////////////////

    // This third-party plugin 'MathJax' allows the proper rendering of
    // math/equations written in LaTeX.

    // https://www.mathjax.org/
</script>
<!-- annotations plugin -->

<script>
    // /////////////////////////
    // DESCRIPTION
    // /////////////////////////

    // This third-party plugin 'Hypothesis' allows public annotation of the
    // manuscript.

    // https://web.hypothes.is/

    // plugin configuration
    window.hypothesisConfig = function() {
        return {
            branding: {
                accentColor: '#2196f3',
                appBackgroundColor: '#f8f8f8',
                ctaBackgroundColor: '#f8f8f8',
                ctaTextColor: '#000000',
                selectionFontFamily: 'Open Sans, Helvetica, sans serif',
                annotationFontFamily: 'Open Sans, Helvetica, sans serif'
            }
        };
    };

    // hypothesis client script
    const embed = 'https://hypothes.is/embed.js';
    // hypothesis annotation count query url
    const query = 'https://api.hypothes.is/api/search?limit=0&url='

    
    // start script
    function start() {
        const button = makeButton();
        document.body.insertBefore(button, document.body.firstChild);
        insertCount(button);
    }

    // make button
    function makeButton() {
        // create button
        const button = document.createElement('button');
        button.id = 'hypothesis_button';
        button.innerHTML = document.querySelector('.icon_hypothesis').innerHTML;
        button.title = 'Hypothesis annotations';
        button.classList.add('icon_button');

        function onClick(event) {
            onButtonClick(event, button);
        }

        // attach click listeners
        button.addEventListener('click', onClick);

        return button;
    }

    // insert annotations count
    async function insertCount(button) {
        // get annotation count from Hypothesis based on url
        let count = '-';
        try {
            const canonical = document.querySelector('link[rel="canonical"]');
            const location = window.location;
            const url = encodeURIComponent((canonical || location).href);
            const response = await fetch(query + url);
            const json = await response.json();
            count = json.total || '-';
        } catch(error) {
            console.log(error);
        }
        
        // put count into button
        const counter = document.createElement('span');
        counter.id = 'hypothesis_count';
        counter.innerHTML = count;
        button.title = 'View ' + count + ' Hypothesis annotations';
        button.append(counter);
    }

    // when button is clicked
    function onButtonClick(event, button) {
        const script = document.createElement('script');
        script.src = embed;
        document.body.append(script);
        button.remove();
    }

    window.addEventListener('load', start);
</script>

<!-- hypothesis icon -->

<template class="icon_hypothesis">
    <!-- modified from: https://simpleicons.org/icons/hypothesis.svg / https://git.io/Jf1VB -->
    <svg width="16" height="16" viewBox="0 0 24 24" tabindex="-1">
        <path
            fill="currentColor"
            d="M3.43 0C2.5 0 1.72 .768 1.72 1.72V18.86C1.72 19.8 2.5 20.57 3.43 20.57H9.38L12 24L14.62 20.57H20.57C21.5 20.57 22.29 19.8 22.29 18.86V1.72C22.29 .77 21.5 0 20.57 0H3.43M5.14 3.43H7.72V9.43S8.58 7.72 10.28 7.72C12 7.72 13.74 8.57 13.74 11.24V17.14H11.16V12C11.16 10.61 10.28 10.07 9.43 10.29C8.57 10.5 7.72 11.41 7.72 13.29V17.14H5.14V3.43M18 13.72C18.95 13.72 19.72 14.5 19.72 15.42A1.71 1.71 0 0 1 18 17.13A1.71 1.71 0 0 1 16.29 15.42C16.29 14.5 17.05 13.71 18 13.71Z"
            tabindex="-1"
        ></path>
    </svg>
</template>
<!-- analytics plugin -->

<!-- copy and paste code from Google Analytics or similar service here -->
</body>
</html>
